--- a/chrome/browser/ui/views/location_bar/location_bar_view.cc
+++ b/chrome/browser/ui/views/location_bar/location_bar_view.cc
@@ -323,8 +323,7 @@ SkColor LocationBarView::GetOpaqueBorder
 }
 
 int LocationBarView::GetBorderRadius() const {
-  return ChromeLayoutProvider::Get()->GetCornerRadiusMetric(
-      views::EMPHASIS_MAXIMUM, size());
+  return 8;
 }
 
 std::unique_ptr<views::Background> LocationBarView::CreateRoundRectBackground(
--- a/ui/views/controls/highlight_path_generator.cc
+++ b/ui/views/controls/highlight_path_generator.cc
@@ -101,8 +101,7 @@ void InstallCircleHighlightPathGenerator
 
 SkPath PillHighlightPathGenerator::GetHighlightPath(const View* view) {
   const SkRect rect = gfx::RectToSkRect(view->GetLocalBounds());
-  const SkScalar corner_radius =
-      SkScalarHalf(std::min(rect.width(), rect.height()));
+  const SkScalar corner_radius = 8;
 
   return SkPath().addRoundRect(gfx::RectToSkRect(view->GetLocalBounds()),
                                corner_radius, corner_radius);
