--- a/chrome/browser/renderer_context_menu/render_view_context_menu.cc
+++ b/chrome/browser/renderer_context_menu/render_view_context_menu.cc
@@ -879,14 +879,14 @@ void RenderViewContextMenu::InitMenu() {
   extension_items_.set_smart_text_selection_enabled(
       supports_smart_text_selection);
 
-  if (content_type_->SupportsGroup(
+  if (false && content_type_->SupportsGroup(
           ContextMenuContentType::ITEM_GROUP_ALL_EXTENSION)) {
     DCHECK(!content_type_->SupportsGroup(
                ContextMenuContentType::ITEM_GROUP_CURRENT_EXTENSION));
     AppendAllExtensionItems();
   }
 
-  if (content_type_->SupportsGroup(
+  if (false && content_type_->SupportsGroup(
           ContextMenuContentType::ITEM_GROUP_CURRENT_EXTENSION)) {
     DCHECK(!content_type_->SupportsGroup(
                ContextMenuContentType::ITEM_GROUP_ALL_EXTENSION));
@@ -1214,9 +1214,9 @@ void RenderViewContextMenu::AppendLinkIt
     }
 #endif  // !defined(OS_CHROMEOS)
 
-    menu_model_.AddSeparator(ui::NORMAL_SEPARATOR);
+    // menu_model_.AddSeparator(ui::NORMAL_SEPARATOR);
 
-    if (browser && send_tab_to_self::ShouldOfferFeatureForLink(
+    if (false && browser && send_tab_to_self::ShouldOfferFeatureForLink(
                        active_web_contents, params_.link_url)) {
       if (send_tab_to_self::GetValidDeviceCount(GetBrowser()->profile()) == 1) {
 #if defined(OS_MACOSX)
@@ -1451,7 +1451,7 @@ void RenderViewContextMenu::AppendPageIt
 
   // Send-Tab-To-Self (user's other devices), page level.
   bool send_tab_to_self_menu_present = false;
-  if (GetBrowser() &&
+  if (false && GetBrowser() &&
       send_tab_to_self::ShouldOfferFeature(
           GetBrowser()->tab_strip_model()->GetActiveWebContents())) {
     menu_model_.AddSeparator(ui::NORMAL_SEPARATOR);
@@ -1515,7 +1515,7 @@ void RenderViewContextMenu::AppendPageIt
     menu_model_.AddSeparator(ui::NORMAL_SEPARATOR);
   }
 
-  if (TranslateService::IsTranslatableURL(params_.page_url)) {
+  if (false && TranslateService::IsTranslatableURL(params_.page_url)) {
     std::unique_ptr<translate::TranslatePrefs> prefs(
         ChromeTranslateClient::CreateTranslatePrefs(
             GetPrefs(browser_context_)));
@@ -1568,6 +1568,7 @@ void RenderViewContextMenu::AppendPrintI
 }
 
 void RenderViewContextMenu::AppendMediaRouterItem() {
+  return;
   if (media_router::MediaRouterEnabled(browser_context_)) {
     menu_model_.AddItemWithStringId(IDC_ROUTE_MEDIA,
                                     IDS_MEDIA_ROUTER_MENU_ITEM_TITLE);
