diff --git a/ash/system/bluetooth/bluetooth_detailed_view.cc b/ash/system/bluetooth/bluetooth_detailed_view.cc
index 778175a27abb..7010656af235 100644
--- a/ash/system/bluetooth/bluetooth_detailed_view.cc
+++ b/ash/system/bluetooth/bluetooth_detailed_view.cc
@@ -13,7 +13,6 @@
 #include "ash/shell.h"
 #include "ash/strings/grit/ash_strings.h"
 #include "ash/style/ash_color_provider.h"
-#include "ash/system/machine_learning/user_settings_event_logger.h"
 #include "ash/system/model/system_tray_model.h"
 #include "ash/system/tray/hover_highlight_view.h"
 #include "ash/system/tray/tray_info_label.h"
@@ -107,13 +106,6 @@ views::View* CreateDisabledPanel() {
   return container;
 }
 
-void LogUserBluetoothEvent(const BluetoothAddress& device_address) {
-  ml::UserSettingsEventLogger* logger = ml::UserSettingsEventLogger::Get();
-  if (logger) {
-    logger->LogBluetoothUkmEvent(device_address);
-  }
-}
-
 HoverHighlightView* GetScrollListItemForDevice(
     const std::unordered_map<HoverHighlightView*, BluetoothAddress>& device_map,
     const BluetoothAddress& address) {
@@ -401,7 +393,6 @@ void BluetoothDetailedView::HandleViewClicked(views::View* view) {
     return;
 
   UpdateClickedDevice(device_address, container);
-  LogUserBluetoothEvent(device_address);
   helper->ConnectToBluetoothDevice(device_address);
 }
 
