diff --git a/ash/BUILD.gn b/ash/BUILD.gn
index 22de1ef05493..0be0c6c96eb3 100644
--- a/ash/BUILD.gn
+++ b/ash/BUILD.gn
@@ -2456,7 +2456,6 @@ test("ash_unittests") {
     "//components/quirks",
     "//components/services/app_service/public/cpp:app_update",
     "//components/sync_preferences:test_support",
-    "//components/ukm:test_support",
     "//components/user_manager",
     "//components/user_manager:test_support",
     "//device/bluetooth",
