diff --git a/ash/BUILD.gn b/ash/BUILD.gn
index bf0b085268a3..cbdde5d04719 100644
--- a/ash/BUILD.gn
+++ b/ash/BUILD.gn
@@ -999,7 +999,6 @@ component("ash") {
     "system/dark_mode/dark_mode_detailed_view_controller.h",
     "system/dark_mode/dark_mode_feature_pod_controller.cc",
     "system/dark_mode/dark_mode_feature_pod_controller.h",
-    "system/enterprise/enterprise_domain_observer.h",
     "system/gesture_education/gesture_education_notification_controller.cc",
     "system/gesture_education/gesture_education_notification_controller.h",
     "system/holding_space/downloads_section.cc",
@@ -1117,8 +1116,6 @@ component("ash") {
     "system/model/clock_model.cc",
     "system/model/clock_model.h",
     "system/model/clock_observer.h",
-    "system/model/enterprise_domain_model.cc",
-    "system/model/enterprise_domain_model.h",
     "system/model/locale_model.cc",
     "system/model/locale_model.h",
     "system/model/session_length_limit_model.cc",
@@ -2488,7 +2485,6 @@ test("ash_unittests") {
     "//components/services/app_service/public/cpp:app_update",
     "//components/session_manager/core",
     "//components/sync_preferences:test_support",
-    "//components/ukm:test_support",
     "//components/user_manager",
     "//components/user_manager:test_support",
     "//device/bluetooth",
