diff --git a/ash/shell.cc b/ash/shell.cc
index 642dcb5f94a8..20c4f0660822 100644
--- a/ash/shell.cc
+++ b/ash/shell.cc
@@ -113,7 +113,6 @@
 #include "ash/system/keyboard_brightness/keyboard_brightness_controller.h"
 #include "ash/system/keyboard_brightness_control_delegate.h"
 #include "ash/system/locale/locale_update_controller_impl.h"
-#include "ash/system/machine_learning/user_settings_event_logger.h"
 #include "ash/system/message_center/message_center_controller.h"
 #include "ash/system/model/system_tray_model.h"
 #include "ash/system/model/virtual_keyboard_model.h"
@@ -696,10 +695,6 @@ Shell::~Shell() {
   // since it might be accessed during this process.
   tablet_mode_controller_->Shutdown();
 
-  // Destroy UserSettingsEventLogger before |system_tray_model_| and
-  // |video_detector_| which it observes.
-  ml::UserSettingsEventLogger::DeleteInstance();
-
   toast_manager_.reset();
 
   tray_bluetooth_helper_.reset();
@@ -1239,10 +1234,6 @@ void Shell::Init(
   frame_throttling_controller_ =
       std::make_unique<FrameThrottlingController>(context_factory);
 
-  // Create UserSettingsEventLogger after |system_tray_model_| and
-  // |video_detector_| which it observes.
-  ml::UserSettingsEventLogger::CreateInstance();
-
   // The compositor thread and main message loop have to be running in
   // order to create mirror window. Run it after the main message loop
   // is started.
