diff --git a/ash/BUILD.gn b/ash/BUILD.gn
index 9552865cf561..2bf6d5797527 100644
--- a/ash/BUILD.gn
+++ b/ash/BUILD.gn
@@ -1765,7 +1765,6 @@ component("ash") {
     "//net",
     "//services/content/public/cpp",
     "//services/data_decoder/public/cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/cpp:cpp",
     "//services/preferences/public/cpp",
 
@@ -2338,7 +2337,6 @@ test("ash_unittests") {
     "//components/quirks",
     "//components/services/app_service/public/cpp:app_update",
     "//components/sync_preferences:test_support",
-    "//components/ukm:test_support",
     "//components/user_manager",
     "//components/user_manager:test_support",
     "//device/bluetooth",
@@ -2351,7 +2349,6 @@ test("ash_unittests") {
     "//services/device/public/cpp/bluetooth",
     "//services/media_session/public/cpp/test:test_support",
     "//services/media_session/public/mojom",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network:test_support",
     "//skia",
     "//testing/gmock",
