diff --git a/ash/keyboard/ui/BUILD.gn b/ash/keyboard/ui/BUILD.gn
index 9ab4d1a74375..8791aa2c24a9 100644
--- a/ash/keyboard/ui/BUILD.gn
+++ b/ash/keyboard/ui/BUILD.gn
@@ -153,7 +153,6 @@ test("keyboard_unittests") {
     "//ash/public/cpp",
     "//base",
     "//base/test:test_support",
-    "//components/ukm:test_support",
     "//mojo/core/embedder",
     "//testing/gmock",
     "//testing/gtest",
