diff --git a/ash/media/media_notification_controller_impl.h b/ash/media/media_notification_controller_impl.h
index 6170360123eb..c962e35f509e 100644
--- a/ash/media/media_notification_controller_impl.h
+++ b/ash/media/media_notification_controller_impl.h
@@ -50,9 +50,6 @@ class ASH_EXPORT MediaNotificationControllerImpl
   void HideNotification(const std::string& id) override;
   void RemoveItem(const std::string& id) override;
   scoped_refptr<base::SequencedTaskRunner> GetTaskRunner() const override;
-  void LogMediaSessionActionButtonPressed(
-      const std::string& id,
-      media_session::mojom::MediaSessionAction action) override {}
 
   std::unique_ptr<MediaNotificationContainerImpl> CreateMediaNotification(
       const message_center::Notification& notification);
