diff --git a/ash/BUILD.gn b/ash/BUILD.gn
index f36ff2c8304b..8ce1048b4b59 100644
--- a/ash/BUILD.gn
+++ b/ash/BUILD.gn
@@ -1724,7 +1724,6 @@ component("ash") {
     "//net",
     "//services/content/public/cpp",
     "//services/data_decoder/public/cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/cpp:cpp",
     "//services/preferences/public/cpp",
 
@@ -2288,7 +2287,6 @@ test("ash_unittests") {
     "//components/quirks",
     "//components/services/app_service/public/cpp:app_update",
     "//components/sync_preferences:test_support",
-    "//components/ukm:test_support",
     "//components/user_manager",
     "//components/user_manager:test_support",
     "//device/bluetooth",
@@ -2301,7 +2299,6 @@ test("ash_unittests") {
     "//services/device/public/cpp/bluetooth",
     "//services/media_session/public/cpp/test:test_support",
     "//services/media_session/public/mojom",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network:test_support",
     "//skia",
     "//testing/gmock",
