diff --git a/ash/system/message_center/notifier_settings_view.cc b/ash/system/message_center/notifier_settings_view.cc
index ca926c5e82d8..d11f7c36a326 100644
--- a/ash/system/message_center/notifier_settings_view.cc
+++ b/ash/system/message_center/notifier_settings_view.cc
@@ -18,7 +18,6 @@
 #include "ash/shell.h"
 #include "ash/strings/grit/ash_strings.h"
 #include "ash/style/ash_color_provider.h"
-#include "ash/system/machine_learning/user_settings_event_logger.h"
 #include "ash/system/message_center/message_center_controller.h"
 #include "ash/system/message_center/message_center_style.h"
 #include "ash/system/tray/tray_popup_utils.h"
@@ -102,13 +101,6 @@ constexpr gfx::Insets kToggleButtonRowLabelPadding(16, 0, 15, 0);
 constexpr SkColor kTopBorderColor = SkColorSetA(SK_ColorBLACK, 0x1F);
 const int kLabelFontSizeDelta = 1;
 
-void LogUserQuietModeEvent(const bool enabled) {
-  auto* logger = ml::UserSettingsEventLogger::Get();
-  if (logger) {
-    logger->LogQuietModeUkmEvent(enabled);
-  }
-}
-
 // NotifierButtonWrapperView ---------------------------------------------------
 
 // A wrapper view of NotifierButton to guarantee the fixed height
@@ -714,7 +706,6 @@ void NotifierSettingsView::AppBadgingTogglePressed() {
 }
 
 void NotifierSettingsView::QuietModeTogglePressed() {
-  LogUserQuietModeEvent(quiet_mode_toggle_->GetIsOn());
   MessageCenter::Get()->SetQuietMode(quiet_mode_toggle_->GetIsOn());
 }
 
