diff --git a/ash/keyboard/ui/keyboard_ui_controller.cc b/ash/keyboard/ui/keyboard_ui_controller.cc
index 1435069d5ac8..1e117fa58641 100644
--- a/ash/keyboard/ui/keyboard_ui_controller.cc
+++ b/ash/keyboard/ui/keyboard_ui_controller.cc
@@ -891,9 +891,6 @@ void KeyboardUIController::PopulateKeyboardContent(
   // are at begin states for animation.
   container_behavior_->InitializeShowAnimationStartingState(keyboard_window);
 
-  LogKeyboardControlEvent(KeyboardControlEvent::kShow);
-  RecordUkmKeyboardShown();
-
   ui::LayerAnimator* container_animator =
       keyboard_window->layer()->GetAnimator();
   container_animator->set_preemption_strategy(
@@ -1083,16 +1080,6 @@ void KeyboardUIController::SetContainerType(
   }
 }
 
-void KeyboardUIController::RecordUkmKeyboardShown() {
-  ui::TextInputClient* text_input_client = GetTextInputClient();
-  if (!text_input_client)
-    return;
-
-  keyboard::RecordUkmKeyboardShown(
-      text_input_client->GetClientSourceForMetrics(),
-      text_input_client->GetTextInputType());
-}
-
 void KeyboardUIController::SetDraggableArea(const gfx::Rect& rect) {
   container_behavior_->SetDraggableArea(rect);
 }
