diff --git a/ash/BUILD.gn b/ash/BUILD.gn
index ca5d00dca448..f55597923707 100644
--- a/ash/BUILD.gn
+++ b/ash/BUILD.gn
@@ -1021,8 +1021,6 @@ component("ash") {
     "system/locale/locale_update_controller_impl.h",
     "system/locale/unified_locale_detailed_view_controller.cc",
     "system/locale/unified_locale_detailed_view_controller.h",
-    "system/machine_learning/user_settings_event_logger.cc",
-    "system/machine_learning/user_settings_event_logger.h",
     "system/media/media_tray.cc",
     "system/media/media_tray.h",
     "system/media/unified_media_controls_container.cc",
@@ -1817,7 +1815,6 @@ component("ash") {
     "//media",
     "//net",
     "//services/data_decoder/public/cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/cpp:cpp",
     "//services/preferences/public/cpp",
 
@@ -2160,7 +2157,6 @@ test("ash_unittests") {
     "system/ime/ime_feature_pod_controller_unittest.cc",
     "system/ime_menu/ime_menu_tray_unittest.cc",
     "system/locale/locale_feature_pod_controller_unittest.cc",
-    "system/machine_learning/user_settings_event_logger_unittest.cc",
     "system/media/media_tray_unittest.cc",
     "system/media/unified_media_controls_container_unittest.cc",
     "system/media/unified_media_controls_controller_unittest.cc",
@@ -2397,7 +2393,6 @@ test("ash_unittests") {
     "//components/quirks",
     "//components/services/app_service/public/cpp:app_update",
     "//components/sync_preferences:test_support",
-    "//components/ukm:test_support",
     "//components/user_manager",
     "//components/user_manager:test_support",
     "//device/bluetooth",
@@ -2410,7 +2405,6 @@ test("ash_unittests") {
     "//services/device/public/cpp/bluetooth",
     "//services/media_session/public/cpp/test:test_support",
     "//services/media_session/public/mojom",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network:test_support",
     "//skia",
     "//testing/gmock",
