diff --git a/ash/system/night_light/night_light_feature_pod_controller.cc b/ash/system/night_light/night_light_feature_pod_controller.cc
index 3f11fde358f1..69fff7a124a3 100644
--- a/ash/system/night_light/night_light_feature_pod_controller.cc
+++ b/ash/system/night_light/night_light_feature_pod_controller.cc
@@ -9,7 +9,6 @@
 #include "ash/session/session_controller_impl.h"
 #include "ash/shell.h"
 #include "ash/strings/grit/ash_strings.h"
-#include "ash/system/machine_learning/user_settings_event_logger.h"
 #include "ash/system/model/system_tray_model.h"
 #include "ash/system/night_light/night_light_controller_impl.h"
 #include "ash/system/tray/tray_popup_utils.h"
@@ -19,18 +18,6 @@
 #include "base/metrics/user_metrics.h"
 #include "ui/base/l10n/l10n_util.h"
 
-namespace ash {
-namespace {
-
-void LogUserNightLightEvent(const bool enabled) {
-  auto* logger = ml::UserSettingsEventLogger::Get();
-  if (logger) {
-    logger->LogNightLightUkmEvent(enabled);
-  }
-}
-
-}  // namespace
-
 NightLightFeaturePodController::NightLightFeaturePodController(
     UnifiedSystemTrayController* tray_controller)
     : tray_controller_(tray_controller) {
@@ -55,7 +42,6 @@ FeaturePodButton* NightLightFeaturePodController::CreateButton() {
 
 void NightLightFeaturePodController::OnIconPressed() {
   Shell::Get()->night_light_controller()->Toggle();
-  LogUserNightLightEvent(Shell::Get()->night_light_controller()->GetEnabled());
   UpdateButton();
 
   if (Shell::Get()->night_light_controller()->GetEnabled()) {
