diff --git a/base/BUILD.gn b/base/BUILD.gn
index 0b65fbd05cdf..d77172b04656 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -413,8 +413,6 @@ component("base") {
     "metrics/sparse_histogram.h",
     "metrics/statistics_recorder.cc",
     "metrics/statistics_recorder.h",
-    "metrics/ukm_source_id.cc",
-    "metrics/ukm_source_id.h",
     "metrics/user_metrics.cc",
     "metrics/user_metrics.h",
     "metrics/user_metrics_action.h",
