diff --git a/base/trace_event/builtin_categories.h b/base/trace_event/builtin_categories.h
index 563674ae96317..ee6a983c01ab7 100644
--- a/base/trace_event/builtin_categories.h
+++ b/base/trace_event/builtin_categories.h
@@ -135,7 +135,6 @@
   X("renderer_host")                                                     \
   X("renderer.scheduler")                                                \
   X("RLZ")                                                               \
-  X("safe_browsing")                                                     \
   X("screenlock_monitor")                                                \
   X("sequence_manager")                                                  \
   X("service_manager")                                                   \
