diff --git a/base/BUILD.gn b/base/BUILD.gn
index 1d97f72804f2..e172ea823ba7 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -408,8 +408,6 @@ component("base") {
     "metrics/sparse_histogram.h",
     "metrics/statistics_recorder.cc",
     "metrics/statistics_recorder.h",
-    "metrics/ukm_source_id.cc",
-    "metrics/ukm_source_id.h",
     "metrics/user_metrics.cc",
     "metrics/user_metrics.h",
     "metrics/user_metrics_action.h",
