diff --git a/base/trace_event/builtin_categories.h b/base/trace_event/builtin_categories.h
index ad3e86fb45ed..fbd9e0ae9e87 100644
--- a/base/trace_event/builtin_categories.h
+++ b/base/trace_event/builtin_categories.h
@@ -132,7 +132,6 @@
   X("renderer_host")                                                     \
   X("renderer.scheduler")                                                \
   X("RLZ")                                                               \
-  X("safe_browsing")                                                     \
   X("screenlock_monitor")                                                \
   X("sequence_manager")                                                  \
   X("service_manager")                                                   \
