diff --git a/base/metrics/sparse_histogram.cc b/base/metrics/sparse_histogram.cc
index 6d2720c23698..1af9c45d0e91 100644
--- a/base/metrics/sparse_histogram.cc
+++ b/base/metrics/sparse_histogram.cc
@@ -115,17 +115,6 @@ void SparseHistogram::Add(Sample value) {
 }
 
 void SparseHistogram::AddCount(Sample value, int count) {
-  if (count <= 0) {
-    NOTREACHED();
-    return;
-  }
-  {
-    base::AutoLock auto_lock(lock_);
-    unlogged_samples_->Accumulate(value, count);
-  }
-
-  if (UNLIKELY(StatisticsRecorder::have_active_callbacks()))
-    FindAndRunCallback(value);
 }
 
 std::unique_ptr<HistogramSamples> SparseHistogram::SnapshotSamples() const {
