diff --git a/base/metrics/histogram.cc b/base/metrics/histogram.cc
index 99524406ed05..be429d13b2ba 100644
--- a/base/metrics/histogram.cc
+++ b/base/metrics/histogram.cc
@@ -510,25 +510,9 @@ bool Histogram::HasConstructionArguments(Sample expected_minimum,
 }
 
 void Histogram::Add(int value) {
-  AddCount(value, 1);
 }
 
 void Histogram::AddCount(int value, int count) {
-  DCHECK_EQ(0, ranges(0));
-  DCHECK_EQ(kSampleType_MAX, ranges(bucket_count()));
-
-  if (value > kSampleType_MAX - 1)
-    value = kSampleType_MAX - 1;
-  if (value < 0)
-    value = 0;
-  if (count <= 0) {
-    NOTREACHED();
-    return;
-  }
-  unlogged_samples_->Accumulate(value, count);
-
-  if (UNLIKELY(StatisticsRecorder::have_active_callbacks()))
-    FindAndRunCallback(value);
 }
 
 std::unique_ptr<HistogramSamples> Histogram::SnapshotSamples() const {
