diff --git a/BUILD.gn b/BUILD.gn
index f73c0c6a4e08..a607207e6f37 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -71,7 +71,6 @@ group("gn_all") {
     "//base:base_perftests",
     "//base:base_unittests",
     "//base/util:base_util_unittests",
-    "//chrome/installer",
     "//chrome/updater",
     "//components:components_unittests",
     "//components/gwp_asan:gwp_asan_unittests",
@@ -113,7 +112,6 @@ group("gn_all") {
       "//chrome/test:interactive_ui_tests",
       "//chrome/test:sync_integration_tests",
       "//chrome/test/chromedriver:chromedriver_unittests",
-      "//components/subresource_filter/tools:subresource_filter_tools",
       "//components/zucchini:zucchini",
       "//components/zucchini:zucchini_unittests",
       "//gpu/gles2_conform_support:gles2_conform_test",
@@ -719,7 +717,6 @@ group("gn_all") {
       "//chrome/tools/convert_dict",
       "//components/constrained_window:unit_tests",
       "//components/metrics:serialization",
-      "//components/rappor:unit_tests",
       "//components/services/filesystem:filesystem_service_unittests",
       "//components/sessions:unit_tests",
       "//media/blink:media_blink_unittests",
