diff --git a/build/config/win/BUILD.gn b/build/config/win/BUILD.gn
index 6cc4b1b9f4d3..85dfb9ebdbd2 100644
--- a/build/config/win/BUILD.gn
+++ b/build/config/win/BUILD.gn
@@ -65,7 +65,8 @@ config("compiler") {
   ]
 
   if (is_clang) {
-    cflags += [ "/Zc:twoPhase" ]
+    cflags += [ "/Zc:twoPhase",
+                "-Wno-unknown-warning-option", ]
   }
 
   # Force C/C++ mode for the given GN detected file type. This is necessary
