diff --git a/BUILD.gn b/BUILD.gn
index 6df566bd7df4..c68f46398641 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -453,14 +453,6 @@ group("gn_all") {
     ]
   }
 
-  if (is_chromeos_ash || is_mac || is_win) {
-    deps += [
-      "//rlz:rlz_id",
-      "//rlz:rlz_lib",
-      "//rlz:rlz_unittests",
-    ]
-  }
-
   if (is_linux || is_chromeos) {
     # The following are definitely linux-only.
     deps += [
@@ -634,7 +626,6 @@ group("gn_all") {
   if (is_win) {
     deps += [
       "//base:pe_image_test",
-      "//chrome/chrome_cleaner",
       "//chrome/chrome_elf:chrome_elf_unittests",
       "//chrome/chrome_elf:dll_hash_main",
       "//chrome/elevation_service:elevation_service_unittests",
@@ -844,10 +835,6 @@ if ((is_linux || is_chromeos || is_win) && enable_remoting &&
       "//remoting/host:host",
       "//tools/traffic_annotation/summary:annotations_xml",
     ]
-
-    if (is_win) {
-      deps += [ "//chrome/chrome_cleaner:shipped_binaries" ]
-    }
   }
 }
 
@@ -1394,7 +1381,6 @@ if (enable_js_type_check) {
       "components/neterror/resources:closure_compile",
       "components/security_interstitials/core/common/resources:closure_compile",
       "components/sync/driver/resources:closure_compile",
-      "components/ukm/debug:closure_compile",
       "content/browser/resources:closure_compile",
       "mojo/public/tools/bindings/generators/js_templates/lite/test:closure_compile",
       "ui/webui/resources:closure_compile",
