diff --git a/BUILD.gn b/BUILD.gn
index adf4629ced45..c1c1d055618c 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -116,7 +116,6 @@ group("gn_all") {
       "//chrome/test:interactive_ui_tests",
       "//chrome/test:sync_integration_tests",
       "//chrome/test/chromedriver:chromedriver_unittests",
-      "//components/subresource_filter/tools:subresource_filter_tools",
       "//components/zucchini:zucchini",
       "//components/zucchini:zucchini_unittests",
       "//gpu/gles2_conform_support:gles2_conform_test",
@@ -154,7 +153,6 @@ group("gn_all") {
       "//third_party/pdfium/samples:pdfium_test",
       "//tools/perf/clear_system_cache",
       "//tools/polymer:polymer_tools_python_unittests",
-      "//tools/privacy_budget:privacy_budget_tools",
       "//ui/accessibility:accessibility_perftests",
       "//ui/accessibility:accessibility_unittests",
       "//ui/accessibility/extensions:extension_tests",
@@ -245,7 +243,6 @@ group("gn_all") {
   } else if (is_ios) {
     deps += [
       "//ios:all",
-      "//third_party/crashpad/crashpad:crashpad_tests",
     ]
   } else if (is_fuchsia) {
     deps += [
@@ -452,14 +449,6 @@ group("gn_all") {
     ]
   }
 
-  if (is_chromeos || is_mac || is_win) {
-    deps += [
-      "//rlz:rlz_id",
-      "//rlz:rlz_lib",
-      "//rlz:rlz_unittests",
-    ]
-  }
-
   if (is_linux || is_chromeos) {
     # The following are definitely linux-only.
     deps += [
@@ -634,7 +623,6 @@ group("gn_all") {
   if (is_win) {
     deps += [
       "//base:pe_image_test",
-      "//chrome/chrome_cleaner",
       "//chrome/chrome_elf:chrome_elf_unittests",
       "//chrome/chrome_elf:dll_hash_main",
       "//chrome/elevation_service:elevation_service_unittests",
@@ -682,14 +670,6 @@ group("gn_all") {
     }
   }
 
-  if (is_mac || is_win || is_android || is_linux || is_chromeos) {
-    deps += [
-      "//third_party/crashpad/crashpad:crashpad_tests",
-      "//third_party/crashpad/crashpad/handler:crashpad_handler",
-      "//third_party/crashpad/crashpad/tools:crashpad_database_util",
-    ]
-  }
-
   if (use_openh264) {
     deps += [
       "//third_party/openh264:common",
@@ -726,7 +706,6 @@ group("gn_all") {
       "//chrome/tools/convert_dict",
       "//components/constrained_window:unit_tests",
       "//components/metrics:serialization",
-      "//components/rappor:unit_tests",
       "//components/services/filesystem:filesystem_service_unittests",
       "//components/sessions:unit_tests",
       "//media/blink:media_blink_unittests",
@@ -853,10 +832,6 @@ if ((is_linux || is_chromeos || is_win) && enable_remoting &&
       "//remoting/host:host",
       "//tools/traffic_annotation/summary:annotations_xml",
     ]
-
-    if (is_win) {
-      deps += [ "//chrome/chrome_cleaner:shipped_binaries" ]
-    }
   }
 }
 
@@ -1258,7 +1233,6 @@ if (!is_ios) {
 
     data = [
       "//build/android/",
-      "//components/crash/content/tools/generate_breakpad_symbols.py",
       "//third_party/blink/renderer/bindings/scripts/",
       "//third_party/blink/renderer/build/scripts/",
       "//third_party/blink/tools/",
@@ -1396,7 +1370,6 @@ if (enable_js_type_check) {
       "components/neterror/resources:closure_compile",
       "components/security_interstitials/core/common/resources:closure_compile",
       "components/sync/driver/resources:closure_compile",
-      "components/ukm/debug:closure_compile",
       "content/browser/resources:closure_compile",
       "mojo/public/tools/bindings/generators/js_templates/lite/test:closure_compile",
       "ui/webui/resources:closure_compile",
