diff --git a/BUILD.gn b/BUILD.gn
index c1277b6c8631..21c02c7790d1 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -71,7 +71,6 @@ group("gn_all") {
     "//base:base_perftests",
     "//base:base_unittests",
     "//base/util:base_util_unittests",
-    "//chrome/installer",
     "//chrome/updater",
     "//components:components_unittests",
     "//components/gwp_asan:gwp_asan_unittests",
@@ -113,7 +112,6 @@ group("gn_all") {
       "//chrome/test:interactive_ui_tests",
       "//chrome/test:sync_integration_tests",
       "//chrome/test/chromedriver:chromedriver_unittests",
-      "//components/subresource_filter/tools:subresource_filter_tools",
       "//components/zucchini:zucchini",
       "//components/zucchini:zucchini_unittests",
       "//gpu/gles2_conform_support:gles2_conform_test",
@@ -239,7 +237,6 @@ group("gn_all") {
   } else if (is_ios) {
     deps += [
       "//ios:all",
-      "//third_party/crashpad/crashpad:crashpad_tests",
     ]
   } else if (is_fuchsia) {
     deps += [
@@ -629,7 +626,6 @@ group("gn_all") {
   if (is_win) {
     deps += [
       "//base:pe_image_test",
-      "//chrome/chrome_cleaner",
       "//chrome/chrome_elf:chrome_elf_unittests",
       "//chrome/chrome_elf:dll_hash_main",
       "//chrome/elevation_service:elevation_service_unittests",
@@ -677,14 +673,6 @@ group("gn_all") {
     }
   }
 
-  if (is_mac || is_win || is_android || (is_linux && !is_chromeos)) {
-    deps += [
-      "//third_party/crashpad/crashpad:crashpad_tests",
-      "//third_party/crashpad/crashpad/handler:crashpad_handler",
-      "//third_party/crashpad/crashpad/tools:crashpad_database_util",
-    ]
-  }
-
   if (use_openh264) {
     deps += [
       "//third_party/openh264:common",
@@ -721,7 +709,6 @@ group("gn_all") {
       "//chrome/tools/convert_dict",
       "//components/constrained_window:unit_tests",
       "//components/metrics:serialization",
-      "//components/rappor:unit_tests",
       "//components/services/filesystem:filesystem_service_unittests",
       "//components/sessions:unit_tests",
       "//media/blink:media_blink_unittests",
@@ -847,10 +834,6 @@ if ((is_linux || is_chromeos || is_win) && enable_remoting &&
       "//remoting/host:host",
       "//tools/traffic_annotation/summary:annotations_xml",
     ]
-
-    if (is_win) {
-      deps += [ "//chrome/chrome_cleaner:shipped_binaries" ]
-    }
   }
 }
 
