diff --git a/BUILD.gn b/BUILD.gn
index 39b9075b1267..085c4a88bf23 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -113,7 +113,6 @@ group("gn_all") {
       "//chrome/test:interactive_ui_tests",
       "//chrome/test:sync_integration_tests",
       "//chrome/test/chromedriver:chromedriver_unittests",
-      "//components/subresource_filter/tools:subresource_filter_tools",
       "//components/zucchini:zucchini",
       "//components/zucchini:zucchini_unittests",
       "//gpu/gles2_conform_support:gles2_conform_test",
@@ -243,7 +242,6 @@ group("gn_all") {
   } else if (is_ios) {
     deps += [
       "//ios:all",
-      "//third_party/crashpad/crashpad:crashpad_tests",
     ]
   } else if (is_fuchsia) {
     deps += [
@@ -448,14 +446,6 @@ group("gn_all") {
     ]
   }
 
-  if (is_chromeos || is_mac || is_win) {
-    deps += [
-      "//rlz:rlz_id",
-      "//rlz:rlz_lib",
-      "//rlz:rlz_unittests",
-    ]
-  }
-
   if (is_linux || is_chromeos) {
     # The following are definitely linux-only.
     deps += [
@@ -630,7 +620,6 @@ group("gn_all") {
   if (is_win) {
     deps += [
       "//base:pe_image_test",
-      "//chrome/chrome_cleaner",
       "//chrome/chrome_elf:chrome_elf_unittests",
       "//chrome/chrome_elf:dll_hash_main",
       "//chrome/elevation_service:elevation_service_unittests",
@@ -678,14 +667,6 @@ group("gn_all") {
     }
   }
 
-  if (is_mac || is_win || is_android || (is_linux && !is_chromeos)) {
-    deps += [
-      "//third_party/crashpad/crashpad:crashpad_tests",
-      "//third_party/crashpad/crashpad/handler:crashpad_handler",
-      "//third_party/crashpad/crashpad/tools:crashpad_database_util",
-    ]
-  }
-
   if (use_openh264) {
     deps += [
       "//third_party/openh264:common",
@@ -722,7 +703,6 @@ group("gn_all") {
       "//chrome/tools/convert_dict",
       "//components/constrained_window:unit_tests",
       "//components/metrics:serialization",
-      "//components/rappor:unit_tests",
       "//components/services/filesystem:filesystem_service_unittests",
       "//components/sessions:unit_tests",
       "//media/blink:media_blink_unittests",
@@ -848,10 +828,6 @@ if ((is_linux || is_chromeos || is_win) && enable_remoting &&
       "//remoting/host:host",
       "//tools/traffic_annotation/summary:annotations_xml",
     ]
-
-    if (is_win) {
-      deps += [ "//chrome/chrome_cleaner:shipped_binaries" ]
-    }
   }
 }
 
@@ -1230,7 +1206,6 @@ if (!is_ios) {
   group("blink_python_tests") {
     data = [
       "//build/android/",
-      "//components/crash/content/tools/generate_breakpad_symbols.py",
       "//testing/scripts/common.py",
       "//testing/scripts/run_isolated_script_test.py",
       "//testing/test_env.py",
@@ -1372,7 +1347,6 @@ if (enable_js_type_check) {
       "components/neterror/resources:closure_compile",
       "components/security_interstitials/core/common/resources:closure_compile",
       "components/sync/driver/resources:closure_compile",
-      "components/ukm/debug:closure_compile",
       "content/browser/resources:closure_compile",
       "mojo/public/tools/bindings/generators/js_templates/lite/test:closure_compile",
       "ui/webui/resources:closure_compile",
