diff --git a/cc/BUILD.gn b/cc/BUILD.gn
index 3465bf70b6c0..235aaa4b259d 100644
--- a/cc/BUILD.gn
+++ b/cc/BUILD.gn
@@ -611,8 +611,6 @@ cc_test_static_library("test_support") {
     "//base/test:test_support",
     "//base/third_party/dynamic_annotations",
     "//cc/paint",
-    "//components/ukm",
-    "//components/ukm:test_support",
     "//components/viz/client",
     "//components/viz/common",
     "//components/viz/service",
@@ -827,7 +825,6 @@ cc_test("cc_unittests") {
     "//build:chromeos_buildflags",
     "//cc/mojo_embedder",
     "//cc/paint",
-    "//components/ukm:test_support",
     "//components/viz/client",
     "//components/viz/common",
     "//components/viz/service",
