diff --git a/cc/BUILD.gn b/cc/BUILD.gn
index 13098ad007436..065ac0555bd5f 100644
--- a/cc/BUILD.gn
+++ b/cc/BUILD.gn
@@ -612,7 +612,6 @@ cc_test_static_library("test_support") {
     "//base/test:test_support",
     "//base/third_party/dynamic_annotations",
     "//cc/paint",
-    "//components/ukm",
     "//components/ukm:test_support",
     "//components/viz/client",
     "//components/viz/common",
