diff --git a/cc/BUILD.gn b/cc/BUILD.gn
index eaa675ba6a9ff..45ad2a7efdd69 100644
--- a/cc/BUILD.gn
+++ b/cc/BUILD.gn
@@ -614,7 +614,6 @@ cc_test_static_library("test_support") {
     "//base/test:test_support",
     "//base/third_party/dynamic_annotations",
     "//cc/paint",
-    "//components/ukm",
     "//components/ukm:test_support",
     "//components/viz/client",
     "//components/viz/common",
