diff --git a/cc/BUILD.gn b/cc/BUILD.gn
index 8c8f34b3727c..87869598192c 100644
--- a/cc/BUILD.gn
+++ b/cc/BUILD.gn
@@ -606,8 +606,6 @@ cc_test_static_library("test_support") {
     "//base/test:test_support",
     "//base/third_party/dynamic_annotations",
     "//cc/paint",
-    "//components/ukm",
-    "//components/ukm:test_support",
     "//components/viz/client",
     "//components/viz/common",
     "//components/viz/service",
@@ -820,7 +818,6 @@ cc_test("cc_unittests") {
     "//build:chromeos_buildflags",
     "//cc/mojo_embedder",
     "//cc/paint",
-    "//components/ukm:test_support",
     "//components/viz/client",
     "//components/viz/common",
     "//components/viz/service",
