diff --git a/cc/metrics/dropped_frame_counter.cc b/cc/metrics/dropped_frame_counter.cc
index bc9e116bcf89..8c362e368720 100644
--- a/cc/metrics/dropped_frame_counter.cc
+++ b/cc/metrics/dropped_frame_counter.cc
@@ -84,11 +84,6 @@ void DroppedFrameCounter::ReportFrames() {
   }
 }
 
-void DroppedFrameCounter::SetUkmSmoothnessDestination(
-    UkmSmoothnessDataShared* smoothness_data) {
-  ukm_smoothness_data_ = smoothness_data;
-}
-
 void DroppedFrameCounter::Reset() {
   total_frames_ = 0;
   total_partial_ = 0;
