diff --git a/cc/metrics/frame_sequence_metrics.cc b/cc/metrics/frame_sequence_metrics.cc
index e79edb0a3c0f..95e2ed11e32d 100644
--- a/cc/metrics/frame_sequence_metrics.cc
+++ b/cc/metrics/frame_sequence_metrics.cc
@@ -15,7 +15,6 @@
 #include "base/trace_event/traced_value.h"
 #include "cc/metrics/frame_sequence_tracker.h"
 #include "cc/metrics/jank_metrics.h"
-#include "cc/metrics/throughput_ukm_reporter.h"
 
 namespace cc {
 
@@ -102,9 +101,8 @@ bool IsInteractionType(FrameSequenceTrackerType sequence_type) {
 
 }  // namespace
 
-FrameSequenceMetrics::FrameSequenceMetrics(FrameSequenceTrackerType type,
-                                           ThroughputUkmReporter* ukm_reporter)
-    : type_(type), throughput_ukm_reporter_(ukm_reporter) {
+FrameSequenceMetrics::FrameSequenceMetrics(FrameSequenceTrackerType type)
+    : type_(type) {
   ThreadType thread_type = GetEffectiveThread();
 
   // Only construct |jank_reporter_| if it has a valid tracker and thread type.
@@ -446,8 +444,6 @@ int FrameSequenceMetrics::ThroughputData::ReportDroppedFramePercentHistogram(
       ShouldReportForAnimation(sequence_type, thread_type);
   const bool is_interaction = ShouldReportForInteraction(metrics, thread_type);
 
-  ThroughputUkmReporter* const ukm_reporter = metrics->ukm_reporter();
-
   if (is_animation) {
     TRACE_EVENT_INSTANT2("cc,benchmark", "PercentDroppedFrames.AllAnimations",
                          TRACE_EVENT_SCOPE_THREAD, "frames_expected",
@@ -456,10 +452,6 @@ int FrameSequenceMetrics::ThroughputData::ReportDroppedFramePercentHistogram(
 
     UMA_HISTOGRAM_PERCENTAGE(
         "Graphics.Smoothness.PercentDroppedFrames.AllAnimations", percent);
-    if (ukm_reporter) {
-      ukm_reporter->ReportAggregateThroughput(AggregationType::kAllAnimations,
-                                              percent);
-    }
   }
 
   if (is_interaction) {
@@ -469,10 +461,6 @@ int FrameSequenceMetrics::ThroughputData::ReportDroppedFramePercentHistogram(
                          data.frames_produced);
     UMA_HISTOGRAM_PERCENTAGE(
         "Graphics.Smoothness.PercentDroppedFrames.AllInteractions", percent);
-    if (ukm_reporter) {
-      ukm_reporter->ReportAggregateThroughput(AggregationType::kAllInteractions,
-                                              percent);
-    }
   }
 
   if (is_animation || is_interaction) {
@@ -482,10 +470,6 @@ int FrameSequenceMetrics::ThroughputData::ReportDroppedFramePercentHistogram(
                          data.frames_produced);
     UMA_HISTOGRAM_PERCENTAGE(
         "Graphics.Smoothness.PercentDroppedFrames.AllSequences", percent);
-    if (ukm_reporter) {
-      ukm_reporter->ReportAggregateThroughput(AggregationType::kAllSequences,
-                                              percent);
-    }
   }
 
   const char* thread_name = thread_type == ThreadType::kCompositor
