diff --git a/cc/metrics/dropped_frame_counter.h b/cc/metrics/dropped_frame_counter.h
index f1c3f5e97a40..c3fa91f40355 100644
--- a/cc/metrics/dropped_frame_counter.h
+++ b/cc/metrics/dropped_frame_counter.h
@@ -51,8 +51,6 @@ class CC_EXPORT DroppedFrameCounter {
   void AddDroppedFrameAffectingSmoothness();
   void ReportFrames();
 
-  void SetUkmSmoothnessDestination(UkmSmoothnessDataShared* smoothness_data);
-
   void Reset();
   void OnFcpReceived();
 
