diff --git a/cc/metrics/frame_sequence_metrics.h b/cc/metrics/frame_sequence_metrics.h
index c663791d76fd..d2e04eed6164 100644
--- a/cc/metrics/frame_sequence_metrics.h
+++ b/cc/metrics/frame_sequence_metrics.h
@@ -13,7 +13,6 @@
 #include "cc/cc_export.h"
 
 namespace cc {
-class ThroughputUkmReporter;
 class JankMetrics;
 
 enum class FrameSequenceTrackerType {
@@ -36,8 +35,7 @@ enum class FrameSequenceTrackerType {
 
 class CC_EXPORT FrameSequenceMetrics {
  public:
-  FrameSequenceMetrics(FrameSequenceTrackerType type,
-                       ThroughputUkmReporter* ukm_reporter);
+  FrameSequenceMetrics(FrameSequenceTrackerType type);
   ~FrameSequenceMetrics();
 
   FrameSequenceMetrics(const FrameSequenceMetrics&) = delete;
@@ -143,9 +141,6 @@ class CC_EXPORT FrameSequenceMetrics {
   uint32_t frames_checkerboarded() const { return frames_checkerboarded_; }
 
   FrameSequenceTrackerType type() const { return type_; }
-  ThroughputUkmReporter* ukm_reporter() const {
-    return throughput_ukm_reporter_;
-  }
 
   // Must be called before destructor.
   void ReportLeftoverData();
@@ -184,9 +179,6 @@ class CC_EXPORT FrameSequenceMetrics {
     void Terminate();
   } trace_data_{this};
 
-  // Pointer to the reporter owned by the FrameSequenceTrackerCollection.
-  ThroughputUkmReporter* const throughput_ukm_reporter_;
-
   ThroughputData impl_throughput_;
   ThroughputData main_throughput_;
 
