diff --git a/cc/metrics/compositor_frame_reporter.h b/cc/metrics/compositor_frame_reporter.h
index d7abf2950efb..aaa5e1475c72 100644
--- a/cc/metrics/compositor_frame_reporter.h
+++ b/cc/metrics/compositor_frame_reporter.h
@@ -29,7 +29,6 @@ struct FrameTimingDetails;
 
 namespace cc {
 class DroppedFrameCounter;
-class LatencyUkmReporter;
 
 // This is used for tracing and reporting the duration of pipeline stages within
 // a single frame.
@@ -146,7 +145,6 @@ class CC_EXPORT CompositorFrameReporter {
 
   CompositorFrameReporter(const ActiveTrackers& active_trackers,
                           const viz::BeginFrameArgs& args,
-                          LatencyUkmReporter* latency_ukm_reporter,
                           bool should_report_metrics,
                           SmoothThread smooth_thread,
                           int layer_tree_host_id);
@@ -306,8 +304,6 @@ class CC_EXPORT CompositorFrameReporter {
 
   const ActiveTrackers active_trackers_;
 
-  LatencyUkmReporter* latency_ukm_reporter_;
-
   // Indicates if work on Impl frame is finished.
   bool did_finish_impl_frame_ = false;
   // The time that work on Impl frame is finished. It's only valid if the
