diff --git a/cc/metrics/dropped_frame_counter.h b/cc/metrics/dropped_frame_counter.h
index 17e9386a5082..2508d3859184 100644
--- a/cc/metrics/dropped_frame_counter.h
+++ b/cc/metrics/dropped_frame_counter.h
@@ -50,7 +50,6 @@ class CC_EXPORT DroppedFrameCounter {
 
   void OnBeginFrame(const viz::BeginFrameArgs& args);
   void OnEndFrame(const viz::BeginFrameArgs& args, bool is_dropped);
-  void SetUkmSmoothnessDestination(UkmSmoothnessDataShared* smoothness_data);
   void OnFcpReceived();
 
   // Reset is used on navigation, which resets frame statistics as well as
