diff --git a/cc/trees/single_thread_proxy.h b/cc/trees/single_thread_proxy.h
index 8b0960dbd4f2..49643189e2a7 100644
--- a/cc/trees/single_thread_proxy.h
+++ b/cc/trees/single_thread_proxy.h
@@ -64,13 +64,11 @@ class CC_EXPORT SingleThreadProxy : public Proxy,
       std::unique_ptr<PaintWorkletLayerPainter> painter) override;
   bool SupportsImplScrolling() const override;
   bool MainFrameWillHappenForTesting() override;
-  void SetSourceURL(ukm::SourceId source_id, const GURL& url) override {
+  void SetSourceURL(const GURL& url) override {
     // Single-threaded mode is only for browser compositing and for renderers in
     // layout tests. This will still get called in the latter case, but we don't
     // need to record UKM in that case.
   }
-  void SetUkmSmoothnessDestination(
-      base::WritableSharedMemoryMapping ukm_smoothness_data) override {}
   void ClearHistory() override;
   void SetRenderFrameObserver(
       std::unique_ptr<RenderFrameMetadataObserver> observer) override;
