diff --git a/cc/trees/proxy_impl.cc b/cc/trees/proxy_impl.cc
index 13cccd11026d..f06f4267e482 100644
--- a/cc/trees/proxy_impl.cc
+++ b/cc/trees/proxy_impl.cc
@@ -33,7 +33,6 @@
 #include "components/viz/common/frame_timing_details.h"
 #include "components/viz/common/gpu/context_provider.h"
 #include "gpu/command_buffer/client/gles2_interface.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
 
 namespace cc {
 
@@ -86,9 +85,7 @@ ProxyImpl::ProxyImpl(base::WeakPtr<ProxyMain> proxy_main_weak_ptr,
   std::unique_ptr<CompositorTimingHistory> compositor_timing_history(
       new CompositorTimingHistory(
           scheduler_settings.using_synchronous_renderer_compositor,
-          CompositorTimingHistory::RENDERER_UMA,
-          layer_tree_host->rendering_stats_instrumentation(),
-          host_impl_->compositor_frame_reporting_controller()));
+          layer_tree_host->rendering_stats_instrumentation()));
   scheduler_.reset(new Scheduler(this, scheduler_settings, layer_tree_host_id_,
                                  task_runner_provider_->ImplThreadTaskRunner(),
                                  std::move(compositor_timing_history)));
@@ -414,11 +411,6 @@ void ProxyImpl::RenewTreePriority() {
   bool user_interaction_in_progress =
       non_scroll_interaction_in_progress || scroll_type_considered_interaction;
 
-  if (host_impl_->ukm_manager()) {
-    host_impl_->ukm_manager()->SetUserInteractionInProgress(
-        user_interaction_in_progress);
-  }
-
   // Schedule expiration if smoothness currently takes priority.
   if (user_interaction_in_progress)
     smoothness_priority_expiration_notifier_.Schedule();
@@ -803,15 +795,9 @@ base::SingleThreadTaskRunner* ProxyImpl::MainThreadTaskRunner() {
   return task_runner_provider_->MainThreadTaskRunner();
 }
 
-void ProxyImpl::SetSourceURL(ukm::SourceId source_id, const GURL& url) {
-  DCHECK(IsImplThread());
-  host_impl_->SetActiveURL(url, source_id);
-}
-
-void ProxyImpl::SetUkmSmoothnessDestination(
-    base::WritableSharedMemoryMapping ukm_smoothness_data) {
+void ProxyImpl::SetSourceURL(const GURL& url) {
   DCHECK(IsImplThread());
-  host_impl_->SetUkmSmoothnessDestination(std::move(ukm_smoothness_data));
+  host_impl_->SetActiveURL(url);
 }
 
 void ProxyImpl::ClearHistory() {
