diff --git a/cc/trees/layer_tree_host_client.h b/cc/trees/layer_tree_host_client.h
index c17aac50b960..fdcb41b8cb35 100644
--- a/cc/trees/layer_tree_host_client.h
+++ b/cc/trees/layer_tree_host_client.h
@@ -11,7 +11,6 @@
 #include "base/time/time.h"
 #include "cc/input/browser_controls_state.h"
 #include "cc/metrics/frame_sequence_tracker_collection.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "ui/gfx/geometry/scroll_offset.h"
 #include "ui/gfx/geometry/vector2d_f.h"
 
@@ -174,8 +173,7 @@ class LayerTreeHostClient {
   virtual void NotifyThroughputTrackerResults(CustomTrackerResults results) = 0;
   // LayerTreeHost calls this when there is new throughput data. The client send
   // the data to the browser process who will aggregate them and report to UKM.
-  virtual void SubmitThroughputData(ukm::SourceId source_id,
-                                    int aggregated_percent,
+  virtual void SubmitThroughputData(int aggregated_percent,
                                     int impl_percent,
                                     base::Optional<int> main_percent) = 0;
 
