diff --git a/cc/trees/proxy.h b/cc/trees/proxy.h
index 5c886bb152f6..749a91e7f1e3 100644
--- a/cc/trees/proxy.h
+++ b/cc/trees/proxy.h
@@ -17,7 +17,6 @@
 #include "cc/trees/paint_holding_commit_trigger.h"
 #include "cc/trees/task_runner_provider.h"
 #include "components/viz/common/frame_sinks/begin_frame_source.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "url/gurl.h"
 
 namespace gfx {
@@ -90,10 +89,7 @@ class CC_EXPORT Proxy {
   // Testing hooks
   virtual bool MainFrameWillHappenForTesting() = 0;
 
-  virtual void SetSourceURL(ukm::SourceId source_id, const GURL& url) = 0;
-
-  virtual void SetUkmSmoothnessDestination(
-      base::WritableSharedMemoryMapping ukm_smoothness_data) = 0;
+  virtual void SetSourceURL(const GURL& url) = 0;
 
   virtual void ClearHistory() = 0;
 
