diff --git a/cc/trees/proxy_main.cc b/cc/trees/proxy_main.cc
index 5f378c07e89b..84d2b73ad2a3 100644
--- a/cc/trees/proxy_main.cc
+++ b/cc/trees/proxy_main.cc
@@ -26,7 +26,6 @@
 #include "cc/trees/render_frame_metadata_observer.h"
 #include "cc/trees/scoped_abort_remaining_swap_promises.h"
 #include "cc/trees/swap_promise.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
 
 namespace cc {
 
@@ -394,12 +393,11 @@ void ProxyMain::NotifyThroughputTrackerResults(CustomTrackerResults results) {
   layer_tree_host_->NotifyThroughputTrackerResults(std::move(results));
 }
 
-void ProxyMain::SubmitThroughputData(ukm::SourceId source_id,
-                                     int aggregated_percent,
+void ProxyMain::SubmitThroughputData(int aggregated_percent,
                                      int impl_percent,
                                      base::Optional<int> main_percent) {
   DCHECK(!task_runner_provider_->IsImplThread());
-  layer_tree_host_->SubmitThroughputData(source_id, aggregated_percent,
+  layer_tree_host_->SubmitThroughputData(aggregated_percent,
                                          impl_percent, main_percent);
 }
 
@@ -692,12 +690,12 @@ base::SingleThreadTaskRunner* ProxyMain::ImplThreadTaskRunner() {
   return task_runner_provider_->ImplThreadTaskRunner();
 }
 
-void ProxyMain::SetSourceURL(ukm::SourceId source_id, const GURL& url) {
+void ProxyMain::SetSourceURL(const GURL& url) {
   DCHECK(IsMainThread());
   ImplThreadTaskRunner()->PostTask(
       FROM_HERE, base::BindOnce(&ProxyImpl::SetSourceURL,
                                 base::Unretained(proxy_impl_.get()),
-                                source_id, url));
+                                url));
 }
 
 void ProxyMain::ClearHistory() {
