diff --git a/cc/trees/layer_tree_host_impl.h b/cc/trees/layer_tree_host_impl.h
index eee1d46bc159..99b3acc06bcc 100644
--- a/cc/trees/layer_tree_host_impl.h
+++ b/cc/trees/layer_tree_host_impl.h
@@ -62,7 +62,6 @@
 #include "cc/trees/presentation_time_callback_buffer.h"
 #include "cc/trees/render_frame_metadata.h"
 #include "cc/trees/task_runner_provider.h"
-#include "cc/trees/ukm_manager.h"
 #include "components/viz/client/client_resource_provider.h"
 #include "components/viz/common/frame_sinks/begin_frame_args.h"
 #include "components/viz/common/gpu/context_cache_controller.h"
@@ -773,9 +772,6 @@ class CC_EXPORT LayerTreeHostImpl : public TileManagerClient,
       base::flat_map<PaintImage::Id, PaintImage::DecodingMode>
           decoding_mode_map);
 
-  void InitializeUkm(std::unique_ptr<ukm::UkmRecorder> recorder);
-  UkmManager* ukm_manager() { return ukm_manager_.get(); }
-
   ActiveFrameSequenceTrackers FrameSequenceTrackerActiveTypes() {
     return frame_trackers_.FrameSequenceTrackerActiveTypes();
   }
@@ -785,10 +781,7 @@ class CC_EXPORT LayerTreeHostImpl : public TileManagerClient,
   void SetRenderFrameObserver(
       std::unique_ptr<RenderFrameMetadataObserver> observer);
 
-  void SetActiveURL(const GURL& url, ukm::SourceId source_id);
-
-  void SetUkmSmoothnessDestination(
-      base::WritableSharedMemoryMapping ukm_smoothness_data);
+  void SetActiveURL(const GURL& url);
 
   CompositorFrameReportingController* compositor_frame_reporting_controller()
       const {
@@ -1127,8 +1120,6 @@ class CC_EXPORT LayerTreeHostImpl : public TileManagerClient,
 
   ImageAnimationController image_animation_controller_;
 
-  std::unique_ptr<UkmManager> ukm_manager_;
-
   // Provides RenderFrameMetadata to the Browser process upon the submission of
   // each CompositorFrame.
   std::unique_ptr<RenderFrameMetadataObserver> render_frame_metadata_observer_;
@@ -1199,8 +1190,6 @@ class CC_EXPORT LayerTreeHostImpl : public TileManagerClient,
   // DroppedFrameCounter. Currently true when first contentful paint is done.
   bool is_measuring_smoothness_ = false;
 
-  base::WritableSharedMemoryMapping ukm_smoothness_mapping_;
-
   // Cache for the results of calls to gfx::ColorSpace::Contains() on sRGB. This
   // computation is deterministic for a given color space, can be called
   // multiple times per frame, and incurs a non-trivial cost.
