diff --git a/cc/trees/layer_tree_host_impl.cc b/cc/trees/layer_tree_host_impl.cc
index fd3ea9b6b153..8af9a326fe49 100644
--- a/cc/trees/layer_tree_host_impl.cc
+++ b/cc/trees/layer_tree_host_impl.cc
@@ -421,7 +421,7 @@ LayerTreeHostImpl::LayerTreeHostImpl(
       paint_image_generator_client_id_(PaintImage::GetNextGeneratorClientId()),
       frame_trackers_(settings.single_thread_proxy_scheduler,
                       compositor_frame_reporting_controller_.get()),
-      lcd_text_metrics_reporter_(LCDTextMetricsReporter::CreateIfNeeded(this)),
+      lcd_text_metrics_reporter_(nullptr),
       frame_rate_estimator_(GetTaskRunner()) {
   // TODO(bokan): Temporary while we decouple input from the layer tree.
   input_delegate_ = static_cast<InputDelegateForCompositor*>(&input_handler_);
