diff --git a/cc/trees/proxy_main.h b/cc/trees/proxy_main.h
index d5bc7291989e..b107c4597dce 100644
--- a/cc/trees/proxy_main.h
+++ b/cc/trees/proxy_main.h
@@ -99,9 +99,7 @@ class CC_EXPORT ProxyMain : public Proxy {
                                   BrowserControlsState current,
                                   bool animate) override;
   void RequestBeginMainFrameNotExpected(bool new_state) override;
-  void SetSourceURL(ukm::SourceId source_id, const GURL& url) override;
-  void SetUkmSmoothnessDestination(
-      base::WritableSharedMemoryMapping ukm_smoothness_data) override;
+  void SetSourceURL(const GURL& url) override;
   void ClearHistory() override;
   void SetRenderFrameObserver(
       std::unique_ptr<RenderFrameMetadataObserver> observer) override;
