diff --git a/cc/test/fake_proxy.h b/cc/test/fake_proxy.h
index 18ef28cefee1..c8bb0678f76e 100644
--- a/cc/test/fake_proxy.h
+++ b/cc/test/fake_proxy.h
@@ -9,7 +9,6 @@
 #include "cc/trees/layer_tree_host.h"
 #include "cc/trees/proxy.h"
 #include "cc/trees/render_frame_metadata_observer.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
 
 namespace cc {
 class PaintWorkletLayerPainter;
@@ -45,9 +44,7 @@ class FakeProxy : public Proxy {
                                   BrowserControlsState current,
                                   bool animate) override {}
   void RequestBeginMainFrameNotExpected(bool new_state) override {}
-  void SetSourceURL(ukm::SourceId source_id, const GURL& url) override {}
-  void SetUkmSmoothnessDestination(
-      base::WritableSharedMemoryMapping ukm_smoothness_data) override {}
+  void SetSourceURL(const GURL& url) override {}
   void ClearHistory() override {}
   void SetRenderFrameObserver(
       std::unique_ptr<RenderFrameMetadataObserver> observer) override {}
