diff --git a/cc/trees/proxy_impl.h b/cc/trees/proxy_impl.h
index 815935a5d41f..2c9ecc6802ef 100644
--- a/cc/trees/proxy_impl.h
+++ b/cc/trees/proxy_impl.h
@@ -64,9 +64,7 @@ class CC_EXPORT ProxyImpl : public LayerTreeHostImplClient,
                                  base::TimeTicks main_thread_start_time,
                                  const viz::BeginFrameArgs& commit_args,
                                  bool hold_commit_for_activation);
-  void SetSourceURL(ukm::SourceId source_id, const GURL& url);
-  void SetUkmSmoothnessDestination(
-      base::WritableSharedMemoryMapping ukm_smoothness_data);
+  void SetSourceURL(const GURL& url);
   void ClearHistory();
   void SetRenderFrameObserver(
       std::unique_ptr<RenderFrameMetadataObserver> observer);
