diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
index 9f1b189d7043..08493c2e6288 100644
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -321,8 +321,6 @@ android_library("chrome_java") {
     "//chrome/browser/ui/android/native_page:java",
     "//chrome/browser/ui/messages/android:java",
     "//chrome/browser/util:java",
-    "//chrome/browser/video_tutorials:factory_java",
-    "//chrome/browser/video_tutorials:java",
     "//chrome/browser/webauthn/android:java",
     "//chrome/browser/xsurface:java",
     "//components/autofill/android:autofill_java",
@@ -604,7 +602,6 @@ java_group("chrome_all_java") {
     "//chrome/browser/test_dummy/internal:base_module_java",
     "//chrome/browser/touch_to_fill/android/internal:java",
     "//chrome/browser/ui/android/appmenu/internal:java",
-    "//chrome/browser/video_tutorials/internal:java",
     "//components/browser_ui/android/bottomsheet/internal:java",
   ]
 
@@ -3086,7 +3083,6 @@ generate_jni("chrome_jni_headers") {
     "java/src/org/chromium/chrome/browser/provider/ChromeBrowserProvider.java",
     "java/src/org/chromium/chrome/browser/push_messaging/PushMessagingServiceObserver.java",
     "java/src/org/chromium/chrome/browser/query_tiles/TileProviderFactory.java",
-    "java/src/org/chromium/chrome/browser/rappor/RapporServiceBridge.java",
     "java/src/org/chromium/chrome/browser/resources/ResourceMapper.java",
     "java/src/org/chromium/chrome/browser/rlz/RevenueStats.java",
     "java/src/org/chromium/chrome/browser/rlz/RlzPingHandler.java",
