diff --git a/chrome/browser/ui/views/tabs/browser_tab_strip_controller.cc b/chrome/browser/ui/views/tabs/browser_tab_strip_controller.cc
index 975817e079eb..eda9528be11d 100644
--- a/chrome/browser/ui/views/tabs/browser_tab_strip_controller.cc
+++ b/chrome/browser/ui/views/tabs/browser_tab_strip_controller.cc
@@ -366,6 +366,12 @@ void BrowserTabStripController::CloseTab(int model_index) {
                              TabStripModel::CLOSE_CREATE_HISTORICAL_TAB);
 }
 
+void BrowserTabStripController::ToggleTabAudioMute(int model_index) {
+  content::WebContents* const contents = model_->GetWebContentsAt(model_index);
+  chrome::SetTabAudioMuted(contents, !contents->IsAudioMuted(),
+                           TabMutedReason::AUDIO_INDICATOR, std::string());
+}
+
 void BrowserTabStripController::AddTabToGroup(
     int model_index,
     const tab_groups::TabGroupId& group) {
