diff --git a/chrome/browser/profiles/profile_info_cache.cc b/chrome/browser/profiles/profile_info_cache.cc
index dcc2d9eb1bf2d..2ac3729ab38cd 100644
--- a/chrome/browser/profiles/profile_info_cache.cc
+++ b/chrome/browser/profiles/profile_info_cache.cc
@@ -122,11 +122,6 @@ ProfileInfoCache::ProfileInfoCache(PrefService* prefs,
     MigrateLegacyProfileNamesAndRecomputeIfNeeded();
     prefs_->SetBoolean(kLegacyProfileNameMigrated, true);
   }
-
-  repeating_timer_ = std::make_unique<signin::PersistentRepeatingTimer>(
-      prefs_, kProfileCountLastUpdatePref, base::TimeDelta::FromHours(24),
-      base::BindRepeating(&ProfileMetrics::LogNumberOfProfiles, this));
-  repeating_timer_->Start();
 #endif  // !defined(OS_ANDROID) && !BUILDFLAG(IS_CHROMEOS_ASH)
 }
 
@@ -183,9 +178,6 @@ void ProfileInfoCache::AddProfileToCache(ProfileAttributesInitParams params) {
 }
 
 void ProfileInfoCache::DisableProfileMetricsForTesting() {
-#if !defined(OS_ANDROID) && !BUILDFLAG(IS_CHROMEOS_ASH)
-  repeating_timer_.reset();
-#endif
 }
 
 void ProfileInfoCache::NotifyIfProfileNamesHaveChanged() {
