diff --git a/chrome/renderer/searchbox/searchbox.h b/chrome/renderer/searchbox/searchbox.h
index 27839c0190db8..a1dbd0074ee60 100644
--- a/chrome/renderer/searchbox/searchbox.h
+++ b/chrome/renderer/searchbox/searchbox.h
@@ -186,9 +186,6 @@ class SearchBox : public content::RenderFrameObserver,
   // Confirms applied theme changes.
   void ConfirmThemeChanges();
 
-  // Called when a user dismisses a promo.
-  void BlocklistPromo(const std::string& promo_id);
-
   // Handles navigation to the chrome://extensions page by calling the browser
   // to do the navigation.
   void OpenExtensionsPage(double button,
