diff --git a/chrome/browser/resources/new_tab_page/app.js b/chrome/browser/resources/new_tab_page/app.js
index dc2539582c27..0095df3fdfb7 100644
--- a/chrome/browser/resources/new_tab_page/app.js
+++ b/chrome/browser/resources/new_tab_page/app.js
@@ -875,11 +875,6 @@ class AppElement extends PolymerElement {
     }
   }
 
-  /** @private */
-  onMiddleSlotPromoLoaded_() {
-    this.middleSlotPromoLoaded_ = true;
-  }
-
   /** @private */
   onModulesLoaded_() {
     this.modulesLoaded_ = true;
