diff --git a/chrome/browser/devtools/devtools_ui_bindings.h b/chrome/browser/devtools/devtools_ui_bindings.h
index a6213ab97923a..1f3020a94f1e3 100644
--- a/chrome/browser/devtools/devtools_ui_bindings.h
+++ b/chrome/browser/devtools/devtools_ui_bindings.h
@@ -170,10 +170,6 @@ class DevToolsUIBindings : public DevToolsEmbedderMessageDispatcher::Delegate,
   void SetOpenNewWindowForPopups(bool value) override;
   void RegisterExtensionsAPI(const std::string& origin,
                              const std::string& script) override;
-  void ShowSurvey(DispatchCallback callback,
-                  const std::string& trigger) override;
-  void CanShowSurvey(DispatchCallback callback,
-                     const std::string& trigger) override;
 
   void EnableRemoteDeviceCounter(bool enable);
 
