diff --git a/chrome/renderer/searchbox/searchbox.cc b/chrome/renderer/searchbox/searchbox.cc
index bdc2683cbc817..b356f2a029a3c 100644
--- a/chrome/renderer/searchbox/searchbox.cc
+++ b/chrome/renderer/searchbox/searchbox.cc
@@ -433,10 +433,6 @@ void SearchBox::ConfirmThemeChanges() {
   embedded_search_service_->ConfirmThemeChanges();
 }
 
-void SearchBox::BlocklistPromo(const std::string& promo_id) {
-  embedded_search_service_->BlocklistPromo(promo_id);
-}
-
 void SearchBox::OpenExtensionsPage(double button,
                                    bool alt_key,
                                    bool ctrl_key,
