diff --git a/chrome/browser/signin/signin_util.cc b/chrome/browser/signin/signin_util.cc
index 2f7471da65d8..81a87611c831 100644
--- a/chrome/browser/signin/signin_util.cc
+++ b/chrome/browser/signin/signin_util.cc
@@ -93,8 +93,7 @@ class DeleteProfileDialogManager : public BrowserListObserver {
                 gaia::ExtractDomainName(primary_account_email_))));
 
     webui::DeleteProfileAtPath(
-        profile_->GetPath(),
-        ProfileMetrics::DELETE_PROFILE_PRIMARY_ACCOUNT_NOT_ALLOWED);
+        profile_->GetPath());
     delegate_->OnProfileDeleted(this);
   }
 
