diff --git a/chrome/browser/ui/views/page_action/page_action_icon_controller.cc b/chrome/browser/ui/views/page_action/page_action_icon_controller.cc
index 98e91d0961d8..27121e13f7cf 100644
--- a/chrome/browser/ui/views/page_action/page_action_icon_controller.cc
+++ b/chrome/browser/ui/views/page_action/page_action_icon_controller.cc
@@ -52,6 +52,7 @@ void PageActionIconController::Init(const PageActionIconParams& params,
         page_action_icons_.push_back(bookmark_star_icon_);
         break;
       case PageActionIconType::kClickToCall:
+#if BUILDFLAG(ENABLE_CLICK_TO_CALL)
         click_to_call_icon_ = new SharingIconView(
             params.icon_label_bubble_delegate, params.page_action_icon_delegate,
             base::BindRepeating([](content::WebContents* contents) {
@@ -61,6 +62,7 @@ void PageActionIconController::Init(const PageActionIconParams& params,
             }),
             base::BindRepeating(SharingDialogView::GetAsBubbleForClickToCall));
         page_action_icons_.push_back(click_to_call_icon_);
+#endif
         break;
       case PageActionIconType::kCookieControls:
         cookie_controls_icon_ =
