diff --git a/chrome/browser/flag_descriptions.h b/chrome/browser/flag_descriptions.h
index e68b38d6fb8bc..eef88c591743a 100644
--- a/chrome/browser/flag_descriptions.h
+++ b/chrome/browser/flag_descriptions.h
@@ -1366,6 +1366,9 @@ extern const char kStylusBatteryStatusDescription[];
 extern const char kSystemKeyboardLockName[];
 extern const char kSystemKeyboardLockDescription[];
 
+extern const char kTabAudioMutingName[];
+extern const char kTabAudioMutingDescription[];
+
 extern const char kSuggestionsWithSubStringMatchName[];
 extern const char kSuggestionsWithSubStringMatchDescription[];
 
