diff --git a/chrome/credential_provider/setup/BUILD.gn b/chrome/credential_provider/setup/BUILD.gn
index 173d5da26745..79bd46dafbca 100644
--- a/chrome/credential_provider/setup/BUILD.gn
+++ b/chrome/credential_provider/setup/BUILD.gn
@@ -77,10 +77,6 @@ executable("gcp_setup") {
     "../gaiacp:util",
     "//build:branding_buildflags",
     "//chrome/common:version_header",
-    "//components/crash/core/app:app",
-    "//components/crash/core/app:crash_export_thunks",
-    "//components/crash/core/app:run_as_crashpad_handler",
-    "//components/crash/core/common",
     "//content/public/common:static_switches",
   ]
   configs += [ "//build/config/win:windowed" ]
