diff --git a/chrome/common/search/instant_mojom_traits.h b/chrome/common/search/instant_mojom_traits.h
index 46126ceb4a9d..dafc132dba20 100644
--- a/chrome/common/search/instant_mojom_traits.h
+++ b/chrome/common/search/instant_mojom_traits.h
@@ -43,7 +43,6 @@ IPC_STRUCT_TRAITS_BEGIN(ntp_tiles::NTPTileImpression)
   IPC_STRUCT_TRAITS_MEMBER(visual_type)
   IPC_STRUCT_TRAITS_MEMBER(icon_type)
   IPC_STRUCT_TRAITS_MEMBER(data_generation_time)
-  IPC_STRUCT_TRAITS_MEMBER(url_for_rappor)
 IPC_STRUCT_TRAITS_END()
 
 IPC_STRUCT_TRAITS_BEGIN(InstantMostVisitedItem)
