diff --git a/chrome/browser/ui/passwords/bubble_controllers/save_unsynced_credentials_locally_bubble_controller.h b/chrome/browser/ui/passwords/bubble_controllers/save_unsynced_credentials_locally_bubble_controller.h
index 54a8df540c52..9e54ababfe4f 100644
--- a/chrome/browser/ui/passwords/bubble_controllers/save_unsynced_credentials_locally_bubble_controller.h
+++ b/chrome/browser/ui/passwords/bubble_controllers/save_unsynced_credentials_locally_bubble_controller.h
@@ -39,7 +39,6 @@ class SaveUnsyncedCredentialsLocallyBubbleController
   base::string16 GetTitle() const override;
   void ReportInteractions() override;
 
-  password_manager::metrics_util::UIDismissalReason dismissal_reason_;
   std::vector<autofill::PasswordForm> unsynced_credentials_;
 };
 
