diff --git a/chrome/browser/extensions/forced_extensions/force_installed_metrics.cc b/chrome/browser/extensions/forced_extensions/force_installed_metrics.cc
index 8cf8185fe9cc..9a18ef76530e 100644
--- a/chrome/browser/extensions/forced_extensions/force_installed_metrics.cc
+++ b/chrome/browser/extensions/forced_extensions/force_installed_metrics.cc
@@ -257,33 +257,17 @@ void ForceInstalledMetrics::ReportMetrics() {
     // In case of CRX_FETCH_FAILURE, report the network error code, HTTP
     // error code and number of fetch tries made.
     if (failure_reason == FailureReason::CRX_FETCH_FAILED) {
-      base::UmaHistogramSparse("Extensions.ForceInstalledNetworkErrorCode",
-                               installation.network_error_code.value());
 
       if (installation.response_code) {
-        base::UmaHistogramSparse("Extensions.ForceInstalledHttpErrorCode2",
-                                 installation.response_code.value());
       }
-      base::UmaHistogramExactLinear("Extensions.ForceInstalledFetchTries",
-                                    installation.fetch_tries.value(),
-                                    ExtensionDownloader::kMaxRetries);
     }
 
     // In case of MANIFEST_FETCH_FAILURE, report the network error code,
     // HTTP error code and number of fetch tries made.
     if (failure_reason == FailureReason::MANIFEST_FETCH_FAILED) {
-      base::UmaHistogramSparse(
-          "Extensions.ForceInstalledManifestFetchFailedNetworkErrorCode",
-          installation.network_error_code.value());
 
       if (installation.response_code) {
-        base::UmaHistogramSparse(
-            "Extensions.ForceInstalledManifestFetchFailedHttpErrorCode2",
-            installation.response_code.value());
       }
-      base::UmaHistogramExactLinear(
-          "Extensions.ForceInstalledManifestFetchFailedFetchTries",
-          installation.fetch_tries.value(), ExtensionDownloader::kMaxRetries);
     }
 #if defined(OS_CHROMEOS)
     // Report type of user in case Force Installed Extensions fail to
