diff --git a/chrome/browser/ui/browser_window.h b/chrome/browser/ui/browser_window.h
index 2063c3dc22ae..2ffb7d6274a5 100644
--- a/chrome/browser/ui/browser_window.h
+++ b/chrome/browser/ui/browser_window.h
@@ -465,13 +465,6 @@ class BrowserWindow : public ui::BaseWindow {
       signin_metrics::AccessPoint access_point,
       bool is_source_keyboard) = 0;
 
-  // Shows User Happiness Tracking Survey's dialog after the survey associated
-  // with |site_id| has been successfully loaded. Failure to load the survey
-  // will result in the dialog not being shown.
-  virtual void ShowHatsDialog(const std::string& site_id,
-                              base::OnceClosure success_callback,
-                              base::OnceClosure failure_callback) = 0;
-
   // Returns object implementing ExclusiveAccessContext interface.
   virtual ExclusiveAccessContext* GetExclusiveAccessContext() = 0;
 
