diff --git a/chrome/browser/profiles/profile_manager.h b/chrome/browser/profiles/profile_manager.h
index cd6fdbba9b435..b20c2f7db1f1c 100644
--- a/chrome/browser/profiles/profile_manager.h
+++ b/chrome/browser/profiles/profile_manager.h
@@ -211,8 +211,7 @@ class ProfileManager : public Profile::Delegate {
   // profile is either scheduling or marked for deletion.
   void MaybeScheduleProfileForDeletion(
       const base::FilePath& profile_dir,
-      ProfileLoadedCallback callback,
-      ProfileMetrics::ProfileDelete deletion_source);
+      ProfileLoadedCallback callback);
 
   // Schedules the profile at the given path to be deleted on shutdown. If we're
   // deleting the last profile, a new one will be created in its place, and in
