diff --git a/chrome/browser/resources/settings/safety_check_page/BUILD.gn b/chrome/browser/resources/settings/safety_check_page/BUILD.gn
index cb9445790033..051d9b4518e5 100644
--- a/chrome/browser/resources/settings/safety_check_page/BUILD.gn
+++ b/chrome/browser/resources/settings/safety_check_page/BUILD.gn
@@ -73,7 +73,6 @@ js_library("safety_check_page") {
     ":safety_check_passwords_child",
     ":safety_check_safe_browsing_child",
     ":safety_check_updates_child",
-    "..:hats_browser_proxy",
     "..:metrics_browser_proxy",
     "../autofill_page:password_manager_proxy",
     "//third_party/polymer/v3_0/components-chromium/iron-a11y-announcer:iron-a11y-announcer",
