diff --git a/chrome/browser/predictors/loading_data_collector.h b/chrome/browser/predictors/loading_data_collector.h
index 4f0e900ec81c0..e0fd1da9bc8dd 100644
--- a/chrome/browser/predictors/loading_data_collector.h
+++ b/chrome/browser/predictors/loading_data_collector.h
@@ -55,7 +55,6 @@ struct PageRequestSummary {
   void AddPreconnectAttempt(const GURL& preconnect_url);
   void AddPrefetchAttempt(const GURL& prefetch_url);
 
-  ukm::SourceId ukm_source_id;
   GURL main_frame_url;
   GURL initial_url;
   base::TimeTicks navigation_started;
