diff --git a/chrome/browser/resource_coordinator/tab_ranker/BUILD.gn b/chrome/browser/resource_coordinator/tab_ranker/BUILD.gn
index e53fa7a32933..7ae7d304913d 100644
--- a/chrome/browser/resource_coordinator/tab_ranker/BUILD.gn
+++ b/chrome/browser/resource_coordinator/tab_ranker/BUILD.gn
@@ -29,7 +29,6 @@ source_set("tab_ranker") {
     "//components/assist_ranker",
     "//components/assist_ranker/proto",
     "//components/sessions",
-    "//services/metrics/public/cpp:ukm_builders",
   ]
 }
 
