diff --git a/chrome/browser/ui/webui/new_tab_page/new_tab_page_handler.h b/chrome/browser/ui/webui/new_tab_page/new_tab_page_handler.h
index f18b85ad3b17..54d290d7d28e 100644
--- a/chrome/browser/ui/webui/new_tab_page/new_tab_page_handler.h
+++ b/chrome/browser/ui/webui/new_tab_page/new_tab_page_handler.h
@@ -18,8 +18,6 @@
 #include "chrome/browser/search/instant_service_observer.h"
 #include "chrome/browser/search/one_google_bar/one_google_bar_service.h"
 #include "chrome/browser/search/one_google_bar/one_google_bar_service_observer.h"
-#include "chrome/browser/search/promos/promo_service.h"
-#include "chrome/browser/search/promos/promo_service_observer.h"
 #include "chrome/browser/ui/omnibox/omnibox_tab_helper.h"
 #include "chrome/browser/ui/search/ntp_user_data_logger.h"
 #include "chrome/browser/ui/webui/new_tab_page/new_tab_page.mojom.h"
@@ -27,6 +25,7 @@
 #include "chrome/common/search/ntp_logging_events.h"
 #include "components/omnibox/browser/autocomplete_controller.h"
 #include "components/omnibox/browser/favicon_cache.h"
+#include "components/prefs/pref_registry_simple.h"
 #include "components/search_provider_logos/logo_common.h"
 #include "content/public/browser/web_contents_observer.h"
 #include "mojo/public/cpp/bindings/pending_receiver.h"
@@ -55,8 +54,7 @@ class NewTabPageHandler : public new_tab_page::mojom::PageHandler,
                           public OmniboxTabHelper::Observer,
                           public OneGoogleBarServiceObserver,
                           public ui::SelectFileDialog::Listener,
-                          public AutocompleteController::Observer,
-                          public PromoServiceObserver {
+                          public AutocompleteController::Observer {
  public:
   NewTabPageHandler(mojo::PendingReceiver<new_tab_page::mojom::PageHandler>
                         pending_page_handler,
@@ -104,7 +102,6 @@ class NewTabPageHandler : public new_tab_page::mojom::PageHandler,
       ChooseLocalCustomBackgroundCallback callback) override;
   void GetOneGoogleBarParts(const std::string& ogdeb_value,
                             GetOneGoogleBarPartsCallback callback) override;
-  void GetPromo(GetPromoCallback callback) override;
   void OnDismissModule(const std::string& module_id) override;
   void OnRestoreModule(const std::string& module_id) override;
   void SetModulesVisible(bool visible) override;
@@ -114,8 +111,6 @@ class NewTabPageHandler : public new_tab_page::mojom::PageHandler,
       std::vector<new_tab_page::mojom::MostVisitedTilePtr> tiles,
       double time) override;
   void OnOneGoogleBarRendered(double time) override;
-  void OnPromoRendered(double time,
-                       const base::Optional<GURL>& log_url) override;
   void OnMostVisitedTileNavigation(new_tab_page::mojom::MostVisitedTilePtr tile,
                                    uint32_t index,
                                    uint8_t mouse_button,
@@ -134,7 +129,6 @@ class NewTabPageHandler : public new_tab_page::mojom::PageHandler,
   void OnDoodleShared(new_tab_page::mojom::DoodleShareChannel channel,
                       const std::string& doodle_id,
                       const base::Optional<std::string>& share_id) override;
-  void OnPromoLinkClicked() override;
   void OnVoiceSearchAction(
       new_tab_page::mojom::VoiceSearchAction action) override;
   void OnVoiceSearchError(new_tab_page::mojom::VoiceSearchError error) override;
@@ -180,10 +174,6 @@ class NewTabPageHandler : public new_tab_page::mojom::PageHandler,
   void OnOneGoogleBarDataUpdated() override;
   void OnOneGoogleBarServiceShuttingDown() override;
 
-  // PromoServiceObserver:
-  void OnPromoDataUpdated() override;
-  void OnPromoServiceShuttingDown() override;
-
   // SelectFileDialog::Listener:
   void FileSelected(const base::FilePath& path,
                     int index,
@@ -247,11 +237,6 @@ class NewTabPageHandler : public new_tab_page::mojom::PageHandler,
   std::unordered_map<const network::SimpleURLLoader*,
                      std::unique_ptr<network::SimpleURLLoader>>
       loader_map_;
-  std::vector<GetPromoCallback> promo_callbacks_;
-  PromoService* promo_service_;
-  base::ScopedObservation<PromoService, PromoServiceObserver>
-      promo_service_observation_{this};
-  base::Optional<base::TimeTicks> promo_load_start_time_;
 
   // These are located at the end of the list of member variables to ensure the
   // WebUI page is disconnected before other members are destroyed.
