diff --git a/chrome/browser/ui/search/search_ipc_router_policy_impl.cc b/chrome/browser/ui/search/search_ipc_router_policy_impl.cc
index 91e6d92abddea..2f5ab09ed377e 100644
--- a/chrome/browser/ui/search/search_ipc_router_policy_impl.cc
+++ b/chrome/browser/ui/search/search_ipc_router_policy_impl.cc
@@ -127,10 +127,6 @@ bool SearchIPCRouterPolicyImpl::ShouldProcessThemeChangeMessages() {
   return !is_incognito_ && search::IsInstantNTP(web_contents_);
 }
 
-bool SearchIPCRouterPolicyImpl::ShouldProcessBlocklistPromo() {
-  return !is_incognito_ && search::IsInstantNTP(web_contents_);
-}
-
 bool SearchIPCRouterPolicyImpl::ShouldProcessOpenExtensionsPage() {
   return !is_incognito_ && search::IsInstantNTP(web_contents_);
 }
