diff --git a/chrome/browser/payments/BUILD.gn b/chrome/browser/payments/BUILD.gn
index d362eb062ec9..c1a748c6679f 100644
--- a/chrome/browser/payments/BUILD.gn
+++ b/chrome/browser/payments/BUILD.gn
@@ -43,7 +43,6 @@ source_set("browsertests") {
     "//components/payments/content:utils",
     "//components/payments/core",
     "//components/payments/core:test_support",
-    "//components/ukm:test_support",
     "//device/fido:test_support",
   ]
 
