diff --git a/chrome/browser/ui/passwords/manage_passwords_ui_controller.h b/chrome/browser/ui/passwords/manage_passwords_ui_controller.h
index 1362eecd9ccb..f504e92366dd 100644
--- a/chrome/browser/ui/passwords/manage_passwords_ui_controller.h
+++ b/chrome/browser/ui/passwords/manage_passwords_ui_controller.h
@@ -119,8 +119,6 @@ class ManagePasswordsUIController
   // PasswordsModelDelegate:
   content::WebContents* GetWebContents() const override;
   url::Origin GetOrigin() const override;
-  password_manager::PasswordFormMetricsRecorder*
-  GetPasswordFormMetricsRecorder() override;
   password_manager::PasswordFeatureManager* GetPasswordFeatureManager()
       override;
   password_manager::ui::State GetState() const override;
