diff --git a/chrome/browser/media/webrtc/permission_bubble_media_access_handler.cc b/chrome/browser/media/webrtc/permission_bubble_media_access_handler.cc
index 819fb30d05c4..a3c9bbcfc29a 100644
--- a/chrome/browser/media/webrtc/permission_bubble_media_access_handler.cc
+++ b/chrome/browser/media/webrtc/permission_bubble_media_access_handler.cc
@@ -32,7 +32,6 @@
 #include <vector>
 
 #include "chrome/browser/media/webrtc/screen_capture_infobar_delegate_android.h"
-#include "components/permissions/permission_uma_util.h"
 #include "components/permissions/permission_util.h"
 #include "content/public/common/content_features.h"
 #endif  // defined(OS_ANDROID)
