diff --git a/chrome/browser/flags/android/chrome_feature_list.cc b/chrome/browser/flags/android/chrome_feature_list.cc
index fc48bb2beae8..b5a64cf0732f 100644
--- a/chrome/browser/flags/android/chrome_feature_list.cc
+++ b/chrome/browser/flags/android/chrome_feature_list.cc
@@ -16,7 +16,6 @@
 #include "chrome/browser/performance_hints/performance_hints_features.h"
 #include "chrome/browser/share/features.h"
 #include "chrome/browser/sharing/shared_clipboard/feature_flags.h"
-#include "chrome/browser/video_tutorials/switches.h"
 #include "chrome/common/chrome_features.h"
 #include "components/autofill/core/common/autofill_features.h"
 #include "components/autofill/core/common/autofill_payments_features.h"
@@ -260,8 +259,6 @@ const base::Feature* kFeaturesExposedToJava[] = {
     &signin::kMobileIdentityConsistency,
     &switches::kDecoupleSyncFromAndroidMasterSync,
     &switches::kSyncUseSessionsUnregisterDelay,
-    &subresource_filter::kSafeBrowsingSubresourceFilter,
-    &video_tutorials::features::kVideoTutorials,
 };
 
 const base::Feature* FindFeatureExposedToJava(const std::string& feature_name) {
