diff --git a/chrome/browser/content_index/content_index_provider_impl.h b/chrome/browser/content_index/content_index_provider_impl.h
index 9d97070c22236..23b67ac5dd397 100644
--- a/chrome/browser/content_index/content_index_provider_impl.h
+++ b/chrome/browser/content_index/content_index_provider_impl.h
@@ -99,7 +99,6 @@ class ContentIndexProviderImpl
       const content::ContentIndexEntry& entry);
 
   Profile* profile_;
-  ContentIndexMetrics metrics_;
   offline_items_collection::OfflineContentAggregator* aggregator_;
   site_engagement::SiteEngagementService* site_engagement_service_;
   absl::optional<std::vector<gfx::Size>> icon_sizes_for_testing_;
