diff --git a/chromecast/browser/extensions/api/tabs/tabs_constants.cc b/chromecast/browser/extensions/api/tabs/tabs_constants.cc
index eeabb9e849b8..a2da828b1419 100644
--- a/chromecast/browser/extensions/api/tabs/tabs_constants.cc
+++ b/chromecast/browser/extensions/api/tabs/tabs_constants.cc
@@ -113,6 +113,8 @@ const char kScreenshotsDisabled[] = "Taking screenshots has been disabled";
 const char kCannotUpdateMuteCaptured[] =
     "Cannot update mute state for tab *, tab has audio or video currently "
     "being captured";
+const char kCannotUpdateMuteDisabled[] =
+    "Failed to update mute state for tab *, --* must be enabled";
 const char kCannotDetermineLanguageOfUnloadedTab[] =
     "Cannot determine language: tab not loaded";
 const char kMissingLockWindowFullscreenPrivatePermission[] =
