diff --git a/chrome/browser/ui/passwords/bubble_controllers/save_unsynced_credentials_locally_bubble_controller.h b/chrome/browser/ui/passwords/bubble_controllers/save_unsynced_credentials_locally_bubble_controller.h
index 07f1d68c7d65e..69b84e24b1570 100644
--- a/chrome/browser/ui/passwords/bubble_controllers/save_unsynced_credentials_locally_bubble_controller.h
+++ b/chrome/browser/ui/passwords/bubble_controllers/save_unsynced_credentials_locally_bubble_controller.h
@@ -44,7 +44,6 @@ class SaveUnsyncedCredentialsLocallyBubbleController
   std::u16string GetTitle() const override;
   void ReportInteractions() override;
 
-  password_manager::metrics_util::UIDismissalReason dismissal_reason_;
   std::vector<password_manager::PasswordForm> unsynced_credentials_;
 };
 
