diff --git a/chrome/browser/metrics/chrome_feature_list_creator.cc b/chrome/browser/metrics/chrome_feature_list_creator.cc
index 88fdfb8d627b..c3a8d1a6c19b 100644
--- a/chrome/browser/metrics/chrome_feature_list_creator.cc
+++ b/chrome/browser/metrics/chrome_feature_list_creator.cc
@@ -40,7 +40,6 @@
 #include "components/prefs/pref_service_factory.h"
 #include "components/variations/pref_names.h"
 #include "components/variations/service/variations_service.h"
-#include "components/variations/variations_crash_keys.h"
 #include "content/public/common/content_switch_dependent_feature_overrides.h"
 #include "services/service_manager/embedder/result_codes.h"
 #include "ui/base/resource/resource_bundle.h"
@@ -187,7 +186,6 @@ void ChromeFeatureListCreator::SetupFieldTrials() {
       content::GetSwitchDependentFeatureOverrides(
           *base::CommandLine::ForCurrentProcess()),
       std::move(feature_list), browser_field_trials_.get());
-  variations::InitCrashKeys();
 }
 
 void ChromeFeatureListCreator::CreateMetricsServices() {
