diff --git a/chrome/browser/android/vr/BUILD.gn b/chrome/browser/android/vr/BUILD.gn
index 4292683f57a8..122ce5e692fe 100644
--- a/chrome/browser/android/vr/BUILD.gn
+++ b/chrome/browser/android/vr/BUILD.gn
@@ -104,7 +104,6 @@ static_library("vr_android") {
     "//components/omnibox/browser",
     "//components/page_info",
     "//components/permissions",
-    "//components/rappor",
     "//components/search_engines:search_engines",
     "//content/public/browser",
     "//content/public/common",
