diff --git a/chrome/app/generated_resources.grd b/chrome/app/generated_resources.grd
index fb19e855f5f40..74eaf76161a00 100644
--- a/chrome/app/generated_resources.grd
+++ b/chrome/app/generated_resources.grd
@@ -308,6 +308,8 @@ are declared in tools/grit/grit_rule.gni.
       <!-- Settings specific strings -->
       <if expr="not is_android">
         <part file="settings_strings.grdp" />
+        <!-- Dumb Browser's settings specific strings -->
+        <part file="dumb_settings_strings.grdp" />
         <part file="shared_settings_strings.grdp"/>
       </if>
 
