diff --git a/chrome/browser/ui/views/page_info/safety_tip_page_info_bubble_view.cc b/chrome/browser/ui/views/page_info/safety_tip_page_info_bubble_view.cc
index fbfb79a207d04..20b1bfcbffa14 100644
--- a/chrome/browser/ui/views/page_info/safety_tip_page_info_bubble_view.cc
+++ b/chrome/browser/ui/views/page_info/safety_tip_page_info_bubble_view.cc
@@ -6,8 +6,6 @@
 
 #include "base/bind.h"
 #include "chrome/browser/platform_util.h"
-#include "chrome/browser/reputation/reputation_service.h"
-#include "chrome/browser/reputation/safety_tip_ui_helper.h"
 #include "chrome/browser/ui/browser.h"
 #include "chrome/browser/ui/browser_finder.h"
 #include "chrome/browser/ui/browser_window.h"
@@ -36,6 +34,7 @@ namespace {
 
 int GetSafetyTipBannerId(security_state::SafetyTipStatus safety_tip_status,
                          bool is_dark) {
+#if 0
   switch (safety_tip_status) {
     case security_state::SafetyTipStatus::kBadReputation:
     case security_state::SafetyTipStatus::kLookalike:
@@ -50,6 +49,8 @@ int GetSafetyTipBannerId(security_state::SafetyTipStatus safety_tip_status,
       NOTREACHED();
       return 0;
   }
+#endif
+  return 0;
 }
 
 }  // namespace
@@ -74,9 +75,11 @@ SafetyTipPageInfoBubbleView::SafetyTipPageInfoBubbleView(
   // created over it, etc).
   set_close_on_deactivate(false);
 
+#if 0
   const std::u16string title_text =
       GetSafetyTipTitle(safety_tip_status, suggested_url);
   SetTitle(title_text);
+#endif
 
   views::BubbleDialogDelegateView::CreateBubble(this);
 
@@ -93,11 +96,6 @@ SafetyTipPageInfoBubbleView::SafetyTipPageInfoBubbleView(
       kSizeDeltaInPixels, gfx::Font::FontStyle::NORMAL,
       gfx::Font::Weight::NORMAL);
 
-  auto new_title = std::make_unique<views::StyledLabel>();
-  new_title->SetText(title_text);
-  new_title->AddStyleRange(gfx::Range(0, title_text.length()), name_style);
-  GetBubbleFrameView()->SetTitleView(std::move(new_title));
-
   ChromeLayoutProvider* layout_provider = ChromeLayoutProvider::Get();
 
   gfx::Insets insets = layout_provider->GetDialogInsetsForContentType(
@@ -142,15 +140,6 @@ SafetyTipPageInfoBubbleView::SafetyTipPageInfoBubbleView(
       layout_provider->GetDistanceMetric(DISTANCE_CONTROL_LIST_VERTICAL);
   bottom_layout->StartRowWithPadding(views::GridLayout::kFixedSize, kColumnId,
                                      views::GridLayout::kFixedSize, spacing);
-  auto text_label = std::make_unique<views::Label>(
-      GetSafetyTipDescription(safety_tip_status, suggested_url_));
-  text_label->SetMultiLine(true);
-  text_label->SetLineHeight(20);
-  text_label->SetHorizontalAlignment(gfx::ALIGN_LEFT);
-  text_label->SizeToFit(layout_provider->GetDistanceMetric(
-                            views::DISTANCE_BUBBLE_PREFERRED_WIDTH) -
-                        insets.left() - insets.right());
-  bottom_layout->AddView(std::move(text_label));
 
   MaybeAddButtons(safety_tip_status, bottom_layout, spacing, kColumnId, insets);
 
@@ -196,16 +185,10 @@ void SafetyTipPageInfoBubbleView::OnWidgetDestroying(views::Widget* widget) {
 
 void SafetyTipPageInfoBubbleView::ExecuteLeaveCommand() {
   action_taken_ = SafetyTipInteraction::kLeaveSite;
-  LeaveSiteFromSafetyTip(
-      web_contents(),
-      safety_tip_status_ == security_state::SafetyTipStatus::kLookalike
-          ? suggested_url_
-          : GURL());
 }
 
 void SafetyTipPageInfoBubbleView::OpenHelpCenter() {
   action_taken_ = SafetyTipInteraction::kLearnMore;
-  OpenHelpCenterFromSafetyTip(web_contents());
 }
 
 void SafetyTipPageInfoBubbleView::RenderFrameDeleted(
@@ -304,8 +287,7 @@ void SafetyTipPageInfoBubbleView::MaybeAddButtons(
           [](SafetyTipPageInfoBubbleView* view) {
             view->ExecuteLeaveCommand();
           },
-          this),
-      l10n_util::GetStringUTF16(GetSafetyTipLeaveButtonId(safety_tip_status)));
+          this));
   leave_button->SetProminent(true);
   leave_button->SetID(PageInfoBubbleView::VIEW_ID_PAGE_INFO_BUTTON_LEAVE_SITE);
   leave_button_ = button_layout->AddView(std::move(leave_button));
