diff --git a/chrome/browser/background_sync/background_sync_controller_impl.h b/chrome/browser/background_sync/background_sync_controller_impl.h
index 86091caa6b00..363c2128abde 100644
--- a/chrome/browser/background_sync/background_sync_controller_impl.h
+++ b/chrome/browser/background_sync/background_sync_controller_impl.h
@@ -14,7 +14,6 @@
 #include "base/macros.h"
 #include "base/time/time.h"
 #include "build/build_config.h"
-#include "chrome/browser/background_sync/background_sync_metrics.h"
 #include "chrome/browser/engagement/site_engagement_observer.h"
 #include "components/content_settings/core/browser/content_settings_observer.h"
 #include "components/keep_alive_registry/keep_alive_types.h"
@@ -161,8 +160,6 @@ class BackgroundSyncControllerImpl : public content::BackgroundSyncController,
   // Same lifetime as |profile_|.
   SiteEngagementService* site_engagement_service_;
 
-  BackgroundSyncMetrics background_sync_metrics_;
-
   std::set<url::Origin> suspended_periodic_sync_origins_;
   std::set<url::Origin> periodic_sync_origins_;
 
