diff --git a/chrome/browser/ui/webui/signin/dice_web_signin_intercept_handler.cc b/chrome/browser/ui/webui/signin/dice_web_signin_intercept_handler.cc
index a223bfb94223..6ba05f597d4c 100644
--- a/chrome/browser/ui/webui/signin/dice_web_signin_intercept_handler.cc
+++ b/chrome/browser/ui/webui/signin/dice_web_signin_intercept_handler.cc
@@ -8,7 +8,6 @@
 #include "base/metrics/field_trial_params.h"
 #include "base/strings/utf_string_conversions.h"
 #include "chrome/browser/browser_process.h"
-#include "chrome/browser/enterprise/browser_management/browser_management_service.h"
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/profiles/profile_attributes_entry.h"
 #include "chrome/browser/profiles/profile_attributes_storage.h"
@@ -205,12 +204,7 @@ bool DiceWebSigninInterceptHandler::ShouldShowManagedDeviceVersion() const {
   // - or anticipating that the user may enable Sync in the new profile and
   //   check the cloud policies attached to the intercepted account (requires
   //   network requests).
-  return policy::PlatformManagementService::GetInstance()
-                 .GetManagementAuthorityTrustworthiness() >
-             policy::ManagementAuthorityTrustworthiness::NONE ||
-         policy::BrowserManagementService(Profile::FromWebUI(web_ui()))
-                 .GetManagementAuthorityTrustworthiness() >
-             policy::ManagementAuthorityTrustworthiness::NONE;
+  return false;
 }
 
 std::string DiceWebSigninInterceptHandler::GetHeaderText() {
