diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 8ca2f3bdf836..e1985372293d 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -155,7 +155,6 @@ static_library("utility") {
 
   if (is_chromeos) {
     deps += [
-      "//chrome/services/file_util",
       "//chrome/services/sharing",
       "//chrome/services/sharing/public/mojom:mojom",
       "//chromeos/assistant:buildflags",
