diff --git a/chrome/browser/prerender/isolated/isolated_prerender_network_context_client.cc b/chrome/browser/prerender/isolated/isolated_prerender_network_context_client.cc
index 0cc60256b753..b2fc0b9cb06a 100644
--- a/chrome/browser/prerender/isolated/isolated_prerender_network_context_client.cc
+++ b/chrome/browser/prerender/isolated/isolated_prerender_network_context_client.cc
@@ -97,6 +97,3 @@ void IsolatedPrerenderNetworkContextClient::OnGenerateHttpNegotiateAuthToken(
 #if defined(OS_CHROMEOS)
 void IsolatedPrerenderNetworkContextClient::OnTrustAnchorUsed() {}
 #endif
-
-void IsolatedPrerenderNetworkContextClient::OnSCTReportReady(
-    const std::string& cache_key) {}
