diff --git a/chrome/browser/optimization_guide/optimization_guide_web_contents_observer.cc b/chrome/browser/optimization_guide/optimization_guide_web_contents_observer.cc
index 7ba9cd0941b7..63de4766aa1c 100644
--- a/chrome/browser/optimization_guide/optimization_guide_web_contents_observer.cc
+++ b/chrome/browser/optimization_guide/optimization_guide_web_contents_observer.cc
@@ -13,7 +13,6 @@
 #include "components/optimization_guide/core/optimization_guide_enums.h"
 #include "components/optimization_guide/core/optimization_guide_features.h"
 #include "components/optimization_guide/proto/hints.pb.h"
-#include "components/page_load_metrics/common/page_load_metrics.mojom.h"
 #include "content/public/browser/browser_thread.h"
 #include "content/public/browser/navigation_handle.h"
 
@@ -141,16 +140,6 @@ void OptimizationGuideWebContentsObserver::NotifyNavigationFinish(
   inflight_optimization_guide_navigation_datas_.erase(navigation_id);
 }
 
-void OptimizationGuideWebContentsObserver::UpdateSessionTimingStatistics(
-    const page_load_metrics::mojom::PageLoadTiming& timing) {
-  DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
-  if (!optimization_guide_keyed_service_)
-    return;
-
-  optimization_guide_keyed_service_->UpdateSessionFCP(
-      timing.paint_timing->first_contentful_paint.value());
-}
-
 void OptimizationGuideWebContentsObserver::FlushLastNavigationData() {
   if (last_navigation_data_)
     last_navigation_data_.reset();
