diff --git a/chrome/browser/web_applications/web_app_provider_factory.cc b/chrome/browser/web_applications/web_app_provider_factory.cc
index 1237beafed1a9..d5b0a5f84553d 100644
--- a/chrome/browser/web_applications/web_app_provider_factory.cc
+++ b/chrome/browser/web_applications/web_app_provider_factory.cc
@@ -4,7 +4,6 @@
 
 #include "chrome/browser/web_applications/web_app_provider_factory.h"
 
-#include "chrome/browser/metrics/ukm_background_recorder_service.h"
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/sync/model_type_store_service_factory.h"
 #include "chrome/browser/web_applications/components/web_app_utils.h"
@@ -32,7 +31,6 @@ WebAppProviderFactory::WebAppProviderFactory()
   WebAppProviderBaseFactory::SetInstance(this);
   DependsOnExtensionsSystem();
   DependsOn(ModelTypeStoreServiceFactory::GetInstance());
-  DependsOn(ukm::UkmBackgroundRecorderFactory::GetInstance());
 }
 
 WebAppProviderFactory::~WebAppProviderFactory() {
