diff --git a/chrome/browser/metrics/chrome_browser_main_extra_parts_metrics.cc b/chrome/browser/metrics/chrome_browser_main_extra_parts_metrics.cc
index 5174562ac1af..2abff096cbbd 100644
--- a/chrome/browser/metrics/chrome_browser_main_extra_parts_metrics.cc
+++ b/chrome/browser/metrics/chrome_browser_main_extra_parts_metrics.cc
@@ -88,24 +88,6 @@
 
 namespace {
 
-void RecordMemoryMetrics();
-
-// Records memory metrics after a delay.
-void RecordMemoryMetricsAfterDelay() {
-  content::GetUIThreadTaskRunner({})->PostDelayedTask(
-      FROM_HERE, base::BindOnce(&RecordMemoryMetrics),
-      memory_instrumentation::GetDelayForNextMemoryLog());
-}
-
-// Records memory metrics, and then triggers memory colleciton after a delay.
-void RecordMemoryMetrics() {
-  scoped_refptr<ProcessMemoryMetricsEmitter> emitter(
-      new ProcessMemoryMetricsEmitter);
-  emitter->FetchAndEmitProcessMemoryMetrics();
-
-  RecordMemoryMetricsAfterDelay();
-}
-
 // These values are written to logs.  New enum values can be added, but existing
 // enums must never be renumbered or deleted and reused.
 enum UMALinuxDistro {
@@ -615,7 +597,6 @@ void ChromeBrowserMainExtraPartsMetrics::PreBrowserStart() {
 }
 
 void ChromeBrowserMainExtraPartsMetrics::PostBrowserStart() {
-  RecordMemoryMetricsAfterDelay();
   RecordLinuxGlibcVersion();
 #if defined(USE_X11)
   if (!features::IsUsingOzonePlatform()) {
