diff --git a/chrome/browser/web_applications/components/BUILD.gn b/chrome/browser/web_applications/components/BUILD.gn
index 031345da1811..f326e5f759b3 100644
--- a/chrome/browser/web_applications/components/BUILD.gn
+++ b/chrome/browser/web_applications/components/BUILD.gn
@@ -21,8 +21,6 @@ source_set("components") {
     "externally_installed_web_app_prefs.h",
     "file_handler_manager.cc",
     "file_handler_manager.h",
-    "install_bounce_metric.cc",
-    "install_bounce_metric.h",
     "install_finalizer.cc",
     "install_finalizer.h",
     "install_manager.cc",
