diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 370ea00c87c0b..927ab7521f1ea 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -10,6 +10,7 @@ import("//chrome/browser/resources/pdf/ink/ink.gni")
 import("//chrome/common/features.gni")
 import("//chromeos/components/chromebox_for_meetings/buildflags/buildflags.gni")
 import("//components/nacl/features.gni")
+import("//dumb/browser/extensions/sources.gni")
 import("//extensions/buildflags/buildflags.gni")
 import("//mojo/public/tools/bindings/mojom.gni")
 import("//pdf/features.gni")
@@ -150,14 +151,10 @@ static_library("extensions") {
     "api/downloads_internal/downloads_internal_api.h",
     "api/enterprise_hardware_platform/enterprise_hardware_platform_api.cc",
     "api/enterprise_hardware_platform/enterprise_hardware_platform_api.h",
-    "api/enterprise_reporting_private/enterprise_reporting_private_api.cc",
-    "api/enterprise_reporting_private/enterprise_reporting_private_api.h",
     "api/extension_action/extension_action_api.cc",
     "api/extension_action/extension_action_api.h",
     "api/extension_action/extension_page_actions_api_constants.cc",
     "api/extension_action/extension_page_actions_api_constants.h",
-    "api/feedback_private/chrome_feedback_private_delegate.cc",
-    "api/feedback_private/chrome_feedback_private_delegate.h",
     "api/file_system/chrome_file_system_delegate.cc",
     "api/file_system/chrome_file_system_delegate.h",
     "api/file_system/file_entry_picker.cc",
@@ -309,14 +306,6 @@ static_library("extensions") {
     "api/resources_private/resources_private_api.h",
     "api/runtime/chrome_runtime_api_delegate.cc",
     "api/runtime/chrome_runtime_api_delegate.h",
-    "api/safe_browsing_private/safe_browsing_private_api.cc",
-    "api/safe_browsing_private/safe_browsing_private_api.h",
-    "api/safe_browsing_private/safe_browsing_private_event_router.cc",
-    "api/safe_browsing_private/safe_browsing_private_event_router.h",
-    "api/safe_browsing_private/safe_browsing_private_event_router_factory.cc",
-    "api/safe_browsing_private/safe_browsing_private_event_router_factory.h",
-    "api/safe_browsing_private/safe_browsing_util.cc",
-    "api/safe_browsing_private/safe_browsing_util.h",
     "api/scripting/scripting_api.cc",
     "api/scripting/scripting_api.h",
     "api/search/search_api.cc",
@@ -617,8 +606,6 @@ static_library("extensions") {
     "external_pref_loader.h",
     "external_provider_impl.cc",
     "external_provider_impl.h",
-    "forced_extensions/force_installed_metrics.cc",
-    "forced_extensions/force_installed_metrics.h",
     "forced_extensions/force_installed_tracker.cc",
     "forced_extensions/force_installed_tracker.h",
     "forced_extensions/install_stage_tracker.cc",
@@ -648,8 +635,6 @@ static_library("extensions") {
     "launch_util.h",
     "load_error_reporter.cc",
     "load_error_reporter.h",
-    "media_router_extension_access_logger_impl.cc",
-    "media_router_extension_access_logger_impl.h",
     "menu_manager.cc",
     "menu_manager.h",
     "menu_manager_factory.cc",
@@ -674,8 +659,6 @@ static_library("extensions") {
     "policy_handlers.h",
     "proxy_overridden_bubble_delegate.cc",
     "proxy_overridden_bubble_delegate.h",
-    "safe_browsing_verdict_handler.cc",
-    "safe_browsing_verdict_handler.h",
     "scoped_active_install.cc",
     "scoped_active_install.h",
     "scripting_permissions_modifier.cc",
@@ -755,9 +738,6 @@ static_library("extensions") {
 
     # TODO(crbug.com/1065748): Remove this circular dependency.
     "//chrome/browser/web_applications/extensions",
-
-    # TODO(crbug/925153): Remove this circular dependency.
-    "//chrome/browser/safe_browsing",
   ]
 
   # Since browser and browser_extensions actually depend on each other,
@@ -770,8 +750,6 @@ static_library("extensions") {
     "//chrome/common",
     "//chrome/common/extensions/api",
     "//components/omnibox/browser",
-    "//components/safe_browsing/core:csd_proto",
-    "//components/safe_browsing/core/db:util",
     "//components/signin/core/browser",
     "//components/translate/content/browser",
     "//content/public/browser",
@@ -804,8 +782,6 @@ static_library("extensions") {
     "//chrome/browser/resource_coordinator:intervention_policy_database_proto",
     "//chrome/browser/resource_coordinator:mojo_bindings",
     "//chrome/browser/resources/pdf/ink:buildflags",
-    "//chrome/browser/safe_browsing",
-    "//chrome/browser/safe_browsing:metrics_collector",
     "//chrome/browser/web_applications",
     "//chrome/browser/web_applications/components",
     "//components/site_engagement/core/mojom:mojo_bindings",
@@ -814,8 +790,6 @@ static_library("extensions") {
     # TODO(crbug.com/1065748): Remove this dependency:
     "//chrome/browser/web_applications/extensions",
     "//chrome/common/extensions/api:extensions_features",
-    "//chrome/common/safe_browsing:proto",
-    "//chrome/services/file_util/public/mojom:mojom",
     "//chrome/services/removable_storage_writer/public/mojom",
     "//components/autofill/content/browser",
     "//components/back_forward_cache",
@@ -833,9 +807,7 @@ static_library("extensions") {
     "//components/drive",
     "//components/embedder_support",
     "//components/embedder_support:browser_util",
-    "//components/enterprise",
     "//components/favicon/content",
-    "//components/feedback",
     "//components/gcm_driver",
     "//components/guest_view/browser",
     "//components/history/core/browser",
@@ -851,7 +823,6 @@ static_library("extensions") {
     "//components/net_log",
     "//components/no_state_prefetch/browser",
     "//components/onc",
-    "//components/page_load_metrics/browser:browser",
     "//components/password_manager/core/browser",
     "//components/password_manager/core/browser:affiliation",
     "//components/password_manager/core/browser/leak_detection",
@@ -865,12 +836,6 @@ static_library("extensions") {
     "//components/privacy_sandbox:privacy_sandbox_prefs",
     "//components/proxy_config",
     "//components/resources",
-    "//components/safe_browsing:buildflags",
-    "//components/safe_browsing/content/web_ui:web_ui",
-    "//components/safe_browsing/core:csd_proto",
-    "//components/safe_browsing/core:features",
-    "//components/safe_browsing/core/common:safe_browsing_prefs",
-    "//components/safe_browsing/core/db:database_manager",
     "//components/search_engines",
     "//components/services/app_service/public/mojom",
     "//components/services/patch/content",
@@ -889,7 +854,6 @@ static_library("extensions") {
     "//components/translate/core/browser",
     "//components/translate/core/common",
     "//components/undo",
-    "//components/unified_consent",
     "//components/update_client",
     "//components/update_client:common_impl",
     "//components/url_formatter",
@@ -922,7 +886,6 @@ static_library("extensions") {
     "//services/audio/public/cpp",
     "//services/data_decoder/public/cpp",
     "//services/device/public/mojom",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/mojom",
     "//services/service_manager/public/cpp",
     "//services/service_manager/public/mojom",
@@ -958,6 +921,8 @@ static_library("extensions") {
     "//url",
   ]
 
+  deps += dumb_browser_extensions_deps
+
   if (is_chromeos_ash) {
     deps += [ "//chromeos/resources:media_app_bundle_resources_grit" ]
   }
@@ -1126,7 +1091,6 @@ static_library("extensions") {
       "//chromeos/ui/base",
       "//components/arc",
       "//components/constrained_window",
-      "//components/crash/content/browser/error_reporting",
       "//components/drive",
       "//components/user_manager",
       "//media/capture:capture_lib",
@@ -1152,10 +1116,6 @@ static_library("extensions") {
   } else {
     assert(!is_chromeos_ash)
     sources += [
-      "api/enterprise_reporting_private/chrome_desktop_report_request_helper.cc",
-      "api/enterprise_reporting_private/chrome_desktop_report_request_helper.h",
-      "api/enterprise_reporting_private/keychain_data_helper_mac.h",
-      "api/enterprise_reporting_private/keychain_data_helper_mac.mm",
       "api/image_writer_private/operation_nonchromeos.cc",
       "api/tabs/tabs_util.cc",
       "chrome_kiosk_delegate.cc",
@@ -1163,8 +1123,6 @@ static_library("extensions") {
       "preinstalled_apps.h",
     ]
     deps += [
-      "//components/enterprise",
-      "//components/enterprise/common/proto:connectors_proto",
       "//components/keep_alive_registry",
     ]
     if (is_linux || is_chromeos) {
