diff --git a/chrome/common/features.gni b/chrome/common/features.gni
index e648b93a7453..054814f46d9a 100644
--- a/chrome/common/features.gni
+++ b/chrome/common/features.gni
@@ -87,6 +87,5 @@ chrome_grit_defines = [
   "enable_tab_search=$enable_tab_search",
   "enable_vr=$enable_vr",
   "enable_webui_tab_strip=$enable_webui_tab_strip",
-  "safe_browsing_mode=$safe_browsing_mode",
   "optimize_webui=$optimize_webui",
 ]
