diff --git a/chrome/browser/tab_contents/navigation_metrics_recorder.cc b/chrome/browser/tab_contents/navigation_metrics_recorder.cc
index dfdd6924d2bb..151c44116a9d 100644
--- a/chrome/browser/tab_contents/navigation_metrics_recorder.cc
+++ b/chrome/browser/tab_contents/navigation_metrics_recorder.cc
@@ -9,7 +9,6 @@
 #include "chrome/browser/engagement/site_engagement_service.h"
 #include "chrome/browser/metrics/chrome_metrics_service_accessor.h"
 #include "chrome/browser/profiles/profile.h"
-#include "chrome/browser/profiles/profile_metrics.h"
 #include "components/navigation_metrics/navigation_metrics.h"
 #include "content/public/browser/browser_context.h"
 #include "content/public/browser/browser_thread.h"
@@ -83,9 +82,6 @@ void NavigationMetricsRecorder::DidFinishNavigation(
 
   const GURL url = last_committed_entry->GetVirtualURL();
   Profile* profile = Profile::FromBrowserContext(context);
-  navigation_metrics::RecordMainFrameNavigation(
-      url, navigation_handle->IsSameDocument(), profile->IsOffTheRecord(),
-      ProfileMetrics::GetBrowserProfileType(profile));
 
   if (url.SchemeIsHTTPOrHTTPS() && !navigation_handle->IsSameDocument() &&
       !navigation_handle->IsDownload() && !profile->IsOffTheRecord()) {
