diff --git a/chrome/browser/profiles/profile_activity_metrics_recorder.cc b/chrome/browser/profiles/profile_activity_metrics_recorder.cc
index 4cb3145e4ebe..bdcdf5016657 100644
--- a/chrome/browser/profiles/profile_activity_metrics_recorder.cc
+++ b/chrome/browser/profiles/profile_activity_metrics_recorder.cc
@@ -209,7 +209,6 @@ void ProfileActivityMetricsRecorder::OnProfileWillBeDestroyed(
 
 ProfileActivityMetricsRecorder::ProfileActivityMetricsRecorder() {
   BrowserList::AddObserver(this);
-  metrics::DesktopSessionDurationTracker::Get()->AddObserver(this);
   action_callback_ = base::BindRepeating(
       &ProfileActivityMetricsRecorder::OnUserAction, base::Unretained(this));
   base::AddActionCallback(action_callback_);
@@ -217,7 +216,6 @@ ProfileActivityMetricsRecorder::ProfileActivityMetricsRecorder() {
 
 ProfileActivityMetricsRecorder::~ProfileActivityMetricsRecorder() {
   BrowserList::RemoveObserver(this);
-  metrics::DesktopSessionDurationTracker::Get()->RemoveObserver(this);
   base::RemoveActionCallback(action_callback_);
 }
 
