diff --git a/chrome/browser/predictors/loading_predictor.h b/chrome/browser/predictors/loading_predictor.h
index 57d1e9521fb25..fbdd365fa033d 100644
--- a/chrome/browser/predictors/loading_predictor.h
+++ b/chrome/browser/predictors/loading_predictor.h
@@ -29,7 +29,6 @@ class Profile;
 namespace predictors {
 
 class ResourcePrefetchPredictor;
-class LoadingStatsCollector;
 
 // Entry point for the Loading predictor.
 // From a high-level request (GURL and motivation) and a database of historical
@@ -68,7 +67,6 @@ class LoadingPredictor : public KeyedService,
 
   // Don't use, internal only.
   ResourcePrefetchPredictor* resource_prefetch_predictor();
-  LoadingDataCollector* loading_data_collector();
   PreconnectManager* preconnect_manager();
   PrefetchManager* prefetch_manager();
 
@@ -158,17 +156,9 @@ class LoadingPredictor : public KeyedService,
     preconnect_manager_ = std::move(preconnect_manager);
   }
 
-  // For testing.
-  void set_mock_loading_data_collector(
-      std::unique_ptr<LoadingDataCollector> loading_data_collector) {
-    loading_data_collector_ = std::move(loading_data_collector);
-  }
-
   LoadingPredictorConfig config_;
   Profile* profile_;
   std::unique_ptr<ResourcePrefetchPredictor> resource_prefetch_predictor_;
-  std::unique_ptr<LoadingStatsCollector> stats_collector_;
-  std::unique_ptr<LoadingDataCollector> loading_data_collector_;
   std::unique_ptr<PreconnectManager> preconnect_manager_;
   std::unique_ptr<PrefetchManager> prefetch_manager_;
   std::map<GURL, base::TimeTicks> active_hints_;
@@ -184,7 +174,6 @@ class LoadingPredictor : public KeyedService,
   friend class LoadingPredictorTest;
   friend class LoadingPredictorPreconnectTest;
   friend class LoadingPredictorTabHelperTest;
-  friend class LoadingPredictorTabHelperTestCollectorTest;
   FRIEND_TEST_ALL_PREFIXES(LoadingPredictorTest,
                            TestMainFrameResponseCancelsHint);
   FRIEND_TEST_ALL_PREFIXES(LoadingPredictorTest,
