diff --git a/chrome/credential_provider/setup/setup.cc b/chrome/credential_provider/setup/setup.cc
index 9e769d1dadca..2bfb777c75fa 100644
--- a/chrome/credential_provider/setup/setup.cc
+++ b/chrome/credential_provider/setup/setup.cc
@@ -36,11 +36,8 @@
 #include "chrome/credential_provider/gaiacp/gcp_utils.h"
 #include "chrome/credential_provider/gaiacp/logging.h"
 #include "chrome/credential_provider/gaiacp/reg_utils.h"
-#include "chrome/credential_provider/setup/gcp_installer_crash_reporting.h"
 #include "chrome/credential_provider/setup/setup_lib.h"
 #include "chrome/credential_provider/setup/setup_utils.h"
-#include "components/crash/core/app/crash_switches.h"
-#include "components/crash/core/app/run_as_crashpad_handler_win.h"
 #include "content/public/common/content_switches.h"
 
 using credential_provider::GetGlobalFlagOrDefault;
@@ -75,11 +72,6 @@ int APIENTRY wWinMain(HINSTANCE hInstance,
   std::string process_type =
       cmdline->GetSwitchValueASCII(switches::kProcessType);
 
-  if (process_type == crash_reporter::switches::kCrashpadHandler) {
-    return crash_reporter::RunAsCrashpadHandler(*cmdline, base::FilePath(),
-                                                switches::kProcessType, "");
-  }
-
   // Initialize logging.
   logging::LoggingSettings settings;
   settings.logging_dest = logging::LOG_NONE;
@@ -128,8 +120,6 @@ int APIENTRY wWinMain(HINSTANCE hInstance,
     return -1;
   }
 
-  credential_provider::ConfigureGcpInstallerCrashReporting(*cmdline);
-
   // If the program is being run to either enable or disable stats, do that
   // and exit.
   if (cmdline->HasSwitch(credential_provider::switches::kEnableStats) ||
