diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index d0b9cdc19631c..d18c4a2f1844a 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -127,10 +127,8 @@ static_library("test_support") {
     "//chrome/renderer",
     "//chrome/utility",
     "//components/gwp_asan/buildflags",
-    "//components/heap_profiling/in_process",
     "//components/nacl/common:buildflags",
     "//components/power_scheduler",
-    "//components/startup_metric_utils/browser",
     "//content/public/app",
     "//content/public/common",
     "//pdf:pdf_ppapi",
