diff --git a/chrome/browser/prefetch/prefetch_proxy/prefetch_proxy_subresource_manager.cc b/chrome/browser/prefetch/prefetch_proxy/prefetch_proxy_subresource_manager.cc
index ee60dfe74c29..383093929664 100644
--- a/chrome/browser/prefetch/prefetch_proxy/prefetch_proxy_subresource_manager.cc
+++ b/chrome/browser/prefetch/prefetch_proxy/prefetch_proxy_subresource_manager.cc
@@ -167,55 +167,23 @@ void PrefetchProxySubresourceManager::RemoveProxiedURLLoaderFactory(
   proxied_loader_factories_.erase(it);
 }
 
-void PrefetchProxySubresourceManager::SetPrefetchMetricsCollector(
-    scoped_refptr<PrefetchProxyPrefetchMetricsCollector> collector) {
-  metrics_collector_ = collector;
-}
-
 void PrefetchProxySubresourceManager::OnResourceFetchComplete(
     const GURL& url,
     network::mojom::URLResponseHeadPtr head,
     const network::URLLoaderCompletionStatus& status) {
-  if (!metrics_collector_)
-    return;
-
-  metrics_collector_->OnSubresourcePrefetched(
-      /*mainframe_url=*/url_,
-      /*subresource_url=*/url, std::move(head), status);
 }
 
 void PrefetchProxySubresourceManager::OnResourceNotEligible(
     const GURL& url,
     PrefetchProxyPrefetchStatus status) {
-  if (!metrics_collector_)
-    return;
-  metrics_collector_->OnSubresourceNotEligible(
-      /*mainframe_url=*/url_,
-      /*subresource_url=*/url, status);
 }
 
 void PrefetchProxySubresourceManager::OnResourceThrottled(const GURL& url) {
-  if (!metrics_collector_)
-    return;
-  metrics_collector_->OnSubresourceNotEligible(
-      /*mainframe_url=*/url_,
-      /*subresource_url=*/url,
-      PrefetchProxyPrefetchStatus::kSubresourceThrottled);
 }
 
 void PrefetchProxySubresourceManager::OnProxyUnavailableForResource(
     const GURL& url) {
-  if (!metrics_collector_)
-    return;
-  metrics_collector_->OnSubresourceNotEligible(
-      /*mainframe_url=*/url_,
-      /*subresource_url=*/url,
-      PrefetchProxyPrefetchStatus::kPrefetchProxyNotAvailable);
 }
 
 void PrefetchProxySubresourceManager::OnResourceUsedFromCache(const GURL& url) {
-  if (!metrics_collector_)
-    return;
-  metrics_collector_->OnCachedSubresourceUsed(/*mainframe_url=*/url_,
-                                              /*subresource_url=*/url);
 }
