diff --git a/chrome/common/extensions/api/autofill_private.idl b/chrome/common/extensions/api/autofill_private.idl
index d7980c99df98..8b142dfbbe9f 100644
--- a/chrome/common/extensions/api/autofill_private.idl
+++ b/chrome/common/extensions/api/autofill_private.idl
@@ -241,9 +241,6 @@ namespace autofillPrivate {
     // Triggers local credit cards migration.
     static void migrateCreditCards();
 
-    // Logs that the server cards edit link was clicked.
-    static void logServerCardLinkClicked();
-
     // Enables or disables FIDO Authentication for credit card unmasking.
     static void setCreditCardFIDOAuthEnabledState(boolean enabled);
 
