diff --git a/chrome/browser/metrics/chromeos_metrics_provider.cc b/chrome/browser/metrics/chromeos_metrics_provider.cc
index 8b87f8806f8b..376dd3079612 100644
--- a/chrome/browser/metrics/chromeos_metrics_provider.cc
+++ b/chrome/browser/metrics/chromeos_metrics_provider.cc
@@ -111,10 +111,6 @@ void ChromeOSMetricsProvider::LogCrash(const std::string& crash_type) {
     IncrementPrefValue(prefs::kStabilitySystemUncleanShutdownCount);
   else
     NOTREACHED() << "Unexpected Chrome OS crash type " << crash_type;
-
-  // Wake up metrics logs sending if necessary now that new
-  // log data is available.
-  g_browser_process->metrics_service()->OnApplicationNotIdle();
 }
 
 EnrollmentStatus ChromeOSMetricsProvider::GetEnrollmentStatus() {
