diff --git a/chrome/browser/notifications/notification_platform_bridge_mac.mm b/chrome/browser/notifications/notification_platform_bridge_mac.mm
index 9e8fcb9f9f91..1e34d7f9841d 100644
--- a/chrome/browser/notifications/notification_platform_bridge_mac.mm
+++ b/chrome/browser/notifications/notification_platform_bridge_mac.mm
@@ -39,7 +39,6 @@
 #include "components/crash/core/app/crashpad.h"
 #include "content/public/browser/browser_task_traits.h"
 #include "third_party/blink/public/common/notifications/notification_constants.h"
-#include "third_party/crashpad/crashpad/client/crashpad_client.h"
 #include "ui/base/l10n/l10n_util_mac.h"
 #include "ui/message_center/public/cpp/notification.h"
 #include "ui/message_center/public/cpp/notification_types.h"
@@ -451,15 +450,6 @@ getDisplayedAlertsForProfileId:(NSString*)profileId
 - (id<NotificationDelivery>)serviceProxy {
   id<NotificationDelivery> proxy = [_xpcConnection remoteObjectProxy];
 
-  if (!_setExceptionPort) {
-    base::mac::ScopedMachSendRight exceptionPort(
-        crash_reporter::GetCrashpadClient().GetHandlerMachPort());
-    base::scoped_nsobject<CrXPCMachPort> xpcPort(
-        [[CrXPCMachPort alloc] initWithMachSendRight:std::move(exceptionPort)]);
-    [proxy setUseUNNotification:NO machExceptionPort:xpcPort];
-    _setExceptionPort = YES;
-  }
-
   return proxy;
 }
 
