diff --git a/chrome/renderer/previews/resource_loading_hints_agent.cc b/chrome/renderer/previews/resource_loading_hints_agent.cc
index 57e005a2399d..199d7bae85d0 100644
--- a/chrome/renderer/previews/resource_loading_hints_agent.cc
+++ b/chrome/renderer/previews/resource_loading_hints_agent.cc
@@ -84,15 +84,6 @@ bool ResourceLoadingHintsAgent::IsMainFrame() const {
   return render_frame()->IsMainFrame();
 }
 
-
-void ResourceLoadingHintsAgent::SetLiteVideoHint(
-    previews::mojom::LiteVideoHintPtr lite_video_hint) {
-  auto* lite_video_hint_agent =
-      lite_video::LiteVideoHintAgent::Get(render_frame());
-  if (lite_video_hint_agent)
-    lite_video_hint_agent->SetLiteVideoHint(std::move(lite_video_hint));
-}
-
 void ResourceLoadingHintsAgent::SetBlinkOptimizationGuideHints(
     blink::mojom::BlinkOptimizationGuideHintsPtr hints) {
   if (!IsMainFrame())
@@ -101,12 +92,6 @@ void ResourceLoadingHintsAgent::SetBlinkOptimizationGuideHints(
 }
 
 void ResourceLoadingHintsAgent::StopThrottlingMediaRequests() {
-  auto* lite_video_hint_agent =
-      lite_video::LiteVideoHintAgent::Get(render_frame());
-  if (lite_video_hint_agent) {
-    LOCAL_HISTOGRAM_BOOLEAN("LiteVideo.HintsAgent.StopThrottling", true);
-    lite_video_hint_agent->StopThrottlingAndClearHints();
-  }
 }
 
 }  // namespace previews
