diff --git a/chrome/renderer/previews/resource_loading_hints_agent.h b/chrome/renderer/previews/resource_loading_hints_agent.h
index 9d0cb0b903b9..8b6d9f481060 100644
--- a/chrome/renderer/previews/resource_loading_hints_agent.h
+++ b/chrome/renderer/previews/resource_loading_hints_agent.h
@@ -10,7 +10,6 @@
 #include "base/bind.h"
 #include "base/macros.h"
 #include "base/optional.h"
-#include "chrome/renderer/lite_video/lite_video_hint_agent.h"
 #include "content/public/renderer/render_frame_observer.h"
 #include "mojo/public/cpp/bindings/associated_receiver.h"
 #include "mojo/public/cpp/bindings/pending_associated_receiver.h"
@@ -47,8 +46,6 @@ class ResourceLoadingHintsAgent
                                    resource_loading_hints) override;
   void SetCompressPublicImagesHints(
       blink::mojom::CompressPublicImagesHintsPtr images_hints) override;
-  void SetLiteVideoHint(
-      blink::mojom::LiteVideoHintPtr lite_video_hint) override;
   void SetBlinkOptimizationGuideHints(
       blink::mojom::BlinkOptimizationGuideHintsPtr hints) override;
   void StopThrottlingMediaRequests() override;
@@ -60,7 +57,6 @@ class ResourceLoadingHintsAgent
   bool IsMainFrame() const;
 
   std::vector<std::string> subresource_patterns_to_block_;
-  base::Optional<int64_t> ukm_source_id_;
 
   mojo::AssociatedReceiver<blink::mojom::PreviewsResourceLoadingHintsReceiver>
       receiver_{this};
