diff --git a/chrome/browser/ui/views/profiles/profile_picker_view.cc b/chrome/browser/ui/views/profiles/profile_picker_view.cc
index 015ac6d2e468c..6f773af1c863a 100644
--- a/chrome/browser/ui/views/profiles/profile_picker_view.cc
+++ b/chrome/browser/ui/views/profiles/profile_picker_view.cc
@@ -497,9 +497,6 @@ void ProfilePickerView::Display(ProfilePicker::EntryPoint entry_point) {
     DCHECK(creation_time_on_startup_.is_null());
     // Display() is called right after the creation of this object.
     creation_time_on_startup_ = base::TimeTicks::Now();
-    base::UmaHistogramTimes("ProfilePicker.StartupTime.BeforeCreation",
-                            creation_time_on_startup_ -
-                                startup_metric_utils::MainEntryPointTicks());
   }
 
   if (state_ == kNotStarted) {
