diff --git a/chrome/browser/win/conflicts/module_database.h b/chrome/browser/win/conflicts/module_database.h
index 0cc076dc1e24c..920e1dd81be45 100644
--- a/chrome/browser/win/conflicts/module_database.h
+++ b/chrome/browser/win/conflicts/module_database.h
@@ -16,7 +16,6 @@
 #include "build/branding_buildflags.h"
 #include "chrome/browser/win/conflicts/module_info.h"
 #include "chrome/browser/win/conflicts/module_inspector.h"
-#include "chrome/browser/win/conflicts/third_party_metrics_recorder.h"
 #include "content/public/common/process_type.h"
 
 class ModuleDatabaseObserver;
@@ -275,9 +274,6 @@ class ModuleDatabase : public ModuleDatabaseEventSource {
   std::unique_ptr<ThirdPartyConflictsManager> third_party_conflicts_manager_;
 #endif
 
-  // Records metrics on third-party modules.
-  ThirdPartyMetricsRecorder third_party_metrics_;
-
   SEQUENCE_CHECKER(sequence_checker_);
 
   DISALLOW_COPY_AND_ASSIGN(ModuleDatabase);
