diff --git a/chrome/browser/prerender/isolated/isolated_prerender_subresource_manager.h b/chrome/browser/prerender/isolated/isolated_prerender_subresource_manager.h
index a50eb9999e9f..d784d3bf80e3 100644
--- a/chrome/browser/prerender/isolated/isolated_prerender_subresource_manager.h
+++ b/chrome/browser/prerender/isolated/isolated_prerender_subresource_manager.h
@@ -33,7 +33,6 @@ class PrerenderHandle;
 }
 
 class PrefetchedMainframeResponseContainer;
-class IsolatedPrerenderPrefetchMetricsCollector;
 class IsolatedPrerenderProxyingURLLoaderFactory;
 
 // This class manages the isolated prerender of a page and its subresources.
@@ -65,10 +64,6 @@ class IsolatedPrerenderSubresourceManager
     return successfully_loaded_subresources_;
   }
 
-  // Sets the prefetch metrics collector to report subresource fetches to.
-  void SetPrefetchMetricsCollector(
-      scoped_refptr<IsolatedPrerenderPrefetchMetricsCollector> collector);
-
   // Takes ownership of |mainframe_response_|.
   std::unique_ptr<PrefetchedMainframeResponseContainer> TakeMainframeResponse();
 
@@ -157,10 +152,6 @@ class IsolatedPrerenderSubresourceManager
   // The mainframe response headers and body.
   std::unique_ptr<PrefetchedMainframeResponseContainer> mainframe_response_;
 
-  // Collects metrics from the implementation of
-  // |IsolatedPrerenderProxyingURLLoaderFactory::ResourceMetricsObserver|.
-  scoped_refptr<IsolatedPrerenderPrefetchMetricsCollector> metrics_collector_;
-
   // State for managing the NoStatePrerender when it is running. If
   // |nsp_handle_| is set, then |on_nsp_done_callback_| is also set and vise
   // versa.
