diff --git a/chrome/browser/resources/new_tab_page/app.js b/chrome/browser/resources/new_tab_page/app.js
index 2863e7c0e999..430734d90d53 100644
--- a/chrome/browser/resources/new_tab_page/app.js
+++ b/chrome/browser/resources/new_tab_page/app.js
@@ -869,11 +869,6 @@ class AppElement extends PolymerElement {
     }
   }
 
-  /** @private */
-  onMiddleSlotPromoLoaded_() {
-    this.middleSlotPromoLoaded_ = true;
-  }
-
   /** @private */
   onModulesLoaded_() {
     this.modulesLoaded_ = true;
