diff --git a/chrome/common/chrome_features.cc b/chrome/common/chrome_features.cc
index 510f8ace25a9..e946b31a33a4 100644
--- a/chrome/common/chrome_features.cc
+++ b/chrome/common/chrome_features.cc
@@ -759,7 +759,7 @@ const base::Feature kTPMFirmwareUpdate{"TPMFirmwareUpdate",
 #if !defined(OS_ANDROID)
 // Enables logging UKMs for background tab activity by TabActivityWatcher.
 const base::Feature kTabMetricsLogging{"TabMetricsLogging",
-                                       base::FEATURE_ENABLED_BY_DEFAULT};
+                                       base::FEATURE_DISABLED_BY_DEFAULT};
 #endif
 
 // Enables the teamfood flags.
