diff --git a/chrome/common/chrome_switches.cc b/chrome/common/chrome_switches.cc
index 3c53afb3eeb0f..4a94f14b45bd9 100644
--- a/chrome/common/chrome_switches.cc
+++ b/chrome/common/chrome_switches.cc
@@ -297,6 +297,9 @@ const char kEnablePotentiallyAnnoyingSecurityFeatures[] =
 // list of port numbers.
 const char kExplicitlyAllowedPorts[] = "explicitly-allowed-ports";
 
+// Enables user control over muting tab audio from the tab strip.
+const char kEnableTabAudioMuting[]  = "enable-tab-audio-muting";
+
 // Name of the command line flag to force content verification to be on in one
 // of various modes.
 const char kExtensionContentVerification[] = "extension-content-verification";
