diff --git a/chrome/browser/chrome_browser_field_trials.cc b/chrome/browser/chrome_browser_field_trials.cc
index 63a2aeadf005..d1c5a77240e9 100644
--- a/chrome/browser/chrome_browser_field_trials.cc
+++ b/chrome/browser/chrome_browser_field_trials.cc
@@ -21,7 +21,6 @@
 #include "chrome/common/chrome_paths.h"
 #include "chrome/common/chrome_switches.h"
 #include "components/metrics/metrics_pref_names.h"
-#include "components/metrics/persistent_histograms.h"
 #include "components/ukm/ukm_recorder_impl.h"
 #include "components/version_info/version_info.h"
 
@@ -114,9 +113,4 @@ void ChromeBrowserFieldTrials::RegisterSyntheticTrials() {
 }
 
 void ChromeBrowserFieldTrials::InstantiateDynamicTrials() {
-  // Persistent histograms must be enabled as soon as possible.
-  base::FilePath metrics_dir;
-  if (base::PathService::Get(chrome::DIR_USER_DATA, &metrics_dir)) {
-    InstantiatePersistentHistograms(metrics_dir);
-  }
 }
