diff --git a/chrome/browser/ui/sad_tab.cc b/chrome/browser/ui/sad_tab.cc
index a5c6d7008f897..22f315f6393ea 100644
--- a/chrome/browser/ui/sad_tab.cc
+++ b/chrome/browser/ui/sad_tab.cc
@@ -55,8 +55,6 @@ void RecordEvent(bool feedback, ui_metrics::SadTabEvent event) {
   }
 }
 
-constexpr char kCategoryTagCrash[] = "Crash";
-
 // Return true if this function has been called in the last 10 seconds.
 bool IsRepeatedlyCrashing() {
   const int kMaxSecondsSinceLastCrash = 10;
@@ -238,14 +236,6 @@ void SadTab::PerformAction(SadTab::Action action) {
       RecordEvent(show_feedback_button_,
                   ui_metrics::SadTabEvent::BUTTON_CLICKED);
       if (show_feedback_button_) {
-        ShowFeedbackPage(
-            chrome::FindBrowserWithWebContents(web_contents_),
-            chrome::kFeedbackSourceSadTabPage,
-            std::string() /* description_template */,
-            l10n_util::GetStringUTF8(kind_ == SAD_TAB_KIND_CRASHED
-                                         ? IDS_CRASHED_TAB_FEEDBACK_MESSAGE
-                                         : IDS_KILLED_TAB_FEEDBACK_MESSAGE),
-            std::string(kCategoryTagCrash), std::string());
       } else {
         web_contents_->GetController().Reload(content::ReloadType::NORMAL,
                                               true);
