diff --git a/chrome/browser/resources/local_ntp/local_ntp.js b/chrome/browser/resources/local_ntp/local_ntp.js
index d9d58e1ddef9..013103cb9a15 100644
--- a/chrome/browser/resources/local_ntp/local_ntp.js
+++ b/chrome/browser/resources/local_ntp/local_ntp.js
@@ -1057,67 +1057,6 @@ function injectOneGoogleBar(ogb) {
   ntpApiHandle.logEvent(LOG_TYPE.NTP_ONE_GOOGLE_BAR_SHOWN);
 }
 
-/**
- * Injects a middle-slot promo into the page. Called asynchronously, so that it
- * doesn't block the main page load.
- */
-function injectPromo(promo) {
-  if (!promo.promoHtml) {
-    if ($(IDS.PROMO)) {
-      $(IDS.PROMO).remove();
-    }
-    return;
-  }
-
-  const promoContainer = document.createElement('div');
-  promoContainer.id = IDS.PROMO;
-  promoContainer.innerHTML += promo.promoHtml;
-  $(IDS.NTP_CONTENTS).appendChild(promoContainer);
-
-  if (promo.promoLogUrl) {
-    navigator.sendBeacon(promo.promoLogUrl);
-  }
-
-  ntpApiHandle.logEvent(LOG_TYPE.NTP_MIDDLE_SLOT_PROMO_SHOWN);
-
-  const link = promoContainer.querySelector('a');
-  if (link) {
-    link.onclick = e => {
-      const url = new URL(link.href);
-      if (promo.canOpenExtensionsPage && url.origin == 'chrome://extensions') {
-        ntpApiHandle.openExtensionsPage(
-            e.button, e.altKey, e.ctrlKey, e.metaKey, e.shiftKey);
-        e.preventDefault();
-      }
-      ntpApiHandle.logEvent(LOG_TYPE.NTP_MIDDLE_SLOT_PROMO_LINK_CLICKED);
-    };
-  }
-
-  if (promo.promoId) {
-    const icon = document.createElement('button');
-    icon.classList.add(CLASSES.DISMISS_ICON);
-
-    icon.title = configData.translatedStrings.dismissPromo;
-    icon.onclick = e => {
-      ntpApiHandle.blocklistPromo(promo.promoId);
-      promoContainer.remove();
-      window.removeEventListener('resize', showPromoIfNotOverlapping);
-    };
-
-    const dismiss = document.createElement('div');
-    dismiss.classList.add(CLASSES.DISMISS_PROMO);
-    dismiss.appendChild(icon);
-
-    promoContainer.querySelector('div').appendChild(dismiss);
-    promoContainer.classList.add(CLASSES.DISMISSABLE);
-  }
-
-  // The the MV tiles are already loaded show the promo immediately.
-  if (tilesAreLoaded) {
-    showPromoIfNotOverlappingAndTrackResizes();
-  }
-}
-
 /**
  * Injects search suggestions into the page. Called *synchronously* with cached
  * data as not to cause shifting of the most visited tiles.
@@ -2097,15 +2036,6 @@ function requestAndInsertGoogleResources() {
       injectOneGoogleBar(og);
     };
   }
-  if (!$('promo-loader')) {
-    const promoScript = document.createElement('script');
-    promoScript.id = 'promo-loader';
-    promoScript.src = 'chrome-search://local-ntp/promo.js';
-    document.body.appendChild(promoScript);
-    promoScript.onload = function() {
-      injectPromo(promo);
-    };
-  }
 }
 
 /**
