diff --git a/chrome/browser/ui/webui/tab_search/tab_search_page_handler.h b/chrome/browser/ui/webui/tab_search/tab_search_page_handler.h
index 55475d31a093..bb239570602d 100644
--- a/chrome/browser/ui/webui/tab_search/tab_search_page_handler.h
+++ b/chrome/browser/ui/webui/tab_search/tab_search_page_handler.h
@@ -46,7 +46,6 @@ class TabSearchPageHandler : public tab_search::mojom::PageHandler,
   void CloseTab(int32_t tab_id) override;
   void GetProfileTabs(GetProfileTabsCallback callback) override;
   void GetTabGroups(GetTabGroupsCallback callback) override;
-  void ShowFeedbackPage() override;
   void SwitchToTab(
       tab_search::mojom::SwitchToTabInfoPtr switch_to_tab_info) override;
   void ShowUI() override;
