diff --git a/chrome/browser/optimization_guide/optimization_guide_navigation_data.h b/chrome/browser/optimization_guide/optimization_guide_navigation_data.h
index 6d63ad9e0cff5..fecb10bf985e5 100644
--- a/chrome/browser/optimization_guide/optimization_guide_navigation_data.h
+++ b/chrome/browser/optimization_guide/optimization_guide_navigation_data.h
@@ -91,13 +91,6 @@ class OptimizationGuideNavigationData {
   }
 
  private:
-  // Records metrics based on data currently held in |this|.
-  void RecordMetrics() const;
-
-  // Records the OptimizationGuide UKM event based on data currently held in
-  // |this|.
-  void RecordOptimizationGuideUKM() const;
-
   // The navigation ID of the navigation handle that this data is associated
   // with.
   const int64_t navigation_id_;
