diff --git a/chrome/installer/setup/install.cc b/chrome/installer/setup/install.cc
index 57861c0a9715..3b200e22005c 100644
--- a/chrome/installer/setup/install.cc
+++ b/chrome/installer/setup/install.cc
@@ -30,7 +30,6 @@
 #include "chrome/install_static/install_util.h"
 #include "chrome/installer/setup/install_params.h"
 #include "chrome/installer/setup/install_worker.h"
-#include "chrome/installer/setup/installer_crash_reporting.h"
 #include "chrome/installer/setup/installer_state.h"
 #include "chrome/installer/setup/setup_constants.h"
 #include "chrome/installer/setup/setup_util.h"
@@ -174,8 +173,6 @@ InstallStatus InstallNewVersion(const InstallParams& install_params,
 
   installer_state.SetStage(BUILDING);
 
-  SetCurrentVersionCrashKey(current_version);
-
   std::unique_ptr<WorkItemList> install_list(WorkItem::CreateWorkItemList());
 
   AddInstallWorkItems(install_params, install_list.get());
