diff --git a/chrome/browser/offline_pages/offline_page_tab_helper.cc b/chrome/browser/offline_pages/offline_page_tab_helper.cc
index 03564986f756..3d1765985fd5 100644
--- a/chrome/browser/offline_pages/offline_page_tab_helper.cc
+++ b/chrome/browser/offline_pages/offline_page_tab_helper.cc
@@ -28,7 +28,6 @@
 #include "components/offline_pages/core/offline_page_model.h"
 #include "components/offline_pages/core/offline_store_utils.h"
 #include "components/offline_pages/core/page_criteria.h"
-#include "components/offline_pages/core/prefetch/offline_metrics_collector.h"
 #include "components/offline_pages/core/prefetch/prefetch_service.h"
 #include "components/offline_pages/core/request_header/offline_page_header.h"
 #include "content/public/browser/back_forward_cache.h"
@@ -165,12 +164,6 @@ void OfflinePageTabHelper::DidStartNavigation(
 
   // The provisional offline info can be cleared no matter how.
   provisional_offline_info_.Clear();
-
-  // Report any attempted navigation as indication that browser is in use.
-  // This doesn't have to be a successful navigation.
-  if (prefetch_service_) {
-    prefetch_service_->GetOfflineMetricsCollector()->OnAppStartupOrResume();
-  }
 }
 
 void OfflinePageTabHelper::DidFinishNavigation(
@@ -261,26 +254,6 @@ void OfflinePageTabHelper::ReportPrefetchMetrics(
 
   if (!prefetch_service_)
     return;
-
-  // Report the kind of navigation (online/offline) to metrics collector.
-  // It accumulates this info to mark a day as 'offline' or 'online'.
-  OfflineMetricsCollector* metrics_collector =
-      prefetch_service_->GetOfflineMetricsCollector();
-  DCHECK(metrics_collector);
-
-  if (offline_page()) {
-    // Report prefetch usage.
-    if (GetPolicy(offline_page()->client_id.name_space).is_suggested)
-      metrics_collector->OnPrefetchedPageOpened();
-    // Note that navigation to offline page may happen even if network is
-    // connected. For the purposes of collecting offline usage statistics,
-    // we still count this as offline navigation.
-    metrics_collector->OnSuccessfulNavigationOffline();
-  } else {
-    metrics_collector->OnSuccessfulNavigationOnline();
-    // The device is apparently online, attempt to report stats to UMA.
-    metrics_collector->ReportAccumulatedStats();
-  }
 }
 
 void OfflinePageTabHelper::TryLoadingOfflinePageOnNetError(
