diff --git a/chrome/browser/prefs/profile_pref_store_manager.h b/chrome/browser/prefs/profile_pref_store_manager.h
index 9f9cd93d325e9..46015aee87306 100644
--- a/chrome/browser/prefs/profile_pref_store_manager.h
+++ b/chrome/browser/prefs/profile_pref_store_manager.h
@@ -16,7 +16,6 @@
 #include "base/sequenced_task_runner.h"
 #include "mojo/public/cpp/bindings/pending_remote.h"
 #include "services/preferences/public/mojom/preferences.mojom-forward.h"
-#include "services/preferences/public/mojom/tracked_preference_validation_delegate.mojom-forward.h"
 
 class PersistentPrefStore;
 class PrefService;
@@ -88,9 +87,7 @@ class ProfilePrefStoreManager {
       size_t reporting_ids_count,
       scoped_refptr<base::SequencedTaskRunner> io_task_runner,
       mojo::PendingRemote<prefs::mojom::ResetOnLoadObserver>
-          reset_on_load_observer,
-      mojo::PendingRemote<prefs::mojom::TrackedPreferenceValidationDelegate>
-          validation_delegate);
+          reset_on_load_observer);
 
   // Initializes the preferences for the managed profile with the preference
   // values in |master_prefs|. Acts synchronously, including blocking IO.
@@ -109,8 +106,6 @@ class ProfilePrefStoreManager {
       size_t reporting_ids_count,
       mojo::PendingRemote<prefs::mojom::ResetOnLoadObserver>
           reset_on_load_observer,
-      mojo::PendingRemote<prefs::mojom::TrackedPreferenceValidationDelegate>
-          validation_delegate,
       service_manager::Connector* connector);
 
   prefs::mojom::TrackedPersistentPrefStoreConfigurationPtr
@@ -119,9 +114,7 @@ class ProfilePrefStoreManager {
           tracking_configuration,
       size_t reporting_ids_count,
       mojo::PendingRemote<prefs::mojom::ResetOnLoadObserver>
-          reset_on_load_observer,
-      mojo::PendingRemote<prefs::mojom::TrackedPreferenceValidationDelegate>
-          validation_delegate);
+          reset_on_load_observer);
 
   const base::FilePath profile_path_;
   const std::string seed_;
