diff --git a/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.cc b/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.cc
index 312b2f9a65e7..433d5d343864 100644
--- a/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.cc
+++ b/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.cc
@@ -13,7 +13,6 @@
 #include "build/build_config.h"
 #include "build/chromeos_buildflags.h"
 #include "chrome/app/vector_icons/vector_icons.h"
-#include "chrome/browser/metrics/tab_count_metrics.h"
 #include "chrome/browser/themes/theme_properties.h"
 #include "chrome/browser/ui/tabs/tab_renderer_data.h"
 #include "chrome/browser/ui/tabs/tab_style.h"
@@ -24,7 +23,6 @@
 #include "chrome/browser/ui/views/tabs/tab.h"
 #include "chrome/browser/ui/views/tabs/tab_controller.h"
 #include "chrome/grit/generated_resources.h"
-#include "components/tab_count_metrics/tab_count_metrics.h"
 #include "components/url_formatter/url_formatter.h"
 #include "ui/accessibility/ax_enums.mojom.h"
 #include "ui/base/l10n/l10n_util.h"
@@ -672,26 +670,6 @@ views::View* TabHoverCardBubbleView::GetDesiredAnchorView() {
 }
 
 void TabHoverCardBubbleView::RecordHoverCardsSeenRatioMetric() {
-  const char kHistogramPrefixHoverCardsSeenBeforeSelection[] =
-      "TabHoverCards.TabHoverCardsSeenBeforeTabSelection";
-  const size_t tab_count = tab_count_metrics::TabCount();
-  const size_t bucket = tab_count_metrics::BucketForTabCount(tab_count);
-  constexpr int kMinHoverCardsSeen = 0;
-  constexpr int kMaxHoverCardsSeen = 100;
-  constexpr int kHistogramBucketCount = 50;
-  STATIC_HISTOGRAM_POINTER_GROUP(
-      tab_count_metrics::HistogramName(
-          kHistogramPrefixHoverCardsSeenBeforeSelection,
-          /* live_tabs_only */ false, bucket),
-      static_cast<int>(bucket),
-      static_cast<int>(tab_count_metrics::kNumTabCountBuckets),
-      Add(hover_cards_seen_count_),
-      base::Histogram::FactoryGet(
-          tab_count_metrics::HistogramName(
-              kHistogramPrefixHoverCardsSeenBeforeSelection,
-              /* live_tabs_only */ false, bucket),
-          kMinHoverCardsSeen, kMaxHoverCardsSeen, kHistogramBucketCount,
-          base::HistogramBase::kUmaTargetedHistogramFlag));
 }
 
 void TabHoverCardBubbleView::OnWidgetVisibilityChanged(views::Widget* widget,
