diff --git a/chrome/renderer/extensions/chrome_extensions_renderer_client.h b/chrome/renderer/extensions/chrome_extensions_renderer_client.h
index 9e682b2d386c..6c92bc16ad20 100644
--- a/chrome/renderer/extensions/chrome_extensions_renderer_client.h
+++ b/chrome/renderer/extensions/chrome_extensions_renderer_client.h
@@ -47,10 +47,6 @@ namespace url {
 class Origin;
 }
 
-namespace ukm {
-class MojoUkmRecorder;
-}
-
 namespace v8 {
 class Isolate;
 template <typename T>
@@ -126,7 +122,6 @@ class ChromeExtensionsRendererClient
   }
 
  private:
-  std::unique_ptr<ukm::MojoUkmRecorder> ukm_recorder_;
   std::unique_ptr<extensions::Dispatcher> extension_dispatcher_;
   std::unique_ptr<extensions::RendererPermissionsPolicyDelegate>
       permissions_policy_delegate_;
