diff --git a/chrome/browser/resource_coordinator/tab_helper.cc b/chrome/browser/resource_coordinator/tab_helper.cc
index ccf38154a1b1..70257cbb6897 100644
--- a/chrome/browser/resource_coordinator/tab_helper.cc
+++ b/chrome/browser/resource_coordinator/tab_helper.cc
@@ -18,7 +18,6 @@
 #include "chrome/browser/resource_coordinator/tab_lifecycle_unit_external.h"
 #include "chrome/browser/resource_coordinator/tab_lifecycle_unit_source.h"
 #include "chrome/browser/resource_coordinator/tab_load_tracker.h"
-#include "chrome/browser/resource_coordinator/tab_memory_metrics_reporter.h"
 #include "chrome/browser/resource_coordinator/utils.h"
 #include "content/public/browser/navigation_handle.h"
 #include "content/public/browser/render_frame_host.h"
@@ -35,12 +34,6 @@ ResourceCoordinatorTabHelper::ResourceCoordinatorTabHelper(
     content::WebContents* web_contents)
     : content::WebContentsObserver(web_contents) {
   TabLoadTracker::Get()->StartTracking(web_contents);
-  if (memory_instrumentation::MemoryInstrumentation::GetInstance()) {
-    auto* rc_parts = g_browser_process->resource_coordinator_parts();
-    DCHECK(rc_parts);
-    rc_parts->tab_memory_metrics_reporter()->StartReporting(
-        TabLoadTracker::Get());
-  }
 }
 
 ResourceCoordinatorTabHelper::~ResourceCoordinatorTabHelper() = default;
@@ -79,11 +72,6 @@ void ResourceCoordinatorTabHelper::DidFinishNavigation(
       navigation_handle->IsSameDocument()) {
     return;
   }
-
-  if (navigation_handle->IsInMainFrame()) {
-    ukm_source_id_ = ukm::ConvertToSourceId(
-        navigation_handle->GetNavigationId(), ukm::SourceIdType::NAVIGATION_ID);
-  }
 }
 
 WEB_CONTENTS_USER_DATA_KEY_IMPL(ResourceCoordinatorTabHelper)
