diff --git a/chrome/browser/ui/search/search_tab_helper.h b/chrome/browser/ui/search/search_tab_helper.h
index 0851dec679f8..afec8d009187 100644
--- a/chrome/browser/ui/search/search_tab_helper.h
+++ b/chrome/browser/ui/search/search_tab_helper.h
@@ -148,7 +148,6 @@ class SearchTabHelper : public content::WebContentsObserver,
   void StopAutocomplete(bool clear_result) override;
   void ToggleSuggestionGroupIdVisibility(int32_t suggestion_group_id) override;
   void LogCharTypedToRepaintLatency(uint32_t latency_ms) override;
-  void BlocklistPromo(const std::string& promo_id) override;
   void OpenExtensionsPage(double button,
                           bool alt_key,
                           bool ctrl_key,
