diff --git a/chromecast/browser/cast_content_browser_client.cc b/chromecast/browser/cast_content_browser_client.cc
index 0861e7d84de3..ed97af011a51 100644
--- a/chromecast/browser/cast_content_browser_client.cc
+++ b/chromecast/browser/cast_content_browser_client.cc
@@ -894,7 +894,7 @@ void CastContentBrowserClient::RegisterNonNetworkNavigationURLLoaderFactories(
   auto* browser_context = web_contents->GetBrowserContext();
   auto extension_factory =
       extensions::CreateExtensionNavigationURLLoaderFactory(
-          browser_context, ukm_source_id,
+          browser_context,
           !!extensions::WebViewGuest::FromWebContents(web_contents));
   factories->emplace(extensions::kExtensionScheme,
                      CastExtensionURLLoaderFactory::Create(
