diff --git a/chrome/browser/profiles/profile_activity_metrics_recorder.cc b/chrome/browser/profiles/profile_activity_metrics_recorder.cc
index 00cbd4abf2835..ae5ad70576a69 100644
--- a/chrome/browser/profiles/profile_activity_metrics_recorder.cc
+++ b/chrome/browser/profiles/profile_activity_metrics_recorder.cc
@@ -211,7 +211,6 @@ void ProfileActivityMetricsRecorder::OnProfileWillBeDestroyed(
 
 ProfileActivityMetricsRecorder::ProfileActivityMetricsRecorder() {
   BrowserList::AddObserver(this);
-  metrics::DesktopSessionDurationTracker::Get()->AddObserver(this);
   action_callback_ = base::BindRepeating(
       &ProfileActivityMetricsRecorder::OnUserAction, base::Unretained(this));
   base::AddActionCallback(action_callback_);
@@ -219,7 +218,6 @@ ProfileActivityMetricsRecorder::ProfileActivityMetricsRecorder() {
 
 ProfileActivityMetricsRecorder::~ProfileActivityMetricsRecorder() {
   BrowserList::RemoveObserver(this);
-  metrics::DesktopSessionDurationTracker::Get()->RemoveObserver(this);
   base::RemoveActionCallback(action_callback_);
 }
 
