diff --git a/chrome/chrome_cleaner/logging/dummy_api_keys.cc b/chrome/chrome_cleaner/logging/dummy_api_keys.cc
index c0ac308b7722..87b3fbc46095 100644
--- a/chrome/chrome_cleaner/logging/dummy_api_keys.cc
+++ b/chrome/chrome_cleaner/logging/dummy_api_keys.cc
@@ -7,11 +7,11 @@
 namespace chrome_cleaner {
 
 const char* kSafeBrowsingCleanerUrl =
-    "https://sb-ssl.google.com/safebrowsing/clientreport/chrome-cct"
+    "https://no-thanks.invalid"
     "?key=DUMMY_KEY";
 
 const char* kSafeBrowsingReporterUrl =
-    "https://sb-ssl.google.com/safebrowsing/clientreport/chrome-sw-reporter"
+    "https://no-thanks.invalid"
     "?key=DUMMY_KEY";
 
 }  // namespace chrome_cleaner
