diff --git a/chrome/browser/devtools/BUILD.gn b/chrome/browser/devtools/BUILD.gn
index 7a6188b2df49..6a301ab51b1a 100644
--- a/chrome/browser/devtools/BUILD.gn
+++ b/chrome/browser/devtools/BUILD.gn
@@ -204,7 +204,6 @@ static_library("devtools") {
   if (!is_android) {
     deps += [
       ":protocol_generated_sources",
-      "//components/subresource_filter/content/browser:browser",
       "//third_party/inspector_protocol:crdtp",
     ]
     sources += [
