diff --git a/chrome/BUILD.gn b/chrome/BUILD.gn
index 14a04f524e5d..c95e30458cb2 100644
--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -143,8 +143,6 @@ if (!is_android && !is_mac) {
         "app/chrome_exe_main_win.cc",
         "app/main_dll_loader_win.cc",
         "app/main_dll_loader_win.h",
-        "common/crash_keys.cc",
-        "common/crash_keys.h",
       ]
 
       deps += [
@@ -163,8 +161,6 @@ if (!is_android && !is_mac) {
         "//chrome/installer/util:constants",
         "//chrome/installer/util:did_run_support",
         "//components/browser_watcher:browser_watcher_client",
-        "//components/crash/core/app:run_as_crashpad_handler",
-        "//components/crash/core/common",
         "//components/flags_ui:switches",
         "//content:sandbox_helper_win",
         "//content/public/common:static_switches",
@@ -173,7 +169,6 @@ if (!is_android && !is_mac) {
         "//sandbox/policy",
         "//third_party/breakpad:breakpad_handler",
         "//third_party/breakpad:breakpad_sender",
-        "//third_party/crashpad/crashpad/util",
       ]
       data_deps = [
         "//chrome/app/version_assembly:version_assembly_manifest",
@@ -324,7 +319,6 @@ if (is_win) {
       "//chrome/install_static:install_static_util",
       "//chrome/install_static:secondary_module",
       "//components/browser_watcher:stability_client",
-      "//components/crash/core/app",
       "//components/policy:generated",
       "//content/app/resources",
       "//content/public/common:service_names",
@@ -698,14 +692,12 @@ if (is_win) {
   bundle_data("chrome_framework_helpers") {
     sources = [
       "$root_out_dir/app_mode_loader",
-      "$root_out_dir/chrome_crashpad_handler",
     ]
 
     outputs = [ "{{bundle_contents_dir}}/Helpers/{{source_file_part}}" ]
 
     public_deps = [
       "//chrome/app_shim:app_mode_loader",
-      "//components/crash/core/app:chrome_crashpad_handler",
     ]
 
     foreach(helper_params, content_mac_helpers) {
@@ -937,7 +929,6 @@ if (is_win) {
       "//chrome/app:command_ids",
       "//chrome/common:buildflags",
       "//chrome/common/profiler",
-      "//components/crash/core/app",
       "//components/policy:generated",
       "//content/public/app",
       "//content/public/common:service_names",
