diff --git a/chrome/browser/resources/new_tab_page/BUILD.gn b/chrome/browser/resources/new_tab_page/BUILD.gn
index 7588d9a7ab6d..1c9b55125742 100644
--- a/chrome/browser/resources/new_tab_page/BUILD.gn
+++ b/chrome/browser/resources/new_tab_page/BUILD.gn
@@ -54,7 +54,6 @@ js_library("new_tab_page") {
 js_library("lazy_load") {
   deps = [
     ":customize_dialog",
-    ":middle_slot_promo",
     ":most_visited",
     ":voice_search_overlay",
     "modules/dummy:module",
