diff --git a/chrome/browser/background_sync/background_sync_delegate_impl.cc b/chrome/browser/background_sync/background_sync_delegate_impl.cc
index 3e9cbbd144984..61ef015c571a7 100644
--- a/chrome/browser/background_sync/background_sync_delegate_impl.cc
+++ b/chrome/browser/background_sync/background_sync_delegate_impl.cc
@@ -5,7 +5,6 @@
 #include "chrome/browser/background_sync/background_sync_delegate_impl.h"
 
 #include "chrome/browser/content_settings/host_content_settings_map_factory.h"
-#include "chrome/browser/metrics/ukm_background_recorder_service.h"
 #include "chrome/browser/profiles/profile_keep_alive_types.h"
 #include "components/content_settings/core/browser/host_content_settings_map.h"
 #include "components/keep_alive_registry/keep_alive_registry.h"
@@ -24,13 +23,9 @@ BackgroundSyncDelegateImpl::BackgroundSyncDelegateImpl(Profile* profile)
     : SiteEngagementObserver(
           site_engagement::SiteEngagementService::Get(profile)),
       profile_(profile),
-      ukm_background_service_(
-          ukm::UkmBackgroundRecorderFactory::GetForProfile(profile)),
-
       site_engagement_service_(
           site_engagement::SiteEngagementService::Get(profile)) {
   DCHECK(profile_);
-  DCHECK(ukm_background_service_);
   DCHECK(site_engagement_service_);
   off_the_record_ = profile_->IsOffTheRecord();
 }
@@ -62,13 +57,6 @@ BackgroundSyncDelegateImpl::CreateBackgroundSyncEventKeepAlive() {
 }
 #endif  // !defined(OS_ANDROID)
 
-void BackgroundSyncDelegateImpl::GetUkmSourceId(
-    const url::Origin& origin,
-    base::OnceCallback<void(base::Optional<ukm::SourceId>)> callback) {
-  ukm_background_service_->GetBackgroundSourceIdIfAllowed(origin,
-                                                          std::move(callback));
-}
-
 void BackgroundSyncDelegateImpl::Shutdown() {
   // Clear the profile as we're not supposed to use it anymore.
   profile_ = nullptr;
