diff --git a/chrome/browser/renderer_context_menu/copy_link_to_text_menu_observer.cc b/chrome/browser/renderer_context_menu/copy_link_to_text_menu_observer.cc
index 05c0bb69094e..f6a88e47c128 100644
--- a/chrome/browser/renderer_context_menu/copy_link_to_text_menu_observer.cc
+++ b/chrome/browser/renderer_context_menu/copy_link_to_text_menu_observer.cc
@@ -11,7 +11,6 @@
 #include "chrome/grit/generated_resources.h"
 #include "components/renderer_context_menu/render_view_context_menu_proxy.h"
 #include "components/shared_highlighting/core/common/disabled_sites.h"
-#include "components/shared_highlighting/core/common/shared_highlighting_metrics.h"
 #include "content/public/browser/context_menu_params.h"
 #include "content/public/browser/render_view_host.h"
 #include "content/public/browser/web_contents.h"
@@ -92,7 +91,6 @@ void CopyLinkToTextMenuObserver::ExecuteCommand(int command_id) {
   // check should happen before iframe check so that if both conditions are
   // present then blocklist error is logged.
   if (!shared_highlighting::ShouldOfferLinkToText(url_)) {
-    shared_highlighting::LogGenerateErrorBlockList();
     OnGeneratedSelector(std::make_unique<ui::DataTransferEndpoint>(
                             main_frame->GetLastCommittedOrigin()),
                         std::string());
@@ -100,7 +98,6 @@ void CopyLinkToTextMenuObserver::ExecuteCommand(int command_id) {
   }
 
   if (main_frame != proxy_->GetWebContents()->GetFocusedFrame()) {
-    shared_highlighting::LogGenerateErrorIFrame();
     OnGeneratedSelector(std::make_unique<ui::DataTransferEndpoint>(
                             main_frame->GetLastCommittedOrigin()),
                         std::string());
