diff --git a/chrome/browser/extensions/api/chrome_extensions_api_client.h b/chrome/browser/extensions/api/chrome_extensions_api_client.h
index 3aecf42019d3a..49c07e7f9b0af 100644
--- a/chrome/browser/extensions/api/chrome_extensions_api_client.h
+++ b/chrome/browser/extensions/api/chrome_extensions_api_client.h
@@ -79,7 +79,6 @@ class ChromeExtensionsAPIClient : public ExtensionsAPIClient {
   MetricsPrivateDelegate* GetMetricsPrivateDelegate() override;
   FileSystemDelegate* GetFileSystemDelegate() override;
   MessagingDelegate* GetMessagingDelegate() override;
-  FeedbackPrivateDelegate* GetFeedbackPrivateDelegate() override;
 
 #if BUILDFLAG(IS_CHROMEOS_ASH)
   MediaPerceptionAPIDelegate* GetMediaPerceptionAPIDelegate() override;
@@ -102,7 +101,6 @@ class ChromeExtensionsAPIClient : public ExtensionsAPIClient {
   std::unique_ptr<ChromeMetricsPrivateDelegate> metrics_private_delegate_;
   std::unique_ptr<FileSystemDelegate> file_system_delegate_;
   std::unique_ptr<MessagingDelegate> messaging_delegate_;
-  std::unique_ptr<FeedbackPrivateDelegate> feedback_private_delegate_;
 
 #if BUILDFLAG(IS_CHROMEOS_ASH)
   std::unique_ptr<MediaPerceptionAPIDelegate> media_perception_api_delegate_;
