diff --git a/chrome/browser/login_detection/login_detection_tab_helper.cc b/chrome/browser/login_detection/login_detection_tab_helper.cc
index c5a202967a369..ffd5012cfdee0 100644
--- a/chrome/browser/login_detection/login_detection_tab_helper.cc
+++ b/chrome/browser/login_detection/login_detection_tab_helper.cc
@@ -18,10 +18,6 @@
 #include "content/public/browser/navigation_handle.h"
 #include "content/public/browser/web_contents.h"
 #include "net/base/registry_controlled_domains/registry_controlled_domain.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
-#include "services/metrics/public/cpp/ukm_source.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 
 namespace login_detection {
 
@@ -34,10 +30,6 @@ PrefService* GetPrefs(content::WebContents* web_contents) {
 
 void RecordLoginDetectionMetrics(LoginDetectionType type,
                                  ukm::SourceId ukm_source_id) {
-  base::UmaHistogramEnumeration("Login.PageLoad.DetectionType", type);
-  ukm::builders::LoginDetection builder(ukm_source_id);
-  builder.SetPage_LoginType(static_cast<int64_t>(type))
-      .Record(ukm::UkmRecorder::Get());
 }
 
 }  // namespace
@@ -130,9 +122,6 @@ void LoginDetectionTabHelper::DidOpenAsPopUp(
 
 void LoginDetectionTabHelper::WebContentsDestroyed() {
   if (auto signedin_site = oauth_login_detector_->GetPopUpLoginFlowSite()) {
-    RecordLoginDetectionMetrics(
-        LoginDetectionType::kOauthPopUpFirstTimeLoginFlow,
-        ukm::GetSourceIdForWebContentsDocument(web_contents()));
     prefs::SaveSiteToOAuthSignedInList(GetPrefs(web_contents()),
                                        *signedin_site);
   }
