diff --git a/chrome/browser/resources/new_tab_page/BUILD.gn b/chrome/browser/resources/new_tab_page/BUILD.gn
index d58e45e12480..517edf63f54f 100644
--- a/chrome/browser/resources/new_tab_page/BUILD.gn
+++ b/chrome/browser/resources/new_tab_page/BUILD.gn
@@ -56,7 +56,6 @@ js_library("new_tab_page") {
 js_library("lazy_load") {
   deps = [
     ":customize_dialog",
-    ":middle_slot_promo",
     ":most_visited",
     ":voice_search_overlay",
     "modules/dummy:module",
