diff --git a/chrome/browser/web_applications/BUILD.gn b/chrome/browser/web_applications/BUILD.gn
index 3b105a6b36dd..e9562ef4e54f 100644
--- a/chrome/browser/web_applications/BUILD.gn
+++ b/chrome/browser/web_applications/BUILD.gn
@@ -16,8 +16,6 @@ group("web_app_test_group") {
 # Extension codepaths have been removed.
 source_set("common") {
   sources = [
-    "daily_metrics_helper.cc",
-    "daily_metrics_helper.h",
     "manifest_update_manager.cc",
     "manifest_update_manager.h",
     "manifest_update_task.cc",
@@ -36,7 +34,6 @@ source_set("common") {
     "//components/content_settings/core/browser",
     "//components/pref_registry",
     "//content/public/browser",
-    "//services/metrics/public/cpp:ukm_builders",
     "//skia",
   ]
 
@@ -235,7 +232,6 @@ source_set("web_applications_unit_tests") {
     "//components/services/app_service/public/cpp:protocol_handling",
     "//content/public/browser",
     "//content/test:test_support",
-    "//services/metrics/public/cpp:ukm_builders",
     "//skia",
     "//testing/gmock",
     "//testing/gtest",
