diff --git a/chrome/browser/download/save_package_file_picker.cc b/chrome/browser/download/save_package_file_picker.cc
index b534612ff65a..b47156f01f27 100644
--- a/chrome/browser/download/save_package_file_picker.cc
+++ b/chrome/browser/download/save_package_file_picker.cc
@@ -45,7 +45,6 @@ namespace {
 bool g_should_prompt_for_filename = true;
 
 void OnSavePackageDownloadCreated(download::DownloadItem* download) {
-  ChromeDownloadManagerDelegate::DisableSafeBrowsing(download);
 }
 
 // Adds "Webpage, HTML Only" type to FileTypeInfo.
