diff --git a/chrome/browser/prefetch/prefetch_proxy/prefetch_proxy_tab_helper.h b/chrome/browser/prefetch/prefetch_proxy/prefetch_proxy_tab_helper.h
index 3ef932fc87fed..b14ca6e272fdd 100644
--- a/chrome/browser/prefetch/prefetch_proxy/prefetch_proxy_tab_helper.h
+++ b/chrome/browser/prefetch/prefetch_proxy/prefetch_proxy_tab_helper.h
@@ -34,7 +34,6 @@
 #include "url/gurl.h"
 
 class PrefetchProxyPageLoadMetricsObserver;
-class PrefetchProxyPrefetchMetricsCollector;
 class PrefetchProxySubresourceManager;
 class Profile;
 
@@ -271,12 +270,6 @@ class PrefetchProxyTabHelper
     // Only set for pages after a Google SRP.
     std::unique_ptr<AfterSRPMetrics> after_srp_metrics_;
 
-    // Collects metrics on all prefetching. This is a scoped refptr so that it
-    // can also be shared with subresource managers until all pointers to it are
-    // destroyed, at which time it logs UKM.
-    scoped_refptr<PrefetchProxyPrefetchMetricsCollector>
-        prefetch_metrics_collector_;
-
     // The status of each prefetch.
     std::map<GURL, PrefetchProxyPrefetchStatus> prefetch_status_by_url_;
 
