diff --git a/chrome/browser/enterprise/connectors/connectors_manager.cc b/chrome/browser/enterprise/connectors/connectors_manager.cc
index 3c0cc458967a..02aa3853512e 100644
--- a/chrome/browser/enterprise/connectors/connectors_manager.cc
+++ b/chrome/browser/enterprise/connectors/connectors_manager.cc
@@ -6,6 +6,7 @@
 
 #include <memory>
 
+#include "base/bind.h"
 #include "base/feature_list.h"
 #include "base/no_destructor.h"
 #include "base/values.h"
@@ -13,7 +14,6 @@
 #include "chrome/browser/enterprise/connectors/connectors_prefs.h"
 #include "components/policy/core/browser/url_util.h"
 #include "components/prefs/pref_service.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "components/url_matcher/url_matcher.h"
 #include "url/gurl.h"
 
@@ -31,7 +31,7 @@ const char kServiceProviderConfig[] = R"({
       "version": {
         "1": {
           "analysis": {
-            "url": "https://safebrowsing.google.com/safebrowsing/uploads/scan",
+            "url": "https://no-thanks.invalid",
             "supported_tags": [
               {
                 "name": "malware",
@@ -84,7 +84,7 @@ const char kServiceProviderConfig[] = R"({
             ]
           },
           "reporting": {
-            "url": "https://chromereporting-pa.googleapis.com/v1/events"
+            "url": "https://no-thanks.invalid"
           }
         }
       }
