diff --git a/chrome/browser/renderer_context_menu/render_view_context_menu.h b/chrome/browser/renderer_context_menu/render_view_context_menu.h
index f4bc78469e2eb..411ebf14115ea 100644
--- a/chrome/browser/renderer_context_menu/render_view_context_menu.h
+++ b/chrome/browser/renderer_context_menu/render_view_context_menu.h
@@ -111,7 +111,6 @@ class RenderViewContextMenu : public RenderViewContextMenuBase,
 #if BUILDFLAG(ENABLE_EXTENSIONS)
   extensions::ContextMenuMatcher extension_items_;
 #endif
-  void RecordUsedItem(int id) override;
 
   // Returns true if the browser is in HTML fullscreen mode, initiated by the
   // page (as opposed to the user). Used to determine which shortcut to display.
@@ -147,7 +146,6 @@ class RenderViewContextMenu : public RenderViewContextMenuBase,
 
   // RenderViewContextMenuBase:
   void InitMenu() override;
-  void RecordShownItem(int id) override;
 #if BUILDFLAG(ENABLE_PLUGINS)
   void HandleAuthorizeAllPlugins() override;
 #endif
