diff --git a/chrome/browser/printing/print_job_worker.cc b/chrome/browser/printing/print_job_worker.cc
index 89c7798d8ddf..e20ba2b2d2e5 100644
--- a/chrome/browser/printing/print_job_worker.cc
+++ b/chrome/browser/printing/print_job_worker.cc
@@ -22,7 +22,6 @@
 #include "chrome/browser/chrome_notification_types.h"
 #include "chrome/browser/printing/print_job.h"
 #include "chrome/grit/generated_resources.h"
-#include "components/crash/core/common/crash_keys.h"
 #include "content/public/browser/browser_task_traits.h"
 #include "content/public/browser/browser_thread.h"
 #include "content/public/browser/notification_service.h"
@@ -208,7 +207,6 @@ void PrintJobWorker::UpdatePrintSettings(base::Value new_settings,
                                          SettingsCallback callback) {
   DCHECK_CURRENTLY_ON(BrowserThread::UI);
 
-  std::unique_ptr<crash_keys::ScopedPrinterInfo> crash_key;
   PrinterType type = static_cast<PrinterType>(
       new_settings.FindIntKey(kSettingPrinterType).value());
   if (type == PrinterType::kLocal) {
@@ -220,8 +218,6 @@ void PrintJobWorker::UpdatePrintSettings(base::Value new_settings,
     scoped_refptr<PrintBackend> print_backend =
         PrintBackend::CreateInstance(g_browser_process->GetApplicationLocale());
     std::string printer_name = *new_settings.FindStringKey(kSettingDeviceName);
-    crash_key = std::make_unique<crash_keys::ScopedPrinterInfo>(
-        print_backend->GetPrinterDriverInfo(printer_name));
 
 #if defined(OS_LINUX) && defined(USE_CUPS) && !defined(OS_CHROMEOS)
     PrinterBasicInfo basic_info;
