diff --git a/chrome/browser/previews/previews_ui_tab_helper.cc b/chrome/browser/previews/previews_ui_tab_helper.cc
index be82fd7d22bc..040dd60f9845 100644
--- a/chrome/browser/previews/previews_ui_tab_helper.cc
+++ b/chrome/browser/previews/previews_ui_tab_helper.cc
@@ -22,7 +22,6 @@
 #include "components/data_reduction_proxy/core/browser/data_reduction_proxy_settings.h"
 #include "components/data_reduction_proxy/core/common/data_reduction_proxy_headers.h"
 #include "components/network_time/network_time_tracker.h"
-#include "components/page_load_metrics/browser/metrics_web_contents_observer.h"
 #include "components/previews/content/previews_decider_impl.h"
 #include "components/previews/content/previews_ui_service.h"
 #include "components/previews/core/previews_experiments.h"
@@ -57,17 +56,6 @@ void AddPreviewNavigationCallback(content::BrowserContext* browser_context,
   }
 }
 
-void InformPLMOfOptOut(content::WebContents* web_contents) {
-  page_load_metrics::MetricsWebContentsObserver* metrics_web_contents_observer =
-      page_load_metrics::MetricsWebContentsObserver::FromWebContents(
-          web_contents);
-  if (!metrics_web_contents_observer)
-    return;
-
-  metrics_web_contents_observer->BroadcastEventToObservers(
-      PreviewsUITabHelper::OptOutEventKey());
-}
-
 }  // namespace
 
 PreviewsUITabHelper::~PreviewsUITabHelper() {
@@ -107,7 +95,6 @@ void PreviewsUITabHelper::ReloadWithoutPreviews() {
 
 void PreviewsUITabHelper::ReloadWithoutPreviews(
     previews::PreviewsType previews_type) {
-  InformPLMOfOptOut(web_contents());
 #if defined(OS_ANDROID)
   should_display_android_omnibox_badge_ = false;
 #endif
