diff --git a/chrome/browser/download/android/download_controller.h b/chrome/browser/download/android/download_controller.h
index 17d81ec3525c..a78cfe4971dc 100644
--- a/chrome/browser/download/android/download_controller.h
+++ b/chrome/browser/download/android/download_controller.h
@@ -50,7 +50,6 @@ class DownloadController : public DownloadControllerBase {
     STORAGE_PERMISSION_NO_WEB_CONTENTS,
     STORAGE_PERMISSION_MAX
   };
-  static void RecordStoragePermission(StoragePermissionType type);
 
   static void CloseTabIfEmpty(content::WebContents* web_contents);
 
