diff --git a/chrome/common/service_process_util.cc b/chrome/common/service_process_util.cc
index 5c968585498f..713289650c41 100644
--- a/chrome/common/service_process_util.cc
+++ b/chrome/common/service_process_util.cc
@@ -29,8 +29,6 @@
 #include "chrome/common/chrome_paths.h"
 #include "chrome/common/chrome_switches.h"
 #include "components/cloud_devices/common/cloud_devices_switches.h"
-#include "components/crash/core/app/crash_switches.h"
-#include "components/crash/core/app/crashpad.h"
 #include "components/network_session_configurator/common/network_switches.h"
 #include "components/version_info/version_info.h"
 #include "content/public/common/content_paths.h"
@@ -116,19 +114,6 @@ std::unique_ptr<base::CommandLine> CreateServiceProcessCommandLine() {
   command_line->AppendArg(switches::kPrefetchArgumentOther);
 #endif  // defined(OS_WIN)
 
-#if defined(OS_LINUX) || defined(OS_CHROMEOS)
-  if (crash_reporter::IsCrashpadEnabled()) {
-    command_line->AppendSwitch(crash_reporter::kEnableCrashpad);
-
-    pid_t pid;
-    if (crash_reporter::GetHandlerSocket(nullptr, &pid)) {
-      command_line->AppendSwitchASCII(
-          crash_reporter::switches::kCrashpadHandlerPid,
-          base::NumberToString(pid));
-    }
-  }
-#endif  // defined(OS_LINUX) || defined(OS_CHROMEOS)
-
   static const char* const kSwitchesToCopy[] = {
     network::switches::kIgnoreUrlFetcherCertRequests,
     switches::kCloudPrintSetupProxy,
