diff --git a/chrome/browser/ui/passwords/manage_passwords_ui_controller.h b/chrome/browser/ui/passwords/manage_passwords_ui_controller.h
index 1bf7ea9fa1125..47aeee8623c76 100644
--- a/chrome/browser/ui/passwords/manage_passwords_ui_controller.h
+++ b/chrome/browser/ui/passwords/manage_passwords_ui_controller.h
@@ -120,8 +120,6 @@ class ManagePasswordsUIController
   // PasswordsModelDelegate:
   content::WebContents* GetWebContents() const override;
   url::Origin GetOrigin() const override;
-  password_manager::PasswordFormMetricsRecorder*
-  GetPasswordFormMetricsRecorder() override;
   password_manager::PasswordFeatureManager* GetPasswordFeatureManager()
       override;
   password_manager::ui::State GetState() const override;
