diff --git a/chrome/chrome_elf/BUILD.gn b/chrome/chrome_elf/BUILD.gn
index 9b08e23e921b..063a7342aa9e 100644
--- a/chrome/chrome_elf/BUILD.gn
+++ b/chrome/chrome_elf/BUILD.gn
@@ -119,7 +119,6 @@ static_library("crash") {
     "//components/crash/core/common",  # crash_keys
     "//components/version_info:channel",
     "//content/public/common:result_codes",
-    "//third_party/crashpad/crashpad/client",  # DumpWithoutCrash
   ]
 }
 
