diff --git a/chrome/browser/payments/BUILD.gn b/chrome/browser/payments/BUILD.gn
index ff5578aa48de..a93c40c591f2 100644
--- a/chrome/browser/payments/BUILD.gn
+++ b/chrome/browser/payments/BUILD.gn
@@ -42,7 +42,6 @@ source_set("browsertests") {
     "//components/payments/content:utils",
     "//components/payments/core",
     "//components/payments/core:test_support",
-    "//components/ukm:test_support",
     "//device/fido:test_support",
   ]
 
