diff --git a/chrome/browser/extensions/chrome_extensions_browser_client.cc b/chrome/browser/extensions/chrome_extensions_browser_client.cc
index bd6c9c3ede262..a5418e9b42aad 100644
--- a/chrome/browser/extensions/chrome_extensions_browser_client.cc
+++ b/chrome/browser/extensions/chrome_extensions_browser_client.cc
@@ -90,10 +90,6 @@ const char kJsonUrlPath[] = "/service/update2/json";
 // new chrome update.
 bool g_did_chrome_update_for_testing = false;
 
-// The fake metrics logger instance to use for testing.
-MediaRouterExtensionAccessLogger* g_media_router_access_logger_for_testing =
-    nullptr;
-
 bool ExtensionsDisabled(const base::CommandLine& command_line) {
   return command_line.HasSwitch(::switches::kDisableExtensions) ||
          command_line.HasSwitch(::switches::kDisableExtensionsExcept);
@@ -567,13 +563,6 @@ void ChromeExtensionsBrowserClient::SetLastSaveFilePath(
   download_prefs->SetSaveFilePath(path);
 }
 
-const MediaRouterExtensionAccessLogger*
-ChromeExtensionsBrowserClient::GetMediaRouterAccessLogger() const {
-  return g_media_router_access_logger_for_testing
-             ? g_media_router_access_logger_for_testing
-             : &media_router_access_logger_;
-}
-
 bool ChromeExtensionsBrowserClient::HasIsolatedStorage(
     const std::string& extension_id,
     content::BrowserContext* context) {
@@ -592,12 +581,6 @@ bool ChromeExtensionsBrowserClient::IsValidTabId(
       tab_id, context, true /* include_incognito */, nullptr /* contents */);
 }
 
-// static
-void ChromeExtensionsBrowserClient::SetMediaRouterAccessLoggerForTesting(
-    MediaRouterExtensionAccessLogger* media_router_access_logger) {
-  g_media_router_access_logger_for_testing = media_router_access_logger;
-}
-
 // static
 void ChromeExtensionsBrowserClient::set_did_chrome_update_for_testing(
     bool did_update) {
