diff --git a/chrome/browser/extensions/api/settings_private/generated_prefs.cc b/chrome/browser/extensions/api/settings_private/generated_prefs.cc
index ba6944df9ab47..15258d517e34c 100644
--- a/chrome/browser/extensions/api/settings_private/generated_prefs.cc
+++ b/chrome/browser/extensions/api/settings_private/generated_prefs.cc
@@ -98,10 +98,6 @@ void GeneratedPrefs::CreatePrefs() {
   prefs_[content_settings::kCookieSessionOnly] =
       std::make_unique<content_settings::GeneratedCookieSessionOnlyPref>(
           profile_);
-  prefs_[kGeneratedPasswordLeakDetectionPref] =
-      std::make_unique<GeneratedPasswordLeakDetectionPref>(profile_);
-  prefs_[safe_browsing::kGeneratedSafeBrowsingPref] =
-      std::make_unique<safe_browsing::GeneratedSafeBrowsingPref>(profile_);
   prefs_[content_settings::kGeneratedNotificationPref] =
       std::make_unique<content_settings::GeneratedNotificationPref>(profile_);
   prefs_[kGeneratedFlocPref] = std::make_unique<GeneratedFlocPref>(profile_);
