diff --git a/chrome/app/chrome_main_delegate.h b/chrome/app/chrome_main_delegate.h
index e597bc9e05eff..0db5132f58965 100644
--- a/chrome/app/chrome_main_delegate.h
+++ b/chrome/app/chrome_main_delegate.h
@@ -29,7 +29,6 @@ class TracingSamplerProfiler;
 }
 
 class ChromeContentBrowserClient;
-class HeapProfilerController;
 
 // Chrome implementation of ContentMainDelegate.
 class ChromeMainDelegate : public content::ContentMainDelegate {
@@ -84,10 +83,6 @@ class ChromeMainDelegate : public content::ContentMainDelegate {
 
   std::unique_ptr<tracing::TracingSamplerProfiler> tracing_sampler_profiler_;
 
-  // The controller schedules UMA heap profiles collections and forwarding down
-  // the reporting pipeline.
-  std::unique_ptr<HeapProfilerController> heap_profiler_controller_;
-
 #if BUILDFLAG(IS_CHROMEOS_LACROS)
   std::unique_ptr<chromeos::LacrosChromeServiceImpl> lacros_chrome_service_;
 #endif
