diff --git a/chrome/browser/password_manager/chrome_password_manager_client.h b/chrome/browser/password_manager/chrome_password_manager_client.h
index cacf5a2d01518..9e48a7ae4d976 100644
--- a/chrome/browser/password_manager/chrome_password_manager_client.h
+++ b/chrome/browser/password_manager/chrome_password_manager_client.h
@@ -29,8 +29,6 @@
 #include "components/password_manager/core/browser/password_manager.h"
 #include "components/password_manager/core/browser/password_manager_client.h"
 #include "components/password_manager/core/browser/password_manager_client_helper.h"
-#include "components/password_manager/core/browser/password_manager_metrics_recorder.h"
-#include "components/password_manager/core/browser/password_manager_metrics_util.h"
 #include "components/password_manager/core/browser/password_reuse_detection_manager.h"
 #include "components/password_manager/core/browser/password_reuse_detector.h"
 #include "components/prefs/pref_member.h"
@@ -190,9 +188,6 @@ class ChromePasswordManagerClient
   void AnnotateNavigationEntry(bool has_password_field) override;
   autofill::LanguageCode GetPageLanguage() const override;
 
-  safe_browsing::PasswordProtectionService* GetPasswordProtectionService()
-      const override;
-
 #if defined(ON_FOCUS_PING_ENABLED)
   void CheckSafeBrowsingReputation(const GURL& form_action,
                                    const GURL& frame_url) override;
@@ -208,8 +203,6 @@ class ChromePasswordManagerClient
   void LogPasswordReuseDetectedEvent() override;
 
   ukm::SourceId GetUkmSourceId() override;
-  password_manager::PasswordManagerMetricsRecorder* GetMetricsRecorder()
-      override;
   password_manager::PasswordRequirementsService*
   GetPasswordRequirementsService() override;
   favicon::FaviconService* GetFaviconService() override;
@@ -342,8 +335,6 @@ class ChromePasswordManagerClient
   password_manager::PasswordManager password_manager_;
   password_manager::PasswordFeatureManagerImpl password_feature_manager_;
   password_manager::HttpAuthManagerImpl httpauth_manager_;
-  password_manager::PasswordReuseDetectionManager
-      password_reuse_detection_manager_;
 
 #if defined(OS_ANDROID)
   // Holds and facilitates a credential store for each origin in this tab.
@@ -400,12 +391,6 @@ class ChromePasswordManagerClient
 
   std::unique_ptr<autofill::LogManager> log_manager_;
 
-  // Recorder of metrics that is associated with the last committed navigation
-  // of the WebContents owning this ChromePasswordManagerClient. May be unset at
-  // times. Sends statistics on destruction.
-  base::Optional<password_manager::PasswordManagerMetricsRecorder>
-      metrics_recorder_;
-
   // Whether navigator.credentials.store() was ever called from this
   // WebContents. Used for testing.
   bool was_store_ever_called_ = false;
