diff --git a/chrome/browser/lifetime/browser_shutdown.h b/chrome/browser/lifetime/browser_shutdown.h
index eed4b72ef087..e293c1c7184d 100644
--- a/chrome/browser/lifetime/browser_shutdown.h
+++ b/chrome/browser/lifetime/browser_shutdown.h
@@ -91,9 +91,6 @@ bool RecordShutdownInfoPrefs();
 void ShutdownPostThreadsStop(RestartMode restart_mode);
 #endif
 
-// Called at startup to create a histogram from our previous shutdown time.
-void ReadLastShutdownInfo();
-
 // There are various situations where the browser process should continue to
 // run after the last browser window has closed - the Mac always continues
 // running until the user explicitly quits, and on Windows/Linux the application
