diff --git a/chrome/browser/devtools/BUILD.gn b/chrome/browser/devtools/BUILD.gn
index 4bfecdc7a0e3..02e5c5dbcc9e 100644
--- a/chrome/browser/devtools/BUILD.gn
+++ b/chrome/browser/devtools/BUILD.gn
@@ -210,7 +210,6 @@ static_library("devtools") {
   if (!is_android) {
     deps += [
       ":protocol_generated_sources",
-      "//components/subresource_filter/content/browser:browser",
       "//third_party/inspector_protocol:crdtp",
     ]
     sources += [
