diff --git a/chrome/browser/renderer_context_menu/render_view_context_menu.h b/chrome/browser/renderer_context_menu/render_view_context_menu.h
index 140c03836951..99dfe1dd7bf1 100644
--- a/chrome/browser/renderer_context_menu/render_view_context_menu.h
+++ b/chrome/browser/renderer_context_menu/render_view_context_menu.h
@@ -39,8 +39,6 @@ class PrintPreviewContextMenuObserver;
 class Profile;
 class QuickAnswersMenuObserver;
 class SharedClipboardContextMenuObserver;
-class SpellingMenuObserver;
-class SpellingOptionsSubMenuObserver;
 
 namespace content {
 class RenderFrameHost;
@@ -73,10 +71,6 @@ class RenderViewContextMenu : public RenderViewContextMenuBase {
 
   ~RenderViewContextMenu() override;
 
-  // Adds the spell check service item to the context menu.
-  static void AddSpellCheckServiceItem(ui::SimpleMenuModel* menu,
-                                       bool is_checked);
-
   // Range of command IDs to use for the items in the send tab to self submenu.
   static const int kMaxSendTabToSelfSubMenuCommandId =
       send_tab_to_self::SendTabToSelfSubMenuModel::kMaxCommandId;
@@ -86,7 +80,6 @@ class RenderViewContextMenu : public RenderViewContextMenuBase {
   bool IsCommandIdVisible(int command_id) const override;
   bool IsCommandIdEnabled(int command_id) const override;
   void ExecuteCommand(int command_id, int event_flags) override;
-  void AddSpellCheckServiceItem(bool is_checked) override;
   void AddAccessibilityLabelsServiceItem(bool is_checked) override;
 
   // Registers a one-time callback that will be called the next time a context
@@ -180,7 +173,6 @@ class RenderViewContextMenu : public RenderViewContextMenuBase {
   void AppendRotationItems();
   void AppendEditableItems();
   void AppendLanguageSettings();
-  void AppendSpellingSuggestionItems();
   // Returns true if the items were appended. This might not happen in all
   // cases, e.g. these are only appended if a screen reader is enabled.
   bool AppendAccessibilityLabelsItems();
@@ -264,22 +256,11 @@ class RenderViewContextMenu : public RenderViewContextMenuBase {
   ui::SimpleMenuModel protocol_handler_submenu_model_;
   ProtocolHandlerRegistry* protocol_handler_registry_;
 
-  // An observer that handles spelling suggestions, "Add to dictionary", and
-  // "Use enhanced spell check" items.
-  std::unique_ptr<SpellingMenuObserver> spelling_suggestions_menu_observer_;
-
   // An observer that handles accessibility labels items.
   std::unique_ptr<AccessibilityLabelsMenuObserver>
       accessibility_labels_menu_observer_;
   ui::SimpleMenuModel accessibility_labels_submenu_model_;
 
-#if !defined(OS_MAC)
-  // An observer that handles the submenu for showing spelling options. This
-  // submenu lets users select the spelling language, for example.
-  std::unique_ptr<SpellingOptionsSubMenuObserver>
-      spelling_options_submenu_observer_;
-#endif
-
 #if defined(OS_CHROMEOS)
   // An observer that handles "Open with <app>" items.
   std::unique_ptr<RenderViewContextMenuObserver> open_with_menu_observer_;
