diff --git a/chrome/browser/devtools/BUILD.gn b/chrome/browser/devtools/BUILD.gn
index 68f80ebb9a060..98350c3bd8e58 100644
--- a/chrome/browser/devtools/BUILD.gn
+++ b/chrome/browser/devtools/BUILD.gn
@@ -234,7 +234,6 @@ static_library("devtools") {
       "//components/media_router/browser",
       "//components/media_router/common/mojom:media_router",
       "//components/security_state/content",
-      "//components/subresource_filter/content/browser:browser",
       "//components/webapps/browser",
       "//third_party/inspector_protocol:crdtp",
     ]
