diff --git a/chrome/browser/resources/settings/hats_browser_proxy.js b/chrome/browser/resources/settings/hats_browser_proxy.js
index 89f4c0eb43d5..94bdd5e659e1 100644
--- a/chrome/browser/resources/settings/hats_browser_proxy.js
+++ b/chrome/browser/resources/settings/hats_browser_proxy.js
@@ -27,4 +27,3 @@ import {addSingletonGetter} from 'chrome://resources/js/cr.m.js';
   }
 
   addSingletonGetter(HatsBrowserProxyImpl);
-
