diff --git a/chrome/browser/extensions/api/autofill_private/autofill_private_api.h b/chrome/browser/extensions/api/autofill_private/autofill_private_api.h
index 16309999a8ad9..fcb77e9f42896 100644
--- a/chrome/browser/extensions/api/autofill_private/autofill_private_api.h
+++ b/chrome/browser/extensions/api/autofill_private/autofill_private_api.h
@@ -181,6 +181,7 @@ class AutofillPrivateMigrateCreditCardsFunction : public ExtensionFunction {
   ResponseAction Run() override;
 };
 
+#if 0
 class AutofillPrivateLogServerCardLinkClickedFunction
     : public ExtensionFunction {
  public:
@@ -198,6 +199,7 @@ class AutofillPrivateLogServerCardLinkClickedFunction
   // ExtensionFunction overrides.
   ResponseAction Run() override;
 };
+#endif
 
 class AutofillPrivateSetCreditCardFIDOAuthEnabledStateFunction
     : public ExtensionFunction {
