diff --git a/chrome/browser/content_index/content_index_provider_impl.h b/chrome/browser/content_index/content_index_provider_impl.h
index f540bcb47173..8086c726408a 100644
--- a/chrome/browser/content_index/content_index_provider_impl.h
+++ b/chrome/browser/content_index/content_index_provider_impl.h
@@ -11,7 +11,6 @@
 #include "base/memory/weak_ptr.h"
 #include "base/observer_list.h"
 #include "base/optional.h"
-#include "chrome/browser/content_index/content_index_metrics.h"
 #include "components/keyed_service/core/keyed_service.h"
 #include "components/offline_items_collection/core/offline_content_provider.h"
 #include "components/offline_items_collection/core/offline_item.h"
@@ -97,7 +96,6 @@ class ContentIndexProviderImpl
       const content::ContentIndexEntry& entry);
 
   Profile* profile_;
-  ContentIndexMetrics metrics_;
   offline_items_collection::OfflineContentAggregator* aggregator_;
   SiteEngagementService* site_engagement_service_;
   base::ObserverList<Observer>::Unchecked observers_;
