diff --git a/chrome/browser/content_index/content_index_provider_factory.cc b/chrome/browser/content_index/content_index_provider_factory.cc
index faac3c4b00ea..5ca113ab2917 100644
--- a/chrome/browser/content_index/content_index_provider_factory.cc
+++ b/chrome/browser/content_index/content_index_provider_factory.cc
@@ -6,7 +6,6 @@
 
 #include "chrome/browser/content_index/content_index_provider_impl.h"
 #include "chrome/browser/engagement/site_engagement_service_factory.h"
-#include "chrome/browser/metrics/ukm_background_recorder_service.h"
 #include "chrome/browser/offline_items_collection/offline_content_aggregator_factory.h"
 #include "chrome/browser/profiles/incognito_helpers.h"
 #include "chrome/browser/profiles/profile.h"
@@ -29,7 +28,6 @@ ContentIndexProviderFactory::ContentIndexProviderFactory()
           "ContentIndexProvider",
           BrowserContextDependencyManager::GetInstance()) {
   DependsOn(OfflineContentAggregatorFactory::GetInstance());
-  DependsOn(ukm::UkmBackgroundRecorderFactory::GetInstance());
   DependsOn(SiteEngagementServiceFactory::GetInstance());
 }
 
