diff --git a/chrome/browser/metrics/chrome_metrics_service_accessor.cc b/chrome/browser/metrics/chrome_metrics_service_accessor.cc
index f2f306546521..19e18c8ec620 100644
--- a/chrome/browser/metrics/chrome_metrics_service_accessor.cc
+++ b/chrome/browser/metrics/chrome_metrics_service_accessor.cc
@@ -33,12 +33,14 @@ void ChromeMetricsServiceAccessor::SetMetricsAndCrashReportingForTesting(
 
 // static
 bool ChromeMetricsServiceAccessor::IsMetricsAndCrashReportingEnabled() {
-  return IsMetricsAndCrashReportingEnabled(g_browser_process->local_state());
+  return false;
 }
 
 // static
 bool ChromeMetricsServiceAccessor::IsMetricsAndCrashReportingEnabled(
     PrefService* local_state) {
+  return false;
+#if 0
   if (g_metrics_consent_for_testing)
     return *g_metrics_consent_for_testing;
 
@@ -57,20 +59,18 @@ bool ChromeMetricsServiceAccessor::IsMetricsAndCrashReportingEnabled(
   }
 
   return IsMetricsReportingEnabled(local_state);
+#endif
 }
 
 // static
 bool ChromeMetricsServiceAccessor::RegisterSyntheticFieldTrial(
     base::StringPiece trial_name,
     base::StringPiece group_name) {
-  return metrics::MetricsServiceAccessor::RegisterSyntheticFieldTrial(
-      g_browser_process->metrics_service(), trial_name, group_name);
+  return true;
 }
 
 void ChromeMetricsServiceAccessor::SetForceIsMetricsReportingEnabledPrefLookup(
     bool value) {
-  metrics::MetricsServiceAccessor::SetForceIsMetricsReportingEnabledPrefLookup(
-      value);
 }
 
 #if BUILDFLAG(ENABLE_PLUGINS)
