diff --git a/chrome/browser/safe_browsing/cloud_content_scanning/binary_upload_service.cc b/chrome/browser/safe_browsing/cloud_content_scanning/binary_upload_service.cc
index 1db9d02f9ad9..6836fb03406a 100644
--- a/chrome/browser/safe_browsing/cloud_content_scanning/binary_upload_service.cc
+++ b/chrome/browser/safe_browsing/cloud_content_scanning/binary_upload_service.cc
@@ -33,7 +33,6 @@
 #include "components/enterprise/common/strings.h"
 #include "components/prefs/pref_service.h"
 #include "components/safe_browsing/content/web_ui/safe_browsing_ui.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "components/safe_browsing/core/features.h"
 #include "components/safe_browsing/core/proto/webprotect.pb.h"
 #include "content/public/browser/browser_task_traits.h"
@@ -47,10 +46,10 @@ namespace {
 const int kScanningTimeoutSeconds = 5 * 60;  // 5 minutes
 
 const char kSbEnterpriseUploadUrl[] =
-    "https://safebrowsing.google.com/safebrowsing/uploads/scan";
+    "https://no-thanks.invalid";
 
 const char kSbAppUploadUrl[] =
-    "https://safebrowsing.google.com/safebrowsing/uploads/app";
+    "https://no-thanks.invalid";
 
 bool IsAdvancedProtectionRequest(const BinaryUploadService::Request& request) {
   if (request.use_legacy_proto()) {
