diff --git a/chrome/browser/ui/autofill/payments/local_card_migration_bubble_controller_impl.cc b/chrome/browser/ui/autofill/payments/local_card_migration_bubble_controller_impl.cc
index 2d624bcd30fd6..e29dbb5fd828d 100644
--- a/chrome/browser/ui/autofill/payments/local_card_migration_bubble_controller_impl.cc
+++ b/chrome/browser/ui/autofill/payments/local_card_migration_bubble_controller_impl.cc
@@ -14,7 +14,6 @@
 #include "chrome/browser/ui/browser.h"
 #include "chrome/browser/ui/browser_finder.h"
 #include "chrome/browser/ui/browser_window.h"
-#include "components/autofill/core/browser/autofill_metrics.h"
 #include "components/autofill/core/browser/payments/local_card_migration_strike_database.h"
 #include "components/autofill/core/browser/strike_database.h"
 #include "components/autofill/core/common/autofill_clock.h"
@@ -44,9 +43,6 @@ void LocalCardMigrationBubbleControllerImpl::ShowBubble(
   local_card_migration_bubble_closure_ =
       std::move(local_card_migration_bubble_closure);
 
-  AutofillMetrics::LogLocalCardMigrationBubbleOfferMetric(
-      AutofillMetrics::LOCAL_CARD_MIGRATION_BUBBLE_REQUESTED, is_reshow_);
-
   Show();
 }
 
@@ -55,8 +51,6 @@ void LocalCardMigrationBubbleControllerImpl::ReshowBubble() {
     return;
 
   is_reshow_ = true;
-  AutofillMetrics::LogLocalCardMigrationBubbleOfferMetric(
-      AutofillMetrics::LOCAL_CARD_MIGRATION_BUBBLE_REQUESTED, is_reshow_);
 
   Show();
 }
@@ -90,30 +84,6 @@ void LocalCardMigrationBubbleControllerImpl::OnBubbleClosed(
     should_add_strikes_on_bubble_close_ = false;
     AddStrikesForBubbleClose();
   }
-
-  // Log local card migration bubble result according to the closed reason.
-  AutofillMetrics::LocalCardMigrationBubbleResultMetric metric;
-  switch (closed_reason) {
-    case PaymentsBubbleClosedReason::kAccepted:
-      metric = AutofillMetrics::LOCAL_CARD_MIGRATION_BUBBLE_ACCEPTED;
-      break;
-    case PaymentsBubbleClosedReason::kClosed:
-      metric = AutofillMetrics::LOCAL_CARD_MIGRATION_BUBBLE_CLOSED;
-      break;
-    case PaymentsBubbleClosedReason::kNotInteracted:
-      metric = AutofillMetrics::LOCAL_CARD_MIGRATION_BUBBLE_NOT_INTERACTED;
-      break;
-    case PaymentsBubbleClosedReason::kLostFocus:
-      metric = AutofillMetrics::LOCAL_CARD_MIGRATION_BUBBLE_LOST_FOCUS;
-      break;
-    case PaymentsBubbleClosedReason::kUnknown:
-      metric = AutofillMetrics::LOCAL_CARD_MIGRATION_BUBBLE_RESULT_UNKNOWN;
-      break;
-    case PaymentsBubbleClosedReason::kCancelled:
-      NOTREACHED();
-      return;
-  }
-  AutofillMetrics::LogLocalCardMigrationBubbleResultMetric(metric, is_reshow_);
 }
 
 PageActionIconType
@@ -132,8 +102,6 @@ void LocalCardMigrationBubbleControllerImpl::DoShowBubble() {
           ->ShowLocalCardMigrationBubble(web_contents(), this, is_reshow_));
   DCHECK(bubble_view());
 
-  AutofillMetrics::LogLocalCardMigrationBubbleOfferMetric(
-      AutofillMetrics::LOCAL_CARD_MIGRATION_BUBBLE_SHOWN, is_reshow_);
 }
 
 void LocalCardMigrationBubbleControllerImpl::AddStrikesForBubbleClose() {
