diff --git a/chrome/browser/ui/passwords/passwords_model_delegate_mock.h b/chrome/browser/ui/passwords/passwords_model_delegate_mock.h
index 89e7aa5e6455..4b1c4a3896a0 100644
--- a/chrome/browser/ui/passwords/passwords_model_delegate_mock.h
+++ b/chrome/browser/ui/passwords/passwords_model_delegate_mock.h
@@ -21,10 +21,6 @@ class PasswordsModelDelegateMock
   ~PasswordsModelDelegateMock() override;
 
   MOCK_METHOD(content::WebContents*, GetWebContents, (), (const override));
-  MOCK_METHOD(password_manager::PasswordFormMetricsRecorder*,
-              GetPasswordFormMetricsRecorder,
-              (),
-              (override));
   MOCK_METHOD(password_manager::PasswordFeatureManager*,
               GetPasswordFeatureManager,
               (),
