diff --git a/chromecast/browser/metrics/cast_browser_metrics.cc b/chromecast/browser/metrics/cast_browser_metrics.cc
index 93129e3745da..667d06defb2a 100644
--- a/chromecast/browser/metrics/cast_browser_metrics.cc
+++ b/chromecast/browser/metrics/cast_browser_metrics.cc
@@ -55,6 +55,7 @@ CastBrowserMetrics::~CastBrowserMetrics() {
 }
 
 void CastBrowserMetrics::Initialize() {
+  return;
   metrics_service_client_->InitializeMetricsService();
 
   auto* metrics_service = metrics_service_client_->GetMetricsService();
