diff --git a/chrome/browser/resources/settings/BUILD.gn b/chrome/browser/resources/settings/BUILD.gn
index 2c3ac524ed8b..125dfca4e554 100644
--- a/chrome/browser/resources/settings/BUILD.gn
+++ b/chrome/browser/resources/settings/BUILD.gn
@@ -162,7 +162,6 @@ group("closure_compile_module") {
     "printing_page:closure_compile_module",
     "privacy_page:closure_compile_module",
     "reset_page:closure_compile_module",
-    "safety_check_page:closure_compile_module",
     "search_engines_page:closure_compile_module",
     "search_page:closure_compile_module",
     "settings_main:closure_compile_module",
@@ -226,7 +225,6 @@ js_library("settings") {
     "autofill_page:password_manager_proxy",
     "people_page:sync_browser_proxy.m",
     "privacy_page:privacy_page_browser_proxy.m",
-    "safety_check_page:safety_check_browser_proxy",
     "search_engines_page:search_engines_browser_proxy.m",
     "settings_ui:settings_ui",
   ]
@@ -364,7 +362,6 @@ group("polymer3_elements") {
     "printing_page:web_components",
     "privacy_page:polymer3_elements",
     "reset_page:web_components",
-    "safety_check_page:web_components",
     "search_engines_page:web_components",
     "search_page:web_components",
     "settings_main:web_components",
