diff --git a/chrome/browser/component_updater/registration.cc b/chrome/browser/component_updater/registration.cc
index 71e027c8f5fa4..ab7490a7f7bd6 100644
--- a/chrome/browser/component_updater/registration.cc
+++ b/chrome/browser/component_updater/registration.cc
@@ -13,21 +13,15 @@
 #include "chrome/browser/browser_process.h"
 #include "chrome/browser/buildflags.h"
 #include "chrome/browser/component_updater/autofill_regex_component_installer.h"
-#include "chrome/browser/component_updater/chrome_origin_trials_component_installer.h"
 #include "chrome/browser/component_updater/crl_set_component_installer.h"
-#include "chrome/browser/component_updater/crowd_deny_component_installer.h"
 #include "chrome/browser/component_updater/desktop_sharing_hub_component_installer.h"
-#include "chrome/browser/component_updater/file_type_policies_component_installer.h"
 #include "chrome/browser/component_updater/first_party_sets_component_installer.h"
-#include "chrome/browser/component_updater/floc_component_installer.h"
 #include "chrome/browser/component_updater/hyphenation_component_installer.h"
 #include "chrome/browser/component_updater/mei_preload_component_installer.h"
 #include "chrome/browser/component_updater/pepper_flash_component_installer.h"
 #include "chrome/browser/component_updater/ssl_error_assistant_component_installer.h"
 #include "chrome/browser/component_updater/sth_set_component_remover.h"
-#include "chrome/browser/component_updater/subresource_filter_component_installer.h"
 #include "chrome/browser/component_updater/trust_token_key_commitments_component_installer.h"
-#include "chrome/browser/component_updater/zxcvbn_data_component_installer.h"
 #include "chrome/common/buildflags.h"
 #include "chrome/common/chrome_paths.h"
 #include "chrome/common/pref_names.h"
@@ -36,7 +30,6 @@
 #include "components/component_updater/installer_policies/autofill_states_component_installer.h"
 #include "components/component_updater/installer_policies/on_device_head_suggest_component_installer.h"
 #include "components/component_updater/installer_policies/optimization_hints_component_installer.h"
-#include "components/component_updater/installer_policies/safety_tips_component_installer.h"
 #include "components/nacl/common/buildflags.h"
 #include "device/vr/buildflags/buildflags.h"
 #include "third_party/widevine/cdm/buildflags.h"
@@ -114,9 +107,6 @@ void RegisterComponentsForUpdate(bool is_off_the_record_profile,
 #endif  // BUILDFLAG(IS_CHROMEOS_ASH)
 #endif  // BUILDFLAG(ENABLE_NACL) && !defined(OS_ANDROID)
 
-  RegisterSubresourceFilterComponent(cus);
-  RegisterFlocComponent(cus,
-                        g_browser_process->floc_sorting_lsh_clusters_service());
   RegisterOnDeviceHeadSuggestComponent(
       cus, g_browser_process->GetApplicationLocale());
   RegisterOptimizationHintsComponent(cus);
@@ -138,7 +128,6 @@ void RegisterComponentsForUpdate(bool is_off_the_record_profile,
 #endif
   }
   RegisterSSLErrorAssistantComponent(cus);
-  RegisterFileTypePoliciesComponent(cus);
   RegisterDesktopSharingHubComponent(cus);
 
 #if !BUILDFLAG(IS_CHROMEOS_ASH)
@@ -148,7 +137,6 @@ void RegisterComponentsForUpdate(bool is_off_the_record_profile,
   component_updater::RegisterCRLSetComponent(cus);
 #endif  // !BUILDFLAG(IS_CHROMEOS_ASH)
 
-  RegisterOriginTrialsComponent(cus);
   RegisterMediaEngagementPreloadComponent(cus, base::OnceClosure());
 
 #if defined(OS_WIN)
@@ -156,7 +144,6 @@ void RegisterComponentsForUpdate(bool is_off_the_record_profile,
   // on chromium build bots, it is always registered here and
   // RegisterSwReporterComponent() has support for running only in official
   // builds or tests.
-  RegisterSwReporterComponent(cus);
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
   RegisterThirdPartyModuleListComponent(cus);
 #endif  // BUILDFLAG(GOOGLE_CHROME_BRANDING)
@@ -168,9 +155,6 @@ void RegisterComponentsForUpdate(bool is_off_the_record_profile,
   }
 #endif
 
-  RegisterSafetyTipsComponent(cus);
-  RegisterCrowdDenyComponent(cus);
-
 #if BUILDFLAG(IS_CHROMEOS_ASH)
   RegisterSmartDimComponent(cus);
 #endif  // !BUILDFLAG(IS_CHROMEOS_ASH)
@@ -179,8 +163,6 @@ void RegisterComponentsForUpdate(bool is_off_the_record_profile,
   RegisterHyphenationComponent(cus);
 #endif
 
-  RegisterZxcvbnDataComponent(cus);
-
   RegisterAutofillStatesComponent(cus, g_browser_process->local_state());
 
   RegisterAutofillRegexComponent(cus);
