diff --git a/chrome/browser/ui/aura/accessibility/automation_manager_aura.cc b/chrome/browser/ui/aura/accessibility/automation_manager_aura.cc
index 0555e3e1520f..595d030f0b03 100644
--- a/chrome/browser/ui/aura/accessibility/automation_manager_aura.cc
+++ b/chrome/browser/ui/aura/accessibility/automation_manager_aura.cc
@@ -10,7 +10,6 @@
 #include "base/no_destructor.h"
 #include "build/build_config.h"
 #include "build/chromeos_buildflags.h"
-#include "components/crash/core/common/crash_key.h"
 #include "ui/accessibility/aura/aura_window_properties.h"
 #include "ui/accessibility/ax_action_data.h"
 #include "ui/accessibility/ax_action_handler_base.h"
@@ -342,11 +341,6 @@ void AutomationManagerAura::OnSerializeFailure(ax::mojom::Event event_type,
   ui::AXTreeSourceChecker<views::AXAuraObjWrapper*> checker(tree_.get());
   checker.CheckAndGetErrorString(&error_string);
 
-  // Add a crash key so we can figure out why this is happening.
-  static crash_reporter::CrashKeyString<256> ax_tree_source_error(
-      "ax_tree_source_error");
-  ax_tree_source_error.Set(error_string);
-
   LOG(ERROR) << "Unable to serialize accessibility event!\n"
              << "Event type: " << event_type << "\n"
              << "Error: " << error_string << "\n"
