diff --git a/chrome/browser/prerender/isolated/isolated_prerender_network_context_client.h b/chrome/browser/prerender/isolated/isolated_prerender_network_context_client.h
index a2d0cccb114e..09029c6634d8 100644
--- a/chrome/browser/prerender/isolated/isolated_prerender_network_context_client.h
+++ b/chrome/browser/prerender/isolated/isolated_prerender_network_context_client.h
@@ -72,7 +72,6 @@ class IsolatedPrerenderNetworkContextClient
 #if defined(OS_CHROMEOS)
   void OnTrustAnchorUsed() override;
 #endif
-  void OnSCTReportReady(const std::string& cache_key) override;
 };
 
 #endif  // CHROME_BROWSER_PRERENDER_ISOLATED_ISOLATED_PRERENDER_NETWORK_CONTEXT_CLIENT_H_
