diff --git a/chrome/browser/ui/webui/settings/about_handler.cc b/chrome/browser/ui/webui/settings/about_handler.cc
index c5214569d93e..46e9f36615bf 100644
--- a/chrome/browser/ui/webui/settings/about_handler.cc
+++ b/chrome/browser/ui/webui/settings/about_handler.cc
@@ -34,7 +34,6 @@
 #include "chrome/browser/ui/browser_commands.h"
 #include "chrome/browser/ui/browser_finder.h"
 #include "chrome/browser/ui/chrome_pages.h"
-#include "chrome/browser/upgrade_detector/upgrade_detector.h"
 #include "chrome/common/pref_names.h"
 #include "chrome/common/url_constants.h"
 #include "chrome/grit/chromium_strings.h"
@@ -256,11 +255,9 @@ AboutHandler::AboutHandler(Profile* profile)
     : profile_(profile),
       apply_changes_from_upgrade_observer_(false),
       clock_(base::DefaultClock::GetInstance()) {
-  UpgradeDetector::GetInstance()->AddObserver(this);
 }
 
 AboutHandler::~AboutHandler() {
-  UpgradeDetector::GetInstance()->RemoveObserver(this);
 }
 
 void AboutHandler::RegisterMessages() {
@@ -271,10 +268,6 @@ void AboutHandler::RegisterMessages() {
       "refreshUpdateStatus",
       base::BindRepeating(&AboutHandler::HandleRefreshUpdateStatus,
                           base::Unretained(this)));
-  web_ui()->RegisterMessageCallback(
-      "openFeedbackDialog",
-      base::BindRepeating(&AboutHandler::HandleOpenFeedbackDialog,
-                          base::Unretained(this)));
   web_ui()->RegisterMessageCallback(
       "openHelpPage", base::BindRepeating(&AboutHandler::HandleOpenHelpPage,
                                           base::Unretained(this)));
@@ -409,11 +402,6 @@ void AboutHandler::PromoteUpdater(const base::ListValue* args) {
 #endif
 
 void AboutHandler::HandleOpenFeedbackDialog(const base::ListValue* args) {
-  DCHECK(args->empty());
-  Browser* browser =
-      chrome::FindBrowserWithWebContents(web_ui()->GetWebContents());
-  chrome::OpenFeedbackDialog(browser,
-                             chrome::kFeedbackSourceMdSettingsAboutPage);
 }
 
 void AboutHandler::HandleOpenHelpPage(const base::ListValue* args) {
