diff --git a/chrome/browser/profiles/avatar_menu.cc b/chrome/browser/profiles/avatar_menu.cc
index 2f7e2769565a1..37934c6e7f77b 100644
--- a/chrome/browser/profiles/avatar_menu.cc
+++ b/chrome/browser/profiles/avatar_menu.cc
@@ -99,8 +99,8 @@ void AvatarMenu::SwitchToProfile(size_t index, bool always_create) {
                             ProfileManager::CreateCallback());
 }
 
-void AvatarMenu::AddNewProfile(ProfileMetrics::ProfileAdd type) {
-  menu_actions_->AddNewProfile(type);
+void AvatarMenu::AddNewProfile() {
+  menu_actions_->AddNewProfile();
 }
 
 void AvatarMenu::EditProfile(size_t index) {
