diff --git a/chrome/browser/offline_pages/prefetch/offline_metrics_collector_impl.h b/chrome/browser/offline_pages/prefetch/offline_metrics_collector_impl.h
index 87bb12ca6a40c..1a4d9a1d59fe3 100644
--- a/chrome/browser/offline_pages/prefetch/offline_metrics_collector_impl.h
+++ b/chrome/browser/offline_pages/prefetch/offline_metrics_collector_impl.h
@@ -9,7 +9,6 @@
 
 #include "base/macros.h"
 #include "base/time/clock.h"
-#include "components/offline_pages/core/prefetch/offline_metrics_collector.h"
 #include "components/prefs/pref_registry_simple.h"
 #include "components/prefs/pref_service.h"
 
