diff --git a/chrome/browser/signin/dice_web_signin_interceptor.cc b/chrome/browser/signin/dice_web_signin_interceptor.cc
index bbb1fce4b8ccc..c633e69e7b73d 100644
--- a/chrome/browser/signin/dice_web_signin_interceptor.cc
+++ b/chrome/browser/signin/dice_web_signin_interceptor.cc
@@ -21,7 +21,6 @@
 #include "chrome/browser/profiles/profile_attributes_storage.h"
 #include "chrome/browser/profiles/profile_avatar_icon_util.h"
 #include "chrome/browser/profiles/profile_manager.h"
-#include "chrome/browser/profiles/profile_metrics.h"
 #include "chrome/browser/profiles/profiles_state.h"
 #include "chrome/browser/signin/dice_intercepted_session_startup_helper.h"
 #include "chrome/browser/signin/dice_signed_in_profile_creator.h"
@@ -528,8 +527,6 @@ void DiceWebSigninInterceptor::OnNewSignedInProfileCreated(
     base::UmaHistogramTimes(
         "Signin.Intercept.ProfileCreationDuration",
         base::TimeTicks::Now() - profile_creation_start_time_);
-    ProfileMetrics::LogProfileAddNewUser(
-        ProfileMetrics::ADD_NEW_USER_SIGNIN_INTERCEPTION);
     if (!new_profile->IsEphemeralGuestProfile()) {
       // Apply the new color to the profile.
       ThemeServiceFactory::GetForProfile(new_profile)
