diff --git a/chrome/browser/ui/webui/new_tab_page/new_tab_page.mojom b/chrome/browser/ui/webui/new_tab_page/new_tab_page.mojom
index 35ce8c998e79f..bdee4b8799fe2 100644
--- a/chrome/browser/ui/webui/new_tab_page/new_tab_page.mojom
+++ b/chrome/browser/ui/webui/new_tab_page/new_tab_page.mojom
@@ -273,8 +273,6 @@ interface PageHandler {
   GetDoodle() => (Doodle? doodle);
   // Choose custom background from local file system.
   ChooseLocalCustomBackground() => (bool success);
-  // Get the middle slot promo if it exists.
-  GetPromo() => (Promo? promo);
   // Called when a module the given id is dismissed.
   OnDismissModule(string module_id);
   // Called when a module the given id is restored.
@@ -295,9 +293,6 @@ interface PageHandler {
   // Logs that the One Google Bar was added to the DOM / loaded in an iframe at
   // |time|.
   OnOneGoogleBarRendered(double time);
-  // Logs that the promo iframe was loaded at |time| and pings |log_url| for
-  // promo metrics logging.
-  OnPromoRendered(double time, url.mojom.Url? log_url);
   // Logs that |tile| at position |index| was triggered to navigate to that
   // most visited entry.
   // |mouse_button| indicates which mouse button was pressed on the entry. See
@@ -329,8 +324,6 @@ interface PageHandler {
   OnDoodleShared(DoodleShareChannel channel,
                  string doodle_id,
                  string? share_id);
-  // Logs that a link on a promo has been clicked.
-  OnPromoLinkClicked();
   // Logs that the <ntp-app> element's |ready| callback method was called.
   OnAppRendered(double time);
 };
