diff --git a/chrome/browser/resource_coordinator/lifecycle_unit.h b/chrome/browser/resource_coordinator/lifecycle_unit.h
index 989493292d1ee..4ad4f471ff1ae 100644
--- a/chrome/browser/resource_coordinator/lifecycle_unit.h
+++ b/chrome/browser/resource_coordinator/lifecycle_unit.h
@@ -152,10 +152,6 @@ class LifecycleUnit {
   // Adds/removes an observer to this LifecycleUnit.
   virtual void AddObserver(LifecycleUnitObserver* observer) = 0;
   virtual void RemoveObserver(LifecycleUnitObserver* observer) = 0;
-
-  // Returns the UKM source ID associated with this LifecycleUnit, if it has
-  // one.
-  virtual ukm::SourceId GetUkmSourceId() const = 0;
 };
 
 using LifecycleUnitSet = base::flat_set<LifecycleUnit*>;
