diff --git a/chrome/browser/net/trial_comparison_cert_verifier_controller.h b/chrome/browser/net/trial_comparison_cert_verifier_controller.h
index f22d29f1935ef..e8b862b77c8e8 100644
--- a/chrome/browser/net/trial_comparison_cert_verifier_controller.h
+++ b/chrome/browser/net/trial_comparison_cert_verifier_controller.h
@@ -14,7 +14,6 @@
 #include "base/memory/ref_counted.h"
 #include "base/memory/weak_ptr.h"
 #include "base/threading/thread_checker.h"
-#include "components/prefs/pref_change_registrar.h"
 #include "mojo/public/cpp/bindings/pending_receiver.h"
 #include "mojo/public/cpp/bindings/pending_remote.h"
 #include "mojo/public/cpp/bindings/receiver_set.h"
@@ -71,7 +70,6 @@ class TrialComparisonCertVerifierController
   void RefreshState();
 
   Profile* profile_;
-  PrefChangeRegistrar pref_change_registrar_;
 
   mojo::ReceiverSet<
       cert_verifier::mojom::TrialComparisonCertVerifierReportClient>
