diff --git a/chrome/browser/ui/views/chrome_browser_main_extra_parts_views.cc b/chrome/browser/ui/views/chrome_browser_main_extra_parts_views.cc
index 09e8f968b6f5..6dd0ae915d6b 100644
--- a/chrome/browser/ui/views/chrome_browser_main_extra_parts_views.cc
+++ b/chrome/browser/ui/views/chrome_browser_main_extra_parts_views.cc
@@ -14,7 +14,6 @@
 #include "chrome/browser/ui/views/chrome_layout_provider.h"
 #include "chrome/browser/ui/views/chrome_views_delegate.h"
 #include "chrome/browser/ui/views/devtools_process_observer.h"
-#include "chrome/browser/ui/views/relaunch_notification/relaunch_notification_controller.h"
 #include "components/constrained_window/constrained_window_views.h"
 #include "components/ui_devtools/connector_delegate.h"
 #include "components/ui_devtools/switches.h"
@@ -134,14 +133,7 @@ void ChromeBrowserMainExtraPartsViews::PreProfileInit() {
 }
 
 void ChromeBrowserMainExtraPartsViews::PostBrowserStart() {
-  relaunch_notification_controller_ =
-      std::make_unique<RelaunchNotificationController>(
-          UpgradeDetector::GetInstance());
 }
 
 void ChromeBrowserMainExtraPartsViews::PostMainMessageLoopRun() {
-  // The relaunch notification controller acts on timer-based events. Tear it
-  // down explicitly here to avoid a case where such an event arrives during
-  // shutdown.
-  relaunch_notification_controller_.reset();
 }
