diff --git a/chrome/browser/ui/views/omnibox/omnibox_view_views.cc b/chrome/browser/ui/views/omnibox/omnibox_view_views.cc
index 1f127e190912b..84d1df4eae950 100644
--- a/chrome/browser/ui/views/omnibox/omnibox_view_views.cc
+++ b/chrome/browser/ui/views/omnibox/omnibox_view_views.cc
@@ -2554,8 +2554,7 @@ gfx::Range OmniboxViewViews::GetSimplifiedDomainBounds(
   url::Component host = GetHostComponentAfterTrivialSubdomain();
 
   GURL url = url_formatter::FixupURL(base::UTF16ToUTF8(text), std::string());
-  if (!OmniboxFieldTrial::ShouldMaybeElideToRegistrableDomain() ||
-      !ShouldElideToRegistrableDomain(url)) {
+  if (!OmniboxFieldTrial::ShouldMaybeElideToRegistrableDomain()) {
     ranges_surrounding_simplified_domain->emplace_back(0, host.begin);
     ranges_surrounding_simplified_domain->emplace_back(host.end(), text.size());
     return gfx::Range(host.begin, host.end());
