diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index 58a6545751b2..8121f4b9a848 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -208,8 +208,6 @@ static_library("ui") {
     "webui/constrained_web_dialog_ui.h",
     "webui/cookies_tree_model_util.cc",
     "webui/cookies_tree_model_util.h",
-    "webui/crashes_ui.cc",
-    "webui/crashes_ui.h",
     "webui/device_log_ui.cc",
     "webui/device_log_ui.h",
     "webui/domain_reliability_internals_ui.cc",
@@ -319,15 +317,9 @@ static_library("ui") {
     ]
   }
 
-  if (safe_browsing_mode == 1) {
-    sources += [
-      "webui/reset_password/reset_password_ui.cc",
-      "webui/reset_password/reset_password_ui.h",
-    ]
-  }
 
   # TODO(crbug/925153): Remove this circular dependency.
-  allow_circular_includes_from = [ "//chrome/browser/safe_browsing" ]
+  allow_circular_includes_from = []
   defines = []
   libs = []
 
@@ -344,7 +336,6 @@ static_library("ui") {
   public_deps = [
     "//chrome/services/machine_learning:machine_learning_tflite_buildflags",
     "//components/dom_distiller/core",
-    "//components/safe_browsing:buildflags",
     "//components/sync",
     "//components/sync_user_events",
     "//components/translate/content/browser",
@@ -381,7 +372,6 @@ static_library("ui") {
     "//chrome/browser/resources/omnibox:resources",
     "//chrome/browser/resources/quota_internals:quota_internals_resources",
     "//chrome/browser/resources/usb_internals:resources",
-    "//chrome/browser/safe_browsing",
     "//chrome/browser/ui/webui/bluetooth_internals",
     "//chrome/browser/ui/webui/downloads:mojo_bindings",
     "//chrome/browser/ui/webui/interventions_internals:mojo_bindings",
@@ -479,21 +469,9 @@ static_library("ui") {
     "//components/previews/core",
     "//components/proxy_config",
     "//components/query_parser",
-    "//components/rappor",
     "//components/reading_list/core",
     "//components/renderer_context_menu",
     "//components/resources",
-    "//components/safe_browsing/content/password_protection",
-    "//components/safe_browsing/content/password_protection:password_protection_metrics_util",
-    "//components/safe_browsing/content/triggers:ad_popup_trigger",
-    "//components/safe_browsing/content/triggers:ad_redirect_trigger",
-    "//components/safe_browsing/content/web_ui",
-    "//components/safe_browsing/core:csd_proto",
-    "//components/safe_browsing/core:features",
-    "//components/safe_browsing/core/common",
-    "//components/safe_browsing/core/common:safe_browsing_prefs",
-    "//components/safe_browsing/core/db:database_manager",
-    "//components/safe_browsing/core/db:util",
     "//components/schema_org/common:improved_mojom",
     "//components/search",
     "//components/search_engines",
@@ -509,10 +487,7 @@ static_library("ui") {
     "//components/signin/public/identity_manager",
     "//components/spellcheck/browser",
     "//components/ssl_errors",
-    "//components/startup_metric_utils/browser",
     "//components/strings",
-    "//components/subresource_filter/content/browser",
-    "//components/subresource_filter/core/browser",
     "//components/sync",
     "//components/sync/driver:resources",
     "//components/sync_preferences",
@@ -628,8 +603,6 @@ static_library("ui") {
   if (!is_fuchsia) {
     # TODO(crbug.com/753619): Enable crash reporting on Fuchsia.
     deps += [
-      "//components/crash/core/app",
-      "//components/crash/core/browser",
     ]
   }
 
@@ -743,8 +716,6 @@ static_library("ui") {
       "android/passwords/password_generation_dialog_view_android.h",
       "android/passwords/password_generation_editing_popup_view_android.cc",
       "android/passwords/password_generation_editing_popup_view_android.h",
-      "android/safe_browsing/password_reuse_dialog_view_android.cc",
-      "android/safe_browsing/password_reuse_dialog_view_android.h",
       "android/simple_message_box_android.cc",
       "android/ssl_client_certificate_selector.cc",
       "android/status_tray_android.cc",
@@ -816,7 +787,6 @@ static_library("ui") {
       "//components/optimization_guide/proto:optimization_guide_proto",
       "//components/page_info",
       "//components/page_info/android:android",
-      "//components/subresource_filter/core/browser",
       "//components/thin_webview:thin_webview",
       "//components/translate/content/android",
       "//crypto:platform",
@@ -1435,8 +1405,6 @@ static_library("ui") {
       "webui/settings/recent_site_settings_helper.h",
       "webui/settings/reset_settings_handler.cc",
       "webui/settings/reset_settings_handler.h",
-      "webui/settings/safety_check_handler.cc",
-      "webui/settings/safety_check_handler.h",
       "webui/settings/search_engines_handler.cc",
       "webui/settings/search_engines_handler.h",
       "webui/settings/settings_clear_browsing_data_handler.cc",
@@ -1513,7 +1481,6 @@ static_library("ui") {
       "//chrome/browser/promo_browser_command:mojo_bindings",
       "//chrome/browser/resource_coordinator:tab_metrics_event_proto",
       "//chrome/browser/resource_coordinator/tab_ranker",
-      "//chrome/browser/safe_browsing:advanced_protection",
       "//chrome/browser/ui/color:color_headers",
       "//chrome/browser/ui/color:mixers",
       "//chrome/browser/ui/webui/app_management:mojo_bindings",
@@ -1530,7 +1497,6 @@ static_library("ui") {
       "//components/performance_manager:site_data_proto",
       "//components/printing/browser",
       "//components/profile_metrics",
-      "//components/safety_check",
       "//components/search_provider_logos",
       "//components/services/app_service/public/cpp:app_update",
       "//components/services/app_service/public/mojom",
@@ -2561,7 +2527,6 @@ static_library("ui") {
       "//chrome/browser/ui/webui/nearby_share/public/mojom",
       "//chrome/browser/ui/webui/settings/chromeos/constants:mojom",
       "//chrome/browser/ui/webui/settings/chromeos/search:mojo_bindings",
-      "//chrome/services/file_util/public/cpp",
       "//chromeos",
       "//chromeos/assistant:buildflags",
       "//chromeos/audio",
@@ -3055,10 +3020,6 @@ static_library("ui") {
       "views/apps/glass_app_window_frame_view_win.cc",
       "views/apps/glass_app_window_frame_view_win.h",
       "views/certificate_viewer_win.cc",
-      "views/chrome_cleaner_dialog_win.cc",
-      "views/chrome_cleaner_dialog_win.h",
-      "views/chrome_cleaner_reboot_dialog_win.cc",
-      "views/chrome_cleaner_reboot_dialog_win.h",
       "views/color_chooser_dialog.cc",
       "views/color_chooser_dialog.h",
       "views/color_chooser_win.cc",
@@ -3081,8 +3042,6 @@ static_library("ui") {
       "views/frame/windows_10_caption_button.cc",
       "views/frame/windows_10_caption_button.h",
       "views/network_profile_bubble_view.cc",
-      "views/settings_reset_prompt_dialog.cc",
-      "views/settings_reset_prompt_dialog.h",
       "views/status_icons/status_icon_win.cc",
       "views/status_icons/status_icon_win.h",
       "views/status_icons/status_tray_state_changer_win.cc",
@@ -3103,8 +3062,6 @@ static_library("ui") {
       "webui/conflicts/conflicts_handler.h",
       "webui/conflicts/conflicts_ui.cc",
       "webui/conflicts/conflicts_ui.h",
-      "webui/settings/chrome_cleanup_handler_win.cc",
-      "webui/settings/chrome_cleanup_handler_win.h",
       "webui/settings_utils_win.cc",
       "webui/version_handler_win.cc",
       "webui/version_handler_win.h",
@@ -3116,7 +3073,6 @@ static_library("ui") {
       "//ui/views/controls/webview",
     ]
     deps += [
-      "//chrome/browser/safe_browsing/chrome_cleaner:public",
       "//chrome/browser/ui/startup:buildflags",
       "//chrome/browser/win/conflicts:module_info",
       "//chrome/credential_provider/common:common_constants",
@@ -3428,7 +3384,6 @@ static_library("ui") {
       "views/device_chooser_content_view.h",
       "views/devtools_process_observer.cc",
       "views/devtools_process_observer.h",
-      "views/download/download_danger_prompt_views.cc",
       "views/download/download_in_progress_dialog_view.cc",
       "views/download/download_in_progress_dialog_view.h",
       "views/download/download_item_view.cc",
@@ -3835,24 +3790,8 @@ static_library("ui") {
       "views/read_later/read_later_button.h",
       "views/reader_mode/reader_mode_icon_view.cc",
       "views/reader_mode/reader_mode_icon_view.h",
-      "views/relaunch_notification/relaunch_notification_controller.cc",
-      "views/relaunch_notification/relaunch_notification_controller.h",
-      "views/relaunch_notification/relaunch_notification_metrics.cc",
-      "views/relaunch_notification/relaunch_notification_metrics.h",
-      "views/relaunch_notification/relaunch_required_timer.cc",
-      "views/relaunch_notification/relaunch_required_timer.h",
-      "views/relaunch_notification/relaunch_required_timer_internal.cc",
-      "views/relaunch_notification/relaunch_required_timer_internal.h",
       "views/sad_tab_view.cc",
       "views/sad_tab_view.h",
-      "views/safe_browsing/deep_scanning_failure_modal_dialog.cc",
-      "views/safe_browsing/deep_scanning_failure_modal_dialog.h",
-      "views/safe_browsing/deep_scanning_modal_dialog.cc",
-      "views/safe_browsing/deep_scanning_modal_dialog.h",
-      "views/safe_browsing/password_reuse_modal_warning_dialog.cc",
-      "views/safe_browsing/password_reuse_modal_warning_dialog.h",
-      "views/safe_browsing/prompt_for_scanning_modal_dialog.cc",
-      "views/safe_browsing/prompt_for_scanning_modal_dialog.h",
       "views/send_tab_to_self/send_tab_to_self_bubble_device_button.cc",
       "views/send_tab_to_self/send_tab_to_self_bubble_device_button.h",
       "views/send_tab_to_self/send_tab_to_self_bubble_view_impl.cc",
@@ -3887,8 +3826,8 @@ static_library("ui") {
       "views/tab_search/tab_search_bubble_view.h",
       "views/tab_sharing/tab_sharing_ui_views.cc",
       "views/tab_sharing/tab_sharing_ui_views.h",
-      "views/tabs/alert_indicator.cc",
-      "views/tabs/alert_indicator.h",
+      "views/tabs/alert_indicator_button.cc",
+      "views/tabs/alert_indicator_button.h",
       "views/tabs/browser_tab_strip_controller.cc",
       "views/tabs/browser_tab_strip_controller.h",
       "views/tabs/color_picker_view.cc",
@@ -4158,8 +4097,6 @@ static_library("ui") {
         "views/frame/opaque_browser_frame_view_layout_delegate.h",
         "views/frame/opaque_browser_frame_view_platform_specific.cc",
         "views/frame/opaque_browser_frame_view_platform_specific.h",
-        "views/outdated_upgrade_bubble_view.cc",
-        "views/outdated_upgrade_bubble_view.h",
         "views/passwords/password_sign_in_promo_view.cc",
         "views/passwords/password_sign_in_promo_view.h",
         "views/policy/enterprise_startup_dialog_view.cc",
@@ -4446,15 +4383,6 @@ static_library("ui") {
     }
   }
 
-  if (safe_browsing_mode == 1) {
-    deps += [
-      "//chrome/browser/ui/webui/reset_password:mojo_bindings",
-      "//chrome/common/safe_browsing:proto",
-      "//components/safe_browsing/content/password_protection:password_protection_metrics_util",
-      "//components/safe_browsing/core:csd_proto",
-    ]
-  }
-
   if (use_dbus) {
     deps += [ "//dbus" ]
   }
