diff --git a/chrome/browser/content_index/content_index_provider_impl.cc b/chrome/browser/content_index/content_index_provider_impl.cc
index 29ec5c99fc6e..d497fc34db16 100644
--- a/chrome/browser/content_index/content_index_provider_impl.cc
+++ b/chrome/browser/content_index/content_index_provider_impl.cc
@@ -12,7 +12,6 @@
 #include "base/task/post_task.h"
 #include "build/build_config.h"
 #include "chrome/browser/engagement/site_engagement_service_factory.h"
-#include "chrome/browser/metrics/ukm_background_recorder_service.h"
 #include "chrome/browser/offline_items_collection/offline_content_aggregator_factory.h"
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/profiles/profile_key.h"
@@ -102,7 +101,6 @@ OfflineItemFilter CategoryToFilter(blink::mojom::ContentCategory category) {
 
 ContentIndexProviderImpl::ContentIndexProviderImpl(Profile* profile)
     : profile_(profile),
-      metrics_(ukm::UkmBackgroundRecorderFactory::GetForProfile(profile)),
       aggregator_(
           OfflineContentAggregatorFactory::GetForKey(profile->GetProfileKey())),
       site_engagement_service_(
@@ -149,9 +147,6 @@ void ContentIndexProviderImpl::OnContentAdded(
 
   for (auto& observer : observers_)
     observer.OnItemsAdded(items);
-
-  metrics_.RecordContentAdded(url::Origin::Create(entry.launch_url.GetOrigin()),
-                              entry.description->category);
 }
 
 void ContentIndexProviderImpl::OnContentDeleted(
@@ -208,7 +203,6 @@ void ContentIndexProviderImpl::DidGetEntryToOpen(
 
 void ContentIndexProviderImpl::DidOpenTab(content::ContentIndexEntry entry,
                                           content::WebContents* web_contents) {
-  metrics_.RecordContentOpened(web_contents, entry.description->category);
 }
 
 void ContentIndexProviderImpl::RemoveItem(const ContentId& id) {
@@ -220,8 +214,6 @@ void ContentIndexProviderImpl::RemoveItem(const ContentId& id) {
   if (!storage_partition || !storage_partition->GetContentIndexContext())
     return;
 
-  metrics_.RecordContentDeletedByUser(components.origin);
-
   storage_partition->GetContentIndexContext()->OnUserDeletedItem(
       components.service_worker_registration_id, components.origin,
       components.description_id);
@@ -309,7 +301,6 @@ void ContentIndexProviderImpl::GetAllItems(MultipleItemCallback callback) {
 void ContentIndexProviderImpl::DidGetAllEntriesAcrossStorageParitions(
     std::unique_ptr<OfflineItemList> item_list,
     MultipleItemCallback callback) {
-  ContentIndexMetrics::RecordContentIndexEntries(item_list->size());
   std::move(callback).Run(*item_list);
 }
 
