diff --git a/chrome/browser/resources/new_tab_page/BUILD.gn b/chrome/browser/resources/new_tab_page/BUILD.gn
index 17570fbe30ff8..a983024af8385 100644
--- a/chrome/browser/resources/new_tab_page/BUILD.gn
+++ b/chrome/browser/resources/new_tab_page/BUILD.gn
@@ -59,7 +59,6 @@ js_library("new_tab_page") {
 js_library("lazy_load") {
   deps = [
     ":customize_dialog",
-    ":middle_slot_promo",
     ":most_visited",
     ":voice_search_overlay",
     "modules/dummy:module",
