diff --git a/chrome/browser/ui/passwords/bubble_controllers/save_unsynced_credentials_locally_bubble_controller.h b/chrome/browser/ui/passwords/bubble_controllers/save_unsynced_credentials_locally_bubble_controller.h
index 4f02b00fe54e..d2f1a8f22462 100644
--- a/chrome/browser/ui/passwords/bubble_controllers/save_unsynced_credentials_locally_bubble_controller.h
+++ b/chrome/browser/ui/passwords/bubble_controllers/save_unsynced_credentials_locally_bubble_controller.h
@@ -44,7 +44,6 @@ class SaveUnsyncedCredentialsLocallyBubbleController
   base::string16 GetTitle() const override;
   void ReportInteractions() override;
 
-  password_manager::metrics_util::UIDismissalReason dismissal_reason_;
   std::vector<password_manager::PasswordForm> unsynced_credentials_;
 };
 
