diff --git a/chrome/browser/browser_switcher/bho/browser_switcher_core.cc b/chrome/browser/browser_switcher/bho/browser_switcher_core.cc
index aad2117e3b6c7..00f8c65dc4d97 100644
--- a/chrome/browser/browser_switcher/bho/browser_switcher_core.cc
+++ b/chrome/browser/browser_switcher/bho/browser_switcher_core.cc
@@ -20,7 +20,7 @@
 namespace {
 
 const wchar_t kChromeKey[] =
-    L"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\App Paths\\chrome.exe";
+    L"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\App Paths\\dumb.exe";
 
 const wchar_t kChromeVarName[] = L"${chrome}";
 const wchar_t kUrlVarName[] = L"${url}";
