diff --git a/chrome/browser/mac/mac_startup_profiler.cc b/chrome/browser/mac/mac_startup_profiler.cc
index 2ae7fe610155..341da9a2513d 100644
--- a/chrome/browser/mac/mac_startup_profiler.cc
+++ b/chrome/browser/mac/mac_startup_profiler.cc
@@ -24,6 +24,7 @@ void MacStartupProfiler::Profile(Location location) {
 }
 
 void MacStartupProfiler::RecordMetrics() {
+#if 0
   const base::TimeTicks main_entry_ticks =
       startup_metric_utils::MainEntryPointTicks();
   DCHECK(!main_entry_ticks.is_null());
@@ -34,6 +35,7 @@ void MacStartupProfiler::RecordMetrics() {
   for (const std::pair<const Location, base::TimeTicks>& entry :
        profiled_ticks_)
     RecordHistogram(entry.first, entry.second - main_entry_ticks);
+#endif
 }
 
 const std::string MacStartupProfiler::HistogramName(Location location) {
