diff --git a/chrome/browser/password_manager/password_store_factory.cc b/chrome/browser/password_manager/password_store_factory.cc
index c44781ca7607..0d0325d27f61 100644
--- a/chrome/browser/password_manager/password_store_factory.cc
+++ b/chrome/browser/password_manager/password_store_factory.cc
@@ -39,9 +39,7 @@
 #include "chrome/browser/password_manager/password_manager_util_win.h"
 #endif
 
-#if defined(PASSWORD_REUSE_DETECTION_ENABLED)
 #include "chrome/browser/password_manager/password_store_signin_notifier_impl.h"
-#endif
 
 using password_manager::PasswordStore;
 
