diff --git a/chrome/common/logging_chrome.cc b/chrome/common/logging_chrome.cc
index 85c2273a345c..8bc327b14b58 100644
--- a/chrome/common/logging_chrome.cc
+++ b/chrome/common/logging_chrome.cc
@@ -62,7 +62,6 @@
 #include <initguid.h>
 #include "base/logging_win.h"
 #include "base/syslog_logging.h"
-#include "chrome/common/win/eventlog_messages.h"
 #include "chrome/install_static/install_details.h"
 #endif
 
