diff --git a/chrome/browser/extensions/api/tabs/tabs_constants.h b/chrome/browser/extensions/api/tabs/tabs_constants.h
index 2be27625a2356..2f03e9cd8a571 100644
--- a/chrome/browser/extensions/api/tabs/tabs_constants.h
+++ b/chrome/browser/extensions/api/tabs/tabs_constants.h
@@ -10,6 +10,7 @@
 namespace extensions {
 namespace tabs_constants {
 
+extern const char kCannotUpdateMuteDisabled[];
 // Keys used in serializing tab data & events.
 extern const char kActiveKey[];
 extern const char kAllFramesKey[];
