diff --git a/chrome/browser/background_sync/background_sync_delegate_impl.h b/chrome/browser/background_sync/background_sync_delegate_impl.h
index 8159bd2f72d0..40c2933b0db3 100644
--- a/chrome/browser/background_sync/background_sync_delegate_impl.h
+++ b/chrome/browser/background_sync/background_sync_delegate_impl.h
@@ -22,10 +22,6 @@ class HostContentSettingsMap;
 class ScopedKeepAlive;
 class ScopedProfileKeepAlive;
 
-namespace ukm {
-class UkmBackgroundRecorderService;
-}  // namespace ukm
-
 // Chrome's customization of the logic in components/background_sync
 class BackgroundSyncDelegateImpl
     : public background_sync::BackgroundSyncDelegate,
@@ -59,9 +55,6 @@ class BackgroundSyncDelegateImpl
   CreateBackgroundSyncEventKeepAlive() override;
 #endif
 
-  void GetUkmSourceId(const url::Origin& origin,
-                      base::OnceCallback<void(base::Optional<ukm::SourceId>)>
-                          callback) override;
   void Shutdown() override;
   HostContentSettingsMap* GetHostContentSettingsMap() override;
   bool IsProfileOffTheRecord() override;
@@ -87,7 +80,6 @@ class BackgroundSyncDelegateImpl
  private:
   Profile* profile_;
   bool off_the_record_;
-  ukm::UkmBackgroundRecorderService* ukm_background_service_;
   // Same lifetime as |profile_|.
   site_engagement::SiteEngagementService* site_engagement_service_;
   std::set<url::Origin> suspended_periodic_sync_origins_;
