diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 0976a68da300b..88fdec91ec9da 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -65,13 +65,6 @@ static_library("utility") {
     "//ui/base:buildflags",
   ]
 
-  if (is_win) {
-    deps += [
-      "//components/services/quarantine",
-      "//components/services/quarantine/public/mojom",
-    ]
-  }
-
   if (is_mac) {
     deps += [ "//chrome/services/mac_notifications" ]
   }
