diff --git a/chrome/browser/ui/passwords/settings/password_manager_presenter.cc b/chrome/browser/ui/passwords/settings/password_manager_presenter.cc
index dbb67b728261a..df8631df3b190 100644
--- a/chrome/browser/ui/passwords/settings/password_manager_presenter.cc
+++ b/chrome/browser/ui/passwords/settings/password_manager_presenter.cc
@@ -38,7 +38,6 @@
 #include "components/password_manager/core/browser/form_fetcher_impl.h"
 #include "components/password_manager/core/browser/password_feature_manager.h"
 #include "components/password_manager/core/browser/password_form.h"
-#include "components/password_manager/core/browser/password_form_metrics_recorder.h"
 #include "components/password_manager/core/browser/password_list_sorter.h"
 #include "components/password_manager/core/browser/password_manager_client.h"
 #include "components/password_manager/core/browser/password_manager_metrics_util.h"
@@ -225,7 +224,7 @@ void PasswordManagerPresenter::MovePasswordToAccountStoreHelper::
   auto save_manager =
       password_manager::PasswordSaveManagerImpl::CreatePasswordSaveManagerImpl(
           client_);
-  save_manager->Init(client_, form_fetcher_.get(), /*metrics_recorder=*/nullptr,
+  save_manager->Init(client_, form_fetcher_.get(),
                      /*votes_uploader=*/nullptr);
   save_manager->CreatePendingCredentials(form_, {}, {}, /*is_http_auth=*/false,
                                          /*is_credential_api_save=*/false);
