diff --git a/chrome/browser/ui/webui/signin/profile_picker_handler.cc b/chrome/browser/ui/webui/signin/profile_picker_handler.cc
index 44585a46709d..a38ab6b6af71 100644
--- a/chrome/browser/ui/webui/signin/profile_picker_handler.cc
+++ b/chrome/browser/ui/webui/signin/profile_picker_handler.cc
@@ -329,7 +329,6 @@ void ProfilePickerHandler::HandleCreateProfile(const base::ListValue* args) {
   DCHECK(profiles::IsDefaultAvatarIconUrl(avatar_url, &icon_index));
 #endif
 
-  ProfileMetrics::LogProfileAddNewUser(ProfileMetrics::ADD_NEW_PROFILE_PICKER);
   ProfileManager::CreateMultiProfileAsync(
       profile_name, avatar_url,
       base::BindRepeating(&ProfilePickerHandler::OnProfileCreated,
@@ -418,8 +417,7 @@ void ProfilePickerHandler::HandleRemoveProfile(const base::ListValue* args) {
     return;
   }
   RecordProfilePickerAction(ProfilePickerAction::kDeleteProfile);
-  webui::DeleteProfileAtPath(*profile_path,
-                             ProfileMetrics::DELETE_PROFILE_USER_MANAGER);
+  webui::DeleteProfileAtPath(*profile_path);
 }
 
 void ProfilePickerHandler::HandleGetProfileStatistics(
