diff --git a/chrome/browser/password_manager/chrome_password_manager_client.h b/chrome/browser/password_manager/chrome_password_manager_client.h
index bc46fd7c8c305..df9713519a73b 100644
--- a/chrome/browser/password_manager/chrome_password_manager_client.h
+++ b/chrome/browser/password_manager/chrome_password_manager_client.h
@@ -29,8 +29,6 @@
 #include "components/password_manager/core/browser/password_manager.h"
 #include "components/password_manager/core/browser/password_manager_client.h"
 #include "components/password_manager/core/browser/password_manager_client_helper.h"
-#include "components/password_manager/core/browser/password_manager_metrics_recorder.h"
-#include "components/password_manager/core/browser/password_manager_metrics_util.h"
 #include "components/password_manager/core/browser/password_reuse_detection_manager.h"
 #include "components/password_manager/core/browser/password_reuse_detector.h"
 #include "components/prefs/pref_member.h"
@@ -191,9 +189,6 @@ class ChromePasswordManagerClient
   void AnnotateNavigationEntry(bool has_password_field) override;
   autofill::LanguageCode GetPageLanguage() const override;
 
-  safe_browsing::PasswordProtectionService* GetPasswordProtectionService()
-      const override;
-
 #if defined(ON_FOCUS_PING_ENABLED)
   void CheckSafeBrowsingReputation(const GURL& form_action,
                                    const GURL& frame_url) override;
@@ -209,8 +204,6 @@ class ChromePasswordManagerClient
   void LogPasswordReuseDetectedEvent() override;
 
   ukm::SourceId GetUkmSourceId() override;
-  password_manager::PasswordManagerMetricsRecorder* GetMetricsRecorder()
-      override;
   password_manager::PasswordRequirementsService*
   GetPasswordRequirementsService() override;
   favicon::FaviconService* GetFaviconService() override;
@@ -343,8 +336,6 @@ class ChromePasswordManagerClient
   password_manager::PasswordManager password_manager_;
   password_manager::PasswordFeatureManagerImpl password_feature_manager_;
   password_manager::HttpAuthManagerImpl httpauth_manager_;
-  password_manager::PasswordReuseDetectionManager
-      password_reuse_detection_manager_;
 
 #if defined(OS_ANDROID)
   // Holds and facilitates a credential store for each origin in this tab.
@@ -402,11 +393,13 @@ class ChromePasswordManagerClient
 
   std::unique_ptr<autofill::LogManager> log_manager_;
 
+#if 0
   // Recorder of metrics that is associated with the last committed navigation
   // of the WebContents owning this ChromePasswordManagerClient. May be unset at
   // times. Sends statistics on destruction.
   absl::optional<password_manager::PasswordManagerMetricsRecorder>
       metrics_recorder_;
+#endif
 
   // Whether navigator.credentials.store() was ever called from this
   // WebContents. Used for testing.
