diff --git a/chrome/services/util_win/public/mojom/BUILD.gn b/chrome/services/util_win/public/mojom/BUILD.gn
index 3afefe242607..6c0d82fa46b9 100644
--- a/chrome/services/util_win/public/mojom/BUILD.gn
+++ b/chrome/services/util_win/public/mojom/BUILD.gn
@@ -45,7 +45,6 @@ mojom("mojom") {
       traits_public_deps = [
         "//base",
         "//chrome/browser/win/conflicts:module_info",
-        "//third_party/metrics_proto",
         "//ui/shell_dialogs",
       ]
     },
