diff --git a/chrome/browser/extensions/api/BUILD.gn b/chrome/browser/extensions/api/BUILD.gn
index bbb5185852056..b00e066e3b1d2 100644
--- a/chrome/browser/extensions/api/BUILD.gn
+++ b/chrome/browser/extensions/api/BUILD.gn
@@ -42,7 +42,6 @@ function_registration("api_registration") {
     "//chrome/common/extensions/api",
     "//components/sync",
     "//skia",
-    "//third_party/metrics_proto",
     "//ui/accessibility:ax_enums_mojo",
   ]
 }
