diff --git a/chrome/browser/download/android/download_controller.h b/chrome/browser/download/android/download_controller.h
index 88c53c269dfd..2d3b25c6c249 100644
--- a/chrome/browser/download/android/download_controller.h
+++ b/chrome/browser/download/android/download_controller.h
@@ -52,7 +52,6 @@ class DownloadController : public DownloadControllerBase {
     STORAGE_PERMISSION_NO_WEB_CONTENTS,
     STORAGE_PERMISSION_MAX
   };
-  static void RecordStoragePermission(StoragePermissionType type);
 
   static void CloseTabIfEmpty(content::WebContents* web_contents);
 
