diff --git a/chrome/browser/policy/chrome_browser_cloud_management_controller_desktop.h b/chrome/browser/policy/chrome_browser_cloud_management_controller_desktop.h
index 3645b616790dc..4ebd8e6ea16c0 100644
--- a/chrome/browser/policy/chrome_browser_cloud_management_controller_desktop.h
+++ b/chrome/browser/policy/chrome_browser_cloud_management_controller_desktop.h
@@ -63,8 +63,6 @@ class ChromeBrowserCloudManagementControllerDesktop
   DeviceManagementService* GetDeviceManagementService() override;
   scoped_refptr<network::SharedURLLoaderFactory> GetSharedURLLoaderFactory()
       override;
-  std::unique_ptr<enterprise_reporting::ReportScheduler> CreateReportScheduler(
-      CloudPolicyClient* client) override;
 
   scoped_refptr<base::SingleThreadTaskRunner> GetBestEffortTaskRunner()
       override;
@@ -80,9 +78,6 @@ class ChromeBrowserCloudManagementControllerDesktop
   bool IsInvalidationsServiceStarted() const override;
 
  private:
-  enterprise_reporting::ReportingDelegateFactoryDesktop
-      reporting_delegate_factory_;
-
   std::unique_ptr<ChromeBrowserCloudManagementRegisterWatcher>
       cloud_management_register_watcher_;
 
