diff --git a/chrome/browser/chrome_browser_main_mac.mm b/chrome/browser/chrome_browser_main_mac.mm
index a006f94f3e08..e98dd4e92289 100644
--- a/chrome/browser/chrome_browser_main_mac.mm
+++ b/chrome/browser/chrome_browser_main_mac.mm
@@ -334,9 +334,6 @@ void RecordBetaAndDevStats() {
       MacStartupProfiler::POST_PROFILE_INIT);
   ChromeBrowserMainPartsPosix::PostProfileInit();
 
-  g_browser_process->metrics_service()->RecordBreakpadRegistration(
-      crash_reporter::GetUploadsEnabled());
-
   // Activation of Keystone is not automatic but done in response to the
   // counting and reporting of profiles.
   KeystoneGlue* glue = [KeystoneGlue defaultKeystoneGlue];
