diff --git a/chrome/browser/extensions/api/chrome_extensions_api_client.h b/chrome/browser/extensions/api/chrome_extensions_api_client.h
index 10e9c251d0281..b6fc4f9e68f2e 100644
--- a/chrome/browser/extensions/api/chrome_extensions_api_client.h
+++ b/chrome/browser/extensions/api/chrome_extensions_api_client.h
@@ -80,7 +80,6 @@ class ChromeExtensionsAPIClient : public ExtensionsAPIClient {
   NetworkingCastPrivateDelegate* GetNetworkingCastPrivateDelegate() override;
   FileSystemDelegate* GetFileSystemDelegate() override;
   MessagingDelegate* GetMessagingDelegate() override;
-  FeedbackPrivateDelegate* GetFeedbackPrivateDelegate() override;
 
 #if BUILDFLAG(IS_CHROMEOS_ASH)
   MediaPerceptionAPIDelegate* GetMediaPerceptionAPIDelegate() override;
@@ -105,7 +104,6 @@ class ChromeExtensionsAPIClient : public ExtensionsAPIClient {
       networking_cast_private_delegate_;
   std::unique_ptr<FileSystemDelegate> file_system_delegate_;
   std::unique_ptr<MessagingDelegate> messaging_delegate_;
-  std::unique_ptr<FeedbackPrivateDelegate> feedback_private_delegate_;
 
 #if BUILDFLAG(IS_CHROMEOS_ASH)
   std::unique_ptr<MediaPerceptionAPIDelegate> media_perception_api_delegate_;
