diff --git a/chrome/browser/mac/mac_startup_profiler.cc b/chrome/browser/mac/mac_startup_profiler.cc
index 2ae7fe610155..3205491ea4fc 100644
--- a/chrome/browser/mac/mac_startup_profiler.cc
+++ b/chrome/browser/mac/mac_startup_profiler.cc
@@ -6,7 +6,6 @@
 
 #include "base/check.h"
 #include "base/metrics/histogram_macros.h"
-#include "components/startup_metric_utils/browser/startup_metric_utils.h"
 
 // static
 MacStartupProfiler* MacStartupProfiler::GetInstance() {
@@ -24,16 +23,6 @@ void MacStartupProfiler::Profile(Location location) {
 }
 
 void MacStartupProfiler::RecordMetrics() {
-  const base::TimeTicks main_entry_ticks =
-      startup_metric_utils::MainEntryPointTicks();
-  DCHECK(!main_entry_ticks.is_null());
-  DCHECK(!recorded_metrics_);
-
-  recorded_metrics_ = true;
-
-  for (const std::pair<const Location, base::TimeTicks>& entry :
-       profiled_ticks_)
-    RecordHistogram(entry.first, entry.second - main_entry_ticks);
 }
 
 const std::string MacStartupProfiler::HistogramName(Location location) {
