diff --git a/chrome/browser/extensions/extension_service.h b/chrome/browser/extensions/extension_service.h
index dee8ffd694ae7..eedab19313454 100644
--- a/chrome/browser/extensions/extension_service.h
+++ b/chrome/browser/extensions/extension_service.h
@@ -23,7 +23,6 @@
 #include "chrome/browser/extensions/blocklist.h"
 #include "chrome/browser/extensions/extension_allowlist.h"
 #include "chrome/browser/extensions/extension_management.h"
-#include "chrome/browser/extensions/forced_extensions/force_installed_metrics.h"
 #include "chrome/browser/extensions/forced_extensions/force_installed_tracker.h"
 #include "chrome/browser/extensions/install_gate.h"
 #include "chrome/browser/extensions/pending_extension_manager.h"
@@ -629,8 +628,6 @@ class ExtensionService : public ExtensionServiceInterface,
 
   ExtensionAllowlist allowlist_;
 
-  SafeBrowsingVerdictHandler safe_browsing_verdict_handler_;
-
   // Sets of enabled/disabled/terminated/blocklisted extensions. Not owned.
   ExtensionRegistry* registry_ = nullptr;
 
@@ -716,8 +713,10 @@ class ExtensionService : public ExtensionServiceInterface,
   // Tracker of enterprise policy forced installation.
   ForceInstalledTracker force_installed_tracker_;
 
+#if 0
   // Reports force-installed extension metrics to UMA.
   ForceInstalledMetrics force_installed_metrics_;
+#endif
 
   base::ScopedObservation<ProfileManager, ProfileManagerObserver>
       profile_manager_observation_{this};
@@ -751,14 +750,6 @@ class ExtensionService : public ExtensionServiceInterface,
                            ExtensionsNotAllowlistedThenBlocklisted);
   FRIEND_TEST_ALL_PREFIXES(ExtensionAllowlistUnitTest,
                            ExtensionsBlocklistedThenNotAllowlisted);
-  FRIEND_TEST_ALL_PREFIXES(SafeBrowsingVerdictHandlerUnitTest,
-                           GreylistedExtensionDisabled);
-  FRIEND_TEST_ALL_PREFIXES(SafeBrowsingVerdictHandlerUnitTest,
-                           GreylistDontEnableManuallyDisabled);
-  FRIEND_TEST_ALL_PREFIXES(SafeBrowsingVerdictHandlerUnitTest,
-                           GreylistUnknownDontChange);
-  FRIEND_TEST_ALL_PREFIXES(SafeBrowsingVerdictHandlerUnitTest,
-                           UnblocklistedExtensionStillGreylisted);
   friend class ::BlocklistedExtensionSyncServiceTest;
 
   DISALLOW_COPY_AND_ASSIGN(ExtensionService);
