diff --git a/chrome/browser/flag_descriptions.h b/chrome/browser/flag_descriptions.h
index 3479dc665a2f..9295025b7e3d 100644
--- a/chrome/browser/flag_descriptions.h
+++ b/chrome/browser/flag_descriptions.h
@@ -1354,6 +1354,9 @@ extern const char kStylusBatteryStatusDescription[];
 extern const char kSystemKeyboardLockName[];
 extern const char kSystemKeyboardLockDescription[];
 
+extern const char kTabAudioMutingName[];
+extern const char kTabAudioMutingDescription[];
+
 extern const char kSuggestionsWithSubStringMatchName[];
 extern const char kSuggestionsWithSubStringMatchDescription[];
 
