diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
index 38c40cd243dc..8b39a51031e4 100644
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -340,8 +340,6 @@ android_library("chrome_java") {
     "//chrome/browser/privacy:java",
     "//chrome/browser/privacy_sandbox/android:java",
     "//chrome/browser/profiles/android:java",
-    "//chrome/browser/safe_browsing/android:java",
-    "//chrome/browser/safety_check/android:java",
     "//chrome/browser/settings:java",
     "//chrome/browser/share:java",
     "//chrome/browser/share/android:java_resources",
@@ -403,7 +401,6 @@ android_library("chrome_java") {
     "//components/content_settings/android:content_settings_enums_java",
     "//components/content_settings/android:java",
     "//components/contextual_search/content/common/mojom:mojom_java",
-    "//components/crash/android:java",
     "//components/dom_distiller/content/browser/android:dom_distiller_content_java",
     "//components/dom_distiller/core/android:dom_distiller_core_java",
     "//components/dom_distiller/core/mojom:mojom_java",
@@ -455,7 +452,6 @@ android_library("chrome_java") {
     "//components/prefs/android:java",
     "//components/profile_metrics:browser_profile_type_enum_java",
     "//components/query_tiles:java",
-    "//components/safe_browsing/android:safe_browsing_java",
     "//components/schema_org/common:mojom_java",
     "//components/search_engines/android:java",
     "//components/security_interstitials/content/android:java",
@@ -862,8 +858,6 @@ junit_binary("chrome_junit_tests") {
     "//chrome/browser/preferences:java",
     "//chrome/browser/preferences:preferences_junit_tests",
     "//chrome/browser/profiles/android:java",
-    "//chrome/browser/safety_check/android:java",
-    "//chrome/browser/safety_check/android:junit",
     "//chrome/browser/share:java",
     "//chrome/browser/signin/services/android:java",
     "//chrome/browser/signin/services/android:junit",
@@ -1128,10 +1122,6 @@ android_library("chrome_test_java") {
     "//chrome/browser/privacy_sandbox/android:java",
     "//chrome/browser/privacy_sandbox/android:javatests",
     "//chrome/browser/profiles/android:java",
-    "//chrome/browser/safe_browsing/android:java",
-    "//chrome/browser/safe_browsing/android:javatests",
-    "//chrome/browser/safety_check/android:java",
-    "//chrome/browser/safety_check/android:javatests",
     "//chrome/browser/settings:java",
     "//chrome/browser/settings:javatests",
     "//chrome/browser/settings:test_support_java",
@@ -1195,7 +1185,6 @@ android_library("chrome_test_java") {
     "//components/browser_ui/widget/android:test_support_java",
     "//components/content_settings/android:content_settings_enums_java",
     "//components/content_settings/android:java",
-    "//components/crash/android:java",
     "//components/dom_distiller/core/android:dom_distiller_core_java",
     "//components/dom_distiller/core/mojom:mojom_java",
     "//components/download/internal/background_service:internal_java",
@@ -1250,7 +1239,6 @@ android_library("chrome_test_java") {
     "//components/profile_metrics:browser_profile_type_enum_java",
     "//components/query_tiles:java",
     "//components/query_tiles:test_support_java",
-    "//components/safe_browsing/android:safe_browsing_java",
     "//components/schema_org/common:mojom_java",
     "//components/search_engines/android:java",
     "//components/security_interstitials/content/android:java",
@@ -1679,7 +1667,6 @@ chrome_common_shared_library("libchromefortest") {
     "//chrome/browser/password_manager/android_test_helpers:android_wrappers",
     "//chrome/browser/subresource_filter:android_test_support",
     "//components/autofill_assistant/browser:test_support",
-    "//components/crash/android:crash_android",
     "//components/minidump_uploader",
     "//components/paint_preview/player/android:test_support",
     "//components/sync",
@@ -1941,7 +1928,6 @@ template("libmonochrome_apk_or_bundle_tmpl") {
     deps = [
       "//android_webview/lib",
       "//android_webview/nonembedded",
-      "//components/crash/android:crashpad_main",
     ]
     if (defined(invoker.deps)) {
       deps += invoker.deps
