diff --git a/chrome/browser/touch_to_fill/touch_to_fill_controller.h b/chrome/browser/touch_to_fill/touch_to_fill_controller.h
index 723988a2fe3f5..d80bdf9aabade 100644
--- a/chrome/browser/touch_to_fill/touch_to_fill_controller.h
+++ b/chrome/browser/touch_to_fill/touch_to_fill_controller.h
@@ -16,7 +16,6 @@
 #include "chrome/browser/touch_to_fill/touch_to_fill_view.h"
 #include "chrome/browser/touch_to_fill/touch_to_fill_view_factory.h"
 #include "components/password_manager/core/browser/biometric_authenticator.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "ui/gfx/native_widget_types.h"
 
 namespace password_manager {
@@ -98,8 +97,6 @@ class TouchToFillController {
   // Authenticator used to trigger a biometric auth before filling.
   scoped_refptr<password_manager::BiometricAuthenticator> authenticator_;
 
-  ukm::SourceId source_id_ = ukm::kInvalidSourceId;
-
   // View used to communicate with the Android frontend. Lazily instantiated so
   // that it can be injected by tests.
   std::unique_ptr<TouchToFillView> view_;
