diff --git a/chrome/browser/ui/webui/system_info_ui.cc b/chrome/browser/ui/webui/system_info_ui.cc
index 2aaa185db2dc9..d2118c0d2cce5 100644
--- a/chrome/browser/ui/webui/system_info_ui.cc
+++ b/chrome/browser/ui/webui/system_info_ui.cc
@@ -20,14 +20,12 @@
 #include "base/time/time.h"
 #include "base/values.h"
 #include "chrome/browser/browser_process.h"
-#include "chrome/browser/feedback/system_logs/about_system_logs_fetcher.h"
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/ui/webui/webui_util.h"
 #include "chrome/common/chrome_paths.h"
 #include "chrome/common/url_constants.h"
 #include "chrome/grit/browser_resources.h"
 #include "chrome/grit/generated_resources.h"
-#include "components/feedback/system_logs/system_logs_fetcher.h"
 #include "content/public/browser/browser_thread.h"
 #include "content/public/browser/web_contents.h"
 #include "content/public/browser/web_ui.h"
@@ -42,7 +40,6 @@
 
 using content::WebContents;
 using content::WebUIMessageHandler;
-using system_logs::SystemLogsResponse;
 
 namespace {
 
@@ -86,8 +83,6 @@ class SystemInfoHandler : public WebUIMessageHandler {
   // system info and eventually returns it to the front end.
   void HandleRequestSystemInfo(const base::ListValue* args);
 
-  void OnSystemInfo(std::unique_ptr<SystemLogsResponse> sys_info);
-
  private:
   std::string callback_id_;
   base::WeakPtrFactory<SystemInfoHandler> weak_ptr_factory_{this};
@@ -117,14 +112,17 @@ void SystemInfoHandler::RegisterMessages() {
 
 void SystemInfoHandler::HandleRequestSystemInfo(const base::ListValue* args) {
   AllowJavascript();
+#if 0
   callback_id_ = args->GetList()[0].GetString();
 
   system_logs::SystemLogsFetcher* fetcher =
       system_logs::BuildAboutSystemLogsFetcher();
   fetcher->Fetch(base::BindOnce(&SystemInfoHandler::OnSystemInfo,
                                 weak_ptr_factory_.GetWeakPtr()));
+#endif
 }
 
+#if 0
 void SystemInfoHandler::OnSystemInfo(
     std::unique_ptr<SystemLogsResponse> sys_info) {
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
@@ -141,6 +139,7 @@ void SystemInfoHandler::OnSystemInfo(
   ResolveJavascriptCallback(base::Value(callback_id_), data);
   callback_id_.clear();
 }
+#endif
 
 ////////////////////////////////////////////////////////////////////////////////
 //
