diff --git a/chrome/chrome_elf/crash/crash_helper.cc b/chrome/chrome_elf/crash/crash_helper.cc
index 886372e11489..1fe56918078c 100644
--- a/chrome/chrome_elf/crash/crash_helper.cc
+++ b/chrome/chrome_elf/crash/crash_helper.cc
@@ -15,7 +15,6 @@
 #include "chrome/chrome_elf/hook_util/hook_util.h"
 #include "components/crash/core/app/crashpad.h"
 #include "components/crash/core/common/crash_keys.h"
-#include "third_party/crashpad/crashpad/client/crashpad_client.h"
 
 namespace {
 
@@ -77,8 +76,6 @@ bool InitializeCrashReporting() {
   g_crash_reports = new std::vector<crash_reporter::Report>;
   g_set_unhandled_exception_filter = new elf_hook::IATHook();
 
-  ChromeCrashReporterClient::InitializeCrashReportingForProcess();
-
   g_crash_helper_enabled = true;
   return true;
 }
@@ -112,9 +109,6 @@ void DisableSetUnhandledExceptionFilter() {
 }
 
 int GenerateCrashDump(EXCEPTION_POINTERS* exception_pointers) {
-  if (g_crash_helper_enabled)
-    crashpad::CrashpadClient::DumpWithoutCrash(
-        *(exception_pointers->ContextRecord));
   return EXCEPTION_CONTINUE_SEARCH;
 }
 
