diff --git a/chrome/browser/flag_descriptions.h b/chrome/browser/flag_descriptions.h
index b9c23379eb1a..694efd27d9a8 100644
--- a/chrome/browser/flag_descriptions.h
+++ b/chrome/browser/flag_descriptions.h
@@ -1273,6 +1273,9 @@ extern const char kStrictOriginIsolationDescription[];
 extern const char kSystemKeyboardLockName[];
 extern const char kSystemKeyboardLockDescription[];
 
+extern const char kTabAudioMutingName[];
+extern const char kTabAudioMutingDescription[];
+
 extern const char kSuggestedContentToggleName[];
 extern const char kSuggestedContentToggleDescription[];
 
