diff --git a/chrome/browser/metrics/BUILD.gn b/chrome/browser/metrics/BUILD.gn
index b4a3aad1b390..334fadbb35b5 100644
--- a/chrome/browser/metrics/BUILD.gn
+++ b/chrome/browser/metrics/BUILD.gn
@@ -16,7 +16,6 @@ source_set("test_support") {
     "//chrome/browser",
     "//components/signin/public/identity_manager",
     "//components/sync/test/fake_server",
-    "//third_party/metrics_proto",
   ]
 
   if (!is_fuchsia) {
