diff --git a/chrome/renderer/BUILD.gn.rej b/chrome/renderer/BUILD.gn.rej
new file mode 100644
index 000000000000..dc1c8d7f86d0
--- /dev/null
+++ b/chrome/renderer/BUILD.gn.rej
@@ -0,0 +1,33 @@
+***************
+*** 96,119 ****
+      "plugins/plugin_uma.h",
+      "previews/resource_loading_hints_agent.cc",
+      "previews/resource_loading_hints_agent.h",
+-     "subresource_redirect/login_robots_decider_agent.cc",
+-     "subresource_redirect/login_robots_decider_agent.h",
+-     "subresource_redirect/public_image_hints_decider_agent.cc",
+-     "subresource_redirect/public_image_hints_decider_agent.h",
+-     "subresource_redirect/public_resource_decider.h",
+-     "subresource_redirect/public_resource_decider_agent.cc",
+-     "subresource_redirect/public_resource_decider_agent.h",
+-     "subresource_redirect/redirect_result.h",
+-     "subresource_redirect/robots_rules_parser.cc",
+-     "subresource_redirect/robots_rules_parser.h",
+-     "subresource_redirect/robots_rules_parser_cache.cc",
+-     "subresource_redirect/robots_rules_parser_cache.h",
+-     "subresource_redirect/subresource_redirect_params.cc",
+-     "subresource_redirect/subresource_redirect_params.h",
+-     "subresource_redirect/subresource_redirect_url_loader_throttle.cc",
+-     "subresource_redirect/subresource_redirect_url_loader_throttle.h",
+-     "subresource_redirect/subresource_redirect_util.cc",
+-     "subresource_redirect/subresource_redirect_util.h",
+      "sync_encryption_keys_extension.cc",
+      "sync_encryption_keys_extension.h",
+      "url_loader_throttle_provider_impl.cc",
+--- 90,95 ----
+      "plugins/plugin_uma.h",
+      "previews/resource_loading_hints_agent.cc",
+      "previews/resource_loading_hints_agent.h",
+      "sync_encryption_keys_extension.cc",
+      "sync_encryption_keys_extension.h",
+      "url_loader_throttle_provider_impl.cc",
