diff --git a/chrome/common/webui_url_constants.cc b/chrome/common/webui_url_constants.cc
index e39322233555..46f86a0895a1 100644
--- a/chrome/common/webui_url_constants.cc
+++ b/chrome/common/webui_url_constants.cc
@@ -7,7 +7,6 @@
 #include "base/stl_util.h"
 #include "base/strings/string_piece.h"
 #include "components/nacl/common/buildflags.h"
-#include "components/safe_browsing/core/web_ui/constants.h"
 #include "extensions/buildflags/buildflags.h"
 
 namespace chrome {
@@ -442,7 +441,6 @@ const char* const kChromeHostURLs[] = {
     kChromeUIBluetoothInternalsHost,
     kChromeUIChromeURLsHost,
     kChromeUIComponentsHost,
-    kChromeUICrashesHost,
     kChromeUICreditsHost,
 #if defined(OS_CHROMEOS) && !defined(OFFICIAL_BUILD)
     kChromeUIDeviceEmulatorHost,
@@ -472,7 +470,6 @@ const char* const kChromeHostURLs[] = {
     kChromeUISignInInternalsHost,
     kChromeUISiteEngagementHost,
     kChromeUINTPTilesInternalsHost,
-    safe_browsing::kChromeUISafeBrowsingHost,
     kChromeUISuggestionsHost,
     kChromeUISupervisedUserInternalsHost,
     kChromeUISyncInternalsHost,
