diff --git a/chrome/browser/ui/webui/crashes_ui.cc b/chrome/browser/ui/webui/crashes_ui.cc
index fa9f7c350c16..52763d75c53f 100644
--- a/chrome/browser/ui/webui/crashes_ui.cc
+++ b/chrome/browser/ui/webui/crashes_ui.cc
@@ -19,7 +19,6 @@
 #include "base/values.h"
 #include "build/build_config.h"
 #include "build/chromeos_buildflags.h"
-#include "chrome/browser/crash_upload_list/crash_upload_list.h"
 #include "chrome/browser/metrics/chrome_metrics_service_accessor.h"
 #include "chrome/browser/metrics/metrics_reporting_state.h"
 #include "chrome/browser/profiles/profile.h"
@@ -103,7 +102,6 @@ class CrashesDOMHandler : public WebUIMessageHandler {
   // Asynchronously requests a user triggered upload. Called from JS.
   void HandleRequestSingleCrashUpload(const base::ListValue* args);
 
-  scoped_refptr<UploadList> upload_list_;
   bool list_available_;
   bool first_load_;
 
@@ -112,16 +110,12 @@ class CrashesDOMHandler : public WebUIMessageHandler {
 
 CrashesDOMHandler::CrashesDOMHandler()
     : list_available_(false), first_load_(true) {
-  upload_list_ = CreateCrashUploadList();
 }
 
 CrashesDOMHandler::~CrashesDOMHandler() {
-  upload_list_->CancelLoadCallback();
 }
 
 void CrashesDOMHandler::RegisterMessages() {
-  upload_list_->Load(base::BindOnce(&CrashesDOMHandler::OnUploadListAvailable,
-                                    base::Unretained(this)));
   web_ui()->RegisterMessageCallback(
       crash_reporter::kCrashesUIRequestCrashList,
       base::BindRepeating(&CrashesDOMHandler::HandleRequestCrashes,
@@ -147,8 +141,6 @@ void CrashesDOMHandler::HandleRequestCrashes(const base::ListValue* args) {
       UpdateUI();
   } else {
     list_available_ = false;
-    upload_list_->Load(base::BindOnce(&CrashesDOMHandler::OnUploadListAvailable,
-                                      base::Unretained(this)));
   }
 }
 
@@ -212,8 +204,6 @@ void CrashesDOMHandler::UpdateUI() {
   bool upload_list = using_crashpad || crash_reporting_enabled;
 
   base::ListValue crash_list;
-  if (upload_list)
-    crash_reporter::UploadListToValue(upload_list_.get(), &crash_list);
 
   base::Value enabled(crash_reporting_enabled);
   base::Value dynamic_backend(system_crash_reporter);
@@ -248,7 +238,6 @@ void CrashesDOMHandler::HandleRequestSingleCrashUpload(
       IsMetricsReportingPolicyManaged()) {
     return;
   }
-  upload_list_->RequestSingleUploadAsync(local_id);
 }
 
 }  // namespace
