diff --git a/chrome/browser/extensions/api/autofill_private/autofill_private_api.cc b/chrome/browser/extensions/api/autofill_private/autofill_private_api.cc
index 07bb9c111523..035aefd435be 100644
--- a/chrome/browser/extensions/api/autofill_private/autofill_private_api.cc
+++ b/chrome/browser/extensions/api/autofill_private/autofill_private_api.cc
@@ -484,22 +484,6 @@ AutofillPrivateMigrateCreditCardsFunction::Run() {
   return RespondNow(NoArguments());
 }
 
-////////////////////////////////////////////////////////////////////////////////
-// AutofillPrivateLogServerCardLinkClickedFunction
-
-ExtensionFunction::ResponseAction
-AutofillPrivateLogServerCardLinkClickedFunction::Run() {
-  autofill::PersonalDataManager* personal_data =
-      autofill::PersonalDataManagerFactory::GetForProfile(
-          chrome_details_.GetProfile());
-
-  if (!personal_data || !personal_data->IsDataLoaded())
-    return RespondNow(Error(kErrorDataUnavailable));
-
-  personal_data->LogServerCardLinkClicked();
-  return RespondNow(NoArguments());
-}
-
 ////////////////////////////////////////////////////////////////////////////////
 // AutofillPrivateSetCreditCardFIDOAuthEnabledStateFunction
 
