diff --git a/chrome/browser/media/router/media_router_feature.cc b/chrome/browser/media/router/media_router_feature.cc
index 843b8d4ef78e..7d83e6122c08 100644
--- a/chrome/browser/media/router/media_router_feature.cc
+++ b/chrome/browser/media/router/media_router_feature.cc
@@ -52,6 +52,7 @@ const base::Feature kAllowAllSitesToInitiateMirroring{
 
 #if defined(OS_ANDROID) || BUILDFLAG(ENABLE_EXTENSIONS)
 namespace {
+ALLOW_UNUSED_TYPE
 const PrefService::Preference* GetMediaRouterPref(
     content::BrowserContext* context) {
   return user_prefs::UserPrefs::Get(context)->FindPreference(
@@ -61,6 +62,8 @@ const PrefService::Preference* GetMediaRouterPref(
 #endif  // defined(OS_ANDROID) || BUILDFLAG(ENABLE_EXTENSIONS)
 
 bool MediaRouterEnabled(content::BrowserContext* context) {
+  return false;
+#if 0
 #if !defined(OFFICIAL_BUILD) && !defined(OS_ANDROID)
   if (!base::FeatureList::IsEnabled(kMediaRouter))
     return false;
@@ -81,6 +84,7 @@ bool MediaRouterEnabled(content::BrowserContext* context) {
 #else   // !(defined(OS_ANDROID) || BUILDFLAG(ENABLE_EXTENSIONS))
   return false;
 #endif  // defined(OS_ANDROID) || BUILDFLAG(ENABLE_EXTENSIONS)
+#endif
 }
 
 #if !defined(OS_ANDROID)
