diff --git a/chrome/installer/mini_installer/BUILD.gn b/chrome/installer/mini_installer/BUILD.gn
index 51fa0d7f4cf77..6886bb9070079 100644
--- a/chrome/installer/mini_installer/BUILD.gn
+++ b/chrome/installer/mini_installer/BUILD.gn
@@ -120,7 +120,7 @@ action("mini_installer_archive") {
 
   inputs = [
     "$root_out_dir/chrome.dll",
-    "$root_out_dir/chrome.exe",
+    "$root_out_dir/dumb.exe",
     "$root_out_dir/locales/en-US.pak",
     "$root_out_dir/setup.exe",
     "//chrome/tools/build/win/makecab.py",
