diff --git a/chrome/common/chrome_content_client.cc b/chrome/common/chrome_content_client.cc
index 4bdfa57a8114..8781fd865dc2 100644
--- a/chrome/common/chrome_content_client.cc
+++ b/chrome/common/chrome_content_client.cc
@@ -34,10 +34,8 @@
 #include "chrome/common/chrome_constants.h"
 #include "chrome/common/chrome_paths.h"
 #include "chrome/common/chrome_switches.h"
-#include "chrome/common/crash_keys.h"
 #include "chrome/common/url_constants.h"
 #include "chrome/grit/common_resources.h"
-#include "components/crash/core/common/crash_key.h"
 #include "components/dom_distiller/core/url_constants.h"
 #include "components/embedder_support/origin_trials/origin_trial_policy_impl.h"
 #include "components/services/heap_profiling/public/cpp/profiling_client.h"
@@ -355,16 +353,9 @@ void ChromeContentClient::SetPDFEntryFunctions(
 
 void ChromeContentClient::SetActiveURL(const GURL& url,
                                        std::string top_origin) {
-  static crash_reporter::CrashKeyString<1024> active_url("url-chunk");
-  active_url.Set(url.possibly_invalid_spec());
-
-  // Use a large enough size for Origin::GetDebugString.
-  static crash_reporter::CrashKeyString<128> top_origin_key("top-origin");
-  top_origin_key.Set(top_origin);
 }
 
 void ChromeContentClient::SetGpuInfo(const gpu::GPUInfo& gpu_info) {
-  gpu::SetKeysForCrashLogging(gpu_info);
 }
 
 #if BUILDFLAG(ENABLE_PLUGINS)
