diff --git a/chrome/browser/ui/cocoa/javascript_app_modal_dialog_cocoa.mm b/chrome/browser/ui/cocoa/javascript_app_modal_dialog_cocoa.mm
index 6c3f4338c6bb..1e1523b13d76 100644
--- a/chrome/browser/ui/cocoa/javascript_app_modal_dialog_cocoa.mm
+++ b/chrome/browser/ui/cocoa/javascript_app_modal_dialog_cocoa.mm
@@ -12,7 +12,6 @@
 #include "base/macros.h"
 #include "base/memory/ptr_util.h"
 #import "chrome/browser/chrome_browser_application_mac.h"
-#include "chrome/browser/ui/blocked_content/popunder_preventer.h"
 #include "chrome/browser/ui/browser.h"
 #include "chrome/browser/ui/browser_list.h"
 #include "chrome/browser/ui/browser_window.h"
@@ -38,7 +37,6 @@ using remote_cocoa::mojom::AlertDisposition;
 JavaScriptAppModalDialogCocoa::JavaScriptAppModalDialogCocoa(
     javascript_dialogs::AppModalDialogController* controller)
     : controller_(controller),
-      popunder_preventer_(new PopunderPreventer(controller->web_contents())),
       weak_factory_(this) {}
 
 JavaScriptAppModalDialogCocoa::~JavaScriptAppModalDialogCocoa() {}
