diff --git a/chrome/browser/ui/views/chrome_browser_main_extra_parts_views.cc b/chrome/browser/ui/views/chrome_browser_main_extra_parts_views.cc
index e95483c592c9..c1450dfb1024 100644
--- a/chrome/browser/ui/views/chrome_browser_main_extra_parts_views.cc
+++ b/chrome/browser/ui/views/chrome_browser_main_extra_parts_views.cc
@@ -15,7 +15,6 @@
 #include "chrome/browser/ui/views/chrome_views_delegate.h"
 #include "chrome/browser/ui/views/devtools_process_observer.h"
 #include "chrome/browser/ui/views/media_router/media_router_dialog_controller_views.h"
-#include "chrome/browser/ui/views/relaunch_notification/relaunch_notification_controller.h"
 #include "components/constrained_window/constrained_window_views.h"
 #include "components/media_router/browser/media_router_dialog_controller.h"
 #include "components/ui_devtools/connector_delegate.h"
@@ -149,14 +148,7 @@ void ChromeBrowserMainExtraPartsViews::PreProfileInit() {
 }
 
 void ChromeBrowserMainExtraPartsViews::PostBrowserStart() {
-  relaunch_notification_controller_ =
-      std::make_unique<RelaunchNotificationController>(
-          UpgradeDetector::GetInstance());
 }
 
 void ChromeBrowserMainExtraPartsViews::PostMainMessageLoopRun() {
-  // The relaunch notification controller acts on timer-based events. Tear it
-  // down explicitly here to avoid a case where such an event arrives during
-  // shutdown.
-  relaunch_notification_controller_.reset();
 }
