diff --git a/chrome/browser/flag_descriptions.h b/chrome/browser/flag_descriptions.h
index ac658f945f54..99141e5ebedb 100644
--- a/chrome/browser/flag_descriptions.h
+++ b/chrome/browser/flag_descriptions.h
@@ -1280,6 +1280,9 @@ extern const char kStrictOriginIsolationDescription[];
 extern const char kSystemKeyboardLockName[];
 extern const char kSystemKeyboardLockDescription[];
 
+extern const char kTabAudioMutingName[];
+extern const char kTabAudioMutingDescription[];
+
 extern const char kSuggestedContentToggleName[];
 extern const char kSuggestedContentToggleDescription[];
 
