diff --git a/chrome/common/extensions/api/api_sources.gni b/chrome/common/extensions/api/api_sources.gni
index 8abfbeb827ff..f828460e1952 100644
--- a/chrome/common/extensions/api/api_sources.gni
+++ b/chrome/common/extensions/api/api_sources.gni
@@ -28,7 +28,6 @@ schema_sources_ = [
   "debugger.json",
   "desktop_capture.json",
   "developer_private.idl",
-  "crash_report_private.idl",
   "downloads.idl",
   "downloads_internal.idl",
   "enterprise_hardware_platform.idl",
@@ -47,7 +46,6 @@ schema_sources_ = [
   "passwords_private.idl",
   "permissions.json",
   "resources_private.idl",
-  "safe_browsing_private.idl",
   "scripting.idl",
   "search.idl",
   "sessions.json",
@@ -125,10 +123,6 @@ if (is_chromeos_ash) {
   schema_sources_ += [ "input_ime.json" ]
 }
 
-if (!is_chromeos_ash) {
-  schema_sources_ += [ "enterprise_reporting_private.idl" ]
-}
-
 if (enable_service_discovery) {
   schema_sources_ += [ "mdns.idl" ]
 }
