diff --git a/chrome/browser/chromeos/extensions/info_private_api.cc b/chrome/browser/chromeos/extensions/info_private_api.cc
index 042916343bd0..c950f4764e9e 100644
--- a/chrome/browser/chromeos/extensions/info_private_api.cc
+++ b/chrome/browser/chromeos/extensions/info_private_api.cc
@@ -248,9 +248,7 @@ bool IsEnterpriseKiosk() {
 }
 
 std::string GetClientId() {
-  return IsEnterpriseKiosk()
-             ? g_browser_process->metrics_service()->GetClientId()
-             : std::string();
+  return std::string();
 }
 
 }  // namespace
