diff --git a/chrome/common/search/search.mojom b/chrome/common/search/search.mojom
index e34dd5f0a96d7..cc2445ab523d7 100644
--- a/chrome/common/search/search.mojom
+++ b/chrome/common/search/search.mojom
@@ -138,9 +138,6 @@ interface EmbeddedSearch {
   // Confirms applied theme changes.
   ConfirmThemeChanges();
 
-  // Called when a user dismisses a promo.
-  BlocklistPromo(string promo_id);
-
   // Handles navigation to chrome://extensions by calling the browser to
   // complete the navigation for the extensions checkup middle slot promo.
   OpenExtensionsPage(double button,
