diff --git a/chrome/browser/resources/settings/site_settings_page/site_settings_page.js b/chrome/browser/resources/settings/site_settings_page/site_settings_page.js
index 2d7471ae2d63..0c7f3c0bea03 100644
--- a/chrome/browser/resources/settings/site_settings_page/site_settings_page.js
+++ b/chrome/browser/resources/settings/site_settings_page/site_settings_page.js
@@ -51,8 +51,7 @@ function getCategoryItemMap() {
       icon: 'settings:ads',
       enabledLabel: 'siteSettingsAllowed',
       disabledLabel: 'siteSettingsAdsBlock',
-      shouldShow: () =>
-          loadTimeData.getBoolean('enableSafeBrowsingSubresourceFilter'),
+      shouldShow: () => false,
     },
     {
       route: routes.SITE_SETTINGS_AR,
