diff --git a/chrome/renderer/extensions/resource_request_policy.cc b/chrome/renderer/extensions/resource_request_policy.cc
index 80a1db5421ca..02fa18f8b848 100644
--- a/chrome/renderer/extensions/resource_request_policy.cc
+++ b/chrome/renderer/extensions/resource_request_policy.cc
@@ -10,7 +10,6 @@
 #include "chrome/common/url_constants.h"
 #include "extensions/common/constants.h"
 #include "extensions/common/extension.h"
-#include "extensions/common/identifiability_metrics.h"
 #include "extensions/common/manifest_constants.h"
 #include "extensions/common/manifest_handlers/icons_handler.h"
 #include "extensions/common/manifest_handlers/web_accessible_resources_info.h"
@@ -119,11 +118,6 @@ bool ResourceRequestPolicy::CanRequestResource(
   // extensions with web accessible resources, since those are inherently
   // identifiable.
   if (!is_dev_tools && !web_accessible_ids_.count(extension_origin.host())) {
-    // Failures are recorded here, successes will be in the browser.
-    RecordExtensionResourceAccessResult(
-        ukm::SourceIdObj::FromInt64(frame->GetDocument().GetUkmSourceId()),
-        resource_url, ExtensionResourceAccessResult::kFailure);
-
     return false;
   }
 
@@ -156,9 +150,6 @@ bool ResourceRequestPolicy::CanRequestResource(
           .ContainsPath(resource_root_relative_path)) {
     LOG(ERROR) << "Denying load of " << resource_url.spec() << " from "
                << "hosted app.";
-    RecordExtensionResourceAccessResult(
-        ukm::SourceIdObj::FromInt64(frame->GetDocument().GetUkmSourceId()),
-        resource_url, ExtensionResourceAccessResult::kFailure);
     return false;
   }
 
@@ -177,9 +168,6 @@ bool ResourceRequestPolicy::CanRequestResource(
     frame->AddMessageToConsole(
         blink::WebConsoleMessage(blink::mojom::ConsoleMessageLevel::kError,
                                  blink::WebString::FromUTF8(message)));
-    RecordExtensionResourceAccessResult(
-        ukm::SourceIdObj::FromInt64(frame->GetDocument().GetUkmSourceId()),
-        resource_url, ExtensionResourceAccessResult::kFailure);
     return false;
   }
 
