diff --git a/chrome/browser/resources/settings/safety_check_page/BUILD.gn b/chrome/browser/resources/settings/safety_check_page/BUILD.gn
index a3bb95d34464d..1333a8c221122 100644
--- a/chrome/browser/resources/settings/safety_check_page/BUILD.gn
+++ b/chrome/browser/resources/settings/safety_check_page/BUILD.gn
@@ -73,7 +73,6 @@ js_library("safety_check_page") {
     ":safety_check_passwords_child",
     ":safety_check_safe_browsing_child",
     ":safety_check_updates_child",
-    "..:hats_browser_proxy",
     "..:metrics_browser_proxy",
     "..:route",
     "../autofill_page:password_manager_proxy",
