diff --git a/chrome/browser/profiles/off_the_record_profile_impl.h b/chrome/browser/profiles/off_the_record_profile_impl.h
index 90de081252443..8ebd634336bf5 100644
--- a/chrome/browser/profiles/off_the_record_profile_impl.h
+++ b/chrome/browser/profiles/off_the_record_profile_impl.h
@@ -14,7 +14,6 @@
 #include "build/chromeos_buildflags.h"
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/profiles/scoped_profile_keep_alive.h"
-#include "components/domain_reliability/clear_mode.h"
 #include "content/public/browser/content_browser_client.h"
 
 #if !defined(OS_ANDROID)
