diff --git a/chrome/browser/translate/translate_ranker_factory.cc b/chrome/browser/translate/translate_ranker_factory.cc
index ac6b0f413de7..e86c3d26885f 100644
--- a/chrome/browser/translate/translate_ranker_factory.cc
+++ b/chrome/browser/translate/translate_ranker_factory.cc
@@ -9,7 +9,6 @@
 #include "components/keyed_service/core/keyed_service.h"
 #include "components/translate/core/browser/translate_ranker_impl.h"
 #include "content/public/browser/browser_context.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
 
 namespace translate {
 
@@ -36,7 +35,7 @@ KeyedService* TranslateRankerFactory::BuildServiceInstanceFor(
     content::BrowserContext* browser_context) const {
   return new TranslateRankerImpl(
       TranslateRankerImpl::GetModelPath(browser_context->GetPath()),
-      TranslateRankerImpl::GetModelURL(), ukm::UkmRecorder::Get());
+      TranslateRankerImpl::GetModelURL());
 }
 
 content::BrowserContext* TranslateRankerFactory::GetBrowserContextToUse(
