diff --git a/chrome/browser/permissions/prediction_based_permission_ui_selector.cc b/chrome/browser/permissions/prediction_based_permission_ui_selector.cc
index fc0f0ebf4fc6b..58d4d9823d3e9 100644
--- a/chrome/browser/permissions/prediction_based_permission_ui_selector.cc
+++ b/chrome/browser/permissions/prediction_based_permission_ui_selector.cc
@@ -20,7 +20,6 @@
 #include "components/permissions/prediction_service/prediction_service.h"
 #include "components/permissions/prediction_service/prediction_service_messages.pb.h"
 #include "components/prefs/pref_service.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 
 namespace {
 
@@ -196,9 +195,7 @@ void PredictionBasedPermissionUiSelector::LookupReponseReceived(
 bool PredictionBasedPermissionUiSelector::IsAllowedToUseAssistedPrompts() {
   // We need to also check `kQuietNotificationPrompts` here since there is no
   // generic safeguard anywhere else in the stack.
-  return base::FeatureList::IsEnabled(features::kQuietNotificationPrompts) &&
-         base::FeatureList::IsEnabled(features::kPermissionPredictions) &&
-         safe_browsing::IsSafeBrowsingEnabled(*(profile_->GetPrefs()));
+  return false;
 }
 
 // static
