diff --git a/chrome/browser/ui/views/renderer_context_menu/render_view_context_menu_views.cc b/chrome/browser/ui/views/renderer_context_menu/render_view_context_menu_views.cc
index f6ff626c408e..c57753704004 100644
--- a/chrome/browser/ui/views/renderer_context_menu/render_view_context_menu_views.cc
+++ b/chrome/browser/ui/views/renderer_context_menu/render_view_context_menu_views.cc
@@ -278,7 +278,6 @@ void RenderViewContextMenuViews::ExecuteCommand(int command_id,
               ? base::i18n::RIGHT_TO_LEFT
               : base::i18n::LEFT_TO_RIGHT);
       view_host->GetWidget()->NotifyTextDirection();
-      RenderViewContextMenu::RecordUsedItem(command_id);
       break;
     }
 
