diff --git a/chrome/common/search/search.mojom b/chrome/common/search/search.mojom
index 9e6fa60f5ee5..ceb570b16703 100644
--- a/chrome/common/search/search.mojom
+++ b/chrome/common/search/search.mojom
@@ -161,9 +161,6 @@ interface EmbeddedSearch {
   // series of characters) was typed until Autocomplete results were painted.
   LogCharTypedToRepaintLatency(uint32 latency_ms);
 
-  // Called when a user dismisses a promo.
-  BlocklistPromo(string promo_id);
-
   // Handles navigation to chrome://extensions by calling the browser to
   // complete the navigation for the extensions checkup middle slot promo.
   OpenExtensionsPage(double button,
