diff --git a/chrome/browser/profile_resetter/resettable_settings_snapshot.cc b/chrome/browser/profile_resetter/resettable_settings_snapshot.cc
index c3a958cdea0ed..207614fc63537 100644
--- a/chrome/browser/profile_resetter/resettable_settings_snapshot.cc
+++ b/chrome/browser/profile_resetter/resettable_settings_snapshot.cc
@@ -222,12 +222,6 @@ std::unique_ptr<reset_report::ChromeResetReport> SerializeSettingsReportToProto(
   return report;
 }
 
-void SendSettingsFeedbackProto(const reset_report::ChromeResetReport& report,
-                               Profile* profile) {
-  ResetReportUploaderFactory::GetForBrowserContext(profile)
-      ->DispatchReport(report);
-}
-
 std::unique_ptr<base::ListValue> GetReadableFeedbackForSnapshot(
     Profile* profile,
     const ResettableSettingsSnapshot& snapshot) {
