diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index 16fe306a6097..c70ece70787d 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -107,8 +107,6 @@ static_library("common") {
     "common_message_generator.cc",
     "common_message_generator.h",
     "content_restriction.h",
-    "crash_keys.cc",
-    "crash_keys.h",
     "custom_handlers/protocol_handler.cc",
     "custom_handlers/protocol_handler.h",
     "google_url_loader_throttle.cc",
@@ -165,7 +163,6 @@ static_library("common") {
     "//chrome/app/theme:theme_resources",
     "//chrome/common:constants",
     "//chrome/common/net",
-    "//chrome/common/privacy_budget",
     "//chrome/common/profiler",
     "//chrome/common/qr_code_generator",
     "//chrome/common/search:mojo_bindings",
@@ -176,7 +173,6 @@ static_library("common") {
     "//components/cloud_devices/common",
     "//components/component_updater",
     "//components/content_settings/core/common",
-    "//components/crash/core/common",
     "//components/dom_distiller/core",
     "//components/embedder_support/origin_trials",
     "//components/error_page/common",
@@ -194,13 +190,11 @@ static_library("common") {
     "//components/offline_pages/buildflags",
     "//components/offline_pages/core:switches",
     "//components/omnibox/common",
-    "//components/page_load_metrics/common:page_load_metrics_mojom",
     "//components/password_manager/core/common",
     "//components/policy:generated",
     "//components/policy/core/common",
     "//components/prefs",
     "//components/safe_browsing:buildflags",
-    "//components/safe_browsing/core/web_ui:constants",
     "//components/services/heap_profiling/public/cpp",
     "//components/strings",
     "//components/translate/content/common",
@@ -240,10 +234,8 @@ static_library("common") {
 
   deps = [
     "//build:branding_buildflags",
-    "//components/crash/core/app",
     "//components/google/core/common",
     "//components/metrics:call_stack_profile_builder",
-    "//components/page_load_metrics/common:common",
     "//components/prerender/common",
     "//components/prerender/common:mojo_bindings",
   ]
@@ -427,7 +419,6 @@ static_library("common") {
     deps += [
       "//chrome/chrome_elf:chrome_elf_main_include",
       "//chrome/common/win:eventlog_messages",
-      "//components/crash/core/app:crash_export_thunk_include",
     ]
 
     public_deps += [
@@ -764,10 +755,6 @@ mojom("mojo_bindings") {
     public_deps += [ "//components/remote_cocoa/common:mojo" ]
   }
 
-  if (safe_browsing_mode == 1) {
-    public_deps += [ "//chrome/services/file_util/public/mojom" ]
-  }
-
   overridden_deps = [
     "//content/public/common:interfaces",
     "//third_party/blink/public/mojom:mojom_platform",
