diff --git a/chrome/utility/services.cc b/chrome/utility/services.cc
index 0728a65184f27..f3a7720cacb29 100644
--- a/chrome/utility/services.cc
+++ b/chrome/utility/services.cc
@@ -35,9 +35,6 @@
 #include "chrome/services/util_win/public/mojom/util_win.mojom.h"
 #include "chrome/services/util_win/util_read_icon.h"
 #include "chrome/services/util_win/util_win_impl.h"
-#include "components/services/quarantine/public/cpp/quarantine_features_win.h"  // nogncheck
-#include "components/services/quarantine/public/mojom/quarantine.mojom.h"  // nogncheck
-#include "components/services/quarantine/quarantine_impl.h"  // nogncheck
 #endif  // defined(OS_WIN)
 
 #if defined(OS_MAC)
