diff --git a/chrome/renderer/chrome_content_renderer_client.h b/chrome/renderer/chrome_content_renderer_client.h
index 4a6fff63e13a..e73202f33bd7 100644
--- a/chrome/renderer/chrome_content_renderer_client.h
+++ b/chrome/renderer/chrome_content_renderer_client.h
@@ -70,10 +70,6 @@ namespace extensions {
 class Extension;
 }
 
-namespace subresource_filter {
-class UnverifiedRulesetDealer;
-}
-
 namespace web_cache {
 class WebCacheImpl;
 }
@@ -266,8 +262,6 @@ class ChromeContentRendererClient
 #if BUILDFLAG(ENABLE_SPELLCHECK)
   std::unique_ptr<SpellCheck> spellcheck_;
 #endif
-  std::unique_ptr<subresource_filter::UnverifiedRulesetDealer>
-      subresource_filter_ruleset_dealer_;
 #if BUILDFLAG(ENABLE_PRINT_PREVIEW)
   std::unique_ptr<ChromePDFPrintClient> pdf_print_client_;
 #endif
