diff --git a/chrome/browser/extensions/omaha_attributes_handler.h b/chrome/browser/extensions/omaha_attributes_handler.h
index 2a3685e1c0f18..8720f4a79d298 100644
--- a/chrome/browser/extensions/omaha_attributes_handler.h
+++ b/chrome/browser/extensions/omaha_attributes_handler.h
@@ -5,6 +5,7 @@
 #ifndef CHROME_BROWSER_EXTENSIONS_OMAHA_ATTRIBUTES_HANDLER_H_
 #define CHROME_BROWSER_EXTENSIONS_OMAHA_ATTRIBUTES_HANDLER_H_
 
+#include "base/feature_list.h"
 #include "chrome/browser/extensions/blocklist.h"
 #include "extensions/browser/blocklist_state.h"
 #include "extensions/common/extension_id.h"
