diff --git a/chrome/browser/notifications/alert_dispatcher_xpc.mm b/chrome/browser/notifications/alert_dispatcher_xpc.mm
index d126ddd3a24d..69b9265d8a2c 100644
--- a/chrome/browser/notifications/alert_dispatcher_xpc.mm
+++ b/chrome/browser/notifications/alert_dispatcher_xpc.mm
@@ -26,7 +26,6 @@
 #include "components/crash/core/app/crashpad.h"
 #include "content/public/browser/browser_task_traits.h"
 #include "content/public/browser/browser_thread.h"
-#include "third_party/crashpad/crashpad/client/crashpad_client.h"
 
 namespace {
 
@@ -181,11 +180,6 @@ void RecordXPCEvent(XPCConnectionEvent event) {
   id<NotificationDelivery> proxy = [_xpcConnection remoteObjectProxy];
 
   if (!_setExceptionPort) {
-    base::mac::ScopedMachSendRight exceptionPort(
-        crash_reporter::GetCrashpadClient().GetHandlerMachPort());
-    base::scoped_nsobject<CrXPCMachPort> xpcPort(
-        [[CrXPCMachPort alloc] initWithMachSendRight:std::move(exceptionPort)]);
-    [proxy setUseUNNotification:NO machExceptionPort:xpcPort];
     _setExceptionPort = YES;
   }
 
