diff --git a/chrome/browser/ui/cocoa/notifications/BUILD.gn b/chrome/browser/ui/cocoa/notifications/BUILD.gn
index 8ad2efbcbc3a..5d78a687b395 100644
--- a/chrome/browser/ui/cocoa/notifications/BUILD.gn
+++ b/chrome/browser/ui/cocoa/notifications/BUILD.gn
@@ -25,7 +25,6 @@ mac_app_bundle("alert_notification_xpc_service") {
   deps = [
     ":common",
     "//base",
-    "//third_party/crashpad/crashpad/client",
   ]
 
   frameworks = [
