diff --git a/chrome/browser/flag_descriptions.h b/chrome/browser/flag_descriptions.h
index 091ad6f2ff3f..0aec9f8d9fff 100644
--- a/chrome/browser/flag_descriptions.h
+++ b/chrome/browser/flag_descriptions.h
@@ -1275,6 +1275,9 @@ extern const char kStrictOriginIsolationDescription[];
 extern const char kSystemKeyboardLockName[];
 extern const char kSystemKeyboardLockDescription[];
 
+extern const char kTabAudioMutingName[];
+extern const char kTabAudioMutingDescription[];
+
 extern const char kSuggestedContentToggleName[];
 extern const char kSuggestedContentToggleDescription[];
 
