diff --git a/chrome/services/file_util/public/mojom/BUILD.gn b/chrome/services/file_util/public/mojom/BUILD.gn
index 8f11a62901c37..809896a1ee0c8 100644
--- a/chrome/services/file_util/public/mojom/BUILD.gn
+++ b/chrome/services/file_util/public/mojom/BUILD.gn
@@ -37,12 +37,8 @@ mojom("mojom") {
           cpp = "::safe_browsing::ArchiveAnalyzerResults"
         },
       ]
-      traits_headers = [ "//chrome/common/safe_browsing/zip_analyzer.h" ]
-      traits_private_headers = [ "safe_archive_analyzer_param_traits.h" ]
       traits_public_deps = [
-        "//chrome/common/safe_browsing:proto",
         "//components/safe_browsing:buildflags",
-        "//components/safe_browsing/core:csd_proto",
       ]
     },
   ]
