diff --git a/chrome/browser/ui/ui_features.h b/chrome/browser/ui/ui_features.h
index b45856eb911a8..afbc3e64d0ffe 100644
--- a/chrome/browser/ui/ui_features.h
+++ b/chrome/browser/ui/ui_features.h
@@ -14,6 +14,7 @@
 #include "build/build_config.h"
 #include "build/chromeos_buildflags.h"
 #include "chrome/common/buildflags.h"
+#include "dumb/browser/ui/ui_features.h"
 
 namespace features {
 
@@ -32,8 +33,6 @@ extern const base::Feature kCommander;
 
 extern const base::Feature kEvDetailsInPageInfo;
 
-extern const base::Feature kExtensionsToolbarMenu;
-
 extern const base::Feature kNewProfilePicker;
 
 extern const base::Feature kNewTabstripAnimation;
