diff --git a/chrome/browser/resources/new_tab_page/app.js b/chrome/browser/resources/new_tab_page/app.js
index af519b04d0505..ca51e99360636 100644
--- a/chrome/browser/resources/new_tab_page/app.js
+++ b/chrome/browser/resources/new_tab_page/app.js
@@ -732,11 +732,6 @@ class AppElement extends mixinBehaviors
     }
   }
 
-  /** @private */
-  onMiddleSlotPromoLoaded_() {
-    this.middleSlotPromoLoaded_ = true;
-  }
-
   /** @private */
   onModulesLoaded_() {
     this.modulesLoaded_ = true;
