diff --git a/chrome/browser/resource_coordinator/tab_activity_watcher.h b/chrome/browser/resource_coordinator/tab_activity_watcher.h
index 91e527edc565..21e042d9c4ae 100644
--- a/chrome/browser/resource_coordinator/tab_activity_watcher.h
+++ b/chrome/browser/resource_coordinator/tab_activity_watcher.h
@@ -16,8 +16,6 @@
 #include "chrome/browser/ui/browser_tab_strip_tracker_delegate.h"
 #include "chrome/browser/ui/tabs/tab_strip_model_observer.h"
 
-class TabMetricsLogger;
-
 namespace resource_coordinator {
 
 class LifecycleUnit;
@@ -76,8 +74,6 @@ class TabActivityWatcher : public BrowserListObserver,
   // Resets internal state.
   void ResetForTesting();
 
-  std::unique_ptr<TabMetricsLogger> tab_metrics_logger_;
-
   // Manages registration of this class as an observer of all TabStripModels as
   // browsers are created and destroyed.
   BrowserTabStripTracker browser_tab_strip_tracker_;
