diff --git a/chrome/browser/apps/intent_helper/apps_navigation_throttle.cc b/chrome/browser/apps/intent_helper/apps_navigation_throttle.cc
index e306456d7350..552614c3b32f 100644
--- a/chrome/browser/apps/intent_helper/apps_navigation_throttle.cc
+++ b/chrome/browser/apps/intent_helper/apps_navigation_throttle.cc
@@ -47,10 +47,6 @@ using ThrottleCheckResult = content::NavigationThrottle::ThrottleCheckResult;
 // Returns true if |url| is a known and valid redirector that will redirect a
 // navigation elsewhere.
 bool IsGoogleRedirectorUrl(const GURL& url) {
-  // This currently only check for redirectors on the "google" domain.
-  if (!page_load_metrics::IsGoogleSearchHostname(url))
-    return false;
-
   return url.path_piece() == "/url" && url.has_query();
 }
 
@@ -179,7 +175,7 @@ void AppsNavigationThrottle::OnIntentPickerClosed(
 
 // static
 bool AppsNavigationThrottle::IsGoogleRedirectorUrlForTesting(const GURL& url) {
-  return IsGoogleRedirectorUrl(url);
+  return false;
 }
 
 // static
