diff --git a/chrome/browser/ui/extensions/extension_action_view_controller.cc b/chrome/browser/ui/extensions/extension_action_view_controller.cc
index f15dba73a0b60..c52528221c329 100644
--- a/chrome/browser/ui/extensions/extension_action_view_controller.cc
+++ b/chrome/browser/ui/extensions/extension_action_view_controller.cc
@@ -28,6 +28,7 @@
 #include "chrome/browser/ui/toolbar/toolbar_action_view_delegate.h"
 #include "chrome/grit/generated_resources.h"
 #include "components/sessions/content/session_tab_helper.h"
+#include "dumb/browser/ui/ui_features.h"
 #include "extensions/browser/extension_action.h"
 #include "extensions/browser/extension_action_manager.h"
 #include "extensions/browser/extension_registry.h"
@@ -258,7 +259,8 @@ bool ExtensionActionViewController::ExecuteAction(PopupShowAction show_action,
   if (!action_runner)
     return false;
 
-  extensions_container_->CloseOverflowMenuIfOpen();
+  if (base::FeatureList::IsEnabled(features::kExtensionsToolbarMenu))
+    extensions_container_->CloseOverflowMenuIfOpen();
 
   if (action_runner->RunAction(extension(), grant_tab_permissions) ==
       extensions::ExtensionAction::ACTION_SHOW_POPUP) {
