diff --git a/chrome/browser/ui/webui/settings/about_handler.cc b/chrome/browser/ui/webui/settings/about_handler.cc
index bf04fc096a58..fcb9576d5845 100644
--- a/chrome/browser/ui/webui/settings/about_handler.cc
+++ b/chrome/browser/ui/webui/settings/about_handler.cc
@@ -33,7 +33,6 @@
 #include "chrome/browser/ui/browser_commands.h"
 #include "chrome/browser/ui/browser_finder.h"
 #include "chrome/browser/ui/chrome_pages.h"
-#include "chrome/browser/upgrade_detector/upgrade_detector.h"
 #include "chrome/common/pref_names.h"
 #include "chrome/common/url_constants.h"
 #include "chrome/grit/chromium_strings.h"
@@ -255,11 +254,9 @@ AboutHandler::AboutHandler(Profile* profile)
     : profile_(profile),
       apply_changes_from_upgrade_observer_(false),
       clock_(base::DefaultClock::GetInstance()) {
-  UpgradeDetector::GetInstance()->AddObserver(this);
 }
 
 AboutHandler::~AboutHandler() {
-  UpgradeDetector::GetInstance()->RemoveObserver(this);
 }
 
 void AboutHandler::RegisterMessages() {
