diff --git a/chrome/browser/resources/settings/privacy_page/BUILD.gn b/chrome/browser/resources/settings/privacy_page/BUILD.gn
index b671664edcb4..2a960ac86b06 100644
--- a/chrome/browser/resources/settings/privacy_page/BUILD.gn
+++ b/chrome/browser/resources/settings/privacy_page/BUILD.gn
@@ -139,7 +139,6 @@ js_library("personalization_options.m") {
 js_library("privacy_page") {
   deps = [
     ":privacy_page_browser_proxy.m",
-    "..:hats_browser_proxy",
     "..:i18n_setup",
     "..:metrics_browser_proxy",
     "..:page_visibility",
