diff --git a/chrome/browser/extensions/chrome_extensions_browser_client.h b/chrome/browser/extensions/chrome_extensions_browser_client.h
index 31792c000a6e6..b492867c7aa11 100644
--- a/chrome/browser/extensions/chrome_extensions_browser_client.h
+++ b/chrome/browser/extensions/chrome_extensions_browser_client.h
@@ -15,7 +15,6 @@
 #include "base/memory/ref_counted.h"
 #include "build/build_config.h"
 #include "build/chromeos_buildflags.h"
-#include "chrome/browser/extensions/media_router_extension_access_logger_impl.h"
 #include "chrome/browser/extensions/user_script_listener.h"
 #include "extensions/browser/extensions_browser_client.h"
 #include "extensions/browser/kiosk/kiosk_delegate.h"
@@ -158,8 +157,6 @@ class ChromeExtensionsBrowserClient : public ExtensionsBrowserClient {
   base::FilePath GetSaveFilePath(content::BrowserContext* context) override;
   void SetLastSaveFilePath(content::BrowserContext* context,
                            const base::FilePath& path) override;
-  const MediaRouterExtensionAccessLogger* GetMediaRouterAccessLogger()
-      const override;
   bool HasIsolatedStorage(const std::string& extension_id,
                           content::BrowserContext* context) override;
   bool IsScreenshotRestricted(
@@ -169,9 +166,6 @@ class ChromeExtensionsBrowserClient : public ExtensionsBrowserClient {
 
   static void set_did_chrome_update_for_testing(bool did_update);
 
-  static void SetMediaRouterAccessLoggerForTesting(
-      MediaRouterExtensionAccessLogger* media_router_access_logger);
-
  private:
   friend struct base::LazyInstanceTraitsBase<ChromeExtensionsBrowserClient>;
 
@@ -189,8 +183,6 @@ class ChromeExtensionsBrowserClient : public ExtensionsBrowserClient {
 
   UserScriptListener user_script_listener_;
 
-  MediaRouterExtensionAccessLoggerImpl media_router_access_logger_;
-
   DISALLOW_COPY_AND_ASSIGN(ChromeExtensionsBrowserClient);
 };
 
