diff --git a/chrome/browser/feedback/system_logs/log_sources/chrome_internal_log_source.h b/chrome/browser/feedback/system_logs/log_sources/chrome_internal_log_source.h
index b04ff4dee68c..48e7c1750b11 100644
--- a/chrome/browser/feedback/system_logs/log_sources/chrome_internal_log_source.h
+++ b/chrome/browser/feedback/system_logs/log_sources/chrome_internal_log_source.h
@@ -30,7 +30,6 @@ class ChromeInternalLogSource : public SystemLogsSource {
   void PopulateSyncLogs(SystemLogsResponse* response);
   void PopulateExtensionInfoLogs(SystemLogsResponse* response);
   void PopulatePowerApiLogs(SystemLogsResponse* response);
-  void PopulateDataReductionProxyLogs(SystemLogsResponse* response);
 
 #if BUILDFLAG(IS_CHROMEOS_ASH)
   void PopulateLocalStateSettings(SystemLogsResponse* response);
