diff --git a/chrome/app/chrome_crash_reporter_client.cc b/chrome/app/chrome_crash_reporter_client.cc
index abacf224d9ee..12f38d4806ca 100644
--- a/chrome/app/chrome_crash_reporter_client.cc
+++ b/chrome/app/chrome_crash_reporter_client.cc
@@ -20,7 +20,6 @@
 #include "chrome/common/chrome_result_codes.h"
 #include "chrome/common/env_vars.h"
 #include "chrome/installer/util/google_update_settings.h"
-#include "components/crash/core/common/crash_keys.h"
 #include "content/public/common/content_switches.h"
 
 #if defined(OS_POSIX) && !defined(OS_MAC)
@@ -92,7 +91,6 @@ ChromeCrashReporterClient::~ChromeCrashReporterClient() {}
 #if !defined(OS_MAC) && !defined(OS_ANDROID)
 void ChromeCrashReporterClient::SetCrashReporterClientIdFromGUID(
     const std::string& client_guid) {
-  crash_keys::SetMetricsClientIdFromGUID(client_guid);
 }
 #endif
 
