diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
index ee79b021ff01..67d974f96326 100644
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -334,8 +334,6 @@ android_library("chrome_java") {
     "//chrome/browser/preferences:java",
     "//chrome/browser/privacy:java",
     "//chrome/browser/profiles/android:java",
-    "//chrome/browser/safe_browsing/android:java",
-    "//chrome/browser/safety_check/android:java",
     "//chrome/browser/settings:java",
     "//chrome/browser/share:java",
     "//chrome/browser/share/android:java_resources",
@@ -387,7 +385,6 @@ android_library("chrome_java") {
     "//components/content_settings/android:content_settings_enums_java",
     "//components/content_settings/android:java",
     "//components/contextual_search/content/common/mojom:mojom_java",
-    "//components/crash/android:java",
     "//components/dom_distiller/content/browser/android:dom_distiller_content_java",
     "//components/dom_distiller/core/android:dom_distiller_core_java",
     "//components/dom_distiller/core/mojom:mojom_java",
@@ -435,7 +432,6 @@ android_library("chrome_java") {
     "//components/policy/android:policy_java",
     "//components/prefs/android:java",
     "//components/query_tiles:java",
-    "//components/safe_browsing/android:safe_browsing_java",
     "//components/schema_org/common:mojom_java",
     "//components/search_engines/android:java",
     "//components/security_interstitials/content/android:java",
@@ -809,8 +805,6 @@ junit_binary("chrome_junit_tests") {
     "//chrome/browser/preferences:java",
     "//chrome/browser/preferences:preferences_junit_tests",
     "//chrome/browser/profiles/android:java",
-    "//chrome/browser/safety_check/android:java",
-    "//chrome/browser/safety_check/android:junit",
     "//chrome/browser/share:java",
     "//chrome/browser/tab:java",
     "//chrome/browser/tabmodel:factory_java",
@@ -1010,10 +1004,6 @@ android_library("chrome_test_java") {
     "//chrome/browser/policy/android:java",
     "//chrome/browser/preferences:java",
     "//chrome/browser/profiles/android:java",
-    "//chrome/browser/safe_browsing/android:java",
-    "//chrome/browser/safe_browsing/android:javatests",
-    "//chrome/browser/safety_check/android:java",
-    "//chrome/browser/safety_check/android:javatests",
     "//chrome/browser/settings:java",
     "//chrome/browser/settings:javatests",
     "//chrome/browser/settings:test_support_java",
@@ -1064,7 +1054,6 @@ android_library("chrome_test_java") {
     "//components/browser_ui/widget/android:test_support_java",
     "//components/content_settings/android:content_settings_enums_java",
     "//components/content_settings/android:java",
-    "//components/crash/android:java",
     "//components/dom_distiller/core/android:dom_distiller_core_java",
     "//components/dom_distiller/core/mojom:mojom_java",
     "//components/download/internal/background_service:internal_java",
@@ -1112,7 +1101,6 @@ android_library("chrome_test_java") {
     "//components/prefs/android:java",
     "//components/query_tiles:public_java",
     "//components/query_tiles:test_support_java",
-    "//components/safe_browsing/android:safe_browsing_java",
     "//components/schema_org/common:mojom_java",
     "//components/search_engines/android:java",
     "//components/security_interstitials/content/android:java",
@@ -1540,7 +1528,6 @@ chrome_common_shared_library("libchromefortest") {
     "//chrome/browser/password_manager/android_test_helpers:android_wrappers",
     "//chrome/browser/subresource_filter:test_support",
     "//components/autofill_assistant/browser:test_support",
-    "//components/crash/android:crash_android",
     "//components/minidump_uploader",
     "//components/paint_preview/player/android:test_support",
     "//components/sync",
@@ -1802,7 +1789,6 @@ template("libmonochrome_apk_or_bundle_tmpl") {
     deps = [
       "//android_webview/lib",
       "//android_webview/nonembedded",
-      "//components/crash/android:crashpad_main",
     ]
     if (defined(invoker.deps)) {
       deps += invoker.deps
@@ -3221,10 +3207,6 @@ generate_jni("chrome_jni_headers") {
     "java/src/org/chromium/chrome/browser/query_tiles/TileProviderFactory.java",
     "java/src/org/chromium/chrome/browser/rappor/RapporServiceBridge.java",
     "java/src/org/chromium/chrome/browser/resources/ResourceMapper.java",
-    "java/src/org/chromium/chrome/browser/rlz/RevenueStats.java",
-    "java/src/org/chromium/chrome/browser/rlz/RlzPingHandler.java",
-    "java/src/org/chromium/chrome/browser/safe_browsing/SafeBrowsingPasswordReuseDialogBridge.java",
-    "java/src/org/chromium/chrome/browser/safe_browsing/SafeBrowsingSettingsLauncher.java",
     "java/src/org/chromium/chrome/browser/screenshot/EditorScreenshotTask.java",
     "java/src/org/chromium/chrome/browser/search_engines/TemplateUrlServiceFactory.java",
     "java/src/org/chromium/chrome/browser/sharing/SharingJNIBridge.java",
