diff --git a/chrome/browser/ui/autofill/chrome_autofill_client.cc b/chrome/browser/ui/autofill/chrome_autofill_client.cc
index 7bb45864bbce..73c28255d8c0 100644
--- a/chrome/browser/ui/autofill/chrome_autofill_client.cc
+++ b/chrome/browser/ui/autofill/chrome_autofill_client.cc
@@ -59,7 +59,6 @@
 #include "components/signin/public/identity_manager/identity_manager.h"
 #include "components/sync/driver/sync_service.h"
 #include "components/translate/core/browser/translate_manager.h"
-#include "components/ukm/content/source_url_recorder.h"
 #include "components/user_prefs/user_prefs.h"
 #include "components/variations/service/variations_service.h"
 #include "content/public/browser/navigation_entry.h"
@@ -164,14 +163,6 @@ StrikeDatabase* ChromeAutofillClient::GetStrikeDatabase() {
   return StrikeDatabaseFactory::GetForProfile(profile);
 }
 
-ukm::UkmRecorder* ChromeAutofillClient::GetUkmRecorder() {
-  return ukm::UkmRecorder::Get();
-}
-
-ukm::SourceId ChromeAutofillClient::GetUkmSourceId() {
-  return ukm::GetSourceIdForWebContentsDocument(web_contents());
-}
-
 AddressNormalizer* ChromeAutofillClient::GetAddressNormalizer() {
   if (base::FeatureList::IsEnabled(features::kAutofillAddressNormalizer))
     return AddressNormalizerFactory::GetInstance();
