diff --git a/chrome/browser/win/conflicts/module_database.cc b/chrome/browser/win/conflicts/module_database.cc
index 83024dbd02e67..1a5684b8398f4 100644
--- a/chrome/browser/win/conflicts/module_database.cc
+++ b/chrome/browser/win/conflicts/module_database.cc
@@ -91,7 +91,6 @@ ModuleDatabase::ModuleDatabase(bool third_party_blocking_policy_enabled)
       module_inspector_(base::BindRepeating(&ModuleDatabase::OnModuleInspected,
                                             base::Unretained(this))) {
   AddObserver(&module_inspector_);
-  AddObserver(&third_party_metrics_);
 
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
   MaybeInitializeThirdPartyConflictsManager(
@@ -413,8 +412,6 @@ void ModuleDatabase::NotifyLoadedModules(ModuleDatabaseObserver* observer) {
 
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
 void ModuleDatabase::OnThirdPartyBlockingDisabled() {
-  third_party_metrics_.SetHookDisabled();
-
   if (third_party_conflicts_manager_)
     third_party_conflicts_manager_->DisableModuleAnalysis();
 }
