diff --git a/chrome/browser/predictors/loading_predictor.cc b/chrome/browser/predictors/loading_predictor.cc
index 9b0afcd76bba9..1db2a0e369c65 100644
--- a/chrome/browser/predictors/loading_predictor.cc
+++ b/chrome/browser/predictors/loading_predictor.cc
@@ -9,8 +9,6 @@
 
 #include "base/metrics/histogram_macros.h"
 #include "build/build_config.h"
-#include "chrome/browser/predictors/loading_data_collector.h"
-#include "chrome/browser/predictors/loading_stats_collector.h"
 #include "chrome/browser/predictors/predictors_features.h"
 #include "chrome/browser/predictors/resource_prefetch_predictor.h"
 #include "content/public/browser/browser_thread.h"
@@ -93,14 +91,7 @@ LoadingPredictor::LoadingPredictor(const LoadingPredictorConfig& config,
     : config_(config),
       profile_(profile),
       resource_prefetch_predictor_(
-          std::make_unique<ResourcePrefetchPredictor>(config, profile)),
-      stats_collector_(std::make_unique<LoadingStatsCollector>(
-          resource_prefetch_predictor_.get(),
-          config)),
-      loading_data_collector_(std::make_unique<LoadingDataCollector>(
-          resource_prefetch_predictor_.get(),
-          stats_collector_.get(),
-          config)) {}
+          std::make_unique<ResourcePrefetchPredictor>(config, profile)) {}
 
 LoadingPredictor::~LoadingPredictor() {
   DCHECK(shutdown_);
@@ -168,10 +159,6 @@ void LoadingPredictor::StartInitialization() {
   resource_prefetch_predictor_->StartInitialization();
 }
 
-LoadingDataCollector* LoadingPredictor::loading_data_collector() {
-  return loading_data_collector_.get();
-}
-
 ResourcePrefetchPredictor* LoadingPredictor::resource_prefetch_predictor() {
   return resource_prefetch_predictor_.get();
 }
@@ -216,8 +203,6 @@ bool LoadingPredictor::OnNavigationStarted(NavigationId navigation_id,
   if (shutdown_)
     return true;
 
-  loading_data_collector()->RecordStartNavigation(
-      navigation_id, ukm_source_id, main_frame_url, creation_time);
   CleanupAbandonedHintsAndNavigations(navigation_id);
   active_navigations_.emplace(navigation_id,
                               NavigationInfo{main_frame_url, creation_time});
@@ -232,8 +217,6 @@ void LoadingPredictor::OnNavigationFinished(NavigationId navigation_id,
   if (shutdown_)
     return;
 
-  loading_data_collector()->RecordFinishNavigation(
-      navigation_id, old_main_frame_url, new_main_frame_url, is_error_page);
   if (active_urls_to_navigations_.find(old_main_frame_url) !=
       active_urls_to_navigations_.end()) {
     active_urls_to_navigations_[old_main_frame_url].erase(navigation_id);
@@ -348,9 +331,6 @@ void LoadingPredictor::PreconnectInitiated(const GURL& url,
   auto nav_id_set_it = active_urls_to_navigations_.find(url);
   if (nav_id_set_it == active_urls_to_navigations_.end())
     return;
-
-  for (const auto& nav_id : nav_id_set_it->second)
-    loading_data_collector_->RecordPreconnectInitiated(nav_id, preconnect_url);
 }
 
 void LoadingPredictor::PreconnectFinished(
@@ -361,7 +341,6 @@ void LoadingPredictor::PreconnectFinished(
 
   DCHECK(stats);
   active_hints_.erase(stats->url);
-  stats_collector_->RecordPreconnectStats(std::move(stats));
 }
 
 void LoadingPredictor::PrefetchInitiated(const GURL& url,
@@ -373,9 +352,6 @@ void LoadingPredictor::PrefetchInitiated(const GURL& url,
   auto nav_id_set_it = active_urls_to_navigations_.find(url);
   if (nav_id_set_it == active_urls_to_navigations_.end())
     return;
-
-  for (const auto& nav_id : nav_id_set_it->second)
-    loading_data_collector_->RecordPrefetchInitiated(nav_id, prefetch_url);
 }
 
 void LoadingPredictor::PrefetchFinished(std::unique_ptr<PrefetchStats> stats) {
