diff --git a/chrome/browser/resources/settings/hats_browser_proxy.js b/chrome/browser/resources/settings/hats_browser_proxy.js
index 5db403ff57e47..47c1aaf6ce192 100644
--- a/chrome/browser/resources/settings/hats_browser_proxy.js
+++ b/chrome/browser/resources/settings/hats_browser_proxy.js
@@ -39,4 +39,3 @@ import {addSingletonGetter} from 'chrome://resources/js/cr.m.js';
   }
 
   addSingletonGetter(HatsBrowserProxyImpl);
-
