diff --git a/chrome/browser/memory/chrome_browser_main_extra_parts_memory.h b/chrome/browser/memory/chrome_browser_main_extra_parts_memory.h
index 1c8b902f3d50..99870df61a8c 100644
--- a/chrome/browser/memory/chrome_browser_main_extra_parts_memory.h
+++ b/chrome/browser/memory/chrome_browser_main_extra_parts_memory.h
@@ -19,10 +19,6 @@ class SystemMemoryPressureEvaluator;
 }  // namespace chromeos
 #endif
 
-namespace memory {
-class EnterpriseMemoryLimitPrefObserver;
-}  // namespace memory
-
 // Wrapper that owns and initialize the browser memory-related extra parts.
 class ChromeBrowserMainExtraPartsMemory : public ChromeBrowserMainExtraParts {
  public:
@@ -34,15 +30,6 @@ class ChromeBrowserMainExtraPartsMemory : public ChromeBrowserMainExtraParts {
   void PostBrowserStart() override;
   void PostMainMessageLoopRun() override;
 
-  // Tracks changes to the MemoryLimitMbEnabled enterprise policy, and
-  // starts/stops the EnterpriseMemoryLimitEvaluator accordingly.
-  //
-  // Only supported on some platforms, see
-  // EnterpriseMemoryLimitPrefObserver::PlatformIsSupported for the list of
-  // supported platforms.
-  std::unique_ptr<memory::EnterpriseMemoryLimitPrefObserver>
-      memory_limit_pref_observer_;
-
 #if BUILDFLAG(IS_CHROMEOS_ASH)
   std::unique_ptr<chromeos::memory::SystemMemoryPressureEvaluator>
       cros_evaluator_;
