diff --git a/chrome/browser/resources/settings/privacy_page/BUILD.gn b/chrome/browser/resources/settings/privacy_page/BUILD.gn
index 1dabaa4ed580..6ec85627a6ae 100644
--- a/chrome/browser/resources/settings/privacy_page/BUILD.gn
+++ b/chrome/browser/resources/settings/privacy_page/BUILD.gn
@@ -140,7 +140,6 @@ js_library("personalization_options.m") {
 js_library("privacy_page") {
   deps = [
     ":privacy_page_browser_proxy.m",
-    "..:hats_browser_proxy",
     "..:i18n_setup",
     "..:metrics_browser_proxy",
     "..:page_visibility",
