diff --git a/chrome/browser/content_settings/page_specific_content_settings_delegate.cc b/chrome/browser/content_settings/page_specific_content_settings_delegate.cc
index 40438973bfd32..fafffcaa9235b 100644
--- a/chrome/browser/content_settings/page_specific_content_settings_delegate.cc
+++ b/chrome/browser/content_settings/page_specific_content_settings_delegate.cc
@@ -20,7 +20,6 @@
 #include "chrome/common/renderer_configuration.mojom.h"
 #include "components/content_settings/browser/page_specific_content_settings.h"
 #include "components/permissions/permission_decision_auto_blocker.h"
-#include "components/permissions/permission_uma_util.h"
 #include "components/prefs/pref_service.h"
 #include "content/public/browser/navigation_handle.h"
 #include "content/public/browser/render_process_host.h"
@@ -178,11 +177,13 @@ void PageSpecificContentSettingsDelegate::OnContentAllowed(
       setting_info.primary_pattern, setting_info.secondary_pattern, type);
   if (grant_time.is_null())
     return;
+#if 0
   permissions::PermissionUmaUtil::RecordTimeElapsedBetweenGrantAndUse(
       type, base::Time::Now() - grant_time);
   permissions::PermissionUmaUtil::RecordPermissionUsage(
       type, web_contents()->GetBrowserContext(), web_contents(),
       web_contents()->GetLastCommittedURL());
+#endif
 }
 
 void PageSpecificContentSettingsDelegate::OnContentBlocked(
