diff --git a/chrome/browser/media/media_engagement_contents_observer.cc b/chrome/browser/media/media_engagement_contents_observer.cc
index dc01e6d177d16..fe209d34fb22d 100644
--- a/chrome/browser/media/media_engagement_contents_observer.cc
+++ b/chrome/browser/media/media_engagement_contents_observer.cc
@@ -654,7 +654,5 @@ MediaEngagementContentsObserver::GetOrCreateSession(
           : MediaEngagementSession::RestoreType::kRestored;
 
   return new MediaEngagementSession(
-      service_, origin, restore_type,
-      ukm::ConvertToSourceId(navigation_handle->GetNavigationId(),
-                             ukm::SourceIdType::NAVIGATION_ID));
+      service_, origin, restore_type);
 }
