diff --git a/chrome/browser/devtools/devtools_ui_bindings.h b/chrome/browser/devtools/devtools_ui_bindings.h
index a4987fd55d67..2ea61d656f00 100644
--- a/chrome/browser/devtools/devtools_ui_bindings.h
+++ b/chrome/browser/devtools/devtools_ui_bindings.h
@@ -171,10 +171,6 @@ class DevToolsUIBindings : public DevToolsEmbedderMessageDispatcher::Delegate,
   void SetOpenNewWindowForPopups(bool value) override;
   void RegisterExtensionsAPI(const std::string& origin,
                              const std::string& script) override;
-  void ShowSurvey(DispatchCallback callback,
-                  const std::string& trigger) override;
-  void CanShowSurvey(DispatchCallback callback,
-                     const std::string& trigger) override;
 
   void EnableRemoteDeviceCounter(bool enable);
 
