diff --git a/chrome/browser/ui/passwords/well_known_change_password_navigation_throttle.cc b/chrome/browser/ui/passwords/well_known_change_password_navigation_throttle.cc
index 18c2d580adad..671ca54e67d0 100644
--- a/chrome/browser/ui/passwords/well_known_change_password_navigation_throttle.cc
+++ b/chrome/browser/ui/passwords/well_known_change_password_navigation_throttle.cc
@@ -12,7 +12,6 @@
 #include "components/password_manager/core/browser/site_affiliation/affiliation_service.h"
 #include "components/password_manager/core/browser/well_known_change_password_state.h"
 #include "components/password_manager/core/common/password_manager_features.h"
-#include "components/ukm/content/source_url_recorder.h"
 #include "content/public/browser/browser_context.h"
 #include "content/public/browser/navigation_handle.h"
 #include "content/public/browser/page_navigator.h"
@@ -20,8 +19,6 @@
 #include "content/public/browser/web_contents.h"
 #include "content/public/browser/web_contents_user_data.h"
 #include "net/base/isolation_info.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
 #include "services/network/public/cpp/shared_url_loader_factory.h"
 #include "url/gurl.h"
 #include "url/origin.h"
@@ -83,9 +80,7 @@ WellKnownChangePasswordNavigationThrottle::MaybeCreateThrottleFor(
 
 WellKnownChangePasswordNavigationThrottle::
     WellKnownChangePasswordNavigationThrottle(NavigationHandle* handle)
-    : NavigationThrottle(handle),
-      source_id_(
-          ukm::GetSourceIdForWebContentsDocument(handle->GetWebContents())) {
+    : NavigationThrottle(handle) {
   if (base::FeatureList::IsEnabled(
           password_manager::features::kChangePasswordAffiliationInfo)) {
     affiliation_service_ =
@@ -154,7 +149,6 @@ const char* WellKnownChangePasswordNavigationThrottle::GetNameForLogging() {
 void WellKnownChangePasswordNavigationThrottle::OnProcessingFinished(
     bool is_supported) {
   if (is_supported) {
-    RecordMetric(WellKnownChangePasswordResult::kUsedWellKnownChangePassword);
     Resume();
     return;
   }
@@ -167,10 +161,8 @@ void WellKnownChangePasswordNavigationThrottle::OnProcessingFinished(
     redirect_url = change_password_url_service_->GetChangePasswordUrl(url);
   }
   if (redirect_url.is_valid()) {
-    RecordMetric(WellKnownChangePasswordResult::kFallbackToOverrideUrl);
     Redirect(redirect_url);
   } else {
-    RecordMetric(WellKnownChangePasswordResult::kFallbackToOriginUrl);
     Redirect(url.GetOrigin());
   }
   CancelDeferredNavigation(NavigationThrottle::CANCEL);
@@ -193,10 +185,3 @@ void WellKnownChangePasswordNavigationThrottle::Redirect(const GURL& url) {
       FROM_HERE, base::BindOnce(&WebContentsLifetimeHelper::NavigateTo,
                                 helper->GetWeakPtr(), std::move(params)));
 }
-
-void WellKnownChangePasswordNavigationThrottle::RecordMetric(
-    WellKnownChangePasswordResult result) {
-  ukm::builders::PasswordManager_WellKnownChangePasswordResult(source_id_)
-      .SetWellKnownChangePasswordResult(static_cast<int64_t>(result))
-      .Record(ukm::UkmRecorder::Get());
-}
