diff --git a/chrome/browser/resources/settings/settings_resources_v3.grdp b/chrome/browser/resources/settings/settings_resources_v3.grdp
index ab5f774e81b3..24c3999aee42 100644
--- a/chrome/browser/resources/settings/settings_resources_v3.grdp
+++ b/chrome/browser/resources/settings/settings_resources_v3.grdp
@@ -586,40 +586,6 @@
            file="${root_gen_dir}/chrome/browser/resources/settings/router.m.js"
            use_base_dir="false"
            type="BINDATA" />
-  <include name="IDR_SETTINGS_SAFETY_CHECK_PAGE_SAFETY_CHECK_BROWSER_PROXY_JS"
-           file="safety_check_page/safety_check_browser_proxy.js"
-           type="BINDATA" />
-  <include name="IDR_SETTINGS_SAFETY_CHECK_PAGE_SAFETY_CHECK_CHILD_JS"
-           file="${root_gen_dir}/chrome/browser/resources/settings/safety_check_page/safety_check_child.js"
-           use_base_dir="false"
-           type="BINDATA" />
-  <if expr="is_win">
-    <include name="IDR_SETTINGS_SAFETY_CHECK_PAGE_SAFETY_CHECK_CHROME_CLEANER_CHILD_JS"
-             file="${root_gen_dir}/chrome/browser/resources/settings/safety_check_page/safety_check_chrome_cleaner_child.js"
-             use_base_dir="false"
-             type="BINDATA" />
-  </if>
-  <include name="IDR_SETTINGS_SAFETY_CHECK_PAGE_SAFETY_CHECK_EXTENSIONS_CHILD_JS"
-           file="${root_gen_dir}/chrome/browser/resources/settings/safety_check_page/safety_check_extensions_child.js"
-           use_base_dir="false"
-           type="BINDATA" />
-  <include name="IDR_SETTINGS_SAFETY_CHECK_PAGE_SAFETY_CHECK_PAGE_JS"
-           preprocess="true"
-           file="${root_gen_dir}/chrome/browser/resources/settings/safety_check_page/safety_check_page.js"
-           use_base_dir="false"
-           type="BINDATA" />
-  <include name="IDR_SETTINGS_SAFETY_CHECK_PAGE_SAFETY_CHECK_PASSWORDS_CHILD_JS"
-           file="${root_gen_dir}/chrome/browser/resources/settings/safety_check_page/safety_check_passwords_child.js"
-           use_base_dir="false"
-           type="BINDATA" />
-  <include name="IDR_SETTINGS_SAFETY_CHECK_PAGE_SAFETY_CHECK_SAFE_BROWSING_CHILD_JS"
-           file="${root_gen_dir}/chrome/browser/resources/settings/safety_check_page/safety_check_safe_browsing_child.js"
-           use_base_dir="false"
-           type="BINDATA" />
-  <include name="IDR_SETTINGS_SAFETY_CHECK_PAGE_SAFETY_CHECK_UPDATES_CHILD_JS"
-           file="${root_gen_dir}/chrome/browser/resources/settings/safety_check_page/safety_check_updates_child.js"
-           use_base_dir="false"
-           type="BINDATA" />
   <include name="IDR_SETTINGS_SEARCH_ENGINES_BROWSER_PROXY_M_JS"
            file="${root_gen_dir}/chrome/browser/resources/settings/search_engines_page/search_engines_browser_proxy.m.js"
            use_base_dir="false"
