diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index f86605d537c5..dcb4d3a8c34f 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -162,7 +162,6 @@ static_library("utility") {
 
   if (is_chromeos) {
     deps += [
-      "//chrome/services/file_util",
       "//chrome/services/sharing",
       "//chrome/services/sharing/public/mojom:mojom",
       "//chromeos/assistant:buildflags",
