diff --git a/chrome/browser/plugins/plugin_observer.cc b/chrome/browser/plugins/plugin_observer.cc
index 280652d8285ce..56173ceab36cc 100644
--- a/chrome/browser/plugins/plugin_observer.cc
+++ b/chrome/browser/plugins/plugin_observer.cc
@@ -28,7 +28,6 @@
 #include "components/content_settings/core/browser/host_content_settings_map.h"
 #include "components/download/public/common/download_url_parameters.h"
 #include "components/infobars/content/content_infobar_manager.h"
-#include "components/metrics_services_manager/metrics_services_manager.h"
 #include "content/public/browser/browser_thread.h"
 #include "content/public/browser/child_process_security_policy.h"
 #include "content/public/browser/download_manager.h"
@@ -176,8 +175,6 @@ void PluginObserver::ShowFlashPermissionBubble() {
 }
 
 void PluginObserver::CouldNotLoadPlugin(const base::FilePath& plugin_path) {
-  g_browser_process->GetMetricsServicesManager()->OnPluginLoadingError(
-      plugin_path);
   std::u16string plugin_name =
       PluginService::GetInstance()->GetPluginDisplayNameByPath(plugin_path);
   CreatePluginObserverInfoBar(
