diff --git a/chrome/browser/DEPS b/chrome/browser/DEPS
index 6c6d580631cb..f43ebbb0ca82 100644
--- a/chrome/browser/DEPS
+++ b/chrome/browser/DEPS
@@ -225,14 +225,12 @@ include_rules = [
   "+components/query_parser",
   "+components/query_tiles",
   "+components/quirks",
-  "+components/rappor",
   "+components/reading_list/core",
   "+components/remote_cocoa/app_shim",
   "+components/remote_cocoa/browser",
   "+components/remote_cocoa/common",
   "+components/renderer_context_menu",
   "+components/rlz",
-  "+components/safe_browsing",
   "+components/safe_search_api",
   "+components/schema_org",
   "+components/search",
