diff --git a/chrome/renderer/DEPS b/chrome/renderer/DEPS
index d57c556a4271..ded9980b1245 100644
--- a/chrome/renderer/DEPS
+++ b/chrome/renderer/DEPS
@@ -37,7 +37,6 @@ include_rules = [
   "+components/prerender",
   "+components/printing/common",
   "+components/printing/renderer",
-  "+components/rappor/public/mojom",
   "+components/safe_browsing/buildflags.h",
   "+components/safe_browsing/content/renderer",
   "+components/safe_browsing/content/common",
