diff --git a/chrome/browser/ui/webui/tab_search/tab_search.mojom b/chrome/browser/ui/webui/tab_search/tab_search.mojom
index e2f0d589721f..93193022abc1 100644
--- a/chrome/browser/ui/webui/tab_search/tab_search.mojom
+++ b/chrome/browser/ui/webui/tab_search/tab_search.mojom
@@ -90,9 +90,6 @@ interface PageHandler {
   // Get tab groups for the current profile.
   GetTabGroups() => (TabGroups tab_groups);
 
-  // Show the feedback page.
-  ShowFeedbackPage();
-
   // Switch to a specific tab.
   SwitchToTab(SwitchToTabInfo switch_to_tab_info);
 
