diff --git a/chrome/browser/extensions/api/autofill_private/autofill_private_api.cc b/chrome/browser/extensions/api/autofill_private/autofill_private_api.cc
index 393bdfaa82115..f4e43149b7e0b 100644
--- a/chrome/browser/extensions/api/autofill_private/autofill_private_api.cc
+++ b/chrome/browser/extensions/api/autofill_private/autofill_private_api.cc
@@ -559,7 +559,7 @@ AutofillPrivateMigrateCreditCardsFunction::Run() {
 
 ////////////////////////////////////////////////////////////////////////////////
 // AutofillPrivateLogServerCardLinkClickedFunction
-
+#if 0
 ExtensionFunction::ResponseAction
 AutofillPrivateLogServerCardLinkClickedFunction::Run() {
   autofill::PersonalDataManager* personal_data =
@@ -572,7 +572,7 @@ AutofillPrivateLogServerCardLinkClickedFunction::Run() {
   personal_data->LogServerCardLinkClicked();
   return RespondNow(NoArguments());
 }
-
+#endif
 ////////////////////////////////////////////////////////////////////////////////
 // AutofillPrivateSetCreditCardFIDOAuthEnabledStateFunction
 
