diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 24b1e6d45fd1..23e2d9d3ac3b 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -218,7 +218,6 @@ static_library("test_support") {
     "//components/permissions:test_support",
     "//components/prefs:test_support",
     "//components/rappor:test_support",
-    "//components/safe_browsing/core/db:v4_test_util",
     "//components/search_engines:test_support",
     "//components/sessions:test_support",
     "//components/signin/public/base:test_support",
@@ -273,8 +272,6 @@ static_library("test_support") {
       # Android uses //chrome:chrome_android_core instead of the //chrome/app
       # target.
       "//chrome/app:test_support",
-      "//components/ukm:test_support",
-      "//components/ukm:ukm_test_helper",
       "//components/zoom:test_support",
     ]
     sources += [
@@ -298,7 +295,6 @@ static_library("test_support") {
     ]
     deps += [ "//build:branding_buildflags" ]
     public_deps += [
-      "//components/crash/core/app",
       "//third_party/breakpad",
     ]
   }
@@ -313,7 +309,6 @@ static_library("test_support") {
     ]
     public_deps += [
       "//chrome/install_static/test:test_support",
-      "//components/crash/core/app",
       "//third_party/wtl",
     ]
   }
@@ -506,13 +501,10 @@ if (is_android) {
       "//chrome/android:app_hooks_java",
       "//chrome/browser/metrics:test_support",
       "//chrome/browser/payments:browsertests",
-      "//chrome/browser/privacy_budget:browser_tests",
       "//chrome/browser/profiling_host:profiling_browsertests",
       "//components/browser_ui/widget/android:java",
-      "//components/crash/android:crashpad_main",
       "//components/games/core/test:test_support",
       "//components/metrics:demographics_test_support",
-      "//components/ukm:ukm_test_helper",
 
       # TODO(crbug.com/961849): This is needed for ShellManager which is what
       # the ChromeBrowserTestsActivity is using to build the java UI. It's
@@ -529,7 +521,6 @@ if (is_android) {
       "../browser/engagement/important_sites_util_browsertest.cc",
       "../browser/games/games_service_browsertest.cc",
       "../browser/metrics/metrics_service_user_demographics_browsertest.cc",
-      "../browser/metrics/ukm_browsertest.cc",
       "../browser/net/cert_verify_proc_browsertest.cc",
       "../browser/profiles/profile_browsertest_android.cc",
       "../browser/ssl/chrome_security_state_client_browsertest.cc",
@@ -664,8 +655,6 @@ if (is_android) {
       "//base:base_java_test_support",
       "//chrome/android:chrome_all_java",
       "//chrome/browser/tabmodel:java",
-      "//components/crash/android:handler_java",
-      "//components/crash/android:java",
       "//components/module_installer/android:module_installer_java",
       "//content/public/android:content_java",
       "//content/public/test/android:android_test_message_pump_support_java",
@@ -734,7 +723,6 @@ if (!is_android) {
       "//chrome/browser/metrics:test_support",
       "//chrome/browser/notifications/scheduler/test:test_support",
       "//chrome/browser/payments:browsertests",
-      "//chrome/browser/privacy_budget:browser_tests",
       "//chrome/browser/profiling_host:profiling_browsertests",
       "//chrome/browser/web_applications:browser_tests",
       "//chrome/renderer",
@@ -762,12 +750,9 @@ if (!is_android) {
       "//components/nacl/common:buildflags",
       "//components/offline_items_collection/core/test_support",
       "//components/optimization_guide:test_support",
-      "//components/page_load_metrics/browser:test_support",
-      "//components/page_load_metrics/common:test_support",
       "//components/policy:chrome_settings_proto_generated_compile",
       "//components/resources",
       "//components/safe_browsing:buildflags",
-      "//components/safe_browsing/core/db:test_database_manager",
       "//components/services/language_detection/public/cpp",
       "//components/services/patch/public/mojom",
       "//components/services/quarantine:test_support",
@@ -862,7 +847,6 @@ if (!is_android) {
 
     data = [
       "data/",
-      "//chrome/browser/page_load_metrics/integration_tests/data/",
       "//chrome/common/extensions/docs/examples/apps/calculator/",
       "//components/test/data/arc/",
       "//components/test/data/autofill/",
@@ -995,7 +979,6 @@ if (!is_android) {
       "../browser/download/download_frame_policy_browsertest.cc",
       "../browser/download/download_started_animation_browsertest.cc",
       "../browser/download/save_page_browsertest.cc",
-      "../browser/enterprise/reporting/report_scheduler_browsertest.cc",
       "../browser/fast_shutdown_browsertest.cc",
       "../browser/favicon/content_favicon_driver_browsertest.cc",
       "../browser/federated_learning/floc_id_provider_browsertest.cc",
@@ -1082,8 +1065,6 @@ if (!is_android) {
       "../browser/metrics/process_memory_metrics_emitter_browsertest.cc",
       "../browser/metrics/startup_metrics_browsertest.cc",
       "../browser/metrics/tab_stats_tracker_browsertest.cc",
-      "../browser/metrics/ukm_background_recorder_browsertest.cc",
-      "../browser/metrics/ukm_browsertest.cc",
       "../browser/metrics/variations/force_field_trials_browsertest.cc",
       "../browser/navigation_predictor/navigation_predictor_browsertest.cc",
       "../browser/navigation_predictor/navigation_predictor_preconnect_client_browsertest.cc",
@@ -1142,10 +1123,8 @@ if (!is_android) {
       "../browser/page_load_metrics/observers/back_forward_cache_page_load_metrics_observer_browsertest.cc",
       "../browser/page_load_metrics/observers/data_saver_site_breakdown_metrics_observer_browsertest.cc",
       "../browser/page_load_metrics/observers/data_use_metrics_observer_browsertest.cc",
-      "../browser/page_load_metrics/observers/foreground_duration_ukm_observer_browsertest.cc",
       "../browser/page_load_metrics/observers/https_engagement_metrics/https_engagement_page_load_metrics_observer_browsertest.cc",
       "../browser/page_load_metrics/observers/isolated_prerender_page_load_metrics_observer_browsertest.cc",
-      "../browser/page_load_metrics/observers/javascript_frameworks_ukm_observer_browsertest.cc",
       "../browser/page_load_metrics/observers/live_tab_count_page_load_metrics_observer_browsertest.cc",
       "../browser/page_load_metrics/observers/multi_tab_loading_page_load_metrics_observer_browsertest.cc",
       "../browser/page_load_metrics/observers/resource_metrics_observer_browsertest.cc",
@@ -1250,19 +1229,6 @@ if (!is_android) {
       "../browser/resource_coordinator/discard_before_unload_helper_browsertest.cc",
       "../browser/resource_coordinator/tab_activity_watcher_browsertest.cc",
       "../browser/resource_coordinator/tab_manager_browsertest.cc",
-      "../browser/safe_browsing/chrome_cleaner/reporter_runner_browsertest_win.cc",
-      "../browser/safe_browsing/client_side_detection_host_browsertest.cc",
-      "../browser/safe_browsing/client_side_detection_service_browsertest.cc",
-      "../browser/safe_browsing/cloud_content_scanning/deep_scanning_browsertest_base.cc",
-      "../browser/safe_browsing/cloud_content_scanning/deep_scanning_browsertest_base.h",
-      "../browser/safe_browsing/cloud_content_scanning/deep_scanning_dialog_delegate_browsertest.cc",
-      "../browser/safe_browsing/cloud_content_scanning/deep_scanning_dialog_views_browsertest.cc",
-      "../browser/safe_browsing/cloud_content_scanning/deep_scanning_test_utils.cc",
-      "../browser/safe_browsing/cloud_content_scanning/deep_scanning_test_utils.h",
-      "../browser/safe_browsing/download_protection/deep_scanning_browsertest.cc",
-      "../browser/safe_browsing/download_protection/download_protection_service_browsertest.cc",
-      "../browser/safe_browsing/test_safe_browsing_database_helper.cc",
-      "../browser/safe_browsing/test_safe_browsing_database_helper.h",
       "../browser/safe_xml_parser_browsertest.cc",
       "../browser/search/local_ntp_navigation_browsertest.cc",
       "../browser/search/ntp_custom_background_enabled_policy_handler_browsertest.cc",
@@ -1431,8 +1397,6 @@ if (!is_android) {
       "../browser/ui/update_chrome_dialog_browsertest.cc",
       "../browser/ui/views/apps/app_info_dialog/app_info_dialog_views_browsertest.cc",
       "../browser/ui/views/bookmarks/bookmark_bar_view_browsertest.cc",
-      "../browser/ui/views/chrome_cleaner_dialog_browsertest_win.cc",
-      "../browser/ui/views/chrome_cleaner_reboot_dialog_browsertest_win.cc",
       "../browser/ui/views/content_setting_bubble_contents_browsertest.cc",
       "../browser/ui/views/content_test_utils.cc",
       "../browser/ui/views/content_test_utils.h",
@@ -1504,7 +1468,6 @@ if (!is_android) {
       "../browser/ui/webui/webui_load_timer_browsertest.cc",
       "../browser/ui/webui/webui_webview_browsertest.cc",
       "../browser/ui/zoom/zoom_controller_browsertest.cc",
-      "../browser/ukm_worker_browsertest.cc",
       "../browser/unload_browsertest.cc",
       "../browser/usb/usb_browsertest.cc",
       "../browser/wake_lock/wake_lock_browsertest.cc",
@@ -1623,7 +1586,6 @@ if (!is_android) {
         "//chrome/browser/media/router:test_support",
         "//chrome/browser/resources/chromeos/accessibility:browser_tests",
         "//chrome/browser/ui/ash/holding_space:browser_tests",
-        "//chrome/services/file_util/public/cpp:browser_tests",
         "//chromeos:test_support",
         "//chromeos/components/drivefs:test_support",
         "//chromeos/components/media_app_ui:browser_test_support",
@@ -1963,7 +1925,6 @@ if (!is_android) {
         "../browser/extensions/external_install_error_browsertest.cc",
         "../browser/extensions/fetch_apitest.cc",
         "../browser/extensions/file_iframe_apitest.cc",
-        "../browser/extensions/google_docs_offline_ukm_browsertest.cc",
         "../browser/extensions/gpu_browsertest.cc",
         "../browser/extensions/identifiability_metrics_test_util.cc",
         "../browser/extensions/identifiability_metrics_test_util.h",
@@ -2006,16 +1967,6 @@ if (!is_android) {
         "../browser/policy/extension_force_install_mixin.cc",
         "../browser/policy/extension_force_install_mixin.h",
         "../browser/policy/extension_policy_browsertest.cc",
-        "../browser/safe_browsing/chrome_cleaner/chrome_cleaner_dialog_controller_impl_browsertest_win.cc",
-        "../browser/safe_browsing/chrome_cleaner/chrome_cleaner_reboot_dialog_controller_impl_browsertest_win.cc",
-        "../browser/safe_browsing/chrome_cleaner/mock_chrome_cleaner_controller_win.cc",
-        "../browser/safe_browsing/chrome_cleaner/mock_chrome_cleaner_controller_win.h",
-        "../browser/safe_browsing/chrome_cleaner/settings_resetter_browsertest_win.cc",
-        "../browser/safe_browsing/settings_reset_prompt/default_settings_fetcher_browsertest.cc",
-        "../browser/safe_browsing/settings_reset_prompt/settings_reset_dependency_browsertest_win.cc",
-        "../browser/safe_browsing/settings_reset_prompt/settings_reset_prompt_model_browsertest_win.cc",
-        "../browser/safe_browsing/settings_reset_prompt/settings_reset_prompt_test_utils.cc",
-        "../browser/safe_browsing/settings_reset_prompt/settings_reset_prompt_test_utils.h",
         "../browser/ui/views/extensions/extension_dialog_browsertest.cc",
         "../browser/ui/views/web_apps/pwa_confirmation_bubble_view_browsertest.cc",
         "../browser/ui/views/web_apps/web_app_confirmation_view_browsertest.cc",
@@ -2753,11 +2704,8 @@ if (!is_android) {
         "../browser/ui/webui/signin/user_manager_ui_browsertest.cc",
 
         # chromeos does not support machine level user cloud policies
-        "../browser/enterprise/reporting/report_scheduler_browsertest.cc",
         "../browser/policy/cloud/chrome_browser_cloud_management_browsertest.cc",
       ]
-
-      deps += [ "//components/crash/content/browser/error_reporting:mock_crash_endpoint" ]
     } else {  # !is_chromeos
       sources -= [
         "../browser/invalidation/profile_invalidation_provider_factory_browsertest.cc",
@@ -3489,7 +3437,6 @@ test("unit_tests") {
     "../browser/page_load_metrics/observers/page_load_metrics_observer_test_harness.cc",
     "../browser/page_load_metrics/observers/page_load_metrics_observer_test_harness.h",
     "../browser/page_load_metrics/observers/previews_page_load_metrics_observer_unittest.cc",
-    "../browser/page_load_metrics/observers/previews_ukm_observer_unittest.cc",
     "../browser/page_load_metrics/observers/protocol_page_load_metrics_observer_unittest.cc",
     "../browser/page_load_metrics/observers/scheme_page_load_metrics_observer_unittest.cc",
     "../browser/page_load_metrics/observers/security_state_page_load_metrics_observer_unittest.cc",
@@ -3498,7 +3445,6 @@ test("unit_tests") {
     "../browser/page_load_metrics/observers/tab_restore_page_load_metrics_observer_unittest.cc",
     "../browser/page_load_metrics/observers/third_party_metrics_observer_unittest.cc",
     "../browser/page_load_metrics/observers/translate_page_load_metrics_observer_unittest.cc",
-    "../browser/page_load_metrics/observers/ukm_page_load_metrics_observer_unittest.cc",
     "../browser/paint_preview/services/paint_preview_demo_service_unittest.cc",
     "../browser/paint_preview/services/paint_preview_tab_service_unittest.cc",
     "../browser/password_manager/chrome_password_manager_client_unittest.cc",
@@ -3742,7 +3688,6 @@ test("unit_tests") {
     "../common/component_flash_hint_file_linux_unittest.cc",
     "../common/conflicts/module_watcher_win_unittest.cc",
     "../common/conflicts/remote_module_watcher_win_unittest.cc",
-    "../common/crash_keys_unittest.cc",
     "../common/google_url_loader_throttle_unittest.cc",
     "../common/ini_parser_unittest.cc",
     "../common/mac/mock_launchd.h",
@@ -3953,7 +3898,7 @@ test("unit_tests") {
   }
   if (is_win) {
     data_deps += [ "//chrome" ]
-    data += [ "$root_out_dir/chrome.exe" ]
+    data += [ "$root_out_dir/dumb.exe" ]
   }
 
   defines = []
@@ -3972,11 +3917,9 @@ test("unit_tests") {
     "//chrome/browser/media/router:unittests",
     "//chrome/browser/notifications:unit_tests",
     "//chrome/browser/payments:unittests",
-    "//chrome/browser/privacy_budget:unit_tests",
     "//chrome/browser/updates/announcement_notification:unit_tests",
     "//chrome/browser/video_tutorials:unit_tests",
     "//chrome/common:test_support",
-    "//chrome/common/privacy_budget:unit_tests",
     "//chrome/services/machine_learning:unit_tests",
     "//chrome/services/machine_learning/public/cpp:test_support",
     "//components/account_id",
@@ -4006,10 +3949,6 @@ test("unit_tests") {
     "//components/query_tiles:unit_tests",
     "//components/resources",
     "//components/safe_browsing:buildflags",
-    "//components/safe_browsing/content/password_protection:mock_password_protection",
-    "//components/safe_browsing/core:features",
-    "//components/safe_browsing/core/db",
-    "//components/safe_browsing/core/db:test_database_manager",
     "//components/safe_search_api:test_support",
     "//components/schema_org/common:improved_mojom",
     "//components/services/patch/content",
@@ -4021,7 +3960,6 @@ test("unit_tests") {
     "//components/sync_device_info:test_support",
     "//components/sync_sessions:test_support",
     "//components/sync_user_events:test_support",
-    "//components/ukm/content",
     "//components/version_info:generate_version_info",
     "//content/app/resources",
     "//content/public/app",
@@ -4216,16 +4154,6 @@ test("unit_tests") {
       "../browser/enterprise/connectors/enterprise_connectors_policy_handler_unittest.cc",
       "../browser/enterprise/connectors/reporting_service_settings_unittest.cc",
       "../browser/enterprise/connectors/service_provider_config_unittest.cc",
-      "../browser/enterprise/reporting/browser_report_generator_unittest.cc",
-      "../browser/enterprise/reporting/extension_info_unittest.cc",
-      "../browser/enterprise/reporting/notification/extension_request_notification_unittest.cc",
-      "../browser/enterprise/reporting/notification/extension_request_observer_factory_unittest.cc",
-      "../browser/enterprise/reporting/notification/extension_request_observer_unittest.cc",
-      "../browser/enterprise/reporting/policy_info_unittest.cc",
-      "../browser/enterprise/reporting/profile_report_generator_unittest.cc",
-      "../browser/enterprise/reporting/report_generator_unittest.cc",
-      "../browser/enterprise/reporting/report_request_queue_generator_unittest.cc",
-      "../browser/enterprise/reporting/report_scheduler_desktop_unittest.cc",
       "../browser/first_run/first_run_unittest.cc",
       "../browser/font_family_cache_unittest.cc",
 
@@ -4316,15 +4244,6 @@ test("unit_tests") {
       "../browser/profile_resetter/reset_report_uploader_unittest.cc",
       "../browser/profile_resetter/triggered_profile_resetter_win_unittest.cc",
       "../browser/renderer_context_menu/render_view_context_menu_unittest.cc",
-      "../browser/safe_browsing/chrome_cleaner/chrome_cleaner_controller_impl_win_unittest.cc",
-      "../browser/safe_browsing/chrome_cleaner/chrome_cleaner_fetcher_win_unittest.cc",
-      "../browser/safe_browsing/chrome_cleaner/chrome_cleaner_runner_win_unittest.cc",
-      "../browser/safe_browsing/chrome_cleaner/chrome_prompt_channel_win_unittest.cc",
-      "../browser/safe_browsing/chrome_cleaner/mock_chrome_cleaner_process_win.cc",
-      "../browser/safe_browsing/chrome_cleaner/mock_chrome_cleaner_process_win.h",
-      "../browser/safe_browsing/chrome_cleaner/reporter_runner_win_unittest.cc",
-      "../browser/safe_browsing/chrome_cleaner/srt_delete_extension_win_unittest.cc",
-      "../browser/safe_browsing/chrome_cleaner/srt_field_trial_win_unittest.cc",
       "../browser/search/background/ntp_background_service_unittest.cc",
       "../browser/search/chrome_colors/chrome_colors_service_unittest.cc",
       "../browser/search/instant_service_unittest.cc",
@@ -4479,7 +4398,6 @@ test("unit_tests") {
       "../browser/ui/webui/settings/profile_info_handler_unittest.cc",
       "../browser/ui/webui/settings/recent_site_settings_helper_unittest.cc",
       "../browser/ui/webui/settings/reset_settings_handler_unittest.cc",
-      "../browser/ui/webui/settings/safety_check_handler_unittest.cc",
       "../browser/ui/webui/settings/settings_cookies_view_handler_unittest.cc",
       "../browser/ui/webui/settings/settings_manage_profile_handler_unittest.cc",
       "../browser/ui/webui/settings/site_settings_handler_unittest.cc",
@@ -4544,25 +4462,19 @@ test("unit_tests") {
       "//chrome/services/sharing/public/cpp",
       "//chrome/services/sharing/public/cpp:unit_tests",
       "//chrome/services/speech:unit_tests",
-      "//components/chrome_cleaner/test:test_name_helper",
       "//components/enterprise",
       "//components/enterprise:test_support",
       "//components/feature_engagement/test:test_support",
-      "//components/safety_check:test_support",
       "//components/send_tab_to_self:test_support",
       "//components/services/app_service:unit_tests",
       "//components/services/app_service/public/cpp:icon_loader_test_support",
       "//components/services/app_service/public/cpp:unit_tests",
       "//components/signin/public/base:signin_buildflags",
       "//components/sync:test_support",
-      "//services/metrics/public/cpp:ukm_builders",
       "//third_party/libaddressinput",
       "//ui/base/idle:test_support",
       "//ui/native_theme:test_support",
     ]
-    if (is_win) {
-      deps += [ "//components/chrome_cleaner/public/proto:test_only_proto" ]
-    }
     if (is_mac) {
       deps += [ ":firefox_importer_interface" ]
     }
@@ -4685,7 +4597,6 @@ test("unit_tests") {
     sources += [
       "../browser/component_updater/cros_component_installer_chromeos_unittest.cc",
       "../browser/component_updater/metadata_table_chromeos_unittest.cc",
-      "../browser/enterprise/reporting/android_app_info_generator_unittest.cc",
       "../browser/extensions/api/terminal/crostini_startup_status_unittest.cc",
       "../browser/google/google_brand_code_map_chromeos_unittest.cc",
       "../browser/media/webrtc/desktop_media_list_ash_unittest.cc",
@@ -5251,7 +5162,6 @@ test("unit_tests") {
 
       # This will add all of the unit tests for the schema compiler to this
       # target.
-      "//third_party/blink/common/privacy_budget:test_support",
       "//tools/json_schema_compiler/test:unit_tests",
     ]
     if (enable_service_discovery) {
@@ -5356,12 +5266,6 @@ test("unit_tests") {
       "../browser/safe_browsing/threat_details_unittest.cc",
       "../browser/safe_browsing/ui_manager_unittest.cc",
     ]
-
-    # TODO(crbug/1056278): Enable this on Fuchsia
-    if (!is_fuchsia) {
-      deps +=
-          [ "//components/safe_browsing/core:file_type_policies_test_support" ]
-    }
   }
 
   if (safe_browsing_mode == 1) {
@@ -5439,13 +5343,6 @@ test("unit_tests") {
     deps += [
       ":test_proto",
       "../common/safe_browsing:mock_binary_feature_extractor",
-      "//chrome/services/file_util/public/cpp:unit_tests",
-      "//components/safe_browsing/content/renderer:websocket_sb_handshake_throttle_unittest",
-      "//components/safe_browsing/content/triggers:ad_redirect_trigger",
-      "//components/safe_browsing/core:ping_manager_unittest",
-      "//components/safe_browsing/core/browser:safe_browsing_url_checker_unittest",
-      "//components/safe_browsing/core/browser:token_fetcher_unittest",
-      "//components/safe_browsing/core/db:v4_test_util",
     ]
   } else if (safe_browsing_mode == 2) {
     sources += [ "../browser/safe_browsing/telemetry/android/android_telemetry_service_unittest.cc" ]
@@ -6239,8 +6136,6 @@ if (!is_android) {
       "//components/sync",
       "//components/sync:test_support_model",
       "//components/translate/content/browser:test_support",
-      "//components/ukm:test_support",
-      "//components/ukm:ukm_test_helper",
       "//content/app/resources",
       "//content/test:test_support",
       "//crypto:platform",
@@ -6252,7 +6147,6 @@ if (!is_android) {
       "//net",
       "//net:net_resources",
       "//net:test_support",
-      "//services/metrics/public/cpp:ukm_builders",
       "//skia",
       "//testing/gmock",
       "//testing/gtest",
@@ -6505,7 +6399,6 @@ test("chrome_app_unittests") {
     "//chrome/child",
     "//chrome/common/profiler",
     "//chrome/gpu",
-    "//components/crash/core/common",
     "//components/flags_ui:switches",
     "//components/gwp_asan/buildflags",
     "//components/heap_profiling/in_process",
@@ -7244,7 +7137,7 @@ if (is_win) {
     # the //chrome target to bundle up both initial/chrome.exe and chrome.exe.
     # As a workaround, explicitly include a data dep on just chrome.exe, and
     # add //chrome to deps above to make sure it's been built.
-    data = [ "$root_out_dir/chrome.exe" ]
+    data = [ "$root_out_dir/dumb.exe" ]
 
     # Don't want the test-specific dependencies to affect load tests.
     # In particular, a few system DLLs cause user32 to be loaded, which is bad.
