diff --git a/chrome/browser/ui/passwords/bubble_controllers/items_bubble_controller.cc b/chrome/browser/ui/passwords/bubble_controllers/items_bubble_controller.cc
index 7f2d2f801a8c4..048a177897267 100644
--- a/chrome/browser/ui/passwords/bubble_controllers/items_bubble_controller.cc
+++ b/chrome/browser/ui/passwords/bubble_controllers/items_bubble_controller.cc
@@ -11,7 +11,6 @@
 #include "chrome/grit/generated_resources.h"
 #include "components/favicon/core/favicon_util.h"
 #include "components/password_manager/core/browser/password_form.h"
-#include "components/password_manager/core/browser/password_form_metrics_recorder.h"
 #include "components/password_manager/core/browser/password_store.h"
 #include "content/public/browser/web_contents.h"
 #include "ui/base/l10n/l10n_util.h"
@@ -93,10 +92,6 @@ void ItemsBubbleController::OnFaviconReady(
 }
 
 void ItemsBubbleController::ReportInteractions() {
-  metrics_util::LogGeneralUIDismissalReason(dismissal_reason_);
-  // Record UKM statistics on dismissal reason.
-  if (metrics_recorder_)
-    metrics_recorder_->RecordUIDismissalReason(dismissal_reason_);
 }
 
 std::u16string ItemsBubbleController::GetTitle() const {
