diff --git a/chrome/browser/safe_browsing/chrome_password_protection_service.cc b/chrome/browser/safe_browsing/chrome_password_protection_service.cc
index c6034a9ef3c9..a91e59a7687e 100644
--- a/chrome/browser/safe_browsing/chrome_password_protection_service.cc
+++ b/chrome/browser/safe_browsing/chrome_password_protection_service.cc
@@ -256,11 +256,6 @@ ChromePasswordProtectionService::ChromePasswordProtectionService(
       base::BindRepeating(
           &ChromePasswordProtectionService::OnWarningTriggerChanged,
           base::Unretained(this)));
-  pref_change_registrar_->Add(
-      prefs::kPasswordProtectionLoginURLs,
-      base::BindRepeating(
-          &ChromePasswordProtectionService::OnEnterprisePasswordUrlChanged,
-          base::Unretained(this)));
   pref_change_registrar_->Add(
       prefs::kPasswordProtectionChangePasswordURL,
       base::BindRepeating(
