diff --git a/chrome/browser/chrome_browser_field_trials.cc b/chrome/browser/chrome_browser_field_trials.cc
index 63a2aeadf005..ec81f35ec2ad 100644
--- a/chrome/browser/chrome_browser_field_trials.cc
+++ b/chrome/browser/chrome_browser_field_trials.cc
@@ -114,9 +114,11 @@ void ChromeBrowserFieldTrials::RegisterSyntheticTrials() {
 }
 
 void ChromeBrowserFieldTrials::InstantiateDynamicTrials() {
+#if 0
   // Persistent histograms must be enabled as soon as possible.
   base::FilePath metrics_dir;
   if (base::PathService::Get(chrome::DIR_USER_DATA, &metrics_dir)) {
     InstantiatePersistentHistograms(metrics_dir);
   }
+#endif
 }
