diff --git a/chrome/common/chrome_features.cc b/chrome/common/chrome_features.cc
index 501bce9096f88..a416aa353e11a 100644
--- a/chrome/common/chrome_features.cc
+++ b/chrome/common/chrome_features.cc
@@ -913,7 +913,7 @@ const base::Feature kTPMFirmwareUpdate{"TPMFirmwareUpdate",
 #if !defined(OS_ANDROID)
 // Enables logging UKMs for background tab activity by TabActivityWatcher.
 const base::Feature kTabMetricsLogging{"TabMetricsLogging",
-                                       base::FEATURE_ENABLED_BY_DEFAULT};
+                                       base::FEATURE_DISABLED_BY_DEFAULT};
 #endif
 
 // Enables the teamfood flags.
