diff --git a/chrome/browser/renderer_context_menu/render_view_context_menu.h b/chrome/browser/renderer_context_menu/render_view_context_menu.h
index 209983847a2c..e9ac4d051f83 100644
--- a/chrome/browser/renderer_context_menu/render_view_context_menu.h
+++ b/chrome/browser/renderer_context_menu/render_view_context_menu.h
@@ -196,7 +196,9 @@ class RenderViewContextMenu : public RenderViewContextMenuBase {
   void AppendPasswordItems();
   void AppendPictureInPictureItem();
   void AppendSharingItems();
+#if BUILDFLAG(ENABLE_CLICK_TO_CALL)
   void AppendClickToCallItem();
+#endif
   void AppendSharedClipboardItem();
   void AppendQRCodeGeneratorItem(bool for_image, bool draw_icon);
 
@@ -304,9 +306,11 @@ class RenderViewContextMenu : public RenderViewContextMenuBase {
   std::unique_ptr<send_tab_to_self::SendTabToSelfSubMenuModel>
       send_tab_to_self_sub_menu_model_;
 
+#if BUILDFLAG(ENABLE_CLICK_TO_CALL)
   // Click to call menu observer.
   std::unique_ptr<ClickToCallContextMenuObserver>
       click_to_call_context_menu_observer_;
+#endif
 
   // Shared clipboard menu observer.
   std::unique_ptr<SharedClipboardContextMenuObserver>
