diff --git a/chrome/browser/resources/new_tab_page/lazy_load.js b/chrome/browser/resources/new_tab_page/lazy_load.js
index 9fb298dff49ae..ea1adc9107d30 100644
--- a/chrome/browser/resources/new_tab_page/lazy_load.js
+++ b/chrome/browser/resources/new_tab_page/lazy_load.js
@@ -11,6 +11,5 @@
  */
 
 import './customize_dialog.js';
-import './middle_slot_promo.js';
 import './most_visited.js';
 import './voice_search_overlay.js';
