diff --git a/chrome/browser/lite_video/lite_video_decider.cc b/chrome/browser/lite_video/lite_video_decider.cc
index d67996aece55..134d06d044c3 100644
--- a/chrome/browser/lite_video/lite_video_decider.cc
+++ b/chrome/browser/lite_video/lite_video_decider.cc
@@ -35,8 +35,7 @@ class ScopedLiteVideoDecisionRecorder {
   explicit ScopedLiteVideoDecisionRecorder(
       lite_video::LiteVideoBlocklistReason blocklist_reason,
       bool is_mainframe)
-      : blocklist_reason_(blocklist_reason),
-        is_mainframe_(is_mainframe),
+      : is_mainframe_(is_mainframe),
         has_hint_for_host_(false) {}
   ~ScopedLiteVideoDecisionRecorder() {
     if (is_mainframe_) {
@@ -56,7 +55,6 @@ class ScopedLiteVideoDecisionRecorder {
   }
 
  private:
-  lite_video::LiteVideoBlocklistReason blocklist_reason_;
   bool is_mainframe_;
   bool has_hint_for_host_;
 };
