diff --git a/chrome/browser/navigation_predictor/navigation_predictor.h b/chrome/browser/navigation_predictor/navigation_predictor.h
index 0e056e2093efc..a67fe3bfa3080 100644
--- a/chrome/browser/navigation_predictor/navigation_predictor.h
+++ b/chrome/browser/navigation_predictor/navigation_predictor.h
@@ -15,8 +15,6 @@
 #include "content/public/browser/frame_service_base.h"
 #include "content/public/browser/visibility.h"
 #include "mojo/public/cpp/bindings/pending_receiver.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "third_party/abseil-cpp/absl/types/optional.h"
 #include "third_party/blink/public/mojom/loader/navigation_predictor.mojom.h"
 #include "ui/gfx/geometry/size.h"
@@ -88,12 +86,6 @@ class NavigationPredictor
   // URLs that were sent to the prediction service.
   std::set<GURL> predicted_urls_;
 
-  // UKM ID for navigation
-  ukm::SourceId ukm_source_id_;
-
-  // UKM recorder
-  ukm::UkmRecorder* ukm_recorder_ = nullptr;
-
   // The time at which the navigation started.
   base::TimeTicks navigation_start_;
 
