diff --git a/chrome/browser/resources/settings/basic_page/BUILD.gn b/chrome/browser/resources/settings/basic_page/BUILD.gn
index dcb01001d62e..66c73151a13e 100644
--- a/chrome/browser/resources/settings/basic_page/BUILD.gn
+++ b/chrome/browser/resources/settings/basic_page/BUILD.gn
@@ -21,7 +21,6 @@ js_library("basic_page") {
     "../chrome_cleanup_page:chrome_cleanup_proxy",
     "../prefs:prefs_behavior.m",
     "../privacy_page:privacy_page",
-    "../safety_check_page:safety_check_page",
     "../search_page",
     "../settings_page:main_page_behavior",
     "//ui/webui/resources/js:load_time_data.m",
