diff --git a/chrome/browser/extensions/webstore_installer.h b/chrome/browser/extensions/webstore_installer.h
index ea5709b47409..249a4e3b1c3d 100644
--- a/chrome/browser/extensions/webstore_installer.h
+++ b/chrome/browser/extensions/webstore_installer.h
@@ -258,9 +258,6 @@ class WebstoreInstaller : public content::NotificationObserver,
   // PendingInstall.
   void ReportSuccess();
 
-  // Records stats regarding an interrupted webstore download item.
-  void RecordInterrupt(const download::DownloadItem* download) const;
-
   content::NotificationRegistrar registrar_;
   ScopedObserver<ExtensionRegistry, ExtensionRegistryObserver>
       extension_registry_observer_{this};
