diff --git a/chrome/browser/resource_coordinator/tab_helper.cc b/chrome/browser/resource_coordinator/tab_helper.cc
index 7e157ea8f732b..eefdb83529913 100644
--- a/chrome/browser/resource_coordinator/tab_helper.cc
+++ b/chrome/browser/resource_coordinator/tab_helper.cc
@@ -16,7 +16,6 @@
 #include "chrome/browser/resource_coordinator/lifecycle_unit_state.mojom-shared.h"
 #include "chrome/browser/resource_coordinator/resource_coordinator_parts.h"
 #include "chrome/browser/resource_coordinator/tab_load_tracker.h"
-#include "chrome/browser/resource_coordinator/tab_memory_metrics_reporter.h"
 #include "chrome/browser/resource_coordinator/utils.h"
 #include "content/public/browser/navigation_handle.h"
 #include "content/public/browser/render_frame_host.h"
@@ -33,12 +32,6 @@ ResourceCoordinatorTabHelper::ResourceCoordinatorTabHelper(
     content::WebContents* web_contents)
     : content::WebContentsObserver(web_contents) {
   TabLoadTracker::Get()->StartTracking(web_contents);
-  if (memory_instrumentation::MemoryInstrumentation::GetInstance()) {
-    auto* rc_parts = g_browser_process->resource_coordinator_parts();
-    DCHECK(rc_parts);
-    rc_parts->tab_memory_metrics_reporter()->StartReporting(
-        TabLoadTracker::Get());
-  }
 }
 
 ResourceCoordinatorTabHelper::~ResourceCoordinatorTabHelper() = default;
@@ -77,11 +70,6 @@ void ResourceCoordinatorTabHelper::DidFinishNavigation(
       navigation_handle->IsSameDocument()) {
     return;
   }
-
-  if (navigation_handle->IsInMainFrame()) {
-    ukm_source_id_ = ukm::ConvertToSourceId(
-        navigation_handle->GetNavigationId(), ukm::SourceIdType::NAVIGATION_ID);
-  }
 }
 
 WEB_CONTENTS_USER_DATA_KEY_IMPL(ResourceCoordinatorTabHelper)
