diff --git a/chrome/browser/extensions/api/enterprise_reporting_private/prefs.h b/chrome/browser/extensions/api/enterprise_reporting_private/prefs.h
index 1896ca3622de..fb23d164cef5 100644
--- a/chrome/browser/extensions/api/enterprise_reporting_private/prefs.h
+++ b/chrome/browser/extensions/api/enterprise_reporting_private/prefs.h
@@ -27,9 +27,6 @@ extern const char kReportUserIDData[];
 // Controls reporting of Chrome extensions and plugins data.
 extern const char kReportExtensionsAndPluginsData[];
 
-// Controls reporting of Safe browsing data.
-extern const char kReportSafeBrowsingData[];
-
 void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry);
 
 }  // namespace enterprise_reporting
