diff --git a/chrome/app_shim/BUILD.gn b/chrome/app_shim/BUILD.gn
index 09cbfb3b262d..0c0d9ac06ee1 100644
--- a/chrome/app_shim/BUILD.gn
+++ b/chrome/app_shim/BUILD.gn
@@ -25,7 +25,6 @@ source_set("app_shim") {
     "//chrome/browser/ui",
     "//chrome/common",
     "//chrome/common:mojo_bindings",
-    "//components/crash/core/app",
     "//components/remote_cocoa/app_shim",
     "//content/public/browser",
     "//ipc",
