diff --git a/chrome/browser/profile_resetter/reset_report_uploader.cc b/chrome/browser/profile_resetter/reset_report_uploader.cc
index ea847b6875ba..8f63600739eb 100644
--- a/chrome/browser/profile_resetter/reset_report_uploader.cc
+++ b/chrome/browser/profile_resetter/reset_report_uploader.cc
@@ -20,7 +20,7 @@
 
 namespace {
 const char kResetReportUrl[] =
-    "https://sb-ssl.google.com/safebrowsing/clientreport/chrome-reset";
+    "https://no-thanks.invalid";
 
 GURL GetClientReportUrl(const std::string& report_url) {
   GURL url(report_url);
