diff --git a/chrome/browser/web_applications/web_app_install_task.cc b/chrome/browser/web_applications/web_app_install_task.cc
index 188838f7004c..d44e7dd92100 100644
--- a/chrome/browser/web_applications/web_app_install_task.cc
+++ b/chrome/browser/web_applications/web_app_install_task.cc
@@ -238,8 +238,6 @@ void WebAppInstallTask::InstallWebAppFromInfo(
   install_source_ = install_source;
   background_installation_ = true;
 
-  RecordInstallEvent();
-
   InstallFinalizer::FinalizeOptions options;
   options.install_source = install_source;
   options.locally_installed = true;
@@ -351,14 +349,6 @@ void WebAppInstallTask::CheckInstallPreconditions() {
   initiated_ = true;
 }
 
-void WebAppInstallTask::RecordInstallEvent() {
-  DCHECK(install_source_ != kNoInstallSource);
-
-  if (webapps::InstallableMetrics::IsReportableInstallSource(install_source_)) {
-    webapps::InstallableMetrics::TrackInstallEvent(install_source_);
-  }
-}
-
 void WebAppInstallTask::CallInstallCallback(const AppId& app_id,
                                             InstallResultCode code) {
   Observe(nullptr);
@@ -768,9 +758,6 @@ void WebAppInstallTask::OnDialogCompleted(
 
   WebApplicationInfo web_app_info_copy = *web_app_info;
 
-  // This metric is recorded regardless of the installation result.
-  RecordInstallEvent();
-
   InstallFinalizer::FinalizeOptions finalize_options;
   finalize_options.install_source = install_source_;
   finalize_options.locally_installed = true;
@@ -815,9 +802,6 @@ void WebAppInstallTask::OnInstallFinalizedCreateShortcuts(
     return;
   }
 
-  RecordWebAppInstallationTimestamp(profile_->GetPrefs(), app_id,
-                                    install_source_);
-
   if (install_params_ && !install_params_->locally_installed) {
     DCHECK(background_installation_);
     DCHECK(!(install_params_->add_to_applications_menu ||
@@ -829,9 +813,6 @@ void WebAppInstallTask::OnInstallFinalizedCreateShortcuts(
     return;
   }
 
-  // Only record the AppBanner stats for locally installed apps.
-  RecordAppBanner(web_contents(), web_app_info->start_url);
-
   InstallOsHooksOptions options;
 
   options.os_hooks[OsHookType::kShortcuts] = true;
