diff --git a/chrome/browser/vr/BUILD.gn b/chrome/browser/vr/BUILD.gn
index fa8a80897475..1d8a7591eea6 100644
--- a/chrome/browser/vr/BUILD.gn
+++ b/chrome/browser/vr/BUILD.gn
@@ -271,7 +271,6 @@ component("vr_common") {
     "//chrome/common:constants",
     "//components/omnibox/browser",
     "//components/omnibox/browser",
-    "//components/rappor/public:public",
     "//components/ukm/content",
     "//content/public/browser",
     "//device/base",
@@ -401,7 +400,6 @@ component("vr_base") {
   deps = [
     "//base",
     "//chrome/app:generated_resources",
-    "//components/rappor:rappor",
     "//components/security_state/core",
     "//components/ukm/content",
     "//components/url_formatter",
