diff --git a/chrome/renderer/chrome_render_frame_observer.cc b/chrome/renderer/chrome_render_frame_observer.cc
index 3ea7900b949e..0883afe625ba 100644
--- a/chrome/renderer/chrome_render_frame_observer.cc
+++ b/chrome/renderer/chrome_render_frame_observer.cc
@@ -192,7 +192,7 @@ void ChromeRenderFrameObserver::DidFinishLoad() {
   // Don't do anything for subframes.
   if (frame->Parent())
     return;
-
+#if 0
   GURL osdd_url = frame->GetDocument().OpenSearchDescriptionURL();
   if (!osdd_url.is_empty()) {
     mojo::AssociatedRemote<chrome::mojom::OpenSearchDescriptionDocumentHandler>
@@ -202,6 +202,7 @@ void ChromeRenderFrameObserver::DidFinishLoad() {
     osdd_handler->PageHasOpenSearchDescriptionDocument(
         frame->GetDocument().Url(), osdd_url);
   }
+#endif
 }
 
 void ChromeRenderFrameObserver::DidCreateNewDocument() {
