diff --git a/chrome/browser/chrome_browser_main.h b/chrome/browser/chrome_browser_main.h
index d5d8c1736133c..4712b5a283be0 100644
--- a/chrome/browser/chrome_browser_main.h
+++ b/chrome/browser/chrome_browser_main.h
@@ -30,7 +30,6 @@ class StartupData;
 class PrefService;
 class Profile;
 class StartupBrowserCreator;
-class ShutdownWatcherHelper;
 class WebUsbDetector;
 
 namespace base {
@@ -101,16 +100,6 @@ class ChromeBrowserMainParts : public content::BrowserMainParts {
  private:
   friend class ChromeBrowserMainPartsTestApi;
 
-  // Constructs the metrics service and initializes metrics recording.
-  void SetupMetrics();
-
-  // Starts recording of metrics. This can only be called after we have a file
-  // thread.
-  static void StartMetricsRecording();
-
-  // Record time from process startup to present time in an UMA histogram.
-  void RecordBrowserStartupTime();
-
   // Reads origin trial policy data from local state and configures command line
   // for child processes.
   void SetupOriginTrialsCommandLine(PrefService* local_state);
@@ -141,11 +130,6 @@ class ChromeBrowserMainParts : public content::BrowserMainParts {
   int result_code_;
 
 #if !defined(OS_ANDROID)
-  // Create ShutdownWatcherHelper object for watching jank during shutdown.
-  // Please keep |shutdown_watcher| as the first object constructed, and hence
-  // it is destroyed last.
-  std::unique_ptr<ShutdownWatcherHelper> shutdown_watcher_;
-
   std::unique_ptr<WebUsbDetector> web_usb_detector_;
 #endif  // !defined(OS_ANDROID)
 
