diff --git a/chrome/browser/content_settings/sound_content_setting_observer.cc b/chrome/browser/content_settings/sound_content_setting_observer.cc
index 6a089fe525b4..1f61af95960b 100644
--- a/chrome/browser/content_settings/sound_content_setting_observer.cc
+++ b/chrome/browser/content_settings/sound_content_setting_observer.cc
@@ -27,7 +27,7 @@
 
 SoundContentSettingObserver::SoundContentSettingObserver(
     content::WebContents* contents)
-    : content::WebContentsObserver(contents), logged_site_muted_ukm_(false) {
+    : content::WebContentsObserver(contents) {
   Profile* profile =
       Profile::FromBrowserContext(web_contents()->GetBrowserContext());
   host_content_settings_map_ =
@@ -87,7 +87,6 @@ void SoundContentSettingObserver::DidFinishNavigation(
   if (navigation_handle->IsInMainFrame() && navigation_handle->HasCommitted() &&
       !navigation_handle->IsSameDocument()) {
     MuteOrUnmuteIfNecessary();
-    logged_site_muted_ukm_ = false;
   }
 }
 
@@ -126,6 +125,10 @@ void SoundContentSettingObserver::MuteOrUnmuteIfNecessary() {
 
   TabMutedReason reason = chrome::GetTabAudioMutedReason(web_contents());
 
+  // Do not unmute if we're muted due to audio indicator.
+  if (!mute && reason == TabMutedReason::AUDIO_INDICATOR)
+    return;
+
   // Do not override the decisions of an extension.
   if (reason == TabMutedReason::EXTENSION)
     return;
@@ -160,23 +163,9 @@ void SoundContentSettingObserver::CheckSoundBlocked(bool is_audible) {
             web_contents()->GetMainFrame());
     if (settings)
       settings->OnAudioBlocked();
-
-    RecordSiteMutedUKM();
   }
 }
 
-void SoundContentSettingObserver::RecordSiteMutedUKM() {
-  // We only want to log 1 event per navigation.
-  if (logged_site_muted_ukm_)
-    return;
-  logged_site_muted_ukm_ = true;
-
-  ukm::builders::Media_SiteMuted(
-      ukm::GetSourceIdForWebContentsDocument(web_contents()))
-      .SetMuteReason(GetSiteMutedReason())
-      .Record(ukm::UkmRecorder::Get());
-}
-
 SoundContentSettingObserver::MuteReason
 SoundContentSettingObserver::GetSiteMutedReason() {
   const GURL url = web_contents()->GetLastCommittedURL();
