diff --git a/chrome/browser/resources/settings/privacy_page/personalization_options.js b/chrome/browser/resources/settings/privacy_page/personalization_options.js
index e8f221291df7..17b71654b8e6 100644
--- a/chrome/browser/resources/settings/privacy_page/personalization_options.js
+++ b/chrome/browser/resources/settings/privacy_page/personalization_options.js
@@ -161,17 +161,6 @@ Polymer({
   },
   // </if>
 
-  /**
-   * @return {boolean}
-   * @private
-   */
-  showSpellCheckControl_() {
-    return (
-        !!this.prefs.spellcheck &&
-        /** @type {!Array<string>} */
-        (this.prefs.spellcheck.dictionaries.value).length > 0);
-  },
-
   /**
    * @return {boolean}
    * @private
