diff --git a/chrome/tools/build/win/FILES.cfg b/chrome/tools/build/win/FILES.cfg
index a809889cec11..675ee3903736 100644
--- a/chrome/tools/build/win/FILES.cfg
+++ b/chrome/tools/build/win/FILES.cfg
@@ -807,12 +807,6 @@ FILES = [
     'archive': 'metrics-metadata.zip',
     'optional': ['dev', 'official'],
   },
-  {
-    'filename': 'rappor.xml',
-    'buildtype': ['dev', 'official'],
-    'archive': 'metrics-metadata.zip',
-    'optional': ['dev', 'official'],
-  },
   {
     'filename': 'ukm.xml',
     'buildtype': ['dev', 'official'],
