diff --git a/chrome/browser/metrics/upgrade_metrics_provider.cc b/chrome/browser/metrics/upgrade_metrics_provider.cc
index cd8c1166ccc32..ceaeec0438933 100644
--- a/chrome/browser/metrics/upgrade_metrics_provider.cc
+++ b/chrome/browser/metrics/upgrade_metrics_provider.cc
@@ -5,7 +5,6 @@
 #include "chrome/browser/metrics/upgrade_metrics_provider.h"
 
 #include "base/metrics/histogram_macros.h"
-#include "chrome/browser/upgrade_detector/upgrade_detector.h"
 
 UpgradeMetricsProvider::UpgradeMetricsProvider() {}
 
@@ -13,8 +12,4 @@ UpgradeMetricsProvider::~UpgradeMetricsProvider() {}
 
 void UpgradeMetricsProvider::ProvideCurrentSessionData(
     metrics::ChromeUserMetricsExtension* uma_proto) {
-  UpgradeDetector* upgrade_detector = UpgradeDetector::GetInstance();
-  UMA_HISTOGRAM_ENUMERATION("UpgradeDetector.NotificationStage",
-                            upgrade_detector->upgrade_notification_stage(),
-                            UpgradeDetector::UPGRADE_ANNOYANCE_MAX_VALUE + 1);
 }
