diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index 19a8c138c42e..20442fcf5330 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -127,9 +127,7 @@ static_library("test_support") {
     "//chrome/renderer",
     "//chrome/utility",
     "//components/gwp_asan/buildflags",
-    "//components/heap_profiling/in_process",
     "//components/nacl/common:buildflags",
-    "//components/startup_metric_utils/browser",
     "//content/public/app",
     "//content/public/common",
     "//pdf:pdf_ppapi",
