diff --git a/chrome/browser/autocomplete/chrome_autocomplete_provider_client.h b/chrome/browser/autocomplete/chrome_autocomplete_provider_client.h
index 01b59b5757e99..4e14f7008a85c 100644
--- a/chrome/browser/autocomplete/chrome_autocomplete_provider_client.h
+++ b/chrome/browser/autocomplete/chrome_autocomplete_provider_client.h
@@ -26,10 +26,6 @@ class StoragePartition;
 class WebContents;
 }
 
-namespace unified_consent {
-class UrlKeyedDataCollectionConsentHelper;
-}
-
 class ChromeAutocompleteProviderClient : public AutocompleteProviderClient {
  public:
   explicit ChromeAutocompleteProviderClient(Profile* profile);
@@ -123,8 +119,6 @@ class ChromeAutocompleteProviderClient : public AutocompleteProviderClient {
   Profile* profile_;
   ChromeAutocompleteSchemeClassifier scheme_classifier_;
   std::unique_ptr<OmniboxPedalProvider> pedal_provider_;
-  std::unique_ptr<unified_consent::UrlKeyedDataCollectionConsentHelper>
-      url_consent_helper_;
 
   // The |most_visited_sites_| is created upon request. It is created at
   // most once by requesting in MostVisitedSitesProvider when the page
