diff --git a/chrome/browser/ui/passwords/bubble_controllers/auto_sign_in_bubble_controller.cc b/chrome/browser/ui/passwords/bubble_controllers/auto_sign_in_bubble_controller.cc
index e77c3936e07ab..faa366cfd7376 100644
--- a/chrome/browser/ui/passwords/bubble_controllers/auto_sign_in_bubble_controller.cc
+++ b/chrome/browser/ui/passwords/bubble_controllers/auto_sign_in_bubble_controller.cc
@@ -5,7 +5,6 @@
 #include "chrome/browser/ui/passwords/bubble_controllers/auto_sign_in_bubble_controller.h"
 
 #include "chrome/browser/ui/passwords/passwords_model_delegate.h"
-#include "components/password_manager/core/browser/password_form_metrics_recorder.h"
 
 namespace metrics_util = password_manager::metrics_util;
 
@@ -32,8 +31,4 @@ std::u16string AutoSignInBubbleController::GetTitle() const {
 }
 
 void AutoSignInBubbleController::ReportInteractions() {
-  metrics_util::LogGeneralUIDismissalReason(dismissal_reason_);
-  // Record UKM statistics on dismissal reason.
-  if (metrics_recorder_)
-    metrics_recorder_->RecordUIDismissalReason(dismissal_reason_);
 }
