diff --git a/chrome/browser/lookalikes/lookalike_url_blocking_page.h b/chrome/browser/lookalikes/lookalike_url_blocking_page.h
index d7281e955b24..6d205a6c6589 100644
--- a/chrome/browser/lookalikes/lookalike_url_blocking_page.h
+++ b/chrome/browser/lookalikes/lookalike_url_blocking_page.h
@@ -12,7 +12,6 @@
 #include "base/time/time.h"
 #include "components/lookalikes/core/lookalike_url_util.h"
 #include "components/security_interstitials/content/security_interstitial_page.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 
 class GURL;
 
@@ -30,7 +29,6 @@ class LookalikeUrlBlockingPage
       content::WebContents* web_contents,
       const GURL& safe_url,
       const GURL& request_url,
-      ukm::SourceId source_id,
       LookalikeUrlMatchType match_type,
       bool is_signed_exchange,
       std::unique_ptr<
@@ -60,7 +58,6 @@ class LookalikeUrlBlockingPage
   // The URL suggested to the user as the safe URL. Can be empty, in which case
   // the default action on the interstitial takes the user to the new tab page.
   const GURL safe_url_;
-  ukm::SourceId source_id_;
   LookalikeUrlMatchType match_type_;
   // True if the throttle encountered a response with
   // is_signed_exchange_inner_response flag. Only checked in tests.
