diff --git a/chrome/browser/ui/passwords/bubble_controllers/save_unsynced_credentials_locally_bubble_controller.cc b/chrome/browser/ui/passwords/bubble_controllers/save_unsynced_credentials_locally_bubble_controller.cc
index b200d271339e..f5e1126c0979 100644
--- a/chrome/browser/ui/passwords/bubble_controllers/save_unsynced_credentials_locally_bubble_controller.cc
+++ b/chrome/browser/ui/passwords/bubble_controllers/save_unsynced_credentials_locally_bubble_controller.cc
@@ -9,7 +9,6 @@
 #include "chrome/browser/ui/passwords/passwords_model_delegate.h"
 #include "chrome/grit/generated_resources.h"
 #include "components/password_manager/core/browser/password_form.h"
-#include "components/password_manager/core/browser/password_form_metrics_recorder.h"
 #include "ui/base/l10n/l10n_util.h"
 
 namespace metrics_util = password_manager::metrics_util;
@@ -21,7 +20,6 @@ SaveUnsyncedCredentialsLocallyBubbleController::
           std::move(delegate),
           /*display_disposition=*/metrics_util::
               AUTOMATIC_SAVE_UNSYNCED_CREDENTIALS_LOCALLY),
-      dismissal_reason_(metrics_util::NO_DIRECT_INTERACTION),
       unsynced_credentials_(delegate_->GetUnsyncedCredentials()) {}
 
 SaveUnsyncedCredentialsLocallyBubbleController::
@@ -46,10 +44,6 @@ void SaveUnsyncedCredentialsLocallyBubbleController::OnCancelClicked() {
 }
 
 void SaveUnsyncedCredentialsLocallyBubbleController::ReportInteractions() {
-  metrics_util::LogGeneralUIDismissalReason(dismissal_reason_);
-  // Record UKM statistics on dismissal reason.
-  if (metrics_recorder_)
-    metrics_recorder_->RecordUIDismissalReason(dismissal_reason_);
 }
 
 base::string16 SaveUnsyncedCredentialsLocallyBubbleController::GetTitle()
