diff --git a/chrome/renderer/extensions/chrome_extensions_renderer_client.h b/chrome/renderer/extensions/chrome_extensions_renderer_client.h
index 90ebb2c3859d..9dd5abd0dbec 100644
--- a/chrome/renderer/extensions/chrome_extensions_renderer_client.h
+++ b/chrome/renderer/extensions/chrome_extensions_renderer_client.h
@@ -48,10 +48,6 @@ namespace url {
 class Origin;
 }
 
-namespace ukm {
-class MojoUkmRecorder;
-}
-
 namespace v8 {
 class Isolate;
 template <typename T>
@@ -127,7 +123,6 @@ class ChromeExtensionsRendererClient
   }
 
  private:
-  std::unique_ptr<ukm::MojoUkmRecorder> ukm_recorder_;
   std::unique_ptr<extensions::Dispatcher> extension_dispatcher_;
   std::unique_ptr<extensions::RendererPermissionsPolicyDelegate>
       permissions_policy_delegate_;
