diff --git a/chrome/browser/ui/views/login_handler_views.cc b/chrome/browser/ui/views/login_handler_views.cc
index 3dc6e20391f6a..6a3f73d309b7d 100644
--- a/chrome/browser/ui/views/login_handler_views.cc
+++ b/chrome/browser/ui/views/login_handler_views.cc
@@ -37,7 +37,6 @@ class LoginHandlerViews : public LoginHandler {
                      web_contents,
                      std::move(auth_required_callback)),
         popunder_preventer_(std::make_unique<PopunderPreventer>(web_contents)) {
-    RecordDialogCreation(DialogIdentifier::LOGIN_HANDLER);
   }
 
   ~LoginHandlerViews() override {
