diff --git a/chrome/browser/extensions/extension_service.h b/chrome/browser/extensions/extension_service.h
index 376fb4111ba8e..8d57af19204d8 100644
--- a/chrome/browser/extensions/extension_service.h
+++ b/chrome/browser/extensions/extension_service.h
@@ -22,12 +22,10 @@
 #include "chrome/browser/extensions/blocklist.h"
 #include "chrome/browser/extensions/extension_allowlist.h"
 #include "chrome/browser/extensions/extension_management.h"
-#include "chrome/browser/extensions/forced_extensions/force_installed_metrics.h"
 #include "chrome/browser/extensions/forced_extensions/force_installed_tracker.h"
 #include "chrome/browser/extensions/install_gate.h"
 #include "chrome/browser/extensions/omaha_attributes_handler.h"
 #include "chrome/browser/extensions/pending_extension_manager.h"
-#include "chrome/browser/extensions/safe_browsing_verdict_handler.h"
 #include "chrome/browser/profiles/profile_manager.h"
 #include "chrome/browser/profiles/profile_manager_observer.h"
 #include "chrome/browser/upgrade_detector/upgrade_observer.h"
@@ -626,8 +624,6 @@ class ExtensionService : public ExtensionServiceInterface,
 
   ExtensionAllowlist allowlist_;
 
-  SafeBrowsingVerdictHandler safe_browsing_verdict_handler_;
-
   OmahaAttributesHandler omaha_attributes_handler_;
 
   // Sets of enabled/disabled/terminated/blocklisted extensions. Not owned.
@@ -715,8 +711,10 @@ class ExtensionService : public ExtensionServiceInterface,
   // Tracker of enterprise policy forced installation.
   ForceInstalledTracker force_installed_tracker_;
 
+#if 0
   // Reports force-installed extension metrics to UMA.
   ForceInstalledMetrics force_installed_metrics_;
+#endif
 
   base::ScopedObservation<ProfileManager, ProfileManagerObserver>
       profile_manager_observation_{this};
