diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 24b1e6d45fd1..ccca1195cf42 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -509,7 +509,6 @@ if (is_android) {
       "//chrome/browser/privacy_budget:browser_tests",
       "//chrome/browser/profiling_host:profiling_browsertests",
       "//components/browser_ui/widget/android:java",
-      "//components/crash/android:crashpad_main",
       "//components/games/core/test:test_support",
       "//components/metrics:demographics_test_support",
       "//components/ukm:ukm_test_helper",
@@ -3953,7 +3952,7 @@ test("unit_tests") {
   }
   if (is_win) {
     data_deps += [ "//chrome" ]
-    data += [ "$root_out_dir/chrome.exe" ]
+    data += [ "$root_out_dir/dumb.exe" ]
   }
 
   defines = []
@@ -7244,7 +7243,7 @@ if (is_win) {
     # the //chrome target to bundle up both initial/chrome.exe and chrome.exe.
     # As a workaround, explicitly include a data dep on just chrome.exe, and
     # add //chrome to deps above to make sure it's been built.
-    data = [ "$root_out_dir/chrome.exe" ]
+    data = [ "$root_out_dir/dumb.exe" ]
 
     # Don't want the test-specific dependencies to affect load tests.
     # In particular, a few system DLLs cause user32 to be loaded, which is bad.
