diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index 8892138ef63f..4e2210e19c02 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -201,7 +201,6 @@ static_library("common") {
     "//components/policy/core/common",
     "//components/prefs",
     "//components/safe_browsing:buildflags",
-    "//components/safe_browsing/core/web_ui:constants",
     "//components/services/heap_profiling/public/cpp",
     "//components/strings",
     "//components/translate/content/common",
@@ -427,7 +426,6 @@ static_library("common") {
     ]
     deps += [
       "//chrome/chrome_elf:chrome_elf_main_include",
-      "//chrome/common/win:eventlog_messages",
       "//components/crash/core/app:crash_export_thunk_include",
     ]
 
@@ -761,10 +759,6 @@ mojom("mojo_bindings") {
     public_deps += [ "//components/remote_cocoa/common:mojo" ]
   }
 
-  if (safe_browsing_mode == 1) {
-    public_deps += [ "//chrome/services/file_util/public/mojom" ]
-  }
-
   overridden_deps = [
     "//content/public/common:interfaces",
     "//third_party/blink/public/mojom:mojom_platform",
