diff --git a/chrome/browser/lifetime/browser_shutdown.cc b/chrome/browser/lifetime/browser_shutdown.cc
index 3e25f91823c6..1f47a9a9cb08 100644
--- a/chrome/browser/lifetime/browser_shutdown.cc
+++ b/chrome/browser/lifetime/browser_shutdown.cc
@@ -35,8 +35,6 @@
 #include "chrome/common/chrome_paths.h"
 #include "chrome/common/chrome_switches.h"
 #include "chrome/common/pref_names.h"
-#include "components/crash/core/common/crash_key.h"
-#include "components/metrics/metrics_service.h"
 #include "components/prefs/pref_registry_simple.h"
 #include "components/prefs/pref_service.h"
 #include "components/tracing/common/tracing_switches.h"
@@ -93,31 +91,6 @@ ShutdownType g_shutdown_type = ShutdownType::kNotValid;
 int g_shutdown_num_processes;
 int g_shutdown_num_processes_slow;
 
-constexpr char kShutdownMsFile[] = "chrome_shutdown_ms.txt";
-
-base::FilePath GetShutdownMsPath() {
-  base::FilePath shutdown_ms_file;
-  base::PathService::Get(chrome::DIR_USER_DATA, &shutdown_ms_file);
-  return shutdown_ms_file.AppendASCII(kShutdownMsFile);
-}
-
-const char* ToShutdownTypeString(ShutdownType type) {
-  switch (type) {
-    case ShutdownType::kNotValid:
-      NOTREACHED();
-      break;
-    case ShutdownType::kWindowClose:
-      return "close";
-    case ShutdownType::kBrowserExit:
-      return "exit";
-    case ShutdownType::kEndSession:
-      return "end";
-    case ShutdownType::kSilentExit:
-      return "silent_exit";
-  }
-  return "";
-}
-
 }  // namespace
 
 void RegisterPrefs(PrefRegistrySimple* registry) {
@@ -132,9 +105,6 @@ void OnShutdownStarting(ShutdownType type) {
   if (g_shutdown_type != ShutdownType::kNotValid)
     return;
 
-  static crash_reporter::CrashKeyString<8> shutdown_type_key("shutdown-type");
-  shutdown_type_key.Set(ToShutdownTypeString(type));
-
   g_shutdown_type = type;
   // For now, we're only counting the number of renderer processes
   // since we can't safely count the number of plugin processes from this
@@ -239,10 +209,6 @@ bool ShutdownPreThreadsStop() {
   // consider putting it in BrowserProcessImpl::EndSession.
   PrefService* prefs = g_browser_process->local_state();
 
-  metrics::MetricsService* metrics = g_browser_process->metrics_service();
-  if (metrics)
-    metrics->RecordCompletedSessionEnd();
-
   bool restart_last_session = RecordShutdownInfoPrefs();
 
   prefs->CommitPendingWrite();
@@ -258,16 +224,6 @@ bool ShutdownPreThreadsStop() {
 
 bool RecordShutdownInfoPrefs() {
   PrefService* prefs = g_browser_process->local_state();
-  if (g_shutdown_type != ShutdownType::kNotValid &&
-      g_shutdown_num_processes > 0) {
-    // Record the shutdown info so that we can put it into a histogram at next
-    // startup.
-    prefs->SetInteger(prefs::kShutdownType, static_cast<int>(g_shutdown_type));
-    prefs->SetInteger(prefs::kShutdownNumProcesses, g_shutdown_num_processes);
-    prefs->SetInteger(prefs::kShutdownNumProcessesSlow,
-                      g_shutdown_num_processes_slow);
-  }
-
   // Check local state for the restart flag so we can restart the session later.
   bool restart_last_session = false;
   if (prefs->HasPrefPath(prefs::kRestartLastSessionOnShutdown)) {
@@ -339,102 +295,12 @@ void ShutdownPostThreadsStop(RestartMode restart_mode) {
 #endif  // BUILDFLAG(IS_CHROMEOS_ASH)
   }
 
-  if (g_shutdown_type != ShutdownType::kNotValid &&
-      g_shutdown_num_processes > 0) {
-    // Measure total shutdown time as late in the process as possible
-    // and then write it to a file to be read at startup.
-    // We can't use prefs since all services are shutdown at this point.
-    TimeDelta shutdown_delta = base::Time::Now() - *g_shutdown_started;
-    std::string shutdown_ms =
-        base::NumberToString(shutdown_delta.InMilliseconds());
-    int len = static_cast<int>(shutdown_ms.length()) + 1;
-    base::FilePath shutdown_ms_file = GetShutdownMsPath();
-    // Note: ReadLastShutdownFile() is done as a BLOCK_SHUTDOWN task so there's
-    // an implicit sequencing between it and this write which happens after
-    // threads have been stopped (and thus ThreadPoolInstance::Shutdown() is
-    // complete).
-    base::WriteFile(shutdown_ms_file, shutdown_ms.c_str(), len);
-  }
-
 #if BUILDFLAG(IS_CHROMEOS_ASH)
   NotifyAndTerminate(false /* fast_path */);
 #endif
 }
 #endif  // !defined(OS_ANDROID)
 
-void ReadLastShutdownFile(ShutdownType type,
-                          int num_procs,
-                          int num_procs_slow) {
-  base::ScopedBlockingCall scoped_blocking_call(FROM_HERE,
-                                                base::BlockingType::MAY_BLOCK);
-
-  base::FilePath shutdown_ms_file = GetShutdownMsPath();
-  std::string shutdown_ms_str;
-  int64_t shutdown_ms = 0;
-  if (base::ReadFileToString(shutdown_ms_file, &shutdown_ms_str))
-    base::StringToInt64(shutdown_ms_str, &shutdown_ms);
-  base::DeleteFile(shutdown_ms_file);
-
-  if (shutdown_ms == 0 || num_procs == 0)
-    return;
-
-  const char* time2_metric_name = nullptr;
-  const char* per_proc_metric_name = nullptr;
-
-  switch (type) {
-    case ShutdownType::kNotValid:
-    case ShutdownType::kSilentExit:
-      // The histograms below have expired, so do not record metrics for silent
-      // exits; see https://crbug.com/975118.
-      break;
-
-    case ShutdownType::kWindowClose:
-      time2_metric_name = "Shutdown.window_close.time2";
-      per_proc_metric_name = "Shutdown.window_close.time_per_process";
-      break;
-
-    case ShutdownType::kBrowserExit:
-      time2_metric_name = "Shutdown.browser_exit.time2";
-      per_proc_metric_name = "Shutdown.browser_exit.time_per_process";
-      break;
-
-    case ShutdownType::kEndSession:
-      time2_metric_name = "Shutdown.end_session.time2";
-      per_proc_metric_name = "Shutdown.end_session.time_per_process";
-      break;
-  }
-  if (!time2_metric_name)
-    return;
-
-  base::UmaHistogramMediumTimes(time2_metric_name,
-                                TimeDelta::FromMilliseconds(shutdown_ms));
-  base::UmaHistogramTimes(per_proc_metric_name,
-                          TimeDelta::FromMilliseconds(shutdown_ms / num_procs));
-  base::UmaHistogramCounts100("Shutdown.renderers.total", num_procs);
-  base::UmaHistogramCounts100("Shutdown.renderers.slow", num_procs_slow);
-}
-
-void ReadLastShutdownInfo() {
-  PrefService* prefs = g_browser_process->local_state();
-  ShutdownType type =
-      static_cast<ShutdownType>(prefs->GetInteger(prefs::kShutdownType));
-  int num_procs = prefs->GetInteger(prefs::kShutdownNumProcesses);
-  int num_procs_slow = prefs->GetInteger(prefs::kShutdownNumProcessesSlow);
-  // clear the prefs immediately so we don't pick them up on a future run
-  prefs->SetInteger(prefs::kShutdownType,
-                    static_cast<int>(ShutdownType::kNotValid));
-  prefs->SetInteger(prefs::kShutdownNumProcesses, 0);
-  prefs->SetInteger(prefs::kShutdownNumProcessesSlow, 0);
-
-  base::UmaHistogramEnumeration("Shutdown.ShutdownType", type);
-
-  base::ThreadPool::PostTask(
-      FROM_HERE,
-      {base::MayBlock(), base::TaskPriority::BEST_EFFORT,
-       base::TaskShutdownBehavior::BLOCK_SHUTDOWN},
-      base::BindOnce(&ReadLastShutdownFile, type, num_procs, num_procs_slow));
-}
-
 void SetTryingToQuit(bool quitting) {
   g_trying_to_quit = quitting;
 
