diff --git a/chrome/browser/ui/passwords/passwords_model_delegate_mock.h b/chrome/browser/ui/passwords/passwords_model_delegate_mock.h
index 11e4207e7e9d..57c1e63062e8 100644
--- a/chrome/browser/ui/passwords/passwords_model_delegate_mock.h
+++ b/chrome/browser/ui/passwords/passwords_model_delegate_mock.h
@@ -22,10 +22,6 @@ class PasswordsModelDelegateMock
   ~PasswordsModelDelegateMock() override;
 
   MOCK_METHOD(content::WebContents*, GetWebContents, (), (const override));
-  MOCK_METHOD(password_manager::PasswordFormMetricsRecorder*,
-              GetPasswordFormMetricsRecorder,
-              (),
-              (override));
   MOCK_METHOD(password_manager::PasswordFeatureManager*,
               GetPasswordFeatureManager,
               (),
