diff --git a/chrome/browser/ui/page_info/chrome_page_info_delegate.cc b/chrome/browser/ui/page_info/chrome_page_info_delegate.cc
index 41f5af52a4fc9..15cd3b0d1defb 100644
--- a/chrome/browser/ui/page_info/chrome_page_info_delegate.cc
+++ b/chrome/browser/ui/page_info/chrome_page_info_delegate.cc
@@ -13,10 +13,8 @@
 #include "chrome/browser/permissions/permission_decision_auto_blocker_factory.h"
 #include "chrome/browser/permissions/permission_manager_factory.h"
 #include "chrome/browser/profiles/profile.h"
-#include "chrome/browser/safe_browsing/chrome_password_protection_service.h"
 #include "chrome/browser/ssl/security_state_tab_helper.h"
 #include "chrome/browser/ssl/stateful_ssl_host_state_delegate_factory.h"
-#include "chrome/browser/subresource_filter/subresource_filter_profile_context_factory.h"
 #include "chrome/browser/usb/usb_chooser_context.h"
 #include "chrome/browser/usb/usb_chooser_context_factory.h"
 #include "chrome/browser/vr/vr_tab_helper.h"
@@ -26,8 +24,6 @@
 #include "components/permissions/permission_manager.h"
 #include "components/permissions/permission_result.h"
 #include "components/security_interstitials/content/stateful_ssl_host_state_delegate.h"
-#include "components/subresource_filter/content/browser/subresource_filter_content_settings_manager.h"
-#include "components/subresource_filter/content/browser/subresource_filter_profile_context.h"
 #include "content/public/browser/web_contents.h"
 #include "content/public/common/content_features.h"
 
@@ -181,7 +177,6 @@ void ChromePageInfoDelegate::OpenConnectionHelpCenterPage(
 }
 
 void ChromePageInfoDelegate::OpenSafetyTipHelpCenterPage() {
-  OpenHelpCenterFromSafetyTip(web_contents_);
 }
 #endif
 
@@ -199,16 +194,6 @@ HostContentSettingsMap* ChromePageInfoDelegate::GetContentSettings() {
   return HostContentSettingsMapFactory::GetForProfile(GetProfile());
 }
 
-bool ChromePageInfoDelegate::IsSubresourceFilterActivated(
-    const GURL& site_url) {
-  subresource_filter::SubresourceFilterContentSettingsManager*
-      settings_manager =
-          SubresourceFilterProfileContextFactory::GetForProfile(GetProfile())
-              ->settings_manager();
-
-  return settings_manager->GetSiteActivationFromMetadata(site_url);
-}
-
 bool ChromePageInfoDelegate::IsContentDisplayedInVrHeadset() {
   return vr::VrTabHelper::IsContentDisplayedInHeadset(web_contents_);
 }
