diff --git a/chrome/common/features.gni b/chrome/common/features.gni
index 2ff23e50e1a7..4b632ab4ead2 100644
--- a/chrome/common/features.gni
+++ b/chrome/common/features.gni
@@ -84,6 +84,5 @@ chrome_grit_defines = [
   "enable_tab_search=$enable_tab_search",
   "enable_vr=$enable_vr",
   "enable_webui_tab_strip=$enable_webui_tab_strip",
-  "safe_browsing_mode=$safe_browsing_mode",
   "optimize_webui=$optimize_webui",
 ]
