diff --git a/chrome/browser/extensions/api/autofill_private/autofill_private_api.cc b/chrome/browser/extensions/api/autofill_private/autofill_private_api.cc
index c3ba3fa17bc3e..3de38697200c5 100644
--- a/chrome/browser/extensions/api/autofill_private/autofill_private_api.cc
+++ b/chrome/browser/extensions/api/autofill_private/autofill_private_api.cc
@@ -494,7 +494,7 @@ AutofillPrivateMigrateCreditCardsFunction::Run() {
 
 ////////////////////////////////////////////////////////////////////////////////
 // AutofillPrivateLogServerCardLinkClickedFunction
-
+#if 0
 ExtensionFunction::ResponseAction
 AutofillPrivateLogServerCardLinkClickedFunction::Run() {
   autofill::PersonalDataManager* personal_data =
@@ -507,7 +507,7 @@ AutofillPrivateLogServerCardLinkClickedFunction::Run() {
   personal_data->LogServerCardLinkClicked();
   return RespondNow(NoArguments());
 }
-
+#endif
 ////////////////////////////////////////////////////////////////////////////////
 // AutofillPrivateSetCreditCardFIDOAuthEnabledStateFunction
 
