diff --git a/chrome/browser/enterprise/connectors/connectors_service.cc b/chrome/browser/enterprise/connectors/connectors_service.cc
index a682d6a5cde8f..06c1b6e9bc8cb 100644
--- a/chrome/browser/enterprise/connectors/connectors_service.cc
+++ b/chrome/browser/enterprise/connectors/connectors_service.cc
@@ -31,7 +31,6 @@
 #include "components/policy/core/common/cloud/machine_level_user_cloud_policy_manager.h"
 #include "components/policy/core/common/cloud/user_cloud_policy_manager.h"
 #include "components/policy/core/common/policy_types.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "components/signin/public/identity_manager/consent_level.h"
 #include "components/signin/public/identity_manager/identity_manager.h"
 #include "components/user_prefs/user_prefs.h"
@@ -345,6 +344,7 @@ std::vector<std::string> ConnectorsService::GetAnalysisServiceProviderNames(
 
 base::Optional<std::string> ConnectorsService::GetDMTokenForRealTimeUrlCheck()
     const {
+#if 0
   if (!ConnectorsEnabled())
     return base::nullopt;
 
@@ -359,9 +359,11 @@ base::Optional<std::string> ConnectorsService::GetDMTokenForRealTimeUrlCheck()
 
   if (dm_token.has_value())
     return dm_token.value().value;
+#endif
   return base::nullopt;
 }
 
+#if 0
 safe_browsing::EnterpriseRealTimeUrlCheckMode
 ConnectorsService::GetAppliedRealTimeUrlCheck() const {
   if (!ConnectorsEnabled() ||
@@ -374,6 +376,7 @@ ConnectorsService::GetAppliedRealTimeUrlCheck() const {
       Profile::FromBrowserContext(context_)->GetPrefs()->GetInteger(
           prefs::kSafeBrowsingEnterpriseRealTimeUrlCheckMode));
 }
+#endif
 
 ConnectorsManager* ConnectorsService::ConnectorsManagerForTesting() {
   return connectors_manager_.get();
