diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 20eb220156efa..3895c0fb22684 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -4434,7 +4434,6 @@ test("unit_tests") {
     "//chrome/common:test_support",
     "//chrome/common/privacy_budget:unit_tests",
     "//chrome/common/profiler:unit_tests",
-    "//chrome/services/file_util:unit_tests",
     "//components/account_id",
     "//components/autofill/content/renderer:test_support",
     "//components/browsing_data/content:test_support",
