diff --git a/chrome/browser/chrome_browser_main_linux.cc b/chrome/browser/chrome_browser_main_linux.cc
index 441a46a27eb5..257789c6fb74 100644
--- a/chrome/browser/chrome_browser_main_linux.cc
+++ b/chrome/browser/chrome_browser_main_linux.cc
@@ -97,8 +97,6 @@ void ChromeBrowserMainPartsLinux::PostProfileInit() {
   } else {
     breakpad_registered = breakpad::IsCrashReporterEnabled();
   }
-  g_browser_process->metrics_service()->RecordBreakpadRegistration(
-      breakpad_registered);
 }
 
 void ChromeBrowserMainPartsLinux::PostMainMessageLoopStart() {
