diff --git a/chrome/browser/download/download_ui_controller.cc b/chrome/browser/download/download_ui_controller.cc
index 4635ab24e3c53..14ab959383cab 100644
--- a/chrome/browser/download/download_ui_controller.cc
+++ b/chrome/browser/download/download_ui_controller.cc
@@ -163,12 +163,6 @@ void DownloadUIController::OnDownloadCreated(content::DownloadManager* manager,
     }
   }
 
-  if (web_contents) {
-    // TODO(crbug.com/1179196): Add test for this metric.
-    RecordDownloadStartPerProfileType(
-        Profile::FromBrowserContext(web_contents->GetBrowserContext()));
-  }
-
   // SavePackage downloads are created in a state where they can be shown in the
   // browser. Call OnDownloadUpdated() once to notify the UI immediately.
   OnDownloadUpdated(manager, item);
