diff --git a/chrome/browser/predictors/navigation_id.h b/chrome/browser/predictors/navigation_id.h
index a6097c19b14de..e99bba7699e31 100644
--- a/chrome/browser/predictors/navigation_id.h
+++ b/chrome/browser/predictors/navigation_id.h
@@ -10,7 +10,6 @@
 #include "base/feature_list.h"
 #include "base/time/time.h"
 #include "components/sessions/core/session_id.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "url/gurl.h"
 
 namespace content {
@@ -24,7 +23,6 @@ struct NavigationID {
   NavigationID();
   explicit NavigationID(content::WebContents* web_contents);
   NavigationID(content::WebContents* web_contents,
-               ukm::SourceId ukm_source_id,
                const GURL& main_frame_url,
                const base::TimeTicks& creation_time);
   NavigationID(const NavigationID& other);
@@ -37,7 +35,6 @@ struct NavigationID {
   bool is_valid() const;
 
   SessionID tab_id;
-  ukm::SourceId ukm_source_id;
   GURL main_frame_url;
 
   // NOTE: Even though we store the creation time here, it is not used during
