diff --git a/chrome/common/chrome_switches.h b/chrome/common/chrome_switches.h
index 627dc6c959bd9..2cf4660ad6128 100644
--- a/chrome/common/chrome_switches.h
+++ b/chrome/common/chrome_switches.h
@@ -21,6 +21,8 @@
 
 namespace switches {
 
+extern const char kEnableTabAudioMuting[];
+
 // -----------------------------------------------------------------------------
 // Can't find the switch you are looking for? Try looking in
 // media/base/media_switches.cc or ui/gl/gl_switches.cc or one of the
