diff --git a/chrome/browser/ui/browser_window.h b/chrome/browser/ui/browser_window.h
index c24ad6977eca..2ffb7d6274a5 100644
--- a/chrome/browser/ui/browser_window.h
+++ b/chrome/browser/ui/browser_window.h
@@ -465,15 +465,6 @@ class BrowserWindow : public ui::BaseWindow {
       signin_metrics::AccessPoint access_point,
       bool is_source_keyboard) = 0;
 
-  // Shows User Happiness Tracking Survey's invitation bubble when possible
-  // (such as having the proper anchor view).
-  // |site_id| is the site identification of the survey the bubble leads to.
-  // Note: |success_callback| and |failure_callback| are discarded for HaTS v1
-  // surveys, which are deprecated (crbug.com/1143176).
-  virtual void ShowHatsBubble(const std::string& site_id,
-                              base::OnceClosure success_callback,
-                              base::OnceClosure failure_callback) = 0;
-
   // Returns object implementing ExclusiveAccessContext interface.
   virtual ExclusiveAccessContext* GetExclusiveAccessContext() = 0;
 
