diff --git a/chrome/browser/background/background_mode_manager.cc b/chrome/browser/background/background_mode_manager.cc
index 481bfdda58e7..c2f0eaa3a246 100644
--- a/chrome/browser/background/background_mode_manager.cc
+++ b/chrome/browser/background/background_mode_manager.cc
@@ -64,7 +64,6 @@
 #include "components/keep_alive_registry/keep_alive_types.h"
 #include "components/prefs/pref_registry_simple.h"
 #include "components/prefs/pref_service.h"
-#include "components/startup_metric_utils/browser/startup_metric_utils.h"
 #include "content/public/browser/notification_service.h"
 #include "extensions/browser/extension_system.h"
 #include "extensions/common/constants.h"
@@ -80,7 +79,6 @@
 #include "chrome/browser/win/app_icon.h"
 #endif
 
-using base::UserMetricsAction;
 using extensions::Extension;
 
 namespace {
@@ -97,10 +95,6 @@ enum MenuItem {
   MENU_ITEM_NUM_STATES
 };
 
-void RecordMenuItemClick(MenuItem item) {
-  UMA_HISTOGRAM_ENUMERATION("BackgroundMode.MenuItemClick", item,
-                            MENU_ITEM_NUM_STATES);
-}
 }  // namespace
 
 // static
@@ -158,7 +152,6 @@ void BackgroundModeManager::BackgroundModeData::ExecuteCommand(
       break;
     default:
       DCHECK(!command_id_handler_vector_->at(command_id).is_null());
-      RecordMenuItemClick(MENU_ITEM_BACKGROUND_CLIENT);
       command_id_handler_vector_->at(command_id).Run();
       break;
   }
@@ -603,7 +596,6 @@ void BackgroundModeManager::ExecuteCommand(int command_id, int event_flags) {
   BackgroundModeData* bmd = GetBackgroundModeDataForLastProfile();
   switch (command_id) {
     case IDC_ABOUT:
-      RecordMenuItemClick(MENU_ITEM_ABOUT);
       if (bmd) {
         chrome::ShowAboutChrome(bmd->GetBrowserWindow());
       } else {
@@ -612,7 +604,6 @@ void BackgroundModeManager::ExecuteCommand(int command_id, int event_flags) {
       }
       break;
     case IDC_TASK_MANAGER:
-    RecordMenuItemClick(MENU_ITEM_TASK_MANAGER);
       if (bmd) {
         chrome::OpenTaskManager(bmd->GetBrowserWindow());
       } else {
@@ -621,8 +612,6 @@ void BackgroundModeManager::ExecuteCommand(int command_id, int event_flags) {
       }
       break;
     case IDC_EXIT:
-      RecordMenuItemClick(MENU_ITEM_EXIT);
-      base::RecordAction(UserMetricsAction("Exit"));
       chrome::CloseAllBrowsers();
       break;
     case IDC_STATUS_TRAY_KEEP_CHROME_RUNNING_IN_BACKGROUND: {
@@ -631,8 +620,6 @@ void BackgroundModeManager::ExecuteCommand(int command_id, int event_flags) {
       DCHECK(IsBackgroundModePrefEnabled());
       DCHECK(KeepAliveRegistry::GetInstance()->IsKeepingAlive());
 
-      RecordMenuItemClick(MENU_ITEM_KEEP_RUNNING);
-
       // Set the background mode pref to "disabled" - the resulting notification
       // will result in a call to DisableBackgroundMode().
       PrefService* service = g_browser_process->local_state();
@@ -693,8 +680,6 @@ void BackgroundModeManager::StartBackgroundMode() {
   if (in_background_mode_)
     return;
 
-  startup_metric_utils::SetBackgroundModeEnabled();
-
   // Mark ourselves as running in background mode.
   in_background_mode_ = true;
 
