diff --git a/chrome/browser/web_applications/BUILD.gn b/chrome/browser/web_applications/BUILD.gn
index 24baac79b235..63e0c45bae1d 100644
--- a/chrome/browser/web_applications/BUILD.gn
+++ b/chrome/browser/web_applications/BUILD.gn
@@ -13,8 +13,6 @@ group("web_app_test_group") {
 
 source_set("web_applications") {
   sources = [
-    "daily_metrics_helper.cc",
-    "daily_metrics_helper.h",
     "extension_status_utils.h",
     "external_web_app_manager.cc",
     "external_web_app_manager.h",
@@ -99,7 +97,6 @@ source_set("web_applications") {
     "//components/user_manager",
     "//components/webapps/browser",
     "//content/public/browser",
-    "//services/metrics/public/cpp:ukm_builders",
     "//skia",
   ]
 
@@ -250,7 +247,6 @@ source_set("web_applications_unit_tests") {
     "//components/webapps/browser",
     "//content/public/browser",
     "//content/test:test_support",
-    "//services/metrics/public/cpp:ukm_builders",
     "//skia",
     "//testing/gmock",
     "//testing/gtest",
