diff --git a/chrome/browser/ui/tabs/tab_utils.h b/chrome/browser/ui/tabs/tab_utils.h
index 16a2b65f05bee..2a64adaa3a3e8 100644
--- a/chrome/browser/ui/tabs/tab_utils.h
+++ b/chrome/browser/ui/tabs/tab_utils.h
@@ -36,6 +36,7 @@ enum class TabAlertState {
 
 enum class TabMutedReason {
   NONE,                    // The tab has never been muted or unmuted.
+  AUDIO_INDICATOR,         // Mute toggled via tab-strip audio icon.
   EXTENSION,               // Mute state changed via extension API.
   CONTENT_SETTING,         // The sound content setting was set to BLOCK.
   CONTENT_SETTING_CHROME,  // Mute toggled on chrome:// URL.
