diff --git a/chrome/browser/ui/webui/signin/profile_picker_handler.cc b/chrome/browser/ui/webui/signin/profile_picker_handler.cc
index fb3cde19cf5f..b3cf3e5777b0 100644
--- a/chrome/browser/ui/webui/signin/profile_picker_handler.cc
+++ b/chrome/browser/ui/webui/signin/profile_picker_handler.cc
@@ -337,7 +337,6 @@ void ProfilePickerHandler::HandleCreateProfile(const base::ListValue* args) {
   DCHECK(profiles::IsDefaultAvatarIconUrl(avatar_url, &icon_index));
 #endif
 
-  ProfileMetrics::LogProfileAddNewUser(ProfileMetrics::ADD_NEW_PROFILE_PICKER);
   ProfileManager::CreateMultiProfileAsync(
       profile_name, avatar_url,
       base::BindRepeating(&ProfilePickerHandler::OnProfileCreated,
@@ -426,8 +425,7 @@ void ProfilePickerHandler::HandleRemoveProfile(const base::ListValue* args) {
     return;
   }
   RecordProfilePickerAction(ProfilePickerAction::kDeleteProfile);
-  webui::DeleteProfileAtPath(*profile_path,
-                             ProfileMetrics::DELETE_PROFILE_USER_MANAGER);
+  webui::DeleteProfileAtPath(*profile_path);
 }
 
 void ProfilePickerHandler::HandleGetProfileStatistics(
