diff --git a/chrome/browser/ui/ui_features.cc b/chrome/browser/ui/ui_features.cc
index 09189740085da..e2ac1bc5b63bd 100644
--- a/chrome/browser/ui/ui_features.cc
+++ b/chrome/browser/ui/ui_features.cc
@@ -30,10 +30,6 @@ const base::Feature kChromeTipsInMainMenuNewBadge{
 const base::Feature kEvDetailsInPageInfo{"EvDetailsInPageInfo",
                                          base::FEATURE_ENABLED_BY_DEFAULT};
 
-// Enables an extension menu in the toolbar. See https://crbug.com/943702
-const base::Feature kExtensionsToolbarMenu{"ExtensionsToolbarMenu",
-                                           base::FEATURE_ENABLED_BY_DEFAULT};
-
 // Enables the new profile picker.
 // https:://crbug.com/1063856
 const base::Feature kNewProfilePicker{"NewProfilePicker",
