diff --git a/chrome/browser/lookalikes/lookalike_url_navigation_throttle.h b/chrome/browser/lookalikes/lookalike_url_navigation_throttle.h
index a3c85a1ad763..c3252c5d28e0 100644
--- a/chrome/browser/lookalikes/lookalike_url_navigation_throttle.h
+++ b/chrome/browser/lookalikes/lookalike_url_navigation_throttle.h
@@ -15,7 +15,6 @@
 #include "components/site_engagement/core/mojom/site_engagement_details.mojom.h"
 #include "components/url_formatter/url_formatter.h"
 #include "content/public/browser/navigation_throttle.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 
 namespace content {
 class NavigationHandle;
@@ -72,7 +71,6 @@ class LookalikeUrlNavigationThrottle : public content::NavigationThrottle {
 
   ThrottleCheckResult ShowInterstitial(const GURL& safe_domain,
                                        const GURL& url,
-                                       ukm::SourceId source_id,
                                        LookalikeUrlMatchType match_type);
 
   Profile* profile_;
