diff --git a/chrome/services/file_util/public/cpp/BUILD.gn b/chrome/services/file_util/public/cpp/BUILD.gn
index 028b8888d353..d0c0783a9c69 100644
--- a/chrome/services/file_util/public/cpp/BUILD.gn
+++ b/chrome/services/file_util/public/cpp/BUILD.gn
@@ -6,7 +6,7 @@ import("//build/config/features.gni")
 import("//components/safe_browsing/buildflags.gni")
 
 source_set("cpp") {
-  deps = [ "//chrome/common/safe_browsing:archive_analyzer_results" ]
+  deps = []
   public_deps = [ "//chrome/services/file_util/public/mojom" ]
 
   if (safe_browsing_mode == 1) {
