diff --git a/chrome/browser/extensions/api/enterprise_reporting_private/context_info_fetcher.h b/chrome/browser/extensions/api/enterprise_reporting_private/context_info_fetcher.h
index 457ece45099e..c69834a15560 100644
--- a/chrome/browser/extensions/api/enterprise_reporting_private/context_info_fetcher.h
+++ b/chrome/browser/extensions/api/enterprise_reporting_private/context_info_fetcher.h
@@ -13,7 +13,6 @@ class BrowserContext;
 }  // namespace content
 
 namespace enterprise_connectors {
-enum AnalysisConnector : int;
 class ConnectorsService;
 }  // namespace enterprise_connectors
 
@@ -55,9 +54,6 @@ class ContextInfoFetcher {
 
   std::vector<std::string> GetProfileAffiliationIDs();
 
-  std::vector<std::string> GetAnalysisConnectorProviders(
-      enterprise_connectors::AnalysisConnector connector);
-
   api::enterprise_reporting_private::RealtimeUrlCheckMode
   GetRealtimeUrlCheckMode();
 
