diff --git a/chrome/browser/accessibility/accessibility_ui.cc b/chrome/browser/accessibility/accessibility_ui.cc
index dff59cf394e29..516359ebfaa54 100644
--- a/chrome/browser/accessibility/accessibility_ui.cc
+++ b/chrome/browser/accessibility/accessibility_ui.cc
@@ -721,7 +721,6 @@ void AccessibilityUIMessageHandler::Callback(const std::string& str) {
 
 void AccessibilityUIMessageHandler::StopRecording(
     content::WebContents* web_contents) {
-  web_contents->RecordAccessibilityEvents(false, base::nullopt);
   observer_.reset(nullptr);
 }
 
@@ -749,9 +748,6 @@ void AccessibilityUIMessageHandler::RequestAccessibilityEvents(
     if (observer_) {
       return;
     }
-    web_contents->RecordAccessibilityEvents(
-        true, base::BindRepeating(&AccessibilityUIMessageHandler::Callback,
-                                  base::Unretained(this)));
     observer_ =
         std::make_unique<AccessibilityUIObserver>(web_contents, &event_logs_);
   } else {
