diff --git a/chrome/browser/first_run/first_run_internal_posix.cc b/chrome/browser/first_run/first_run_internal_posix.cc
index c9f885283968..63468d20adeb 100644
--- a/chrome/browser/first_run/first_run_internal_posix.cc
+++ b/chrome/browser/first_run/first_run_internal_posix.cc
@@ -23,7 +23,6 @@
 #include "components/metrics/metrics_pref_names.h"
 #include "components/metrics/metrics_reporting_default_state.h"
 #include "components/prefs/pref_service.h"
-#include "components/startup_metric_utils/browser/startup_metric_utils.h"
 
 namespace first_run {
 
@@ -105,7 +104,6 @@ void DoPostImportPlatformSpecificTasks(Profile* profile) {
     std::move(GetBeforeShowFirstRunDialogHookForTesting()).Run();
 
   ShowFirstRunDialog(profile);
-  startup_metric_utils::SetNonBrowserUIDisplayed();
 #endif  // !OS_CHROMEOS
 }
 
