diff --git a/chrome/browser/reputation/reputation_service.h b/chrome/browser/reputation/reputation_service.h
index 54f950f6ad15..e6b379fa2b97 100644
--- a/chrome/browser/reputation/reputation_service.h
+++ b/chrome/browser/reputation/reputation_service.h
@@ -13,7 +13,6 @@
 #include "chrome/browser/reputation/safety_tip_ui.h"
 #include "components/keyed_service/core/keyed_service.h"
 #include "components/security_state/core/security_state.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "url/gurl.h"
 #include "url/origin.h"
 
