diff --git a/chrome/browser/ui/passwords/bubble_controllers/password_bubble_controller_base.cc b/chrome/browser/ui/passwords/bubble_controllers/password_bubble_controller_base.cc
index 55dd56159559..59f188d73267 100644
--- a/chrome/browser/ui/passwords/bubble_controllers/password_bubble_controller_base.cc
+++ b/chrome/browser/ui/passwords/bubble_controllers/password_bubble_controller_base.cc
@@ -6,21 +6,14 @@
 
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/ui/passwords/passwords_model_delegate.h"
-#include "components/password_manager/core/browser/password_form_metrics_recorder.h"
 #include "content/public/browser/web_contents.h"
 
 PasswordBubbleControllerBase::PasswordBubbleControllerBase(
     base::WeakPtr<PasswordsModelDelegate> delegate,
     password_manager::metrics_util::UIDisplayDisposition display_disposition)
-    : metrics_recorder_(delegate->GetPasswordFormMetricsRecorder()),
-      are_passwords_revealed_when_bubble_is_opened_(
+    : are_passwords_revealed_when_bubble_is_opened_(
           delegate->ArePasswordsRevealedWhenBubbleIsOpened()),
       delegate_(std::move(delegate)) {
-  if (metrics_recorder_) {
-    metrics_recorder_->RecordPasswordBubbleShown(
-        delegate_->GetCredentialSource(), display_disposition);
-  }
-  password_manager::metrics_util::LogUIDisplayDisposition(display_disposition);
 
   delegate_->OnBubbleShown();
 }
