diff --git a/chrome/browser/extensions/api/passwords_private/passwords_private_delegate_factory.cc b/chrome/browser/extensions/api/passwords_private/passwords_private_delegate_factory.cc
index b39d37820387b..8bdb7cae1d549 100644
--- a/chrome/browser/extensions/api/passwords_private/passwords_private_delegate_factory.cc
+++ b/chrome/browser/extensions/api/passwords_private/passwords_private_delegate_factory.cc
@@ -36,7 +36,6 @@ PasswordsPrivateDelegateFactory::PasswordsPrivateDelegateFactory()
     : BrowserContextKeyedServiceFactory(
           "PasswordsPrivateDelegate",
           BrowserContextDependencyManager::GetInstance()) {
-  DependsOn(BulkLeakCheckServiceFactory::GetInstance());
   DependsOn(PasswordStoreFactory::GetInstance());
   DependsOn(ProfileSyncServiceFactory::GetInstance());
 }
