diff --git a/chrome/browser/ui/webui/management_ui_handler.cc b/chrome/browser/ui/webui/management_ui_handler.cc
index 518f0baa72c8..774794b26aa9 100644
--- a/chrome/browser/ui/webui/management_ui_handler.cc
+++ b/chrome/browser/ui/webui/management_ui_handler.cc
@@ -26,7 +26,6 @@
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/common/chrome_features.h"
 #include "chrome/common/pref_names.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "components/strings/grit/components_strings.h"
 #include "content/public/browser/storage_partition.h"
 #include "content/public/browser/web_contents.h"
@@ -788,9 +787,7 @@ base::Value ManagementUIHandler::GetThreatProtectionInfo(
 
   auto* on_page_visited_event =
       chrome_policies.GetValue(policy::key::kEnterpriseRealTimeUrlCheckMode);
-  if (on_page_visited_event && on_page_visited_event->is_int() &&
-      on_page_visited_event->GetInt() !=
-          safe_browsing::REAL_TIME_CHECK_DISABLED) {
+  if (on_page_visited_event && on_page_visited_event->is_int()) {
     base::Value value(base::Value::Type::DICTIONARY);
     value.SetStringKey("title", kManagementOnPageVisitedEvent);
     value.SetStringKey("permission", kManagementOnPageVisitedVisibleData);
