diff --git a/chrome/browser/media/kaleidoscope/kaleidoscope_tab_helper.cc b/chrome/browser/media/kaleidoscope/kaleidoscope_tab_helper.cc
index 414e4e2aaf51..22c5d60fa18d 100644
--- a/chrome/browser/media/kaleidoscope/kaleidoscope_tab_helper.cc
+++ b/chrome/browser/media/kaleidoscope/kaleidoscope_tab_helper.cc
@@ -7,9 +7,6 @@
 #include "base/metrics/histogram_functions.h"
 #include "chrome/browser/media/kaleidoscope/constants.h"
 #include "content/public/browser/navigation_handle.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
-#include "services/metrics/public/cpp/ukm_entry_builder.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
 #include "third_party/blink/public/common/associated_interfaces/associated_interface_provider.h"
 #include "third_party/blink/public/mojom/autoplay/autoplay.mojom.h"
 
@@ -27,9 +24,6 @@ const url::Origin& KaleidoscopeUntrustedOrigin() {
   return *origin;
 }
 
-const char kKaleidoscopeNavigationHistogramName[] =
-    "Media.Kaleidoscope.Navigation";
-
 // These values are persisted to logs. Entries should not be renumbered and
 // numeric values should never be reused.
 enum class KaleidoscopeNavigation {
@@ -78,21 +72,6 @@ void KaleidoscopeTabHelper::ReadyToCommitNavigation(
       &client);
   client->AddAutoplayFlags(url::Origin::Create(handle->GetURL()),
                            blink::mojom::kAutoplayFlagUserException);
-
-  // Only record metrics if this page was opened by Kaleidoscope.
-  if (IsOpenedFromKaleidoscope(handle)) {
-    base::UmaHistogramEnumeration(kKaleidoscopeNavigationHistogramName,
-                                  KaleidoscopeNavigation::kNormal);
-
-    ukm::UkmRecorder* ukm_recorder = ukm::UkmRecorder::Get();
-    if (!ukm_recorder)
-      return;
-
-    ukm::builders::Media_Kaleidoscope_Navigation(
-        handle->GetNextPageUkmSourceId())
-        .SetWasFromKaleidoscope(true)
-        .Record(ukm_recorder);
-  }
 }
 
 WEB_CONTENTS_USER_DATA_KEY_IMPL(KaleidoscopeTabHelper)
