diff --git a/chrome/browser/resources/settings/languages_page/BUILD.gn b/chrome/browser/resources/settings/languages_page/BUILD.gn
index 2ebe03316281..792a9da67615 100644
--- a/chrome/browser/resources/settings/languages_page/BUILD.gn
+++ b/chrome/browser/resources/settings/languages_page/BUILD.gn
@@ -58,7 +58,6 @@ js_type_check("closure_compile_module") {
   closure_flags = settings_closure_flags
   deps = [
     ":add_languages_dialog",
-    ":edit_dictionary_page",
     ":languages.m",
     ":languages_browser_proxy.m",
     ":languages_metrics_proxy",
@@ -166,7 +165,6 @@ group("polymer3_elements") {
 html_to_js("web_components") {
   js_files = [
     "add_languages_dialog.js",
-    "edit_dictionary_page.js",
     "languages_page.js",
   ]
 }
