diff --git a/chrome/common/chrome_switches.h b/chrome/common/chrome_switches.h
index df5bf330c8b5..506f79f6b74b 100644
--- a/chrome/common/chrome_switches.h
+++ b/chrome/common/chrome_switches.h
@@ -100,6 +100,7 @@ extern const char kEnableExtensionActivityLogTesting[];
 extern const char kEnableNaCl[];
 extern const char kEnableNetBenchmarking[];
 extern const char kEnablePotentiallyAnnoyingSecurityFeatures[];
+extern const char kEnableTabAudioMuting[];
 extern const char kExtensionContentVerification[];
 extern const char kExtensionContentVerificationBootstrap[];
 extern const char kExtensionContentVerificationEnforce[];
