diff --git a/chrome/browser/extensions/api/chrome_extensions_api_client.h b/chrome/browser/extensions/api/chrome_extensions_api_client.h
index dd7459b81d6b..3878959bb2e5 100644
--- a/chrome/browser/extensions/api/chrome_extensions_api_client.h
+++ b/chrome/browser/extensions/api/chrome_extensions_api_client.h
@@ -77,7 +77,6 @@ class ChromeExtensionsAPIClient : public ExtensionsAPIClient {
   NetworkingCastPrivateDelegate* GetNetworkingCastPrivateDelegate() override;
   FileSystemDelegate* GetFileSystemDelegate() override;
   MessagingDelegate* GetMessagingDelegate() override;
-  FeedbackPrivateDelegate* GetFeedbackPrivateDelegate() override;
 
 #if BUILDFLAG(IS_CHROMEOS_ASH)
   MediaPerceptionAPIDelegate* GetMediaPerceptionAPIDelegate() override;
@@ -100,7 +99,6 @@ class ChromeExtensionsAPIClient : public ExtensionsAPIClient {
       networking_cast_private_delegate_;
   std::unique_ptr<FileSystemDelegate> file_system_delegate_;
   std::unique_ptr<MessagingDelegate> messaging_delegate_;
-  std::unique_ptr<FeedbackPrivateDelegate> feedback_private_delegate_;
 
 #if BUILDFLAG(IS_CHROMEOS_ASH)
   std::unique_ptr<MediaPerceptionAPIDelegate> media_perception_api_delegate_;
