diff --git a/chrome/browser/badging/badge_manager.h b/chrome/browser/badging/badge_manager.h
index 75e0c47c80ae..d3d83e162a94 100644
--- a/chrome/browser/badging/badge_manager.h
+++ b/chrome/browser/badging/badge_manager.h
@@ -25,10 +25,6 @@ class RenderFrameHost;
 class RenderProcessHost;
 }  // namespace content
 
-namespace ukm {
-class UkmRecorder;
-}
-
 namespace badging {
 class BadgeManagerDelegate;
 
@@ -75,12 +71,6 @@ class BadgeManager : public KeyedService, public blink::mojom::BadgeService {
   // badged.
   base::Optional<BadgeValue> GetBadgeValue(const web_app::AppId& app_id);
 
-  void SetBadgeForTesting(const web_app::AppId& app_id,
-                          BadgeValue value,
-                          ukm::UkmRecorder* test_recorder);
-  void ClearBadgeForTesting(const web_app::AppId& app_id,
-                            ukm::UkmRecorder* test_recorder);
-
  private:
   // The BindingContext of a mojo request. Allows mojo calls to be tied back
   // to the execution context they belong to without trusting the renderer for
