diff --git a/chrome/browser/ui/tabs/tab_utils.h b/chrome/browser/ui/tabs/tab_utils.h
index 4355d65c52f20..9261c2793bb55 100644
--- a/chrome/browser/ui/tabs/tab_utils.h
+++ b/chrome/browser/ui/tabs/tab_utils.h
@@ -36,6 +36,7 @@ enum class TabAlertState {
 
 enum class TabMutedReason {
   NONE,                    // The tab has never been muted or unmuted.
+  AUDIO_INDICATOR,         // Mute toggled via tab-strip audio icon.
   CONTEXT_MENU,            // Mute/Unmute chosen from tab context menu.
   EXTENSION,               // Mute state changed via extension API.
   CONTENT_SETTING,         // The sound content setting was set to BLOCK.
