diff --git a/chrome/browser/download/android/download_controller.h b/chrome/browser/download/android/download_controller.h
index 8044a094cc9f9..dd10a427a9b1d 100644
--- a/chrome/browser/download/android/download_controller.h
+++ b/chrome/browser/download/android/download_controller.h
@@ -52,7 +52,6 @@ class DownloadController : public DownloadControllerBase {
     STORAGE_PERMISSION_NO_WEB_CONTENTS,
     STORAGE_PERMISSION_MAX
   };
-  static void RecordStoragePermission(StoragePermissionType type);
 
   // Close the |web_contents| for |download|. |download| could be null
   // if the download is created by Android DownloadManager.
