diff --git a/chrome/browser/ui/startup/bad_flags_prompt.cc b/chrome/browser/ui/startup/bad_flags_prompt.cc
index d27d0eaa766ae..6a5ac04460ac4 100644
--- a/chrome/browser/ui/startup/bad_flags_prompt.cc
+++ b/chrome/browser/ui/startup/bad_flags_prompt.cc
@@ -22,13 +22,11 @@
 #include "chrome/grit/chromium_strings.h"
 #include "chrome/grit/generated_resources.h"
 #include "components/autofill/core/common/autofill_switches.h"
-#include "components/data_reduction_proxy/core/common/data_reduction_proxy_switches.h"
 #include "components/infobars/content/content_infobar_manager.h"
 #include "components/infobars/core/infobar_delegate.h"
 #include "components/nacl/common/buildflags.h"
 #include "components/nacl/common/nacl_switches.h"
 #include "components/network_session_configurator/common/network_switches.h"
-#include "components/startup_metric_utils/browser/startup_metric_utils.h"
 #include "components/translate/core/common/translate_switches.h"
 #include "content/public/common/content_features.h"
 #include "content/public/common/content_switches.h"
@@ -133,10 +131,6 @@ static const char* kBadFlags[] = {
     // updating components won't be performed until shutdown.
     switches::kDisableBestEffortTasks,
 
-    // Enables save data feature which can cause user traffic to be proxied via
-    // Google's data reduction proxy servers.
-    data_reduction_proxy::switches::kEnableDataReductionProxy,
-
     // GPU sanboxing isn't implemented for the Web GPU API yet meaning it would
     // be possible to read GPU data for other Chromium processes.
     switches::kEnableUnsafeWebGPU,
@@ -215,7 +209,6 @@ void MaybeShowInvalidUserDataDirWarningDialog() {
   if (user_data_dir.empty())
     return;
 
-  startup_metric_utils::SetNonBrowserUIDisplayed();
 
   // Ensure the ResourceBundle is initialized for string resource access.
   bool cleanup_resource_bundle = false;
