diff --git a/chrome/browser/ui/webui/signin/profile_picker_handler.cc b/chrome/browser/ui/webui/signin/profile_picker_handler.cc
index 5073dbcd727f..f92f0ea25784 100644
--- a/chrome/browser/ui/webui/signin/profile_picker_handler.cc
+++ b/chrome/browser/ui/webui/signin/profile_picker_handler.cc
@@ -412,8 +412,6 @@ void ProfilePickerHandler::HandleCreateProfile(const base::ListValue* args) {
 #endif
 
   std::string avatar_url = profiles::GetDefaultAvatarIconUrl(avatar_index);
-  ProfileMetrics::LogProfileAddNewUser(
-      ProfileMetrics::ADD_NEW_PROFILE_PICKER_LOCAL);
   ProfileManager::CreateMultiProfileAsync(
       profile_name, avatar_url,
       base::BindRepeating(&ProfilePickerHandler::OnProfileCreated,
@@ -534,8 +532,7 @@ void ProfilePickerHandler::HandleRemoveProfile(const base::ListValue* args) {
     return;
   }
   RecordProfilePickerAction(ProfilePickerAction::kDeleteProfile);
-  webui::DeleteProfileAtPath(*profile_path,
-                             ProfileMetrics::DELETE_PROFILE_USER_MANAGER);
+  webui::DeleteProfileAtPath(*profile_path);
 }
 
 void ProfilePickerHandler::HandleGetProfileStatistics(
