diff --git a/chrome/browser/profiles/avatar_menu_actions_desktop.cc b/chrome/browser/profiles/avatar_menu_actions_desktop.cc
index 93ab73145b70d..827af5144bcdb 100644
--- a/chrome/browser/profiles/avatar_menu_actions_desktop.cc
+++ b/chrome/browser/profiles/avatar_menu_actions_desktop.cc
@@ -21,7 +21,7 @@ AvatarMenuActions* AvatarMenuActions::Create() {
   return new AvatarMenuActionsDesktop();
 }
 
-void AvatarMenuActionsDesktop::AddNewProfile(ProfileMetrics::ProfileAdd type) {
+void AvatarMenuActionsDesktop::AddNewProfile() {
   // TODO: Remove dependency on Browser by delegating AddNewProfile and
   // and EditProfile actions.
 
@@ -32,7 +32,6 @@ void AvatarMenuActionsDesktop::AddNewProfile(ProfileMetrics::ProfileAdd type) {
     settings_browser = Browser::Create(params);
   }
   chrome::ShowSettingsSubPage(settings_browser, chrome::kCreateProfileSubPage);
-  ProfileMetrics::LogProfileAddNewUser(type);
 }
 
 void AvatarMenuActionsDesktop::EditProfile(Profile* profile) {
