diff --git a/chrome/notification_helper/BUILD.gn b/chrome/notification_helper/BUILD.gn
index 7d4de239c58e..6787fa7910b2 100644
--- a/chrome/notification_helper/BUILD.gn
+++ b/chrome/notification_helper/BUILD.gn
@@ -11,8 +11,6 @@ executable("notification_helper") {
   sources = [
     "notification_helper.cc",
     "notification_helper.rc",
-    "notification_helper_crash_reporter_client.cc",
-    "notification_helper_crash_reporter_client.h",
   ]
 
   configs -= [
@@ -30,8 +28,6 @@ executable("notification_helper") {
     "//chrome/common:version_header",
     "//chrome/install_static:install_static_util",
     "//chrome/install_static:primary_module",
-    "//components/crash/core/app",
-    "//components/crash/core/app:crash_export_thunks",
     "//components/version_info:channel",
   ]
 
