diff --git a/chrome/browser/accessibility/accessibility_ui.cc b/chrome/browser/accessibility/accessibility_ui.cc
index eed11ee2d85b..94ea3a85cba9 100644
--- a/chrome/browser/accessibility/accessibility_ui.cc
+++ b/chrome/browser/accessibility/accessibility_ui.cc
@@ -719,7 +719,6 @@ void AccessibilityUIMessageHandler::Callback(const std::string& str) {
 
 void AccessibilityUIMessageHandler::StopRecording(
     content::WebContents* web_contents) {
-  web_contents->RecordAccessibilityEvents(false, base::nullopt);
   observer_.reset(nullptr);
 }
 
@@ -747,9 +746,6 @@ void AccessibilityUIMessageHandler::RequestAccessibilityEvents(
     if (observer_) {
       return;
     }
-    web_contents->RecordAccessibilityEvents(
-        true, base::BindRepeating(&AccessibilityUIMessageHandler::Callback,
-                                  base::Unretained(this)));
     observer_ =
         std::make_unique<AccessibilityUIObserver>(web_contents, &event_logs_);
   } else {
