diff --git a/chrome/browser/devtools/devtools_embedder_message_dispatcher.cc b/chrome/browser/devtools/devtools_embedder_message_dispatcher.cc
index fb0b1a3afddad..af0d04b5b984e 100644
--- a/chrome/browser/devtools/devtools_embedder_message_dispatcher.cc
+++ b/chrome/browser/devtools/devtools_embedder_message_dispatcher.cc
@@ -250,8 +250,5 @@ DevToolsEmbedderMessageDispatcher::CreateForDevToolsFrontend(
                      &Delegate::SetOpenNewWindowForPopups, delegate);
   d->RegisterHandler("registerExtensionsAPI", &Delegate::RegisterExtensionsAPI,
                      delegate);
-  d->RegisterHandlerWithCallback("showSurvey", &Delegate::ShowSurvey, delegate);
-  d->RegisterHandlerWithCallback("canShowSurvey", &Delegate::CanShowSurvey,
-                                 delegate);
   return d;
 }
