diff --git a/chrome/browser/metrics/BUILD.gn b/chrome/browser/metrics/BUILD.gn
index 564454e7e1c13..15b329ba6622a 100644
--- a/chrome/browser/metrics/BUILD.gn
+++ b/chrome/browser/metrics/BUILD.gn
@@ -17,7 +17,6 @@ source_set("test_support") {
     "//chrome/browser",
     "//components/signin/public/identity_manager",
     "//components/sync/test/fake_server",
-    "//third_party/metrics_proto",
   ]
 
   if (!is_fuchsia) {
