diff --git a/chrome/common/extensions/api/api_sources.gni b/chrome/common/extensions/api/api_sources.gni
index 5b0625bd56447..034226804e751 100644
--- a/chrome/common/extensions/api/api_sources.gni
+++ b/chrome/common/extensions/api/api_sources.gni
@@ -31,7 +31,6 @@ schema_sources_ = [
   "downloads.idl",
   "downloads_internal.idl",
   "enterprise_hardware_platform.idl",
-  "enterprise_reporting_private.idl",
   "font_settings.json",
   "gcm.json",
   "history.json",
@@ -47,7 +46,6 @@ schema_sources_ = [
   "passwords_private.idl",
   "permissions.json",
   "resources_private.idl",
-  "safe_browsing_private.idl",
   "scripting.idl",
   "search.idl",
   "sessions.json",
