diff --git a/chrome/browser/media/router/BUILD.gn b/chrome/browser/media/router/BUILD.gn
index c9f142c95dca6..6b978b5321f94 100644
--- a/chrome/browser/media/router/BUILD.gn
+++ b/chrome/browser/media/router/BUILD.gn
@@ -91,7 +91,6 @@ static_library("router") {
       "//components/paint_preview/buildflags",
       "//components/signin/public/base:signin_buildflags",
       "//components/translate/content/common",
-      "//components/ukm/content:content",
       "//components/version_info:version_info",
       "//extensions/browser",
       "//extensions/buildflags",
