diff --git a/chrome/browser/ui/webui/new_tab_page/promo_browser_command/promo_browser_command_handler.cc b/chrome/browser/ui/webui/new_tab_page/promo_browser_command/promo_browser_command_handler.cc
index 8b632fdaa583..2f59764a37ff 100644
--- a/chrome/browser/ui/webui/new_tab_page/promo_browser_command/promo_browser_command_handler.cc
+++ b/chrome/browser/ui/webui/new_tab_page/promo_browser_command/promo_browser_command_handler.cc
@@ -16,9 +16,6 @@
 #include "chrome/browser/ui/browser_navigator.h"
 #include "chrome/browser/ui/chrome_pages.h"
 #include "chrome/common/webui_url_constants.h"
-#include "components/safe_browsing/content/web_ui/safe_browsing_ui.h"
-#include "components/safe_browsing/core/common/safe_browsing_policy_handler.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "ui/base/page_transition_types.h"
 #include "ui/base/window_open_disposition.h"
 
@@ -47,24 +44,6 @@ void PromoBrowserCommandHandler::CanShowPromoWithCommand(
     promo_browser_command::mojom::Command command_id,
     CanShowPromoWithCommandCallback callback) {
   bool can_show = false;
-  switch (static_cast<Command>(command_id)) {
-    case Command::kUnknownCommand:
-      // Nothing to do.
-      break;
-    case Command::kOpenSafetyCheck:
-      can_show = chrome::enterprise_util::HasBrowserPoliciesApplied(profile_);
-      break;
-    case Command::kOpenSafeBrowsingEnhancedProtectionSettings: {
-      bool managed = safe_browsing::SafeBrowsingPolicyHandler::
-          IsSafeBrowsingProtectionLevelSetByPolicy(profile_->GetPrefs());
-      bool already_enabled =
-          safe_browsing::IsEnhancedProtectionEnabled(*(profile_->GetPrefs()));
-      can_show = !managed && !already_enabled;
-    } break;
-    default:
-      NOTREACHED() << "Unspecified behavior for command " << command_id;
-      break;
-  }
   std::move(callback).Run(can_show);
 }
 
