diff --git a/chrome/browser/devtools/devtools_ui_bindings.h b/chrome/browser/devtools/devtools_ui_bindings.h
index 32aa300af86ac..9314f6fd3c5b2 100644
--- a/chrome/browser/devtools/devtools_ui_bindings.h
+++ b/chrome/browser/devtools/devtools_ui_bindings.h
@@ -173,10 +173,6 @@ class DevToolsUIBindings : public DevToolsEmbedderMessageDispatcher::Delegate,
   void SetOpenNewWindowForPopups(bool value) override;
   void RegisterExtensionsAPI(const std::string& origin,
                              const std::string& script) override;
-  void ShowSurvey(DispatchCallback callback,
-                  const std::string& trigger) override;
-  void CanShowSurvey(DispatchCallback callback,
-                     const std::string& trigger) override;
 
   void EnableRemoteDeviceCounter(bool enable);
 
