diff --git a/chrome/browser/predictors/loading_predictor_tab_helper.cc b/chrome/browser/predictors/loading_predictor_tab_helper.cc
index 67f842d7c7978..22dfc90904ddd 100644
--- a/chrome/browser/predictors/loading_predictor_tab_helper.cc
+++ b/chrome/browser/predictors/loading_predictor_tab_helper.cc
@@ -26,7 +26,6 @@
 #include "content/public/browser/browser_thread.h"
 #include "content/public/browser/navigation_handle.h"
 #include "content/public/browser/render_frame_host.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "services/network/public/mojom/fetch_api.mojom.h"
 #include "third_party/blink/public/mojom/loader/resource_load_info.mojom.h"
 
@@ -251,8 +250,7 @@ void LoadingPredictorTabHelper::DidStartNavigation(
   page_data.has_local_preconnect_predictions_for_current_navigation_ =
       predictor_->OnNavigationStarted(
           page_data.navigation_id_,
-          ukm::ConvertToSourceId(navigation_handle->GetNavigationId(),
-                                 ukm::SourceIdType::NAVIGATION_ID),
+          ukm::kInvalidSourceId,
           navigation_handle->GetURL(), navigation_handle->NavigationStart());
   if (page_data.has_local_preconnect_predictions_for_current_navigation_ &&
       !features::ShouldAlwaysRetrieveOptimizationGuidePredictions()) {
@@ -363,9 +361,6 @@ void LoadingPredictorTabHelper::ResourceLoadComplete(
   auto* page_data = PageData::GetForDocument(*render_frame_host);
   if (!page_data)
     return;
-
-  predictor_->loading_data_collector()->RecordResourceLoadComplete(
-      page_data->navigation_id_, resource_load_info);
 }
 
 void LoadingPredictorTabHelper::DidLoadResourceFromMemoryCache(
@@ -391,8 +386,6 @@ void LoadingPredictorTabHelper::DidLoadResourceFromMemoryCache(
       GetRequestPriority(resource_load_info.request_destination);
   resource_load_info.network_info =
       blink::mojom::CommonNetworkInfo::New(false, false, absl::nullopt);
-  predictor_->loading_data_collector()->RecordResourceLoadComplete(
-      page_data->navigation_id_, resource_load_info);
 }
 
 void LoadingPredictorTabHelper::DocumentOnLoadCompletedInMainFrame(
@@ -405,10 +398,6 @@ void LoadingPredictorTabHelper::DocumentOnLoadCompletedInMainFrame(
   if (!page_data)
     return;
 
-  predictor_->loading_data_collector()->RecordMainFrameLoadComplete(
-      page_data->navigation_id_,
-      page_data->last_optimization_guide_prediction_);
-
   // Clear out Optimization Guide Prediction, as it is no longer needed.
   page_data->last_optimization_guide_prediction_ = absl::nullopt;
 }
@@ -423,9 +412,6 @@ void LoadingPredictorTabHelper::RecordFirstContentfulPaint(
   auto* page_data = PageData::GetForDocument(*render_frame_host);
   if (!page_data)
     return;
-
-  predictor_->loading_data_collector()->RecordFirstContentfulPaint(
-      page_data->navigation_id_, first_contentful_paint);
 }
 
 void LoadingPredictorTabHelper::OnOptimizationGuideDecision(
