diff --git a/chrome/browser/ui/managed_ui.cc b/chrome/browser/ui/managed_ui.cc
index 952e12319fc9..5cbd521847fb 100644
--- a/chrome/browser/ui/managed_ui.cc
+++ b/chrome/browser/ui/managed_ui.cc
@@ -33,7 +33,7 @@ bool ShouldDisplayManagedUi(Profile* profile) {
     return false;
 #endif
 
-  return enterprise_util::HasBrowserPoliciesApplied(profile);
+  return false;
 }
 
 base::string16 GetManagedUiMenuItemLabel(Profile* profile) {
