diff --git a/chrome/BUILD.gn b/chrome/BUILD.gn
index ec09d0836a976..b480e0d101ef7 100644
--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -126,7 +126,7 @@ if (!is_android && !is_mac) {
     # the output name for that platform.
     # See crbug.com/1146571.
     if (is_win) {
-      public_deps += [ ":reorder_imports" ]
+      public_deps += [ ":reorder_imports", "//dumb/build/win:copy_exe_and_pdb" ]
       data_deps += [ ":reorder_imports" ]
     }
   }
@@ -1338,11 +1338,8 @@ group("browser_dependencies") {
     "//chrome/common",
     "//components/crash/core/app",
     "//components/gwp_asan/buildflags",
-    "//components/heap_profiling/in_process",
     "//components/power_scheduler",
-    "//components/startup_metric_utils/browser",
     "//components/sync",
-    "//components/upload_list:upload_list",
     "//services/tracing/public/cpp",
   ]
   if (enable_plugins) {
@@ -1408,7 +1405,6 @@ group("child_dependencies") {
     "//chrome/renderer",
     "//chrome/utility",
     "//components/gwp_asan/buildflags",
-    "//components/services/heap_profiling",
     "//content/public/child",
     "//pdf",
     "//pdf:pdf_ppapi",
@@ -1437,7 +1433,7 @@ group("child_dependencies") {
 
 if (is_win) {
   process_version_rc_template("chrome_exe_version") {
-    sources = [ "app/chrome_exe.ver" ]
+    sources = [ "app/dumb_exe.ver" ]
     output = "$target_gen_dir/chrome_exe_version.rc"
   }
 
@@ -1651,12 +1647,8 @@ if (is_android) {
       "//chrome/gpu",
       "//chrome/renderer",
       "//components/gwp_asan/buildflags",
-      "//components/heap_profiling/in_process",
       "//components/minidump_uploader",
       "//components/safe_browsing:buildflags",
-      "//components/safe_browsing/android:safe_browsing_api_handler",
-      "//components/safe_browsing/android:safe_browsing_mobile",
-      "//components/services/heap_profiling",
       "//content/public/app",
     ]
 
