diff --git a/chrome/browser/accessibility/accessibility_ui.cc b/chrome/browser/accessibility/accessibility_ui.cc
index 8905cd790b7d4..7a09c88f8df9e 100644
--- a/chrome/browser/accessibility/accessibility_ui.cc
+++ b/chrome/browser/accessibility/accessibility_ui.cc
@@ -722,7 +722,6 @@ void AccessibilityUIMessageHandler::Callback(const std::string& str) {
 
 void AccessibilityUIMessageHandler::StopRecording(
     content::WebContents* web_contents) {
-  web_contents->RecordAccessibilityEvents(false, absl::nullopt);
   observer_.reset(nullptr);
 }
 
@@ -750,9 +749,6 @@ void AccessibilityUIMessageHandler::RequestAccessibilityEvents(
     if (observer_) {
       return;
     }
-    web_contents->RecordAccessibilityEvents(
-        true, base::BindRepeating(&AccessibilityUIMessageHandler::Callback,
-                                  base::Unretained(this)));
     observer_ =
         std::make_unique<AccessibilityUIObserver>(web_contents, &event_logs_);
   } else {
