diff --git a/chrome/browser/login_detection/login_detection_tab_helper.cc b/chrome/browser/login_detection/login_detection_tab_helper.cc
index 696952084f88f..4629184cb3480 100644
--- a/chrome/browser/login_detection/login_detection_tab_helper.cc
+++ b/chrome/browser/login_detection/login_detection_tab_helper.cc
@@ -19,10 +19,6 @@
 #include "content/public/browser/navigation_handle.h"
 #include "content/public/browser/web_contents.h"
 #include "net/base/registry_controlled_domains/registry_controlled_domain.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
-#include "services/metrics/public/cpp/ukm_source.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 
 namespace login_detection {
 
@@ -35,10 +31,6 @@ PrefService* GetPrefs(content::WebContents* web_contents) {
 
 void RecordLoginDetectionMetrics(LoginDetectionType type,
                                  ukm::SourceId ukm_source_id) {
-  base::UmaHistogramEnumeration("Login.PageLoad.DetectionType", type);
-  ukm::builders::LoginDetection builder(ukm_source_id);
-  builder.SetPage_LoginType(static_cast<int64_t>(type))
-      .Record(ukm::UkmRecorder::Get());
 }
 
 }  // namespace
