diff --git a/chrome/browser/mac/mac_startup_profiler.h b/chrome/browser/mac/mac_startup_profiler.h
index d3ace2fb0f83..01caabab27ca 100644
--- a/chrome/browser/mac/mac_startup_profiler.h
+++ b/chrome/browser/mac/mac_startup_profiler.h
@@ -55,7 +55,7 @@ class MacStartupProfiler {
   std::map<Location, base::TimeTicks> profiled_ticks_;
 
   // Whether UMA metrics have been recorded. Only record UMA metrics once.
-  bool recorded_metrics_;
+  ALLOW_UNUSED_TYPE bool recorded_metrics_;
 
   DISALLOW_COPY_AND_ASSIGN(MacStartupProfiler);
 };
