diff --git a/chrome/browser/resources/settings/privacy_page/BUILD.gn b/chrome/browser/resources/settings/privacy_page/BUILD.gn
index 44d93bce35de2..463ee3ec5a737 100644
--- a/chrome/browser/resources/settings/privacy_page/BUILD.gn
+++ b/chrome/browser/resources/settings/privacy_page/BUILD.gn
@@ -106,7 +106,6 @@ js_library("personalization_options") {
 js_library("privacy_page") {
   deps = [
     ":privacy_page_browser_proxy",
-    "..:hats_browser_proxy",
     "..:i18n_setup",
     "..:metrics_browser_proxy",
     "..:page_visibility",
@@ -193,7 +192,6 @@ js_library("security_keys_subpage") {
 
 js_library("security_page") {
   deps = [
-    ":disable_safebrowsing_dialog",
     ":privacy_page_browser_proxy",
     "..:i18n_setup",
     "..:metrics_browser_proxy",
@@ -210,7 +208,6 @@ html_to_js("polymer3_elements") {
   js_files = [
     "collapse_radio_button.js",
     "cookies_page.js",
-    "disable_safebrowsing_dialog.js",
     "do_not_track_toggle.js",
     "personalization_options.js",
     "privacy_page.js",
