diff --git a/chrome/browser/ui/views/tabs/fake_base_tab_strip_controller.cc b/chrome/browser/ui/views/tabs/fake_base_tab_strip_controller.cc
index 3bd3542a627d..3c7c25bed88f 100644
--- a/chrome/browser/ui/views/tabs/fake_base_tab_strip_controller.cc
+++ b/chrome/browser/ui/views/tabs/fake_base_tab_strip_controller.cc
@@ -205,6 +205,9 @@ void FakeBaseTabStripController::CloseTab(int index) {
   RemoveTab(index);
 }
 
+void FakeBaseTabStripController::ToggleTabAudioMute(int index) {
+}
+
 void FakeBaseTabStripController::ShowContextMenuForTab(
     Tab* tab,
     const gfx::Point& p,
