diff --git a/chromeos/components/media_app_ui/resources/js/BUILD.gn b/chromeos/components/media_app_ui/resources/js/BUILD.gn
index 9d5bbb3603aa..7b0087fd9b21 100644
--- a/chromeos/components/media_app_ui/resources/js/BUILD.gn
+++ b/chromeos/components/media_app_ui/resources/js/BUILD.gn
@@ -41,7 +41,6 @@ js_library("launch") {
     "media_app.externs.js",
   ]
   deps = [
-    ":error_reporter",
     ":message_types",
     ":mojo_api_bootstrap",
     "//chromeos/components/system_apps/public/js:message_pipe",
