diff --git a/chrome/browser/optimization_guide/optimization_guide_navigation_data.cc b/chrome/browser/optimization_guide/optimization_guide_navigation_data.cc
index bc1af3ab035d0..e4b46af1768b3 100644
--- a/chrome/browser/optimization_guide/optimization_guide_navigation_data.cc
+++ b/chrome/browser/optimization_guide/optimization_guide_navigation_data.cc
@@ -12,17 +12,12 @@
 #include "components/optimization_guide/core/hints_processing_util.h"
 #include "content/public/browser/navigation_handle.h"
 #include "net/nqe/effective_connection_type.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
-#include "services/metrics/public/cpp/ukm_source.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 
 OptimizationGuideNavigationData::OptimizationGuideNavigationData(
     int64_t navigation_id)
     : navigation_id_(navigation_id) {}
 
 OptimizationGuideNavigationData::~OptimizationGuideNavigationData() {
-  RecordMetrics();
 }
 
 // static
@@ -39,6 +34,7 @@ OptimizationGuideNavigationData::GetFromNavigationHandle(
       ->GetOrCreateOptimizationGuideNavigationData(navigation_handle);
 }
 
+#if 0
 void OptimizationGuideNavigationData::RecordMetrics() const {
   RecordOptimizationGuideUKM();
 }
@@ -87,6 +83,7 @@ void OptimizationGuideNavigationData::RecordOptimizationGuideUKM() const {
   if (did_record_metric)
     builder.Record(ukm::UkmRecorder::Get());
 }
+#endif
 
 base::Optional<base::TimeDelta>
 OptimizationGuideNavigationData::hints_fetch_latency() const {
