diff --git a/chrome/browser/web_applications/components/BUILD.gn b/chrome/browser/web_applications/components/BUILD.gn
index d51025045b14b..69f370a80347c 100644
--- a/chrome/browser/web_applications/components/BUILD.gn
+++ b/chrome/browser/web_applications/components/BUILD.gn
@@ -24,8 +24,6 @@ source_set("components") {
     "externally_installed_web_app_prefs.h",
     "file_handler_manager.cc",
     "file_handler_manager.h",
-    "install_bounce_metric.cc",
-    "install_bounce_metric.h",
     "install_finalizer.cc",
     "install_finalizer.h",
     "install_manager.cc",
