diff --git a/chrome/browser/ui/cocoa/first_run_dialog.mm b/chrome/browser/ui/cocoa/first_run_dialog.mm
index 125931b4454f..4f173dc0da8e 100644
--- a/chrome/browser/ui/cocoa/first_run_dialog.mm
+++ b/chrome/browser/ui/cocoa/first_run_dialog.mm
@@ -17,7 +17,6 @@
 #include "chrome/browser/browser_process_impl.h"
 #include "chrome/browser/first_run/first_run.h"
 #include "chrome/browser/first_run/first_run_dialog.h"
-#include "chrome/browser/metrics/metrics_reporting_state.h"
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/shell_integration.h"
 #include "chrome/browser/ui/cocoa/first_run_dialog_controller.h"
@@ -68,11 +67,6 @@ void ShowFirstRunModal(Profile* profile) {
 
   [dialog.get() showWindow:nil];
 
-  // If the dialog asked the user to opt-in for stats and crash reporting,
-  // record the decision and enable the crash reporter if appropriate.
-  bool consent_given = [dialog.get() isStatsReportingEnabled];
-  ChangeMetricsReportingState(consent_given);
-
   // If selected, set as default browser. Skip in automated tests so that an OS
   // dialog confirming the default browser choice isn't left on screen.
   BOOL make_default_browser =
