diff --git a/chrome/browser/profiles/avatar_menu.h b/chrome/browser/profiles/avatar_menu.h
index 9a14085fc9ca9..3e47e2b938ef7 100644
--- a/chrome/browser/profiles/avatar_menu.h
+++ b/chrome/browser/profiles/avatar_menu.h
@@ -15,7 +15,6 @@
 #include "base/macros.h"
 #include "base/scoped_observer.h"
 #include "chrome/browser/profiles/profile_attributes_storage.h"
-#include "chrome/browser/profiles/profile_metrics.h"
 #include "chrome/common/buildflags.h"
 #include "content/public/browser/web_contents.h"
 #include "content/public/browser/web_contents_observer.h"
@@ -118,7 +117,7 @@ class AvatarMenu :
   void SwitchToProfile(size_t index, bool always_create);
 
   // Creates a new profile.
-  void AddNewProfile(ProfileMetrics::ProfileAdd type);
+  void AddNewProfile();
 
   // Opens the profile settings in response to clicking the edit button next to
   // an item.
