diff --git a/chrome/browser/extensions/api/autofill_private/autofill_private_api.h b/chrome/browser/extensions/api/autofill_private/autofill_private_api.h
index 9d390257b8256..f5cacc27a71b2 100644
--- a/chrome/browser/extensions/api/autofill_private/autofill_private_api.h
+++ b/chrome/browser/extensions/api/autofill_private/autofill_private_api.h
@@ -179,6 +179,7 @@ class AutofillPrivateMigrateCreditCardsFunction : public ExtensionFunction {
   ResponseAction Run() override;
 };
 
+#if 0
 class AutofillPrivateLogServerCardLinkClickedFunction
     : public ExtensionFunction {
  public:
@@ -196,6 +197,7 @@ class AutofillPrivateLogServerCardLinkClickedFunction
   // ExtensionFunction overrides.
   ResponseAction Run() override;
 };
+#endif
 
 class AutofillPrivateSetCreditCardFIDOAuthEnabledStateFunction
     : public ExtensionFunction {
