diff --git a/chrome/browser/ui/passwords/bubble_controllers/password_bubble_controller_base.h b/chrome/browser/ui/passwords/bubble_controllers/password_bubble_controller_base.h
index 0536c58b68a0..8b68b6337584 100644
--- a/chrome/browser/ui/passwords/bubble_controllers/password_bubble_controller_base.h
+++ b/chrome/browser/ui/passwords/bubble_controllers/password_bubble_controller_base.h
@@ -12,10 +12,6 @@ namespace content {
 class WebContents;
 }
 
-namespace password_manager {
-class PasswordFormMetricsRecorder;
-}
-
 class PasswordsModelDelegate;
 class Profile;
 
@@ -51,12 +47,6 @@ class PasswordBubbleControllerBase {
   content::WebContents* GetWebContents() const;
 
  protected:
-  // Reference to metrics recorder of the PasswordForm presented to the user by
-  // |this|. We hold on to this because |delegate_| may not be able to provide
-  // the reference anymore when we need it.
-  scoped_refptr<password_manager::PasswordFormMetricsRecorder>
-      metrics_recorder_;
-
   // True if the model has already recorded all the necessary statistics when
   // the bubble is closing.
   bool interaction_reported_ = false;
