diff --git a/chrome/browser/optimization_guide/optimization_guide_hints_manager.cc b/chrome/browser/optimization_guide/optimization_guide_hints_manager.cc
index bcc6bd4f7ccb..2d5d1ced0f7d 100644
--- a/chrome/browser/optimization_guide/optimization_guide_hints_manager.cc
+++ b/chrome/browser/optimization_guide/optimization_guide_hints_manager.cc
@@ -51,10 +51,6 @@
 #include "components/prefs/scoped_user_pref_update.h"
 #include "content/public/browser/browser_thread.h"
 #include "content/public/browser/navigation_handle.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
-#include "services/metrics/public/cpp/ukm_source.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "services/network/public/cpp/shared_url_loader_factory.h"
 
 namespace {
@@ -179,25 +175,6 @@ bool IsOptimizationTypeAllowed(
   return false;
 }
 
-// Logs an OptimizationAutotuning event for the navigation with |navigation_id|,
-// if |navigation_id| and |tuning_version| are non-null.
-void MaybeLogOptimizationAutotuningUKMForNavigation(
-    base::Optional<int64_t> navigation_id,
-    optimization_guide::proto::OptimizationType optimization_type,
-    base::Optional<int64_t> tuning_version) {
-  if (!navigation_id || !tuning_version) {
-    // Only log if we can correlate the tuning event with a navigation.
-    return;
-  }
-
-  ukm::SourceId ukm_source_id =
-      ukm::ConvertToSourceId(*navigation_id, ukm::SourceIdType::NAVIGATION_ID);
-  ukm::builders::OptimizationGuideAutotuning builder(ukm_source_id);
-  builder.SetOptimizationType(optimization_type)
-      .SetTuningVersion(*tuning_version)
-      .Record(ukm::UkmRecorder::Get());
-}
-
 // Util class for recording whether a hints fetch race against the current
 // navigation was attempted. The result is recorded when it goes out of scope
 // and its destructor is called.
@@ -1092,8 +1069,6 @@ OptimizationGuideHintsManager::CanApplyOptimization(
           url_keyed_hint->page_hints(0).whitelisted_optimizations(),
           optimization_type, optimization_metadata, &tuning_version);
       if (is_allowed || tuning_version) {
-        MaybeLogOptimizationAutotuningUKMForNavigation(
-            navigation_id, optimization_type, tuning_version);
         return is_allowed ? optimization_guide::OptimizationTypeDecision::
                                 kAllowedByHint
                           : optimization_guide::OptimizationTypeDecision::
@@ -1130,8 +1105,6 @@ OptimizationGuideHintsManager::CanApplyOptimization(
       loaded_hint->whitelisted_optimizations(), optimization_type,
       optimization_metadata, &tuning_version);
   if (is_allowed || tuning_version) {
-    MaybeLogOptimizationAutotuningUKMForNavigation(
-        navigation_id, optimization_type, tuning_version);
     return is_allowed
                ? optimization_guide::OptimizationTypeDecision::kAllowedByHint
                : optimization_guide::OptimizationTypeDecision::
@@ -1148,8 +1121,6 @@ OptimizationGuideHintsManager::CanApplyOptimization(
   is_allowed = IsOptimizationTypeAllowed(
       matched_page_hint->whitelisted_optimizations(), optimization_type,
       optimization_metadata, &tuning_version);
-  MaybeLogOptimizationAutotuningUKMForNavigation(
-      navigation_id, optimization_type, tuning_version);
   return is_allowed
              ? optimization_guide::OptimizationTypeDecision::kAllowedByHint
              : optimization_guide::OptimizationTypeDecision::kNotAllowedByHint;
