diff --git a/chrome/browser/browser_switcher/bho/mini_bho.cc b/chrome/browser/browser_switcher/bho/mini_bho.cc
index 824f6356f680..75a2be048f8f 100644
--- a/chrome/browser/browser_switcher/bho/mini_bho.cc
+++ b/chrome/browser/browser_switcher/bho/mini_bho.cc
@@ -61,7 +61,7 @@ const wchar_t kHttpsPrefix[] = L"https://";
 const wchar_t kFilePrefix[] = L"file://";
 
 const wchar_t kChromeKey[] =
-    L"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\App Paths\\chrome.exe";
+    L"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\App Paths\\dumb.exe";
 
 // TODO(crbug/950039): Add other varnames (${firefox}, ${opera}, ...)
 const wchar_t kChromeVarName[] = L"${chrome}";
