diff --git a/chrome/credential_provider/setup/BUILD.gn b/chrome/credential_provider/setup/BUILD.gn
index b58775b83698..8d2221377c1a 100644
--- a/chrome/credential_provider/setup/BUILD.gn
+++ b/chrome/credential_provider/setup/BUILD.gn
@@ -71,10 +71,6 @@ executable("gcp_setup") {
     "../gaiacp:common",
     "//build:branding_buildflags",
     "//chrome/common:version_header",
-    "//components/crash/core/app:app",
-    "//components/crash/core/app:crash_export_thunks",
-    "//components/crash/core/app:run_as_crashpad_handler",
-    "//components/crash/core/common",
     "//content/public/common:static_switches",
   ]
   configs += [ "//build/config/win:windowed" ]
