diff --git a/chrome/browser/ui/cocoa/notifications/notification_delivery.h b/chrome/browser/ui/cocoa/notifications/notification_delivery.h
index 72732cf10930..72eb6df023e7 100644
--- a/chrome/browser/ui/cocoa/notifications/notification_delivery.h
+++ b/chrome/browser/ui/cocoa/notifications/notification_delivery.h
@@ -15,12 +15,6 @@
 // Protocol for the XPC notification service.
 @protocol NotificationDelivery
 
-// Sets the Mach exception handler port to use for the XPCService, and sets
-// which notification API to be used. This method must be called first before
-// using the other methods in this protocol.
-- (void)setUseUNNotification:(BOOL)useUNNotification
-           machExceptionPort:(CrXPCMachPort*)port;
-
 // |notificationData| is generated using a NofiticationBuilder object.
 - (void)deliverNotification:(NSDictionary*)notificationData;
 
