diff --git a/chrome/test/data/webui/settings/BUILD.gn b/chrome/test/data/webui/settings/BUILD.gn
index 282b4d6503f78..3c259dafb0e4f 100644
--- a/chrome/test/data/webui/settings/BUILD.gn
+++ b/chrome/test/data/webui/settings/BUILD.gn
@@ -100,8 +100,6 @@ js_type_check("closure_compile") {
     #":reset_page_test",
     #":reset_profile_banner_test",
     #":route_tests",
-    ":safety_check_chrome_cleaner_test",
-    ":safety_check_page_test",
 
     #":search_engines_page_test",
     ":search_page_test",
