diff --git a/chrome/browser/chromeos/BUILD.gn b/chrome/browser/chromeos/BUILD.gn
index d50d68cf2627..e545731f0fc2 100644
--- a/chrome/browser/chromeos/BUILD.gn
+++ b/chrome/browser/chromeos/BUILD.gn
@@ -261,7 +261,6 @@ source_set("chromeos") {
     "//components/printing/common:mojo_interfaces",
     "//components/proxy_config",
     "//components/quirks",
-    "//components/rappor",
     "//components/renderer_context_menu",
     "//components/rlz",
     "//components/safe_browsing/core:csd_proto",
