diff --git a/chrome/browser/resource_coordinator/tab_activity_watcher.cc b/chrome/browser/resource_coordinator/tab_activity_watcher.cc
index cb42c263b7b9..211c2e1949c8 100644
--- a/chrome/browser/resource_coordinator/tab_activity_watcher.cc
+++ b/chrome/browser/resource_coordinator/tab_activity_watcher.cc
@@ -22,7 +22,6 @@
 #include "chrome/browser/ui/browser_list.h"
 #include "chrome/browser/ui/browser_window.h"
 #include "chrome/browser/ui/tabs/tab_strip_model.h"
-#include "components/ukm/content/source_url_recorder.h"
 #include "content/public/browser/browser_context.h"
 #include "content/public/browser/navigation_handle.h"
 #include "content/public/browser/render_view_host.h"
@@ -30,7 +29,6 @@
 #include "content/public/browser/web_contents.h"
 #include "content/public/browser/web_contents_observer.h"
 #include "content/public/browser/web_contents_user_data.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "third_party/blink/public/common/input/web_input_event.h"
 
 // Use a 1-day max for tab visibility histograms since it's not uncommon to keep
@@ -51,18 +49,6 @@ constexpr float kFrecencyScoreDecay = 0.8f;
 static int32_t reactivation_index = 0;
 // Used for generating label_ids and query_ids.
 int64_t internal_id_for_logging = 0;
-// Returns an int64_t number as label_id or query_id.
-int64_t NewInt64ForLabelIdOrQueryId() {
-  // The id is shifted 16 bits so that the lower bits are reserved for counting
-  // multiple queries.
-  // We choose 16 so that the lower bits for counting multiple queries and
-  // higher bits for labeling queries are both unlikely to overflow. (lower bits
-  // only overflows when we have more than 65536 queries without labeling
-  // events; higher bits only overflow when we have more than 100 billion
-  // discards.
-  constexpr int kIdShiftBits = 16;
-  return (++internal_id_for_logging) << kIdShiftBits;
-}
 
 }  // namespace
 
@@ -115,9 +101,6 @@ class TabActivityWatcher::WebContentsData
     // Copy the replaced tab's stats.
     page_metrics_ = replaced_tab.page_metrics_;
 
-    // Recover the ukm_source_id from the |replaced_tab|.
-    ukm_source_id_ = replaced_tab.ukm_source_id_;
-
     // Copy the replaced label_id_.
     label_id_ = replaced_tab.label_id_;
 
@@ -154,35 +137,6 @@ class TabActivityWatcher::WebContentsData
     }
   }
 
-  // Logs TabMetrics for the tab if it is considered to be backgrounded.
-  void LogTabIfBackgrounded() {
-    if (backgrounded_time_.is_null() || DisableBackgroundLogWithTabRanker())
-      return;
-
-    base::Optional<TabFeatures> tab = GetTabFeatures();
-    if (tab.has_value()) {
-      // Background time logging always logged with label_id == 0, since we
-      // only use label_id for query time logging for now.
-      TabActivityWatcher::GetInstance()->tab_metrics_logger_->LogTabMetrics(
-          ukm_source_id_, tab.value(), web_contents(), 0);
-    }
-  }
-
-  // Logs current TabFeatures; skips if current tab is null.
-  void LogCurrentTabFeatures(const base::Optional<TabFeatures>& tab) {
-    if (!tab.has_value())
-      return;
-    // Update label_id_: a new label_id is generated for this query if the
-    // label_id_ is 0; otherwise the old label_id_ is incremented. This allows
-    // us to better pairing TabMetrics with ForegroundedOrClosed events offline.
-    // The same label_id_ will be logged with ForegroundedOrClosed event later
-    // on so that TabFeatures can be paired with ForegroundedOrClosed.
-    label_id_ = label_id_ ? label_id_ + 1 : NewInt64ForLabelIdOrQueryId();
-
-    TabActivityWatcher::GetInstance()->tab_metrics_logger_->LogTabMetrics(
-        ukm_source_id_, tab.value(), web_contents(), label_id_);
-  }
-
   // Sets foregrounded_time_ to NowTicks() so this becomes the
   // most-recently-used tab.
   void TabWindowActivated() { foregrounded_time_ = NowTicks(); }
@@ -208,9 +162,6 @@ class TabActivityWatcher::WebContentsData
 
     creation_time_ = NowTicks();
 
-    // A navigation may already have completed if this is a replacement tab.
-    ukm_source_id_ = ukm::GetSourceIdForWebContentsDocument(web_contents);
-
     // When a tab is discarded, a new null_web_contents will be created (with
     // WasDiscarded set as true) applied as a replacement of the discarded tab.
     // We want to record this discarded state for later logging.
@@ -238,7 +189,6 @@ class TabActivityWatcher::WebContentsData
 
     backgrounded_time_ = NowTicks();
     discarded_since_backgrounded_ = false;
-    LogTabIfBackgrounded();
   }
 
   void WasShown() {
@@ -251,9 +201,6 @@ class TabActivityWatcher::WebContentsData
     if (browser && browser->tab_strip_model()->closing_all())
       return;
 
-    // Log the event before updating times.
-    LogForegroundedOrClosedMetrics(true /* is_foregrounded */);
-
     backgrounded_time_ = base::TimeTicks();
     foregrounded_time_ = NowTicks();
 
@@ -276,14 +223,6 @@ class TabActivityWatcher::WebContentsData
       return;
     }
 
-    // Use the same SourceId that SourceUrlRecorderWebContentsObserver populates
-    // and updates.
-    ukm::SourceId new_source_id = ukm::ConvertToSourceId(
-        navigation_handle->GetNavigationId(), ukm::SourceIdType::NAVIGATION_ID);
-    DCHECK_NE(new_source_id, ukm_source_id_)
-        << "Expected a unique Source ID for the navigation";
-    ukm_source_id_ = new_source_id;
-
     // Update navigation time for UKM reporting.
     navigation_time_ = navigation_handle->NavigationStart();
 
@@ -298,9 +237,6 @@ class TabActivityWatcher::WebContentsData
   // after a navigation commits, so we can have some idea of its status and
   // contents.
   void DidStopLoading() override {
-    // Ignore load events in foreground tabs. The tab state of a foreground tab
-    // will be logged if/when it is backgrounded.
-    LogTabIfBackgrounded();
   }
 
   void OnVisibilityChanged(content::Visibility visibility) override {
@@ -412,55 +348,7 @@ class TabActivityWatcher::WebContentsData
   // TODO(charleszhao): refactor TabMetricsLogger::GetTabFeatures to return a
   // full TabFeatures instead of a partial TabFeatures.
   base::Optional<TabFeatures> GetTabFeatures() {
-    if (web_contents()->IsBeingDestroyed() || backgrounded_time_.is_null())
-      return base::nullopt;
-    // For tab features.
-    base::Optional<TabFeatures> tab =
-        TabMetricsLogger::GetTabFeatures(page_metrics_, web_contents());
-    if (!tab.has_value())
-      return tab;
-
-    tab->time_from_backgrounded =
-        backgrounded_time_.is_null()
-            ? 0
-            : (NowTicks() - backgrounded_time_).InMilliseconds();
-
-    // For mru features.
-    const tab_ranker::MRUFeatures& mru = GetMRUFeatures();
-    tab->mru_index = mru.index;
-    tab->total_tab_count = mru.total;
-
-    // For frecency_score;
-    tab->frecency_score = GetFrecencyScore();
-
-    return tab;
-  }
-
-  // Collect current ForegroundedOrClosedMetrics and send to ukm.
-  void LogForegroundedOrClosedMetrics(bool is_foregrounded) {
-    // If background time logging is disabled, then we only log the case where
-    // the label_id_ != 0 (a feature is logged and a label has not been logged).
-    if (DisableBackgroundLogWithTabRanker() && label_id_ == 0)
-      return;
-
-    TabMetricsLogger::ForegroundedOrClosedMetrics metrics;
-    metrics.is_foregrounded = is_foregrounded;
-    metrics.is_discarded = discarded_since_backgrounded_;
-    metrics.time_from_backgrounded =
-        (NowTicks() - backgrounded_time_).InMilliseconds();
-    metrics.label_id = label_id_;
-
-    TabActivityWatcher::GetInstance()
-        ->tab_metrics_logger_->LogForegroundedOrClosedMetrics(ukm_source_id_,
-                                                              metrics);
-    // label_id_ is reset whenever a label is logged.
-    // A new label_id_ is generated when a query happens inside
-    // CalculateReactivationScore, after that this ForegroundedOrClosed logging
-    // can happen many times (tabs may get backgrounded and reactivated several
-    // times). In such cases, we only count the first time as the true label,
-    // the rest are considered to be query time logging irrelevant, for which we
-    // log with label_id == 0.
-    label_id_ = 0;
+    return base::nullopt;
   }
 
   // Returns frecency score of this tab.
@@ -485,9 +373,6 @@ class TabActivityWatcher::WebContentsData
     frecency_score_.score = GetFrecencyScore() + 1.0f - kFrecencyScoreDecay;
   }
 
-  // Updated when a navigation is finished.
-  ukm::SourceId ukm_source_id_ = 0;
-
   // When the tab was created.
   base::TimeTicks creation_time_;
 
@@ -539,8 +424,7 @@ class TabActivityWatcher::WebContentsData
 WEB_CONTENTS_USER_DATA_KEY_IMPL(TabActivityWatcher::WebContentsData)
 
 TabActivityWatcher::TabActivityWatcher()
-    : tab_metrics_logger_(std::make_unique<TabMetricsLogger>()),
-      browser_tab_strip_tracker_(this, this),
+    : browser_tab_strip_tracker_(this, this),
       predictor_(std::make_unique<tab_ranker::TabScorePredictor>()) {
   BrowserList::AddObserver(this);
   browser_tab_strip_tracker_.Init();
@@ -561,9 +445,6 @@ base::Optional<float> TabActivityWatcher::CalculateReactivationScore(
 
 void TabActivityWatcher::LogAndMaybeSortLifecycleUnitWithTabRanker(
     std::vector<LifecycleUnit*>* tabs) {
-  // Set query_id so that all TabFeatures logged in this query can be joined.
-  tab_metrics_logger_->set_query_id(NewInt64ForLabelIdOrQueryId());
-
   const bool should_sort_tabs =
       base::FeatureList::IsEnabled(features::kTabRanker);
 
@@ -589,7 +470,6 @@ void TabActivityWatcher::LogAndMaybeSortLifecycleUnitWithTabRanker(
     }
 
     const base::Optional<TabFeatures> tab = web_contents_data->GetTabFeatures();
-    web_contents_data->LogCurrentTabFeatures(tab);
 
     // No reason to store TabFeatures if TabRanker is disabled.
     if (should_sort_tabs) {
@@ -672,7 +552,6 @@ void TabActivityWatcher::OnTabStripModelChanged(
 void TabActivityWatcher::TabPinnedStateChanged(TabStripModel* tab_strip_model,
                                                content::WebContents* contents,
                                                int index) {
-  WebContentsData::FromWebContents(contents)->LogTabIfBackgrounded();
 }
 
 bool TabActivityWatcher::ShouldTrackBrowser(Browser* browser) {
@@ -684,7 +563,6 @@ bool TabActivityWatcher::ShouldTrackBrowser(Browser* browser) {
 }
 
 void TabActivityWatcher::ResetForTesting() {
-  tab_metrics_logger_ = std::make_unique<TabMetricsLogger>();
   predictor_ = std::make_unique<tab_ranker::TabScorePredictor>();
   internal_id_for_logging = 0;
 }
@@ -696,16 +574,6 @@ TabActivityWatcher* TabActivityWatcher::GetInstance() {
 }
 
 void TabActivityWatcher::OnTabClosed(WebContentsData* web_contents_data) {
-  // Log TabLifetime event.
-  tab_metrics_logger_->LogTabLifetime(
-      web_contents_data->ukm_source_id_,
-      NowTicks() - web_contents_data->navigation_time_);
-
-  // Log ForegroundedOrClosed event.
-  if (!web_contents_data->backgrounded_time_.is_null()) {
-    web_contents_data->LogForegroundedOrClosedMetrics(
-        false /*is_foregrounded */);
-  }
 }
 
 }  // namespace resource_coordinator
