diff --git a/chrome/browser/resources/new_tab_page/BUILD.gn b/chrome/browser/resources/new_tab_page/BUILD.gn
index 2b0e896532bc8..7565088e433e7 100644
--- a/chrome/browser/resources/new_tab_page/BUILD.gn
+++ b/chrome/browser/resources/new_tab_page/BUILD.gn
@@ -61,7 +61,6 @@ js_library("new_tab_page") {
 js_library("lazy_load") {
   deps = [
     ":customize_dialog",
-    ":middle_slot_promo",
     ":most_visited",
     ":voice_search_overlay",
     "modules/dummy:module",
