diff --git a/chrome/renderer/extensions/chrome_extensions_renderer_client.h b/chrome/renderer/extensions/chrome_extensions_renderer_client.h
index a70ac78af01d..eb27aa0f4014 100644
--- a/chrome/renderer/extensions/chrome_extensions_renderer_client.h
+++ b/chrome/renderer/extensions/chrome_extensions_renderer_client.h
@@ -44,10 +44,6 @@ namespace url {
 class Origin;
 }
 
-namespace ukm {
-class MojoUkmRecorder;
-}
-
 namespace v8 {
 class Isolate;
 template <typename T>
@@ -116,7 +112,6 @@ class ChromeExtensionsRendererClient
   }
 
  private:
-  std::unique_ptr<ukm::MojoUkmRecorder> ukm_recorder_;
   std::unique_ptr<extensions::Dispatcher> extension_dispatcher_;
   std::unique_ptr<extensions::RendererPermissionsPolicyDelegate>
       permissions_policy_delegate_;
