diff --git a/chromecast/browser/extensions/api/tabs/tabs_constants.cc b/chromecast/browser/extensions/api/tabs/tabs_constants.cc
index eeabb9e849b8..df10b0b7f1c5 100644
--- a/chromecast/browser/extensions/api/tabs/tabs_constants.cc
+++ b/chromecast/browser/extensions/api/tabs/tabs_constants.cc
@@ -110,6 +110,8 @@ const char kSupportedInWindowsOnlyError[] = "Supported in Windows only";
 const char kInvalidWindowTypeError[] = "Invalid value for type";
 const char kInvalidWindowStateError[] = "Invalid value for state";
 const char kScreenshotsDisabled[] = "Taking screenshots has been disabled";
+const char kCannotUpdateMuteDisabled[] =
+    "Failed to update mute state for tab *, --* must be enabled";
 const char kCannotUpdateMuteCaptured[] =
     "Cannot update mute state for tab *, tab has audio or video currently "
     "being captured";
