diff --git a/chrome/browser/translate/chrome_translate_client.cc b/chrome/browser/translate/chrome_translate_client.cc
index 694e406a96ad..37742781f8f3 100644
--- a/chrome/browser/translate/chrome_translate_client.cc
+++ b/chrome/browser/translate/chrome_translate_client.cc
@@ -61,27 +61,6 @@ namespace {
 using base::FeatureList;
 using metrics::TranslateEventProto;
 
-#if !defined(OS_ANDROID)
-TranslateEventProto::EventType BubbleResultToTranslateEvent(
-    ShowTranslateBubbleResult result) {
-  switch (result) {
-    case ShowTranslateBubbleResult::BROWSER_WINDOW_NOT_VALID:
-      return TranslateEventProto::BROWSER_WINDOW_IS_INVALID;
-    case ShowTranslateBubbleResult::BROWSER_WINDOW_MINIMIZED:
-      return TranslateEventProto::BROWSER_WINDOW_IS_MINIMIZED;
-    case ShowTranslateBubbleResult::BROWSER_WINDOW_NOT_ACTIVE:
-      return TranslateEventProto::BROWSER_WINDOW_NOT_ACTIVE;
-    case ShowTranslateBubbleResult::WEB_CONTENTS_NOT_ACTIVE:
-      return TranslateEventProto::WEB_CONTENTS_NOT_ACTIVE;
-    case ShowTranslateBubbleResult::EDITABLE_FIELD_IS_ACTIVE:
-      return TranslateEventProto::EDITABLE_FIELD_IS_ACTIVE;
-    default:
-      NOTREACHED();
-      return metrics::TranslateEventProto::UNKNOWN;
-  }
-}
-#endif
-
 }  // namespace
 
 ChromeTranslateClient::ChromeTranslateClient(content::WebContents* web_contents)
@@ -254,13 +233,6 @@ bool ChromeTranslateClient::ShowTranslateUI(
     return false;
   }
 
-  ShowTranslateBubbleResult result =
-      ShowBubble(step, source_language, target_language, error_type);
-  if (result != ShowTranslateBubbleResult::SUCCESS &&
-      step == translate::TRANSLATE_STEP_BEFORE_TRANSLATE) {
-    translate_manager_->RecordTranslateEvent(
-        BubbleResultToTranslateEvent(result));
-  }
 #endif
 
   return true;
