diff --git a/chrome/browser/component_updater/registration.cc b/chrome/browser/component_updater/registration.cc
index e4d08bbf6754..35ddf19672ad 100644
--- a/chrome/browser/component_updater/registration.cc
+++ b/chrome/browser/component_updater/registration.cc
@@ -12,7 +12,6 @@
 #include "chrome/browser/component_updater/crl_set_component_installer.h"
 #include "chrome/browser/component_updater/crowd_deny_component_installer.h"
 #include "chrome/browser/component_updater/file_type_policies_component_installer.h"
-#include "chrome/browser/component_updater/floc_blocklist_component_installer.h"
 #include "chrome/browser/component_updater/games_component_installer.h"
 #include "chrome/browser/component_updater/mei_preload_component_installer.h"
 #include "chrome/browser/component_updater/optimization_hints_component_installer.h"
@@ -118,9 +117,6 @@ void RegisterComponentsForUpdate(bool is_off_the_record_profile,
   whitelist_installer->RegisterComponents();
 #endif
 
-  RegisterSubresourceFilterComponent(cus);
-  RegisterFlocBlocklistComponent(cus,
-                                 g_browser_process->floc_blocklist_service());
   RegisterOnDeviceHeadSuggestComponent(
       cus, g_browser_process->GetApplicationLocale());
   RegisterOptimizationHintsComponent(cus, is_off_the_record_profile);
@@ -148,8 +144,6 @@ void RegisterComponentsForUpdate(bool is_off_the_record_profile,
 
     RegisterOriginTrialsComponent(cus, path);
 
-    RegisterFileTypePoliciesComponent(cus, path);
-
     RegisterTrustTokenKeyCommitmentsComponentIfTrustTokensEnabled(cus, path);
 
     RegisterSSLErrorAssistantComponent(cus, path);
@@ -162,7 +156,6 @@ void RegisterComponentsForUpdate(bool is_off_the_record_profile,
   // on chromium build bots, it is always registered here and
   // RegisterSwReporterComponent() has support for running only in official
   // builds or tests.
-  RegisterSwReporterComponent(cus);
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
   RegisterThirdPartyModuleListComponent(cus);
 #endif  // BUILDFLAG(GOOGLE_CHROME_BRANDING)
