diff --git a/chrome/browser/policy/chrome_browser_cloud_management_controller_desktop.cc b/chrome/browser/policy/chrome_browser_cloud_management_controller_desktop.cc
index f4f4b5165bcbf..6ed52c02e7f4a 100644
--- a/chrome/browser/policy/chrome_browser_cloud_management_controller_desktop.cc
+++ b/chrome/browser/policy/chrome_browser_cloud_management_controller_desktop.cc
@@ -193,6 +193,7 @@ ChromeBrowserCloudManagementControllerDesktop::GetSharedURLLoaderFactory() {
       ->GetSharedURLLoaderFactory();
 }
 
+#if 0
 std::unique_ptr<enterprise_reporting::ReportScheduler>
 ChromeBrowserCloudManagementControllerDesktop::CreateReportScheduler(
     CloudPolicyClient* client) {
@@ -205,6 +206,7 @@ ChromeBrowserCloudManagementControllerDesktop::CreateReportScheduler(
       client, std::move(generator), std::move(real_time_generator),
       &reporting_delegate_factory_);
 }
+#endif
 
 scoped_refptr<base::SingleThreadTaskRunner>
 ChromeBrowserCloudManagementControllerDesktop::GetBestEffortTaskRunner() {
@@ -258,6 +260,7 @@ void ChromeBrowserCloudManagementControllerDesktop::StartInvalidations() {
       0 /* highest_handled_invalidation_version */);
   policy_invalidator_->Initialize(invalidation_service_.get());
 
+#if 0
   if (base::FeatureList::IsEnabled(policy::features::kCBCMRemoteCommands)) {
     g_browser_process->browser_policy_connector()
         ->machine_level_user_cloud_policy_manager()
@@ -273,6 +276,7 @@ void ChromeBrowserCloudManagementControllerDesktop::StartInvalidations() {
         base::DefaultClock::GetInstance(), PolicyInvalidationScope::kCBCM);
     commands_invalidator_->Initialize(invalidation_service_.get());
   }
+#endif
 }
 
 scoped_refptr<network::SharedURLLoaderFactory>
