diff --git a/chrome/browser/ui/passwords/bubble_controllers/items_bubble_controller.cc b/chrome/browser/ui/passwords/bubble_controllers/items_bubble_controller.cc
index 7a25478f0753..128043e340f1 100644
--- a/chrome/browser/ui/passwords/bubble_controllers/items_bubble_controller.cc
+++ b/chrome/browser/ui/passwords/bubble_controllers/items_bubble_controller.cc
@@ -10,7 +10,6 @@
 #include "chrome/browser/ui/passwords/passwords_model_delegate.h"
 #include "chrome/grit/generated_resources.h"
 #include "components/favicon/core/favicon_util.h"
-#include "components/password_manager/core/browser/password_form_metrics_recorder.h"
 #include "components/password_manager/core/browser/password_store.h"
 #include "content/public/browser/web_contents.h"
 #include "ui/base/l10n/l10n_util.h"
@@ -91,10 +90,6 @@ void ItemsBubbleController::OnFaviconReady(
 }
 
 void ItemsBubbleController::ReportInteractions() {
-  metrics_util::LogGeneralUIDismissalReason(dismissal_reason_);
-  // Record UKM statistics on dismissal reason.
-  if (metrics_recorder_)
-    metrics_recorder_->RecordUIDismissalReason(dismissal_reason_);
 }
 
 base::string16 ItemsBubbleController::GetTitle() const {
