diff --git a/chrome/browser/resources/BUILD.gn b/chrome/browser/resources/BUILD.gn
index fc31b0249f126..5ad18feacbb29 100644
--- a/chrome/browser/resources/BUILD.gn
+++ b/chrome/browser/resources/BUILD.gn
@@ -91,7 +91,6 @@ if (enable_js_type_check) {
     deps = [
       "accessibility:closure_compile",
       "components:closure_compile",
-      "domain_reliability_internals:closure_compile",
       "engagement:closure_compile",
       "media:closure_compile",
       "memory_internals:closure_compile",
