diff --git a/chrome/browser/predictors/loading_predictor.cc b/chrome/browser/predictors/loading_predictor.cc
index f2fb8573f3e79..e8625ba88b551 100644
--- a/chrome/browser/predictors/loading_predictor.cc
+++ b/chrome/browser/predictors/loading_predictor.cc
@@ -8,8 +8,6 @@
 #include <vector>
 
 #include "base/metrics/histogram_macros.h"
-#include "chrome/browser/predictors/loading_data_collector.h"
-#include "chrome/browser/predictors/loading_stats_collector.h"
 #include "chrome/browser/predictors/navigation_id.h"
 #include "chrome/browser/predictors/predictors_features.h"
 #include "chrome/browser/predictors/resource_prefetch_predictor.h"
@@ -58,14 +56,7 @@ LoadingPredictor::LoadingPredictor(const LoadingPredictorConfig& config,
     : config_(config),
       profile_(profile),
       resource_prefetch_predictor_(
-          std::make_unique<ResourcePrefetchPredictor>(config, profile)),
-      stats_collector_(std::make_unique<LoadingStatsCollector>(
-          resource_prefetch_predictor_.get(),
-          config)),
-      loading_data_collector_(std::make_unique<LoadingDataCollector>(
-          resource_prefetch_predictor_.get(),
-          stats_collector_.get(),
-          config)) {}
+          std::make_unique<ResourcePrefetchPredictor>(config, profile)) {}
 
 LoadingPredictor::~LoadingPredictor() {
   DCHECK(shutdown_);
@@ -133,10 +124,6 @@ void LoadingPredictor::StartInitialization() {
   resource_prefetch_predictor_->StartInitialization();
 }
 
-LoadingDataCollector* LoadingPredictor::loading_data_collector() {
-  return loading_data_collector_.get();
-}
-
 ResourcePrefetchPredictor* LoadingPredictor::resource_prefetch_predictor() {
   return resource_prefetch_predictor_.get();
 }
@@ -178,7 +165,6 @@ bool LoadingPredictor::OnNavigationStarted(const NavigationID& navigation_id) {
   if (shutdown_)
     return true;
 
-  loading_data_collector()->RecordStartNavigation(navigation_id);
   CleanupAbandonedHintsAndNavigations(navigation_id);
   active_navigations_.emplace(navigation_id);
   active_urls_to_navigations_[navigation_id.main_frame_url].insert(
@@ -194,8 +180,6 @@ void LoadingPredictor::OnNavigationFinished(
   if (shutdown_)
     return;
 
-  loading_data_collector()->RecordFinishNavigation(
-      old_navigation_id, new_navigation_id, is_error_page);
   if (active_urls_to_navigations_.find(old_navigation_id.main_frame_url) !=
       active_urls_to_navigations_.end()) {
     active_urls_to_navigations_[old_navigation_id.main_frame_url].erase(
@@ -304,9 +288,6 @@ void LoadingPredictor::PreconnectInitiated(const GURL& url,
   auto nav_id_set_it = active_urls_to_navigations_.find(url);
   if (nav_id_set_it == active_urls_to_navigations_.end())
     return;
-
-  for (const auto& nav_id : nav_id_set_it->second)
-    loading_data_collector_->RecordPreconnectInitiated(nav_id, preconnect_url);
 }
 
 void LoadingPredictor::PreconnectFinished(
@@ -317,7 +298,6 @@ void LoadingPredictor::PreconnectFinished(
 
   DCHECK(stats);
   active_hints_.erase(stats->url);
-  stats_collector_->RecordPreconnectStats(std::move(stats));
 }
 
 void LoadingPredictor::PrefetchInitiated(const GURL& url,
@@ -329,9 +309,6 @@ void LoadingPredictor::PrefetchInitiated(const GURL& url,
   auto nav_id_set_it = active_urls_to_navigations_.find(url);
   if (nav_id_set_it == active_urls_to_navigations_.end())
     return;
-
-  for (const auto& nav_id : nav_id_set_it->second)
-    loading_data_collector_->RecordPrefetchInitiated(nav_id, prefetch_url);
 }
 
 void LoadingPredictor::PrefetchFinished(std::unique_ptr<PrefetchStats> stats) {
