diff --git a/chrome/browser/background_sync/background_sync_delegate_impl.h b/chrome/browser/background_sync/background_sync_delegate_impl.h
index eef61ecd5e004..f6f152ee94fa4 100644
--- a/chrome/browser/background_sync/background_sync_delegate_impl.h
+++ b/chrome/browser/background_sync/background_sync_delegate_impl.h
@@ -22,10 +22,6 @@ class HostContentSettingsMap;
 class ScopedKeepAlive;
 class ScopedProfileKeepAlive;
 
-namespace ukm {
-class UkmBackgroundRecorderService;
-}  // namespace ukm
-
 // Chrome's customization of the logic in components/background_sync
 class BackgroundSyncDelegateImpl
     : public background_sync::BackgroundSyncDelegate,
@@ -87,7 +83,6 @@ class BackgroundSyncDelegateImpl
  private:
   Profile* profile_;
   bool off_the_record_;
-  ukm::UkmBackgroundRecorderService* ukm_background_service_;
   // Same lifetime as |profile_|.
   site_engagement::SiteEngagementService* site_engagement_service_;
   std::set<url::Origin> suspended_periodic_sync_origins_;
