diff --git a/chrome/installer/setup/BUILD.gn b/chrome/installer/setup/BUILD.gn
index 7bf0b6d6db61..4548dc8f1416 100644
--- a/chrome/installer/setup/BUILD.gn
+++ b/chrome/installer/setup/BUILD.gn
@@ -38,11 +38,7 @@ if (is_win) {
       "//chrome/common:constants",
       "//chrome/install_static:install_static_util",
       "//chrome/installer/util:with_rc_strings",
-      "//components/crash/core/app:app",
-      "//components/crash/core/app:crash_export_thunks",
-      "//components/crash/core/app:run_as_crashpad_handler",
       "//content/public/common:static_switches",
-      "//rlz:rlz_lib_no_network",
     ]
 
     libs = [ "netapi32.lib" ]
@@ -59,10 +55,6 @@ if (is_win) {
       "install_params.h",
       "install_worker.cc",
       "install_worker.h",
-      "installer_crash_reporter_client.cc",
-      "installer_crash_reporter_client.h",
-      "installer_crash_reporting.cc",
-      "installer_crash_reporting.h",
       "installer_state.cc",
       "installer_state.h",
       "launch_chrome.cc",
@@ -101,15 +93,11 @@ if (is_win) {
       "//chrome/install_static:buildflags",
       "//chrome/install_static:install_static_util",
       "//components/base32",
-      "//components/crash/core/app",
-      "//components/crash/core/common",
       "//components/metrics",
       "//components/zucchini:zucchini_io",
       "//components/zucchini:zucchini_lib",
       "//courgette:courgette_lib",
       "//third_party/bspatch",
-      "//third_party/crashpad/crashpad/client",
-      "//third_party/crashpad/crashpad/util",
       "//ui/base:fullscreen_win",
     ]
   }
@@ -154,7 +142,6 @@ if (is_win) {
       "//chrome/installer/mini_installer:unit_tests",
       "//chrome/installer/util:serial_unittests",
       "//chrome/installer/util:test_support",
-      "//components/crash/core/app:test_support",
       "//testing/gmock",
       "//testing/gtest",
     ]
