diff --git a/chrome/browser/renderer_host/chrome_navigation_ui_data.cc b/chrome/browser/renderer_host/chrome_navigation_ui_data.cc
index 78982fd08979e..e84e86c0728b8 100644
--- a/chrome/browser/renderer_host/chrome_navigation_ui_data.cc
+++ b/chrome/browser/renderer_host/chrome_navigation_ui_data.cc
@@ -6,7 +6,6 @@
 
 #include "chrome/browser/prefetch/no_state_prefetch/chrome_no_state_prefetch_contents_delegate.h"
 #include "components/no_state_prefetch/browser/no_state_prefetch_contents.h"
-#include "components/no_state_prefetch/browser/prerender_histograms.h"
 #include "content/public/browser/navigation_handle.h"
 #include "extensions/buildflags/buildflags.h"
 #include "ui/base/window_open_disposition.h"
@@ -40,9 +39,6 @@ ChromeNavigationUIData::ChromeNavigationUIData(
           web_contents);
   if (no_state_prefetch_contents) {
     is_no_state_prefetching_ = true;
-    prerender_histogram_prefix_ =
-        prerender::PrerenderHistograms::GetHistogramPrefix(
-            no_state_prefetch_contents->origin());
   }
 }
 
@@ -93,7 +89,6 @@ std::unique_ptr<content::NavigationUIData> ChromeNavigationUIData::Clone() {
 #endif
 
   copy->is_no_state_prefetching_ = is_no_state_prefetching_;
-  copy->prerender_histogram_prefix_ = prerender_histogram_prefix_;
 
   return std::move(copy);
 }
