diff --git a/chrome/browser/resources/settings/site_settings_page/site_settings_page.js b/chrome/browser/resources/settings/site_settings_page/site_settings_page.js
index 5349a0bd98c8d..406f3ca144c6f 100644
--- a/chrome/browser/resources/settings/site_settings_page/site_settings_page.js
+++ b/chrome/browser/resources/settings/site_settings_page/site_settings_page.js
@@ -55,8 +55,7 @@ function getCategoryItemMap() {
                                       'siteSettingsAllowed',
       disabledLabel: redesignEnabled ? 'siteSettingsAdsBlocked' :
                                        'siteSettingsAdsBlock',
-      shouldShow: () =>
-          loadTimeData.getBoolean('enableSafeBrowsingSubresourceFilter'),
+      shouldShow: () => false,
     },
     {
       route: routes.SITE_SETTINGS_AR,
