diff --git a/chrome/browser/web_applications/web_app_install_task.h b/chrome/browser/web_applications/web_app_install_task.h
index 437a54afc59b0..02f1e6121174d 100644
--- a/chrome/browser/web_applications/web_app_install_task.h
+++ b/chrome/browser/web_applications/web_app_install_task.h
@@ -19,7 +19,6 @@
 #include "chrome/browser/web_applications/components/web_app_install_utils.h"
 #include "chrome/browser/web_applications/components/web_app_url_loader.h"
 #include "chrome/browser/web_applications/components/web_application_info.h"
-#include "components/webapps/browser/installable/installable_metrics.h"
 #include "content/public/browser/web_contents_observer.h"
 
 class GURL;
@@ -164,7 +163,6 @@ class WebAppInstallTask : content::WebContentsObserver {
 
  private:
   void CheckInstallPreconditions();
-  void RecordInstallEvent();
 
   // Calling the callback may destroy |this| task. Callers shouldn't work with
   // any |this| class members after calling it.
