diff --git a/chrome/browser/ui/passwords/manage_passwords_ui_controller.cc b/chrome/browser/ui/passwords/manage_passwords_ui_controller.cc
index 1bdc6041c6a8..5fff7cd16a34 100644
--- a/chrome/browser/ui/passwords/manage_passwords_ui_controller.cc
+++ b/chrome/browser/ui/passwords/manage_passwords_ui_controller.cc
@@ -354,15 +354,6 @@ url::Origin ManagePasswordsUIController::GetOrigin() const {
   return passwords_data_.origin();
 }
 
-password_manager::PasswordFormMetricsRecorder*
-ManagePasswordsUIController::GetPasswordFormMetricsRecorder() {
-  // The form manager may be null for example for auto sign-in toasts of the
-  // credential manager API.
-  password_manager::PasswordFormManagerForUI* form_manager =
-      passwords_data_.form_manager();
-  return form_manager ? form_manager->GetMetricsRecorder() : nullptr;
-}
-
 password_manager::PasswordFeatureManager*
 ManagePasswordsUIController::GetPasswordFeatureManager() {
   password_manager::PasswordManagerClient* client = passwords_data_.client();
@@ -498,11 +489,7 @@ void ManagePasswordsUIController::SavePassword(const base::string16& username,
 
   UMA_HISTOGRAM_BOOLEAN("PasswordManager.PasswordSavedWithManualFallback",
                         BubbleIsManualFallbackForSaving());
-  if (GetPasswordFormMetricsRecorder() && BubbleIsManualFallbackForSaving()) {
-    GetPasswordFormMetricsRecorder()->RecordDetailedUserAction(
-        password_manager::PasswordFormMetricsRecorder::DetailedUserAction::
-            kTriggeredManualFallbackForSaving);
-  }
+
   save_fallback_timer_.Stop();
   passwords_data_.form_manager()->Save();
 
