diff --git a/chrome/browser/extensions/api/cryptotoken_private/cryptotoken_private_api.cc b/chrome/browser/extensions/api/cryptotoken_private/cryptotoken_private_api.cc
index 4190dc3f2cfae..c6e12946faa6f 100644
--- a/chrome/browser/extensions/api/cryptotoken_private/cryptotoken_private_api.cc
+++ b/chrome/browser/extensions/api/cryptotoken_private/cryptotoken_private_api.cc
@@ -16,7 +16,6 @@
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/common/chrome_features.h"
 #include "chrome/common/pref_names.h"
-#include "components/page_load_metrics/browser/metrics_web_contents_observer.h"
 #include "components/permissions/permission_request_manager.h"
 #include "components/pref_registry/pref_registry_syncable.h"
 #include "components/prefs/pref_service.h"
@@ -276,8 +275,6 @@ CryptotokenPrivateRecordRegisterRequestFunction::Run() {
     return RespondNow(Error("cannot find specified tab or frame"));
   }
 
-  page_load_metrics::MetricsWebContentsObserver::RecordFeatureUsage(
-      frame, blink::mojom::WebFeature::kU2FCryptotokenRegister);
   return RespondNow(NoArguments());
 }
 
@@ -292,8 +289,6 @@ CryptotokenPrivateRecordSignRequestFunction::Run() {
     return RespondNow(Error("cannot find specified tab or frame"));
   }
 
-  page_load_metrics::MetricsWebContentsObserver::RecordFeatureUsage(
-      frame, blink::mojom::WebFeature::kU2FCryptotokenSign);
   return RespondNow(NoArguments());
 }
 
