diff --git a/chrome/common/features.gni b/chrome/common/features.gni
index 1b2d4f0fffb1..b0436f49684b 100644
--- a/chrome/common/features.gni
+++ b/chrome/common/features.gni
@@ -86,7 +86,6 @@ chrome_grit_defines = [
   "enable_supervised_users=$enable_supervised_users",
   "enable_vr=$enable_vr",
   "enable_webui_tab_strip=$enable_webui_tab_strip",
-  "safe_browsing_mode=$safe_browsing_mode",
   "optimize_webui=$optimize_webui",
   "enable_feed_v1=$enable_feed_v1",
   "enable_feed_v2=$enable_feed_v2",
