diff --git a/chrome/common/chrome_features.cc b/chrome/common/chrome_features.cc
index a26a9c95fb1b..32b8aebbae00 100644
--- a/chrome/common/chrome_features.cc
+++ b/chrome/common/chrome_features.cc
@@ -880,7 +880,7 @@ const base::Feature kTPMFirmwareUpdate{"TPMFirmwareUpdate",
 #if !defined(OS_ANDROID)
 // Enables logging UKMs for background tab activity by TabActivityWatcher.
 const base::Feature kTabMetricsLogging{"TabMetricsLogging",
-                                       base::FEATURE_ENABLED_BY_DEFAULT};
+                                       base::FEATURE_DISABLED_BY_DEFAULT};
 #endif
 
 // Enables the teamfood flags.
