diff --git a/chrome/browser/ui/global_media_controls/media_session_notification_producer.h b/chrome/browser/ui/global_media_controls/media_session_notification_producer.h
index 99d374a35be92..9987449e075ff 100644
--- a/chrome/browser/ui/global_media_controls/media_session_notification_producer.h
+++ b/chrome/browser/ui/global_media_controls/media_session_notification_producer.h
@@ -81,9 +81,6 @@ class MediaSessionNotificationProducer
   // There is at most one session per WebContents.
   std::string GetActiveControllableSessionForWebContents(
       content::WebContents* web_contents) const;
-  void LogMediaSessionActionButtonPressed(
-      const std::string& id,
-      media_session::mojom::MediaSessionAction action);
 
   // Used by a |MediaNotificationDeviceSelectorView| to query the system
   // for connected audio output devices.
