diff --git a/chrome/browser/ui/views/page_action/page_action_icon_controller.cc b/chrome/browser/ui/views/page_action/page_action_icon_controller.cc
index 59936ed9d754f..a227e53b20d34 100644
--- a/chrome/browser/ui/views/page_action/page_action_icon_controller.cc
+++ b/chrome/browser/ui/views/page_action/page_action_icon_controller.cc
@@ -85,6 +85,7 @@ void PageActionIconController::Init(const PageActionIconParams& params,
                                        params.page_action_icon_delegate));
         break;
       case PageActionIconType::kClickToCall:
+#if 0
         add_page_action_icon(
             type, std::make_unique<SharingIconView>(
                       params.icon_label_bubble_delegate,
@@ -96,6 +97,7 @@ void PageActionIconController::Init(const PageActionIconParams& params,
                       }),
                       base::BindRepeating(
                           SharingDialogView::GetAsBubbleForClickToCall)));
+#endif
         break;
       case PageActionIconType::kCookieControls:
         add_page_action_icon(type, std::make_unique<CookieControlsIconView>(
