diff --git a/chrome/browser/ui/views/renderer_context_menu/render_view_context_menu_views.cc b/chrome/browser/ui/views/renderer_context_menu/render_view_context_menu_views.cc
index 0271d19c5c33b..a89d74dea462e 100644
--- a/chrome/browser/ui/views/renderer_context_menu/render_view_context_menu_views.cc
+++ b/chrome/browser/ui/views/renderer_context_menu/render_view_context_menu_views.cc
@@ -284,7 +284,6 @@ void RenderViewContextMenuViews::ExecuteCommand(int command_id,
               ? base::i18n::RIGHT_TO_LEFT
               : base::i18n::LEFT_TO_RIGHT);
       view_host->GetWidget()->NotifyTextDirection();
-      RenderViewContextMenu::RecordUsedItem(command_id);
       break;
     }
 
