diff --git a/chrome/browser/chrome_browser_field_trials.cc b/chrome/browser/chrome_browser_field_trials.cc
index 4205959135d64..7961e2e44c3d0 100644
--- a/chrome/browser/chrome_browser_field_trials.cc
+++ b/chrome/browser/chrome_browser_field_trials.cc
@@ -22,8 +22,6 @@
 #include "chrome/common/chrome_paths.h"
 #include "chrome/common/chrome_switches.h"
 #include "components/metrics/metrics_pref_names.h"
-#include "components/metrics/persistent_histograms.h"
-#include "components/ukm/ukm_recorder_impl.h"
 #include "components/version_info/version_info.h"
 
 #if defined(OS_ANDROID)
@@ -52,13 +50,6 @@ void CreateFallbackSamplingTrialIfNeeded(base::FeatureList* feature_list) {
 #endif  // defined(OS_WIN) || defined(OS_ANDROID)
 }
 
-// Create a field trial to control UKM sampling for Stable if no variations
-// seed was applied.
-void CreateFallbackUkmSamplingTrialIfNeeded(base::FeatureList* feature_list) {
-  ukm::UkmRecorderImpl::CreateFallbackSamplingTrial(
-      chrome::GetChannel() == version_info::Channel::STABLE, feature_list);
-}
-
 }  // namespace
 
 ChromeBrowserFieldTrials::ChromeBrowserFieldTrials(PrefService* local_state)
@@ -89,7 +80,6 @@ void ChromeBrowserFieldTrials::SetupFeatureControllingFieldTrials(
   // is needed.
   if (!has_seed) {
     CreateFallbackSamplingTrialIfNeeded(feature_list);
-    CreateFallbackUkmSamplingTrialIfNeeded(feature_list);
 #if BUILDFLAG(IS_CHROMEOS_ASH)
     chromeos::multidevice_setup::CreateFirstRunFieldTrial(feature_list);
 #endif
@@ -180,9 +170,4 @@ void ChromeBrowserFieldTrials::RegisterSyntheticTrials() {
 }
 
 void ChromeBrowserFieldTrials::InstantiateDynamicTrials() {
-  // Persistent histograms must be enabled as soon as possible.
-  base::FilePath metrics_dir;
-  if (base::PathService::Get(chrome::DIR_USER_DATA, &metrics_dir)) {
-    InstantiatePersistentHistograms(metrics_dir);
-  }
 }
