diff --git a/chrome/browser/ui/global_media_controls/media_notification_service.cc b/chrome/browser/ui/global_media_controls/media_notification_service.cc
index 98c6ede2aedeb..bffc0b2116ff1 100644
--- a/chrome/browser/ui/global_media_controls/media_notification_service.cc
+++ b/chrome/browser/ui/global_media_controls/media_notification_service.cc
@@ -26,14 +26,11 @@
 #include "components/media_message_center/media_notification_util.h"
 #include "components/media_message_center/media_session_notification_item.h"
 #include "components/media_router/browser/presentation/start_presentation_context.h"
-#include "components/ukm/content/source_url_recorder.h"
 #include "content/public/browser/audio_service.h"
 #include "content/public/browser/media_session.h"
 #include "content/public/browser/media_session_service.h"
 #include "media/base/media_switches.h"
 #include "services/media_session/public/mojom/media_session.mojom.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
 
 MediaNotificationService::MediaNotificationService(
     Profile* profile,
@@ -107,13 +104,6 @@ MediaNotificationService::GetTaskRunner() const {
   return nullptr;
 }
 
-void MediaNotificationService::LogMediaSessionActionButtonPressed(
-    const std::string& id,
-    media_session::mojom::MediaSessionAction action) {
-  media_session_notification_producer_->LogMediaSessionActionButtonPressed(
-      id, action);
-}
-
 void MediaNotificationService::Shutdown() {
   // |cast_notification_producer_| and
   // |presentation_request_notification_producer_| depend on MediaRouter,
