diff --git a/chrome/browser/resources/local_ntp/externs.js b/chrome/browser/resources/local_ntp/externs.js
index 11ad0c32056f..b3ec30739e86 100644
--- a/chrome/browser/resources/local_ntp/externs.js
+++ b/chrome/browser/resources/local_ntp/externs.js
@@ -104,17 +104,6 @@ let ddl;
  */
 let og;
 
-/**
- * The type of the middle-slot promo data object. The definition is based on
- * chrome/browser/search/local_ntp_source.cc:
- *     ConvertPromoDataToDict()
- * @typedef {{promoHtml: (string|undefined),
- *            promoLogUrl: (string|undefined),
- *            promoId: (string|undefined),
- *            canOpenExtensionsPage: boolean}}
- */
-let promo;
-
 /**
  * The type of the search suggestions data object. The definition is based on
  * chrome/browser/search/local_ntp_source.cc:
@@ -392,9 +381,6 @@ window.chrome.embeddedSearch.newTabPage.undoMostVisitedDeletion;
  */
 window.chrome.embeddedSearch.newTabPage.updateCustomLink;
 
-/** @param {string} promoId */
-window.chrome.embeddedSearch.newTabPage.blocklistPromo;
-
 /**
  * @param {number} button
  * @param {boolean} altKey
@@ -510,7 +496,6 @@ configData.translatedStrings.copyLink;
 configData.translatedStrings.customizeThisPage;
 configData.translatedStrings.defaultWallpapers;
 configData.translatedStrings.details;
-configData.translatedStrings.dismissPromo;
 configData.translatedStrings.editLinkTitle;
 configData.translatedStrings.editLinkTooltip;
 configData.translatedStrings.fakeboxMicrophoneTooltip;
