diff --git a/chrome/browser/ui/cocoa/renderer_context_menu/render_view_context_menu_mac.mm b/chrome/browser/ui/cocoa/renderer_context_menu/render_view_context_menu_mac.mm
index c5a6e69483ba..93bc56409a92 100644
--- a/chrome/browser/ui/cocoa/renderer_context_menu/render_view_context_menu_mac.mm
+++ b/chrome/browser/ui/cocoa/renderer_context_menu/render_view_context_menu_mac.mm
@@ -86,8 +86,6 @@ void RenderViewContextMenuMac::UpdateTextDirection(
   content::RenderViewHost* view_host = GetRenderViewHost();
   view_host->GetWidget()->UpdateTextDirection(direction);
   view_host->GetWidget()->NotifyTextDirection();
-
-  RenderViewContextMenu::RecordUsedItem(command_id);
 }
 
 void RenderViewContextMenuMac::AppendPlatformEditableItems() {
