diff --git a/chrome/browser/extensions/api/autofill_private/autofill_private_api.h b/chrome/browser/extensions/api/autofill_private/autofill_private_api.h
index 64e55183c8ae..fb753b555c0d 100644
--- a/chrome/browser/extensions/api/autofill_private/autofill_private_api.h
+++ b/chrome/browser/extensions/api/autofill_private/autofill_private_api.h
@@ -189,25 +189,6 @@ class AutofillPrivateMigrateCreditCardsFunction : public ExtensionFunction {
   DISALLOW_COPY_AND_ASSIGN(AutofillPrivateMigrateCreditCardsFunction);
 };
 
-class AutofillPrivateLogServerCardLinkClickedFunction
-    : public ExtensionFunction {
- public:
-  AutofillPrivateLogServerCardLinkClickedFunction();
-  DECLARE_EXTENSION_FUNCTION("autofillPrivate.logServerCardLinkClicked",
-                             AUTOFILLPRIVATE_SERVERCARDLINKCLICKED)
-
- protected:
-  ~AutofillPrivateLogServerCardLinkClickedFunction() override;
-
-  // ExtensionFunction overrides.
-  ResponseAction Run() override;
-
- private:
-  ChromeExtensionFunctionDetails chrome_details_;
-
-  DISALLOW_COPY_AND_ASSIGN(AutofillPrivateLogServerCardLinkClickedFunction);
-};
-
 class AutofillPrivateSetCreditCardFIDOAuthEnabledStateFunction
     : public ExtensionFunction {
  public:
