diff --git a/chrome/browser/chrome_browser_main_mac.mm b/chrome/browser/chrome_browser_main_mac.mm
index a006f94f3e08..f762f304ab24 100644
--- a/chrome/browser/chrome_browser_main_mac.mm
+++ b/chrome/browser/chrome_browser_main_mac.mm
@@ -254,8 +254,6 @@ void RecordBetaAndDevStats() {
 }
 
 void ChromeBrowserMainPartsMac::PreMainMessageLoopStart() {
-  MacStartupProfiler::GetInstance()->Profile(
-      MacStartupProfiler::PRE_MAIN_MESSAGE_LOOP_START);
   ChromeBrowserMainPartsPosix::PreMainMessageLoopStart();
 
   // ChromeBrowserMainParts should have loaded the resource bundle by this
@@ -310,8 +308,6 @@ void RecordBetaAndDevStats() {
 }
 
 void ChromeBrowserMainPartsMac::PostMainMessageLoopStart() {
-  MacStartupProfiler::GetInstance()->Profile(
-      MacStartupProfiler::POST_MAIN_MESSAGE_LOOP_START);
   ChromeBrowserMainPartsPosix::PostMainMessageLoopStart();
 
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
@@ -320,8 +316,6 @@ void RecordBetaAndDevStats() {
 }
 
 void ChromeBrowserMainPartsMac::PreProfileInit() {
-  MacStartupProfiler::GetInstance()->Profile(
-      MacStartupProfiler::PRE_PROFILE_INIT);
   ChromeBrowserMainPartsPosix::PreProfileInit();
 
   // This is called here so that the app shim socket is only created after
@@ -330,13 +324,8 @@ void RecordBetaAndDevStats() {
 }
 
 void ChromeBrowserMainPartsMac::PostProfileInit() {
-  MacStartupProfiler::GetInstance()->Profile(
-      MacStartupProfiler::POST_PROFILE_INIT);
   ChromeBrowserMainPartsPosix::PostProfileInit();
 
-  g_browser_process->metrics_service()->RecordBreakpadRegistration(
-      crash_reporter::GetUploadsEnabled());
-
   // Activation of Keystone is not automatic but done in response to the
   // counting and reporting of profiles.
   KeystoneGlue* glue = [KeystoneGlue defaultKeystoneGlue];
