diff --git a/chrome/browser/ui/web_applications/web_app_ui_manager_impl.cc b/chrome/browser/ui/web_applications/web_app_ui_manager_impl.cc
index 926a10ae9d091..c4e66707ea3ca 100644
--- a/chrome/browser/ui/web_applications/web_app_ui_manager_impl.cc
+++ b/chrome/browser/ui/web_applications/web_app_ui_manager_impl.cc
@@ -20,7 +20,6 @@
 #include "chrome/browser/ui/web_applications/app_browser_controller.h"
 #include "chrome/browser/ui/web_applications/web_app_dialog_manager.h"
 #include "chrome/browser/ui/web_applications/web_app_launch_utils.h"
-#include "chrome/browser/ui/web_applications/web_app_metrics.h"
 #include "chrome/browser/web_applications/components/app_registry_controller.h"
 #include "chrome/browser/web_applications/extensions/web_app_extension_shortcut.h"
 #include "chrome/browser/web_applications/system_web_apps/system_web_app_manager.h"
@@ -319,12 +318,6 @@ void WebAppUiManagerImpl::NotifyOnAssociatedAppChanged(
     content::WebContents* web_contents,
     const AppId& previous_app_id,
     const AppId& new_app_id) const {
-  WebAppMetrics* web_app_metrics = WebAppMetrics::Get(profile_);
-  // Unavailable in guest sessions.
-  if (!web_app_metrics)
-    return;
-  web_app_metrics->NotifyOnAssociatedAppChanged(web_contents, previous_app_id,
-                                                new_app_id);
 }
 
 bool WebAppUiManagerImpl::CanReparentAppTabToWindow(
