diff --git a/chrome/browser/ui/views/location_bar/location_bar_view.cc b/chrome/browser/ui/views/location_bar/location_bar_view.cc
index 2889d7f988f2..f3b3be152c46 100644
--- a/chrome/browser/ui/views/location_bar/location_bar_view.cc
+++ b/chrome/browser/ui/views/location_bar/location_bar_view.cc
@@ -255,8 +255,10 @@ void LocationBarView::Init() {
     // The send tab to self icon is intentionally the first one added so it is
     // the left most icon.
     params.types_enabled.push_back(PageActionIconType::kSendTabToSelf);
+#if BUILDFLAG(ENABLE_CLICK_TO_CALL)
     if (base::FeatureList::IsEnabled(kClickToCallUI))
       params.types_enabled.push_back(PageActionIconType::kClickToCall);
+#endif
     if (base::FeatureList::IsEnabled(kSharingQRCodeGenerator))
       params.types_enabled.push_back(PageActionIconType::kQRCodeGenerator);
     if (base::FeatureList::IsEnabled(kSharedClipboardUI))
@@ -334,8 +336,7 @@ SkColor LocationBarView::GetOpaqueBorderColor() const {
 }
 
 int LocationBarView::GetBorderRadius() const {
-  return ChromeLayoutProvider::Get()->GetCornerRadiusMetric(
-      views::EMPHASIS_MAXIMUM, size());
+  return 8;
 }
 
 std::unique_ptr<views::Background> LocationBarView::CreateRoundRectBackground(
