diff --git a/chrome/browser/web_applications/components/BUILD.gn b/chrome/browser/web_applications/components/BUILD.gn
index cd56b58374fc8..e33a1141d2fa0 100644
--- a/chrome/browser/web_applications/components/BUILD.gn
+++ b/chrome/browser/web_applications/components/BUILD.gn
@@ -28,8 +28,6 @@ source_set("components") {
     "file_handling_permission_context.h",
     "file_handling_permission_request_impl.cc",
     "file_handling_permission_request_impl.h",
-    "install_bounce_metric.cc",
-    "install_bounce_metric.h",
     "install_finalizer.cc",
     "install_finalizer.h",
     "install_manager.cc",
