diff --git a/chrome/browser/ui/search/search_tab_helper.cc b/chrome/browser/ui/search/search_tab_helper.cc
index d3370d3171528..c33d91d7d1c37 100644
--- a/chrome/browser/ui/search/search_tab_helper.cc
+++ b/chrome/browser/ui/search/search_tab_helper.cc
@@ -493,6 +493,7 @@ void SearchTabHelper::OnConfirmThemeChanges() {
   }
 }
 
+#if 0
 void SearchTabHelper::BlocklistPromo(const std::string& promo_id) {
   auto* promo_service = PromoServiceFactory::GetForProfile(profile());
   if (!promo_service) {
@@ -502,6 +503,7 @@ void SearchTabHelper::BlocklistPromo(const std::string& promo_id) {
 
   promo_service->BlocklistPromo(promo_id);
 }
+#endif
 
 void SearchTabHelper::OpenExtensionsPage(double button,
                                          bool alt_key,
