diff --git a/chrome/browser/download/download_stats.cc b/chrome/browser/download/download_stats.cc
index 9d7cbb2034828..bc0fd7f3802c8 100644
--- a/chrome/browser/download/download_stats.cc
+++ b/chrome/browser/download/download_stats.cc
@@ -4,7 +4,6 @@
 
 #include "chrome/browser/download/download_stats.h"
 #include "components/profile_metrics/browser_profile_type.h"
-#include "components/safe_browsing/core/browser/download/download_stats.h"
 
 #include "base/metrics/histogram_functions.h"
 #include "base/metrics/user_metrics.h"
@@ -26,8 +25,6 @@ void RecordDangerousDownloadWarningShown(
     bool has_user_gesture) {
   base::UmaHistogramEnumeration("Download.ShowedDownloadWarning", danger_type,
                                 download::DOWNLOAD_DANGER_TYPE_MAX);
-  safe_browsing::RecordDangerousDownloadWarningShown(
-      danger_type, file_path, is_https, has_user_gesture);
 }
 
 void RecordOpenedDangerousConfirmDialog(
