diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index d4435c818602..da8e7856bd84 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -147,14 +147,10 @@ static_library("extensions") {
     "api/downloads/downloads_api.h",
     "api/downloads_internal/downloads_internal_api.cc",
     "api/downloads_internal/downloads_internal_api.h",
-    "api/enterprise_hardware_platform/enterprise_hardware_platform_api.cc",
-    "api/enterprise_hardware_platform/enterprise_hardware_platform_api.h",
     "api/extension_action/extension_action_api.cc",
     "api/extension_action/extension_action_api.h",
     "api/extension_action/extension_page_actions_api_constants.cc",
     "api/extension_action/extension_page_actions_api_constants.h",
-    "api/feedback_private/chrome_feedback_private_delegate.cc",
-    "api/feedback_private/chrome_feedback_private_delegate.h",
     "api/file_system/chrome_file_system_delegate.cc",
     "api/file_system/chrome_file_system_delegate.h",
     "api/file_system/file_entry_picker.cc",
@@ -304,14 +300,6 @@ static_library("extensions") {
     "api/resources_private/resources_private_api.h",
     "api/runtime/chrome_runtime_api_delegate.cc",
     "api/runtime/chrome_runtime_api_delegate.h",
-    "api/safe_browsing_private/safe_browsing_private_api.cc",
-    "api/safe_browsing_private/safe_browsing_private_api.h",
-    "api/safe_browsing_private/safe_browsing_private_event_router.cc",
-    "api/safe_browsing_private/safe_browsing_private_event_router.h",
-    "api/safe_browsing_private/safe_browsing_private_event_router_factory.cc",
-    "api/safe_browsing_private/safe_browsing_private_event_router_factory.h",
-    "api/safe_browsing_private/safe_browsing_util.cc",
-    "api/safe_browsing_private/safe_browsing_util.h",
     "api/scripting/scripting_api.cc",
     "api/scripting/scripting_api.h",
     "api/search/search_api.cc",
@@ -346,8 +334,6 @@ static_library("extensions") {
     "api/signed_in_devices/signed_in_devices_api.h",
     "api/signed_in_devices/signed_in_devices_manager.cc",
     "api/signed_in_devices/signed_in_devices_manager.h",
-    "api/storage/managed_value_store_cache.cc",
-    "api/storage/managed_value_store_cache.h",
     "api/storage/policy_value_store.cc",
     "api/storage/policy_value_store.h",
     "api/storage/setting_sync_data.cc",
@@ -608,8 +594,6 @@ static_library("extensions") {
     "external_pref_loader.h",
     "external_provider_impl.cc",
     "external_provider_impl.h",
-    "forced_extensions/force_installed_metrics.cc",
-    "forced_extensions/force_installed_metrics.h",
     "forced_extensions/force_installed_tracker.cc",
     "forced_extensions/force_installed_tracker.h",
     "forced_extensions/install_stage_tracker.cc",
@@ -639,8 +623,6 @@ static_library("extensions") {
     "launch_util.h",
     "load_error_reporter.cc",
     "load_error_reporter.h",
-    "media_router_extension_access_logger_impl.cc",
-    "media_router_extension_access_logger_impl.h",
     "menu_manager.cc",
     "menu_manager.h",
     "menu_manager_factory.cc",
@@ -746,9 +728,6 @@ static_library("extensions") {
 
     # TODO(crbug.com/1065748): Remove this circular dependency.
     "//chrome/browser/web_applications/extensions",
-
-    # TODO(crbug/925153): Remove this circular dependency.
-    "//chrome/browser/safe_browsing",
   ]
 
   # Since browser and browser_extensions actually depend on each other,
@@ -761,8 +740,6 @@ static_library("extensions") {
     "//chrome/common",
     "//chrome/common/extensions/api",
     "//components/omnibox/browser",
-    "//components/safe_browsing/core:csd_proto",
-    "//components/safe_browsing/core/db:util",
     "//components/signin/core/browser",
     "//components/translate/content/browser",
     "//content/public/browser",
@@ -792,7 +769,6 @@ static_library("extensions") {
     "//chrome/browser/profiles:profile",
     "//chrome/browser/resource_coordinator:intervention_policy_database_proto",
     "//chrome/browser/resource_coordinator:mojo_bindings",
-    "//chrome/browser/safe_browsing",
     "//chrome/browser/web_applications",
     "//chrome/browser/web_applications/components",
     "//components/site_engagement/core/mojom:mojo_bindings",
@@ -801,7 +777,6 @@ static_library("extensions") {
     # TODO(crbug.com/1065748): Remove this dependency:
     "//chrome/browser/web_applications/extensions",
     "//chrome/common/extensions/api:extensions_features",
-    "//chrome/common/safe_browsing:proto",
     "//chrome/services/removable_storage_writer/public/mojom",
     "//components/autofill/content/browser",
     "//components/bookmarks/browser",
@@ -818,9 +793,7 @@ static_library("extensions") {
     "//components/drive",
     "//components/embedder_support",
     "//components/embedder_support:browser_util",
-    "//components/enterprise",
     "//components/favicon/content",
-    "//components/feedback",
     "//components/gcm_driver",
     "//components/guest_view/browser",
     "//components/history/core/browser",
@@ -834,7 +807,6 @@ static_library("extensions") {
     "//components/net_log",
     "//components/no_state_prefetch/browser",
     "//components/onc",
-    "//components/page_load_metrics/browser:browser",
     "//components/password_manager/core/browser",
     "//components/password_manager/core/browser:affiliation",
     "//components/password_manager/core/browser/leak_detection",
@@ -848,12 +820,6 @@ static_library("extensions") {
     "//components/privacy_sandbox:privacy_sandbox_prefs",
     "//components/proxy_config",
     "//components/resources",
-    "//components/safe_browsing:buildflags",
-    "//components/safe_browsing/content/web_ui:web_ui",
-    "//components/safe_browsing/core:csd_proto",
-    "//components/safe_browsing/core:features",
-    "//components/safe_browsing/core/common:safe_browsing_prefs",
-    "//components/safe_browsing/core/db:database_manager",
     "//components/search_engines",
     "//components/services/app_service/public/mojom",
     "//components/services/patch/content",
@@ -904,7 +870,6 @@ static_library("extensions") {
     "//services/audio/public/cpp",
     "//services/data_decoder/public/cpp",
     "//services/device/public/mojom",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/mojom",
     "//services/service_manager/public/cpp",
     "//services/service_manager/public/mojom",
@@ -955,8 +920,6 @@ static_library("extensions") {
 
   if (is_chromeos_lacros || is_chromeos_ash) {
     sources += [
-      "api/enterprise_device_attributes/enterprise_device_attributes_api.h",
-      "api/enterprise_platform_keys/enterprise_platform_keys_api.h",
       "api/platform_keys/platform_keys_api.h",
     ]
     deps += [
@@ -965,10 +928,6 @@ static_library("extensions") {
     ]
     if (is_chromeos_lacros) {
       sources += [
-        "api/enterprise_device_attributes/enterprise_device_attributes_api_lacros.cc",
-        "api/enterprise_device_attributes/enterprise_device_attributes_api_lacros.h",
-        "api/enterprise_platform_keys/enterprise_platform_keys_api_lacros.cc",
-        "api/enterprise_platform_keys/enterprise_platform_keys_api_lacros.h",
         "api/platform_keys/platform_keys_api_lacros.cc",
         "api/platform_keys/platform_keys_api_lacros.h",
       ]
@@ -982,14 +941,6 @@ static_library("extensions") {
       "api/certificate_provider/certificate_provider_api.h",
       "api/crash_report_private/crash_report_private_api.cc",
       "api/crash_report_private/crash_report_private_api.h",
-      "api/enterprise_device_attributes/enterprise_device_attributes_api_ash.cc",
-      "api/enterprise_device_attributes/enterprise_device_attributes_api_ash.h",
-      "api/enterprise_networking_attributes/enterprise_networking_attributes_api.cc",
-      "api/enterprise_networking_attributes/enterprise_networking_attributes_api.h",
-      "api/enterprise_platform_keys/enterprise_platform_keys_api_ash.cc",
-      "api/enterprise_platform_keys/enterprise_platform_keys_api_ash.h",
-      "api/enterprise_platform_keys_private/enterprise_platform_keys_private_api.cc",
-      "api/enterprise_platform_keys_private/enterprise_platform_keys_private_api.h",
       "api/file_handlers/non_native_file_system_delegate_chromeos.cc",
       "api/file_handlers/non_native_file_system_delegate_chromeos.h",
       "api/file_system/consent_provider.cc",
@@ -1089,7 +1040,6 @@ static_library("extensions") {
       "//chromeos/ui/base",
       "//components/arc",
       "//components/constrained_window",
-      "//components/crash/content/browser/error_reporting",
       "//components/drive",
       "//components/user_manager",
       "//media/capture:capture_lib",
@@ -1118,14 +1068,6 @@ static_library("extensions") {
   } else {
     assert(!is_chromeos_ash)
     sources += [
-      "api/enterprise_reporting_private/chrome_desktop_report_request_helper.cc",
-      "api/enterprise_reporting_private/chrome_desktop_report_request_helper.h",
-      "api/enterprise_reporting_private/context_info_fetcher.cc",
-      "api/enterprise_reporting_private/context_info_fetcher.h",
-      "api/enterprise_reporting_private/enterprise_reporting_private_api.cc",
-      "api/enterprise_reporting_private/enterprise_reporting_private_api.h",
-      "api/enterprise_reporting_private/keychain_data_helper_mac.h",
-      "api/enterprise_reporting_private/keychain_data_helper_mac.mm",
       "api/image_writer_private/operation_nonchromeos.cc",
       "api/messaging/native_message_process_host.cc",
       "api/messaging/native_message_process_host.h",
@@ -1139,8 +1081,6 @@ static_library("extensions") {
       "default_apps.h",
     ]
     deps += [
-      "//components/enterprise",
-      "//components/enterprise/common/proto:connectors_proto",
       "//components/keep_alive_registry",
     ]
     if (is_linux || is_chromeos) {
