diff --git a/chrome/browser/devtools/devtools_embedder_message_dispatcher.cc b/chrome/browser/devtools/devtools_embedder_message_dispatcher.cc
index 52a9e66664543..435fafb89d5a8 100644
--- a/chrome/browser/devtools/devtools_embedder_message_dispatcher.cc
+++ b/chrome/browser/devtools/devtools_embedder_message_dispatcher.cc
@@ -233,8 +233,5 @@ DevToolsEmbedderMessageDispatcher::CreateForDevToolsFrontend(
                      &Delegate::SetOpenNewWindowForPopups, delegate);
   d->RegisterHandler("registerExtensionsAPI", &Delegate::RegisterExtensionsAPI,
                      delegate);
-  d->RegisterHandlerWithCallback("showSurvey", &Delegate::ShowSurvey, delegate);
-  d->RegisterHandlerWithCallback("canShowSurvey", &Delegate::CanShowSurvey,
-                                 delegate);
   return d;
 }
