diff --git a/chrome/browser/prefs/pref_service_incognito_allowlist.cc b/chrome/browser/prefs/pref_service_incognito_allowlist.cc
index 63db3b0071e95..6300914063f35 100644
--- a/chrome/browser/prefs/pref_service_incognito_allowlist.cc
+++ b/chrome/browser/prefs/pref_service_incognito_allowlist.cc
@@ -14,7 +14,6 @@
 #include "components/content_settings/core/common/pref_names.h"
 #include "components/metrics/metrics_pref_names.h"
 #include "components/reading_list/core/reading_list_pref_names.h"
-#include "components/ukm/ukm_pref_names.h"
 
 #if !defined(OS_ANDROID)
 #include "chrome/browser/accessibility/animation_policy_prefs.h"
@@ -147,13 +146,6 @@ const char* const kPersistentPrefNames[] = {
     // Reading list preferences are common between incognito and regular mode.
     reading_list::prefs::kReadingListHasUnseenEntries,
 
-    // Although UKMs are not collected in incognito, theses preferences may be
-    // changed by UMA/Sync/Unity consent, and need to be the same between
-    // incognito and regular modes.
-    ukm::prefs::kUkmClientId,
-    ukm::prefs::kUkmUnsentLogStore,
-    ukm::prefs::kUkmSessionId,
-
     // Cookie controls preference is, as in an initial release, surfaced only in
     // the incognito mode and therefore should be persisted between incognito
     // sessions.
