diff --git a/chrome/common/extensions/api/autofill_private.idl b/chrome/common/extensions/api/autofill_private.idl
index ed1c4c3ae5546..1cc3900060be2 100644
--- a/chrome/common/extensions/api/autofill_private.idl
+++ b/chrome/common/extensions/api/autofill_private.idl
@@ -245,9 +245,6 @@ namespace autofillPrivate {
     // Triggers local credit cards migration.
     static void migrateCreditCards();
 
-    // Logs that the server cards edit link was clicked.
-    static void logServerCardLinkClicked();
-
     // Enables or disables FIDO Authentication for credit card unmasking.
     static void setCreditCardFIDOAuthEnabledState(boolean enabled);
 
