diff --git a/chrome/browser/ui/views/profiles/profile_picker_view.cc b/chrome/browser/ui/views/profiles/profile_picker_view.cc
index 1cc806ee815b..28574fa43853 100644
--- a/chrome/browser/ui/views/profiles/profile_picker_view.cc
+++ b/chrome/browser/ui/views/profiles/profile_picker_view.cc
@@ -495,9 +495,6 @@ void ProfilePickerView::Display(ProfilePicker::EntryPoint entry_point) {
     DCHECK(creation_time_on_startup_.is_null());
     // Display() is called right after the creation of this object.
     creation_time_on_startup_ = base::TimeTicks::Now();
-    base::UmaHistogramTimes("ProfilePicker.StartupTime.BeforeCreation",
-                            creation_time_on_startup_ -
-                                startup_metric_utils::MainEntryPointTicks());
   }
 
   if (state_ == kNotStarted) {
@@ -1117,8 +1114,6 @@ void ProfilePickerView::FinishSignedInCreationFlowImpl(
   }
   entry->SetLocalProfileName(sign_in_->name_for_signed_in_profile,
                              /*is_default_name=*/false);
-  ProfileMetrics::LogProfileAddNewUser(
-      ProfileMetrics::ADD_NEW_PROFILE_PICKER_SIGNED_IN);
 
   // If sync is not enabled (and will not likely be enabled with an enterprise
   // consent), apply a new color to the profile (otherwise, a more complicated
