diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index d85c8180793d..df8546c6ec77 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -42,10 +42,7 @@ source_set("chrome_dll_resources") {
   deps = [ ":command_ids" ]
 
   if (is_win) {
-    sources += [
-      "chrome_dll.rc",
-      "etw_events/chrome_events_win.rc",
-    ]
+    sources += [ "chrome_dll.rc" ]
 
     deps += [
       "//build:branding_buildflags",
@@ -128,7 +125,6 @@ static_library("test_support") {
     "//components/gwp_asan/buildflags",
     "//components/heap_profiling/in_process",
     "//components/nacl/common:buildflags",
-    "//components/startup_metric_utils/browser",
     "//content/public/app",
     "//content/public/common",
     "//content/public/common:service_names",
@@ -143,7 +139,6 @@ static_library("test_support") {
   if (!is_fuchsia) {
     # TODO(crbug.com/753619): Enable crash reporting on Fuchsia.
     deps += [
-      "//components/crash/core/app",
       "//components/crash/core/app:test_support",
     ]
   }
