diff --git a/chrome/browser/lifetime/browser_shutdown.h b/chrome/browser/lifetime/browser_shutdown.h
index c7c1a310600e..a8d010c3d488 100644
--- a/chrome/browser/lifetime/browser_shutdown.h
+++ b/chrome/browser/lifetime/browser_shutdown.h
@@ -94,9 +94,6 @@ bool RecordShutdownInfoPrefs();
 void ShutdownPostThreadsStop(RestartMode restart_mode);
 #endif
 
-// Called at startup to create a histogram from our previous shutdown time.
-void ReadLastShutdownInfo();
-
 // There are various situations where the browser process should continue to
 // run after the last browser window has closed - the Mac always continues
 // running until the user explicitly quits, and on Windows/Linux the application
