diff --git a/chrome/browser/media/router/mojo/media_router_mojo_impl.cc b/chrome/browser/media/router/mojo/media_router_mojo_impl.cc
index d369537922fd..d41c003b8d2b 100644
--- a/chrome/browser/media/router/mojo/media_router_mojo_impl.cc
+++ b/chrome/browser/media/router/mojo/media_router_mojo_impl.cc
@@ -17,7 +17,9 @@
 #include "base/stl_util.h"
 #include "base/strings/stringprintf.h"
 #include "chrome/browser/media/cast_mirroring_service_host.h"
+#if BUILDFLAG(ENABLE_MEDIA_REMOTING)
 #include "chrome/browser/media/cast_remoting_connector.h"
+#endif
 #include "chrome/browser/media/router/event_page_request_manager.h"
 #include "chrome/browser/media/router/event_page_request_manager_factory.h"
 #include "chrome/browser/media/router/media_router_feature.h"
@@ -315,10 +317,12 @@ void MediaRouterMojoImpl::CreateRoute(const MediaSource::Id& source_id,
 
   const MediaSource source(source_id);
   if (source.IsTabMirroringSource() || source.IsLocalFileSource()) {
+#if BUILDFLAG(ENABLE_MEDIA_REMOTING)
     // Ensure the CastRemotingConnector is created before mirroring starts.
     CastRemotingConnector* const connector =
         CastRemotingConnector::Get(web_contents);
     connector->ResetRemotingPermission();
+#endif
 
     MediaRouterMojoMetrics::RecordTabMirroringMetrics(web_contents);
   }
