diff --git a/chrome/app/theme/chromium/BRANDING b/chrome/app/theme/chromium/BRANDING
index a2020a0b6f92..285c5e2d137a 100644
--- a/chrome/app/theme/chromium/BRANDING
+++ b/chrome/app/theme/chromium/BRANDING
@@ -1,10 +1,10 @@
-COMPANY_FULLNAME=The Chromium Authors
-COMPANY_SHORTNAME=The Chromium Authors
-PRODUCT_FULLNAME=Chromium
-PRODUCT_SHORTNAME=Chromium
-PRODUCT_INSTALLER_FULLNAME=Chromium Installer
-PRODUCT_INSTALLER_SHORTNAME=Chromium Installer
-COPYRIGHT=Copyright 2020 The Chromium Authors. All rights reserved.
-MAC_BUNDLE_ID=org.chromium.Chromium
+COMPANY_FULLNAME=The Dumb Browser Authors
+COMPANY_SHORTNAME=The Dumb Browser Authors
+PRODUCT_FULLNAME=Dumb Browser
+PRODUCT_SHORTNAME=Dumb
+PRODUCT_INSTALLER_FULLNAME=Dumb Browser Installer
+PRODUCT_INSTALLER_SHORTNAME=Dumb Installer
+COPYRIGHT=Copyright 2020 The Dumb Browser Authors. All rights reserved.
+MAC_BUNDLE_ID=com.example.dumb-browser
 MAC_CREATOR_CODE=Cr24
 MAC_TEAM_ID=
