diff --git a/chrome/browser/plugins/plugin_info_host_impl.cc b/chrome/browser/plugins/plugin_info_host_impl.cc
index 19de17320066..f571d5c3590a 100644
--- a/chrome/browser/plugins/plugin_info_host_impl.cc
+++ b/chrome/browser/plugins/plugin_info_host_impl.cc
@@ -40,7 +40,6 @@
 #include "components/nacl/common/buildflags.h"
 #include "components/pref_registry/pref_registry_syncable.h"
 #include "components/prefs/pref_service.h"
-#include "components/ukm/content/source_url_recorder.h"
 #include "content/public/browser/browser_task_traits.h"
 #include "content/public/browser/plugin_service.h"
 #include "content/public/browser/plugin_service_filter.h"
@@ -49,8 +48,6 @@
 #include "extensions/buildflags/buildflags.h"
 #include "net/base/registry_controlled_domains/registry_controlled_domain.h"
 #include "ppapi/buildflags/buildflags.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
 #include "url/gurl.h"
 #include "url/origin.h"
 
@@ -449,10 +446,6 @@ void PluginInfoHostImpl::ReportMetrics(int render_frame_id,
       mime_type != content::kFlashPluginSplMimeType) {
     return;
   }
-
-  ukm::builders::Plugins_FlashInstance(
-      ukm::GetSourceIdForWebContentsDocument(web_contents))
-      .Record(ukm::UkmRecorder::Get());
 }
 
 void PluginInfoHostImpl::Context::MaybeGrantAccess(
