diff --git a/chrome/browser/ui/webui/tab_search/tab_search_page_handler.cc b/chrome/browser/ui/webui/tab_search/tab_search_page_handler.cc
index c0110fff2366..c93aad51a787 100644
--- a/chrome/browser/ui/webui/tab_search/tab_search_page_handler.cc
+++ b/chrome/browser/ui/webui/tab_search/tab_search_page_handler.cc
@@ -31,12 +31,6 @@
 namespace {
 constexpr base::TimeDelta kTabsChangeDelay =
     base::TimeDelta::FromMilliseconds(50);
-
-#if BUILDFLAG(IS_CHROMEOS_ASH)
-constexpr char kFeedbackCategoryTag[] = "FromTabSearch";
-#else
-constexpr char kFeedbackCategoryTag[] = "FromTabSearchBrowser";
-#endif
 }
 
 TabSearchPageHandler::TabSearchPageHandler(
@@ -129,18 +123,6 @@ void TabSearchPageHandler::GetTabGroups(GetTabGroupsCallback callback) {
   NOTIMPLEMENTED();
 }
 
-void TabSearchPageHandler::ShowFeedbackPage() {
-  Browser* browser = chrome::FindLastActive();
-  if (!browser)
-    return;
-  chrome::ShowFeedbackPage(browser,
-                           chrome::FeedbackSource::kFeedbackSourceTabSearch,
-                           std::string() /* description_template */,
-                           std::string() /* description_placeholder_text */,
-                           std::string(kFeedbackCategoryTag) /* category_tag */,
-                           std::string() /* extra_diagnostics */);
-}
-
 void TabSearchPageHandler::SwitchToTab(
     tab_search::mojom::SwitchToTabInfoPtr switch_to_tab_info) {
   base::Optional<TabDetails> optional_details =
