diff --git a/chrome/browser/resources/new_tab_page/BUILD.gn b/chrome/browser/resources/new_tab_page/BUILD.gn
index e6c75abb1e5a..b31075c151bd 100644
--- a/chrome/browser/resources/new_tab_page/BUILD.gn
+++ b/chrome/browser/resources/new_tab_page/BUILD.gn
@@ -58,7 +58,6 @@ js_library("new_tab_page") {
 js_library("lazy_load") {
   deps = [
     ":customize_dialog",
-    ":middle_slot_promo",
     ":most_visited",
     ":voice_search_overlay",
     "modules/dummy:module",
