diff --git a/chrome/browser/enterprise/connectors/content_analysis_delegate.h b/chrome/browser/enterprise/connectors/content_analysis_delegate.h
index 2029534e2244..879fa9df7376 100644
--- a/chrome/browser/enterprise/connectors/content_analysis_delegate.h
+++ b/chrome/browser/enterprise/connectors/content_analysis_delegate.h
@@ -18,8 +18,6 @@
 #include "base/time/time.h"
 #include "chrome/browser/enterprise/connectors/common.h"
 #include "chrome/browser/enterprise/connectors/connectors_manager.h"
-#include "chrome/browser/safe_browsing/cloud_content_scanning/binary_upload_service.h"
-#include "chrome/browser/safe_browsing/cloud_content_scanning/deep_scanning_utils.h"
 #include "chrome/browser/ui/tab_modal_confirm_dialog.h"
 #include "chrome/browser/ui/tab_modal_confirm_dialog_delegate.h"
 #include "components/enterprise/common/proto/connectors.pb.h"
@@ -118,23 +116,6 @@ class ContentAnalysisDelegate {
     std::string mime_type;
   };
 
-  // File contents used as input for |file_info_| and the BinaryUploadService.
-  struct FileContents {
-    FileContents();
-    explicit FileContents(safe_browsing::BinaryUploadService::Result result);
-    FileContents(FileContents&&);
-    FileContents& operator=(FileContents&&);
-
-    safe_browsing::BinaryUploadService::Result result =
-        safe_browsing::BinaryUploadService::Result::UNKNOWN;
-    safe_browsing::BinaryUploadService::Request::Data data;
-
-    // Store the file size separately instead of using data.contents.size() to
-    // keep track of size for large files.
-    int64_t size = 0;
-    std::string sha256;
-  };
-
   // Enum to identify which message to show once scanning is complete. Ordered
   // by precedence for when multiple files have conflicting results.
   // TODO(crbug.com/1055785): Refactor this to whatever solution is chosen.
