diff --git a/chrome/browser/ui/browser_window.h b/chrome/browser/ui/browser_window.h
index e1581f7eb6abf..118e6c0850c22 100644
--- a/chrome/browser/ui/browser_window.h
+++ b/chrome/browser/ui/browser_window.h
@@ -488,6 +488,7 @@ class BrowserWindow : public ui::BaseWindow {
   // customization bubble is shown, the IPH should be shown after.
   virtual void MaybeShowProfileSwitchIPH() = 0;
 
+#if 0
   // Shows User Happiness Tracking Survey's dialog after the survey associated
   // with |site_id| has been successfully loaded. Failure to load the survey
   // will result in the dialog not being shown. |product_specific_data| should
@@ -499,6 +500,7 @@ class BrowserWindow : public ui::BaseWindow {
       base::OnceClosure success_callback,
       base::OnceClosure failure_callback,
       const std::map<std::string, bool>& product_specific_data) = 0;
+#endif
 
   // Returns object implementing ExclusiveAccessContext interface.
   virtual ExclusiveAccessContext* GetExclusiveAccessContext() = 0;
