diff --git a/chromecast/browser/extensions/api/tabs/tabs_constants.h b/chromecast/browser/extensions/api/tabs/tabs_constants.h
index 57130c94b93b..f361549d62bd 100644
--- a/chromecast/browser/extensions/api/tabs/tabs_constants.h
+++ b/chromecast/browser/extensions/api/tabs/tabs_constants.h
@@ -102,6 +102,7 @@ extern const char kSupportedInWindowsOnlyError[];
 extern const char kInvalidWindowTypeError[];
 extern const char kInvalidWindowStateError[];
 extern const char kScreenshotsDisabled[];
+extern const char kCannotUpdateMuteDisabled[];
 extern const char kCannotUpdateMuteCaptured[];
 extern const char kCannotDetermineLanguageOfUnloadedTab[];
 extern const char kMissingLockWindowFullscreenPrivatePermission[];
