diff --git a/chrome/browser/payments/BUILD.gn b/chrome/browser/payments/BUILD.gn
index 5d4544f346e07..9043762085593 100644
--- a/chrome/browser/payments/BUILD.gn
+++ b/chrome/browser/payments/BUILD.gn
@@ -47,7 +47,6 @@ source_set("browser_tests") {
     "//components/payments/content:utils",
     "//components/payments/core",
     "//components/payments/core:test_support",
-    "//components/ukm:test_support",
     "//device/fido:test_support",
   ]
 
