diff --git a/chrome/browser/ui/cocoa/notifications/BUILD.gn b/chrome/browser/ui/cocoa/notifications/BUILD.gn
index 281f436ef4b7..e2ff4ce4d619 100644
--- a/chrome/browser/ui/cocoa/notifications/BUILD.gn
+++ b/chrome/browser/ui/cocoa/notifications/BUILD.gn
@@ -28,7 +28,6 @@ mac_app_bundle("alert_notification_xpc_service") {
     ":common",
     "//base",
     "//chrome/services/mac_notifications/public/cpp:constants",
-    "//third_party/crashpad/crashpad/client",
   ]
 
   frameworks = [
