diff --git a/chrome/browser/ui/page_info/chrome_page_info_delegate.h b/chrome/browser/ui/page_info/chrome_page_info_delegate.h
index 9b910432efd6a..c89515114b898 100644
--- a/chrome/browser/ui/page_info/chrome_page_info_delegate.h
+++ b/chrome/browser/ui/page_info/chrome_page_info_delegate.h
@@ -63,7 +63,6 @@ class ChromePageInfoDelegate : public PageInfoDelegate {
       override;
   StatefulSSLHostStateDelegate* GetStatefulSSLHostStateDelegate() override;
   HostContentSettingsMap* GetContentSettings() override;
-  bool IsSubresourceFilterActivated(const GURL& site_url) override;
   bool IsContentDisplayedInVrHeadset() override;
   security_state::SecurityLevel GetSecurityLevel() override;
   security_state::VisibleSecurityState GetVisibleSecurityState() override;
