diff --git a/chrome/child/BUILD.gn b/chrome/child/BUILD.gn
index f877dad8e6d3..cc871333005b 100644
--- a/chrome/child/BUILD.gn
+++ b/chrome/child/BUILD.gn
@@ -15,8 +15,6 @@ static_library("child") {
   if (is_win) {
     sources += [
       "delay_load_failure_hook.cc",
-      "v8_crashpad_support_win.cc",
-      "v8_crashpad_support_win.h",
     ]
     deps += [
       "//components/crash/core/app:crash_export_thunk_include",
