diff --git a/chrome/browser/profiles/profile_window.cc b/chrome/browser/profiles/profile_window.cc
index e41d9e945806..fcab1859a6b4 100644
--- a/chrome/browser/profiles/profile_window.cc
+++ b/chrome/browser/profiles/profile_window.cc
@@ -291,8 +291,7 @@ bool HasProfileSwitchTargets(Profile* profile) {
   return number_of_profiles >= min_profiles;
 }
 
-void CreateAndSwitchToNewProfile(ProfileManager::CreateCallback callback,
-                                 ProfileMetrics::ProfileAdd metric) {
+void CreateAndSwitchToNewProfile(ProfileManager::CreateCallback callback) {
   ProfileAttributesStorage& storage =
       g_browser_process->profile_manager()->GetProfileAttributesStorage();
 
@@ -302,7 +301,6 @@ void CreateAndSwitchToNewProfile(ProfileManager::CreateCallback callback,
       profiles::GetDefaultAvatarIconUrl(placeholder_avatar_index),
       base::Bind(&profiles::OpenBrowserWindowForProfile, callback, true, true,
                  false));
-  ProfileMetrics::LogProfileAddNewUser(metric);
 }
 
 void ProfileBrowserCloseSuccess(const base::FilePath& profile_path) {
