diff --git a/chrome/browser/extensions/webstore_installer.h b/chrome/browser/extensions/webstore_installer.h
index cf9e87234d42a..e84a8a4dd6cdd 100644
--- a/chrome/browser/extensions/webstore_installer.h
+++ b/chrome/browser/extensions/webstore_installer.h
@@ -263,9 +263,6 @@ class WebstoreInstaller : public content::NotificationObserver,
   // PendingInstall.
   void ReportSuccess();
 
-  // Records stats regarding an interrupted webstore download item.
-  void RecordInterrupt(const download::DownloadItem* download) const;
-
   content::NotificationRegistrar registrar_;
   base::ScopedObservation<ExtensionRegistry, ExtensionRegistryObserver>
       extension_registry_observation_{this};
