diff --git a/chrome/browser/resources/settings/languages_page/languages_types.js b/chrome/browser/resources/settings/languages_page/languages_types.js
index 8e839effbddb..9036bf0f29ce 100644
--- a/chrome/browser/resources/settings/languages_page/languages_types.js
+++ b/chrome/browser/resources/settings/languages_page/languages_types.js
@@ -173,13 +173,6 @@ class LanguageHelper {
    */
   disableTranslateLanguage(languageCode) {}
 
-  /**
-   * Enables or disables spell check for the given language.
-   * @param {string} languageCode
-   * @param {boolean} enable
-   */
-  toggleSpellCheck(languageCode, enable) {}
-
   /**
    * Converts the language code for translate. There are some differences
    * between the language set the Translate server uses and that for
@@ -203,9 +196,6 @@ class LanguageHelper {
    */
   getLanguage(languageCode) {}
 
-  /** @param {string} languageCode */
-  retryDownloadDictionary(languageCode) {}
-
   // <if expr="chromeos">
   /** @param {string} id */
   addInputMethod(id) {}
