diff --git a/chrome/browser/chromeos/BUILD.gn b/chrome/browser/chromeos/BUILD.gn
index 7010c8608f56..837610b695c7 100644
--- a/chrome/browser/chromeos/BUILD.gn
+++ b/chrome/browser/chromeos/BUILD.gn
@@ -281,9 +281,7 @@ source_set("chromeos") {
     "//components/printing/common:mojo_interfaces",
     "//components/proxy_config",
     "//components/quirks",
-    "//components/rappor",
     "//components/renderer_context_menu",
-    "//components/rlz",
     "//components/safe_browsing/core:csd_proto",
     "//components/safe_browsing/core/db:metadata_proto",
     "//components/services/app_service:lib",
@@ -344,7 +342,6 @@ source_set("chromeos") {
     "//services/device/public/cpp/usb",
     "//services/device/public/mojom",
     "//services/device/public/mojom:usb",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/preferences/public/mojom",
     "//services/resource_coordinator/public/cpp:resource_coordinator_cpp",
     "//services/service_manager/public/cpp",
