diff --git a/chrome/browser/resources/settings/privacy_page/privacy_page.js b/chrome/browser/resources/settings/privacy_page/privacy_page.js
index b80d9bfac40f..dea480401f9b 100644
--- a/chrome/browser/resources/settings/privacy_page/privacy_page.js
+++ b/chrome/browser/resources/settings/privacy_page/privacy_page.js
@@ -82,13 +82,6 @@ Polymer({
     /** @private */
     showClearBrowsingDataDialog_: Boolean,
 
-    /** @private */
-    enableSafeBrowsingSubresourceFilter_: {
-      type: Boolean,
-      value() {
-        return loadTimeData.getBoolean('enableSafeBrowsingSubresourceFilter');
-      }
-    },
 
     /** @private */
     cookieSettingDescription_: String,
