diff --git a/chrome/browser/ui/views/login_handler_views.cc b/chrome/browser/ui/views/login_handler_views.cc
index a7957c623994..642ec4c08a31 100644
--- a/chrome/browser/ui/views/login_handler_views.cc
+++ b/chrome/browser/ui/views/login_handler_views.cc
@@ -36,7 +36,6 @@ class LoginHandlerViews : public LoginHandler {
                      web_contents,
                      std::move(auth_required_callback)),
         popunder_preventer_(std::make_unique<PopunderPreventer>(web_contents)) {
-    RecordDialogCreation(DialogIdentifier::LOGIN_HANDLER);
   }
 
   ~LoginHandlerViews() override {
