diff --git a/chrome/renderer/previews/resource_loading_hints_agent.h b/chrome/renderer/previews/resource_loading_hints_agent.h
index b7b910c21106e..5b6866c1889ff 100644
--- a/chrome/renderer/previews/resource_loading_hints_agent.h
+++ b/chrome/renderer/previews/resource_loading_hints_agent.h
@@ -11,7 +11,6 @@
 #include "base/macros.h"
 #include "base/optional.h"
 #include "chrome/common/previews_resource_loading_hints.mojom.h"
-#include "chrome/renderer/lite_video/lite_video_hint_agent.h"
 #include "content/public/renderer/render_frame_observer.h"
 #include "mojo/public/cpp/bindings/associated_receiver.h"
 #include "mojo/public/cpp/bindings/pending_associated_receiver.h"
@@ -43,8 +42,6 @@ class ResourceLoadingHintsAgent
   GURL GetDocumentURL() const;
 
   // previews::mojom::PreviewsResourceLoadingHintsReceiver:
-  void SetLiteVideoHint(
-      previews::mojom::LiteVideoHintPtr lite_video_hint) override;
   void SetBlinkOptimizationGuideHints(
       blink::mojom::BlinkOptimizationGuideHintsPtr hints) override;
   void StopThrottlingMediaRequests() override;
