diff --git a/chrome/tools/build/win/FILES.cfg b/chrome/tools/build/win/FILES.cfg
index 0cc4d6f56cc3..b3beb888c06f 100644
--- a/chrome/tools/build/win/FILES.cfg
+++ b/chrome/tools/build/win/FILES.cfg
@@ -814,12 +814,6 @@ FILES = [
     'archive': 'metrics-metadata.zip',
     'optional': ['dev', 'official'],
   },
-  {
-    'filename': 'rappor.xml',
-    'buildtype': ['dev', 'official'],
-    'archive': 'metrics-metadata.zip',
-    'optional': ['dev', 'official'],
-  },
   {
     'filename': 'ukm.xml',
     'buildtype': ['dev', 'official'],
