diff --git a/chrome/browser/ui/views/tabs/tab.h b/chrome/browser/ui/views/tabs/tab.h
index a80a4013bd2e7..eec73a3ce3444 100644
--- a/chrome/browser/ui/views/tabs/tab.h
+++ b/chrome/browser/ui/views/tabs/tab.h
@@ -28,7 +28,7 @@
 #include "ui/views/masked_targeter_delegate.h"
 #include "ui/views/view_observer.h"
 
-class AlertIndicator;
+class AlertIndicatorButton;
 class TabCloseButton;
 class TabController;
 class TabIcon;
@@ -117,6 +117,10 @@ class Tab : public gfx::AnimationDelegate,
   // Returns true if this tab is the active tab.
   bool IsActive() const;
 
+  // Returns the width of the largest part of the tab that is available for the
+  // user to click to select/activate the tab.
+  int GetWidthOfLargestSelectableRegion() const;
+
   // Notifies the AlertIndicatorButton that the active state of this tab has
   // changed.
   void ActiveStateChanged();
@@ -234,7 +238,7 @@ class Tab : public gfx::AnimationDelegate,
   bool closing_ = false;
 
   TabIcon* icon_ = nullptr;
-  AlertIndicator* alert_indicator_ = nullptr;
+  AlertIndicatorButton* alert_indicator_button_ = nullptr;
   TabCloseButton* close_button_ = nullptr;
 
   views::Label* title_;
