diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 8428b7508092..96a4f639680b 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -123,8 +123,6 @@ static_library("test_support") {
     "../browser/notifications/stub_notification_platform_bridge.h",
     "../browser/password_manager/password_manager_test_util.cc",
     "../browser/password_manager/password_manager_test_util.h",
-    "../browser/permissions/crowd_deny_fake_safe_browsing_database_manager.cc",
-    "../browser/permissions/crowd_deny_fake_safe_browsing_database_manager.h",
     "../browser/search_engines/template_url_service_factory_test_util.cc",
     "../browser/search_engines/template_url_service_factory_test_util.h",
     "../browser/search_engines/template_url_service_test_util.cc",
@@ -327,7 +325,6 @@ static_library("test_support") {
     ]
     deps += [ "//build:branding_buildflags" ]
     public_deps += [
-      "//components/crash/core/app",
       "//third_party/breakpad",
     ]
   }
@@ -357,7 +354,6 @@ static_library("test_support") {
     ]
     public_deps += [
       "//chrome/install_static/test:test_support",
-      "//components/crash/core/app",
       "//third_party/wtl",
     ]
     libs = [ "runtimeobject.lib" ]
@@ -577,12 +573,9 @@ if (is_android) {
       "../browser/browsing_data/access_context_audit_browsertest.cc",
       "../browser/engagement/important_sites_util_browsertest.cc",
       "../browser/metrics/metrics_service_user_demographics_browsertest.cc",
-      "../browser/metrics/ukm_browsertest.cc",
       "../browser/net/cert_verify_proc_browsertest.cc",
       "../browser/page_load_metrics/observers/ad_metrics/ad_density_intervention_android_browsertest.cc",
       "../browser/profiles/profile_browsertest_android.cc",
-      "../browser/safe_browsing/test_safe_browsing_database_helper.cc",
-      "../browser/safe_browsing/test_safe_browsing_database_helper.h",
       "../browser/ssl/chrome_security_state_client_browsertest.cc",
       "../browser/ssl/crlset_browsertest.cc",
       "../browser/subresource_filter/subresource_filter_browser_test_harness.cc",
@@ -731,8 +724,6 @@ if (is_android) {
       "//chrome/android:base_module_java",
       "//chrome/android:chrome_all_java",
       "//chrome/browser/tabmodel:java",
-      "//components/crash/android:handler_java",
-      "//components/crash/android:java",
       "//components/module_installer/android:module_installer_java",
       "//content/public/android:content_java",
       "//content/public/test/android:android_test_message_pump_support_java",
@@ -802,7 +793,6 @@ if (!is_android) {
       "//chrome/browser/metrics:test_support",
       "//chrome/browser/notifications/scheduler/test:test_support",
       "//chrome/browser/payments:browser_tests",
-      "//chrome/browser/privacy_budget:browser_tests",
       "//chrome/browser/profiling_host:profiling_browsertests",
       "//chrome/browser/web_applications:browser_tests",
       "//chrome/browser/webaudio:browser_tests",
@@ -933,7 +923,6 @@ if (!is_android) {
 
     data = [
       "data/",
-      "//chrome/browser/page_load_metrics/integration_tests/data/",
       "//components/test/data/arc/",
       "//components/test/data/autofill/",
       "//components/test/data/optimization_guide/",
@@ -1282,7 +1271,6 @@ if (!is_android) {
       "../browser/policy/policy_test_utils.cc",
       "../browser/policy/policy_test_utils.h",
       "../browser/policy/restore_on_startup_policy_browsertest.cc",
-      "../browser/policy/safe_browsing_policy_browsertest.cc",
       "../browser/policy/signed_exchange_browsertest.cc",
       "../browser/policy/site_isolation_policy_browsertest.cc",
       "../browser/policy/ssl_error_overriding_allowed_policy_browsertest.cc",
@@ -1336,16 +1324,6 @@ if (!is_android) {
       "../browser/resource_coordinator/discard_before_unload_helper_browsertest.cc",
       "../browser/resource_coordinator/tab_activity_watcher_browsertest.cc",
       "../browser/resource_coordinator/tab_manager_browsertest.cc",
-      "../browser/safe_browsing/client_side_detection_host_browsertest.cc",
-      "../browser/safe_browsing/client_side_detection_service_browsertest.cc",
-      "../browser/safe_browsing/cloud_content_scanning/deep_scanning_browsertest_base.cc",
-      "../browser/safe_browsing/cloud_content_scanning/deep_scanning_browsertest_base.h",
-      "../browser/safe_browsing/cloud_content_scanning/deep_scanning_test_utils.cc",
-      "../browser/safe_browsing/cloud_content_scanning/deep_scanning_test_utils.h",
-      "../browser/safe_browsing/download_protection/deep_scanning_browsertest.cc",
-      "../browser/safe_browsing/download_protection/download_protection_service_browsertest.cc",
-      "../browser/safe_browsing/test_safe_browsing_database_helper.cc",
-      "../browser/safe_browsing/test_safe_browsing_database_helper.h",
       "../browser/safe_xml_parser_browsertest.cc",
       "../browser/search/local_ntp_navigation_browsertest.cc",
       "../browser/search/ntp_custom_background_enabled_policy_handler_browsertest.cc",
@@ -1427,7 +1405,6 @@ if (!is_android) {
       "../browser/ui/autofill/save_address_profile_bubble_controller_impl_browsertest.cc",
       "../browser/ui/blocked_content/popup_opener_tab_helper_browsertest.cc",
       "../browser/ui/blocked_content/popup_tracker_browsertest.cc",
-      "../browser/ui/blocked_content/safe_browsing_triggered_popup_blocker_browsertest.cc",
       "../browser/ui/blocked_content/tab_under_blocker_browsertest.cc",
       "../browser/ui/bookmarks/bookmark_browsertest.cc",
       "../browser/ui/browser_browsertest.cc",
@@ -1581,7 +1558,6 @@ if (!is_android) {
       "../browser/ui/webui/webui_load_timer_browsertest.cc",
       "../browser/ui/webui/webui_webview_browsertest.cc",
       "../browser/ui/zoom/zoom_controller_browsertest.cc",
-      "../browser/ukm_worker_browsertest.cc",
       "../browser/unload_browsertest.cc",
       "../browser/usb/usb_browsertest.cc",
       "../browser/wake_lock/wake_lock_browsertest.cc",
@@ -1682,7 +1658,6 @@ if (!is_android) {
         "../browser/importer/ie_importer_browsertest_win.cc",
         "../browser/policy/locale_policy_browsertest.cc",
         "../browser/process_singleton_browsertest.cc",
-        "../browser/safe_browsing/chrome_cleaner/reporter_runner_browsertest_win.cc",
         "../browser/ui/startup/startup_browser_creator_corrupt_profiles_browsertest_win.cc",
         "../browser/ui/startup/startup_browser_creator_triggered_reset_browsertest_win.cc",
         "../browser/ui/views/chrome_cleaner_dialog_browsertest_win.cc",
@@ -1756,7 +1731,6 @@ if (!is_android) {
         "//chrome/browser/media/router:test_support",
         "//chrome/browser/resources/chromeos/accessibility:browser_tests",
         "//chrome/browser/ui/ash/holding_space:browser_tests",
-        "//chrome/services/file_util/public/cpp:browser_tests",
         "//chromeos:test_support",
         "//chromeos/components/drivefs:test_support",
         "//chromeos/components/media_app_ui:browser_test_support",
@@ -2107,7 +2081,6 @@ if (!is_android) {
         "../browser/extensions/external_install_error_browsertest.cc",
         "../browser/extensions/fetch_apitest.cc",
         "../browser/extensions/file_iframe_apitest.cc",
-        "../browser/extensions/google_docs_offline_ukm_browsertest.cc",
         "../browser/extensions/gpu_browsertest.cc",
         "../browser/extensions/identifiability_metrics_test_util.cc",
         "../browser/extensions/identifiability_metrics_test_util.h",
@@ -2154,9 +2127,6 @@ if (!is_android) {
         "../browser/policy/extension_force_install_mixin.cc",
         "../browser/policy/extension_force_install_mixin.h",
         "../browser/policy/extension_policy_browsertest.cc",
-        "../browser/safe_browsing/settings_reset_prompt/default_settings_fetcher_browsertest.cc",
-        "../browser/safe_browsing/settings_reset_prompt/settings_reset_prompt_test_utils.cc",
-        "../browser/safe_browsing/settings_reset_prompt/settings_reset_prompt_test_utils.h",
         "../browser/ui/views/extensions/extension_dialog_browsertest.cc",
         "../browser/ui/views/web_apps/pwa_confirmation_bubble_view_browsertest.cc",
         "../browser/ui/views/web_apps/web_app_confirmation_view_browsertest.cc",
@@ -2172,18 +2142,6 @@ if (!is_android) {
         sources += [ "../browser/extensions/background_app_browsertest.cc" ]
       }
 
-      if (is_win) {
-        sources += [
-          "../browser/safe_browsing/chrome_cleaner/chrome_cleaner_dialog_controller_impl_browsertest_win.cc",
-          "../browser/safe_browsing/chrome_cleaner/chrome_cleaner_reboot_dialog_controller_impl_browsertest_win.cc",
-          "../browser/safe_browsing/chrome_cleaner/mock_chrome_cleaner_controller_win.cc",
-          "../browser/safe_browsing/chrome_cleaner/mock_chrome_cleaner_controller_win.h",
-          "../browser/safe_browsing/chrome_cleaner/settings_resetter_browsertest_win.cc",
-          "../browser/safe_browsing/settings_reset_prompt/settings_reset_dependency_browsertest_win.cc",
-          "../browser/safe_browsing/settings_reset_prompt/settings_reset_prompt_model_browsertest_win.cc",
-        ]
-      }
-
       if (is_chromeos_ash) {
         sources += [
           "../browser/accessibility/accessibility_extension_api_browsertest.cc",
@@ -2357,7 +2315,6 @@ if (!is_android) {
         "../browser/ui/views/profiles/profile_menu_view_browsertest.cc",
         "../browser/ui/views/qrcode_generator/qrcode_generator_bubble_browsertest.cc",
         "../browser/ui/views/read_later/read_later_button_browsertest.cc",
-        "../browser/ui/views/safe_browsing/password_reuse_modal_warning_dialog_browsertest.cc",
         "../browser/ui/views/select_file_dialog_extension_browsertest.cc",
         "../browser/ui/views/session_crashed_bubble_view_browsertest.cc",
         "../browser/ui/views/status_bubble_views_browsertest.cc",
@@ -3545,10 +3502,6 @@ test("unit_tests") {
     "../browser/enterprise/util/affiliation_unittest.cc",
     "../browser/enterprise/util/managed_browser_utils_unittest.cc",
     "../browser/external_protocol/external_protocol_handler_unittest.cc",
-    "../browser/federated_learning/floc_eligibility_unittest.cc",
-    "../browser/federated_learning/floc_event_logger_unittest.cc",
-    "../browser/federated_learning/floc_id_provider_unittest.cc",
-    "../browser/federated_learning/floc_remote_permission_service_unittest.cc",
     "../browser/file_select_helper_unittest.cc",
     "../browser/file_system_access/chrome_file_system_access_permission_context_unittest.cc",
     "../browser/font_pref_change_notifier_unittest.cc",
@@ -4195,7 +4148,7 @@ test("unit_tests") {
   }
   if (is_win) {
     data_deps += [ "//chrome" ]
-    data += [ "$root_out_dir/chrome.exe" ]
+    data += [ "$root_out_dir/dumb.exe" ]
   }
 
   defines = []
@@ -4414,7 +4367,6 @@ test("unit_tests") {
       "../browser/metrics/chrome_android_metrics_provider_unittest.cc",
       "../browser/offline_pages/prefetch/notifications/prefetch_notification_service_impl_unittest.cc",
       "../browser/optimization_guide/android/optimization_guide_bridge_unittest.cc",
-      "../browser/page_load_metrics/observers/android_page_load_metrics_observer_unittest.cc",
       "../browser/password_manager/android/account_chooser_dialog_android_unittest.cc",
       "../browser/password_manager/android/all_passwords_bottom_sheet_controller_unittest.cc",
       "../browser/password_manager/android/all_passwords_bottom_sheet_helper_unittest.cc",
@@ -4499,17 +4451,6 @@ test("unit_tests") {
       "../browser/enterprise/connectors/file_system/service_settings_unittest.cc",
       "../browser/enterprise/connectors/reporting/reporting_service_settings_unittest.cc",
       "../browser/enterprise/connectors/service_provider_config_unittest.cc",
-      "../browser/enterprise/reporting/browser_report_generator_unittest.cc",
-      "../browser/enterprise/reporting/extension_info_unittest.cc",
-      "../browser/enterprise/reporting/extension_request/extension_request_notification_unittest.cc",
-      "../browser/enterprise/reporting/extension_request/extension_request_observer_factory_unittest.cc",
-      "../browser/enterprise/reporting/extension_request/extension_request_observer_unittest.cc",
-      "../browser/enterprise/reporting/extension_request/extension_request_report_throttler_unittest.cc",
-      "../browser/enterprise/reporting/policy_info_unittest.cc",
-      "../browser/enterprise/reporting/profile_report_generator_unittest.cc",
-      "../browser/enterprise/reporting/report_generator_unittest.cc",
-      "../browser/enterprise/reporting/report_request_queue_generator_unittest.cc",
-      "../browser/enterprise/reporting/report_scheduler_desktop_unittest.cc",
       "../browser/first_run/first_run_unittest.cc",
       "../browser/font_family_cache_unittest.cc",
 
@@ -4757,7 +4698,6 @@ test("unit_tests") {
       "../browser/ui/webui/settings/profile_info_handler_unittest.cc",
       "../browser/ui/webui/settings/recent_site_settings_helper_unittest.cc",
       "../browser/ui/webui/settings/reset_settings_handler_unittest.cc",
-      "../browser/ui/webui/settings/safety_check_handler_unittest.cc",
       "../browser/ui/webui/settings/settings_cookies_view_handler_unittest.cc",
       "../browser/ui/webui/settings/settings_manage_profile_handler_unittest.cc",
       "../browser/ui/webui/settings/settings_security_key_handler_unittest.cc",
@@ -5010,7 +4950,6 @@ test("unit_tests") {
     sources += [
       "../browser/component_updater/cros_component_installer_chromeos_unittest.cc",
       "../browser/component_updater/metadata_table_chromeos_unittest.cc",
-      "../browser/enterprise/reporting/android_app_info_generator_unittest.cc",
       "../browser/extensions/api/terminal/crostini_startup_status_unittest.cc",
       "../browser/google/google_brand_code_map_chromeos_unittest.cc",
       "../browser/media/webrtc/desktop_media_list_ash_unittest.cc",
@@ -5599,7 +5538,6 @@ test("unit_tests") {
 
       # This will add all of the unit tests for the schema compiler to this
       # target.
-      "//third_party/blink/common/privacy_budget:test_support",
       "//tools/json_schema_compiler/test:unit_tests",
     ]
     if (enable_service_discovery) {
@@ -5793,13 +5731,6 @@ test("unit_tests") {
     deps += [
       ":test_proto",
       "../common/safe_browsing:mock_binary_feature_extractor",
-      "//chrome/services/file_util/public/cpp:unit_tests",
-      "//components/safe_browsing/content/renderer:websocket_sb_handshake_throttle_unittest",
-      "//components/safe_browsing/content/triggers:ad_redirect_trigger",
-      "//components/safe_browsing/core:ping_manager_unittest",
-      "//components/safe_browsing/core/browser:safe_browsing_url_checker_unittest",
-      "//components/safe_browsing/core/browser/sync:unittests",
-      "//components/safe_browsing/core/db:v4_test_util",
     ]
   } else if (safe_browsing_mode == 2 && is_android) {
     sources += [ "../browser/safe_browsing/telemetry/android/android_telemetry_service_unittest.cc" ]
@@ -6673,7 +6604,6 @@ if (!is_android) {
       "//net",
       "//net:net_resources",
       "//net:test_support",
-      "//services/metrics/public/cpp:ukm_builders",
       "//skia",
       "//testing/gmock",
       "//testing/gtest",
@@ -6973,7 +6903,6 @@ test("chrome_app_unittests") {
     "//chrome/child",
     "//chrome/common/profiler",
     "//chrome/gpu",
-    "//components/crash/core/common",
     "//components/flags_ui:switches",
     "//components/gwp_asan/buildflags",
     "//components/heap_profiling/in_process",
@@ -7730,7 +7659,7 @@ if (is_win) {
     # the //chrome target to bundle up both initial/chrome.exe and chrome.exe.
     # As a workaround, explicitly include a data dep on just chrome.exe, and
     # add //chrome to deps above to make sure it's been built.
-    data = [ "$root_out_dir/chrome.exe" ]
+    data = [ "$root_out_dir/dumb.exe" ]
 
     # Don't want the test-specific dependencies to affect load tests.
     # In particular, a few system DLLs cause user32 to be loaded, which is bad.
