diff --git a/chrome/browser/ui/webui/tab_search/tab_search.mojom b/chrome/browser/ui/webui/tab_search/tab_search.mojom
index 4447b144831f..420e9ab0964b 100644
--- a/chrome/browser/ui/webui/tab_search/tab_search.mojom
+++ b/chrome/browser/ui/webui/tab_search/tab_search.mojom
@@ -89,9 +89,6 @@ interface PageHandler {
   // Get tab groups for the current profile.
   GetTabGroups() => (TabGroups tab_groups);
 
-  // Show the feedback page.
-  ShowFeedbackPage();
-
   // Switch to a specific tab.
   SwitchToTab(SwitchToTabInfo switch_to_tab_info);
 
