diff --git a/chrome/chrome_elf/third_party_dlls/hook.cc b/chrome/chrome_elf/third_party_dlls/hook.cc
index c14be1128b28..e0eafd96a8a7 100644
--- a/chrome/chrome_elf/third_party_dlls/hook.cc
+++ b/chrome/chrome_elf/third_party_dlls/hook.cc
@@ -10,7 +10,6 @@
 
 #include <assert.h>
 
-#include "chrome/chrome_elf/crash/crash_helper.h"
 #include "chrome/chrome_elf/hook_util/hook_util.h"
 #include "chrome/chrome_elf/pe_image_safe/pe_image_safe.h"
 #include "chrome/chrome_elf/sha1/sha1.h"
@@ -383,7 +382,7 @@ NewNtMapViewOfSection(NtMapViewOfSectionFunction orig_MapViewOfSection,
     ret = NewNtMapViewOfSectionImpl(
         orig_MapViewOfSection, section, process, base, zero_bits, commit_size,
         offset, view_size, inherit, allocation_type, protect);
-  } __except (elf_crash::GenerateCrashDump(GetExceptionInformation())) {
+  } __except (EXCEPTION_CONTINUE_SEARCH) {
   }
 
   return ret;
