diff --git a/chrome/browser/ui/passwords/well_known_change_password_navigation_throttle.h b/chrome/browser/ui/passwords/well_known_change_password_navigation_throttle.h
index 1c24eb12c605..e5dd0f1ef604 100644
--- a/chrome/browser/ui/passwords/well_known_change_password_navigation_throttle.h
+++ b/chrome/browser/ui/passwords/well_known_change_password_navigation_throttle.h
@@ -11,7 +11,6 @@
 
 #include "components/password_manager/core/browser/well_known_change_password_state.h"
 #include "components/password_manager/core/browser/well_known_change_password_util.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 
 class GURL;
 namespace content {
@@ -54,8 +53,6 @@ class WellKnownChangePasswordNavigationThrottle
   void OnProcessingFinished(bool is_supported) override;
   // Redirects to a given URL in the same tab.
   void Redirect(const GURL& url);
-  // Records the given UKM metric.
-  void RecordMetric(password_manager::WellKnownChangePasswordResult result);
 
   // Stores `navigation_handle()->GetURL()` if the first navigation was to
   // .well-known/change-password. It is later used to derive the URL for the
@@ -63,7 +60,6 @@ class WellKnownChangePasswordNavigationThrottle
   const GURL request_url_;
   password_manager::WellKnownChangePasswordState
       well_known_change_password_state_{this};
-  ukm::SourceId source_id_ = ukm::kInvalidSourceId;
   password_manager::ChangePasswordUrlService* change_password_url_service_ =
       nullptr;
   password_manager::AffiliationService* affiliation_service_ = nullptr;
