diff --git a/chrome/browser/web_applications/BUILD.gn b/chrome/browser/web_applications/BUILD.gn
index 2ae37bca711b..6306a32ccf66 100644
--- a/chrome/browser/web_applications/BUILD.gn
+++ b/chrome/browser/web_applications/BUILD.gn
@@ -11,8 +11,6 @@ group("web_app_test_group") {
 
 source_set("web_applications") {
   sources = [
-    "daily_metrics_helper.cc",
-    "daily_metrics_helper.h",
     "extension_status_utils.h",
     "external_web_app_manager.cc",
     "external_web_app_manager.h",
@@ -85,7 +83,6 @@ source_set("web_applications") {
     "//components/sync",
     "//components/user_manager",
     "//content/public/browser",
-    "//services/metrics/public/cpp:ukm_builders",
     "//skia",
   ]
 
@@ -215,7 +212,6 @@ source_set("web_applications_unit_tests") {
     "//components/services/app_service/public/cpp:protocol_handling",
     "//content/public/browser",
     "//content/test:test_support",
-    "//services/metrics/public/cpp:ukm_builders",
     "//skia",
     "//testing/gmock",
     "//testing/gtest",
