diff --git a/chrome/installer/mini_installer/BUILD.gn b/chrome/installer/mini_installer/BUILD.gn
index 7c7af8c0d948..9bc2aaf8dc90 100644
--- a/chrome/installer/mini_installer/BUILD.gn
+++ b/chrome/installer/mini_installer/BUILD.gn
@@ -138,7 +138,7 @@ template("generate_mini_installer") {
 
     inputs = [
       "$chrome_dll_file",
-      "$root_out_dir/chrome.exe",
+      "$root_out_dir/dumb.exe",
       "$root_out_dir/locales/en-US.pak",
       "$root_out_dir/setup.exe",
       "//chrome/tools/build/win/makecab.py",
