diff --git a/chrome/browser/profiles/profile_window.h b/chrome/browser/profiles/profile_window.h
index 2ee8d99e1804..6dac55b72302 100644
--- a/chrome/browser/profiles/profile_window.h
+++ b/chrome/browser/profiles/profile_window.h
@@ -95,8 +95,7 @@ bool HasProfileSwitchTargets(Profile* profile);
 // Creates a new profile from the next available profile directory, and
 // opens a new browser window for the profile once it is ready. When the browser
 // is opened, |callback| will be run if it isn't null.
-void CreateAndSwitchToNewProfile(ProfileManager::CreateCallback callback,
-                                 ProfileMetrics::ProfileAdd metric);
+void CreateAndSwitchToNewProfile(ProfileManager::CreateCallback callback);
 
 // Closes all browser windows that belong to the guest profile and opens the
 // user manager.
