diff --git a/chrome/renderer/previews/resource_loading_hints_agent.h b/chrome/renderer/previews/resource_loading_hints_agent.h
index fa11d806bd0df..23d027b816b8a 100644
--- a/chrome/renderer/previews/resource_loading_hints_agent.h
+++ b/chrome/renderer/previews/resource_loading_hints_agent.h
@@ -8,7 +8,6 @@
 #include "base/bind.h"
 #include "base/macros.h"
 #include "chrome/common/previews_resource_loading_hints.mojom.h"
-#include "chrome/renderer/lite_video/lite_video_hint_agent.h"
 #include "content/public/renderer/render_frame_observer.h"
 #include "mojo/public/cpp/bindings/associated_receiver.h"
 #include "mojo/public/cpp/bindings/pending_associated_receiver.h"
@@ -41,8 +40,6 @@ class ResourceLoadingHintsAgent
   GURL GetDocumentURL() const;
 
   // previews::mojom::PreviewsResourceLoadingHintsReceiver:
-  void SetLiteVideoHint(
-      previews::mojom::LiteVideoHintPtr lite_video_hint) override;
   void SetBlinkOptimizationGuideHints(
       blink::mojom::BlinkOptimizationGuideHintsPtr hints) override;
   void StopThrottlingMediaRequests() override;
