diff --git a/chrome/app_shim/BUILD.gn b/chrome/app_shim/BUILD.gn
index 1b2729aaaebb..8176defb2a1b 100644
--- a/chrome/app_shim/BUILD.gn
+++ b/chrome/app_shim/BUILD.gn
@@ -26,7 +26,6 @@ source_set("app_shim") {
     "//chrome/browser/ui",
     "//chrome/common",
     "//chrome/common:mojo_bindings",
-    "//components/crash/core/app",
     "//components/remote_cocoa/app_shim",
     "//content/public/browser",
     "//ipc",
