diff --git a/chrome/browser/devtools/protocol/page_handler.cc b/chrome/browser/devtools/protocol/page_handler.cc
index 5822cc323a23d..ea05709ecaba3 100644
--- a/chrome/browser/devtools/protocol/page_handler.cc
+++ b/chrome/browser/devtools/protocol/page_handler.cc
@@ -4,7 +4,6 @@
 
 #include "chrome/browser/devtools/protocol/page_handler.h"
 
-#include "components/subresource_filter/content/browser/devtools_interaction_tracker.h"
 #include "components/webapps/browser/installable/installable_manager.h"
 #include "ui/gfx/image/image.h"
 
@@ -22,15 +21,6 @@ PageHandler::~PageHandler() {
 void PageHandler::ToggleAdBlocking(bool enabled) {
   if (!web_contents())
     return;
-
-  // Create the DevtoolsInteractionTracker lazily (note that this call is a
-  // no-op if the object was already created).
-  subresource_filter::DevtoolsInteractionTracker::CreateForWebContents(
-      web_contents());
-
-  subresource_filter::DevtoolsInteractionTracker::FromWebContents(
-      web_contents())
-      ->ToggleForceActivation(enabled);
 }
 
 protocol::Response PageHandler::Enable() {
