diff --git a/chrome/credential_provider/gaiacp/BUILD.gn b/chrome/credential_provider/gaiacp/BUILD.gn
index b182615da50a..50f12ded0110 100644
--- a/chrome/credential_provider/gaiacp/BUILD.gn
+++ b/chrome/credential_provider/gaiacp/BUILD.gn
@@ -139,7 +139,6 @@ source_set("gaiacp_lib") {
     "//chrome/installer/launcher_support",
     "//components/crash/core/app:app",
     "//components/crash/core/app:crash_export_thunks",
-    "//components/crash/core/app:run_as_crashpad_handler",
     "//content/public/browser",
     "//crypto",
     "//google_apis",
@@ -246,7 +245,6 @@ shared_library("gaia1_0") {
     "//build:branding_buildflags",
     "//chrome/common:version_header",
     "//components/crash/core/app",
-    "//components/crash/core/app:run_as_crashpad_handler",
     "//content/public/browser",
   ]
   configs += [ "//build/config/win:windowed" ]
