diff --git a/chrome/browser/enterprise/connectors/device_trust/signal_reporter.h b/chrome/browser/enterprise/connectors/device_trust/signal_reporter.h
index 70dc93de69644..8a0ad56d178e0 100644
--- a/chrome/browser/enterprise/connectors/device_trust/signal_reporter.h
+++ b/chrome/browser/enterprise/connectors/device_trust/signal_reporter.h
@@ -38,15 +38,14 @@ class DeviceTrustSignalReporter {
 
   // Helper methods made virtual to be overwritten in unit tests.
   virtual policy::DMToken GetDmToken() const;
-  virtual void PostCreateReportQueueTask(
-      reporting::ReportQueueProvider::CreateReportQueueCallback create_queue_cb,
-      std::unique_ptr<reporting::ReportQueueConfiguration> config);
 
  private:
   std::unique_ptr<reporting::ReportQueue> report_queue_;
 
+#if 0
   enum class CreateQueueStatus { NOT_STARTED, IN_PROGRESS, DONE };
   CreateQueueStatus create_queue_status_{CreateQueueStatus::NOT_STARTED};
+#endif
 
   base::WeakPtrFactory<DeviceTrustSignalReporter> weak_factory_{this};
 };
