diff --git a/chrome/browser/ui/passwords/well_known_change_password_navigation_throttle.h b/chrome/browser/ui/passwords/well_known_change_password_navigation_throttle.h
index bec600dace07..6296c9e4f473 100644
--- a/chrome/browser/ui/passwords/well_known_change_password_navigation_throttle.h
+++ b/chrome/browser/ui/passwords/well_known_change_password_navigation_throttle.h
@@ -11,7 +11,6 @@
 
 #include "components/password_manager/core/browser/well_known_change_password_state.h"
 #include "components/password_manager/core/browser/well_known_change_password_util.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 
 class GURL;
 namespace content {
@@ -54,12 +53,9 @@ class WellKnownChangePasswordNavigationThrottle
   void OnProcessingFinished(bool is_supported) override;
   // Redirects to a given URL in the same tab.
   void Redirect(const GURL& url);
-  // Records the given UKM metric.
-  void RecordMetric(password_manager::WellKnownChangePasswordResult result);
 
   password_manager::WellKnownChangePasswordState
       well_known_change_password_state_{this};
-  ukm::SourceId source_id_ = ukm::kInvalidSourceId;
   password_manager::ChangePasswordUrlService* change_password_url_service_ =
       nullptr;
   password_manager::AffiliationService* affiliation_service_ = nullptr;
