diff --git a/chrome/browser/ui/views/frame/browser_view.h b/chrome/browser/ui/views/frame/browser_view.h
index f6db8a6928831..241055e1600e2 100644
--- a/chrome/browser/ui/views/frame/browser_view.h
+++ b/chrome/browser/ui/views/frame/browser_view.h
@@ -485,11 +485,13 @@ class BrowserView : public BrowserWindow,
       signin_metrics::AccessPoint access_point,
       bool is_source_keyboard) override;
   void MaybeShowProfileSwitchIPH() override;
+#if 0
   void ShowHatsDialog(
       const std::string& site_id,
       base::OnceClosure success_callback,
       base::OnceClosure failure_callback,
       const std::map<std::string, bool>& product_specific_data) override;
+#endif
   ExclusiveAccessContext* GetExclusiveAccessContext() override;
   std::string GetWorkspace() const override;
   bool IsVisibleOnAllWorkspaces() const override;
