diff --git a/chrome/browser/resource_coordinator/tab_ranker/BUILD.gn b/chrome/browser/resource_coordinator/tab_ranker/BUILD.gn
index 57f37f3784f25..955f884f0a0a7 100644
--- a/chrome/browser/resource_coordinator/tab_ranker/BUILD.gn
+++ b/chrome/browser/resource_coordinator/tab_ranker/BUILD.gn
@@ -30,7 +30,6 @@ source_set("tab_ranker") {
     "//components/assist_ranker",
     "//components/assist_ranker/proto",
     "//components/sessions",
-    "//services/metrics/public/cpp:ukm_builders",
   ]
 }
 
