diff --git a/chrome/browser/tracing/chrome_tracing_delegate.h b/chrome/browser/tracing/chrome_tracing_delegate.h
index 69cb03c59259..a88e8a54ce27 100644
--- a/chrome/browser/tracing/chrome_tracing_delegate.h
+++ b/chrome/browser/tracing/chrome_tracing_delegate.h
@@ -35,9 +35,6 @@ class ChromeTracingDelegate : public content::TracingDelegate,
 
   static void RegisterPrefs(PrefRegistrySimple* registry);
 
-  std::unique_ptr<content::TraceUploader> GetTraceUploader(
-      scoped_refptr<network::SharedURLLoaderFactory> factory) override;
-
   bool IsAllowedToBeginBackgroundScenario(
       const content::BackgroundTracingConfig& config,
       bool requires_anonymized_data) override;
