diff --git a/chrome/browser/extensions/extension_service.h b/chrome/browser/extensions/extension_service.h
index f86268b678f6..0024d42f1ffe 100644
--- a/chrome/browser/extensions/extension_service.h
+++ b/chrome/browser/extensions/extension_service.h
@@ -24,7 +24,6 @@
 #include "chrome/browser/extensions/blocklist.h"
 #include "chrome/browser/extensions/extension_allowlist.h"
 #include "chrome/browser/extensions/extension_management.h"
-#include "chrome/browser/extensions/forced_extensions/force_installed_metrics.h"
 #include "chrome/browser/extensions/forced_extensions/force_installed_tracker.h"
 #include "chrome/browser/extensions/install_gate.h"
 #include "chrome/browser/extensions/pending_extension_manager.h"
@@ -721,9 +720,6 @@ class ExtensionService : public ExtensionServiceInterface,
   // Tracker of enterprise policy forced installation.
   ForceInstalledTracker force_installed_tracker_;
 
-  // Reports force-installed extension metrics to UMA.
-  ForceInstalledMetrics force_installed_metrics_;
-
   ScopedObserver<ProfileManager, ProfileManagerObserver>
       profile_manager_observer_{this};
 
