diff --git a/chrome/browser/ui/passwords/well_known_change_password_navigation_throttle.cc b/chrome/browser/ui/passwords/well_known_change_password_navigation_throttle.cc
index e863e6481cd7..701359ecd7dd 100644
--- a/chrome/browser/ui/passwords/well_known_change_password_navigation_throttle.cc
+++ b/chrome/browser/ui/passwords/well_known_change_password_navigation_throttle.cc
@@ -15,7 +15,6 @@
 #include "components/password_manager/core/browser/well_known_change_password_state.h"
 #include "components/password_manager/core/browser/well_known_change_password_util.h"
 #include "components/password_manager/core/common/password_manager_features.h"
-#include "components/ukm/content/source_url_recorder.h"
 #include "content/public/browser/browser_context.h"
 #include "content/public/browser/navigation_handle.h"
 #include "content/public/browser/page_navigator.h"
@@ -23,8 +22,6 @@
 #include "content/public/browser/web_contents.h"
 #include "content/public/browser/web_contents_user_data.h"
 #include "net/base/isolation_info.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
 #include "services/network/public/cpp/shared_url_loader_factory.h"
 #include "ui/base/page_transition_types.h"
 #include "url/gurl.h"
@@ -109,9 +106,7 @@ WellKnownChangePasswordNavigationThrottle::MaybeCreateThrottleFor(
 WellKnownChangePasswordNavigationThrottle::
     WellKnownChangePasswordNavigationThrottle(NavigationHandle* handle)
     : NavigationThrottle(handle),
-      request_url_(handle->GetURL()),
-      source_id_(
-          ukm::GetSourceIdForWebContentsDocument(handle->GetWebContents())) {
+      request_url_(handle->GetURL()) {
   if (base::FeatureList::IsEnabled(
           password_manager::features::kChangePasswordAffiliationInfo)) {
     affiliation_service_ =
@@ -182,7 +177,6 @@ const char* WellKnownChangePasswordNavigationThrottle::GetNameForLogging() {
 void WellKnownChangePasswordNavigationThrottle::OnProcessingFinished(
     bool is_supported) {
   if (is_supported) {
-    RecordMetric(WellKnownChangePasswordResult::kUsedWellKnownChangePassword);
     Resume();
     return;
   }
@@ -195,10 +189,8 @@ void WellKnownChangePasswordNavigationThrottle::OnProcessingFinished(
         change_password_url_service_->GetChangePasswordUrl(request_url_);
   }
   if (redirect_url.is_valid()) {
-    RecordMetric(WellKnownChangePasswordResult::kFallbackToOverrideUrl);
     Redirect(redirect_url);
   } else {
-    RecordMetric(WellKnownChangePasswordResult::kFallbackToOriginUrl);
     Redirect(request_url_.GetOrigin());
   }
   CancelDeferredNavigation(NavigationThrottle::CANCEL);
@@ -221,10 +213,3 @@ void WellKnownChangePasswordNavigationThrottle::Redirect(const GURL& url) {
       FROM_HERE, base::BindOnce(&WebContentsLifetimeHelper::NavigateTo,
                                 helper->GetWeakPtr(), std::move(params)));
 }
-
-void WellKnownChangePasswordNavigationThrottle::RecordMetric(
-    WellKnownChangePasswordResult result) {
-  ukm::builders::PasswordManager_WellKnownChangePasswordResult(source_id_)
-      .SetWellKnownChangePasswordResult(static_cast<int64_t>(result))
-      .Record(ukm::UkmRecorder::Get());
-}
