diff --git a/chrome/browser/vr/BUILD.gn b/chrome/browser/vr/BUILD.gn
index 95a7e5f0e6fbd..6b45917d4cfca 100644
--- a/chrome/browser/vr/BUILD.gn
+++ b/chrome/browser/vr/BUILD.gn
@@ -254,7 +254,6 @@ component("vr_common") {
     "//chrome/common:constants",
     "//components/omnibox/browser",
     "//components/omnibox/browser",
-    "//components/ukm/content",
     "//content/public/browser",
     "//device/base",
     "//device/vr:vr_util",
@@ -263,7 +262,6 @@ component("vr_common") {
     "//media",
     "//net",
     "//services/audio/public/cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//skia",
     "//ui/base",
     "//ui/display",
@@ -381,13 +379,11 @@ component("vr_base") {
     "//base",
     "//chrome/app:generated_resources",
     "//components/security_state/core",
-    "//components/ukm/content",
     "//components/url_formatter",
     "//content/public/browser",
     "//device/vr/public/mojom:isolated_xr_service",
     "//media",
     "//net",
-    "//services/metrics/public/cpp:ukm_builders",
     "//skia",
   ]
 
