diff --git a/chrome/browser/predictors/loading_predictor_tab_helper.cc b/chrome/browser/predictors/loading_predictor_tab_helper.cc
index adbf0f923d46..dfa7ab46b4db 100644
--- a/chrome/browser/predictors/loading_predictor_tab_helper.cc
+++ b/chrome/browser/predictors/loading_predictor_tab_helper.cc
@@ -25,7 +25,6 @@
 #include "content/public/browser/browser_thread.h"
 #include "content/public/browser/navigation_handle.h"
 #include "content/public/browser/render_frame_host.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "services/network/public/mojom/fetch_api.mojom.h"
 #include "third_party/blink/public/mojom/loader/resource_load_info.mojom.h"
 
@@ -181,8 +180,6 @@ void LoadingPredictorTabHelper::DidStartNavigation(
 
   auto navigation_id = NavigationID(
       web_contents(),
-      ukm::ConvertToSourceId(navigation_handle->GetNavigationId(),
-                             ukm::SourceIdType::NAVIGATION_ID),
       navigation_handle->GetURL(), navigation_handle->NavigationStart());
   if (!navigation_id.is_valid())
     return;
@@ -228,8 +225,6 @@ void LoadingPredictorTabHelper::DidRedirectNavigation(
 
   auto navigation_id = NavigationID(
       web_contents(),
-      ukm::ConvertToSourceId(navigation_handle->GetNavigationId(),
-                             ukm::SourceIdType::NAVIGATION_ID),
       navigation_handle->GetURL(), navigation_handle->NavigationStart());
   if (!navigation_id.is_valid())
     return;
@@ -269,14 +264,10 @@ void LoadingPredictorTabHelper::DidFinishNavigation(
 
   auto old_navigation_id =
       NavigationID(web_contents(),
-                   ukm::ConvertToSourceId(navigation_handle->GetNavigationId(),
-                                          ukm::SourceIdType::NAVIGATION_ID),
                    navigation_handle->GetRedirectChain().front(),
                    navigation_handle->NavigationStart());
   auto new_navigation_id = NavigationID(
       web_contents(),
-      ukm::ConvertToSourceId(navigation_handle->GetNavigationId(),
-                             ukm::SourceIdType::NAVIGATION_ID),
       navigation_handle->GetURL(), navigation_handle->NavigationStart());
   if (!old_navigation_id.is_valid() || !new_navigation_id.is_valid())
     return;
@@ -300,9 +291,6 @@ void LoadingPredictorTabHelper::ResourceLoadComplete(
   auto navigation_id = NavigationID(web_contents());
   if (!navigation_id.is_valid())
     return;
-
-  predictor_->loading_data_collector()->RecordResourceLoadComplete(
-      navigation_id, resource_load_info);
 }
 
 void LoadingPredictorTabHelper::DidLoadResourceFromMemoryCache(
@@ -327,8 +315,6 @@ void LoadingPredictorTabHelper::DidLoadResourceFromMemoryCache(
       GetRequestPriority(resource_load_info.request_destination);
   resource_load_info.network_info =
       blink::mojom::CommonNetworkInfo::New(false, false, base::nullopt);
-  predictor_->loading_data_collector()->RecordResourceLoadComplete(
-      navigation_id, resource_load_info);
 }
 
 void LoadingPredictorTabHelper::DocumentOnLoadCompletedInMainFrame() {
@@ -340,9 +326,6 @@ void LoadingPredictorTabHelper::DocumentOnLoadCompletedInMainFrame() {
   if (!navigation_id.is_valid())
     return;
 
-  predictor_->loading_data_collector()->RecordMainFrameLoadComplete(
-      navigation_id, last_optimization_guide_prediction_);
-
   // Clear out Optimization Guide Prediction, as it is no longer needed.
   last_optimization_guide_prediction_ = base::nullopt;
 }
