diff --git a/chrome/browser/chrome_browser_field_trials.cc b/chrome/browser/chrome_browser_field_trials.cc
index 63a2aeadf005..4c51b45c3108 100644
--- a/chrome/browser/chrome_browser_field_trials.cc
+++ b/chrome/browser/chrome_browser_field_trials.cc
@@ -21,8 +21,6 @@
 #include "chrome/common/chrome_paths.h"
 #include "chrome/common/chrome_switches.h"
 #include "components/metrics/metrics_pref_names.h"
-#include "components/metrics/persistent_histograms.h"
-#include "components/ukm/ukm_recorder_impl.h"
 #include "components/version_info/version_info.h"
 
 #if defined(OS_ANDROID)
@@ -49,13 +47,6 @@ void CreateFallbackSamplingTrialIfNeeded(base::FeatureList* feature_list) {
 #endif  // defined(OS_WIN) || defined(OS_ANDROID)
 }
 
-// Create a field trial to control UKM sampling for Stable if no variations
-// seed was applied.
-void CreateFallbackUkmSamplingTrialIfNeeded(base::FeatureList* feature_list) {
-  ukm::UkmRecorderImpl::CreateFallbackSamplingTrial(
-      chrome::GetChannel() == version_info::Channel::STABLE, feature_list);
-}
-
 }  // namespace
 
 ChromeBrowserFieldTrials::ChromeBrowserFieldTrials(PrefService* local_state)
@@ -87,7 +78,6 @@ void ChromeBrowserFieldTrials::SetupFeatureControllingFieldTrials(
   // is needed.
   if (!has_seed) {
     CreateFallbackSamplingTrialIfNeeded(feature_list);
-    CreateFallbackUkmSamplingTrialIfNeeded(feature_list);
 #if defined(OS_CHROMEOS)
     chromeos::multidevice_setup::CreateFirstRunFieldTrial(feature_list);
 #endif
@@ -114,9 +104,4 @@ void ChromeBrowserFieldTrials::RegisterSyntheticTrials() {
 }
 
 void ChromeBrowserFieldTrials::InstantiateDynamicTrials() {
-  // Persistent histograms must be enabled as soon as possible.
-  base::FilePath metrics_dir;
-  if (base::PathService::Get(chrome::DIR_USER_DATA, &metrics_dir)) {
-    InstantiatePersistentHistograms(metrics_dir);
-  }
 }
