diff --git a/chrome/browser/ui/passwords/bubble_controllers/save_update_with_account_store_bubble_controller.cc b/chrome/browser/ui/passwords/bubble_controllers/save_update_with_account_store_bubble_controller.cc
index 516f98512e41..dd9fe80812d4 100644
--- a/chrome/browser/ui/passwords/bubble_controllers/save_update_with_account_store_bubble_controller.cc
+++ b/chrome/browser/ui/passwords/bubble_controllers/save_update_with_account_store_bubble_controller.cc
@@ -17,7 +17,6 @@
 #include "chrome/grit/generated_resources.h"
 #include "chrome/grit/theme_resources.h"
 #include "components/password_manager/core/browser/password_feature_manager.h"
-#include "components/password_manager/core/browser/password_form_metrics_recorder.h"
 #include "components/password_manager/core/browser/password_manager_features_util.h"
 #include "components/password_manager/core/browser/password_manager_metrics_util.h"
 #include "components/password_manager/core/browser/password_store.h"
@@ -361,8 +360,4 @@ void SaveUpdateWithAccountStoreBubbleController::ReportInteractions() {
   if (no_interaction && delegate_) {
     delegate_->OnNoInteraction();
   }
-
-  // Record UKM statistics on dismissal reason.
-  if (metrics_recorder_)
-    metrics_recorder_->RecordUIDismissalReason(dismissal_reason_);
 }
