diff --git a/chrome/browser/permissions/chrome_permissions_client.cc b/chrome/browser/permissions/chrome_permissions_client.cc
index 99a6ff9bace4..033538047c06 100644
--- a/chrome/browser/permissions/chrome_permissions_client.cc
+++ b/chrome/browser/permissions/chrome_permissions_client.cc
@@ -12,7 +12,6 @@
 #include "chrome/browser/content_settings/host_content_settings_map_factory.h"
 #include "chrome/browser/engagement/important_sites_util.h"
 #include "chrome/browser/engagement/site_engagement_service.h"
-#include "chrome/browser/metrics/ukm_background_recorder_service.h"
 #include "chrome/browser/permissions/abusive_origin_permission_revocation_request.h"
 #include "chrome/browser/permissions/adaptive_quiet_notification_permission_ui_enabler.h"
 #include "chrome/browser/permissions/contextual_notification_permission_ui_selector.h"
@@ -21,9 +20,6 @@
 #include "chrome/browser/permissions/quiet_notification_permission_ui_config.h"
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/search_engines/ui_thread_search_terms_data.h"
-#include "chrome/browser/subresource_filter/subresource_filter_content_settings_manager.h"
-#include "chrome/browser/subresource_filter/subresource_filter_profile_context.h"
-#include "chrome/browser/subresource_filter/subresource_filter_profile_context_factory.h"
 #include "chrome/browser/usb/usb_chooser_context.h"
 #include "chrome/browser/usb/usb_chooser_context_factory.h"
 #include "chrome/common/pref_names.h"
@@ -32,7 +28,6 @@
 #include "components/google/core/common/google_util.h"
 #include "components/permissions/features.h"
 #include "components/prefs/pref_service.h"
-#include "components/ukm/content/source_url_recorder.h"
 #include "extensions/common/constants.h"
 #include "net/base/registry_controlled_domains/registry_controlled_domain.h"
 #include "url/origin.h"
@@ -77,10 +72,7 @@ ChromePermissionsClient::GetCookieSettings(
 bool ChromePermissionsClient::IsSubresourceFilterActivated(
     content::BrowserContext* browser_context,
     const GURL& url) {
-  return SubresourceFilterProfileContextFactory::GetForProfile(
-             Profile::FromBrowserContext(browser_context))
-      ->settings_manager()
-      ->GetSiteActivationFromMetadata(url);
+  return false;
 }
 
 permissions::ChooserContextBase* ChromePermissionsClient::GetChooserContext(
@@ -166,25 +158,6 @@ bool ChromePermissionsClient::IsCookieDeletionDisabled(
 }
 #endif
 
-void ChromePermissionsClient::GetUkmSourceId(
-    content::BrowserContext* browser_context,
-    const content::WebContents* web_contents,
-    const GURL& requesting_origin,
-    GetUkmSourceIdCallback callback) {
-  if (web_contents) {
-    ukm::SourceId source_id =
-        ukm::GetSourceIdForWebContentsDocument(web_contents);
-    std::move(callback).Run(source_id);
-  } else {
-    // We only record a permission change if the origin is in the user's
-    // history.
-    ukm::UkmBackgroundRecorderFactory::GetForProfile(
-        Profile::FromBrowserContext(browser_context))
-        ->GetBackgroundSourceIdIfAllowed(url::Origin::Create(requesting_origin),
-                                         std::move(callback));
-  }
-}
-
 permissions::PermissionRequest::IconId
 ChromePermissionsClient::GetOverrideIconId(ContentSettingsType type) {
 #if defined(OS_CHROMEOS)
