diff --git a/chrome/browser/ui/content_settings/content_setting_bubble_model.h b/chrome/browser/ui/content_settings/content_setting_bubble_model.h
index 2bfba8146251a..d7e112c669f39 100644
--- a/chrome/browser/ui/content_settings/content_setting_bubble_model.h
+++ b/chrome/browser/ui/content_settings/content_setting_bubble_model.h
@@ -59,7 +59,6 @@ class Event;
 // Forward declaration necessary for downcasts.
 class ContentSettingSimpleBubbleModel;
 class ContentSettingMediaStreamBubbleModel;
-class ContentSettingSubresourceFilterBubbleModel;
 class ContentSettingDownloadsBubbleModel;
 class ContentSettingFramebustBlockBubbleModel;
 class ContentSettingNotificationsBubbleModel;
@@ -203,11 +202,6 @@ class ContentSettingBubbleModel {
   // Cast this bubble into ContentSettingMediaStreamBubbleModel if possible.
   virtual ContentSettingMediaStreamBubbleModel* AsMediaStreamBubbleModel();
 
-  // Cast this bubble into ContentSettingSubresourceFilterBubbleModel
-  // if possible.
-  virtual ContentSettingSubresourceFilterBubbleModel*
-  AsSubresourceFilterBubbleModel();
-
   // Cast this bubble into ContentSettingDownloadsBubbleModel if possible.
   virtual ContentSettingDownloadsBubbleModel* AsDownloadsBubbleModel();
 
@@ -425,33 +419,6 @@ class ContentSettingNotificationsBubbleModel
   DISALLOW_COPY_AND_ASSIGN(ContentSettingNotificationsBubbleModel);
 };
 
-// The model for the deceptive content bubble.
-class ContentSettingSubresourceFilterBubbleModel
-    : public ContentSettingBubbleModel {
- public:
-  ContentSettingSubresourceFilterBubbleModel(
-      Delegate* delegate,
-      content::WebContents* web_contents);
-
-  ~ContentSettingSubresourceFilterBubbleModel() override;
-
- private:
-  void SetMessage();
-  void SetTitle();
-  void SetManageText();
-
-  // ContentSettingBubbleModel:
-  void OnManageCheckboxChecked(bool is_checked) override;
-  ContentSettingSubresourceFilterBubbleModel* AsSubresourceFilterBubbleModel()
-      override;
-  void OnLearnMoreClicked() override;
-  void CommitChanges() override;
-
-  bool is_checked_ = false;
-
-  DISALLOW_COPY_AND_ASSIGN(ContentSettingSubresourceFilterBubbleModel);
-};
-
 // The model for automatic downloads setting.
 class ContentSettingDownloadsBubbleModel : public ContentSettingBubbleModel {
  public:
