diff --git a/chrome/browser/ui/ui_features.cc b/chrome/browser/ui/ui_features.cc
index abb8120729fec..beab8ce34fc7e 100644
--- a/chrome/browser/ui/ui_features.cc
+++ b/chrome/browser/ui/ui_features.cc
@@ -117,6 +117,7 @@ const base::Feature kTabGroupsFeedback{"TabGroupsFeedback",
 const base::Feature kTabGroupsNewBadgePromo{"TabGroupsNewBadgePromo",
                                             base::FEATURE_DISABLED_BY_DEFAULT};
 
+#if 0
 // Enables popup cards containing tab information when hovering over a tab.
 // https://crbug.com/910739
 const base::Feature kTabHoverCards{"TabHoverCards",
@@ -125,6 +126,7 @@ const base::Feature kTabHoverCards{"TabHoverCards",
 // Parameter name used for tab hover cards user study.
 // TODO(corising): Removed this after tab hover cards user study.
 const char kTabHoverCardsFeatureParameterName[] = "setting";
+#endif
 
 // Enables preview images in hover cards. See kTabHoverCards.
 // https://crbug.com/928954
