diff --git a/chrome/browser/win/conflicts/module_database.cc b/chrome/browser/win/conflicts/module_database.cc
index 60a15f7d1a5d..0d42ed9b192e 100644
--- a/chrome/browser/win/conflicts/module_database.cc
+++ b/chrome/browser/win/conflicts/module_database.cc
@@ -90,7 +90,6 @@ ModuleDatabase::ModuleDatabase(bool third_party_blocking_policy_enabled)
       module_inspector_(base::BindRepeating(&ModuleDatabase::OnModuleInspected,
                                             base::Unretained(this))) {
   AddObserver(&module_inspector_);
-  AddObserver(&third_party_metrics_);
 
 #if defined(GOOGLE_CHROME_BUILD)
   MaybeInitializeThirdPartyConflictsManager(
@@ -412,8 +411,6 @@ void ModuleDatabase::NotifyLoadedModules(ModuleDatabaseObserver* observer) {
 
 #if defined(GOOGLE_CHROME_BUILD)
 void ModuleDatabase::OnThirdPartyBlockingDisabled() {
-  third_party_metrics_.SetHookDisabled();
-
   if (third_party_conflicts_manager_)
     third_party_conflicts_manager_->DisableModuleAnalysis();
 }
