diff --git a/chrome/browser/metrics/chrome_metrics_service_client.h b/chrome/browser/metrics/chrome_metrics_service_client.h
index 116765ed1155..29471232bb16 100644
--- a/chrome/browser/metrics/chrome_metrics_service_client.h
+++ b/chrome/browser/metrics/chrome_metrics_service_client.h
@@ -110,10 +110,6 @@ class ChromeMetricsServiceClient : public metrics::MetricsServiceClient,
   // Completes the two-phase initialization of ChromeMetricsServiceClient.
   void Initialize();
 
-  // Registers providers to the MetricsService. These provide data from
-  // alternate sources.
-  void RegisterMetricsServiceProviders();
-
   // Registers providers to the UkmService. These provide data from alternate
   // sources.
   void RegisterUKMProviders();
