diff --git a/chrome/renderer/subresource_redirect/subresource_redirect_url_loader_throttle.cc b/chrome/renderer/subresource_redirect/subresource_redirect_url_loader_throttle.cc
index 0fcd6db52edb..bc8f483ea287 100644
--- a/chrome/renderer/subresource_redirect/subresource_redirect_url_loader_throttle.cc
+++ b/chrome/renderer/subresource_redirect/subresource_redirect_url_loader_throttle.cc
@@ -260,11 +260,6 @@ void SubresourceRedirectURLLoaderThrottle::BeforeWillProcessResponse(
       net::HttpUtil::ParseRetryAfterHeader(retry_after_string,
                                            base::Time::Now(), &retry_after);
     }
-    if (auto* public_resource_decider_agent =
-            GetPublicResourceDeciderAgent(render_frame_id_)) {
-      public_resource_decider_agent->NotifyCompressedResourceFetchFailed(
-          retry_after);
-    }
   }
 
   // Non 2XX responses from the compression server need to have unaltered
@@ -372,11 +367,6 @@ void SubresourceRedirectURLLoaderThrottle::OnRedirectTimeout() {
   DCHECK_EQ(RedirectState::kRedirectAttempted, redirect_state_);
   redirect_state_ = RedirectState::kRedirectFailed;
   delegate_->RestartWithURLResetAndFlagsNow(net::LOAD_NORMAL);
-  if (auto* public_resource_decider_agent =
-          GetPublicResourceDeciderAgent(render_frame_id_)) {
-    public_resource_decider_agent->NotifyCompressedResourceFetchFailed(
-        base::TimeDelta());
-  }
   UMA_HISTOGRAM_BOOLEAN("SubresourceRedirect.CompressionFetchTimeout", true);
 }
 
