diff --git a/chrome/browser/ui/webui/discards/graph_dump_impl.h b/chrome/browser/ui/webui/discards/graph_dump_impl.h
index 79b4762a383b..7a8689b39909 100644
--- a/chrome/browser/ui/webui/discards/graph_dump_impl.h
+++ b/chrome/browser/ui/webui/discards/graph_dump_impl.h
@@ -120,8 +120,6 @@ class DiscardsGraphDumpImpl : public discards::mojom::GraphDump,
       const performance_manager::PageNode* page_node) override {}  // Ignored.
   void OnIsLoadingChanged(
       const performance_manager::PageNode* page_node) override {}  // Ignored.
-  void OnUkmSourceIdChanged(
-      const performance_manager::PageNode* page_node) override {}  // Ignored.
   // Ignored.
   void OnPageLifecycleStateChanged(
       const performance_manager::PageNode* page_node) override {}
