diff --git a/chrome/browser/ui/views/reader_mode/reader_mode_icon_view.cc b/chrome/browser/ui/views/reader_mode/reader_mode_icon_view.cc
index 2b18e55d65783..9ee85dc1b43ef 100644
--- a/chrome/browser/ui/views/reader_mode/reader_mode_icon_view.cc
+++ b/chrome/browser/ui/views/reader_mode/reader_mode_icon_view.cc
@@ -11,12 +11,9 @@
 #include "components/dom_distiller/core/dom_distiller_features.h"
 #include "components/dom_distiller/core/url_utils.h"
 #include "components/prefs/pref_service.h"
-#include "components/ukm/content/source_url_recorder.h"
 #include "content/public/browser/navigation_details.h"
 #include "content/public/browser/navigation_handle.h"
 #include "content/public/browser/web_contents.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
 #include "ui/base/l10n/l10n_util.h"
 #include "ui/base/metadata/metadata_impl_macros.h"
 
@@ -155,10 +152,6 @@ void ReaderModeIconView::OnExecuting(
   content::WebContents* contents = GetWebContents();
   if (!contents || IsDistilledPage(contents->GetLastCommittedURL()))
     return;
-  ukm::SourceId source_id = ukm::GetSourceIdForWebContentsDocument(contents);
-  ukm::builders::ReaderModeActivated(source_id)
-      .SetActivatedViaOmnibox(true)
-      .Record(ukm::UkmRecorder::Get());
 }
 
 void ReaderModeIconView::OnResult(
@@ -169,14 +162,6 @@ void ReaderModeIconView::OnResult(
     return;
   }
 
-  if (result.is_last) {
-    ukm::SourceId source_id =
-        ukm::GetSourceIdForWebContentsDocument(web_contents);
-    ukm::builders::ReaderModeReceivedDistillability(source_id)
-        .SetIsPageDistillable(result.is_distillable)
-        .Record(ukm::UkmRecorder::Get());
-  }
-
   Update();
 
   // Once we know the type of page we are on (distillable or not), we can
