diff --git a/chrome/browser/ui/views/tabs/fake_base_tab_strip_controller.cc b/chrome/browser/ui/views/tabs/fake_base_tab_strip_controller.cc
index c25d65f9eb5f5..4f894c114a3f4 100644
--- a/chrome/browser/ui/views/tabs/fake_base_tab_strip_controller.cc
+++ b/chrome/browser/ui/views/tabs/fake_base_tab_strip_controller.cc
@@ -235,6 +235,9 @@ void FakeBaseTabStripController::CloseTab(int index) {
   RemoveTab(index);
 }
 
+void FakeBaseTabStripController::ToggleTabAudioMute(int index) {
+}
+
 void FakeBaseTabStripController::ShowContextMenuForTab(
     Tab* tab,
     const gfx::Point& p,
