diff --git a/chrome/browser/ui/webui/help/version_updater_basic.cc b/chrome/browser/ui/webui/help/version_updater_basic.cc
index 3545b63afb7e7..7b5ca0bf24aba 100644
--- a/chrome/browser/ui/webui/help/version_updater_basic.cc
+++ b/chrome/browser/ui/webui/help/version_updater_basic.cc
@@ -6,13 +6,9 @@
 
 #include <string>
 
-#include "chrome/browser/upgrade_detector/upgrade_detector.h"
-
 void VersionUpdaterBasic::CheckForUpdate(StatusCallback status_callback,
                                          PromoteCallback) {
-  const Status status = UpgradeDetector::GetInstance()->notify_upgrade()
-                            ? NEARLY_UPDATED
-                            : DISABLED;
+  const Status status = DISABLED;
   status_callback.Run(status, 0, false, false, std::string(), 0,
                       std::u16string());
 }
