diff --git a/chrome/browser/ui/ui_features.h b/chrome/browser/ui/ui_features.h
index 26f1eab90e97e..fd1de002f082d 100644
--- a/chrome/browser/ui/ui_features.h
+++ b/chrome/browser/ui/ui_features.h
@@ -14,6 +14,7 @@
 #include "build/build_config.h"
 #include "build/chromeos_buildflags.h"
 #include "chrome/common/buildflags.h"
+#include "dumb/browser/ui/ui_features.h"
 
 namespace features {
 
@@ -72,8 +73,10 @@ extern const base::Feature kTabGroupsFeedback;
 
 extern const base::Feature kTabGroupsNewBadgePromo;
 
+#if 0
 extern const base::Feature kTabHoverCards;
 extern const char kTabHoverCardsFeatureParameterName[];
+#endif
 
 extern const base::Feature kTabHoverCardImages;
 extern const char kTabHoverCardImagesNotReadyDelayParameterName[];
