diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index d85c8180793d..fc4bcb2dcd22 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -128,7 +128,6 @@ static_library("test_support") {
     "//components/gwp_asan/buildflags",
     "//components/heap_profiling/in_process",
     "//components/nacl/common:buildflags",
-    "//components/startup_metric_utils/browser",
     "//content/public/app",
     "//content/public/common",
     "//content/public/common:service_names",
@@ -143,7 +142,6 @@ static_library("test_support") {
   if (!is_fuchsia) {
     # TODO(crbug.com/753619): Enable crash reporting on Fuchsia.
     deps += [
-      "//components/crash/core/app",
       "//components/crash/core/app:test_support",
     ]
   }
