diff --git a/chrome/browser/tracing/chrome_tracing_delegate.cc b/chrome/browser/tracing/chrome_tracing_delegate.cc
index 3cfc73bc23c5d..858bdbb73499a 100644
--- a/chrome/browser/tracing/chrome_tracing_delegate.cc
+++ b/chrome/browser/tracing/chrome_tracing_delegate.cc
@@ -19,7 +19,6 @@
 #include "chrome/browser/browser_process.h"
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/profiles/profile_manager.h"
-#include "chrome/browser/tracing/crash_service_uploader.h"
 #include "chrome/browser/ui/browser_otr_state.h"
 #include "chrome/common/pref_names.h"
 #include "components/metrics/metrics_pref_names.h"
@@ -115,12 +114,6 @@ void ChromeTracingDelegate::OnBrowserAdded(Browser* browser) {
 }
 #endif  // defined(OS_ANDROID)
 
-std::unique_ptr<content::TraceUploader> ChromeTracingDelegate::GetTraceUploader(
-    scoped_refptr<network::SharedURLLoaderFactory> factory) {
-  return std::unique_ptr<content::TraceUploader>(
-      new TraceCrashServiceUploader(std::move(factory)));
-}
-
 namespace {
 
 enum PermitMissingProfile { PROFILE_REQUIRED, PROFILE_NOT_REQUIRED };
