diff --git a/chrome/browser/chromeos/BUILD.gn b/chrome/browser/chromeos/BUILD.gn
index be2ca2bec78b..ba4883ebe970 100644
--- a/chrome/browser/chromeos/BUILD.gn
+++ b/chrome/browser/chromeos/BUILD.gn
@@ -263,9 +263,7 @@ source_set("chromeos") {
     "//components/printing/common:mojo_interfaces",
     "//components/proxy_config",
     "//components/quirks",
-    "//components/rappor",
     "//components/renderer_context_menu",
-    "//components/rlz",
     "//components/safe_browsing/core:csd_proto",
     "//components/safe_browsing/core/db:metadata_proto",
     "//components/services/app_service:lib",
