diff --git a/chrome/browser/resources/new_tab_page/new_tab_page_resources.grd b/chrome/browser/resources/new_tab_page/new_tab_page_resources.grd
index 47786cd0109b..d6cfc550227c 100644
--- a/chrome/browser/resources/new_tab_page/new_tab_page_resources.grd
+++ b/chrome/browser/resources/new_tab_page/new_tab_page_resources.grd
@@ -20,9 +20,6 @@
           file="${root_gen_dir}/chrome/browser/resources/new_tab_page/app.js"
           use_base_dir="false" type="BINDATA" compress="false"
           preprocess="true" />
-      <include name="IDR_NEW_TAB_PAGE_MIDDLE_SLOT_PROMO_JS"
-          file="${root_gen_dir}/chrome/browser/resources/new_tab_page/middle_slot_promo.js"
-          use_base_dir="false" type="BINDATA" compress="false" />
       <include name="IDR_NEW_TAB_PAGE_MOST_VISITED_JS"
           file="${root_gen_dir}/chrome/browser/resources/new_tab_page/most_visited.js"
           use_base_dir="false" type="BINDATA" compress="false" />
