diff --git a/chrome/browser/chrome_browser_main_mac.mm b/chrome/browser/chrome_browser_main_mac.mm
index c30b73c2575f..2da60e613959 100644
--- a/chrome/browser/chrome_browser_main_mac.mm
+++ b/chrome/browser/chrome_browser_main_mac.mm
@@ -75,8 +75,6 @@
 }
 
 void ChromeBrowserMainPartsMac::PreMainMessageLoopStart() {
-  MacStartupProfiler::GetInstance()->Profile(
-      MacStartupProfiler::PRE_MAIN_MESSAGE_LOOP_START);
   ChromeBrowserMainPartsPosix::PreMainMessageLoopStart();
 
   // ChromeBrowserMainParts should have loaded the resource bundle by this
@@ -135,8 +133,6 @@
 }
 
 void ChromeBrowserMainPartsMac::PostMainMessageLoopStart() {
-  MacStartupProfiler::GetInstance()->Profile(
-      MacStartupProfiler::POST_MAIN_MESSAGE_LOOP_START);
   ChromeBrowserMainPartsPosix::PostMainMessageLoopStart();
 
   if (base::FeatureList::IsEnabled(network::features::kCertVerifierService) &&
@@ -147,8 +143,6 @@
 }
 
 void ChromeBrowserMainPartsMac::PreProfileInit() {
-  MacStartupProfiler::GetInstance()->Profile(
-      MacStartupProfiler::PRE_PROFILE_INIT);
   ChromeBrowserMainPartsPosix::PreProfileInit();
 
   // This is called here so that the app shim socket is only created after
@@ -157,13 +151,8 @@
 }
 
 void ChromeBrowserMainPartsMac::PostProfileInit() {
-  MacStartupProfiler::GetInstance()->Profile(
-      MacStartupProfiler::POST_PROFILE_INIT);
   ChromeBrowserMainPartsPosix::PostProfileInit();
 
-  g_browser_process->metrics_service()->RecordBreakpadRegistration(
-      crash_reporter::GetUploadsEnabled());
-
   // Activation of Keystone is not automatic but done in response to the
   // counting and reporting of profiles.
   KeystoneGlue* glue = [KeystoneGlue defaultKeystoneGlue];
