diff --git a/chrome/browser/browser_switcher/alternative_browser_driver_win.cc b/chrome/browser/browser_switcher/alternative_browser_driver_win.cc
index f079e72d7474..6e0ce90612a6 100644
--- a/chrome/browser/browser_switcher/alternative_browser_driver_win.cc
+++ b/chrome/browser/browser_switcher/alternative_browser_driver_win.cc
@@ -43,7 +43,7 @@ const wchar_t kOperaKey[] =
 const wchar_t kSafariKey[] =
     L"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\App Paths\\safari.exe";
 const wchar_t kChromeKey[] =
-    L"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\App Paths\\chrome.exe";
+    L"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\App Paths\\dumb.exe";
 const wchar_t kEdgeKey[] =
     L"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\App Paths\\msedge.exe";
 
@@ -57,7 +57,7 @@ const wchar_t kSafariVarName[] = L"${safari}";
 const wchar_t kEdgeVarName[] = L"${edge}";
 
 // Case-insensitive, typical filenames for popular browsers' executables.
-const wchar_t kChromeTypicalExecutable[] = L"chrome.exe";
+const wchar_t kChromeTypicalExecutable[] = L"dumb.exe";
 const wchar_t kIETypicalExecutable[] = L"iexplore.exe";
 const wchar_t kFirefoxTypicalExecutable[] = L"firefox.exe";
 const wchar_t kOperaTypicalExecutable[] = L"launcher.exe";
