diff --git a/chrome/browser/offline_pages/prefetch/offline_metrics_collector_impl.h b/chrome/browser/offline_pages/prefetch/offline_metrics_collector_impl.h
index fa899cbc3002f..9f24397a4de45 100644
--- a/chrome/browser/offline_pages/prefetch/offline_metrics_collector_impl.h
+++ b/chrome/browser/offline_pages/prefetch/offline_metrics_collector_impl.h
@@ -7,7 +7,6 @@
 
 #include "base/macros.h"
 #include "base/time/clock.h"
-#include "components/offline_pages/core/prefetch/offline_metrics_collector.h"
 #include "components/prefs/pref_registry_simple.h"
 #include "components/prefs/pref_service.h"
 
