diff --git a/chrome/browser/ui/search/search_ipc_router.cc b/chrome/browser/ui/search/search_ipc_router.cc
index 6d4cf0d262f62..e885df0bf0db4 100644
--- a/chrome/browser/ui/search/search_ipc_router.cc
+++ b/chrome/browser/ui/search/search_ipc_router.cc
@@ -441,14 +441,6 @@ void SearchIPCRouter::ConfirmThemeChanges() {
   delegate_->OnConfirmThemeChanges();
 }
 
-void SearchIPCRouter::BlocklistPromo(const std::string& promo_id) {
-  if (!policy_->ShouldProcessBlocklistPromo()) {
-    return;
-  }
-
-  delegate_->BlocklistPromo(promo_id);
-}
-
 void SearchIPCRouter::OpenExtensionsPage(double button,
                                          bool alt_key,
                                          bool ctrl_key,
