diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index 8d66d50525f9..86bd77a50a1a 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -56,7 +56,6 @@
 #include "chrome/browser/ui/views/toolbar/toolbar_account_icon_container_view.h"
 #include "chrome/browser/ui/views/toolbar/toolbar_button.h"
 #include "chrome/browser/ui/web_applications/app_browser_controller.h"
-#include "chrome/browser/upgrade_detector/upgrade_detector.h"
 #include "chrome/common/chrome_features.h"
 #include "chrome/common/pref_names.h"
 #include "chrome/grit/chromium_strings.h"
@@ -99,7 +98,6 @@
 #else
 #include "chrome/browser/signin/signin_global_error_factory.h"
 #include "chrome/browser/ui/bookmarks/bookmark_bubble_sign_in_delegate.h"
-#include "chrome/browser/ui/views/outdated_upgrade_bubble_view.h"
 #endif
 
 #if BUILDFLAG(ENABLE_WEBUI_TAB_STRIP)
@@ -155,7 +153,6 @@ ToolbarView::ToolbarView(Browser* browser, BrowserView* browser_view)
       display_mode_(GetDisplayMode(browser)) {
   SetID(VIEW_ID_TOOLBAR);
 
-  UpgradeDetector::GetInstance()->AddObserver(this);
 
   if (display_mode_ == DisplayMode::NORMAL) {
     SetBackground(std::make_unique<TopContainerBackground>(browser_view));
@@ -166,7 +163,6 @@ ToolbarView::ToolbarView(Browser* browser, BrowserView* browser_view)
 }
 
 ToolbarView::~ToolbarView() {
-  UpgradeDetector::GetInstance()->RemoveObserver(this);
 
   if (display_mode_ != DisplayMode::NORMAL)
     return;
@@ -517,19 +513,6 @@ void ToolbarView::ButtonPressed(views::Button* sender,
       browser_, sender->tag(), ui::DispositionFromEventFlags(event.flags()));
 }
 
-////////////////////////////////////////////////////////////////////////////////
-// ToolbarView, UpgradeObserver implementation:
-void ToolbarView::OnOutdatedInstall() {
-  ShowOutdatedInstallNotification(true);
-}
-
-void ToolbarView::OnOutdatedInstallNoAutoUpdate() {
-  ShowOutdatedInstallNotification(false);
-}
-
-void ToolbarView::OnCriticalUpgradeInstalled() {
-  ShowCriticalNotification();
-}
 
 ////////////////////////////////////////////////////////////////////////////////
 // ToolbarView, ui::AcceleratorProvider implementation:
@@ -918,8 +901,6 @@ void ToolbarView::ShowCriticalNotification() {
 
 void ToolbarView::ShowOutdatedInstallNotification(bool auto_update_enabled) {
 #if !defined(OS_CHROMEOS)
-  OutdatedUpgradeBubbleView::ShowBubble(app_menu_button_, browser_,
-                                        auto_update_enabled);
 #endif
 }
 
