diff --git a/chrome/browser/memory/chrome_browser_main_extra_parts_memory.cc b/chrome/browser/memory/chrome_browser_main_extra_parts_memory.cc
index 9988d5561602..71c55bd0348f 100644
--- a/chrome/browser/memory/chrome_browser_main_extra_parts_memory.cc
+++ b/chrome/browser/memory/chrome_browser_main_extra_parts_memory.cc
@@ -8,7 +8,6 @@
 #include "build/build_config.h"
 #include "build/chromeos_buildflags.h"
 #include "chrome/browser/browser_process.h"
-#include "chrome/browser/memory/enterprise_memory_limit_pref_observer.h"
 
 #if BUILDFLAG(IS_CHROMEOS_ASH)
 #include "base/logging.h"
@@ -24,12 +23,6 @@ ChromeBrowserMainExtraPartsMemory::~ChromeBrowserMainExtraPartsMemory() =
 void ChromeBrowserMainExtraPartsMemory::PostBrowserStart() {
   // The MemoryPressureMonitor might not be available in some tests.
   if (base::MemoryPressureMonitor::Get()) {
-    if (memory::EnterpriseMemoryLimitPrefObserver::PlatformIsSupported()) {
-      memory_limit_pref_observer_ =
-          std::make_unique<memory::EnterpriseMemoryLimitPrefObserver>(
-              g_browser_process->local_state());
-    }
-
 #if BUILDFLAG(IS_CHROMEOS_ASH)
     if (base::SysInfo::IsRunningOnChromeOS()) {
       cros_evaluator_ =
@@ -43,11 +36,6 @@ void ChromeBrowserMainExtraPartsMemory::PostBrowserStart() {
 }
 
 void ChromeBrowserMainExtraPartsMemory::PostMainMessageLoopRun() {
-  // |memory_limit_pref_observer_| must be destroyed before its |pref_service_|
-  // is destroyed, as the observer's PrefChangeRegistrar's destructor uses the
-  // pref_service.
-  memory_limit_pref_observer_.reset();
-
 #if BUILDFLAG(IS_CHROMEOS_ASH)
   cros_evaluator_.reset();
 #endif
