diff --git a/chrome/browser/ui/cocoa/notifications/BUILD.gn b/chrome/browser/ui/cocoa/notifications/BUILD.gn
index 722621ed41f0..f1e1056c62e9 100644
--- a/chrome/browser/ui/cocoa/notifications/BUILD.gn
+++ b/chrome/browser/ui/cocoa/notifications/BUILD.gn
@@ -27,7 +27,6 @@ mac_app_bundle("alert_notification_xpc_service") {
   deps = [
     ":common",
     "//base",
-    "//third_party/crashpad/crashpad/client",
   ]
 
   frameworks = [
