diff --git a/chrome/renderer/extensions/chrome_extensions_dispatcher_delegate.cc b/chrome/renderer/extensions/chrome_extensions_dispatcher_delegate.cc
index 4195d36ff1a8..deaf019fbea7 100644
--- a/chrome/renderer/extensions/chrome_extensions_dispatcher_delegate.cc
+++ b/chrome/renderer/extensions/chrome_extensions_dispatcher_delegate.cc
@@ -11,7 +11,6 @@
 #include "build/chromeos_buildflags.h"
 #include "chrome/common/channel_info.h"
 #include "chrome/common/chrome_switches.h"
-#include "chrome/common/crash_keys.h"
 #include "chrome/grit/renderer_resources.h"
 #include "chrome/renderer/extensions/accessibility_private_hooks_delegate.h"
 #include "chrome/renderer/extensions/app_hooks_delegate.h"
@@ -261,7 +260,6 @@ void ChromeExtensionsDispatcherDelegate::OnActiveExtensionsUpdated(
   if (base::CommandLine::ForCurrentProcess()->HasSwitch(
           ::switches::kSingleProcess))
     return;
-  crash_keys::SetActiveExtensions(extension_ids);
 }
 
 void ChromeExtensionsDispatcherDelegate::InitializeBindingsSystem(
