diff --git a/chrome/browser/extensions/api/tabs/tabs_constants.cc b/chrome/browser/extensions/api/tabs/tabs_constants.cc
index 0e1640ae85a50..aac4f747e4121 100644
--- a/chrome/browser/extensions/api/tabs/tabs_constants.cc
+++ b/chrome/browser/extensions/api/tabs/tabs_constants.cc
@@ -7,6 +7,8 @@
 namespace extensions {
 namespace tabs_constants {
 
+const char kCannotUpdateMuteDisabled[] =
+  "Failed to update mute state for tab *, --* must be enabled";
 const char kActiveKey[] = "active";
 const char kAllFramesKey[] = "allFrames";
 const char kAlwaysOnTopKey[] = "alwaysOnTop";
