diff --git a/chrome/browser/password_manager/password_store_factory.cc b/chrome/browser/password_manager/password_store_factory.cc
index 11534a4b1222..f939205690ff 100644
--- a/chrome/browser/password_manager/password_store_factory.cc
+++ b/chrome/browser/password_manager/password_store_factory.cc
@@ -46,9 +46,7 @@
 #include "ui/base/ui_base_features.h"
 #endif
 
-#if defined(PASSWORD_REUSE_DETECTION_ENABLED)
 #include "chrome/browser/password_manager/password_store_signin_notifier_impl.h"
-#endif
 
 using password_manager::PasswordStore;
 
