diff --git a/chrome/chrome_elf/chrome_elf_x64.def b/chrome/chrome_elf/chrome_elf_x64.def
index cdc6f52fb846..8e2c561954a9 100644
--- a/chrome/chrome_elf/chrome_elf_x64.def
+++ b/chrome/chrome_elf/chrome_elf_x64.def
@@ -7,16 +7,6 @@ EXPORTS
   ; When functions are added to this file, they must also be added to
   ; chrome_elf_x86.def and chrome_elf_arm64.def
 
-  ; From components/crash/core/app/crash_export_stubs.cc
-  CrashForException_ExportThunk
-  DumpHungProcessWithPtype_ExportThunk
-  GetCrashpadDatabasePath_ExportThunk
-  ClearReportsBetween_ExportThunk
-  GetCrashReports_ExportThunk
-  InjectDumpForHungInput_ExportThunk
-  RequestSingleCrashUpload_ExportThunk
-  SetUploadConsent_ExportThunk
-
   ; From chrome/chrome_elf/crash/crash_helper.cc
   SetMetricsClientId
 
@@ -24,7 +14,6 @@ EXPORTS
   DumpProcessWithoutCrash
   GetUserDataDirectoryThunk
   SignalChromeElf
-  SignalInitializeCrashReporting
 
   ; From chrome/install_static
   GetInstallDetailsPayload
