diff --git a/chrome/browser/resources/BUILD.gn b/chrome/browser/resources/BUILD.gn
index 48c8af41a890f..755800ecbdbd9 100644
--- a/chrome/browser/resources/BUILD.gn
+++ b/chrome/browser/resources/BUILD.gn
@@ -95,7 +95,6 @@ if (enable_js_type_check) {
     deps = [
       "accessibility:closure_compile",
       "components:closure_compile",
-      "domain_reliability_internals:closure_compile",
       "engagement:closure_compile",
       "media:closure_compile",
       "memory_internals:closure_compile",
