diff --git a/chrome/browser/predictors/loading_data_collector.h b/chrome/browser/predictors/loading_data_collector.h
index b0de2852bdee..9f22878dc8e3 100644
--- a/chrome/browser/predictors/loading_data_collector.h
+++ b/chrome/browser/predictors/loading_data_collector.h
@@ -47,7 +47,6 @@ struct PageRequestSummary {
   void AddPreconnectAttempt(const GURL& preconnect_url);
   void AddPrefetchAttempt(const GURL& prefetch_url);
 
-  ukm::SourceId ukm_source_id;
   GURL main_frame_url;
   GURL initial_url;
   base::TimeTicks navigation_started;
