diff --git a/chrome/installer/setup/BUILD.gn b/chrome/installer/setup/BUILD.gn
index db3796b6b6ef7..430d4a614aed0 100644
--- a/chrome/installer/setup/BUILD.gn
+++ b/chrome/installer/setup/BUILD.gn
@@ -45,7 +45,6 @@ if (is_win) {
       "//components/crash/core/app:crash_export_thunks",
       "//components/crash/core/app:run_as_crashpad_handler",
       "//content/public/common:static_switches",
-      "//rlz:rlz_lib_no_network",
     ]
 
     libs = [ "netapi32.lib" ]
@@ -64,10 +63,6 @@ if (is_win) {
       "install_params.h",
       "install_worker.cc",
       "install_worker.h",
-      "installer_crash_reporter_client.cc",
-      "installer_crash_reporter_client.h",
-      "installer_crash_reporting.cc",
-      "installer_crash_reporting.h",
       "installer_state.cc",
       "installer_state.h",
       "last_breaking_installer_version.cc",
