diff --git a/chrome/browser/extensions/api/BUILD.gn b/chrome/browser/extensions/api/BUILD.gn
index aac94e9465ff..fb754147b2c9 100644
--- a/chrome/browser/extensions/api/BUILD.gn
+++ b/chrome/browser/extensions/api/BUILD.gn
@@ -39,7 +39,6 @@ function_registration("api_registration") {
     "//chrome/common/extensions/api",
     "//components/sync",
     "//skia",
-    "//third_party/metrics_proto",
     "//ui/accessibility:ax_enums_mojo",
   ]
   deps += [ "//extensions/common/api" ]
