diff --git a/chrome/android/chrome_java_sources.gni b/chrome/android/chrome_java_sources.gni
index 17cdabb8a1de..5f3d0f98bb73 100644
--- a/chrome/android/chrome_java_sources.gni
+++ b/chrome/android/chrome_java_sources.gni
@@ -1316,7 +1316,6 @@ chrome_java_sources = [
   "java/src/org/chromium/chrome/browser/query_tiles/QueryTileSection.java",
   "java/src/org/chromium/chrome/browser/query_tiles/QueryTileUtils.java",
   "java/src/org/chromium/chrome/browser/query_tiles/TileProviderFactory.java",
-  "java/src/org/chromium/chrome/browser/rappor/RapporServiceBridge.java",
   "java/src/org/chromium/chrome/browser/reengagement/ReengagementNotificationController.java",
   "java/src/org/chromium/chrome/browser/resources/ResourceMapper.java",
   "java/src/org/chromium/chrome/browser/rlz/RevenueStats.java",
