diff --git a/chrome/browser/performance_manager/observers/metrics_collector.h b/chrome/browser/performance_manager/observers/metrics_collector.h
index 1ef40f9dbb277..71062b83dbd5d 100644
--- a/chrome/browser/performance_manager/observers/metrics_collector.h
+++ b/chrome/browser/performance_manager/observers/metrics_collector.h
@@ -82,7 +82,6 @@ class MetricsCollector : public FrameNode::ObserverDefaultImpl,
 
  private:
   static MetricsReportRecord* GetMetricsReportRecord(const PageNode* page_node);
-  static UkmCollectionState* GetUkmCollectionState(const PageNode* page_node);
 
   // (Un)registers the various node observer flavors of this object with the
   // graph. These are invoked by OnPassedToGraph and OnTakenFromGraph, but
