diff --git a/chrome/browser/web_applications/chrome_pwa_launcher/BUILD.gn b/chrome/browser/web_applications/chrome_pwa_launcher/BUILD.gn
index 9234f69facbc4..f657430c35ca0 100644
--- a/chrome/browser/web_applications/chrome_pwa_launcher/BUILD.gn
+++ b/chrome/browser/web_applications/chrome_pwa_launcher/BUILD.gn
@@ -43,18 +43,12 @@ source_set("util") {
   public = [
     "chrome_pwa_launcher_util.h",
     "last_browser_file_util.h",
-    "launcher_log_reporter.h",
     "launcher_update.h",
   ]
 
   sources = [
     "chrome_pwa_launcher_util.cc",
     "last_browser_file_util.cc",
-    "launcher_log.cc",
-    "launcher_log.h",
-    "launcher_log_reporter.cc",
-    "launcher_log_util.cc",
-    "launcher_log_util.h",
     "launcher_update.cc",
   ]
 
