diff --git a/chrome/common/safe_browsing/BUILD.gn b/chrome/common/safe_browsing/BUILD.gn
index 5b3e64a2144d..a61275c05e2e 100644
--- a/chrome/common/safe_browsing/BUILD.gn
+++ b/chrome/common/safe_browsing/BUILD.gn
@@ -87,21 +87,6 @@ if (safe_browsing_mode == 1) {
 
   source_set("binary_feature_extractor") {
     sources = [
-      "binary_feature_extractor.cc",
-      "binary_feature_extractor.h",
-      "binary_feature_extractor_mac.cc",
-      "binary_feature_extractor_win.cc",
-      "mach_o_image_reader_mac.cc",
-      "mach_o_image_reader_mac.h",
-    ]
-    if (is_posix) {
-      sources += [ "binary_feature_extractor_posix.cc" ]
-    }
-
-    public_deps = [
-      "//base",
-      "//components/safe_browsing/core:csd_proto",
-      "//crypto",
     ]
   }
 
