diff --git a/cloud_print/virtual_driver/win/port_monitor/BUILD.gn b/cloud_print/virtual_driver/win/port_monitor/BUILD.gn
index cf9bdb92c465a..4226259da31e3 100644
--- a/cloud_print/virtual_driver/win/port_monitor/BUILD.gn
+++ b/cloud_print/virtual_driver/win/port_monitor/BUILD.gn
@@ -87,7 +87,6 @@ source_set("lib") {
   deps = [
     "//base",
     "//chrome/common:constants",
-    "//chrome/installer/launcher_support",
     "//cloud_print/common",
     "//cloud_print/virtual_driver/win",
   ]
