diff --git a/components/services/print_compositor/BUILD.gn b/components/services/print_compositor/BUILD.gn
index 8c14ddce91b4..251278d1c108 100644
--- a/components/services/print_compositor/BUILD.gn
+++ b/components/services/print_compositor/BUILD.gn
@@ -12,7 +12,6 @@ static_library("print_compositor") {
 
   deps = [
     "//base",
-    "//components/crash/core/common:crash_key",
     "//components/discardable_memory/client",
     "//components/discardable_memory/public/mojom",
     "//components/services/print_compositor/public/cpp",
@@ -47,7 +46,6 @@ if (enable_basic_printing) {
       "//base",
       "//base/test:test_support",
       "//cc/paint:paint",
-      "//components/crash/core/common:crash_key",
       "//components/services/print_compositor/public/cpp",
       "//components/services/print_compositor/public/mojom",
       "//skia",
