diff --git a/components/permissions/android/android_permission_util.cc b/components/permissions/android/android_permission_util.cc
index 959ed6958c1c..6af5000203c9 100644
--- a/components/permissions/android/android_permission_util.cc
+++ b/components/permissions/android/android_permission_util.cc
@@ -6,7 +6,6 @@
 
 #include "base/android/jni_array.h"
 #include "components/permissions/android/jni_headers/PermissionUtil_jni.h"
-#include "components/permissions/permission_uma_util.h"
 #include "content/public/browser/web_contents.h"
 #include "ui/android/window_android.h"
 
