diff --git a/components/gwp_asan/client/BUILD.gn b/components/gwp_asan/client/BUILD.gn
index 17a315aeca04..4fb07cc77334 100644
--- a/components/gwp_asan/client/BUILD.gn
+++ b/components/gwp_asan/client/BUILD.gn
@@ -46,13 +46,8 @@ component("client") {
   deps = [
     "//base",
     "//base/allocator:buildflags",
-    "//components/crash/core/common:crash_key",
     "//components/gwp_asan/common",
   ]
-
-  if (is_android) {
-    deps += [ "//components/crash/core/app" ]
-  }
 }
 
 source_set("unit_tests") {
@@ -74,7 +69,6 @@ source_set("unit_tests") {
     ":client",
     "//base/allocator:buildflags",
     "//base/test:test_support",
-    "//components/crash/core/common:crash_key",
     "//components/gwp_asan/common",
     "//testing/gmock",
     "//testing/gtest",
