diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index c5c87536d757..1d0201d029a3 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -135,7 +135,6 @@ source_set("test_support") {
     "//base",
     "//components/content_settings/core/browser",
     "//components/sync_preferences:test_support",
-    "//components/ukm/content",
     "//components/vector_icons",
     "//content/public/browser",
     "//testing/gmock",
@@ -172,8 +171,6 @@ source_set("unit_tests") {
     "//components/content_settings/core/browser",
     "//components/keyed_service/content",
     "//components/prefs:test_support",
-    "//components/ukm:test_support",
-    "//components/ukm/content",
     "//components/variations",
     "//content/test:test_support",
     "//sql",
