diff --git a/components/safe_browsing/core/realtime/policy_engine.cc b/components/safe_browsing/core/realtime/policy_engine.cc
index ed86fc15f6ba..6f867bc5337c 100644
--- a/components/safe_browsing/core/realtime/policy_engine.cc
+++ b/components/safe_browsing/core/realtime/policy_engine.cc
@@ -9,7 +9,6 @@
 #include "base/stl_util.h"
 #include "build/build_config.h"
 #include "components/prefs/pref_service.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "components/safe_browsing/core/common/safebrowsing_constants.h"
 #include "components/safe_browsing/core/features.h"
 #include "components/signin/public/identity_manager/account_info.h"
@@ -80,7 +79,7 @@ bool RealTimePolicyEngine::IsUserMbbOptedIn(PrefService* pref_service) {
 
 // static
 bool RealTimePolicyEngine::IsUserEpOptedIn(PrefService* pref_service) {
-  return IsEnhancedProtectionEnabled(*pref_service);
+  return false;
 }
 
 // static
