diff --git a/components/viz/service/main/BUILD.gn b/components/viz/service/main/BUILD.gn
index 1526d19ae939..e7c0aec7497b 100644
--- a/components/viz/service/main/BUILD.gn
+++ b/components/viz/service/main/BUILD.gn
@@ -30,8 +30,6 @@ source_set("main") {
     "//gpu/ipc/service",
     "//ipc",
     "//mojo/public/cpp/system",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/mojom",
     "//services/service_manager/public/cpp",
     "//services/tracing/public/cpp",
     "//services/viz/privileged/mojom",
