diff --git a/components/lookalikes/core/lookalike_url_ui_util.h b/components/lookalikes/core/lookalike_url_ui_util.h
index c15ea9a31860..0683cb83c9ef 100644
--- a/components/lookalikes/core/lookalike_url_ui_util.h
+++ b/components/lookalikes/core/lookalike_url_ui_util.h
@@ -6,24 +6,11 @@
 #define COMPONENTS_LOOKALIKES_CORE_LOOKALIKE_URL_UI_UTIL_H_
 
 #include "components/lookalikes/core/lookalike_url_util.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 
 namespace base {
 class DictionaryValue;
 }  // namespace base
 
-// Allow easier reporting of UKM when no interstitial is shown.
-void RecordUkmForLookalikeUrlBlockingPage(
-    ukm::SourceId source_id,
-    LookalikeUrlMatchType match_type,
-    LookalikeUrlBlockingPageUserAction user_action);
-
-// Record UKM if not already reported for this page.
-void ReportUkmForLookalikeUrlBlockingPageIfNeeded(
-    ukm::SourceId& source_id,
-    LookalikeUrlMatchType match_type,
-    LookalikeUrlBlockingPageUserAction action);
-
 // Populates |load_time_data| for interstitial HTML.
 void PopulateLookalikeUrlBlockingPageStrings(
     base::DictionaryValue* load_time_data,
