diff --git a/components/javascript_dialogs/BUILD.gn b/components/javascript_dialogs/BUILD.gn
index 94034255947c..f682702c32eb 100644
--- a/components/javascript_dialogs/BUILD.gn
+++ b/components/javascript_dialogs/BUILD.gn
@@ -30,7 +30,6 @@ static_library("javascript_dialogs") {
     "//content/public/browser",
     "//content/public/common",
     "//net",
-    "//services/metrics/public/cpp:metrics_cpp",
     "//services/metrics/public/cpp:ukm_builders",
     "//skia",
     "//ui/base",
