diff --git a/components/gwp_asan/crash_handler/BUILD.gn b/components/gwp_asan/crash_handler/BUILD.gn
index c06efe9cf415..5208fbe70235 100644
--- a/components/gwp_asan/crash_handler/BUILD.gn
+++ b/components/gwp_asan/crash_handler/BUILD.gn
@@ -57,13 +57,5 @@ source_set("unit_tests") {
       "//third_party/crashpad/crashpad/minidump",
       "//third_party/crashpad/crashpad/tools:tool_support",
     ]
-
-    if (is_android) {
-      metadata = {
-        shared_libraries = [ "$root_out_dir/libchrome_crashpad_handler.so" ]
-      }
-      deps +=
-          [ "//components/crash/core/app:chrome_crashpad_handler_named_as_so" ]
-    }
   }
 }
