diff --git a/components/security_interstitials/content/insecure_form_blocking_page.cc b/components/security_interstitials/content/insecure_form_blocking_page.cc
index 241f33961c4ab..9153fdee925f5 100644
--- a/components/security_interstitials/content/insecure_form_blocking_page.cc
+++ b/components/security_interstitials/content/insecure_form_blocking_page.cc
@@ -94,12 +94,6 @@ void InsecureFormBlockingPage::PopulateInterstitialStrings(
   load_time_data->SetString(
       "primaryButtonText",
       l10n_util::GetStringUTF16(IDS_INSECURE_FORM_BACK_BUTTON));
-  load_time_data->SetString(
-      "optInLink",
-      l10n_util::GetStringUTF16(IDS_SAFE_BROWSING_SCOUT_REPORTING_AGREE));
-  load_time_data->SetString(
-      "enhancedProtectionMessage",
-      l10n_util::GetStringUTF16(IDS_SAFE_BROWSING_ENHANCED_PROTECTION_MESSAGE));
 }
 
 void InsecureFormBlockingPage::PopulateValuesForSharedHTML(
