diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index ea4ac9763945..56f866e62a67 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -33,8 +33,6 @@ source_set("permissions") {
     "permission_request_manager.h",
     "permission_result.cc",
     "permission_result.h",
-    "permission_uma_util.cc",
-    "permission_uma_util.h",
     "permission_util.cc",
     "permission_util.h",
     "permissions_client.cc",
@@ -53,12 +51,10 @@ source_set("permissions") {
     "//components/content_settings/core/browser",
     "//components/keyed_service/content",
     "//components/strings:components_strings_grit",
-    "//components/ukm/content",
     "//components/url_formatter",
     "//components/variations",
     "//components/vector_icons",
     "//content/public/browser",
-    "//services/metrics/public/cpp:ukm_builders",
     "//third_party/blink/public/common",
     "//ui/base",
     "//url",
@@ -109,7 +105,6 @@ source_set("test_support") {
     "//base",
     "//components/content_settings/core/browser",
     "//components/sync_preferences:test_support",
-    "//components/ukm/content",
     "//components/vector_icons",
     "//content/public/browser",
     "//testing/gmock",
@@ -143,8 +138,6 @@ source_set("unit_tests") {
     "//components/content_settings/core/browser",
     "//components/keyed_service/content",
     "//components/prefs:test_support",
-    "//components/ukm:test_support",
-    "//components/ukm/content",
     "//components/variations",
     "//content/test:test_support",
     "//testing/gtest",
