diff --git a/components/autofill/core/browser/BUILD.gn b/components/autofill/core/browser/BUILD.gn
index f47429dd1ac37..6af3ace1d37f9 100644
--- a/components/autofill/core/browser/BUILD.gn
+++ b/components/autofill/core/browser/BUILD.gn
@@ -444,7 +444,6 @@ static_library("browser") {
     "//net",
     "//services/data_decoder/public/cpp:cpp",
     "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/cpp",
     "//services/network/public/mojom",
     "//sql",
@@ -569,8 +568,6 @@ static_library("test_support") {
     "//components/sync:test_support",
     "//components/translate/core/browser",
     "//components/translate/core/browser:test_support",
-    "//components/ukm",
-    "//components/ukm:test_support",
     "//components/version_info:version_info",
     "//google_apis:test_support",
     "//services/network:test_support",
