diff --git a/components/exo/text_input.cc b/components/exo/text_input.cc
index f39285c8d0c0..099697139fbc 100644
--- a/components/exo/text_input.cc
+++ b/components/exo/text_input.cc
@@ -330,11 +330,6 @@ bool TextInput::IsTextEditCommandEnabled(ui::TextEditCommand command) const {
 void TextInput::SetTextEditCommandForNextKeyEvent(ui::TextEditCommand command) {
 }
 
-ukm::SourceId TextInput::GetClientSourceForMetrics() const {
-  NOTIMPLEMENTED_LOG_ONCE();
-  return ukm::kInvalidSourceId;
-}
-
 bool TextInput::ShouldDoLearning() {
   return should_do_learning_;
 }
