diff --git a/components/download/database/download_info.cc b/components/download/database/download_info.cc
index fec67b4048e30..e46fb78f3ebca 100644
--- a/components/download/database/download_info.cc
+++ b/components/download/database/download_info.cc
@@ -13,7 +13,7 @@ DownloadInfo::DownloadInfo(const DownloadInfo& other) = default;
 DownloadInfo::~DownloadInfo() = default;
 
 bool DownloadInfo::operator==(const DownloadInfo& other) const {
-  return guid == other.guid && id == other.id && ukm_info == other.ukm_info &&
+  return guid == other.guid && id == other.id &&
          in_progress_info == other.in_progress_info;
 }
 
