diff --git a/components/paint_preview/browser/BUILD.gn b/components/paint_preview/browser/BUILD.gn
index fe905fd1dd9f..798dab2ba8f6 100644
--- a/components/paint_preview/browser/BUILD.gn
+++ b/components/paint_preview/browser/BUILD.gn
@@ -36,12 +36,9 @@ source_set("browser") {
     "//components/discardable_memory/service",
     "//components/keyed_service/core",
     "//components/strings:components_strings_grit",
-    "//components/ukm/content",
     "//content/public/browser",
     "//mojo/public/cpp/base",
     "//mojo/public/cpp/bindings",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//third_party/blink/public:blink_headers",
     "//third_party/blink/public/common",
     "//third_party/zlib/google:zip",
