diff --git a/components/components_strings.grd b/components/components_strings.grd
index b453bbfd7ff4..d3bd25c112b0 100644
--- a/components/components_strings.grd
+++ b/components/components_strings.grd
@@ -314,7 +314,6 @@
       <part file="print_media_strings.grdp" />
       <part file="printing_component_strings.grdp" />
       <part file="reset_password_strings.grdp" />
-      <part file="safe_browsing_strings.grdp" />
       <part file="security_interstitials_strings.grdp" />
       <part file="security_state_strings.grdp" />
       <part file="send_tab_to_self_strings.grdp" />
@@ -327,6 +326,7 @@
       <part file="undo_strings.grdp" />
       <part file="version_ui_strings.grdp" />
       <part file="webapps_strings.grdp" />
+      <part file="dumb_version_ui_strings.grdp" />
 
       <if expr="not is_android and not is_ios">
         <part file="management_strings.grdp" />
