diff --git a/components/ntp_tiles/BUILD.gn b/components/ntp_tiles/BUILD.gn
index 6c9a0838a5a2..08c3a8045a76 100644
--- a/components/ntp_tiles/BUILD.gn
+++ b/components/ntp_tiles/BUILD.gn
@@ -60,7 +60,6 @@ static_library("ntp_tiles") {
     "//components/image_fetcher/core",
     "//components/pref_registry",
     "//components/prefs",
-    "//components/rappor/public",
     "//components/resources",
     "//components/search_engines",
     "//components/strings",
@@ -95,7 +94,6 @@ source_set("unit_tests") {
     "//components/image_fetcher/core",
     "//components/image_fetcher/core:test_support",
     "//components/pref_registry:pref_registry",
-    "//components/rappor:test_support",
     "//components/sync_preferences:test_support",
     "//net:test_support",
     "//services/data_decoder/public/cpp:test_support",
