diff --git a/components/password_manager/core/browser/BUILD.gn b/components/password_manager/core/browser/BUILD.gn
index d44ce6a08130..c5cb316b09df 100644
--- a/components/password_manager/core/browser/BUILD.gn
+++ b/components/password_manager/core/browser/BUILD.gn
@@ -166,12 +166,6 @@ static_library("browser") {
     "password_manager_util.h",
     "password_requirements_service.cc",
     "password_requirements_service.h",
-    "password_reuse_detection_manager.cc",
-    "password_reuse_detection_manager.h",
-    "password_reuse_detector.cc",
-    "password_reuse_detector.h",
-    "password_reuse_detector_consumer.cc",
-    "password_reuse_detector_consumer.h",
     "password_save_manager.h",
     "password_save_manager_impl.cc",
     "password_save_manager_impl.h",
@@ -282,9 +276,6 @@ static_library("browser") {
     "//components/password_manager/core/common",
     "//components/pref_registry",
     "//components/prefs",
-    "//components/profile_metrics",
-    "//components/safe_browsing/core:features",
-    "//components/safe_browsing/core/common:safe_browsing_prefs",
     "//components/security_state/core",
     "//components/signin/public/identity_manager",
     "//components/strings",
@@ -526,7 +517,6 @@ static_library("test_support") {
     "//components/safe_browsing:buildflags",
     "//components/signin/public/identity_manager:test_support",
     "//components/sync:test_support",
-    "//components/ukm",
     "//services/network/public/cpp",
     "//testing/gmock",
     "//testing/gtest",
@@ -725,8 +715,6 @@ source_set("unit_tests") {
     "//components/password_manager/core/browser/leak_detection:unit_tests",
     "//components/password_manager/core/common",
     "//components/prefs:test_support",
-    "//components/safe_browsing/core:features",
-    "//components/safe_browsing/core/common:safe_browsing_prefs",
     "//components/security_state/core",
     "//components/signin/public/identity_manager:test_support",
     "//components/strings",
