diff --git a/components/download/public/common/mock_download_file.h b/components/download/public/common/mock_download_file.h
index f68ccde80e000..9e777713cc739 100644
--- a/components/download/public/common/mock_download_file.h
+++ b/components/download/public/common/mock_download_file.h
@@ -52,7 +52,6 @@ class MockDownloadFile : public DownloadFile {
            const std::string& client_guid,
            const GURL& source_url,
            const GURL& referrer_url,
-           mojo::PendingRemote<quarantine::mojom::Quarantine> remote_quarantine,
            RenameCompletionCallback callback));
   MOCK_METHOD0(Detach, void());
   MOCK_METHOD0(Cancel, void());
