diff --git a/components/password_manager/core/browser/password_form_filling.h b/components/password_manager/core/browser/password_form_filling.h
index bf9da1a367f7..5e1d0cf49406 100644
--- a/components/password_manager/core/browser/password_form_filling.h
+++ b/components/password_manager/core/browser/password_form_filling.h
@@ -15,7 +15,6 @@ struct PasswordFormFillData;
 }  // namespace autofill
 
 namespace password_manager {
-class PasswordFormMetricsRecorder;
 class PasswordManagerClient;
 class PasswordManagerDriver;
 struct PasswordForm;
@@ -46,8 +45,7 @@ LikelyFormFilling SendFillInformationToRenderer(
     const std::vector<const PasswordForm*>& best_matches,
     const std::vector<const PasswordForm*>& federated_matches,
     const PasswordForm* preferred_match,
-    bool blocked_by_user,
-    PasswordFormMetricsRecorder* metrics_recorder);
+    bool blocked_by_user);
 
 // Create a PasswordFormFillData structure in preparation for filling a form
 // identified by |form_on_page|, with credentials from |preferred_match| and
