diff --git a/components/translate/content/browser/BUILD.gn b/components/translate/content/browser/BUILD.gn
index 770893c3dead..ba11ed98b39d 100644
--- a/components/translate/content/browser/BUILD.gn
+++ b/components/translate/content/browser/BUILD.gn
@@ -35,7 +35,6 @@ static_library("browser") {
     "//components/services/language_detection/public/cpp",
     "//components/services/language_detection/public/mojom",
     "//components/translate/core/language_detection:language_detection",
-    "//components/ukm/content",
     "//content/public/common",
     "//net",
   ]
