diff --git a/components/password_manager/core/browser/browser_save_password_progress_logger.cc b/components/password_manager/core/browser/browser_save_password_progress_logger.cc
index 5f1fa0d169de..93ad7d011a3c 100644
--- a/components/password_manager/core/browser/browser_save_password_progress_logger.cc
+++ b/components/password_manager/core/browser/browser_save_password_progress_logger.cc
@@ -91,7 +91,7 @@ std::string FormSignatureToDebugString(autofill::FormSignature form_signature) {
   return base::StrCat(
       {NumberToString(form_signature.value()), " - ",
        NumberToString(
-           PasswordFormMetricsRecorder::HashFormSignature(form_signature))});
+           static_cast<uint64_t>(form_signature) % 1021)});
 }
 
 BrowserSavePasswordProgressLogger::StringID FormSchemeToStringID(
