diff --git a/components/password_manager/core/browser/password_store_signin_notifier.cc b/components/password_manager/core/browser/password_store_signin_notifier.cc
index 80300be5a345..3b1e50b8e350 100644
--- a/components/password_manager/core/browser/password_store_signin_notifier.cc
+++ b/components/password_manager/core/browser/password_store_signin_notifier.cc
@@ -4,7 +4,6 @@
 
 #include "components/password_manager/core/browser/password_store_signin_notifier.h"
 
-#include "components/password_manager/core/browser/password_manager_metrics_util.h"
 #include "components/password_manager/core/browser/password_store.h"
 
 namespace password_manager {
@@ -19,14 +18,8 @@ void PasswordStoreSigninNotifier::NotifySignedOut(const std::string& username,
     return;
 
   if (primary_account) {
-    metrics_util::LogGaiaPasswordHashChange(
-        metrics_util::GaiaPasswordHashChange::CLEARED_ON_CHROME_SIGNOUT,
-        /*is_sync_password=*/true);
     store_->ClearAllGaiaPasswordHash();
   } else {
-    metrics_util::LogGaiaPasswordHashChange(
-        metrics_util::GaiaPasswordHashChange::CLEARED_ON_CHROME_SIGNOUT,
-        /*is_sync_password=*/false);
     store_->ClearGaiaPasswordHash(username);
   }
 }
