diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index 799755985b57..5c1ef55a5558 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -77,7 +77,6 @@ source_set("permissions") {
     "//components/permissions/prediction_service",
     "//components/permissions/vector_icons",
     "//components/strings:components_strings_grit",
-    "//components/ukm/content",
     "//components/url_formatter",
     "//components/variations",
     "//components/vector_icons",
@@ -136,7 +135,6 @@ source_set("test_support") {
     "//base",
     "//components/content_settings/core/browser",
     "//components/sync_preferences:test_support",
-    "//components/ukm/content",
     "//components/vector_icons",
     "//content/public/browser",
     "//testing/gmock",
@@ -173,8 +171,6 @@ source_set("unit_tests") {
     "//components/content_settings/core/browser",
     "//components/keyed_service/content",
     "//components/prefs:test_support",
-    "//components/ukm:test_support",
-    "//components/ukm/content",
     "//components/variations",
     "//content/test:test_support",
     "//sql",
