diff --git a/components/metrics/persistent_histograms.cc b/components/metrics/persistent_histograms.cc
index 202c461b10692..a31fa3b85bd4a 100644
--- a/components/metrics/persistent_histograms.cc
+++ b/components/metrics/persistent_histograms.cc
@@ -23,20 +23,6 @@
 
 namespace {
 
-// Creating a "spare" file for persistent metrics involves a lot of I/O and
-// isn't important so delay the operation for a while after startup.
-#if defined(OS_ANDROID)
-// Android needs the spare file and also launches faster.
-constexpr bool kSpareFileRequired = true;
-constexpr int kSpareFileCreateDelaySeconds = 10;
-#else
-// Desktop may have to restore a lot of tabs so give it more time before doing
-// non-essential work. The spare file is still a performance boost but not as
-// significant of one so it's not required.
-constexpr bool kSpareFileRequired = false;
-constexpr int kSpareFileCreateDelaySeconds = 90;
-#endif
-
 #if defined(OS_WIN)
 
 // Windows sometimes creates files of the form MyFile.pma~RF71cb1793.TMP
