diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index 43035485af85c..ad128e2c06d4c 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -232,7 +232,6 @@ static_library("browser") {
     "//components/omnibox/common",
     "//components/omnibox/resources:omnibox_resources",
     "//components/security_state/core",
-    "//third_party/metrics_proto",
     "//url",
   ]
   deps = [
@@ -312,7 +311,6 @@ static_library("location_bar") {
     "//base",
     "//components/omnibox/common",
     "//components/security_state/core",
-    "//third_party/metrics_proto",
     "//url",
   ]
   deps = [
@@ -471,7 +469,6 @@ static_library("test_support") {
     "//services/network:test_support",
     "//testing/gmock",
     "//testing/gtest",
-    "//third_party/metrics_proto",
     "//ui/base",
     "//ui/gfx",
   ]
@@ -604,7 +601,6 @@ fuzzer_test("autocomplete_input_fuzzer") {
     ":test_support",
     "//base",
     "//base:i18n",
-    "//third_party/metrics_proto:metrics_proto",
   ]
 }
 
@@ -624,7 +620,6 @@ fuzzer_test("search_suggestion_parser_fuzzer") {
     "//base",
     "//base:i18n",
     "//third_party/icu:icui18n",
-    "//third_party/metrics_proto:metrics_proto",
   ]
   seed_corpus = "fuzz_corpus_suggest"
 }
