diff --git a/components/permissions/permissions_client.h b/components/permissions/permissions_client.h
index c8e8c1f94a56..1418aeafbf7c 100644
--- a/components/permissions/permissions_client.h
+++ b/components/permissions/permissions_client.h
@@ -13,7 +13,6 @@
 #include "components/permissions/notification_permission_ui_selector.h"
 #include "components/permissions/permission_prompt.h"
 #include "components/permissions/permission_util.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "url/origin.h"
 
 class GURL;
@@ -105,17 +104,6 @@ class PermissionsClient {
       const GURL& origin);
 #endif
 
-  // Retrieves the ukm::SourceId (if any) associated with this |browser_context|
-  // and |web_contents|. |web_contents| may be null. |callback| will be called
-  // with the result, and may be run synchronously if the result is available
-  // immediately.
-  using GetUkmSourceIdCallback =
-      base::OnceCallback<void(base::Optional<ukm::SourceId>)>;
-  virtual void GetUkmSourceId(content::BrowserContext* browser_context,
-                              const content::WebContents* web_contents,
-                              const GURL& requesting_origin,
-                              GetUkmSourceIdCallback callback);
-
   // Returns the icon ID that should be used for permissions UI for |type|. If
   // the embedder returns an empty IconId, the default icon for |type| will be
   // used.
