diff --git a/components/signin/public/identity_manager/identity_manager.cc b/components/signin/public/identity_manager/identity_manager.cc
index b39d993c80e7..1b6c7ffeb46f 100644
--- a/components/signin/public/identity_manager/identity_manager.cc
+++ b/components/signin/public/identity_manager/identity_manager.cc
@@ -101,6 +101,7 @@ IdentityManager::~IdentityManager() {
     Java_IdentityManager_destroy(base::android::AttachCurrentThread(),
                                  java_identity_manager_);
 #endif
+  observer_list_.Clear();
 }
 
 void IdentityManager::AddObserver(Observer* observer) {
