diff --git a/components/resources/BUILD.gn b/components/resources/BUILD.gn
index a36546c69781..30df821da9a8 100644
--- a/components/resources/BUILD.gn
+++ b/components/resources/BUILD.gn
@@ -108,7 +108,6 @@ python2_action("about_credits") {
 if (is_android) {
   source_set("android_resources") {
     sources = [
-      "android/blocked_content_resource_id.h",
       "android/page_info_resource_id.h",
       "android/permissions_resource_id.h",
       "android/sms_resource_id.h",
