diff --git a/components/page_info/BUILD.gn b/components/page_info/BUILD.gn
index c897a2e35d92..07e9ab85f088 100644
--- a/components/page_info/BUILD.gn
+++ b/components/page_info/BUILD.gn
@@ -34,8 +34,6 @@ static_library("page_info") {
     "//components/ssl_errors:ssl_errors",
     "//components/strings:components_chromium_strings_grit",
     "//components/strings:components_strings_grit",
-    "//components/subresource_filter/core/browser",
-    "//components/ukm/content",
     "//components/url_formatter",
     "//components/vector_icons:vector_icons",
     "//content/public/browser:browser",
