diff --git a/components/optimization_guide/core/BUILD.gn b/components/optimization_guide/core/BUILD.gn
index 876bd32341866..31f517e430c3a 100644
--- a/components/optimization_guide/core/BUILD.gn
+++ b/components/optimization_guide/core/BUILD.gn
@@ -72,11 +72,9 @@ static_library("core") {
 
   deps = [
     "//base",
-    "//components/data_reduction_proxy/core/browser",
     "//components/leveldb_proto",
     "//components/optimization_guide/proto:optimization_guide_proto",
     "//components/prefs",
-    "//components/unified_consent",
     "//components/variations",
     "//components/variations/net",
     "//google_apis",
