diff --git a/components/autofill/core/browser/payments/autofill_credit_card_filling_infobar_delegate_mobile.cc b/components/autofill/core/browser/payments/autofill_credit_card_filling_infobar_delegate_mobile.cc
index 7a77ca039f30..df9a75a81533 100644
--- a/components/autofill/core/browser/payments/autofill_credit_card_filling_infobar_delegate_mobile.cc
+++ b/components/autofill/core/browser/payments/autofill_credit_card_filling_infobar_delegate_mobile.cc
@@ -34,12 +34,6 @@ AutofillCreditCardFillingInfoBarDelegateMobile::
 
 AutofillCreditCardFillingInfoBarDelegateMobile::
     ~AutofillCreditCardFillingInfoBarDelegateMobile() {
-  if (was_shown_) {
-    AutofillMetrics::LogCreditCardFillingInfoBarMetric(
-        AutofillMetrics::INFOBAR_SHOWN);
-    if (!had_user_interaction_)
-      LogUserAction(AutofillMetrics::INFOBAR_IGNORED);
-  }
 }
 
 int AutofillCreditCardFillingInfoBarDelegateMobile::GetIconId() const {
@@ -59,17 +53,14 @@ base::string16 AutofillCreditCardFillingInfoBarDelegateMobile::GetMessageText()
 }
 
 void AutofillCreditCardFillingInfoBarDelegateMobile::InfoBarDismissed() {
-  LogUserAction(AutofillMetrics::INFOBAR_DENIED);
 }
 
 bool AutofillCreditCardFillingInfoBarDelegateMobile::Accept() {
   std::move(card_filling_callback_).Run();
-  LogUserAction(AutofillMetrics::INFOBAR_ACCEPTED);
   return true;
 }
 
 bool AutofillCreditCardFillingInfoBarDelegateMobile::Cancel() {
-  LogUserAction(AutofillMetrics::INFOBAR_DENIED);
   return true;
 }
 
@@ -85,12 +76,4 @@ base::string16 AutofillCreditCardFillingInfoBarDelegateMobile::GetButtonLabel(
                           : IDS_AUTOFILL_CREDIT_CARD_FILLING_INFOBAR_NO_THANKS);
 }
 
-void AutofillCreditCardFillingInfoBarDelegateMobile::LogUserAction(
-    AutofillMetrics::InfoBarMetric user_action) {
-  DCHECK(!had_user_interaction_);
-
-  AutofillMetrics::LogCreditCardFillingInfoBarMetric(user_action);
-  had_user_interaction_ = true;
-}
-
 }  // namespace autofill
