diff --git a/components/crash/core/app/BUILD.gn b/components/crash/core/app/BUILD.gn
index 93c330b46525..290fa1776570 100644
--- a/components/crash/core/app/BUILD.gn
+++ b/components/crash/core/app/BUILD.gn
@@ -162,7 +162,6 @@ if (is_win) {
 
     deps = [
       "//base",
-      "//components/browser_watcher:activity_report",
       "//components/gwp_asan/buildflags",
       "//third_party/crashpad/crashpad/client",
       "//third_party/crashpad/crashpad/handler",
