diff --git a/components/performance_manager/public/graph/graph.h b/components/performance_manager/public/graph/graph.h
index 4e45c4e16e1e..cc97f7ad9cde 100644
--- a/components/performance_manager/public/graph/graph.h
+++ b/components/performance_manager/public/graph/graph.h
@@ -13,10 +13,6 @@
 #include "base/macros.h"
 #include "base/memory/ptr_util.h"
 
-namespace ukm {
-class UkmRecorder;
-}  // namespace ukm
-
 namespace performance_manager {
 
 class GraphObserver;
@@ -110,9 +106,6 @@ class Graph {
   // Returns true if the graph is currently empty.
   virtual bool IsEmpty() const = 0;
 
-  // Returns the associated UKM recorder if it is defined.
-  virtual ukm::UkmRecorder* GetUkmRecorder() const = 0;
-
   // Returns the data describer registry.
   virtual NodeDataDescriberRegistry* GetNodeDataDescriberRegistry() const = 0;
 
