diff --git a/components/components_strings.grd b/components/components_strings.grd
index 346579eb41400..172c240e608e8 100644
--- a/components/components_strings.grd
+++ b/components/components_strings.grd
@@ -320,7 +320,6 @@
       <part file="printing_component_strings.grdp" />
       <part file="privacy_sandbox_strings.grdp" />
       <part file="reset_password_strings.grdp" />
-      <part file="safe_browsing_strings.grdp" />
       <part file="security_interstitials_strings.grdp" />
       <part file="security_state_strings.grdp" />
       <part file="send_tab_to_self_strings.grdp" />
@@ -334,6 +333,7 @@
       <part file="undo_strings.grdp" />
       <part file="version_ui_strings.grdp" />
       <part file="webapps_strings.grdp" />
+      <part file="dumb_version_ui_strings.grdp" />
 
       <if expr="not is_android and not is_ios">
         <part file="management_strings.grdp" />
