diff --git a/components/password_manager/core/browser/password_form_manager.h b/components/password_manager/core/browser/password_form_manager.h
index dceeaa93c851..42cb1080f243 100644
--- a/components/password_manager/core/browser/password_form_manager.h
+++ b/components/password_manager/core/browser/password_form_manager.h
@@ -24,7 +24,6 @@
 #include "components/password_manager/core/browser/form_parsing/form_parser.h"
 #include "components/password_manager/core/browser/form_parsing/password_field_prediction.h"
 #include "components/password_manager/core/browser/password_form_manager_for_ui.h"
-#include "components/password_manager/core/browser/password_form_metrics_recorder.h"
 #include "components/password_manager/core/browser/password_save_manager.h"
 #include "components/password_manager/core/browser/password_store.h"
 #include "components/password_manager/core/browser/votes_uploader.h"
@@ -32,7 +31,6 @@
 
 namespace password_manager {
 
-class PasswordFormMetricsRecorder;
 class PasswordManagerClient;
 class PasswordManagerDriver;
 struct PossibleUsernameData;
@@ -53,14 +51,6 @@ class PasswordFormManager : public PasswordFormManagerForUI,
       const base::WeakPtr<PasswordManagerDriver>& driver,
       const autofill::FormData& observed_form_data,
       FormFetcher* form_fetcher,
-      std::unique_ptr<PasswordSaveManager> password_save_manager,
-      scoped_refptr<PasswordFormMetricsRecorder> metrics_recorder);
-
-  // Constructor for http authentication (aka basic authentication).
-  PasswordFormManager(
-      PasswordManagerClient* client,
-      PasswordStore::FormDigest observed_http_auth_digest,
-      FormFetcher* form_fetcher,
       std::unique_ptr<PasswordSaveManager> password_save_manager);
 
   ~PasswordFormManager() override;
@@ -152,7 +142,6 @@ class PasswordFormManager : public PasswordFormManagerForUI,
   std::vector<const PasswordForm*> GetFederatedMatches() const override;
   const PasswordForm& GetPendingCredentials() const override;
   metrics_util::CredentialSourceType GetCredentialSource() const override;
-  PasswordFormMetricsRecorder* GetMetricsRecorder() override;
   base::span<const InteractionsStats> GetInteractionsStats() const override;
   base::span<const CompromisedCredentials> GetCompromisedCredentials()
       const override;
@@ -263,13 +252,15 @@ class PasswordFormManager : public PasswordFormManagerForUI,
   using FormOrDigest =
       absl::variant<autofill::FormData, PasswordStore::FormDigest>;
 
+public:
   // Delegating constructor.
   PasswordFormManager(
       PasswordManagerClient* client,
       FormOrDigest observed_form_or_digest,
       FormFetcher* form_fetcher,
-      std::unique_ptr<PasswordSaveManager> password_save_manager,
-      scoped_refptr<PasswordFormMetricsRecorder> metrics_recorder);
+      std::unique_ptr<PasswordSaveManager> password_save_manager);
+
+private:
   // Records the status of readonly fields during parsing, combined with the
   // overall success of the parsing. It reports through two different metrics,
   // depending on whether |mode| indicates parsing for saving or filling.
@@ -357,9 +348,6 @@ class PasswordFormManager : public PasswordFormManagerForUI,
   // |PasswordManager.ResultOfSavingAfterUnblacklisting|.
   bool was_unblacklisted_while_on_page_ = false;
 
-  // Takes care of recording metrics and events for |*this|.
-  scoped_refptr<PasswordFormMetricsRecorder> metrics_recorder_;
-
   // When not null, then this is the object which |form_fetcher_| points to.
   std::unique_ptr<FormFetcher> owned_form_fetcher_;
 
