diff --git a/components/password_manager/core/browser/BUILD.gn b/components/password_manager/core/browser/BUILD.gn
index 930ff903a00f..55d713331b20 100644
--- a/components/password_manager/core/browser/BUILD.gn
+++ b/components/password_manager/core/browser/BUILD.gn
@@ -11,9 +11,9 @@ if (is_android) {
 
 # TODO(crbug.com/1006430): Fix password reuse detection not fully functional on
 # Android.
-password_reuse_detection_support = !is_ios
-password_reuse_warning_support = !is_ios
-password_on_focus_ping_support = !is_ios
+password_reuse_detection_support = false
+password_reuse_warning_support = false
+password_on_focus_ping_support = false
 
 config("password_reuse_detection_config") {
   defines = []
@@ -149,8 +149,6 @@ static_library("browser") {
     "password_form_manager.cc",
     "password_form_manager.h",
     "password_form_manager_for_ui.h",
-    "password_form_metrics_recorder.cc",
-    "password_form_metrics_recorder.h",
     "password_generation_frame_helper.cc",
     "password_generation_frame_helper.h",
     "password_generation_manager.cc",
@@ -169,20 +167,12 @@ static_library("browser") {
     "password_manager_features_util.cc",
     "password_manager_features_util.h",
     "password_manager_interface.h",
-    "password_manager_metrics_recorder.cc",
-    "password_manager_metrics_recorder.h",
     "password_manager_metrics_util.cc",
     "password_manager_metrics_util.h",
     "password_manager_util.cc",
     "password_manager_util.h",
     "password_requirements_service.cc",
     "password_requirements_service.h",
-    "password_reuse_detection_manager.cc",
-    "password_reuse_detection_manager.h",
-    "password_reuse_detector.cc",
-    "password_reuse_detector.h",
-    "password_reuse_detector_consumer.cc",
-    "password_reuse_detector_consumer.h",
     "password_save_manager.h",
     "password_save_manager_impl.cc",
     "password_save_manager_impl.h",
@@ -292,8 +282,6 @@ static_library("browser") {
     "//components/password_manager/core/common",
     "//components/pref_registry",
     "//components/prefs",
-    "//components/safe_browsing/core:features",
-    "//components/safe_browsing/core/common:safe_browsing_prefs",
     "//components/security_state/core",
     "//components/signin/public/identity_manager",
     "//components/strings",
@@ -304,8 +292,6 @@ static_library("browser") {
     "//components/webdata/common",
     "//google_apis",
     "//net",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/cpp",
     "//sql",
     "//third_party/abseil-cpp:absl",
@@ -528,7 +514,6 @@ static_library("test_support") {
     "//components/safe_browsing:buildflags",
     "//components/signin/public/identity_manager:test_support",
     "//components/sync:test_support",
-    "//components/ukm",
     "//services/network/public/cpp",
     "//testing/gmock",
     "//testing/gtest",
@@ -637,7 +622,6 @@ source_set("unit_tests") {
     "password_feature_manager_impl_unittest.cc",
     "password_form_filling_unittest.cc",
     "password_form_manager_unittest.cc",
-    "password_form_metrics_recorder_unittest.cc",
     "password_generation_frame_helper_unittest.cc",
     "password_generation_manager_unittest.cc",
     "password_hash_data_unittest.cc",
@@ -721,18 +705,14 @@ source_set("unit_tests") {
     "//components/password_manager/core/browser/leak_detection:unit_tests",
     "//components/password_manager/core/common",
     "//components/prefs:test_support",
-    "//components/safe_browsing/core:features",
-    "//components/safe_browsing/core/common:safe_browsing_prefs",
     "//components/security_state/core",
     "//components/signin/public/identity_manager:test_support",
     "//components/strings",
     "//components/sync:test_support",
     "//components/sync_preferences:test_support",
-    "//components/ukm:test_support",
     "//components/url_formatter",
     "//google_apis:google_apis",
     "//net:test_support",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network:network_service",
     "//services/network:test_support",
     "//services/network/public/cpp",
