diff --git a/components/autofill/core/browser/autofill_client.h b/components/autofill/core/browser/autofill_client.h
index d8bb5ce5779a..6a49bb4dd762 100644
--- a/components/autofill/core/browser/autofill_client.h
+++ b/components/autofill/core/browser/autofill_client.h
@@ -24,7 +24,6 @@
 #include "components/autofill/core/browser/ui/popup_types.h"
 #include "components/security_state/core/security_state.h"
 #include "components/translate/core/browser/language_state.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "ui/base/window_open_disposition.h"
 #include "ui/gfx/geometry/rect_f.h"
 #include "url/gurl.h"
@@ -47,10 +46,6 @@ namespace syncer {
 class SyncService;
 }
 
-namespace ukm {
-class UkmRecorder;
-}
-
 namespace version_info {
 enum class Channel;
 }
@@ -267,12 +262,6 @@ class AutofillClient : public RiskDataLoader {
   // Gets the StrikeDatabase associated with the client.
   virtual StrikeDatabase* GetStrikeDatabase() = 0;
 
-  // Gets the UKM service associated with this client (for metrics).
-  virtual ukm::UkmRecorder* GetUkmRecorder() = 0;
-
-  // Gets the UKM source id associated with this client (for metrics).
-  virtual ukm::SourceId GetUkmSourceId() = 0;
-
   // Gets an AddressNormalizer instance (can be null).
   virtual AddressNormalizer* GetAddressNormalizer() = 0;
 
