diff --git a/components/shared_highlighting/core/common/BUILD.gn b/components/shared_highlighting/core/common/BUILD.gn
index 39e5b8a38582..848007226952 100644
--- a/components/shared_highlighting/core/common/BUILD.gn
+++ b/components/shared_highlighting/core/common/BUILD.gn
@@ -4,8 +4,6 @@
 
 static_library("common") {
   sources = [
-    "shared_highlighting_metrics.cc",
-    "shared_highlighting_metrics.h",
     "text_fragment.cc",
     "text_fragment.h",
     "text_fragments_constants.cc",
@@ -17,8 +15,6 @@ static_library("common") {
   deps = [
     "//base",
     "//components/search_engines:search_engine_utils",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
   ]
 }
 
@@ -32,9 +28,6 @@ source_set("unit_tests") {
   deps = [
     "//base/test:test_support",
     "//components/shared_highlighting/core/common",
-    "//components/ukm:test_support",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//testing/gtest",
     "//url",
   ]
