diff --git a/components/permissions/permission_request_manager.cc b/components/permissions/permission_request_manager.cc
index fd412be2fd8c..254fb3622c5c 100644
--- a/components/permissions/permission_request_manager.cc
+++ b/components/permissions/permission_request_manager.cc
@@ -503,8 +503,6 @@ void PermissionRequestManager::ShowBubble() {
     return;
 
   if (!current_request_already_displayed_) {
-    PermissionUmaUtil::PermissionPromptShown(requests_);
-
     if (ShouldCurrentRequestUseQuietUI()) {
       switch (ReasonForUsingQuietUi()) {
         case QuietUiReason::kEnabledInPrefs:
@@ -553,10 +551,6 @@ void PermissionRequestManager::FinalizeBubble(
     PermissionAction permission_action) {
   DCHECK(IsRequestInProgress());
 
-  PermissionUmaUtil::PermissionPromptResolved(
-      requests_, web_contents(), permission_action,
-      DetermineCurrentRequestUIDispositionForUMA());
-
   content::BrowserContext* browser_context =
       web_contents()->GetBrowserContext();
   PermissionDecisionAutoBlocker* autoblocker =
@@ -593,7 +587,6 @@ void PermissionRequestManager::FinalizeBubble(
         embargo_status = PermissionEmbargoStatus::REPEATED_IGNORES;
       }
     }
-    PermissionUmaUtil::RecordEmbargoStatus(embargo_status);
   }
   std::vector<PermissionRequest*>::iterator requests_iter;
   for (requests_iter = requests_.begin(); requests_iter != requests_.end();
