diff --git a/components/assist_ranker/BUILD.gn b/components/assist_ranker/BUILD.gn
index dd11864c26a4..e74c2ce9921f 100644
--- a/components/assist_ranker/BUILD.gn
+++ b/components/assist_ranker/BUILD.gn
@@ -43,7 +43,6 @@ static_library("assist_ranker") {
     "//components/assist_ranker/proto",
     "//components/keyed_service/core",
     "//net",
-    "//services/metrics/public/cpp:metrics_cpp",
     "//services/network:network_service",
     "//services/network/public/cpp",
     "//services/network/public/mojom",
