diff --git a/components/no_state_prefetch/browser/BUILD.gn b/components/no_state_prefetch/browser/BUILD.gn
index 88010bdfbe85..b3aec0b14579 100644
--- a/components/no_state_prefetch/browser/BUILD.gn
+++ b/components/no_state_prefetch/browser/BUILD.gn
@@ -38,7 +38,6 @@ static_library("browser") {
     "//components/no_state_prefetch/common:mojo_bindings",
     "//content/public/browser",
     "//net",
-    "//services/metrics/public/cpp:ukm_builders",
     "//ui/gfx/geometry",
     "//ui/gfx/image/mojom:mojom_traits",
     "//url",
