diff --git a/components/performance_manager/graph/page_node_impl_describer.cc b/components/performance_manager/graph/page_node_impl_describer.cc
index 6032b9688476..21d313e9ff8b 100644
--- a/components/performance_manager/graph/page_node_impl_describer.cc
+++ b/components/performance_manager/graph/page_node_impl_describer.cc
@@ -61,9 +61,6 @@ base::Value PageNodeImplDescriber::DescribePageNodeData(
   result.SetBoolKey("is_visible", page_node_impl->is_visible_.value());
   result.SetBoolKey("is_audible", page_node_impl->is_audible_.value());
   result.SetBoolKey("is_loading", page_node_impl->is_loading_.value());
-  result.SetStringKey(
-      "ukm_source_id",
-      base::NumberToString(page_node_impl->ukm_source_id_.value()));
   result.SetStringKey(
       "lifecycle_state",
       MojoEnumToString(page_node_impl->lifecycle_state_.value()));
