diff --git a/components/arc/ime/arc_ime_service.cc b/components/arc/ime/arc_ime_service.cc
index ead34dd74752..0b7409283d8c 100644
--- a/components/arc/ime/arc_ime_service.cc
+++ b/components/arc/ime/arc_ime_service.cc
@@ -610,12 +610,6 @@ bool ArcImeService::IsTextEditCommandEnabled(
   return false;
 }
 
-ukm::SourceId ArcImeService::GetClientSourceForMetrics() const {
-  // TODO(yhanada): Implement this method. crbug.com/752657
-  NOTIMPLEMENTED_LOG_ONCE();
-  return ukm::SourceId();
-}
-
 bool ArcImeService::ShouldDoLearning() {
   return is_personalized_learning_allowed_;
 }
