diff --git a/components/subresource_filter/content/browser/subresource_filter_observer_manager.cc b/components/subresource_filter/content/browser/subresource_filter_observer_manager.cc
index bd7a16fa87aa..10685a98cf02 100644
--- a/components/subresource_filter/content/browser/subresource_filter_observer_manager.cc
+++ b/components/subresource_filter/content/browser/subresource_filter_observer_manager.cc
@@ -27,10 +27,9 @@ void SubresourceFilterObserverManager::RemoveObserver(
 }
 
 void SubresourceFilterObserverManager::NotifySafeBrowsingChecksComplete(
-    content::NavigationHandle* navigation_handle,
-    const SubresourceFilterSafeBrowsingClient::CheckResult& result) {
+    content::NavigationHandle* navigation_handle) {
   for (auto& observer : observers_) {
-    observer.OnSafeBrowsingChecksComplete(navigation_handle, result);
+    observer.OnSafeBrowsingChecksComplete(navigation_handle);
   }
 }
 
