diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index cbc77b40788e..711cb7ab90c9 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -85,8 +85,6 @@ static_library("metrics") {
     "metrics_switches.h",
     "metrics_upload_scheduler.cc",
     "metrics_upload_scheduler.h",
-    "persistent_histograms.cc",
-    "persistent_histograms.h",
     "persistent_system_profile.cc",
     "persistent_system_profile.h",
     "reporting_service.cc",
@@ -96,7 +94,6 @@ static_library("metrics") {
     "stability_metrics_provider.cc",
     "stability_metrics_provider.h",
     "system_memory_stats_recorder.h",
-    "ukm_demographic_metrics_provider.h",
     "unsent_log_store.cc",
     "unsent_log_store.h",
     "unsent_log_store_metrics.cc",
@@ -164,7 +161,6 @@ static_library("metrics") {
       "system_session_analyzer/system_session_analyzer_win.cc",
       "system_session_analyzer/system_session_analyzer_win.h",
     ]
-    deps += [ "//components/browser_watcher:stability_client" ]
     libs = [ "wevtapi.lib" ]
   } else {
     sources += [ "machine_id_provider_nonwin.cc" ]
@@ -229,9 +225,6 @@ if (!is_ios) {
       "//extensions/buildflags",
       "//gpu/config",
     ]
-    if (is_android) {
-      deps += [ "//components/crash/content/browser" ]
-    }
   }
 }
 
