diff --git a/components/permissions/permission_request_manager.cc b/components/permissions/permission_request_manager.cc
index c930f2f499c9..c4f1ee96666c 100644
--- a/components/permissions/permission_request_manager.cc
+++ b/components/permissions/permission_request_manager.cc
@@ -556,8 +556,6 @@ void PermissionRequestManager::ShowBubble() {
     return;
 
   if (!current_request_already_displayed_) {
-    PermissionUmaUtil::PermissionPromptShown(requests_);
-
     if (ShouldCurrentRequestUseQuietUI()) {
       switch (ReasonForUsingQuietUi()) {
         case QuietUiReason::kEnabledInPrefs:
@@ -608,11 +606,6 @@ void PermissionRequestManager::ResetViewStateForCurrentRequest() {
 void PermissionRequestManager::FinalizeCurrentRequests(
     PermissionAction permission_action) {
   DCHECK(IsRequestInProgress());
-  PermissionUmaUtil::PermissionPromptResolved(
-      requests_, web_contents(), permission_action,
-      DetermineCurrentRequestUIDispositionForUMA(),
-      DetermineCurrentRequestUIDispositionReasonForUMA(),
-      prediction_grant_likelihood_);
 
   content::BrowserContext* browser_context =
       web_contents()->GetBrowserContext();
@@ -650,7 +643,6 @@ void PermissionRequestManager::FinalizeCurrentRequests(
         embargo_status = PermissionEmbargoStatus::REPEATED_IGNORES;
       }
     }
-    PermissionUmaUtil::RecordEmbargoStatus(embargo_status);
   }
   ResetViewStateForCurrentRequest();
   std::vector<PermissionRequest*>::iterator requests_iter;
