diff --git a/components/password_manager/core/browser/password_form_manager.cc b/components/password_manager/core/browser/password_form_manager.cc
index 325b6fb640a4..6fb60391f081 100644
--- a/components/password_manager/core/browser/password_form_manager.cc
+++ b/components/password_manager/core/browser/password_form_manager.cc
@@ -620,7 +620,6 @@ std::unique_ptr<PasswordFormManager> PasswordFormManager::Clone() {
   }
   result->is_submitted_ = is_submitted_;
   result->password_save_manager_->Init(result->client_, result->form_fetcher_,
-                                       result->metrics_recorder_,
                                        &result->votes_uploader_);
   return result;
 }
@@ -899,7 +898,7 @@ PasswordFormManager::PasswordFormManager(
         client_->IsCommittedMainFrameSecure(), client_->GetUkmSourceId(),
         client_->GetPrefs());
   }
-  password_save_manager_->Init(client_, form_fetcher_, metrics_recorder_,
+  password_save_manager_->Init(client_, form_fetcher_,
                                &votes_uploader_);
 }
 
