diff --git a/components/autofill_assistant/browser/BUILD.gn b/components/autofill_assistant/browser/BUILD.gn
index 3f994f2b8d70..8ff0fb032a04 100644
--- a/components/autofill_assistant/browser/BUILD.gn
+++ b/components/autofill_assistant/browser/BUILD.gn
@@ -267,13 +267,10 @@ static_library("browser") {
     "//components/password_manager/core/browser/form_parsing:form_parsing",
     "//components/signin/public/identity_manager",
     "//components/strings:components_strings_grit",
-    "//components/ukm/content:content",
     "//components/version_info",
     "//content/public/browser",
     "//google_apis",
     "//net",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//third_party/re2",
   ]
 }
@@ -419,8 +416,6 @@ source_set("unit_tests") {
     "//components/password_manager/core/browser:test_support",
     "//components/password_manager/core/common",
     "//components/strings:components_strings_grit",
-    "//components/ukm:test_support",
-    "//components/ukm/content:content",
     "//components/version_info",
     "//content/test:test_support",
     "//services/network:test_support",
