diff --git a/components/viz/service/display/display.cc b/components/viz/service/display/display.cc
index 94877489b132..da3626afb0e4 100644
--- a/components/viz/service/display/display.cc
+++ b/components/viz/service/display/display.cc
@@ -919,27 +919,6 @@ void Display::DidReceiveSwapBuffersAck(const gfx::SwapTimings& timings) {
 
   // We should have at least one group that hasn't received a SwapBuffersAck
   DCHECK(!draw_start_timestamp.is_null());
-
-  // Check that the swap timings correspond with the timestamp from when
-  // the swap was triggered. Note that not all output surfaces provide timing
-  // information, hence the check for a valid swap_start.
-  if (!timings.swap_start.is_null()) {
-    DCHECK_LE(draw_start_timestamp, timings.swap_start);
-    base::TimeDelta delta = timings.swap_start - draw_start_timestamp;
-    UMA_HISTOGRAM_CUSTOM_MICROSECONDS_TIMES(
-        "Compositing.Display.DrawToSwapUs", delta, kDrawToSwapMin,
-        kDrawToSwapMax, kDrawToSwapUsBuckets);
-  }
-
-  if (!timings.viz_scheduled_draw.is_null()) {
-    DCHECK(!timings.gpu_started_draw.is_null());
-    DCHECK_LE(timings.viz_scheduled_draw, timings.gpu_started_draw);
-    base::TimeDelta delta =
-        timings.gpu_started_draw - timings.viz_scheduled_draw;
-    UMA_HISTOGRAM_CUSTOM_MICROSECONDS_TIMES(
-        "Compositing.Display.VizScheduledDrawToGpuStartedDrawUs", delta,
-        kDrawToSwapMin, kDrawToSwapMax, kDrawToSwapUsBuckets);
-  }
 }
 
 void Display::DidReceiveTextureInUseResponses(
