diff --git a/components/BUILD.gn b/components/BUILD.gn
index abf17f45f5d9..7343546fbde7 100644
--- a/components/BUILD.gn
+++ b/components/BUILD.gn
@@ -80,7 +80,6 @@ test("components_unittests") {
     "//components/content_capture/common:unit_tests",
     "//components/content_settings/core/browser:unit_tests",
     "//components/content_settings/core/common:unit_tests",
-    "//components/crash/core/common:unit_tests",
     "//components/crx_file:unit_tests",
     "//components/device_event_log:unit_tests",
     "//components/dom_distiller/core:unit_tests",
@@ -125,7 +124,6 @@ test("components_unittests") {
     "//components/policy/core/common:unit_tests",
     "//components/power_scheduler:unit_tests",
     "//components/prefs:unit_tests",
-    "//components/previews/core:unit_tests",
     "//components/proxy_config:unit_tests",
     "//components/qr_code_generator:unit_tests",
     "//components/query_parser:unit_tests",
@@ -168,7 +166,6 @@ test("components_unittests") {
     "//components/translate/core/browser:unit_tests",
     "//components/translate/core/common:unit_tests",
     "//components/translate/core/language_detection:unit_tests",
-    "//components/ukm:unit_tests",
     "//components/undo:unit_tests",
     "//components/unified_consent:unit_tests",
     "//components/upload_list:unit_tests",
@@ -238,7 +235,6 @@ test("components_unittests") {
       "//components/autofill/content/renderer:unit_tests",
       "//components/autofill/core/common/mojom:unit_tests",
       "//components/background_sync:unit_tests",
-      "//components/blocked_content:unit_tests",
       "//components/browsing_data/content:unit_tests",
       "//components/captive_portal/content:unit_tests",
       "//components/cast_certificate:unit_tests",
@@ -257,7 +253,6 @@ test("components_unittests") {
       "//components/drive:unit_tests",
       "//components/embedder_support:unit_tests",
       "//components/favicon/content:unit_tests",
-      "//components/federated_learning:unit_tests",
       "//components/feed:unit_tests",
       "//components/gcm_driver/instance_id:unit_tests",
       "//components/history/content/browser:unit_tests",
@@ -278,8 +273,6 @@ test("components_unittests") {
       "//components/optimization_guide/core:unit_tests",
       "//components/page_image_annotation/content/renderer:unit_tests",
       "//components/page_image_annotation/core:unit_tests",
-      "//components/page_load_metrics/browser:unit_tests",
-      "//components/page_load_metrics/renderer:unit_tests",
       "//components/paint_preview/browser:unit_tests",
       "//components/paint_preview/common:unit_tests",
       "//components/paint_preview/player:unit_tests",
@@ -321,7 +314,6 @@ test("components_unittests") {
       "//components/tracing:unit_tests",
       "//components/translate/content/browser:unit_tests",
       "//components/translate/content/renderer:unit_tests",
-      "//components/ukm/content:unit_tests",
       "//components/visitedlink/test:unit_tests",
       "//components/web_cache/browser:unit_tests",
       "//components/web_package:unit_tests",
@@ -417,8 +409,6 @@ test("components_unittests") {
   # Desktop-only deps.
   if (!is_android && !is_ios) {
     deps += [
-      "//components/feedback:unit_tests",
-
       # See comment in components/guest_view/browser/BUILD.gn for why
       # guest_view is currently non-mobile.
       "//components/guest_view/browser:unit_tests",
@@ -456,10 +446,6 @@ test("components_unittests") {
     deps += [ "//components/constrained_window:unit_tests" ]
   }
 
-  if (is_win) {
-    deps += [ "//components/browser_watcher:unit_tests" ]
-  }
-
   if (enable_basic_printing) {
     deps += [ "//components/services/print_compositor:unit_tests" ]
   }
@@ -618,7 +604,6 @@ if (!is_ios && !is_fuchsia) {
       "optimization_guide/content/renderer/page_text_agent_browsertest.cc",
       "paint_preview/renderer/paint_preview_recorder_browsertest.cc",
       "security_state/content/content_utils_browsertest.cc",
-      "ukm/content/source_url_recorder_browsertest.cc",
     ]
 
     data = [
@@ -658,8 +643,6 @@ if (!is_ios && !is_fuchsia) {
       "//components/security_state/core",
       "//components/shared_highlighting/core/common",
       "//components/strings",
-      "//components/ukm:test_support",
-      "//components/ukm/content",
       "//content/public/browser",
       "//content/shell:content_shell_lib",
       "//content/test:browsertest_support",
@@ -802,16 +785,6 @@ if (!is_ios && !is_fuchsia) {
         deps += [ "//v8:v8_external_startup_data_assets" ]
       }
     }
-
-    if (!is_android) {
-      sources += [ "safe_browsing/core/db/v4_store_perftest.cc" ]
-      deps += [
-        "//components/safe_browsing/core/db:v4_protocol_manager_util",
-        "//components/safe_browsing/core/db:v4_store",
-        "//components/safe_browsing/core/db:v4_test_util",
-        "//crypto",
-      ]
-    }
   }
 }
 if (is_android) {
