diff --git a/components/gwp_asan/client/sampling_malloc_shims.cc b/components/gwp_asan/client/sampling_malloc_shims.cc
index 83d1d934beee..503907e73538 100644
--- a/components/gwp_asan/client/sampling_malloc_shims.cc
+++ b/components/gwp_asan/client/sampling_malloc_shims.cc
@@ -15,11 +15,9 @@
 #include "base/process/process_metrics.h"
 #include "base/rand_util.h"
 #include "build/build_config.h"
-#include "components/crash/core/common/crash_key.h"
 #include "components/gwp_asan/client/export.h"
 #include "components/gwp_asan/client/guarded_page_allocator.h"
 #include "components/gwp_asan/client/sampling_state.h"
-#include "components/gwp_asan/common/crash_key_name.h"
 
 #if defined(OS_APPLE)
 #include <pthread.h>
@@ -265,11 +263,9 @@ void InstallMallocHooks(size_t max_allocated_pages,
                         size_t total_pages,
                         size_t sampling_frequency,
                         GuardedPageAllocator::OutOfMemoryCallback callback) {
-  static crash_reporter::CrashKeyString<24> malloc_crash_key(kMallocCrashKey);
   gpa = new GuardedPageAllocator();
   gpa->Init(max_allocated_pages, num_metadata, total_pages, std::move(callback),
             false);
-  malloc_crash_key.Set(gpa->GetCrashKey());
   sampling_state.Init(sampling_frequency);
   base::allocator::InsertAllocatorDispatch(&g_allocator_dispatch);
 }
