diff --git a/components/security_interstitials/DEPS b/components/security_interstitials/DEPS
index 71b130792b0c..fcbe74070e85 100644
--- a/components/security_interstitials/DEPS
+++ b/components/security_interstitials/DEPS
@@ -3,7 +3,6 @@ include_rules = [
   "+components/history/core/browser",
   "+components/metrics",
   "+components/prefs",
-  "+components/rappor",
   "+components/safe_browsing/core/common",
   "+components/safe_browsing/core/db",
   "+components/ssl_errors",
