diff --git a/components/download/public/common/download_file_impl.h b/components/download/public/common/download_file_impl.h
index 6d2a0fe7cb6d..fcf457932973 100644
--- a/components/download/public/common/download_file_impl.h
+++ b/components/download/public/common/download_file_impl.h
@@ -363,7 +363,6 @@ class COMPONENTS_DOWNLOAD_EXPORT DownloadFileImpl : public DownloadFile {
   base::TimeTicks download_start_;
   RateEstimator rate_estimator_;
   int num_active_streams_;
-  bool record_stream_bandwidth_;
   base::TimeTicks last_update_time_;
   size_t bytes_seen_with_parallel_streams_;
   size_t bytes_seen_without_parallel_streams_;
