diff --git a/components/metrics_services_manager/metrics_services_manager.cc b/components/metrics_services_manager/metrics_services_manager.cc
index 107632b0ad8f..09dac0068d37 100644
--- a/components/metrics_services_manager/metrics_services_manager.cc
+++ b/components/metrics_services_manager/metrics_services_manager.cc
@@ -15,8 +15,6 @@
 #include "components/metrics/metrics_state_manager.h"
 #include "components/metrics/metrics_switches.h"
 #include "components/metrics_services_manager/metrics_services_manager_client.h"
-#include "components/rappor/rappor_service_impl.h"
-#include "components/ukm/ukm_service.h"
 #include "components/variations/service/variations_service.h"
 #include "services/network/public/cpp/shared_url_loader_factory.h"
 
@@ -43,20 +41,6 @@ metrics::MetricsService* MetricsServicesManager::GetMetricsService() {
   return GetMetricsServiceClient()->GetMetricsService();
 }
 
-rappor::RapporServiceImpl* MetricsServicesManager::GetRapporServiceImpl() {
-  DCHECK(thread_checker_.CalledOnValidThread());
-  if (!rappor_service_) {
-    rappor_service_ = client_->CreateRapporServiceImpl();
-    rappor_service_->Initialize(client_->GetURLLoaderFactory());
-  }
-  return rappor_service_.get();
-}
-
-ukm::UkmService* MetricsServicesManager::GetUkmService() {
-  DCHECK(thread_checker_.CalledOnValidThread());
-  return GetMetricsServiceClient()->GetUkmService();
-}
-
 variations::VariationsService* MetricsServicesManager::GetVariationsService() {
   DCHECK(thread_checker_.CalledOnValidThread());
   if (!variations_service_)
@@ -90,15 +74,6 @@ void MetricsServicesManager::UpdatePermissions(bool current_may_record,
                                                bool current_consent_given,
                                                bool current_may_upload) {
   DCHECK(thread_checker_.CalledOnValidThread());
-  // If the user has opted out of metrics, delete local UKM state. We only check
-  // consent for UKM.
-  if (consent_given_ && !current_consent_given) {
-    ukm::UkmService* ukm = GetUkmService();
-    if (ukm) {
-      ukm->Purge();
-      ukm->ResetClientState(ukm::ResetReason::kUpdatePermissions);
-    }
-  }
 
   // Stash the current permissions so that we can update the RapporServiceImpl
   // correctly when the Rappor preference changes.
@@ -115,62 +90,16 @@ void MetricsServicesManager::UpdateRunningServices() {
   const base::CommandLine* cmdline = base::CommandLine::ForCurrentProcess();
   if (cmdline->HasSwitch(metrics::switches::kMetricsRecordingOnly)) {
     metrics->StartRecordingForTests();
-    GetRapporServiceImpl()->Update(true, false);
     return;
   }
 
   client_->UpdateRunningServices(may_record_, may_upload_);
 
-  if (may_record_) {
-    if (!metrics->recording_active())
-      metrics->Start();
-    if (may_upload_)
-      metrics->EnableReporting();
-    else
-      metrics->DisableReporting();
-  } else {
-    metrics->Stop();
-  }
-
-  UpdateUkmService();
-
-  GetRapporServiceImpl()->Update(may_record_, may_upload_);
-}
-
-void MetricsServicesManager::UpdateUkmService() {
-  ukm::UkmService* ukm = GetUkmService();
-  if (!ukm)
-    return;
-
-  bool listeners_active =
-      metrics_service_client_->AreNotificationListenersEnabledOnAllProfiles();
-  bool sync_enabled =
-      metrics_service_client_->IsMetricsReportingForceEnabled() ||
-      metrics_service_client_->IsUkmAllowedForAllProfiles();
-  bool is_incognito = client_->IsOffTheRecordSessionActive();
-
-  if (consent_given_ && listeners_active && sync_enabled && !is_incognito) {
-    // TODO(skare): revise this - merged in a big change
-    ukm->EnableRecording(
-        metrics_service_client_->IsUkmAllowedWithExtensionsForAllProfiles());
-    if (may_upload_)
-      ukm->EnableReporting();
-    else
-      ukm->DisableReporting();
-  } else {
-    ukm->DisableRecording();
-    ukm->DisableReporting();
-  }
+  metrics->Stop();
 }
 
 void MetricsServicesManager::UpdateUploadPermissions(bool may_upload) {
-  if (metrics_service_client_->IsMetricsReportingForceEnabled()) {
-    UpdatePermissions(true, true, true);
-    return;
-  }
-
-  UpdatePermissions(client_->IsMetricsReportingEnabled(),
-                    client_->IsMetricsConsentGiven(), may_upload);
+  UpdatePermissions(false, false, false);
 }
 
 bool MetricsServicesManager::IsMetricsReportingEnabled() const {
