diff --git a/components/password_manager/core/browser/password_manager_client.h b/components/password_manager/core/browser/password_manager_client.h
index e5d0849d47206..365fb5ffba998 100644
--- a/components/password_manager/core/browser/password_manager_client.h
+++ b/components/password_manager/core/browser/password_manager_client.h
@@ -62,10 +62,6 @@ class Origin;
 
 class GURL;
 
-namespace safe_browsing {
-class PasswordProtectionService;
-}
-
 namespace password_manager {
 
 class FieldInfoManager;
@@ -331,10 +327,6 @@ class PasswordManagerClient {
   // Returns the current best guess as to the page's display language.
   virtual autofill::LanguageCode GetPageLanguage() const;
 
-  // Return the PasswordProtectionService associated with this instance.
-  virtual safe_browsing::PasswordProtectionService*
-  GetPasswordProtectionService() const = 0;
-
 #if defined(ON_FOCUS_PING_ENABLED)
   // Checks the safe browsing reputation of the webpage when the
   // user focuses on a username/password field. This is used for reporting
@@ -363,13 +355,6 @@ class PasswordManagerClient {
   // and its last committed main frame navigation.
   virtual ukm::SourceId GetUkmSourceId() = 0;
 
-  // Gets a metrics recorder for the currently committed navigation.
-  // As PasswordManagerMetricsRecorder submits metrics on destruction, a new
-  // instance will be returned for each committed navigation. A caller must not
-  // hold on to the pointer. This method returns a nullptr if the client
-  // does not support metrics recording.
-  virtual PasswordManagerMetricsRecorder* GetMetricsRecorder() = 0;
-
   // Gets the PasswordRequirementsService associated with the client. It is
   // valid that this method returns a nullptr if the PasswordRequirementsService
   // has not been implemented for a specific platform or the context is an
