diff --git a/components/policy/core/common/BUILD.gn b/components/policy/core/common/BUILD.gn
index 10290f8679e4a..d29f61be2fdbc 100644
--- a/components/policy/core/common/BUILD.gn
+++ b/components/policy/core/common/BUILD.gn
@@ -166,8 +166,6 @@ source_set("internal") {
     ":common_constants",
     "//components/policy:generated",
     "//components/policy/proto",
-    "//components/reporting/proto:record_constants",
-    "//components/reporting/proto:record_proto",
   ]
 
   deps = [
@@ -379,8 +377,6 @@ static_library("test_support") {
     "//components/account_id",
     "//components/policy:generated",
     "//components/policy/proto",
-    "//components/reporting/proto:record_constants",
-    "//components/reporting/proto:record_proto",
     "//components/strings",
     "//crypto",
     "//net",
