diff --git a/components/assist_ranker/base_predictor.cc b/components/assist_ranker/base_predictor.cc
index c5d774e6293cd..3fd647c384906 100644
--- a/components/assist_ranker/base_predictor.cc
+++ b/components/assist_ranker/base_predictor.cc
@@ -9,8 +9,6 @@
 #include "components/assist_ranker/proto/ranker_model.pb.h"
 #include "components/assist_ranker/ranker_example_util.h"
 #include "components/assist_ranker/ranker_model.h"
-#include "services/metrics/public/cpp/ukm_entry_builder.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
 #include "url/gurl.h"
 
 namespace assist_ranker {
@@ -53,6 +51,7 @@ bool BasePredictor::IsReady() {
   return is_ready_;
 }
 
+#if 0
 void BasePredictor::LogFeatureToUkm(const std::string& feature_name,
                                     const Feature& feature,
                                     ukm::UkmEntryBuilder* ukm_builder) {
@@ -110,6 +109,7 @@ void BasePredictor::LogExampleToUkm(const RankerExample& example,
   }
   builder.Record(ukm::UkmRecorder::Get());
 }
+#endif
 
 std::string BasePredictor::GetModelName() const {
   return config_.model_name;
