diff --git a/components/autofill/core/browser/BUILD.gn b/components/autofill/core/browser/BUILD.gn
index b947cefc2bec5..3915c4d390d15 100644
--- a/components/autofill/core/browser/BUILD.gn
+++ b/components/autofill/core/browser/BUILD.gn
@@ -434,7 +434,6 @@ static_library("browser") {
     "//net",
     "//services/data_decoder/public/cpp:cpp",
     "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/cpp",
     "//services/network/public/mojom",
     "//sql",
@@ -558,8 +557,6 @@ static_library("test_support") {
     "//components/sync:test_support",
     "//components/translate/core/browser",
     "//components/translate/core/browser:test_support",
-    "//components/ukm",
-    "//components/ukm:test_support",
     "//components/version_info:version_info",
     "//google_apis:test_support",
     "//services/network:test_support",
