diff --git a/components/exo/text_input.cc b/components/exo/text_input.cc
index 71fdd0cc09e0..b49df2e8c3b3 100644
--- a/components/exo/text_input.cc
+++ b/components/exo/text_input.cc
@@ -310,11 +310,6 @@ bool TextInput::IsTextEditCommandEnabled(ui::TextEditCommand command) const {
 void TextInput::SetTextEditCommandForNextKeyEvent(ui::TextEditCommand command) {
 }
 
-ukm::SourceId TextInput::GetClientSourceForMetrics() const {
-  NOTIMPLEMENTED_LOG_ONCE();
-  return ukm::kInvalidSourceId;
-}
-
 bool TextInput::ShouldDoLearning() {
   return should_do_learning_;
 }
