diff --git a/components/variations/child_process_field_trial_syncer.cc b/components/variations/child_process_field_trial_syncer.cc
index 6825bdd2d14c..203856846073 100644
--- a/components/variations/child_process_field_trial_syncer.cc
+++ b/components/variations/child_process_field_trial_syncer.cc
@@ -9,7 +9,6 @@
 
 #include "base/base_switches.h"
 #include "base/command_line.h"
-#include "components/variations/variations_crash_keys.h"
 
 namespace variations {
 
@@ -21,8 +20,6 @@ ChildProcessFieldTrialSyncer::~ChildProcessFieldTrialSyncer() {}
 
 void ChildProcessFieldTrialSyncer::InitFieldTrialObserving(
     const base::CommandLine& command_line) {
-  // Set up initial set of crash dump data for field trials in this process.
-  variations::InitCrashKeys();
 
   // Listen for field trial activations to report them to the browser.
   base::FieldTrialList::AddObserver(observer_);
