diff --git a/components/BUILD.gn b/components/BUILD.gn
index 020764dfebf9..cee227163bfe 100644
--- a/components/BUILD.gn
+++ b/components/BUILD.gn
@@ -78,7 +78,6 @@ test("components_unittests") {
     "//components/content_capture/common:unit_tests",
     "//components/content_settings/core/browser:unit_tests",
     "//components/content_settings/core/common:unit_tests",
-    "//components/crash/core/common:unit_tests",
     "//components/crx_file:unit_tests",
     "//components/device_event_log:unit_tests",
     "//components/dom_distiller/core:unit_tests",
@@ -127,7 +126,6 @@ test("components_unittests") {
     "//components/previews/core:unit_tests",
     "//components/proxy_config:unit_tests",
     "//components/query_parser:unit_tests",
-    "//components/rappor:unit_tests",
     "//components/reading_list/core:unit_tests",
     "//components/safe_search_api:unit_tests",
     "//components/scheduling_metrics:unit_tests",
@@ -274,12 +272,6 @@ test("components_unittests") {
       "//components/prerender/browser:unit_tests",
       "//components/previews/content:unit_tests",
       "//components/query_tiles:unit_tests",
-      "//components/safe_browsing/content/password_protection:password_protection_unittest",
-      "//components/safe_browsing/content/triggers:unit_tests",
-      "//components/safe_browsing/content/web_ui:unit_tests",
-      "//components/safe_browsing/core/common:unit_tests",
-      "//components/safe_browsing/core/triggers:unit_tests",
-      "//components/safety_check:unit_tests",
       "//components/security_interstitials/content:unit_tests",
       "//components/security_state/content:unit_tests",
       "//components/services/heap_profiling:unit_tests",
@@ -304,9 +296,6 @@ test("components_unittests") {
 
     if (!is_fuchsia) {  # !iOS and !Fuchsia
       deps += [
-        "//components/crash/content/browser:unit_tests",
-        "//components/crash/content/browser/error_reporting:unit_tests",
-        "//components/crash/core/app:unit_tests",
         "//components/data_reduction_proxy/core/browser:unit_tests",
         "//components/data_reduction_proxy/core/common:unit_tests",
       ]
@@ -332,8 +321,6 @@ test("components_unittests") {
       "//base:base_java_unittest_support",
       "//components/autofill_assistant/browser:unit_tests",
       "//components/cdm/browser:unit_tests",
-      "//components/crash/android:java",
-      "//components/crash/android:unit_tests",
       "//components/download/internal/common:internal_java",
       "//components/embedder_support/android:native_java_unittests_java",
       "//components/embedder_support/android/metrics:test_support_java",
@@ -408,10 +395,6 @@ test("components_unittests") {
     deps += [ "//components/constrained_window:unit_tests" ]
   }
 
-  if (is_win) {
-    deps += [ "//components/browser_watcher:unit_tests" ]
-  }
-
   if (enable_basic_printing) {
     deps += [
       "//components/printing/browser:unit_tests",
@@ -439,7 +422,6 @@ test("components_unittests") {
 
   if (!is_ios) {
     deps += [
-      "//components/safe_browsing/core/realtime:unit_tests",
       "//components/webrtc_logging/browser:unit_tests",
       "//components/webrtc_logging/common:unit_tests",
     ]
@@ -760,9 +742,6 @@ if (!is_ios && !is_fuchsia) {
     if (!is_android) {
       sources += [ "safe_browsing/core/db/v4_store_perftest.cc" ]
       deps += [
-        "//components/safe_browsing/core/db:v4_protocol_manager_util",
-        "//components/safe_browsing/core/db:v4_store",
-        "//components/safe_browsing/core/db:v4_test_util",
         "//crypto",
       ]
     }
