diff --git a/components/autofill/core/browser/payments/credit_card_access_manager.h b/components/autofill/core/browser/payments/credit_card_access_manager.h
index 6bdf8b8c647f..1440a72ac2ca 100644
--- a/components/autofill/core/browser/payments/credit_card_access_manager.h
+++ b/components/autofill/core/browser/payments/credit_card_access_manager.h
@@ -19,7 +19,6 @@
 #include "components/autofill/core/browser/autofill_client.h"
 #include "components/autofill/core/browser/autofill_driver.h"
 #include "components/autofill/core/browser/data_model/credit_card.h"
-#include "components/autofill/core/browser/metrics/credit_card_form_event_logger.h"
 #include "components/autofill/core/browser/payments/credit_card_cvc_authenticator.h"
 #include "components/autofill/core/browser/payments/payments_client.h"
 #include "components/autofill/core/browser/personal_data_manager.h"
@@ -78,12 +77,9 @@ class CreditCardAccessManager : public CreditCardCVCAuthenticator::Requester,
   CreditCardAccessManager(
       AutofillDriver* driver,
       AutofillClient* client,
-      PersonalDataManager* personal_data_manager,
-      CreditCardFormEventLogger* credit_card_form_event_logger);
+      PersonalDataManager* personal_data_manager);
   ~CreditCardAccessManager() override;
 
-  // Logs information about current credit card data.
-  void UpdateCreditCardFormEventLogger();
   // Returns all credit cards.
   std::vector<CreditCard*> GetCreditCards();
   // Returns credit cards in the order to be suggested to the user.
@@ -262,9 +258,6 @@ class CreditCardAccessManager : public CreditCardCVCAuthenticator::Requester,
   // May be NULL. NULL indicates OTR.
   PersonalDataManager* personal_data_manager_;
 
-  // For logging metrics.
-  CreditCardFormEventLogger* form_event_logger_;
-
   // Timestamp used for preflight call metrics.
   base::TimeTicks preflight_call_timestamp_;
 
