diff --git a/components/password_manager/core/browser/password_form_filling.h b/components/password_manager/core/browser/password_form_filling.h
index 32963800834f..39e44f3c8935 100644
--- a/components/password_manager/core/browser/password_form_filling.h
+++ b/components/password_manager/core/browser/password_form_filling.h
@@ -18,7 +18,6 @@ struct PasswordFormFillData;
 namespace password_manager {
 class PasswordManagerClient;
 class PasswordManagerDriver;
-class PasswordFormMetricsRecorder;
 
 // Enum detailing the browser process' best belief what kind of credential
 // filling is used in the renderer for a given password form.
@@ -45,8 +44,7 @@ LikelyFormFilling SendFillInformationToRenderer(
     const PasswordForm& observed_form,
     const std::vector<const PasswordForm*>& best_matches,
     const std::vector<const PasswordForm*>& federated_matches,
-    const PasswordForm* preferred_match,
-    PasswordFormMetricsRecorder* metrics_recorder);
+    const PasswordForm* preferred_match);
 
 // Create a PasswordFormFillData structure in preparation for filling a form
 // identified by |form_on_page|, with credentials from |preferred_match| and
