diff --git a/components/crash/core/app/BUILD.gn b/components/crash/core/app/BUILD.gn
index 5cd0b2b043bb..7e9eeda3ab82 100644
--- a/components/crash/core/app/BUILD.gn
+++ b/components/crash/core/app/BUILD.gn
@@ -154,7 +154,6 @@ if (is_win) {
 
     deps = [
       "//base",
-      "//components/browser_watcher:activity_report",
       "//components/gwp_asan/buildflags",
       "//third_party/crashpad/crashpad/client",
       "//third_party/crashpad/crashpad/handler",
