diff --git a/components/password_manager/core/browser/password_form_manager_for_ui.h b/components/password_manager/core/browser/password_form_manager_for_ui.h
index 5f9e5147b959..fc62f6c50b9f 100644
--- a/components/password_manager/core/browser/password_form_manager_for_ui.h
+++ b/components/password_manager/core/browser/password_form_manager_for_ui.h
@@ -17,7 +17,6 @@ namespace password_manager {
 struct CompromisedCredentials;
 struct InteractionsStats;
 struct PasswordForm;
-class PasswordFormMetricsRecorder;
 
 // Interface that contains all methods from PasswordFormManager that are used in
 // UI.
@@ -44,10 +43,6 @@ class PasswordFormManagerForUI {
   // origin.
   virtual metrics_util::CredentialSourceType GetCredentialSource() const = 0;
 
-  // Returns metric recorder which responsible for recording metrics for this
-  // form.
-  virtual PasswordFormMetricsRecorder* GetMetricsRecorder() = 0;
-
   // Statistics for recent password bubble usage.
   virtual base::span<const InteractionsStats> GetInteractionsStats() const = 0;
 
