diff --git a/components/arc/mojom/BUILD.gn b/components/arc/mojom/BUILD.gn
index 825de163af295..cc84ec61efc03 100644
--- a/components/arc/mojom/BUILD.gn
+++ b/components/arc/mojom/BUILD.gn
@@ -32,7 +32,6 @@ if (is_chromeos_ash) {
       "dark_theme.mojom",
       "digital_goods.mojom",
       "disk_quota.mojom",
-      "enterprise_reporting.mojom",
       "file_system.mojom",
       "iio_sensor.mojom",
       "ime.mojom",
