diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index 06ca8e270053..36c501587dde 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -226,7 +226,6 @@ static_library("browser") {
     "//components/omnibox/common",
     "//components/omnibox/resources:omnibox_resources",
     "//components/security_state/core",
-    "//third_party/metrics_proto",
     "//url",
   ]
   deps = [
@@ -305,7 +304,6 @@ static_library("location_bar") {
     "//base",
     "//components/omnibox/common",
     "//components/security_state/core",
-    "//third_party/metrics_proto",
     "//url",
   ]
   deps = [
@@ -465,7 +463,6 @@ static_library("test_support") {
     "//services/network:test_support",
     "//testing/gmock",
     "//testing/gtest",
-    "//third_party/metrics_proto",
     "//ui/base",
     "//ui/gfx",
   ]
@@ -598,7 +595,6 @@ fuzzer_test("autocomplete_input_fuzzer") {
     ":test_support",
     "//base",
     "//base:i18n",
-    "//third_party/metrics_proto:metrics_proto",
   ]
 }
 
@@ -618,7 +614,6 @@ fuzzer_test("search_suggestion_parser_fuzzer") {
     "//base",
     "//base:i18n",
     "//third_party/icu:icui18n",
-    "//third_party/metrics_proto:metrics_proto",
   ]
   seed_corpus = "fuzz_corpus_suggest"
 }
