diff --git a/components/password_manager/core/browser/password_store.cc b/components/password_manager/core/browser/password_store.cc
index ffa72a0c752f0..64ebe848b62ce 100644
--- a/components/password_manager/core/browser/password_store.cc
+++ b/components/password_manager/core/browser/password_store.cc
@@ -42,7 +42,6 @@
 #include "components/password_manager/core/common/password_manager_features.h"
 #include "components/password_manager/core/common/password_manager_pref_names.h"
 #include "components/prefs/pref_service.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "components/sync/model/client_tag_based_model_type_processor.h"
 #include "components/sync/model/proxy_model_type_controller_delegate.h"
 
@@ -111,6 +110,7 @@ void PasswordStore::DatabaseInsecureCredentialsObserver::
   OnInsecureCredentialsChanged();
 }
 
+#if 0
 PasswordStore::CheckReuseRequest::CheckReuseRequest(
     PasswordReuseDetectorConsumer* consumer)
     : origin_task_runner_(base::SequencedTaskRunnerHandle::Get()),
@@ -134,6 +134,7 @@ void PasswordStore::CheckReuseRequest::OnReuseCheckDone(
                      matching_reused_credentials, saved_passwords));
   TRACE_EVENT_NESTABLE_ASYNC_END0("passwords", "CheckReuseRequest", this);
 }
+#endif
 
 PasswordStore::FormDigest::FormDigest(PasswordForm::Scheme new_scheme,
                                       const std::string& new_signon_realm,
@@ -547,6 +548,7 @@ void PasswordStore::SetSyncTaskTimeoutForTest(base::TimeDelta timeout) {
   sync_task_timeout_ = timeout;
 }
 
+#if 0
 void PasswordStore::CheckReuse(const std::u16string& input,
                                const std::string& domain,
                                PasswordReuseDetectorConsumer* consumer) {
@@ -554,6 +556,7 @@ void PasswordStore::CheckReuse(const std::u16string& input,
                               std::make_unique<CheckReuseRequest>(consumer),
                               input, domain));
 }
+#endif
 
 void PasswordStore::PreparePasswordHashData(const std::string& sync_username,
                                             const bool is_signed_in) {
@@ -564,30 +567,24 @@ void PasswordStore::PreparePasswordHashData(const std::string& sync_username,
 
 void PasswordStore::SaveGaiaPasswordHash(const std::string& username,
                                          const std::u16string& password,
-                                         bool is_primary_account,
-                                         GaiaPasswordHashChange event) {
+                                         bool is_primary_account) {
   SaveProtectedPasswordHash(username, password, is_primary_account,
-                            /*is_gaia_password=*/true, event);
+                            /*is_gaia_password=*/true);
 }
 
 void PasswordStore::SaveEnterprisePasswordHash(const std::string& username,
                                                const std::u16string& password) {
   SaveProtectedPasswordHash(
       username, password, /*is_primary_account=*/false,
-      /*is_gaia_password=*/false,
-      GaiaPasswordHashChange::NON_GAIA_ENTERPRISE_PASSWORD_CHANGE);
+      /*is_gaia_password=*/false);
 }
 
 void PasswordStore::SaveProtectedPasswordHash(const std::string& username,
                                               const std::u16string& password,
                                               bool is_primary_account,
-                                              bool is_gaia_password,
-                                              GaiaPasswordHashChange event) {
+                                              bool is_gaia_password) {
   if (hash_password_manager_.SavePasswordHash(username, password,
                                               is_gaia_password)) {
-    if (is_gaia_password) {
-      metrics_util::LogGaiaPasswordHashChange(event, is_primary_account);
-    }
     // This method is not being called on startup so it shouldn't log metrics.
     // |is_signed_in| is only used when |should_log_metrics| is true so
     // it doesn't matter what the value is here.
@@ -597,11 +594,8 @@ void PasswordStore::SaveProtectedPasswordHash(const std::string& username,
 }
 
 void PasswordStore::SaveSyncPasswordHash(
-    const PasswordHashData& sync_password_data,
-    GaiaPasswordHashChange event) {
+    const PasswordHashData& sync_password_data) {
   if (hash_password_manager_.SavePasswordHash(sync_password_data)) {
-    metrics_util::LogGaiaPasswordHashChange(event,
-                                            /*is_sync_password=*/true);
     SchedulePasswordHashUpdate(/*should_log_metrics=*/false,
                                /*does_primary_account_exists=*/false,
                                /*is_signed_in=*/false);
@@ -658,15 +652,6 @@ void PasswordStore::SchedulePasswordHashUpdate(bool should_log_metrics,
 }
 
 void PasswordStore::ScheduleEnterprisePasswordURLUpdate() {
-  std::vector<GURL> enterprise_login_urls;
-  safe_browsing::GetPasswordProtectionLoginURLsPref(*prefs_,
-                                                    &enterprise_login_urls);
-  GURL enterprise_change_password_url =
-      safe_browsing::GetPasswordProtectionChangePasswordURLPref(*prefs_);
-
-  ScheduleTask(base::BindOnce(&PasswordStore::SaveEnterprisePasswordURLs, this,
-                              std::move(enterprise_login_urls),
-                              std::move(enterprise_change_password_url)));
 }
 
 PasswordStore::~PasswordStore() {
@@ -695,16 +680,6 @@ bool PasswordStore::InitOnBackgroundSequence(
           syncer::PASSWORDS, base::DoNothing()),
       /*password_store_sync=*/this, sync_enabled_or_disabled_cb_,
       force_initial_sync_cycle));
-
-  if (IsPasswordReuseDetectionEnabled()) {
-    reuse_detector_ = new PasswordReuseDetector;
-
-    base::SequencedTaskRunnerHandle::Get()->PostTask(
-        FROM_HERE,
-        base::BindOnce(&PasswordStoreConsumer::OnGetPasswordStoreResultsFrom,
-                       reuse_detector_->GetWeakPtr(), base::RetainedRef(this),
-                       GetAutofillableLoginsImpl()));
-  }
   return true;
 }
 
@@ -766,8 +741,10 @@ void PasswordStore::NotifyLoginsChanged(
     if (sync_bridge_)
       sync_bridge_->ActOnPasswordStoreChanges(changes);
 
+#if 0
     if (reuse_detector_)
       reuse_detector_->OnLoginsChanged(changes);
+#endif
   }
 
   if (base::ranges::any_of(changes, [](const auto& change) {
@@ -828,6 +805,7 @@ void PasswordStore::NotifyUnsyncedCredentialsWillBeDeleted(
   }
 }
 
+#if 0
 void PasswordStore::CheckReuseImpl(std::unique_ptr<CheckReuseRequest> request,
                                    const std::u16string& input,
                                    const std::string& domain) {
@@ -837,13 +815,14 @@ void PasswordStore::CheckReuseImpl(std::unique_ptr<CheckReuseRequest> request,
     request->OnReuseCheckDone(false, 0, absl::nullopt, {}, 0);
   }
 }
+#endif
 
 void PasswordStore::SaveProtectedPasswordHashImpl(
     PasswordHashDataList protected_password_data_list,
     bool should_log_metrics,
     bool does_primary_account_exists,
     bool is_signed_in) {
-  if (!reuse_detector_ || !protected_password_data_list.has_value())
+  if (!protected_password_data_list.has_value())
     return;
   TRACE_EVENT0("passwords", "PasswordStore::SaveProtectedPasswordHashImpl");
 
@@ -861,44 +840,28 @@ void PasswordStore::SaveProtectedPasswordHashImpl(
         gaia_password_hash_list.size(), enterprise_password_hash_list.size(),
         does_primary_account_exists, is_signed_in);
   }
-  reuse_detector_->UseGaiaPasswordHash(std::move(gaia_password_hash_list));
-  reuse_detector_->UseNonGaiaEnterprisePasswordHash(
-      std::move(enterprise_password_hash_list));
 }
 
 void PasswordStore::SaveEnterprisePasswordURLs(
     const std::vector<GURL>& enterprise_login_urls,
     const GURL& enterprise_change_password_url) {
-  if (!reuse_detector_)
-    return;
-  TRACE_EVENT0("passwords", "PasswordStore::SaveEnterprisePasswordURLs");
-  reuse_detector_->UseEnterprisePasswordURLs(std::move(enterprise_login_urls),
-                                             enterprise_change_password_url);
 }
 
 void PasswordStore::ClearGaiaPasswordHashImpl(const std::string& username) {
   TRACE_EVENT0("passwords", "PasswordStore::ClearGaiaPasswordHashImpl");
-  if (reuse_detector_)
-    reuse_detector_->ClearGaiaPasswordHash(username);
 }
 
 void PasswordStore::ClearAllGaiaPasswordHashImpl() {
   TRACE_EVENT0("passwords", "PasswordStore::ClearAllGaiaPasswordHashImpl");
-  if (reuse_detector_)
-    reuse_detector_->ClearAllGaiaPasswordHash();
 }
 
 void PasswordStore::ClearAllEnterprisePasswordHashImpl() {
   TRACE_EVENT0("passwords",
                "PasswordStore::ClearAllEnterprisePasswordHashImpl");
-  if (reuse_detector_)
-    reuse_detector_->ClearAllEnterprisePasswordHash();
 }
 
 void PasswordStore::ClearAllNonGmailPasswordHashImpl() {
   TRACE_EVENT0("passwords", "PasswordStore::ClearAllNonGmailPasswordHashImpl");
-  if (reuse_detector_)
-    reuse_detector_->ClearAllNonGmailPasswordHash();
 }
 
 void PasswordStore::OnInitCompleted(bool success) {
@@ -1378,9 +1341,6 @@ PasswordStore::GetSyncControllerDelegateOnBackgroundSequence() {
 void PasswordStore::DestroyOnBackgroundSequence() {
   DCHECK(background_task_runner_->RunsTasksInCurrentSequence());
   sync_bridge_.reset();
-
-  delete reuse_detector_;
-  reuse_detector_ = nullptr;
 }
 
 }  // namespace password_manager
