diff --git a/components/policy/core/common/BUILD.gn b/components/policy/core/common/BUILD.gn
index d397b6de091e..0f7856d08796 100644
--- a/components/policy/core/common/BUILD.gn
+++ b/components/policy/core/common/BUILD.gn
@@ -62,10 +62,6 @@ source_set("internal") {
     "cloud/dm_token.h",
     "cloud/dmserver_job_configurations.cc",
     "cloud/dmserver_job_configurations.h",
-    "cloud/encrypted_reporting_job_configuration.cc",
-    "cloud/encrypted_reporting_job_configuration.h",
-    "cloud/enterprise_metrics.cc",
-    "cloud/enterprise_metrics.h",
     "cloud/external_policy_data_fetcher.cc",
     "cloud/external_policy_data_fetcher.h",
     "cloud/external_policy_data_updater.cc",
@@ -76,10 +72,6 @@ source_set("internal") {
     "cloud/machine_level_user_cloud_policy_store.h",
     "cloud/policy_invalidation_scope.h",
     "cloud/policy_value_validator.h",
-    "cloud/realtime_reporting_job_configuration.cc",
-    "cloud/realtime_reporting_job_configuration.h",
-    "cloud/reporting_job_configuration_base.cc",
-    "cloud/reporting_job_configuration_base.h",
     "cloud/resource_cache.cc",
     "cloud/resource_cache.h",
     "cloud/signing_service.h",
@@ -137,14 +129,6 @@ source_set("internal") {
     "policy_service_impl.h",
     "policy_statistics_collector.cc",
     "policy_statistics_collector.h",
-    "remote_commands/remote_command_job.cc",
-    "remote_commands/remote_command_job.h",
-    "remote_commands/remote_commands_factory.cc",
-    "remote_commands/remote_commands_factory.h",
-    "remote_commands/remote_commands_queue.cc",
-    "remote_commands/remote_commands_queue.h",
-    "remote_commands/remote_commands_service.cc",
-    "remote_commands/remote_commands_service.h",
     "schema.cc",
     "schema.h",
     "schema_internal.h",
@@ -164,8 +148,6 @@ source_set("internal") {
     ":common_constants",
     "//components/policy:generated",
     "//components/policy/proto",
-    "//components/reporting/proto:record_constants",
-    "//components/reporting/proto:record_proto",
   ]
 
   deps = [
@@ -173,7 +155,6 @@ source_set("internal") {
     "//base/third_party/dynamic_annotations",
     "//build:chromeos_buildflags",
     "//components/account_id",
-    "//components/enterprise/common:strings",
     "//components/prefs",
     "//components/signin/public/identity_manager",
     "//components/strings",
@@ -377,8 +358,6 @@ static_library("test_support") {
     "//components/account_id",
     "//components/policy:generated",
     "//components/policy/proto",
-    "//components/reporting/proto:record_constants",
-    "//components/reporting/proto:record_proto",
     "//components/strings",
     "//crypto",
     "//net",
