diff --git a/components/module_installer/android/BUILD.gn b/components/module_installer/android/BUILD.gn
index 67028f66c5d3..0df4865b4513 100644
--- a/components/module_installer/android/BUILD.gn
+++ b/components/module_installer/android/BUILD.gn
@@ -35,7 +35,6 @@ android_library("module_installer_java") {
   deps = [
     "//base:base_java",
     "//base:jni_java",
-    "//components/crash/android:java",
     "//third_party/android_deps:androidx_annotation_annotation_java",
     "//third_party/google_android_play_core:com_google_android_play_core_java",
   ]
