diff --git a/components/autofill/core/browser/payments/credit_card_fido_authenticator.h b/components/autofill/core/browser/payments/credit_card_fido_authenticator.h
index 93b11355f9ae..eeb843aa763f 100644
--- a/components/autofill/core/browser/payments/credit_card_fido_authenticator.h
+++ b/components/autofill/core/browser/payments/credit_card_fido_authenticator.h
@@ -224,9 +224,6 @@ class CreditCardFIDOAuthenticator
   // Returns true if |request_options| contains a challenge.
   bool IsValidCreationOptions(const base::Value& creation_options);
 
-  // Logs the result of a WebAuthn prompt.
-  void LogWebauthnResult(AuthenticatorStatus status);
-
   // Updates the user preference to the value of |user_is_opted_in_|.
   void UpdateUserPref();
 
