diff --git a/components/autofill/core/browser/BUILD.gn b/components/autofill/core/browser/BUILD.gn
index b0dcbce1521a..44c96a7df0cb 100644
--- a/components/autofill/core/browser/BUILD.gn
+++ b/components/autofill/core/browser/BUILD.gn
@@ -61,8 +61,6 @@ static_library("browser") {
     "autofill_manager.cc",
     "autofill_manager.h",
     "autofill_manager_test_delegate.h",
-    "autofill_metrics.cc",
-    "autofill_metrics.h",
     "autofill_observer.cc",
     "autofill_observer.h",
     "autofill_profile_sync_util.cc",
@@ -186,13 +184,6 @@ static_library("browser") {
     "logging/log_receiver.h",
     "logging/log_router.cc",
     "logging/log_router.h",
-    "metrics/address_form_event_logger.cc",
-    "metrics/address_form_event_logger.h",
-    "metrics/credit_card_form_event_logger.cc",
-    "metrics/credit_card_form_event_logger.h",
-    "metrics/form_event_logger_base.cc",
-    "metrics/form_event_logger_base.h",
-    "metrics/form_events.h",
     "pattern_provider/pattern_configuration_parser.cc",
     "pattern_provider/pattern_configuration_parser.h",
     "pattern_provider/pattern_provider.cc",
@@ -321,13 +312,6 @@ static_library("browser") {
     ]
   }
 
-  if (is_ios) {
-    sources += [
-      "keyboard_accessory_metrics_logger.h",
-      "keyboard_accessory_metrics_logger.mm",
-    ]
-  }
-
   if (is_ios || is_android) {
     sources += [
       "payments/autofill_credit_card_filling_infobar_delegate_mobile.cc",
@@ -383,7 +367,6 @@ static_library("browser") {
     "//components/history/core/browser",
     "//components/infobars/core",
     "//components/keyed_service/core",
-    "//components/language_usage_metrics",
     "//components/leveldb_proto",
     "//components/os_crypt",
     "//components/policy/core/browser",
@@ -403,8 +386,6 @@ static_library("browser") {
     "//google_apis",
     "//net",
     "//services/data_decoder/public/cpp:cpp",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/cpp",
     "//services/network/public/mojom",
     "//sql",
@@ -531,8 +512,6 @@ static_library("test_support") {
     "//components/sync:test_support",
     "//components/translate/core/browser",
     "//components/translate/core/browser:test_support",
-    "//components/ukm",
-    "//components/ukm:test_support",
     "//google_apis:test_support",
     "//services/network:test_support",
     "//services/network/public/cpp",
@@ -605,7 +584,6 @@ source_set("unit_tests") {
     "autofill_external_delegate_unittest.cc",
     "autofill_manager_unittest.cc",
     "autofill_merge_unittest.cc",
-    "autofill_metrics_unittest.cc",
     "autofill_profile_sync_util_unittest.cc",
     "autofill_profile_validation_util_unittest.cc",
     "autofill_profile_validator_unittest.cc",
@@ -738,8 +716,6 @@ source_set("unit_tests") {
     "//components/sync:test_support_model",
     "//components/sync/driver:test_support",
     "//components/translate/core/browser",
-    "//components/ukm",
-    "//components/ukm:test_support",
     "//components/unified_consent",
     "//components/variations",
     "//components/variations:test_support",
@@ -750,7 +726,6 @@ source_set("unit_tests") {
     "//google_apis:test_support",
     "//net:test_support",
     "//services/data_decoder/public/cpp:test_support",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network:test_support",
     "//services/network/public/cpp",
     "//sql",
