diff --git a/components/autofill/core/browser/autofill_manager.h b/components/autofill/core/browser/autofill_manager.h
index 387029bbda51..dc5eb8af12ed 100644
--- a/components/autofill/core/browser/autofill_manager.h
+++ b/components/autofill/core/browser/autofill_manager.h
@@ -27,11 +27,8 @@
 #include "components/autofill/core/browser/autofill_download_manager.h"
 #include "components/autofill/core/browser/autofill_driver.h"
 #include "components/autofill/core/browser/autofill_handler.h"
-#include "components/autofill/core/browser/autofill_metrics.h"
 #include "components/autofill/core/browser/field_filler.h"
 #include "components/autofill/core/browser/form_types.h"
-#include "components/autofill/core/browser/metrics/address_form_event_logger.h"
-#include "components/autofill/core/browser/metrics/credit_card_form_event_logger.h"
 #include "components/autofill/core/browser/payments/card_unmask_delegate.h"
 #include "components/autofill/core/browser/payments/credit_card_access_manager.h"
 #include "components/autofill/core/browser/payments/full_card_request.h"
@@ -298,13 +295,6 @@ class AutofillManager : public AutofillHandler,
     return MakeFrontendID(cc_backend_id, profile_backend_id);
   }
 
-  // A public wrapper that calls |form_interactions_ukm_logger| for testing
-  // purposes only.
-  AutofillMetrics::FormInteractionsUkmLogger*
-  form_interactions_ukm_logger_for_test() {
-    return form_interactions_ukm_logger();
-  }
-
   // A public wrapper that calls |ShouldTriggerRefill| for testing purposes
   // only.
   bool ShouldTriggerRefillForTest(const FormStructure& form_structure) {
@@ -379,10 +369,6 @@ class AutofillManager : public AutofillHandler,
   void OnFormsParsed(const std::vector<const FormData*>& forms,
                      const base::TimeTicks timestamp) override;
 
-  AutofillMetrics::FormInteractionsUkmLogger* form_interactions_ukm_logger() {
-    return form_interactions_ukm_logger_.get();
-  }
-
   // Exposed for testing.
   void set_download_manager(AutofillDownloadManager* manager) {
     download_manager_.reset(manager);
@@ -614,11 +600,6 @@ class AutofillManager : public AutofillHandler,
   bool ShouldShowVirtualCardOption(FormStructure* form_structure);
 #endif
 
-  // Returns an appropriate EventFormLogger for the given |field_type_group|.
-  // May return nullptr.
-  FormEventLoggerBase* GetEventFormLogger(
-      FieldTypeGroup field_type_group) const;
-
   void SetDataList(const std::vector<base::string16>& values,
                    const std::vector<base::string16>& labels);
   AutofillClient* const client_;
@@ -646,14 +627,6 @@ class AutofillManager : public AutofillHandler,
   // May be NULL.  NULL indicates OTR.
   base::WeakPtr<AutocompleteHistoryManager> autocomplete_history_manager_;
 
-  // Utility for logging URL keyed metrics.
-  std::unique_ptr<AutofillMetrics::FormInteractionsUkmLogger>
-      form_interactions_ukm_logger_;
-
-  // Utilities for logging form events.
-  std::unique_ptr<AddressFormEventLogger> address_form_event_logger_;
-  std::unique_ptr<CreditCardFormEventLogger> credit_card_form_event_logger_;
-
   // Have we logged whether Autofill is enabled for this page load?
   bool has_logged_autofill_enabled_ = false;
   // Have we logged an address suggestions count metric for this page?
