diff --git a/components/policy/core/common/policy_loader_common.cc b/components/policy/core/common/policy_loader_common.cc
index c77a7774262f..ad381207646c 100644
--- a/components/policy/core/common/policy_loader_common.cc
+++ b/components/policy/core/common/policy_loader_common.cc
@@ -36,7 +36,6 @@ const char* kSensitivePolicies[] = {
     key::kMetricsReportingEnabled,
     key::kNewTabPageLocation,
     key::kPasswordProtectionChangePasswordURL,
-    key::kPasswordProtectionLoginURLs,
     key::kRestoreOnStartup,
     key::kRestoreOnStartupURLs,
     key::kSafeBrowsingForTrustedSourcesEnabled,
