diff --git a/components/browser_watcher/BUILD.gn b/components/browser_watcher/BUILD.gn
index 2c673d73311e..706ed07349bd 100644
--- a/components/browser_watcher/BUILD.gn
+++ b/components/browser_watcher/BUILD.gn
@@ -20,7 +20,6 @@ if (is_win) {
       ":stability_common",
       "//base",
       "//components/metrics",
-      "//third_party/crashpad/crashpad/client",
     ]
   }
 
@@ -36,11 +35,6 @@ if (is_win) {
       ":stability_client",
       ":stability_common",
       "//base",
-      "//third_party/crashpad/crashpad/client",
-      "//third_party/crashpad/crashpad/compat",
-      "//third_party/crashpad/crashpad/handler",
-      "//third_party/crashpad/crashpad/minidump",
-      "//third_party/crashpad/crashpad/snapshot",
     ]
   }
 
@@ -55,8 +49,6 @@ if (is_win) {
       ":stability_client",
       "//base",
       "//components/variations",
-      "//third_party/crashpad/crashpad/client:client",
-      "//third_party/crashpad/crashpad/util",
     ]
   }
 
@@ -76,8 +68,6 @@ if (is_win) {
     deps = [
       ":activity_report_proto",
       "//base",
-      "//third_party/crashpad/crashpad/client",
-      "//third_party/crashpad/crashpad/util",
     ]
 
     # This needs to be a public dep because of base::win::PEImage.
@@ -104,12 +94,6 @@ if (is_win) {
       "//components/metrics:metrics",
       "//testing/gmock",
       "//testing/gtest",
-      "//third_party/crashpad/crashpad/client",
-
-      # TODO(manzagop): remove this lib once Crashpad writes the minidumps.
-      "//third_party/crashpad/crashpad/minidump",
-      "//third_party/crashpad/crashpad/snapshot",
-      "//third_party/crashpad/crashpad/util",
     ]
   }
 
