diff --git a/components/download/internal/common/resource_downloader.cc b/components/download/internal/common/resource_downloader.cc
index 64b8adb711a9..d76216b39322 100644
--- a/components/download/internal/common/resource_downloader.cc
+++ b/components/download/internal/common/resource_downloader.cc
@@ -168,7 +168,7 @@ void ResourceDownloader::Start(
       download_url_parameters->request_headers(),
       download_url_parameters->request_origin(),
       download_url_parameters->download_source(),
-      std::vector<GURL>(1, resource_request_->url), is_background_mode);
+      std::vector<GURL>(1, resource_request_->url));
 
   mojo::PendingRemote<network::mojom::URLLoaderClient> url_loader_client_remote;
   url_loader_client_receiver_ =
@@ -207,8 +207,7 @@ void ResourceDownloader::InterceptResponse(
       network::mojom::RedirectMode::kFollow,
       download::DownloadUrlParameters::RequestHeadersType(),
       std::string(), /* request_origin */
-      download::DownloadSource::NAVIGATION, std::move(url_chain),
-      false /* is_background_mode */);
+      download::DownloadSource::NAVIGATION, std::move(url_chain));
 
   // Simulate on the new URLLoaderClient calls that happened on the old client.
   response_head->cert_status = cert_status;
