diff --git a/components/download/internal/common/download_worker.cc b/components/download/internal/common/download_worker.cc
index 87155179737d0..f82bd534f46ac 100644
--- a/components/download/internal/common/download_worker.cc
+++ b/components/download/internal/common/download_worker.cc
@@ -10,7 +10,6 @@
 #include "components/download/internal/common/resource_downloader.h"
 #include "components/download/public/common/download_create_info.h"
 #include "components/download/public/common/download_interrupt_reasons.h"
-#include "components/download/public/common/download_stats.h"
 #include "components/download/public/common/download_task_runner.h"
 #include "components/download/public/common/download_utils.h"
 #include "components/download/public/common/input_stream.h"
@@ -131,7 +130,6 @@ void DownloadWorker::OnUrlDownloadStarted(
     create_info->result = DOWNLOAD_INTERRUPT_REASON_SERVER_NO_RANGE;
 
   if (create_info->result != DOWNLOAD_INTERRUPT_REASON_NONE) {
-    RecordParallelRequestCreationFailure(create_info->result);
     VLOG(kWorkerVerboseLevel)
         << "Parallel download sub-request failed. reason = "
         << create_info->result;
