diff --git a/components/page_info/page_info_delegate.h b/components/page_info/page_info_delegate.h
index 599d25310aca2..4aaafa6a9e4a3 100644
--- a/components/page_info/page_info_delegate.h
+++ b/components/page_info/page_info_delegate.h
@@ -13,7 +13,6 @@
 #include "components/permissions/permission_result.h"
 #include "components/permissions/permission_uma_util.h"
 #include "components/safe_browsing/buildflags.h"
-#include "components/safe_browsing/core/password_protection/metrics_util.h"
 #include "components/security_state/core/security_state.h"
 
 namespace permissions {
@@ -21,10 +20,6 @@ class ObjectPermissionContextBase;
 class PermissionDecisionAutoBlocker;
 }  // namespace permissions
 
-namespace safe_browsing {
-class PasswordProtectionService;
-}  // namespace safe_browsing
-
 namespace ui {
 class Event;
 }  // namespace ui
@@ -79,10 +74,6 @@ class PageInfoDelegate {
   // content settings (aka. site permissions).
   virtual HostContentSettingsMap* GetContentSettings() = 0;
 
-  // The subresource filter service determines whether ads should be blocked on
-  // the site and relevant permission prompts should be shown respectively.
-  virtual bool IsSubresourceFilterActivated(const GURL& site_url) = 0;
-
   virtual std::unique_ptr<
       content_settings::PageSpecificContentSettings::Delegate>
   GetPageSpecificContentSettingsDelegate() = 0;
