diff --git a/components/ntp_snippets/BUILD.gn b/components/ntp_snippets/BUILD.gn
index 64b1abd65b57..7c5e4e5b1aa4 100644
--- a/components/ntp_snippets/BUILD.gn
+++ b/components/ntp_snippets/BUILD.gn
@@ -108,8 +108,6 @@ static_library("ntp_snippets") {
     "//components/variations/net",
     "//components/variations/service",
     "//components/web_resource",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/cpp",
     "//services/network/public/mojom",
     "//third_party/icu/",
@@ -159,12 +157,10 @@ source_set("unit_tests") {
     "//components/strings",
     "//components/sync:test_support",
     "//components/sync_preferences:test_support",
-    "//components/ukm:test_support",
     "//components/variations:test_support",
     "//components/variations/net",
     "//components/web_resource:web_resource",
     "//net:test_support",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network:test_support",
     "//services/network/public/cpp",
     "//services/network/public/mojom",
