diff --git a/components/download/database/download_info.h b/components/download/database/download_info.h
index 05b9312babd87..d79b354dff335 100644
--- a/components/download/database/download_info.h
+++ b/components/download/database/download_info.h
@@ -8,7 +8,6 @@
 #include <string>
 
 #include "components/download/database/in_progress/in_progress_info.h"
-#include "components/download/database/in_progress/ukm_info.h"
 #include "third_party/abseil-cpp/absl/types/optional.h"
 
 namespace download {
@@ -29,9 +28,6 @@ struct DownloadInfo {
   // Deprecated, only kept for the purpose of download extension API.
   int id = -1;
 
-  // UKM information for reporting.
-  absl::optional<UkmInfo> ukm_info;
-
   // In progress information for active download.
   absl::optional<InProgressInfo> in_progress_info;
 };
