diff --git a/components/crash/content/browser/BUILD.gn b/components/crash/content/browser/BUILD.gn
index 7649900ccecb..61882eeb7764 100644
--- a/components/crash/content/browser/BUILD.gn
+++ b/components/crash/content/browser/BUILD.gn
@@ -27,10 +27,6 @@ source_set("browser") {
     ]
   }
 
-  if (!is_chromeos) {
-    deps += [ "//third_party/crashpad/crashpad/client" ]
-  }
-
   # This is not in the GYP build but this target includes breakpad client
   # headers, so add the dependency here.
   if ((is_posix && !is_ios) || is_fuchsia) {
