diff --git a/components/enterprise/BUILD.gn b/components/enterprise/BUILD.gn
index 35a6655b58cea..6d4616ffb7c0d 100644
--- a/components/enterprise/BUILD.gn
+++ b/components/enterprise/BUILD.gn
@@ -24,7 +24,7 @@ static_library("enterprise") {
     "//services/network/public/cpp",
   ]
 
-  if (!is_android) {
+  if (false) {
     sources += [
       "browser/reporting/browser_report_generator.cc",
       "browser/reporting/browser_report_generator.h",
