diff --git a/components/download/internal/common/download_job_factory.cc b/components/download/internal/common/download_job_factory.cc
index 052f69f77693..8290864f6914 100644
--- a/components/download/internal/common/download_job_factory.cc
+++ b/components/download/internal/common/download_job_factory.cc
@@ -13,7 +13,6 @@
 #include "components/download/internal/common/save_package_download_job.h"
 #include "components/download/public/common/download_features.h"
 #include "components/download/public/common/download_item.h"
-#include "components/download/public/common/download_stats.h"
 #include "net/http/http_response_info.h"
 
 namespace download {
@@ -96,43 +95,6 @@ bool IsParallelizableDownload(const DownloadCreateInfo& create_info,
   if (!IsParallelDownloadEnabled())
     return is_parallelizable;
 
-  RecordParallelDownloadCreationEvent(
-      is_parallelizable
-          ? ParallelDownloadCreationEvent::STARTED_PARALLEL_DOWNLOAD
-          : ParallelDownloadCreationEvent::FELL_BACK_TO_NORMAL_DOWNLOAD);
-  if (!has_strong_validator) {
-    RecordParallelDownloadCreationEvent(
-        ParallelDownloadCreationEvent::FALLBACK_REASON_STRONG_VALIDATORS);
-  }
-  if (!range_support_allowed) {
-    RecordParallelDownloadCreationEvent(
-        ParallelDownloadCreationEvent::FALLBACK_REASON_ACCEPT_RANGE_HEADER);
-    if (create_info.accept_range == RangeRequestSupportType::kUnknown) {
-      RecordParallelDownloadCreationEvent(
-          ParallelDownloadCreationEvent::FALLBACK_REASON_UNKNOWN_RANGE_SUPPORT);
-    }
-  }
-  if (!has_content_length) {
-    RecordParallelDownloadCreationEvent(
-        ParallelDownloadCreationEvent::FALLBACK_REASON_CONTENT_LENGTH_HEADER);
-  }
-  if (!satisfy_min_file_size) {
-    RecordParallelDownloadCreationEvent(
-        ParallelDownloadCreationEvent::FALLBACK_REASON_FILE_SIZE);
-  }
-  if (!satisfy_connection_type) {
-    RecordParallelDownloadCreationEvent(
-        ParallelDownloadCreationEvent::FALLBACK_REASON_CONNECTION_TYPE);
-  }
-  if (!http_get_method) {
-    RecordParallelDownloadCreationEvent(
-        ParallelDownloadCreationEvent::FALLBACK_REASON_HTTP_METHOD);
-  }
-  if (!can_support_parallel_requests) {
-    RecordParallelDownloadCreationEvent(
-        ParallelDownloadCreationEvent::
-            FALLBACK_REASON_RESUMPTION_WITHOUT_SLICES);
-  }
   return is_parallelizable;
 }
 
