diff --git a/components/BUILD.gn b/components/BUILD.gn
index 020764dfebf9..29b70e95dde0 100644
--- a/components/BUILD.gn
+++ b/components/BUILD.gn
@@ -127,7 +127,6 @@ test("components_unittests") {
     "//components/previews/core:unit_tests",
     "//components/proxy_config:unit_tests",
     "//components/query_parser:unit_tests",
-    "//components/rappor:unit_tests",
     "//components/reading_list/core:unit_tests",
     "//components/safe_search_api:unit_tests",
     "//components/scheduling_metrics:unit_tests",
