diff --git a/components/offline_pages/core/prefetch/stub_prefetch_service.cc b/components/offline_pages/core/prefetch/stub_prefetch_service.cc
index 4030c674fe335..76b6dc864b8d6 100644
--- a/components/offline_pages/core/prefetch/stub_prefetch_service.cc
+++ b/components/offline_pages/core/prefetch/stub_prefetch_service.cc
@@ -29,10 +29,6 @@ OfflineEventLogger* StubPrefetchService::GetLogger() {
   return nullptr;
 }
 
-OfflineMetricsCollector* StubPrefetchService::GetOfflineMetricsCollector() {
-  return nullptr;
-}
-
 PrefetchDispatcher* StubPrefetchService::GetPrefetchDispatcher() {
   return nullptr;
 }
