diff --git a/components/renderer_context_menu/render_view_context_menu_proxy.h b/components/renderer_context_menu/render_view_context_menu_proxy.h
index 1f8a9ea18353..b092f629a19d 100644
--- a/components/renderer_context_menu/render_view_context_menu_proxy.h
+++ b/components/renderer_context_menu/render_view_context_menu_proxy.h
@@ -107,9 +107,6 @@ class RenderViewContextMenuProxy {
   // Removes separators so that any adjacent duplicates are reduced to only 1.
   virtual void RemoveAdjacentSeparators() = 0;
 
-  // Add spell check service item to the context menu.
-  virtual void AddSpellCheckServiceItem(bool is_checked) = 0;
-
   // Add accessibility labels service item to the context menu.
   virtual void AddAccessibilityLabelsServiceItem(bool is_checked) = 0;
 
