diff --git a/components/ntp_snippets/BUILD.gn b/components/ntp_snippets/BUILD.gn
index 64b1abd65b57..56e06a75e1ed 100644
--- a/components/ntp_snippets/BUILD.gn
+++ b/components/ntp_snippets/BUILD.gn
@@ -108,7 +108,6 @@ static_library("ntp_snippets") {
     "//components/variations/net",
     "//components/variations/service",
     "//components/web_resource",
-    "//services/metrics/public/cpp:metrics_cpp",
     "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/cpp",
     "//services/network/public/mojom",
