diff --git a/components/password_manager/core/browser/browser_save_password_progress_logger.cc b/components/password_manager/core/browser/browser_save_password_progress_logger.cc
index e6037b494ed9..3d57ad478d23 100644
--- a/components/password_manager/core/browser/browser_save_password_progress_logger.cc
+++ b/components/password_manager/core/browser/browser_save_password_progress_logger.cc
@@ -88,7 +88,7 @@ std::string FormSignatureToDebugString(autofill::FormSignature form_signature) {
   return base::StrCat(
       {NumberToString(form_signature.value()), " - ",
        NumberToString(
-           PasswordFormMetricsRecorder::HashFormSignature(form_signature))});
+           static_cast<uint64_t>(form_signature) % 1021)});
 }
 
 BrowserSavePasswordProgressLogger::StringID FormSchemeToStringID(
