diff --git a/components/metal_util/test_shader.mm b/components/metal_util/test_shader.mm
index a9c265748186..41eea4d93bc0 100644
--- a/components/metal_util/test_shader.mm
+++ b/components/metal_util/test_shader.mm
@@ -18,7 +18,6 @@
 #include "base/synchronization/lock.h"
 #include "base/task/post_task.h"
 #include "base/task/thread_pool.h"
-#include "components/crash/core/common/crash_key.h"
 #include "components/metal_util/device.h"
 #include "components/metal_util/switches.h"
 
@@ -659,10 +658,6 @@ const size_t kTestLibHashOffset = 0x104;
 const size_t kTestLibBitcodeOffset = 0xd16;
 const size_t kTestLibBitcodeSize = 0xa50;
 
-crash_reporter::CrashKeyString<32>& GetLinkIndexCrashKey() {
-  static crash_reporter::CrashKeyString<32> crash_key("metal-link-index");
-  return crash_key;
-}
 
 }  // namespace
 
@@ -682,7 +677,6 @@ std::vector<uint8_t> GetAlteredLibraryData() {
 
   // Shader link crashes have been observed on bots. Record what index caused
   // them if they appear in the wild.
-  GetLinkIndexCrashKey().Set(base::StringPrintf("%llu", index));
   return data;
 }
 
@@ -775,11 +769,6 @@ class TestShaderState : public base::RefCountedThreadSafe<TestShaderState> {
                                compile_time);
     }
     if (closure) {
-      // If this is a link that has completed, then we set the crash key when
-      // constructing the shader binary. Clear it now, because Metal didn't
-      // crash while linking the shader.
-      if (component_ == TestShaderComponent::kLink)
-        GetLinkIndexCrashKey().Clear();
       std::move(closure).Run();
     }
   }
