diff --git a/components/renderer_context_menu/render_view_context_menu_base.h b/components/renderer_context_menu/render_view_context_menu_base.h
index 2dca58a6553e..1d45d1cb0cd8 100644
--- a/components/renderer_context_menu/render_view_context_menu_base.h
+++ b/components/renderer_context_menu/render_view_context_menu_base.h
@@ -140,12 +140,6 @@ class RenderViewContextMenuBase : public ui::SimpleMenuModel::Delegate,
   // Menu Construction.
   virtual void InitMenu();
 
-  // Increments histogram value for used items specified by |id|.
-  virtual void RecordUsedItem(int id) = 0;
-
-  // Increments histogram value for visible context menu item specified by |id|.
-  virtual void RecordShownItem(int id) = 0;
-
 #if BUILDFLAG(ENABLE_PLUGINS)
   virtual void HandleAuthorizeAllPlugins() = 0;
 #endif
