diff --git a/components/enterprise/common/proto/connectors.proto b/components/enterprise/common/proto/connectors.proto
index a49072adcce6..d5a1cf26f138 100644
--- a/components/enterprise/common/proto/connectors.proto
+++ b/components/enterprise/common/proto/connectors.proto
@@ -8,9 +8,6 @@ option optimize_for = LITE_RUNTIME;
 
 package enterprise_connectors;
 
-// For ClientDownloadRequest.
-import "components/safe_browsing/core/proto/csd.proto";
-
 // Which connector is calling BinaryUploadService so that the proper rules can
 // be triggered.  BinaryUploadService also uses this value to find the URL that
 // the payload should be uploaded to.
@@ -35,9 +32,6 @@ message ContentMetaData {
   // Sha256 digest of file.
   optional string digest = 3;
 
-  // Specifically for the download case.
-  optional safe_browsing.ClientDownloadRequest csd = 4;
-
   // Optional email address of user.  This field may be empty if the user
   // is not signed in.
   optional string email = 5;
