diff --git a/components/payments/content/payment_request_state.h b/components/payments/content/payment_request_state.h
index d277d415b20af..2eb646c41544b 100644
--- a/components/payments/content/payment_request_state.h
+++ b/components/payments/content/payment_request_state.h
@@ -19,7 +19,6 @@
 #include "components/payments/content/payment_response_helper.h"
 #include "components/payments/content/service_worker_payment_app.h"
 #include "components/payments/content/service_worker_payment_app_factory.h"
-#include "components/payments/core/journey_logger.h"
 #include "components/payments/core/payments_profile_comparator.h"
 #include "content/public/browser/global_routing_id.h"
 #include "content/public/browser/payment_app_provider.h"
@@ -106,8 +105,7 @@ class PaymentRequestState : public PaymentAppFactory::Delegate,
                       base::WeakPtr<Delegate> delegate,
                       const std::string& app_locale,
                       autofill::PersonalDataManager* personal_data_manager,
-                      ContentPaymentRequestDelegate* payment_request_delegate,
-                      JourneyLogger* journey_logger);
+                      ContentPaymentRequestDelegate* payment_request_delegate);
   ~PaymentRequestState() override;
 
   // PaymentAppFactory::Delegate
@@ -372,7 +370,6 @@ class PaymentRequestState : public PaymentAppFactory::Delegate,
 
   // Not owned. Never null. Will outlive this object.
   autofill::PersonalDataManager* personal_data_manager_;
-  JourneyLogger* journey_logger_;
 
   StatusCallback can_make_payment_callback_;
   StatusCallback has_enrolled_instrument_callback_;
