diff --git a/components/autofill/core/browser/ui/label_formatter.cc b/components/autofill/core/browser/ui/label_formatter.cc
index 5f87d687d7b1..107c76915f90 100644
--- a/components/autofill/core/browser/ui/label_formatter.cc
+++ b/components/autofill/core/browser/ui/label_formatter.cc
@@ -10,7 +10,6 @@
 
 #include "build/build_config.h"
 #include "components/autofill/core/browser/autofill_data_util.h"
-#include "components/autofill/core/browser/autofill_metrics.h"
 #include "components/autofill/core/browser/ui/address_contact_form_label_formatter.h"
 #include "components/autofill/core/browser/ui/address_email_form_label_formatter.h"
 #include "components/autofill/core/browser/ui/address_form_label_formatter.h"
