diff --git a/components/safe_browsing/core/db/v4_get_hash_protocol_manager.cc b/components/safe_browsing/core/db/v4_get_hash_protocol_manager.cc
index 81a4d3aee4e3..b46d676dc9cd 100644
--- a/components/safe_browsing/core/db/v4_get_hash_protocol_manager.cc
+++ b/components/safe_browsing/core/db/v4_get_hash_protocol_manager.cc
@@ -14,8 +14,6 @@
 #include "base/stl_util.h"
 #include "base/strings/string_split.h"
 #include "base/timer/timer.h"
-#include "base/trace_event/trace_event.h"
-#include "base/trace_event/traced_value.h"
 #include "components/safe_browsing/core/common/thread_utils.h"
 #include "components/safe_browsing/core/features.h"
 #include "net/base/load_flags.h"
@@ -610,9 +608,6 @@ bool V4GetHashProtocolManager::ParseHashResponse(
     FullHashInfo full_hash_info(match.threat().hash(), list_id,
                                 positive_expiry);
     ParseMetadata(match, &full_hash_info.metadata);
-    TRACE_EVENT2("safe_browsing", "V4GetHashProtocolManager::ParseHashResponse",
-                 "threat_type", full_hash_info.list_id.threat_type(),
-                 "metadata", full_hash_info.metadata.ToTracedValue());
     full_hash_infos->push_back(full_hash_info);
   }
   return true;
