diff --git a/components/download/internal/common/parallel_download_job.cc b/components/download/internal/common/parallel_download_job.cc
index 6c53b1be9badf..8cf226c8afb9f 100644
--- a/components/download/internal/common/parallel_download_job.cc
+++ b/components/download/internal/common/parallel_download_job.cc
@@ -11,7 +11,6 @@
 #include "base/time/time.h"
 #include "components/download/internal/common/parallel_download_utils.h"
 #include "components/download/public/common/download_create_info.h"
-#include "components/download/public/common/download_stats.h"
 #include "mojo/public/cpp/bindings/pending_remote.h"
 #include "net/traffic_annotation/network_traffic_annotation.h"
 #include "net/url_request/referrer_policy.h"
@@ -200,8 +199,6 @@ void ParallelDownloadJob::BuildParallelRequests() {
     return;
 
   ForkSubRequests(slices_to_download);
-  RecordParallelDownloadRequestCount(
-      static_cast<int>(slices_to_download.size()));
   requests_sent_ = true;
 }
 
