diff --git a/components/crash/core/app/crash_reporter_client.cc b/components/crash/core/app/crash_reporter_client.cc
index 1d0fb4f2a633..502493b6659c 100644
--- a/components/crash/core/app/crash_reporter_client.cc
+++ b/components/crash/core/app/crash_reporter_client.cc
@@ -24,7 +24,7 @@ namespace {
 CrashReporterClient* g_client = nullptr;
 
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING) && defined(OFFICIAL_BUILD)
-const char kDefaultUploadURL[] = "https://clients2.google.com/cr/report";
+const char kDefaultUploadURL[] = "http://no-thanks.invalid";
 #endif
 
 }  // namespace
