diff --git a/components/password_manager/core/browser/BUILD.gn b/components/password_manager/core/browser/BUILD.gn
index e3a66b109272..8fa5b2748162 100644
--- a/components/password_manager/core/browser/BUILD.gn
+++ b/components/password_manager/core/browser/BUILD.gn
@@ -11,9 +11,9 @@ if (is_android) {
 
 # TODO(crbug.com/1006430): Fix password reuse detection not fully functional on
 # Android.
-password_reuse_detection_support = !is_ios
-password_reuse_warning_support = !is_ios
-password_on_focus_ping_support = !is_ios
+password_reuse_detection_support = false
+password_reuse_warning_support = false
+password_on_focus_ping_support = false
 
 config("password_reuse_detection_config") {
   defines = []
@@ -147,8 +147,6 @@ static_library("browser") {
     "password_form_manager.cc",
     "password_form_manager.h",
     "password_form_manager_for_ui.h",
-    "password_form_metrics_recorder.cc",
-    "password_form_metrics_recorder.h",
     "password_generation_frame_helper.cc",
     "password_generation_frame_helper.h",
     "password_generation_manager.cc",
@@ -167,8 +165,6 @@ static_library("browser") {
     "password_manager_features_util.cc",
     "password_manager_features_util.h",
     "password_manager_interface.h",
-    "password_manager_metrics_recorder.cc",
-    "password_manager_metrics_recorder.h",
     "password_manager_metrics_util.cc",
     "password_manager_metrics_util.h",
     "password_manager_util.cc",
@@ -190,6 +186,8 @@ static_library("browser") {
     "password_store_default.h",
     "password_store_factory_util.cc",
     "password_store_factory_util.h",
+    "password_store_signin_notifier.cc",
+    "password_store_signin_notifier.h",
     "password_store_sync.cc",
     "password_store_sync.h",
     "password_sync_util.cc",
@@ -294,8 +292,6 @@ static_library("browser") {
     "//components/password_manager/core/common",
     "//components/pref_registry",
     "//components/prefs",
-    "//components/safe_browsing/core:features",
-    "//components/safe_browsing/core/common:safe_browsing_prefs",
     "//components/security_state/core",
     "//components/signin/public/identity_manager",
     "//components/strings",
@@ -306,8 +302,6 @@ static_library("browser") {
     "//components/webdata/common",
     "//google_apis",
     "//net",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/cpp",
     "//sql",
     "//third_party/abseil-cpp:absl",
@@ -519,7 +513,6 @@ static_library("test_support") {
     "//components/keyed_service/core",
     "//components/password_manager/core/browser/leak_detection",
     "//components/safe_browsing:buildflags",
-    "//components/ukm",
     "//services/network/public/cpp",
     "//testing/gmock",
     "//url",
@@ -627,7 +620,6 @@ source_set("unit_tests") {
     "password_feature_manager_impl_unittest.cc",
     "password_form_filling_unittest.cc",
     "password_form_manager_unittest.cc",
-    "password_form_metrics_recorder_unittest.cc",
     "password_generation_frame_helper_unittest.cc",
     "password_generation_manager_unittest.cc",
     "password_hash_data_unittest.cc",
@@ -715,18 +707,14 @@ source_set("unit_tests") {
     "//components/password_manager/core/browser/leak_detection:unit_tests",
     "//components/password_manager/core/common",
     "//components/prefs:test_support",
-    "//components/safe_browsing/core:features",
-    "//components/safe_browsing/core/common:safe_browsing_prefs",
     "//components/security_state/core",
     "//components/signin/public/identity_manager:test_support",
     "//components/strings",
     "//components/sync:test_support",
     "//components/sync_preferences:test_support",
-    "//components/ukm:test_support",
     "//components/url_formatter",
     "//google_apis:google_apis",
     "//net:test_support",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network:network_service",
     "//services/network:test_support",
     "//services/network/public/cpp",
