diff --git a/components/security_interstitials/content/cert_report_helper.h b/components/security_interstitials/content/cert_report_helper.h
index 41a72e96a7110..0fa0ef699eae7 100644
--- a/components/security_interstitials/content/cert_report_helper.h
+++ b/components/security_interstitials/content/cert_report_helper.h
@@ -110,11 +110,13 @@ class CertReportHelper {
   const GURL request_url_;
   // The SSLInfo used in this helper's report.
   const net::SSLInfo ssl_info_;
+#if 0
   // The reason for the interstitial, included in this helper's report.
   CertificateErrorReport::InterstitialReason interstitial_reason_;
   // True if the user was given the option to proceed through the
   // certificate chain error being reported.
   bool overridable_;
+#endif
   // The time at which the interstitial was constructed.
   const base::Time interstitial_time_;
   // Whether to show enhanced protection message in the interstitial. If it is
@@ -124,8 +126,10 @@ class CertReportHelper {
   // true in other platforms. Remove this member once WebLayer supports enhanced
   // protection.
   bool can_show_enhanced_protection_message_;
+#if 0
   // Helpful for recording metrics about cert reports.
   security_interstitials::MetricsHelper* metrics_helper_;
+#endif
   // Appends additional details to a report.
   ClientDetailsCallback client_details_callback_;
   // Default to DID_NOT_PROCEED. If no user action is processed via
