diff --git a/components/permissions/permission_request_manager.cc b/components/permissions/permission_request_manager.cc
index c4c308f0349ea..66b56a4a0a998 100644
--- a/components/permissions/permission_request_manager.cc
+++ b/components/permissions/permission_request_manager.cc
@@ -624,11 +624,6 @@ void PermissionRequestManager::FinalizeCurrentRequests(
       permission_action != PermissionAction::IGNORED) {
     time_to_decision = base::Time::Now() - current_request_first_display_time_;
   }
-  PermissionUmaUtil::PermissionPromptResolved(
-      requests_, web_contents(), permission_action, time_to_decision,
-      DetermineCurrentRequestUIDispositionForUMA(),
-      DetermineCurrentRequestUIDispositionReasonForUMA(),
-      prediction_grant_likelihood_);
 
   content::BrowserContext* browser_context =
       web_contents()->GetBrowserContext();
@@ -666,7 +661,6 @@ void PermissionRequestManager::FinalizeCurrentRequests(
         embargo_status = PermissionEmbargoStatus::REPEATED_IGNORES;
       }
     }
-    PermissionUmaUtil::RecordEmbargoStatus(embargo_status);
   }
   ResetViewStateForCurrentRequest();
   std::vector<PermissionRequest*>::iterator requests_iter;
