diff --git a/components/history/core/browser/history_backend.cc b/components/history/core/browser/history_backend.cc
index 711eb3ffa602..c16769b1d366 100644
--- a/components/history/core/browser/history_backend.cc
+++ b/components/history/core/browser/history_backend.cc
@@ -150,10 +150,6 @@ const int kCommitIntervalSeconds = 10;
 // deleting some.
 const int kMaxRedirectCount = 32;
 
-// The number of days old a history entry can be before it is considered "old"
-// and is deleted.
-const int kExpireDaysThreshold = 90;
-
 // The maximum number of days for which domain visit metrics are computed
 // each time HistoryBackend::GetDomainDiversity() is called.
 constexpr int kDomainDiversityMaxBacktrackedDays = 7;
@@ -878,9 +874,6 @@ void HistoryBackend::InitImpl(
   // Get the first item in our database.
   db_->GetStartDate(&first_recorded_time_);
 
-  // Start expiring old stuff.
-  expirer_.StartExpiringOldStuff(TimeDelta::FromDays(kExpireDaysThreshold));
-
 #if defined(OS_ANDROID)
   if (backend_client_) {
     backend_client_->OnHistoryBackendInitialized(this, db_.get(),
