diff --git a/components/download/database/in_progress/download_entry.h b/components/download/database/in_progress/download_entry.h
index a68a0b6e08a81..1a6ddb7195582 100644
--- a/components/download/database/in_progress/download_entry.h
+++ b/components/download/database/in_progress/download_entry.h
@@ -10,7 +10,6 @@
 #include "components/download/public/common/download_item.h"
 #include "components/download/public/common/download_source.h"
 #include "components/download/public/common/download_url_parameters.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 
 namespace download {
 
@@ -22,10 +21,8 @@ struct DownloadEntry {
   DownloadEntry(
       const std::string& guid,
       const std::string& request_origin,
-      DownloadSource download_source,
       bool fetch_error_body,
-      const DownloadUrlParameters::RequestHeadersType& request_headers,
-      int64_t ukm_id);
+      const DownloadUrlParameters::RequestHeadersType& request_headers);
   ~DownloadEntry();
 
   bool operator==(const DownloadEntry& other) const;
@@ -38,13 +35,6 @@ struct DownloadEntry {
   // Represents the origin information for this download. Used by offline pages.
   std::string request_origin;
 
-  // The source that triggered the download.
-  DownloadSource download_source = DownloadSource::UNKNOWN;
-
-  // Unique ID that tracks the download UKM entry, where 0 means the
-  // download_id is not yet initialized.
-  uint64_t ukm_download_id = 0;
-
   // Count for how many (extra) bytes were used (including resumption).
   int64_t bytes_wasted = 0;
 
