diff --git a/components/javascript_dialogs/BUILD.gn b/components/javascript_dialogs/BUILD.gn
index 94034255947c..5eddc8bcac73 100644
--- a/components/javascript_dialogs/BUILD.gn
+++ b/components/javascript_dialogs/BUILD.gn
@@ -25,12 +25,10 @@ static_library("javascript_dialogs") {
     "//base:i18n",
     "//components/navigation_metrics/",
     "//components/strings",
-    "//components/ukm/content",
     "//components/url_formatter",
     "//content/public/browser",
     "//content/public/common",
     "//net",
-    "//services/metrics/public/cpp:metrics_cpp",
     "//services/metrics/public/cpp:ukm_builders",
     "//skia",
     "//ui/base",
