diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index 2faa3caadfea1..fd8a2aea21adf 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -86,7 +86,6 @@ source_set("permissions") {
     "//components/permissions/prediction_service",
     "//components/permissions/vector_icons",
     "//components/strings:components_strings_grit",
-    "//components/ukm/content",
     "//components/url_formatter",
     "//components/variations",
     "//components/vector_icons",
@@ -148,7 +147,6 @@ source_set("test_support") {
     "//base",
     "//components/content_settings/core/browser",
     "//components/sync_preferences:test_support",
-    "//components/ukm/content",
     "//components/vector_icons",
     "//content/public/browser",
     "//testing/gmock",
@@ -186,8 +184,6 @@ source_set("unit_tests") {
     "//components/content_settings/core/browser",
     "//components/keyed_service/content",
     "//components/prefs:test_support",
-    "//components/ukm:test_support",
-    "//components/ukm/content",
     "//components/variations",
     "//content/test:test_support",
     "//sql",
