diff --git a/components/download/internal/common/download_item_impl_delegate.cc b/components/download/internal/common/download_item_impl_delegate.cc
index 92c931b6448d7..b4975e09ff46b 100644
--- a/components/download/internal/common/download_item_impl_delegate.cc
+++ b/components/download/internal/common/download_item_impl_delegate.cc
@@ -106,10 +106,12 @@ void DownloadItemImplDelegate::ReportBytesWasted(DownloadItemImpl* download) {}
 void DownloadItemImplDelegate::BindWakeLockProvider(
     mojo::PendingReceiver<device::mojom::WakeLockProvider> receiver) {}
 
+#if 0
 QuarantineConnectionCallback
 DownloadItemImplDelegate::GetQuarantineConnectionCallback() {
   return base::NullCallback();
 }
+#endif
 
 std::unique_ptr<DownloadItemRenameHandler>
 DownloadItemImplDelegate::GetRenameHandlerForDownload(
