diff --git a/components/lookalikes/core/BUILD.gn b/components/lookalikes/core/BUILD.gn
index 30aab561767c..9970685fe270 100644
--- a/components/lookalikes/core/BUILD.gn
+++ b/components/lookalikes/core/BUILD.gn
@@ -22,8 +22,6 @@ static_library("core") {
     "//components/url_formatter/spoof_checks/top_domains:top500_domains",
     "//components/url_formatter/spoof_checks/top_domains:top500_domains_header",
     "//net",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//ui/base",
   ]
 }
