diff --git a/components/offline_pages/core/prefetch/stub_prefetch_service.h b/components/offline_pages/core/prefetch/stub_prefetch_service.h
index 1454cc81ba0b..1263f9628dbf 100644
--- a/components/offline_pages/core/prefetch/stub_prefetch_service.h
+++ b/components/offline_pages/core/prefetch/stub_prefetch_service.h
@@ -22,7 +22,6 @@ class StubPrefetchService : public PrefetchService {
   void ForceRefreshSuggestions() override;
   PrefetchGCMHandler* GetPrefetchGCMHandler() override;
   OfflineEventLogger* GetLogger() override;
-  OfflineMetricsCollector* GetOfflineMetricsCollector() override;
   PrefetchDispatcher* GetPrefetchDispatcher() override;
   PrefetchNetworkRequestFactory* GetPrefetchNetworkRequestFactory() override;
   PrefetchDownloader* GetPrefetchDownloader() override;
