diff --git a/components/search_engines/BUILD.gn b/components/search_engines/BUILD.gn
index c2b09f071782..95d403b051a7 100644
--- a/components/search_engines/BUILD.gn
+++ b/components/search_engines/BUILD.gn
@@ -59,7 +59,6 @@ static_library("search_engines") {
     "//components/prefs",
     "//components/sync",
     "//components/webdata/common",
-    "//third_party/metrics_proto",
   ]
 
   deps = [
@@ -81,7 +80,6 @@ static_library("search_engines") {
     "//services/network/public/cpp",
     "//services/network/public/mojom",
     "//sql",
-    "//third_party/metrics_proto",
     "//ui/base",
     "//ui/gfx",
     "//ui/gfx/geometry",
