diff --git a/components/translate/content/browser/per_frame_content_translate_driver.h b/components/translate/content/browser/per_frame_content_translate_driver.h
index a9b631bdce7f..2dddea5b4f11 100644
--- a/components/translate/content/browser/per_frame_content_translate_driver.h
+++ b/components/translate/content/browser/per_frame_content_translate_driver.h
@@ -19,7 +19,6 @@
 #include "mojo/public/cpp/bindings/pending_associated_remote.h"
 #include "mojo/public/cpp/bindings/pending_receiver.h"
 #include "mojo/public/cpp/bindings/receiver_set.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "third_party/blink/public/common/associated_interfaces/associated_interface_provider.h"
 
 namespace content {
