diff --git a/components/payments/core/BUILD.gn b/components/payments/core/BUILD.gn
index cb46c8bed0d7..3ade745d66c1 100644
--- a/components/payments/core/BUILD.gn
+++ b/components/payments/core/BUILD.gn
@@ -24,8 +24,6 @@ static_library("core") {
     "error_message_util.h",
     "features.cc",
     "features.h",
-    "journey_logger.cc",
-    "journey_logger.h",
     "payer_data.cc",
     "payer_data.h",
     "payment_address.cc",
@@ -91,8 +89,6 @@ static_library("core") {
     "//components/pref_registry",
     "//components/strings:components_strings_grit",
     "//net",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/cpp",
     "//third_party/re2",
     "//ui/base",
@@ -195,10 +191,7 @@ source_set("unit_tests") {
     "//components/autofill/core/browser:test_support",
     "//components/leveldb_proto:leveldb_proto",
     "//components/strings:components_strings_grit",
-    "//components/ukm",
-    "//components/ukm:test_support",
     "//net:test_support",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network:test_support",
     "//services/network/public/cpp",
     "//testing/gmock",
