diff --git a/components/omnibox/browser/omnibox_metrics_provider.cc b/components/omnibox/browser/omnibox_metrics_provider.cc
index 6c838baace937..4e672ebbfbc64 100644
--- a/components/omnibox/browser/omnibox_metrics_provider.cc
+++ b/components/omnibox/browser/omnibox_metrics_provider.cc
@@ -51,6 +51,7 @@ void OmniboxMetricsProvider::OnURLOpenedFromOmnibox(OmniboxLog* log) {
 }
 
 void OmniboxMetricsProvider::RecordOmniboxOpenedURL(const OmniboxLog& log) {
+#if 0
   std::vector<base::StringPiece16> terms = base::SplitStringPiece(
       log.text, base::kWhitespaceUTF16,
       base::KEEP_WHITESPACE, base::SPLIT_WANT_NONEMPTY);
@@ -123,4 +124,5 @@ void OmniboxMetricsProvider::RecordOmniboxOpenedURL(const OmniboxLog& log) {
     omnibox_event->add_feature_triggered_in_session(
         static_cast<size_t>(feature));
   }
+#endif
 }
