diff --git a/components/download/internal/common/BUILD.gn b/components/download/internal/common/BUILD.gn
index b2e26585bbe8..0a5a8f7830e5 100644
--- a/components/download/internal/common/BUILD.gn
+++ b/components/download/internal/common/BUILD.gn
@@ -47,7 +47,6 @@ source_set("internal") {
     "download_response_handler.cc",
     "download_stats.cc",
     "download_task_runner.cc",
-    "download_ukm_helper.cc",
     "download_utils.cc",
     "download_worker.cc",
     "download_worker.h",
@@ -86,7 +85,6 @@ source_set("internal") {
     "//mojo/public/c/system",
     "//net",
     "//services/device/public/mojom:mojom",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/cpp",
     "//services/service_manager/public/cpp:cpp",
   ]
@@ -162,7 +160,6 @@ source_set("unit_tests") {
     "download_item_impl_unittest.cc",
     "download_path_reservation_tracker_unittest.cc",
     "download_stats_unittest.cc",
-    "download_ukm_helper_unittest.cc",
     "parallel_download_job_unittest.cc",
     "parallel_download_utils_unittest.cc",
     "rate_estimator_unittest.cc",
@@ -175,10 +172,8 @@ source_set("unit_tests") {
     "//components/download/public/common:test_support",
     "//components/leveldb_proto",
     "//components/leveldb_proto:test_support",
-    "//components/ukm:test_support",
     "//crypto",
     "//net",
-    "//services/metrics/public/cpp:ukm_builders",
     "//testing/gmock",
     "//testing/gtest",
   ]
