diff --git a/components/arc/BUILD.gn b/components/arc/BUILD.gn
index 0619879c2b94..8e175f996b25 100644
--- a/components/arc/BUILD.gn
+++ b/components/arc/BUILD.gn
@@ -116,7 +116,6 @@ static_library("arc") {
     "//chromeos/login/login_state",
     "//chromeos/network",
     "//components/account_id",
-    "//components/arc/enterprise",
     "//components/exo",
     "//components/google/core/common",
     "//components/guest_os",
@@ -268,7 +267,6 @@ static_library("arc_base") {
     "//chromeos/memory:memory",
     "//chromeos/system:system",
     "//components/account_id",
-    "//components/arc/enterprise",
     "//components/keyed_service/content",
     "//components/prefs",
     "//components/user_manager",
@@ -371,7 +369,6 @@ static_library("arc_test_support") {
 
   deps = [
     "//base",
-    "//components/arc/enterprise",
     "//components/keyed_service/content",
     "//components/prefs:test_support",
     "//components/user_prefs",
