diff --git a/components/security_interstitials/content/origin_policy_interstitial_page.cc b/components/security_interstitials/content/origin_policy_interstitial_page.cc
index 3a7c53639d277..cc3cfd41b93d1 100644
--- a/components/security_interstitials/content/origin_policy_interstitial_page.cc
+++ b/components/security_interstitials/content/origin_policy_interstitial_page.cc
@@ -67,9 +67,6 @@ void OriginPolicyInterstitialPage::PopulateInterstitialStrings(
       {"finalParagraph", IDS_ORIGIN_POLICY_FINAL_PARAGRAPH},
       {"heading", IDS_ORIGIN_POLICY_HEADING},
       {"openDetails", IDS_ORIGIN_POLICY_DETAILS},
-      {"optInLink", IDS_SAFE_BROWSING_SCOUT_REPORTING_AGREE},
-      {"enhancedProtectionMessage",
-       IDS_SAFE_BROWSING_ENHANCED_PROTECTION_MESSAGE},
       {"primaryButtonText", IDS_ORIGIN_POLICY_BUTTON},
       {"primaryParagraph", IDS_ORIGIN_POLICY_INFO},
       {"recurrentErrorParagraph", IDS_ORIGIN_POLICY_INFO2},
