diff --git a/components/remote_cocoa/app_shim/BUILD.gn b/components/remote_cocoa/app_shim/BUILD.gn
index 80269ea657be..328b33aef037 100644
--- a/components/remote_cocoa/app_shim/BUILD.gn
+++ b/components/remote_cocoa/app_shim/BUILD.gn
@@ -56,7 +56,6 @@ component("app_shim") {
   deps = [
     "//base",
     "//base:i18n",
-    "//components/crash/core/common",
     "//components/remote_cocoa/common:mojo",
     "//mojo/public/cpp/bindings",
     "//ui/accelerated_widget_mac",
