diff --git a/components/background_sync/background_sync_delegate.h b/components/background_sync/background_sync_delegate.h
index 5a4566b8cdb61..8e00918d8718e 100644
--- a/components/background_sync/background_sync_delegate.h
+++ b/components/background_sync/background_sync_delegate.h
@@ -36,12 +36,6 @@ class BackgroundSyncDelegate {
   CreateBackgroundSyncEventKeepAlive() = 0;
 #endif
 
-  // Gets the source_ID to log the UKM event for, and calls |callback| with that
-  // source_id, or with base::nullopt if UKM recording is not allowed.
-  virtual void GetUkmSourceId(
-      const url::Origin& origin,
-      base::OnceCallback<void(base::Optional<ukm::SourceId>)> callback) = 0;
-
   // Handles browser shutdown.
   virtual void Shutdown() = 0;
 
