diff --git a/components/ntp_tiles/BUILD.gn b/components/ntp_tiles/BUILD.gn
index b54019299876..8e204039cebe 100644
--- a/components/ntp_tiles/BUILD.gn
+++ b/components/ntp_tiles/BUILD.gn
@@ -58,7 +58,6 @@ static_library("ntp_tiles") {
     "//components/image_fetcher/core",
     "//components/pref_registry",
     "//components/prefs",
-    "//components/rappor/public",
     "//components/resources",
     "//components/search",
     "//components/search_engines",
@@ -101,7 +100,6 @@ source_set("unit_tests") {
     "//components/image_fetcher/core",
     "//components/image_fetcher/core:test_support",
     "//components/pref_registry:pref_registry",
-    "//components/rappor:test_support",
     "//components/sync_preferences:test_support",
     "//net:test_support",
     "//services/data_decoder/public/cpp:test_support",
