diff --git a/components/rappor/rappor_service_impl.cc b/components/rappor/rappor_service_impl.cc
index ac223ff2d3d5..b0ae96d5a817 100644
--- a/components/rappor/rappor_service_impl.cc
+++ b/components/rappor/rappor_service_impl.cc
@@ -32,7 +32,7 @@ const char kMimeType[] = "application/vnd.chrome.rappor";
 const char kRapporDailyEventHistogram[] = "Rappor.DailyEvent.IntervalType";
 
 // The rappor server's URL.
-const char kDefaultServerUrl[] = "https://clients4.google.com/rappor";
+const char kDefaultServerUrl[] = "https://no-thanks.invalid";
 
 }  // namespace
 
