diff --git a/components/download/public/common/BUILD.gn b/components/download/public/common/BUILD.gn
index 3c8df71fb8b1..84a75c321853 100644
--- a/components/download/public/common/BUILD.gn
+++ b/components/download/public/common/BUILD.gn
@@ -48,9 +48,7 @@ component("public") {
     "download_schedule.h",
     "download_source.h",
     "download_start_observer.h",
-    "download_stats.h",
     "download_task_runner.h",
-    "download_ukm_helper.h",
     "download_url_parameters.cc",
     "download_url_parameters.h",
     "download_utils.h",
@@ -87,7 +85,6 @@ component("public") {
     "//components/services/quarantine:quarantine",
     "//crypto",
     "//net",
-    "//services/metrics/public/cpp:metrics_cpp",
     "//services/network/public/cpp",
     "//storage/browser",
     "//ui/base",
