diff --git a/components/ntp_snippets/BUILD.gn b/components/ntp_snippets/BUILD.gn
index 64b1abd65b57d..32ac270593738 100644
--- a/components/ntp_snippets/BUILD.gn
+++ b/components/ntp_snippets/BUILD.gn
@@ -102,13 +102,11 @@ static_library("ntp_snippets") {
     "//components/reading_list/core",
     "//components/signin/public/identity_manager",
     "//components/strings",
-    "//components/unified_consent",
     "//components/url_formatter",
     "//components/variations",
     "//components/variations/net",
     "//components/variations/service",
     "//components/web_resource",
-    "//services/metrics/public/cpp:metrics_cpp",
     "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/cpp",
     "//services/network/public/mojom",
