diff --git a/components/payments/content/android/BUILD.gn b/components/payments/content/android/BUILD.gn
index 06ec330c48041..e0e0d6d4db342 100644
--- a/components/payments/content/android/BUILD.gn
+++ b/components/payments/content/android/BUILD.gn
@@ -45,7 +45,6 @@ static_library("android") {
     "//components/payments/content:utils",
     "//components/payments/content/utility",
     "//components/payments/core",
-    "//components/ukm/content:content",
     "//content/public/browser",
     "//net",
     "//url:gurl_android",
