diff --git a/components/discardable_memory/service/BUILD.gn b/components/discardable_memory/service/BUILD.gn
index 3763694b4550..9f0175eed9d7 100644
--- a/components/discardable_memory/service/BUILD.gn
+++ b/components/discardable_memory/service/BUILD.gn
@@ -17,7 +17,6 @@ component("service") {
   deps = [
     "//base",
     "//build:chromecast_buildflags",
-    "//components/crash/core/common:crash_key",
     "//components/discardable_memory/common",
     "//components/discardable_memory/public/mojom",
   ]
