diff --git a/components/payments/content/BUILD.gn b/components/payments/content/BUILD.gn
index 382995bdd9898..f35360ffcee09 100644
--- a/components/payments/content/BUILD.gn
+++ b/components/payments/content/BUILD.gn
@@ -73,7 +73,6 @@ static_library("content") {
     "//components/payments/mojom",
     "//components/prefs",
     "//components/strings:components_strings_grit",
-    "//components/ukm/content",
     "//components/url_formatter",
     "//components/webdata/common",
     "//content/public/browser",
@@ -232,7 +231,6 @@ source_set("unit_tests") {
     "//components/webdata/common",
     "//content/test:test_support",
     "//net:test_support",
-    "//services/metrics/public/cpp:metrics_cpp",
     "//services/network:test_support",
     "//sql",
     "//testing/gtest",
