diff --git a/components/omnibox/browser/search_provider.cc b/components/omnibox/browser/search_provider.cc
index 9b63bf673711..ef526db5608a 100644
--- a/components/omnibox/browser/search_provider.cc
+++ b/components/omnibox/browser/search_provider.cc
@@ -1599,11 +1599,6 @@ void SearchProvider::PrefetchImages(SearchSuggestionParser::Results* results) {
       suggestion.answer()->AddImageURLsTo(&prefetch_image_urls);
   }
 
-  UMA_HISTOGRAM_EXACT_LINEAR(
-      "Omnibox.SuggestRequest.Success.PrefetchImagesCount",
-      prefetch_image_urls.size(),
-      AutocompleteResult::kMaxAutocompletePositionValue);
-
   for (const GURL& url : prefetch_image_urls)
     client()->PrefetchImage(url);
 }
