diff --git a/components/gwp_asan/common/BUILD.gn b/components/gwp_asan/common/BUILD.gn
index 993b729862fb..78f12834bb83 100644
--- a/components/gwp_asan/common/BUILD.gn
+++ b/components/gwp_asan/common/BUILD.gn
@@ -8,7 +8,6 @@ static_library("common") {
   sources = [
     "allocator_state.cc",
     "allocator_state.h",
-    "crash_key_name.h",
     "pack_stack_trace.cc",
     "pack_stack_trace.h",
   ]
