diff --git a/components/password_manager/core/browser/mock_password_form_manager_for_ui.h b/components/password_manager/core/browser/mock_password_form_manager_for_ui.h
index ff1fa8718ae1..5169dc79be3d 100644
--- a/components/password_manager/core/browser/mock_password_form_manager_for_ui.h
+++ b/components/password_manager/core/browser/mock_password_form_manager_for_ui.h
@@ -35,7 +35,6 @@ class MockPasswordFormManagerForUI : public PasswordFormManagerForUI {
               GetCredentialSource,
               (),
               (const override));
-  MOCK_METHOD(PasswordFormMetricsRecorder*, GetMetricsRecorder, (), (override));
   MOCK_METHOD(base::span<const InteractionsStats>,
               GetInteractionsStats,
               (),
