diff --git a/components/viz/service/BUILD.gn b/components/viz/service/BUILD.gn
index c93313243875..567e590c842e 100644
--- a/components/viz/service/BUILD.gn
+++ b/components/viz/service/BUILD.gn
@@ -186,7 +186,6 @@ viz_component("service") {
     "//build:chromecast_buildflags",
     "//cc/base",
     "//cc/paint",
-    "//components/crash/core/common:crash_key",
     "//gpu/command_buffer/client:gles2_cmd_helper",
     "//gpu/command_buffer/client:gles2_implementation",
     "//gpu/command_buffer/client:raster",
