diff --git a/components/security_interstitials/content/mitm_software_blocking_page.cc b/components/security_interstitials/content/mitm_software_blocking_page.cc
index 2238540e6f92..54585b6d0bd8 100644
--- a/components/security_interstitials/content/mitm_software_blocking_page.cc
+++ b/components/security_interstitials/content/mitm_software_blocking_page.cc
@@ -6,7 +6,6 @@
 
 #include <utility>
 
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "components/security_interstitials/content/cert_report_helper.h"
 #include "components/security_interstitials/content/security_interstitial_controller_client.h"
 #include "components/security_interstitials/content/security_interstitial_page.h"
