diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index c5c87536d757..abfd79e24a05 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -76,7 +76,6 @@ source_set("permissions") {
     "//components/keyed_service/content",
     "//components/permissions/prediction_service",
     "//components/strings:components_strings_grit",
-    "//components/ukm/content",
     "//components/url_formatter",
     "//components/variations",
     "//components/vector_icons",
@@ -135,7 +134,6 @@ source_set("test_support") {
     "//base",
     "//components/content_settings/core/browser",
     "//components/sync_preferences:test_support",
-    "//components/ukm/content",
     "//components/vector_icons",
     "//content/public/browser",
     "//testing/gmock",
@@ -172,8 +170,6 @@ source_set("unit_tests") {
     "//components/content_settings/core/browser",
     "//components/keyed_service/content",
     "//components/prefs:test_support",
-    "//components/ukm:test_support",
-    "//components/ukm/content",
     "//components/variations",
     "//content/test:test_support",
     "//sql",
