diff --git a/components/crash/content/browser/BUILD.gn b/components/crash/content/browser/BUILD.gn
index efc8041b4133..821597a0bddd 100644
--- a/components/crash/content/browser/BUILD.gn
+++ b/components/crash/content/browser/BUILD.gn
@@ -45,10 +45,6 @@ source_set("browser") {
     ]
   }
 
-  if (!is_chromeos) {
-    deps += [ "//third_party/crashpad/crashpad/client" ]
-  }
-
   if (!is_android) {
     deps += [ "//third_party/breakpad:client" ]
   }
