diff --git a/components/viz/service/main/viz_main_impl.h b/components/viz/service/main/viz_main_impl.h
index 27e0c52cf9b9f..946d65ed601d7 100644
--- a/components/viz/service/main/viz_main_impl.h
+++ b/components/viz/service/main/viz_main_impl.h
@@ -39,10 +39,6 @@ class SyncPointManager;
 class SharedImageManager;
 }  // namespace gpu
 
-namespace ukm {
-class MojoUkmRecorder;
-}
-
 namespace gfx {
 class RenderingPipeline;
 }
@@ -95,7 +91,6 @@ class VizMainImpl : public mojom::VizMain,
     gpu::SharedImageManager* shared_image_manager = nullptr;
     base::WaitableEvent* shutdown_event = nullptr;
     scoped_refptr<base::SingleThreadTaskRunner> io_thread_task_runner;
-    std::unique_ptr<ukm::MojoUkmRecorder> ukm_recorder;
     VizCompositorThreadRunner* viz_compositor_thread_runner = nullptr;
 
    private:
