diff --git a/components/safe_browsing/content/password_protection/password_protection_service.cc b/components/safe_browsing/content/password_protection/password_protection_service.cc
index 60c866b0b637..fa3224887960 100644
--- a/components/safe_browsing/content/password_protection/password_protection_service.cc
+++ b/components/safe_browsing/content/password_protection/password_protection_service.cc
@@ -49,7 +49,7 @@ namespace {
 // Keys for storing password protection verdict into a DictionaryValue.
 const int kRequestTimeoutMs = 10000;
 const char kPasswordProtectionRequestUrl[] =
-    "https://sb-ssl.google.com/safebrowsing/clientreport/login";
+    "https://no-thanks.invalid";
 
 }  // namespace
 
@@ -247,9 +247,7 @@ bool PasswordProtectionService::CanSendPing(
     LoginReputationClientRequest::TriggerType trigger_type,
     const GURL& main_frame_url,
     ReusedPasswordAccountType password_type) {
-  return IsPingingEnabled(trigger_type, password_type) &&
-         !IsURLWhitelistedForPasswordEntry(main_frame_url) &&
-         !IsInExcludedCountry();
+  return false;
 }
 
 void PasswordProtectionService::RequestFinished(
@@ -437,8 +435,7 @@ PasswordProtectionService::MaybeCreateNavigationThrottle(
 
 bool PasswordProtectionService::IsWarningEnabled(
     ReusedPasswordAccountType password_type) {
-  return GetPasswordProtectionWarningTriggerPref(password_type) ==
-         PHISHING_REUSE;
+  return false;
 }
 
 // static
