diff --git a/components/password_manager/core/browser/password_generation_manager.cc b/components/password_manager/core/browser/password_generation_manager.cc
index 06509e7815f5..193b11f2ec6d 100644
--- a/components/password_manager/core/browser/password_generation_manager.cc
+++ b/components/password_manager/core/browser/password_generation_manager.cc
@@ -48,7 +48,6 @@ class PasswordDataForUI : public PasswordFormManagerForUI {
   std::vector<const PasswordForm*> GetFederatedMatches() const override;
   const PasswordForm& GetPendingCredentials() const override;
   metrics_util::CredentialSourceType GetCredentialSource() const override;
-  PasswordFormMetricsRecorder* GetMetricsRecorder() override;
   base::span<const InteractionsStats> GetInteractionsStats() const override;
   base::span<const CompromisedCredentials> GetCompromisedCredentials()
       const override;
@@ -120,10 +119,6 @@ metrics_util::CredentialSourceType PasswordDataForUI::GetCredentialSource()
   return metrics_util::CredentialSourceType::kPasswordManager;
 }
 
-PasswordFormMetricsRecorder* PasswordDataForUI::GetMetricsRecorder() {
-  return nullptr;
-}
-
 base::span<const InteractionsStats> PasswordDataForUI::GetInteractionsStats()
     const {
   return {};
