diff --git a/components/translate/core/browser/BUILD.gn b/components/translate/core/browser/BUILD.gn
index b3464b30ea80b..e3f0a815baa44 100644
--- a/components/translate/core/browser/BUILD.gn
+++ b/components/translate/core/browser/BUILD.gn
@@ -126,8 +126,6 @@ source_set("unit_tests") {
     "//components/prefs:test_support",
     "//components/sync_preferences:test_support",
     "//components/translate/core/common",
-    "//components/ukm",
-    "//components/ukm:test_support",
     "//components/variations",
     "//net:test_support",
     "//services/metrics/public/cpp:metrics_cpp",
