diff --git a/components/security_interstitials/core/browser/resources/interstitial_large.js b/components/security_interstitials/core/browser/resources/interstitial_large.js
index 197ac59e88ed1..607e24bb12e34 100644
--- a/components/security_interstitials/core/browser/resources/interstitial_large.js
+++ b/components/security_interstitials/core/browser/resources/interstitial_large.js
@@ -243,8 +243,6 @@ function setupEvents() {
   }
 
   preventDefaultOnPoundLinkClicks();
-  setupExtendedReportingCheckbox();
-  setupEnhancedProtectionMessage();
   setupSSLDebuggingInfo();
   document.addEventListener('keypress', handleKeypress);
 }
