diff --git a/components/page_info/page_info_delegate.h b/components/page_info/page_info_delegate.h
index 68f98b921f24..472bfbaee18b 100644
--- a/components/page_info/page_info_delegate.h
+++ b/components/page_info/page_info_delegate.h
@@ -10,15 +10,9 @@
 #include "components/content_settings/browser/page_specific_content_settings.h"
 #include "components/content_settings/core/common/content_settings_types.h"
 #include "components/permissions/permission_result.h"
-#include "components/permissions/permission_uma_util.h"
 #include "components/safe_browsing/buildflags.h"
-#include "components/safe_browsing/content/password_protection/metrics_util.h"
 #include "components/security_state/core/security_state.h"
 
-namespace safe_browsing {
-class PasswordProtectionService;
-}  // namespace safe_browsing
-
 namespace permissions {
 class ChooserContextBase;
 class PermissionDecisionAutoBlocker;
