diff --git a/components/policy/core/browser/BUILD.gn b/components/policy/core/browser/BUILD.gn
index 0e8afdd37118f..89ec389dfb00b 100644
--- a/components/policy/core/browser/BUILD.gn
+++ b/components/policy/core/browser/BUILD.gn
@@ -58,7 +58,6 @@ source_set("internal") {
     "//components/keyed_service/core",
     "//components/pref_registry",
     "//components/prefs",
-    "//components/reporting/proto:record_proto",
     "//components/strings",
     "//components/url_formatter",
     "//components/url_matcher",
