diff --git a/components/performance_manager/graph/graph_impl.cc b/components/performance_manager/graph/graph_impl.cc
index 2eaf547b6c6b..409ceba7d98a 100644
--- a/components/performance_manager/graph/graph_impl.cc
+++ b/components/performance_manager/graph/graph_impl.cc
@@ -24,10 +24,6 @@
 #include "components/performance_manager/public/graph/node_data_describer.h"
 #include "components/performance_manager/public/graph/node_data_describer_registry.h"
 
-namespace ukm {
-class UkmEntryBuilder;
-}  // namespace ukm
-
 namespace performance_manager {
 
 namespace {
@@ -305,11 +301,6 @@ bool GraphImpl::IsEmpty() const {
   return nodes_.empty();
 }
 
-ukm::UkmRecorder* GraphImpl::GetUkmRecorder() const {
-  DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
-  return ukm_recorder();
-}
-
 NodeDataDescriberRegistry* GraphImpl::GetNodeDataDescriberRegistry() const {
   if (!describer_registry_)
     describer_registry_ = std::make_unique<NodeDataDescriberRegistryImpl>();
