diff --git a/components/security_interstitials/core/BUILD.gn b/components/security_interstitials/core/BUILD.gn
index a7e6e10e50bd..28950f5c3354 100644
--- a/components/security_interstitials/core/BUILD.gn
+++ b/components/security_interstitials/core/BUILD.gn
@@ -6,8 +6,6 @@ static_library("core") {
   sources = [
     "bad_clock_ui.cc",
     "bad_clock_ui.h",
-    "base_safe_browsing_error_ui.cc",
-    "base_safe_browsing_error_ui.h",
     "blocked_interception_ui.cc",
     "blocked_interception_ui.h",
     "common_string_util.cc",
@@ -24,10 +22,6 @@ static_library("core") {
     "mitm_software_ui.h",
     "pref_names.cc",
     "pref_names.h",
-    "safe_browsing_loud_error_ui.cc",
-    "safe_browsing_loud_error_ui.h",
-    "safe_browsing_quiet_error_ui.cc",
-    "safe_browsing_quiet_error_ui.h",
     "ssl_error_options_mask.cc",
     "ssl_error_options_mask.h",
     "ssl_error_ui.cc",
@@ -72,9 +66,6 @@ static_library("unsafe_resource") {
 
   deps = [
     "//base",
-    "//components/safe_browsing/core/common:common",
-    "//components/safe_browsing/core/db:hit_report",
-    "//components/safe_browsing/core/db:util",
     "//net",
     "//services/network/public/mojom",
   ]
