diff --git a/components/autofill/core/browser/data_model/autofill_profile.cc b/components/autofill/core/browser/data_model/autofill_profile.cc
index 165605125ad5..8d45b7c45dbd 100644
--- a/components/autofill/core/browser/data_model/autofill_profile.cc
+++ b/components/autofill/core/browser/data_model/autofill_profile.cc
@@ -23,7 +23,6 @@
 #include "base/strings/utf_string_conversion_utils.h"
 #include "base/strings/utf_string_conversions.h"
 #include "components/autofill/core/browser/autofill_field.h"
-#include "components/autofill/core/browser/autofill_metrics.h"
 #include "components/autofill/core/browser/autofill_type.h"
 #include "components/autofill/core/browser/data_model/address.h"
 #include "components/autofill/core/browser/data_model/autofill_metadata.h"
@@ -764,24 +763,9 @@ bool AutofillProfile::SaveAdditionalInfo(const AutofillProfile& profile,
   // we can merge two unverified profiles or merge verified profile data into an
   // unverified profile.
   if (!IsVerified() || profile.IsVerified()) {
-    if (MergeDataFrom(profile, app_locale)) {
-      AutofillMetrics::LogProfileActionOnFormSubmitted(
-          AutofillMetrics::EXISTING_PROFILE_UPDATED);
-    } else {
-      AutofillMetrics::LogProfileActionOnFormSubmitted(
-          AutofillMetrics::EXISTING_PROFILE_USED);
-    }
     return true;
   }
-  // If the profile is verified, only the structured name information should be
-  // merged.
-  if (MergeStructuredDataFrom(profile, app_locale)) {
-    AutofillMetrics::LogProfileActionOnFormSubmitted(
-        AutofillMetrics::EXISTING_PROFILE_UPDATED);
-  } else {
-    AutofillMetrics::LogProfileActionOnFormSubmitted(
-        AutofillMetrics::EXISTING_PROFILE_USED);
-  }
+
   return true;
 }
 
