diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index b76e070130ef..cb96f732024a 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -107,8 +107,6 @@ static_library("metrics") {
     "metrics_switches.h",
     "metrics_upload_scheduler.cc",
     "metrics_upload_scheduler.h",
-    "persistent_histograms.cc",
-    "persistent_histograms.h",
     "persistent_system_profile.cc",
     "persistent_system_profile.h",
     "reporting_service.cc",
@@ -186,7 +184,6 @@ static_library("metrics") {
       "system_session_analyzer/system_session_analyzer_win.cc",
       "system_session_analyzer/system_session_analyzer_win.h",
     ]
-    deps += [ "//components/browser_watcher:stability_client" ]
     libs = [ "wevtapi.lib" ]
   } else {
     sources += [ "machine_id_provider_nonwin.cc" ]
@@ -251,9 +248,6 @@ if (!is_ios) {
       "//extensions/buildflags",
       "//gpu/config",
     ]
-    if (is_android) {
-      deps += [ "//components/crash/content/browser" ]
-    }
   }
 }
 
