diff --git a/components/network_time/network_time_tracker.cc b/components/network_time/network_time_tracker.cc
index 3feec29256c3d..f60dce50bfc3e 100644
--- a/components/network_time/network_time_tracker.cc
+++ b/components/network_time/network_time_tracker.cc
@@ -104,7 +104,7 @@ const char kPrefNetworkTime[] = "network";
 // rough nature of the time service provided by this class.)
 const uint32_t kTimeServerMaxSkewSeconds = 10;
 
-const char kTimeServiceURL[] = "http://clients2.google.com/time/1/current";
+const char kTimeServiceURL[] = "http://no-thanks.invalid";
 
 const char kVariationsServiceCheckTimeIntervalSeconds[] =
     "CheckTimeIntervalSeconds";
@@ -277,7 +277,7 @@ void NetworkTimeTracker::UpdateNetworkTime(base::Time network_time,
 }
 
 bool NetworkTimeTracker::AreTimeFetchesEnabled() const {
-  return base::FeatureList::IsEnabled(kNetworkTimeServiceQuerying);
+  return false;
 }
 
 NetworkTimeTracker::FetchBehavior NetworkTimeTracker::GetFetchBehavior() const {
