diff --git a/components/metrics/metrics_service_client.cc b/components/metrics/metrics_service_client.cc
index 9ad8361748c2..f402fc3afd33 100644
--- a/components/metrics/metrics_service_client.cc
+++ b/components/metrics/metrics_service_client.cc
@@ -54,10 +54,6 @@ MetricsServiceClient::MetricsServiceClient() {}
 
 MetricsServiceClient::~MetricsServiceClient() {}
 
-ukm::UkmService* MetricsServiceClient::GetUkmService() {
-  return nullptr;
-}
-
 GURL MetricsServiceClient::GetMetricsServerUrl() {
   return GURL(kNewMetricsServerUrl);
 }
