diff --git a/components/printing/browser/BUILD.gn b/components/printing/browser/BUILD.gn
index 1b3c83d06c85..019b7a21dbfd 100644
--- a/components/printing/browser/BUILD.gn
+++ b/components/printing/browser/BUILD.gn
@@ -50,7 +50,6 @@ static_library("browser") {
 
   deps = [
     "//base",
-    "//components/crash/core/common",
     "//components/discardable_memory/service",
     "//components/printing/common",
     "//components/printing/common:mojo_interfaces",
