diff --git a/components/payments/content/payment_app.cc b/components/payments/content/payment_app.cc
index 69f2c695efc6..033e431b4735 100644
--- a/components/payments/content/payment_app.cc
+++ b/components/payments/content/payment_app.cc
@@ -86,10 +86,6 @@ const std::set<std::string>& PaymentApp::GetAppMethodNames() const {
   return app_method_names_;
 }
 
-ukm::SourceId PaymentApp::UkmSourceId() {
-  return ukm::kInvalidSourceId;
-}
-
 bool PaymentApp::IsWaitingForPaymentDetailsUpdate() const {
   return false;
 }
