diff --git a/components/password_manager/core/browser/password_save_manager.h b/components/password_manager/core/browser/password_save_manager.h
index e8e51586f0d5..2da76ce1479a 100644
--- a/components/password_manager/core/browser/password_save_manager.h
+++ b/components/password_manager/core/browser/password_save_manager.h
@@ -24,7 +24,6 @@ class PasswordManagerClient;
 class FormFetcher;
 class VotesUploader;
 class FormSaver;
-class PasswordFormMetricsRecorder;
 class PasswordManagerDriver;
 
 // Implementations of this interface should encapsulate the password Save/Update
@@ -41,7 +40,6 @@ class PasswordSaveManager {
 
   virtual void Init(PasswordManagerClient* client,
                     const FormFetcher* form_fetcher,
-                    scoped_refptr<PasswordFormMetricsRecorder> metrics_recorder,
                     VotesUploader* votes_uploader) = 0;
 
   virtual const autofill::PasswordForm& GetPendingCredentials() const = 0;
