diff --git a/components/paint_preview/browser/paint_preview_client.cc b/components/paint_preview/browser/paint_preview_client.cc
index 7232b9be21b21..0204cc5caa9b3 100644
--- a/components/paint_preview/browser/paint_preview_client.cc
+++ b/components/paint_preview/browser/paint_preview_client.cc
@@ -78,19 +78,6 @@ PaintPreviewCaptureResponseToPaintPreviewFrameProto(
   return frame_guids;
 }
 
-// Records UKM data for the capture.
-// TODO(crbug/1038390): Add more metrics;
-// - Peak memory during capture (bucketized).
-// - Compressed on disk size (bucketized).
-void RecordUkmCaptureData(ukm::SourceId source_id,
-                          base::TimeDelta blink_recording_time) {
-  if (source_id == ukm::kInvalidSourceId)
-    return;
-  ukm::builders::PaintPreviewCapture(source_id)
-      .SetBlinkCaptureTime(blink_recording_time.InMilliseconds())
-      .Record(ukm::UkmRecorder::Get());
-}
-
 base::flat_set<base::UnguessableToken> CreateAcceptedTokenList(
     content::RenderFrameHost* render_frame_host) {
   auto rfhs = render_frame_host->GetFramesInSubtree();
@@ -313,8 +300,6 @@ void PaintPreviewClient::CapturePaintPreview(
   chromeVersion->set_build(CHROME_VERSION_BUILD);
   chromeVersion->set_patch(CHROME_VERSION_PATCH);
   document_data.callback = std::move(callback);
-  document_data.source_id =
-      ukm::GetSourceIdForWebContentsDocument(web_contents());
   document_data.accepted_tokens = CreateAcceptedTokenList(render_frame_host);
   document_data.capture_links = params.inner.capture_links;
   document_data.max_per_capture_size = params.inner.max_capture_size;
@@ -571,9 +556,6 @@ void PaintPreviewClient::OnFinished(
         "Browser.PaintPreview.Capture.NumberOfFramesCaptured",
         document_data->finished_subframes.size());
 
-    RecordUkmCaptureData(document_data->source_id,
-                         document_data->main_frame_blink_recording_time);
-
     // At a minimum one frame was captured successfully, it is up to the
     // caller to decide if a partial success is acceptable based on what is
     // contained in the proto.
