diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 4d970f45ddc7e..f56143d172f1e 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -45,15 +45,11 @@
 #endif
 #include "build/chromeos_buildflags.h"
 #include "components/safe_browsing/buildflags.h"
-#include "components/safe_browsing/content/password_protection/password_protection_service.h"
-#include "components/safe_browsing/core/password_protection/metrics_util.h"
-#include "components/safe_browsing/core/proto/csd.pb.h"
 #include "components/security_interstitials/content/stateful_ssl_host_state_delegate.h"
 #include "components/signin/public/identity_manager/account_info.h"
 #include "components/ssl_errors/error_info.h"
 #include "components/strings/grit/components_chromium_strings.h"
 #include "components/strings/grit/components_strings.h"
-#include "components/subresource_filter/core/browser/subresource_filter_features.h"
 #include "components/ukm/content/source_url_recorder.h"
 #include "components/url_formatter/elide_url.h"
 #include "content/public/browser/browser_context.h"
@@ -65,8 +61,6 @@
 #include "net/cert/x509_certificate.h"
 #include "net/ssl/ssl_cipher_suite_names.h"
 #include "net/ssl/ssl_connection_status_flags.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
 #include "third_party/boringssl/src/include/openssl/ssl.h"
 #include "ui/base/l10n/l10n_util.h"
 #include "url/origin.h"
@@ -75,8 +69,6 @@ using base::ASCIIToUTF16;
 using base::UTF16ToUTF8;
 using base::UTF8ToUTF16;
 using content::BrowserThread;
-using safe_browsing::LoginReputationClientResponse;
-using safe_browsing::RequestOutcome;
 
 namespace {
 
@@ -129,17 +121,11 @@ bool ShouldShowPermission(const PageInfo::PermissionInfo& info,
                           const GURL& site_url,
                           HostContentSettingsMap* content_settings,
                           content::WebContents* web_contents,
-                          bool changed_since_last_page_load,
-                          bool is_subresource_filter_activated) {
+                          bool changed_since_last_page_load) {
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
-    if (!base::FeatureList::IsEnabled(
-            subresource_filter::kSafeBrowsingSubresourceFilter)) {
-      return false;
-    }
-
-    return is_subresource_filter_activated;
+    return false;
   }
 
   if (info.type == ContentSettingsType::SOUND) {
@@ -298,7 +284,6 @@ PageInfo::PageInfo(std::unique_ptr<PageInfoDelegate> delegate,
       show_info_bar_(false),
       site_url_(url),
       site_identity_status_(SITE_IDENTITY_STATUS_UNKNOWN),
-      safe_browsing_status_(SAFE_BROWSING_STATUS_NONE),
       safety_tip_info_({security_state::SafetyTipStatus::kUnknown, GURL()}),
       site_connection_status_(SITE_CONNECTION_STATUS_UNKNOWN),
       show_ssl_decision_revoke_button_(false),
@@ -423,13 +408,6 @@ void PageInfo::RecordPageInfoAction(PageInfoAction action) {
 
   UMA_HISTOGRAM_ENUMERATION("WebsiteSettings.Action", action, PAGE_INFO_COUNT);
 
-  if (web_contents()) {
-    ukm::builders::PageInfoBubble(
-        ukm::GetSourceIdForWebContentsDocument(web_contents()))
-        .SetActionTaken(action)
-        .Record(ukm::UkmRecorder::Get());
-  }
-
   base::UmaHistogramEnumeration(
       security_state::GetSafetyTipHistogramName(
           "Security.SafetyTips.PageInfo.Action", safety_tip_info_.status),
@@ -516,11 +494,6 @@ void PageInfo::OnSitePermissionChanged(ContentSettingsType type,
     }
   }
 
-  permissions::PermissionUmaUtil::ScopedRevocationReporter
-      scoped_revocation_reporter(web_contents()->GetBrowserContext(), site_url_,
-                                 site_url_, type,
-                                 permissions::PermissionSourceUI::OIB);
-
   // The permission may have been blocked due to being under embargo, so if it
   // was changed away from BLOCK, clear embargo status if it exists.
   if (setting != CONTENT_SETTING_BLOCK) {
@@ -981,8 +954,7 @@ void PageInfo::PresentSitePermissions() {
 
     if (ShouldShowPermission(
             permission_info, site_url_, content_settings, web_contents(),
-            HasContentSettingChangedViaPageInfo(permission_info.type),
-            delegate_->IsSubresourceFilterActivated(site_url_))) {
+            HasContentSettingChangedViaPageInfo(permission_info.type))) {
       permission_info_list.push_back(permission_info);
     }
   }
