diff --git a/components/security_interstitials/core/browser/resources/interstitial_large.js b/components/security_interstitials/core/browser/resources/interstitial_large.js
index 3f0a1dc6cc20..89fa28e5a60f 100644
--- a/components/security_interstitials/core/browser/resources/interstitial_large.js
+++ b/components/security_interstitials/core/browser/resources/interstitial_large.js
@@ -244,8 +244,6 @@ function setupEvents() {
   }
 
   preventDefaultOnPoundLinkClicks();
-  setupExtendedReportingCheckbox();
-  setupEnhancedProtectionMessage();
   setupSSLDebuggingInfo();
   document.addEventListener('keypress', handleKeypress);
 }
