diff --git a/components/blocked_content/popup_opener_tab_helper.h b/components/blocked_content/popup_opener_tab_helper.h
index 2bd8b065fdf9..a0b06dc2a601 100644
--- a/components/blocked_content/popup_opener_tab_helper.h
+++ b/components/blocked_content/popup_opener_tab_helper.h
@@ -38,7 +38,7 @@ class PopupOpenerTabHelper
  public:
   ~PopupOpenerTabHelper() override;
 
-  void OnOpenedPopup(PopupTracker* popup_tracker);
+  void OnOpenedPopup();
   void OnDidTabUnder();
 
   bool has_opened_popup_since_last_user_gesture() const {
@@ -86,9 +86,6 @@ class PopupOpenerTabHelper
 
   bool has_opened_popup_since_last_user_gesture_ = false;
 
-  // The last source id used for logging Popup_Page.
-  ukm::SourceId last_opener_source_id_ = ukm::kInvalidSourceId;
-
   // The settings map for the web contents this object is associated with.
   HostContentSettingsMap* settings_map_ = nullptr;
 
