diff --git a/components/performance_manager/BUILD.gn b/components/performance_manager/BUILD.gn
index eba36a7aea871..eb0605e1653b5 100644
--- a/components/performance_manager/BUILD.gn
+++ b/components/performance_manager/BUILD.gn
@@ -194,7 +194,6 @@ static_library("performance_manager") {
     "//base/allocator:buildflags",
     "//components/performance_manager/public/mojom",
     "//content/public/browser",
-    "//services/metrics/public/cpp:metrics_cpp",
     "//third_party/blink/public/common",
     "//url",
   ]
