diff --git a/components/download/internal/common/download_create_info.cc b/components/download/internal/common/download_create_info.cc
index 677d4cf2aa9f3..460548ebe93dc 100644
--- a/components/download/internal/common/download_create_info.cc
+++ b/components/download/internal/common/download_create_info.cc
@@ -30,7 +30,6 @@ DownloadCreateInfo::DownloadCreateInfo(
       accept_range(RangeRequestSupportType::kNoSupport),
       connection_info(net::HttpResponseInfo::CONNECTION_INFO_UNKNOWN),
       method("GET"),
-      ukm_source_id(ukm::kInvalidSourceId),
       is_content_initiated(false) {}
 
 DownloadCreateInfo::DownloadCreateInfo()
