diff --git a/components/autofill/core/browser/BUILD.gn b/components/autofill/core/browser/BUILD.gn
index 5031372d31e2..3d057fe4a794 100644
--- a/components/autofill/core/browser/BUILD.gn
+++ b/components/autofill/core/browser/BUILD.gn
@@ -549,8 +549,6 @@ static_library("test_support") {
     "//components/sync:test_support",
     "//components/translate/core/browser",
     "//components/translate/core/browser:test_support",
-    "//components/ukm",
-    "//components/ukm:test_support",
     "//components/version_info:version_info",
     "//google_apis:test_support",
     "//services/network:test_support",
@@ -759,8 +757,6 @@ source_set("unit_tests") {
     "//components/sync:test_support_model",
     "//components/sync/driver:test_support",
     "//components/translate/core/browser",
-    "//components/ukm",
-    "//components/ukm:test_support",
     "//components/unified_consent",
     "//components/variations",
     "//components/variations:test_support",
