diff --git a/components/metrics/metrics_service_accessor.cc b/components/metrics/metrics_service_accessor.cc
index ca1fb8ca159d..a88cbc4ba180 100644
--- a/components/metrics/metrics_service_accessor.cc
+++ b/components/metrics/metrics_service_accessor.cc
@@ -17,23 +17,12 @@ namespace {
 
 bool g_force_official_enabled_test = false;
 
-bool IsMetricsReportingEnabledForOfficialBuild(PrefService* pref_service) {
-  return pref_service->GetBoolean(prefs::kMetricsReportingEnabled);
-}
-
 }  // namespace
 
 // static
 bool MetricsServiceAccessor::IsMetricsReportingEnabled(
     PrefService* pref_service) {
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
-  return IsMetricsReportingEnabledForOfficialBuild(pref_service);
-#else
-  // In non-official builds, disable metrics reporting completely.
-  return g_force_official_enabled_test
-             ? IsMetricsReportingEnabledForOfficialBuild(pref_service)
-             : false;
-#endif  // BUILDFLAG(GOOGLE_CHROME_BRANDING)
+  return false;
 }
 
 // static
@@ -51,12 +40,6 @@ bool MetricsServiceAccessor::RegisterSyntheticFieldTrialWithNameAndGroupHash(
     MetricsService* metrics_service,
     uint32_t trial_name_hash,
     uint32_t group_name_hash) {
-  if (!metrics_service)
-    return false;
-
-  variations::SyntheticTrialGroup trial_group(trial_name_hash, group_name_hash);
-  metrics_service->synthetic_trial_registry()->RegisterSyntheticFieldTrial(
-      trial_group);
   return true;
 }
 
