diff --git a/components/password_manager/core/browser/mock_password_form_manager_for_ui.h b/components/password_manager/core/browser/mock_password_form_manager_for_ui.h
index de5c3929e704f..3989f5852f231 100644
--- a/components/password_manager/core/browser/mock_password_form_manager_for_ui.h
+++ b/components/password_manager/core/browser/mock_password_form_manager_for_ui.h
@@ -32,7 +32,6 @@ class MockPasswordFormManagerForUI : public PasswordFormManagerForUI {
               GetCredentialSource,
               (),
               (const override));
-  MOCK_METHOD(PasswordFormMetricsRecorder*, GetMetricsRecorder, (), (override));
   MOCK_METHOD(base::span<const InteractionsStats>,
               GetInteractionsStats,
               (),
