diff --git a/components/unified_consent/unified_consent_service.cc b/components/unified_consent/unified_consent_service.cc
index 5251ac16d5e1d..7fc7b591eaedf 100644
--- a/components/unified_consent/unified_consent_service.cc
+++ b/components/unified_consent/unified_consent_service.cc
@@ -57,7 +57,7 @@ void UnifiedConsentService::SetUrlKeyedAnonymizedDataCollectionEnabled(
     SetMigrationState(MigrationState::kCompleted);
 
   pref_service_->SetBoolean(prefs::kUrlKeyedAnonymizedDataCollectionEnabled,
-                            enabled);
+                            false);
 }
 
 void UnifiedConsentService::Shutdown() {
@@ -169,6 +169,7 @@ void UnifiedConsentService::UpdateSettingsForMigration() {
     return;
   }
 
+#if 0
   // Set URL-keyed anonymized metrics to the state it had before unified
   // consent.
   bool url_keyed_metrics_enabled =
@@ -176,6 +177,8 @@ void UnifiedConsentService::UpdateSettingsForMigration() {
       sync_service_->GetUserSettings()->GetSelectedTypes().Has(
           syncer::UserSelectableType::kHistory) &&
       !sync_service_->GetUserSettings()->IsUsingExplicitPassphrase();
+#endif
+  bool url_keyed_metrics_enabled = false;
   SetUrlKeyedAnonymizedDataCollectionEnabled(url_keyed_metrics_enabled);
 }
 
