diff --git a/components/subresource_filter/content/browser/subresource_filter_observer_manager.h b/components/subresource_filter/content/browser/subresource_filter_observer_manager.h
index 77e2b54e7db5..53aa2f23ee86 100644
--- a/components/subresource_filter/content/browser/subresource_filter_observer_manager.h
+++ b/components/subresource_filter/content/browser/subresource_filter_observer_manager.h
@@ -40,8 +40,7 @@ class SubresourceFilterObserverManager
   // the latest. Right now it will only include phishing and subresource filter
   // threat types.
   virtual void NotifySafeBrowsingChecksComplete(
-      content::NavigationHandle* navigation_handle,
-      const SubresourceFilterSafeBrowsingClient::CheckResult& result);
+      content::NavigationHandle* navigation_handle);
 
   // Will be called at the latest in the WillProcessResponse stage from a
   // NavigationThrottle that was registered before the throttle manager's
