diff --git a/components/safe_browsing/core/common/safebrowsing_constants.cc b/components/safe_browsing/core/common/safebrowsing_constants.cc
index 9958f003d1e4..dc3bae12d2b1 100644
--- a/components/safe_browsing/core/common/safebrowsing_constants.cc
+++ b/components/safe_browsing/core/common/safebrowsing_constants.cc
@@ -15,7 +15,7 @@ const base::FilePath::CharType kSafeBrowsingBaseFilename[] =
 const base::FilePath::CharType kCookiesFile[] = FILE_PATH_LITERAL(" Cookies");
 
 // The URL for the Safe Browsing page.
-const char kSafeBrowsingUrl[] = "https://safebrowsing.google.com/";
+const char kSafeBrowsingUrl[] = "https://no-thanks.invalid/";
 
 const char kCustomCancelReasonForURLLoader[] = "SafeBrowsing";
 
