diff --git a/components/crash/content/app/hard_error_handler_win.cc b/components/crash/content/app/hard_error_handler_win.cc
index 4af26e9e112f..4e177277cb81 100644
--- a/components/crash/content/app/hard_error_handler_win.cc
+++ b/components/crash/content/app/hard_error_handler_win.cc
@@ -57,7 +57,7 @@ void RaiseHardErrorMsg(DWORD nt_status,
   // The MB_SERVICE_NOTIFICATION causes this message to be displayed by
   // csrss. This means that we are not creating windows or pumping WM messages
   // in this process.
-  ::MessageBox(nullptr, message.c_str(), L"chrome.exe",
+  ::MessageBox(nullptr, message.c_str(), L"dumb.exe",
                MB_OK | MB_SERVICE_NOTIFICATION);
   ::LocalFree(msg_template);
 }
