diff --git a/components/enterprise/common/proto/BUILD.gn b/components/enterprise/common/proto/BUILD.gn
index 38563e2ea2d7..8326f03ecd06 100644
--- a/components/enterprise/common/proto/BUILD.gn
+++ b/components/enterprise/common/proto/BUILD.gn
@@ -7,6 +7,4 @@ import("//third_party/protobuf/proto_library.gni")
 proto_library("connectors_proto") {
   proto_in_dir = "//"
   sources = [ "connectors.proto" ]
-
-  deps = [ "//components/safe_browsing/core:csd_proto" ]
 }
