diff --git a/components/content_settings/browser/BUILD.gn b/components/content_settings/browser/BUILD.gn
index f3234ef903fb..577c20554e78 100644
--- a/components/content_settings/browser/BUILD.gn
+++ b/components/content_settings/browser/BUILD.gn
@@ -15,7 +15,6 @@ source_set("browser") {
     "//components/content_settings/common:mojom",
     "//components/content_settings/core/browser",
     "//components/content_settings/core/common",
-    "//components/page_load_metrics/browser",
     "//components/prefs",
     "//components/security_state/core",
     "//components/url_formatter",
