diff --git a/components/autofill/core/browser/payments/autofill_save_card_infobar_delegate_mobile.h b/components/autofill/core/browser/payments/autofill_save_card_infobar_delegate_mobile.h
index f1f9011b1e24..1bd3d79bdb08 100644
--- a/components/autofill/core/browser/payments/autofill_save_card_infobar_delegate_mobile.h
+++ b/components/autofill/core/browser/payments/autofill_save_card_infobar_delegate_mobile.h
@@ -12,7 +12,6 @@
 #include "base/strings/string16.h"
 #include "build/build_config.h"
 #include "components/autofill/core/browser/autofill_client.h"
-#include "components/autofill/core/browser/autofill_metrics.h"
 #include "components/autofill/core/browser/payments/legal_message_line.h"
 #include "components/infobars/core/confirm_infobar_delegate.h"
 
@@ -103,8 +102,6 @@ class AutofillSaveCardInfoBarDelegateMobile : public ConfirmInfoBarDelegate {
       AutofillClient::SaveCardOfferUserDecision user_decision,
       AutofillClient::UserProvidedCardDetails user_provided_details);
 
-  void LogUserAction(AutofillMetrics::InfoBarMetric user_action);
-
   // Whether the action is an upload or a local save.
   bool upload_;
 
