diff --git a/components/security_interstitials/content/BUILD.gn b/components/security_interstitials/content/BUILD.gn
index dd19bf9033f6..a8d1ad21d487 100644
--- a/components/security_interstitials/content/BUILD.gn
+++ b/components/security_interstitials/content/BUILD.gn
@@ -70,7 +70,6 @@ static_library("security_interstitial_page") {
 
   public_deps = [
     "//components/captive_portal/core:buildflags",
-    "//components/safe_browsing/core/db:hit_report",
   ]
 
   deps = [
@@ -86,7 +85,6 @@ static_library("security_interstitial_page") {
     "//components/pref_registry",
     "//components/prefs:prefs",
     "//components/resources",
-    "//components/safe_browsing/core/common:safe_browsing_prefs",
     "//components/security_interstitials/core:core",
     "//components/security_interstitials/core:unsafe_resource",
     "//components/ssl_errors",
