diff --git a/components/password_manager/core/browser/stub_password_manager_client.cc b/components/password_manager/core/browser/stub_password_manager_client.cc
index fe63a39b284c..1f766ba53b00 100644
--- a/components/password_manager/core/browser/stub_password_manager_client.cc
+++ b/components/password_manager/core/browser/stub_password_manager_client.cc
@@ -98,43 +98,6 @@ StubPasswordManagerClient::GetPasswordFeatureManager() {
   return &password_feature_manager_;
 }
 
-#if defined(ON_FOCUS_PING_ENABLED) || defined(PASSWORD_REUSE_DETECTION_ENABLED)
-safe_browsing::PasswordProtectionService*
-StubPasswordManagerClient::GetPasswordProtectionService() const {
-  return nullptr;
-}
-#endif
-
-#if defined(ON_FOCUS_PING_ENABLED)
-void StubPasswordManagerClient::CheckSafeBrowsingReputation(
-    const GURL& form_action,
-    const GURL& frame_url) {}
-#endif
-
-#if defined(PASSWORD_REUSE_DETECTION_ENABLED)
-void StubPasswordManagerClient::CheckProtectedPasswordEntry(
-    metrics_util::PasswordType reused_password_type,
-    const std::string& username,
-    const std::vector<MatchingReusedCredential>& matching_reused_credentials,
-    bool password_field_exists) {}
-#endif
-
-#if defined(PASSWORD_REUSE_WARNING_ENABLED)
-void StubPasswordManagerClient::LogPasswordReuseDetectedEvent() {}
-#endif
-
-ukm::SourceId StubPasswordManagerClient::GetUkmSourceId() {
-  return ukm_source_id_;
-}
-
-PasswordManagerMetricsRecorder*
-StubPasswordManagerClient::GetMetricsRecorder() {
-  if (!metrics_recorder_) {
-    metrics_recorder_.emplace(GetUkmSourceId(), nullptr);
-  }
-  return base::OptionalOrNullptr(metrics_recorder_);
-}
-
 signin::IdentityManager* StubPasswordManagerClient::GetIdentityManager() {
   return nullptr;
 }
