diff --git a/components/exo/text_input.h b/components/exo/text_input.h
index bf4664de087f..f88b02407f6b 100644
--- a/components/exo/text_input.h
+++ b/components/exo/text_input.h
@@ -135,7 +135,6 @@ class TextInput : public ui::TextInputClient,
   void EnsureCaretNotInRect(const gfx::Rect& rect) override;
   bool IsTextEditCommandEnabled(ui::TextEditCommand command) const override;
   void SetTextEditCommandForNextKeyEvent(ui::TextEditCommand command) override;
-  ukm::SourceId GetClientSourceForMetrics() const override;
   bool ShouldDoLearning() override;
   bool SetCompositionFromExistingText(
       const gfx::Range& range,
