diff --git a/components/metrics_services_manager/DEPS b/components/metrics_services_manager/DEPS
index d90fc81dc778..365cfc4ab2cc 100644
--- a/components/metrics_services_manager/DEPS
+++ b/components/metrics_services_manager/DEPS
@@ -2,7 +2,6 @@ include_rules = [
   "-components",
   "+components/metrics",
   "+components/metrics_services_manager",
-  "+components/rappor",
   "+components/ukm",
   "+components/variations",
   "+services/network",
