diff --git a/components/discardable_memory/client/client_discardable_shared_memory_manager.cc b/components/discardable_memory/client/client_discardable_shared_memory_manager.cc
index 528e02d11ece..35ffaa22ceb5 100644
--- a/components/discardable_memory/client/client_discardable_shared_memory_manager.cc
+++ b/components/discardable_memory/client/client_discardable_shared_memory_manager.cc
@@ -26,7 +26,6 @@
 #include "base/trace_event/memory_dump_manager.h"
 #include "base/trace_event/trace_event.h"
 #include "build/build_config.h"
-#include "components/crash/core/common/crash_key.h"
 
 namespace discardable_memory {
 
@@ -634,13 +633,6 @@ void ClientDiscardableSharedMemoryManager::DeletedDiscardableSharedMemory(
 void ClientDiscardableSharedMemoryManager::MemoryUsageChanged(
     size_t new_bytes_total,
     size_t new_bytes_free) const {
-  static crash_reporter::CrashKeyString<24> discardable_memory_allocated(
-      "discardable-memory-allocated");
-  discardable_memory_allocated.Set(base::NumberToString(new_bytes_total));
-
-  static crash_reporter::CrashKeyString<24> discardable_memory_free(
-      "discardable-memory-free");
-  discardable_memory_free.Set(base::NumberToString(new_bytes_free));
 }
 
 }  // namespace discardable_memory
