diff --git a/components/autofill/core/browser/ui/payments/card_unmask_prompt_controller_impl.h b/components/autofill/core/browser/ui/payments/card_unmask_prompt_controller_impl.h
index 0fcaffcf3f36..86bd856fdc9f 100644
--- a/components/autofill/core/browser/ui/payments/card_unmask_prompt_controller_impl.h
+++ b/components/autofill/core/browser/ui/payments/card_unmask_prompt_controller_impl.h
@@ -11,7 +11,6 @@
 #include "base/macros.h"
 #include "base/memory/weak_ptr.h"
 #include "build/build_config.h"
-#include "components/autofill/core/browser/autofill_metrics.h"
 #include "components/autofill/core/browser/data_model/credit_card.h"
 #include "components/autofill/core/browser/payments/card_unmask_delegate.h"
 #include "components/autofill/core/browser/ui/payments/card_unmask_prompt_controller.h"
@@ -73,8 +72,6 @@ class CardUnmaskPromptControllerImpl : public CardUnmaskPromptController {
 
  private:
   bool AllowsRetry(AutofillClient::PaymentsRpcResult result);
-  void LogOnCloseEvents();
-  AutofillMetrics::UnmaskPromptEvent GetCloseReasonEvent();
 
   PrefService* pref_service_;
   bool new_card_link_clicked_ = false;
