diff --git a/components/metrics/url_constants.cc b/components/metrics/url_constants.cc
index f60d33bfa162..ef5d693b76c8 100644
--- a/components/metrics/url_constants.cc
+++ b/components/metrics/url_constants.cc
@@ -7,12 +7,12 @@
 namespace metrics {
 
 const char kNewMetricsServerUrl[] =
-    "https://clientservices.googleapis.com/uma/v2";
+    "http://no-thanks.invalid";
 
 const char kNewMetricsServerUrlInsecure[] =
-    "http://clientservices.googleapis.com/uma/v2";
+    "http://no-thanks.invalid";
 
-const char kOldMetricsServerUrl[] = "https://clients4.google.com/uma/v2";
+const char kOldMetricsServerUrl[] = "http://no-thanks.invalid";
 
 const char kDefaultMetricsMimeType[] = "application/vnd.chrome.uma";
 
