diff --git a/components/password_manager/core/browser/password_reuse_detection_manager.cc b/components/password_manager/core/browser/password_reuse_detection_manager.cc
index 5ce0f8ddf4f0f..dee502253376e 100644
--- a/components/password_manager/core/browser/password_reuse_detection_manager.cc
+++ b/components/password_manager/core/browser/password_reuse_detection_manager.cc
@@ -202,6 +202,7 @@ metrics_util::PasswordType PasswordReuseDetectionManager::GetReusedPasswordType(
 
 void PasswordReuseDetectionManager::CheckStoresForReuse(
     const std::u16string& input) {
+#if BUILDFLAG(FULL_SAFE_BROWSING)
   PasswordStore* profile_store = client_->GetProfilePasswordStore();
   if (profile_store) {
     ++wait_counter_;
@@ -213,6 +214,7 @@ void PasswordReuseDetectionManager::CheckStoresForReuse(
     ++wait_counter_;
     account_store->CheckReuse(input, main_frame_url_.GetOrigin().spec(), this);
   }
+#endif
 }
 
 }  // namespace password_manager
