diff --git a/components/arc/ime/arc_ime_service.h b/components/arc/ime/arc_ime_service.h
index acaee65d23776..372a054b59744 100644
--- a/components/arc/ime/arc_ime_service.h
+++ b/components/arc/ime/arc_ime_service.h
@@ -147,7 +147,6 @@ class ArcImeService : public KeyedService,
   bool IsTextEditCommandEnabled(ui::TextEditCommand command) const override;
   void SetTextEditCommandForNextKeyEvent(ui::TextEditCommand command) override {
   }
-  ukm::SourceId GetClientSourceForMetrics() const override;
   bool ShouldDoLearning() override;
   bool SetCompositionFromExistingText(
       const gfx::Range& range,
