diff --git a/components/page_load_metrics/browser/BUILD.gn b/components/page_load_metrics/browser/BUILD.gn
index ce601841e237..6d12b29a5a37 100644
--- a/components/page_load_metrics/browser/BUILD.gn
+++ b/components/page_load_metrics/browser/BUILD.gn
@@ -48,7 +48,6 @@ source_set("browser") {
     "//content/public/browser",
     "//content/public/common",
     "//net",
-    "//services/metrics/public/cpp:ukm_builders",
     "//ui/base",
   ]
 }
@@ -71,7 +70,6 @@ source_set("test_support") {
     "//base/test:test_support",
     "//components/page_load_metrics/common:page_load_metrics_mojom",
     "//components/page_load_metrics/common:test_support",
-    "//components/ukm:test_support",
     "//content/public/browser",
     "//content/public/common",
     "//content/test:test_support",
@@ -98,11 +96,8 @@ source_set("unit_tests") {
     "//base/test:test_support",
     "//components/page_load_metrics/common:page_load_metrics_mojom",
     "//components/page_load_metrics/common:test_support",
-    "//components/ukm:test_support",
-    "//components/ukm/content",
     "//content/public/browser",
     "//content/test:test_support",
-    "//services/metrics/public/cpp:ukm_builders",
     "//testing/gtest",
     "//url",
   ]
