diff --git a/components/password_manager/core/browser/password_save_manager.h b/components/password_manager/core/browser/password_save_manager.h
index 61c6ab09c60c..64b0958da77d 100644
--- a/components/password_manager/core/browser/password_save_manager.h
+++ b/components/password_manager/core/browser/password_save_manager.h
@@ -23,7 +23,6 @@ class PasswordManagerClient;
 class FormFetcher;
 class VotesUploader;
 class FormSaver;
-class PasswordFormMetricsRecorder;
 class PasswordManagerDriver;
 struct PasswordForm;
 
@@ -41,7 +40,6 @@ class PasswordSaveManager {
 
   virtual void Init(PasswordManagerClient* client,
                     const FormFetcher* form_fetcher,
-                    scoped_refptr<PasswordFormMetricsRecorder> metrics_recorder,
                     VotesUploader* votes_uploader) = 0;
 
   virtual const PasswordForm& GetPendingCredentials() const = 0;
