diff --git a/components/offline_pages/core/prefetch/BUILD.gn b/components/offline_pages/core/prefetch/BUILD.gn
index 28a349a0c759..bae4e2a224bd 100644
--- a/components/offline_pages/core/prefetch/BUILD.gn
+++ b/components/offline_pages/core/prefetch/BUILD.gn
@@ -13,7 +13,6 @@ static_library("prefetch") {
     "generate_page_bundle_request.h",
     "get_operation_request.cc",
     "get_operation_request.h",
-    "offline_metrics_collector.h",
     "prefetch_background_task.cc",
     "prefetch_background_task.h",
     "prefetch_background_task_handler.h",
@@ -144,7 +143,6 @@ static_library("test_support") {
     "test_download_client.h",
     "test_download_service.cc",
     "test_download_service.h",
-    "test_offline_metrics_collector.h",
     "test_prefetch_dispatcher.cc",
     "test_prefetch_dispatcher.h",
     "test_prefetch_downloader.cc",
