diff --git a/components/shared_highlighting/core/common/BUILD.gn b/components/shared_highlighting/core/common/BUILD.gn
index 0cd5cd6206d1..dbf1af42d55e 100644
--- a/components/shared_highlighting/core/common/BUILD.gn
+++ b/components/shared_highlighting/core/common/BUILD.gn
@@ -38,9 +38,6 @@ source_set("unit_tests") {
   deps = [
     "//base/test:test_support",
     "//components/shared_highlighting/core/common",
-    "//components/ukm:test_support",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//testing/gtest",
     "//url",
   ]
