diff --git a/components/password_manager/core/browser/password_manager.h b/components/password_manager/core/browser/password_manager.h
index ebdf9f995bf8..5152592a5270 100644
--- a/components/password_manager/core/browser/password_manager.h
+++ b/components/password_manager/core/browser/password_manager.h
@@ -26,7 +26,6 @@
 #include "components/password_manager/core/browser/leak_detection/leak_detection_check_factory.h"
 #include "components/password_manager/core/browser/leak_detection_delegate.h"
 #include "components/password_manager/core/browser/password_manager_interface.h"
-#include "components/password_manager/core/browser/password_manager_metrics_recorder.h"
 #include "components/password_manager/core/browser/possible_username_data.h"
 
 class PrefRegistrySimple;
@@ -51,7 +50,6 @@ class PasswordManagerClient;
 class PasswordManagerDriver;
 class PasswordFormManagerForUI;
 class PasswordFormManager;
-class PasswordManagerMetricsRecorder;
 struct PasswordForm;
 struct PossibleUsernameData;
 
@@ -194,15 +192,6 @@ class PasswordManager : public PasswordManagerInterface {
 
 #endif  // defined(UNIT_TEST)
 
-#if !defined(OS_IOS)
-  // Reports the success from the renderer's PasswordAutofillAgent to fill
-  // credentials into a site. This may be called multiple times, but only
-  // the first result will be recorded for each PasswordFormManager.
-  void LogFirstFillingResult(PasswordManagerDriver* driver,
-                             autofill::FormRendererId form_renderer_id,
-                             int32_t result);
-#endif  // !defined(OS_IOS)
-
   // Notifies that Credential Management API function store() is called.
   void NotifyStorePasswordCalled();
 
@@ -287,13 +276,6 @@ class PasswordManager : public PasswordManagerInterface {
   // gone.
   std::unique_ptr<PasswordFormManagerForUI> MoveOwnedSubmittedManager();
 
-  // Records provisional save failure using current |client_| and
-  // |main_frame_url_|.
-  void RecordProvisionalSaveFailure(
-      PasswordManagerMetricsRecorder::ProvisionalSaveFailure failure,
-      const GURL& form_origin,
-      BrowserSavePasswordProgressLogger* logger);
-
   // Returns the manager which manages |form|. |driver| is needed to determine
   // the match. Returns nullptr when no matched manager is found.
   PasswordFormManager* GetMatchedManager(PasswordManagerDriver* driver,
