diff --git a/components/optimization_guide/proto/BUILD.gn b/components/optimization_guide/proto/BUILD.gn
index 51136ae7029a2..2990a9da542e9 100644
--- a/components/optimization_guide/proto/BUILD.gn
+++ b/components/optimization_guide/proto/BUILD.gn
@@ -16,7 +16,6 @@ proto_library("optimization_guide_proto") {
     "delay_competing_low_priority_requests_metadata.proto",
     "hint_cache.proto",
     "hints.proto",
-    "lite_video_metadata.proto",
     "loading_predictor_metadata.proto",
     "models.proto",
     "page_topics_model_metadata.proto",
@@ -34,7 +33,6 @@ if (is_android) {
       "delay_async_script_execution_metadata.proto",
       "delay_competing_low_priority_requests_metadata.proto",
       "hints.proto",
-      "lite_video_metadata.proto",
       "loading_predictor_metadata.proto",
       "models.proto",
       "performance_hints_metadata.proto",
