diff --git a/components/omnibox/browser/search_provider.cc b/components/omnibox/browser/search_provider.cc
index 3d03370544a88..2dfc98dfec8cb 100644
--- a/components/omnibox/browser/search_provider.cc
+++ b/components/omnibox/browser/search_provider.cc
@@ -1594,11 +1594,6 @@ void SearchProvider::PrefetchImages(SearchSuggestionParser::Results* results) {
       suggestion.answer()->AddImageURLsTo(&prefetch_image_urls);
   }
 
-  UMA_HISTOGRAM_EXACT_LINEAR(
-      "Omnibox.SuggestRequest.Success.PrefetchImagesCount",
-      prefetch_image_urls.size(),
-      AutocompleteResult::kMaxAutocompletePositionValue);
-
   for (const GURL& url : prefetch_image_urls)
     client()->PrefetchImage(url);
 }
