diff --git a/components/policy/core/common/policy_loader_common.cc b/components/policy/core/common/policy_loader_common.cc
index 08b7687caa7af..9da5db4c1a350 100644
--- a/components/policy/core/common/policy_loader_common.cc
+++ b/components/policy/core/common/policy_loader_common.cc
@@ -36,7 +36,6 @@ const char* kSensitivePolicies[] = {
     key::kMetricsReportingEnabled,
     key::kNewTabPageLocation,
     key::kPasswordProtectionChangePasswordURL,
-    key::kPasswordProtectionLoginURLs,
     key::kRestoreOnStartup,
     key::kRestoreOnStartupURLs,
     key::kSafeBrowsingForTrustedSourcesEnabled,
