diff --git a/components/background_sync/BUILD.gn b/components/background_sync/BUILD.gn
index 6a3f330984873..4eee2dd008658 100644
--- a/components/background_sync/BUILD.gn
+++ b/components/background_sync/BUILD.gn
@@ -7,8 +7,6 @@ static_library("background_sync") {
     "background_sync_controller_impl.cc",
     "background_sync_controller_impl.h",
     "background_sync_delegate.h",
-    "background_sync_metrics.cc",
-    "background_sync_metrics.h",
     "background_sync_permission_context.cc",
     "background_sync_permission_context.h",
   ]
@@ -19,7 +17,6 @@ static_library("background_sync") {
     "//components/permissions",
     "//components/variations",
     "//content/public/browser",
-    "//services/metrics/public/cpp:ukm_builders",
     "//third_party/blink/public/common:headers",
   ]
   if (!is_android) {
