diff --git a/components/password_manager/core/browser/password_ui_utils.cc b/components/password_manager/core/browser/password_ui_utils.cc
index 163062b94ddf..3b31353d4b5d 100644
--- a/components/password_manager/core/browser/password_ui_utils.cc
+++ b/components/password_manager/core/browser/password_ui_utils.cc
@@ -15,7 +15,6 @@
 #include "components/password_manager/core/browser/android_affiliation/affiliation_utils.h"
 #include "components/password_manager/core/browser/password_form.h"
 #include "components/password_manager/core/browser/password_form_manager_for_ui.h"
-#include "components/password_manager/core/browser/password_form_metrics_recorder.h"
 #include "components/url_formatter/elide_url.h"
 
 namespace password_manager {
@@ -83,19 +82,9 @@ void UpdatePasswordFormUsernameAndPassword(
   bool password_changed = pending_credentials.password_value != password;
   if (username_edited) {
     form_manager->OnUpdateUsernameFromPrompt(username);
-    if (form_manager->GetMetricsRecorder()) {
-      form_manager->GetMetricsRecorder()->RecordDetailedUserAction(
-          password_manager::PasswordFormMetricsRecorder::DetailedUserAction::
-              kEditedUsernameInBubble);
-    }
   }
   if (password_changed) {
     form_manager->OnUpdatePasswordFromPrompt(password);
-    if (form_manager->GetMetricsRecorder()) {
-      form_manager->GetMetricsRecorder()->RecordDetailedUserAction(
-          password_manager::PasswordFormMetricsRecorder::DetailedUserAction::
-              kSelectedDifferentPasswordInBubble);
-    }
   }
 
   // Values of this histogram are a bit mask. Only the lower two bits are used:
