diff --git a/components/safe_browsing/core/db/hit_report.h b/components/safe_browsing/core/db/hit_report.h
index 8bc0796c277d..6edf5bb77eb7 100644
--- a/components/safe_browsing/core/db/hit_report.h
+++ b/components/safe_browsing/core/db/hit_report.h
@@ -7,7 +7,6 @@
 #ifndef COMPONENTS_SAFE_BROWSING_CORE_DB_HIT_REPORT_H_
 #define COMPONENTS_SAFE_BROWSING_CORE_DB_HIT_REPORT_H_
 
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "components/safe_browsing/core/db/util.h"
 #include "url/gurl.h"
 
@@ -44,7 +43,6 @@ struct HitReport {
   // NOTE(vakh): Unused at the moment, but may be used later.
   std::string population_id;
 
-  ExtendedReportingLevel extended_reporting_level;
   bool is_enhanced_protection = false;
   bool is_metrics_reporting_active;
 
