diff --git a/components/password_manager/core/browser/password_form_filling.cc b/components/password_manager/core/browser/password_form_filling.cc
index bcc720dce57a..171102c64046 100644
--- a/components/password_manager/core/browser/password_form_filling.cc
+++ b/components/password_manager/core/browser/password_form_filling.cc
@@ -126,8 +126,7 @@ LikelyFormFilling SendFillInformationToRenderer(
     const PasswordForm& observed_form,
     const std::vector<const PasswordForm*>& best_matches,
     const std::vector<const PasswordForm*>& federated_matches,
-    const PasswordForm* preferred_match,
-    PasswordFormMetricsRecorder* metrics_recorder) {
+    const PasswordForm* preferred_match) {
   DCHECK(driver);
   DCHECK_EQ(PasswordForm::Scheme::kHtml, observed_form.scheme);
 
@@ -147,8 +146,6 @@ LikelyFormFilling SendFillInformationToRenderer(
         client->GetPasswordFeatureManager()->ShouldShowAccountStorageReSignin(
             client->GetLastCommittedURL());
     driver->InformNoSavedCredentials(should_show_popup_without_passwords);
-    metrics_recorder->RecordFillEvent(
-        PasswordFormMetricsRecorder::kManagerFillEventNoCredential);
     return LikelyFormFilling::kNoFilling;
   }
   DCHECK(preferred_match);
@@ -194,29 +191,9 @@ LikelyFormFilling SendFillInformationToRenderer(
     wait_for_username_reason = WaitForUsernameReason::kFoasFeature;
   }
 
-  // Record no "FirstWaitForUsernameReason" metrics for a form that is not meant
-  // for filling. The renderer won't record a "FirstFillingResult" either.
-  if (!no_sign_in_form) {
-    metrics_recorder->RecordFirstWaitForUsernameReason(
-        wait_for_username_reason);
-  }
-
   bool wait_for_username =
       wait_for_username_reason != WaitForUsernameReason::kDontWait;
 
-  if (wait_for_username) {
-    metrics_recorder->SetManagerAction(
-        PasswordFormMetricsRecorder::kManagerActionNone);
-    metrics_recorder->RecordFillEvent(
-        PasswordFormMetricsRecorder::kManagerFillEventBlockedOnInteraction);
-  } else {
-    metrics_recorder->SetManagerAction(
-        PasswordFormMetricsRecorder::kManagerActionAutofilled);
-    metrics_recorder->RecordFillEvent(
-        PasswordFormMetricsRecorder::kManagerFillEventAutofilled);
-    base::RecordAction(base::UserMetricsAction("PasswordManager_Autofilled"));
-  }
-
   // Continue with autofilling any password forms as traditionally has been
   // done.
   Autofill(client, driver, observed_form, best_matches, federated_matches,
