diff --git a/components/permissions/permissions_client.h b/components/permissions/permissions_client.h
index 0cc6da8ec5772..0d38974b92bc8 100644
--- a/components/permissions/permissions_client.h
+++ b/components/permissions/permissions_client.h
@@ -63,11 +63,6 @@ class PermissionsClient {
   virtual scoped_refptr<content_settings::CookieSettings> GetCookieSettings(
       content::BrowserContext* browser_context) = 0;
 
-  // Retrieves the subresource filter activation from browser website settings.
-  virtual bool IsSubresourceFilterActivated(
-      content::BrowserContext* browser_context,
-      const GURL& url) = 0;
-
   // Retrieves the PermissionDecisionAutoBlocker for this context. The returned
   // pointer has the same lifetime as |browser_context|.
   virtual PermissionDecisionAutoBlocker* GetPermissionDecisionAutoBlocker(
