diff --git a/components/security_interstitials/core/ssl_error_ui.cc b/components/security_interstitials/core/ssl_error_ui.cc
index 575476a1e352b..1ba4141c28758 100644
--- a/components/security_interstitials/core/ssl_error_ui.cc
+++ b/components/security_interstitials/core/ssl_error_ui.cc
@@ -89,10 +89,11 @@ void SSLErrorUI::PopulateStringsForHTML(base::DictionaryValue* load_time_data) {
   load_time_data->SetBoolean("show_recurrent_error_paragraph",
                              controller_->HasSeenRecurrentError());
 
-  if (soft_override_enabled_)
-    PopulateOverridableStrings(load_time_data);
-  else
-    PopulateNonOverridableStrings(load_time_data);
+  // if (soft_override_enabled_)
+  //   PopulateOverridableStrings(load_time_data);
+  // else
+  //   PopulateNonOverridableStrings(load_time_data);
+  PopulateOverridableStrings(load_time_data);
 }
 
 const net::SSLInfo& SSLErrorUI::ssl_info() const {
