diff --git a/components/blocked_content/popup_tracker.cc b/components/blocked_content/popup_tracker.cc
index f949085b5244..4ad77dbd46ca 100644
--- a/components/blocked_content/popup_tracker.cc
+++ b/components/blocked_content/popup_tracker.cc
@@ -11,12 +11,8 @@
 #include "base/metrics/histogram_macros.h"
 #include "base/time/default_tick_clock.h"
 #include "components/blocked_content/popup_opener_tab_helper.h"
-#include "components/ukm/content/source_url_recorder.h"
 #include "content/public/browser/navigation_handle.h"
 #include "content/public/browser/web_contents.h"
-#include "services/metrics/public/cpp/metrics_utils.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
 
 namespace blocked_content {
 namespace {
@@ -54,13 +50,6 @@ PopupTracker::PopupTracker(content::WebContents* contents,
       window_open_disposition_(disposition) {
   if (auto* popup_opener = PopupOpenerTabHelper::FromWebContents(opener))
     popup_opener->OnOpenedPopup(this);
-
-  auto* observation_manager =
-      subresource_filter::SubresourceFilterObserverManager::FromWebContents(
-          contents);
-  if (observation_manager) {
-    scoped_observation_.Observe(observation_manager);
-  }
 }
 
 void PopupTracker::WebContentsDestroyed() {
@@ -172,7 +161,6 @@ void PopupTracker::OnSafeBrowsingChecksComplete(
 }
 
 void PopupTracker::OnSubresourceFilterGoingAway() {
-  scoped_observation_.RemoveObservation();
 }
 
 WEB_CONTENTS_USER_DATA_KEY_IMPL(PopupTracker)
