diff --git a/components/page_info/BUILD.gn b/components/page_info/BUILD.gn
index 1de931eaf7a40..375f2a9cd446d 100644
--- a/components/page_info/BUILD.gn
+++ b/components/page_info/BUILD.gn
@@ -23,10 +23,6 @@ static_library("page_info") {
     "//components/password_manager/core/browser",
     "//components/permissions",
     "//components/prefs",
-    "//components/safe_browsing:buildflags",
-    "//components/safe_browsing/content/password_protection",
-    "//components/safe_browsing/core:csd_proto",
-    "//components/safe_browsing/core/password_protection:password_protection_metrics_util",
     "//components/security_interstitials/content:security_interstitial_page",
     "//components/security_interstitials/core",
     "//components/security_state/core",
@@ -34,13 +30,10 @@ static_library("page_info") {
     "//components/ssl_errors:ssl_errors",
     "//components/strings:components_chromium_strings_grit",
     "//components/strings:components_strings_grit",
-    "//components/subresource_filter/core/browser",
-    "//components/ukm/content",
     "//components/url_formatter",
     "//components/vector_icons:vector_icons",
     "//content/public/browser:browser",
     "//services/device/public/cpp:device_features",
-    "//services/metrics/public/cpp:ukm_builders",
   ]
 
   if (is_android) {
