diff --git a/components/exo/text_input.h b/components/exo/text_input.h
index 1fb945552c71..7a769c8e67eb 100644
--- a/components/exo/text_input.h
+++ b/components/exo/text_input.h
@@ -133,7 +133,6 @@ class TextInput : public ui::TextInputClient,
   void EnsureCaretNotInRect(const gfx::Rect& rect) override;
   bool IsTextEditCommandEnabled(ui::TextEditCommand command) const override;
   void SetTextEditCommandForNextKeyEvent(ui::TextEditCommand command) override;
-  ukm::SourceId GetClientSourceForMetrics() const override;
   bool ShouldDoLearning() override;
   bool SetCompositionFromExistingText(
       const gfx::Range& range,
