diff --git a/components/content_settings/browser/BUILD.gn b/components/content_settings/browser/BUILD.gn
index 4a1cc00f444f7..8249cefb32ca9 100644
--- a/components/content_settings/browser/BUILD.gn
+++ b/components/content_settings/browser/BUILD.gn
@@ -16,7 +16,6 @@ source_set("browser") {
     "//components/content_settings/common:mojom",
     "//components/content_settings/core/browser",
     "//components/content_settings/core/common",
-    "//components/page_load_metrics/browser",
     "//components/prefs",
     "//components/security_state/core",
     "//components/url_formatter",
