diff --git a/components/autofill/core/browser/payments/local_card_migration_manager.h b/components/autofill/core/browser/payments/local_card_migration_manager.h
index 3f2c73c37207..129bfbdbe744 100644
--- a/components/autofill/core/browser/payments/local_card_migration_manager.h
+++ b/components/autofill/core/browser/payments/local_card_migration_manager.h
@@ -13,7 +13,6 @@
 
 #include "base/strings/string16.h"
 #include "components/autofill/core/browser/autofill_client.h"
-#include "components/autofill/core/browser/autofill_metrics.h"
 #include "components/autofill/core/browser/payments/legal_message_line.h"
 #include "components/autofill/core/browser/payments/local_card_migration_strike_database.h"
 #include "components/autofill/core/browser/payments/payments_client.h"
@@ -241,9 +240,6 @@ class LocalCardMigrationManager {
   // on the main dialog.
   bool user_accepted_main_migration_dialog_ = false;
 
-  // Record the triggering source of the local card migration.
-  AutofillMetrics::LocalCardMigrationOrigin local_card_migration_origin_;
-
   // Initialized only during tests.
   ObserverForTest* observer_for_testing_ = nullptr;
 
