diff --git a/components/javascript_dialogs/tab_modal_dialog_manager.h b/components/javascript_dialogs/tab_modal_dialog_manager.h
index 549dc5725a2ce..31c2e396893d4 100644
--- a/components/javascript_dialogs/tab_modal_dialog_manager.h
+++ b/components/javascript_dialogs/tab_modal_dialog_manager.h
@@ -133,9 +133,6 @@ class TabModalDialogManager
       content::WebContents* web_contents,
       std::unique_ptr<TabModalDialogManagerDelegate> delegate);
 
-  // Logs the cause of a dialog dismissal in UMA.
-  void LogDialogDismissalCause(DismissalCause cause);
-
   // Handles the case when the user switches away from a tab.
   void HandleTabSwitchAway(DismissalCause cause);
 
