diff --git a/components/arc/mojom/BUILD.gn b/components/arc/mojom/BUILD.gn
index 2cebdecd8b29..e9d083723fac 100644
--- a/components/arc/mojom/BUILD.gn
+++ b/components/arc/mojom/BUILD.gn
@@ -30,7 +30,6 @@ if (is_chromeos_ash) {
       "crash_collector.mojom",
       "digital_goods.mojom",
       "disk_quota.mojom",
-      "enterprise_reporting.mojom",
       "file_system.mojom",
       "iio_sensor.mojom",
       "ime.mojom",
