diff --git a/components/discardable_memory/client/BUILD.gn b/components/discardable_memory/client/BUILD.gn
index 9ed7a020abd0..b5ee03c07d80 100644
--- a/components/discardable_memory/client/BUILD.gn
+++ b/components/discardable_memory/client/BUILD.gn
@@ -16,7 +16,6 @@ component("client") {
 
   deps = [
     "//base",
-    "//components/crash/core/common:crash_key",
     "//components/discardable_memory/common",
     "//components/discardable_memory/public/mojom",
   ]
