diff --git a/components/no_state_prefetch/browser/prerender_util.h b/components/no_state_prefetch/browser/prerender_util.h
index 77a3cc54eb94..e8dd283e79e6 100644
--- a/components/no_state_prefetch/browser/prerender_util.h
+++ b/components/no_state_prefetch/browser/prerender_util.h
@@ -5,8 +5,6 @@
 #ifndef COMPONENTS_NO_STATE_PREFETCH_BROWSER_PRERENDER_UTIL_H_
 #define COMPONENTS_NO_STATE_PREFETCH_BROWSER_PRERENDER_UTIL_H_
 
-#include "services/metrics/public/cpp/ukm_source_id.h"
-
 class GURL;
 
 namespace content {
@@ -20,13 +18,6 @@ class NoStatePrefetchManager;
 // Indicates whether the URL provided is a GWS origin.
 bool IsGoogleOriginURL(const GURL& origin_url);
 
-// Records the metrics for the nostate prefetch to an event with UKM source ID
-// |source_id|.
-void RecordNoStatePrefetchMetrics(
-    content::NavigationHandle* navigation_handle,
-    ukm::SourceId source_id,
-    prerender::NoStatePrefetchManager* no_state_prefetch_manager);
-
 }  // namespace prerender
 
 #endif  // COMPONENTS_NO_STATE_PREFETCH_BROWSER_PRERENDER_UTIL_H_
