diff --git a/components/autofill/core/browser/personal_data_manager.h b/components/autofill/core/browser/personal_data_manager.h
index eea8c5166585..c414e1b93ea8 100644
--- a/components/autofill/core/browser/personal_data_manager.h
+++ b/components/autofill/core/browser/personal_data_manager.h
@@ -390,10 +390,6 @@ class PersonalDataManager : public KeyedService,
   // Records the sync transport consent.
   void OnUserAcceptedCardsFromAccountOption();
 
-  // Logs the fact that the server card link was clicked including information
-  // about the current sync state.
-  void LogServerCardLinkClicked() const;
-
   // Records the sync transport consent if the user is in sync transport mode.
   virtual void OnUserAcceptedUpstreamOffer();
 
