diff --git a/components/paint_preview/browser/paint_preview_client.h b/components/paint_preview/browser/paint_preview_client.h
index 1b3537afcc534..2497b820033ce 100644
--- a/components/paint_preview/browser/paint_preview_client.h
+++ b/components/paint_preview/browser/paint_preview_client.h
@@ -108,9 +108,6 @@ class PaintPreviewClient
     bool capture_links = true;
     size_t max_per_capture_size = 0;
 
-    // UKM Source ID of the WebContent.
-    ukm::SourceId source_id;
-
     // Main frame capture time.
     base::TimeDelta main_frame_blink_recording_time;
 
