diff --git a/components/components_strings.grd b/components/components_strings.grd
index 5bd448ca54a4..dac2288a246b 100644
--- a/components/components_strings.grd
+++ b/components/components_strings.grd
@@ -308,7 +308,6 @@
       <part file="print_media_strings.grdp" />
       <part file="printing_component_strings.grdp" />
       <part file="reset_password_strings.grdp" />
-      <part file="safe_browsing_strings.grdp" />
       <part file="security_interstitials_strings.grdp" />
       <part file="security_state_strings.grdp" />
       <part file="send_tab_to_self_strings.grdp" />
