diff --git a/components/autofill/core/browser/BUILD.gn b/components/autofill/core/browser/BUILD.gn
index 3447ad7a3033..3e2467ebf5ea 100644
--- a/components/autofill/core/browser/BUILD.gn
+++ b/components/autofill/core/browser/BUILD.gn
@@ -59,8 +59,6 @@ static_library("browser") {
     "autofill_manager.cc",
     "autofill_manager.h",
     "autofill_manager_test_delegate.h",
-    "autofill_metrics.cc",
-    "autofill_metrics.h",
     "autofill_observer.cc",
     "autofill_observer.h",
     "autofill_profile_sync_util.cc",
@@ -176,13 +174,6 @@ static_library("browser") {
     "logging/log_receiver.h",
     "logging/log_router.cc",
     "logging/log_router.h",
-    "metrics/address_form_event_logger.cc",
-    "metrics/address_form_event_logger.h",
-    "metrics/credit_card_form_event_logger.cc",
-    "metrics/credit_card_form_event_logger.h",
-    "metrics/form_event_logger_base.cc",
-    "metrics/form_event_logger_base.h",
-    "metrics/form_events.h",
     "pattern_provider/pattern_provider.cc",
     "pattern_provider/pattern_provider.h",
     "payments/account_info_getter.h",
@@ -309,13 +300,6 @@ static_library("browser") {
     ]
   }
 
-  if (is_ios) {
-    sources += [
-      "keyboard_accessory_metrics_logger.h",
-      "keyboard_accessory_metrics_logger.mm",
-    ]
-  }
-
   if (is_ios || is_android) {
     sources += [
       "payments/autofill_credit_card_filling_infobar_delegate_mobile.cc",
@@ -371,7 +355,6 @@ static_library("browser") {
     "//components/history/core/browser",
     "//components/infobars/core",
     "//components/keyed_service/core",
-    "//components/language_usage_metrics",
     "//components/leveldb_proto",
     "//components/os_crypt",
     "//components/policy/core/browser",
@@ -390,8 +373,6 @@ static_library("browser") {
     "//crypto",
     "//google_apis",
     "//net",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/cpp",
     "//services/network/public/mojom",
     "//sql",
@@ -514,8 +495,6 @@ static_library("test_support") {
     "//components/sync:test_support",
     "//components/translate/core/browser",
     "//components/translate/core/browser:test_support",
-    "//components/ukm",
-    "//components/ukm:test_support",
     "//google_apis:test_support",
     "//services/network:test_support",
     "//services/network/public/cpp",
@@ -587,7 +566,6 @@ source_set("unit_tests") {
     "autofill_external_delegate_unittest.cc",
     "autofill_manager_unittest.cc",
     "autofill_merge_unittest.cc",
-    "autofill_metrics_unittest.cc",
     "autofill_profile_sync_util_unittest.cc",
     "autofill_profile_validation_util_unittest.cc",
     "autofill_profile_validator_unittest.cc",
@@ -716,8 +694,6 @@ source_set("unit_tests") {
     "//components/sync:test_support_model",
     "//components/sync/driver:test_support",
     "//components/translate/core/browser",
-    "//components/ukm",
-    "//components/ukm:test_support",
     "//components/unified_consent",
     "//components/variations",
     "//components/variations:test_support",
@@ -727,7 +703,6 @@ source_set("unit_tests") {
     "//google_apis",
     "//google_apis:test_support",
     "//net:test_support",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network:test_support",
     "//services/network/public/cpp",
     "//sql",
