diff --git a/components/lookalikes/core/BUILD.gn b/components/lookalikes/core/BUILD.gn
index fccb48046c86..badd30f5a139 100644
--- a/components/lookalikes/core/BUILD.gn
+++ b/components/lookalikes/core/BUILD.gn
@@ -23,8 +23,6 @@ static_library("core") {
     "//components/url_formatter/spoof_checks/top_domains:top500_domains",
     "//components/url_formatter/spoof_checks/top_domains:top500_domains_header",
     "//net",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//ui/base",
   ]
 }
