diff --git a/components/blocked_content/popup_tracker.h b/components/blocked_content/popup_tracker.h
index 29483926eb67..9e35434c4e7e 100644
--- a/components/blocked_content/popup_tracker.h
+++ b/components/blocked_content/popup_tracker.h
@@ -9,8 +9,6 @@
 #include "base/optional.h"
 #include "base/scoped_observer.h"
 #include "base/time/time.h"
-#include "components/subresource_filter/content/browser/subresource_filter_observer.h"
-#include "components/subresource_filter/content/browser/subresource_filter_observer_manager.h"
 #include "content/public/browser/web_contents_observer.h"
 #include "content/public/browser/web_contents_user_data.h"
 #include "services/metrics/public/cpp/ukm_source_id.h"
@@ -28,8 +26,7 @@ namespace blocked_content {
 // TODO(csharrison): Consider adding more metrics like total visibility for the
 // lifetime of the WebContents.
 class PopupTracker : public content::WebContentsObserver,
-                     public content::WebContentsUserData<PopupTracker>,
-                     public subresource_filter::SubresourceFilterObserver {
+                     public content::WebContentsUserData<PopupTracker> {
  public:
   // These values are persisted to logs. Entries should not be renumbered and
   // numeric values should never be reused.
@@ -61,17 +58,6 @@ class PopupTracker : public content::WebContentsObserver,
   void OnVisibilityChanged(content::Visibility visibility) override;
   void DidGetUserInteraction(const blink::WebInputEvent& event) override;
 
-  // subresource_filter::SubresourceFilterObserver:
-  void OnSafeBrowsingChecksComplete(
-      content::NavigationHandle* navigation_handle,
-      const subresource_filter::SubresourceFilterSafeBrowsingClient::
-          CheckResult& result) override;
-  void OnSubresourceFilterGoingAway() override;
-
-  ScopedObserver<subresource_filter::SubresourceFilterObserverManager,
-                 subresource_filter::SubresourceFilterObserver>
-      scoped_observer_;
-
   // Will be unset until the first navigation commits. Will be set to the total
   // time the contents was visible at commit time.
   base::Optional<base::TimeDelta> first_load_visible_time_start_;
