diff --git a/components/media_message_center/media_session_notification_item.cc b/components/media_message_center/media_session_notification_item.cc
index 805590d1a7e1..4b880eec22f4 100644
--- a/components/media_message_center/media_session_notification_item.cc
+++ b/components/media_message_center/media_session_notification_item.cc
@@ -155,7 +155,6 @@ void MediaSessionNotificationItem::OnMediaSessionActionButtonPressed(
   if (frozen_)
     return;
 
-  controller_->LogMediaSessionActionButtonPressed(request_id_, action);
   media_session::PerformMediaSessionAction(action, media_controller_remote_);
 }
 
