diff --git a/components/gwp_asan/client/BUILD.gn b/components/gwp_asan/client/BUILD.gn
index e4e59ea9b1b4..091bb50abdc1 100644
--- a/components/gwp_asan/client/BUILD.gn
+++ b/components/gwp_asan/client/BUILD.gn
@@ -42,13 +42,8 @@ component("client") {
   deps = [
     "//base",
     "//base/allocator:buildflags",
-    "//components/crash/core/common:crash_key",
     "//components/gwp_asan/common",
   ]
-
-  if (is_android) {
-    deps += [ "//components/crash/core/app" ]
-  }
 }
 
 source_set("unit_tests") {
@@ -70,7 +65,6 @@ source_set("unit_tests") {
     ":client",
     "//base/allocator:buildflags",
     "//base/test:test_support",
-    "//components/crash/core/common:crash_key",
     "//components/gwp_asan/common",
     "//testing/gmock",
     "//testing/gtest",
