diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index b76e070130ef..f653d1d84a1e 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -107,8 +107,6 @@ static_library("metrics") {
     "metrics_switches.h",
     "metrics_upload_scheduler.cc",
     "metrics_upload_scheduler.h",
-    "persistent_histograms.cc",
-    "persistent_histograms.h",
     "persistent_system_profile.cc",
     "persistent_system_profile.h",
     "reporting_service.cc",
