diff --git a/components/variations/BUILD.gn b/components/variations/BUILD.gn
index 5d97b39c222c..4252ab237223 100644
--- a/components/variations/BUILD.gn
+++ b/components/variations/BUILD.gn
@@ -67,8 +67,6 @@ static_library("variations") {
     "variations_associated_data.cc",
     "variations_associated_data.h",
     "variations_client.h",
-    "variations_crash_keys.cc",
-    "variations_crash_keys.h",
     "variations_id_collection.cc",
     "variations_id_collection.h",
     "variations_ids_provider.cc",
@@ -182,7 +180,6 @@ source_set("unit_tests") {
     "study_filtering_unittest.cc",
     "synthetic_trial_registry_unittest.cc",
     "variations_associated_data_unittest.cc",
-    "variations_crash_keys_unittest.cc",
     "variations_id_collection_unittest.cc",
     "variations_ids_provider_unittest.cc",
     "variations_murmur_hash_unittest.cc",
@@ -204,7 +201,6 @@ source_set("unit_tests") {
     "net",
     "proto",
     "//base/test:test_support",
-    "//components/crash/core/common:crash_key",
     "//components/prefs:test_support",
     "//components/variations/field_trial_config:field_trial_config",
     "//services/network/public/cpp:cpp_base",
