diff --git a/components/autofill_assistant/browser/BUILD.gn b/components/autofill_assistant/browser/BUILD.gn
index 7498c5f33dde6..a32e99660139c 100644
--- a/components/autofill_assistant/browser/BUILD.gn
+++ b/components/autofill_assistant/browser/BUILD.gn
@@ -282,14 +282,11 @@ static_library("browser") {
     "//components/password_manager/core/browser/form_parsing:form_parsing",
     "//components/signin/public/identity_manager",
     "//components/strings:components_strings_grit",
-    "//components/ukm/content:content",
     "//components/url_matcher",
     "//components/version_info",
     "//content/public/browser",
     "//google_apis",
     "//net",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//third_party/re2",
   ]
 }
