diff --git a/components/viz/service/BUILD.gn b/components/viz/service/BUILD.gn
index 92acbc3a25c6..db39b0b89416 100644
--- a/components/viz/service/BUILD.gn
+++ b/components/viz/service/BUILD.gn
@@ -186,7 +186,6 @@ viz_component("service") {
     "//build:chromecast_buildflags",
     "//cc/base",
     "//cc/paint",
-    "//components/crash/core/common:crash_key",
     "//gpu/command_buffer/client:gles2_cmd_helper",
     "//gpu/command_buffer/client:gles2_implementation",
     "//gpu/command_buffer/client:raster",
@@ -256,7 +255,7 @@ viz_component("service") {
     ]
 
     deps += [ "//ui/accelerated_widget_mac" ]
-    frameworks = [ "IOSurface.framework" ]
+    frameworks = [ "CoreFoundation.framework", "IOSurface.framework" ]
   }
 
   if (is_android || use_ozone) {
