diff --git a/components/component_updater/installer_policies/BUILD.gn b/components/component_updater/installer_policies/BUILD.gn
index 92a11b9f7837..3e730bcfeeeb 100644
--- a/components/component_updater/installer_policies/BUILD.gn
+++ b/components/component_updater/installer_policies/BUILD.gn
@@ -8,8 +8,6 @@ static_library("installer_policies") {
     "on_device_head_suggest_component_installer.h",
     "optimization_hints_component_installer.cc",
     "optimization_hints_component_installer.h",
-    "safety_tips_component_installer.cc",
-    "safety_tips_component_installer.h",
   ]
 
   deps = [
@@ -18,8 +16,6 @@ static_library("installer_policies") {
     "//components/omnibox/browser",
     "//components/omnibox/common",
     "//components/optimization_guide/core",
-    "//components/reputation/core",
-    "//components/reputation/core:proto",
   ]
 }
 
