diff --git a/components/remote_cocoa/app_shim/BUILD.gn b/components/remote_cocoa/app_shim/BUILD.gn
index a9bf35e0cddc..c144640d8632 100644
--- a/components/remote_cocoa/app_shim/BUILD.gn
+++ b/components/remote_cocoa/app_shim/BUILD.gn
@@ -58,7 +58,6 @@ component("app_shim") {
   deps = [
     "//base",
     "//base:i18n",
-    "//components/crash/core/common",
     "//components/remote_cocoa/common:mojo",
     "//mojo/public/cpp/bindings",
     "//net",
