diff --git a/components/performance_manager/performance_manager_tab_helper.cc b/components/performance_manager/performance_manager_tab_helper.cc
index b4dc3e9ccb48..749da02917cb 100644
--- a/components/performance_manager/performance_manager_tab_helper.cc
+++ b/components/performance_manager/performance_manager_tab_helper.cc
@@ -509,12 +509,6 @@ void PerformanceManagerTabHelper::OnMainFrameNavigation(int64_t navigation_id,
   last_navigation_id_ = navigation_id;
   if (!same_doc)
     last_new_doc_navigation_id_ = navigation_id;
-  ukm_source_id_ =
-      ukm::ConvertToSourceId(navigation_id, ukm::SourceIdType::NAVIGATION_ID);
-  PerformanceManagerImpl::CallOnGraphImpl(
-      FROM_HERE,
-      base::BindOnce(&PageNodeImpl::SetUkmSourceId,
-                     base::Unretained(page_node_.get()), ukm_source_id_));
 
   first_time_title_set_ = false;
   first_time_favicon_set_ = false;
