diff --git a/components/offline_pages/core/prefetch/prefetch_service_impl.h b/components/offline_pages/core/prefetch/prefetch_service_impl.h
index e672e8244c9e..01e9fea528e1 100644
--- a/components/offline_pages/core/prefetch/prefetch_service_impl.h
+++ b/components/offline_pages/core/prefetch/prefetch_service_impl.h
@@ -47,7 +47,6 @@ class PrefetchServiceImpl : public PrefetchService {
 
   // Internal usage only functions.
   void ForceRefreshSuggestions() override;
-  OfflineMetricsCollector* GetOfflineMetricsCollector() override;
   PrefetchDispatcher* GetPrefetchDispatcher() override;
   PrefetchNetworkRequestFactory* GetPrefetchNetworkRequestFactory() override;
   OfflinePageModel* GetOfflinePageModel() override;
