diff --git a/components/download/database/in_progress/download_entry.cc b/components/download/database/in_progress/download_entry.cc
index 176b90db04e1..267e4b2894ed 100644
--- a/components/download/database/in_progress/download_entry.cc
+++ b/components/download/database/in_progress/download_entry.cc
@@ -13,14 +13,10 @@ DownloadEntry::DownloadEntry(const DownloadEntry& other) = default;
 DownloadEntry::DownloadEntry(
     const std::string& guid,
     const std::string& request_origin,
-    DownloadSource download_source,
     bool fetch_error_body,
-    const DownloadUrlParameters::RequestHeadersType& request_headers,
-    int64_t ukm_download_id)
+    const DownloadUrlParameters::RequestHeadersType& request_headers)
     : guid(guid),
       request_origin(request_origin),
-      download_source(download_source),
-      ukm_download_id(ukm_download_id),
       fetch_error_body(fetch_error_body),
       request_headers(request_headers) {}
 
@@ -28,8 +24,6 @@ DownloadEntry::~DownloadEntry() = default;
 
 bool DownloadEntry::operator==(const DownloadEntry& other) const {
   return guid == other.guid && request_origin == other.request_origin &&
-         download_source == other.download_source &&
-         ukm_download_id == other.ukm_download_id &&
          bytes_wasted == other.bytes_wasted &&
          fetch_error_body == other.fetch_error_body &&
          request_headers == other.request_headers;
