diff --git a/components/data_reduction_proxy/core/browser/data_reduction_proxy_settings.cc b/components/data_reduction_proxy/core/browser/data_reduction_proxy_settings.cc
index 67659c93c176a..0fb9cd871feef 100644
--- a/components/data_reduction_proxy/core/browser/data_reduction_proxy_settings.cc
+++ b/components/data_reduction_proxy/core/browser/data_reduction_proxy_settings.cc
@@ -209,9 +209,6 @@ base::Time DataReductionProxySettings::GetLastEnabledTime() const {
 }
 
 void DataReductionProxySettings::RegisterDataReductionProxyFieldTrial() {
-  register_synthetic_field_trial_.Run(
-      "SyntheticDataReductionProxySetting",
-      IsDataReductionProxyEnabled() ? "Enabled" : "Disabled");
 }
 
 void DataReductionProxySettings::OnProxyEnabledPrefChange() {
