diff --git a/components/services/print_compositor/print_compositor_impl.cc b/components/services/print_compositor/print_compositor_impl.cc
index f8f81cc9ae11..f54d5d373495 100644
--- a/components/services/print_compositor/print_compositor_impl.cc
+++ b/components/services/print_compositor/print_compositor_impl.cc
@@ -16,7 +16,6 @@
 #include "base/stl_util.h"
 #include "base/trace_event/trace_event.h"
 #include "build/build_config.h"
-#include "components/crash/core/common/crash_key.h"
 #include "components/discardable_memory/client/client_discardable_shared_memory_manager.h"
 #include "components/services/print_compositor/public/cpp/print_service_mojo_types.h"
 #include "content/public/utility/utility_thread.h"
@@ -223,10 +222,6 @@ void PrintCompositorImpl::CompleteDocumentToPdf(
 }
 
 void PrintCompositorImpl::SetWebContentsURL(const GURL& url) {
-  // Record the most recent url we tried to print. This should be sufficient
-  // for users using print preview by default.
-  static crash_reporter::CrashKeyString<1024> crash_key("main-frame-url");
-  crash_key.Set(url.spec());
 }
 
 void PrintCompositorImpl::SetUserAgent(const std::string& user_agent) {
