diff --git a/components/autofill/core/browser/autofill_handler.cc b/components/autofill/core/browser/autofill_handler.cc
index b8f647955aed..8e8d8bb10eef 100644
--- a/components/autofill/core/browser/autofill_handler.cc
+++ b/components/autofill/core/browser/autofill_handler.cc
@@ -88,7 +88,6 @@ void AutofillHandler::OnFormsSeen(const std::vector<FormData>& forms,
 
   std::vector<const FormData*> new_forms;
   for (const FormData& form : forms) {
-    const auto parse_form_start_time = AutofillTickClock::NowTicks();
     FormStructure* cached_form_structure =
         FindCachedFormByRendererId(form.unique_renderer_id);
     // Autofill used to ignore cache hits for non-credit-card forms. The
@@ -122,8 +121,6 @@ void AutofillHandler::OnFormsSeen(const std::vector<FormData>& forms,
       form_structure->set_form_signature(CalculateFormSignature(form));
 
     new_forms.push_back(&form);
-    AutofillMetrics::LogParseFormTiming(AutofillTickClock::NowTicks() -
-                                        parse_form_start_time);
   }
 
   if (new_forms.empty())
