diff --git a/components/translate/core/browser/BUILD.gn b/components/translate/core/browser/BUILD.gn
index 5198ffb0317a..8f7710c113cc 100644
--- a/components/translate/core/browser/BUILD.gn
+++ b/components/translate/core/browser/BUILD.gn
@@ -67,8 +67,6 @@ static_library("browser") {
     "//components/variations/net",
     "//google_apis",
     "//net",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/cpp:cpp",
     "//third_party/icu",
     "//third_party/metrics_proto",
@@ -122,12 +120,8 @@ source_set("unit_tests") {
     "//components/prefs:test_support",
     "//components/sync_preferences:test_support",
     "//components/translate/core/common",
-    "//components/ukm",
-    "//components/ukm:test_support",
     "//components/variations",
     "//net:test_support",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network:test_support",
     "//services/network/public/cpp:cpp",
     "//ui/base",
@@ -152,7 +146,6 @@ source_set("test_support") {
     "//components/sync_preferences:test_support",
     "//components/translate/core/browser",
     "//components/translate/core/common",
-    "//services/metrics/public/cpp:metrics_cpp",
     "//testing/gmock",
     "//testing/gtest",
     "//third_party/metrics_proto:metrics_proto",
