diff --git a/components/safe_browsing/core/ping_manager.cc b/components/safe_browsing/core/ping_manager.cc
index 93f3a275f68f..7ff6b88e4eff 100644
--- a/components/safe_browsing/core/ping_manager.cc
+++ b/components/safe_browsing/core/ping_manager.cc
@@ -140,7 +140,7 @@ GURL PingManager::SafeBrowsingHitUrl(
          hit_report.threat_type == SB_THREAT_TYPE_URL_CLIENT_SIDE_PHISHING ||
          hit_report.threat_type == SB_THREAT_TYPE_URL_CLIENT_SIDE_MALWARE);
   std::string url =
-      GetReportUrl(config_, "report", &hit_report.extended_reporting_level,
+      GetReportUrl(config_, "report",
                    hit_report.is_enhanced_protection);
   std::string threat_list = "none";
   switch (hit_report.threat_type) {
