diff --git a/components/payments/content/payment_request_state.cc b/components/payments/content/payment_request_state.cc
index ef44f43febc8..cbb615fbc406 100644
--- a/components/payments/content/payment_request_state.cc
+++ b/components/payments/content/payment_request_state.cc
@@ -65,8 +65,7 @@ PaymentRequestState::PaymentRequestState(
     base::WeakPtr<Delegate> delegate,
     const std::string& app_locale,
     autofill::PersonalDataManager* personal_data_manager,
-    ContentPaymentRequestDelegate* payment_request_delegate,
-    JourneyLogger* journey_logger)
+    ContentPaymentRequestDelegate* payment_request_delegate)
     : frame_routing_id_(content::GlobalFrameRoutingId(
           initiator_render_frame_host->GetProcess()->GetID(),
           initiator_render_frame_host->GetRoutingID())),
@@ -77,7 +76,6 @@ PaymentRequestState::PaymentRequestState(
       spec_(spec),
       delegate_(delegate),
       personal_data_manager_(personal_data_manager),
-      journey_logger_(journey_logger),
       are_requested_methods_supported_(
           !spec_->supported_card_networks().empty()),
       payment_request_delegate_(payment_request_delegate),
@@ -171,17 +169,6 @@ bool PaymentRequestState::IsOffTheRecord() const {
 }
 
 void PaymentRequestState::OnPaymentAppCreated(std::unique_ptr<PaymentApp> app) {
-  if (app->type() == PaymentApp::Type::AUTOFILL) {
-    journey_logger_->SetAvailableMethod(
-        JourneyLogger::PaymentMethodCategory::kBasicCard);
-  } else if (base::Contains(app->GetAppMethodNames(), methods::kGooglePay) ||
-             base::Contains(app->GetAppMethodNames(), methods::kAndroidPay)) {
-    journey_logger_->SetAvailableMethod(
-        JourneyLogger::PaymentMethodCategory::kGoogle);
-  } else {
-    journey_logger_->SetAvailableMethod(
-        JourneyLogger::PaymentMethodCategory::kOther);
-  }
   available_apps_.emplace_back(std::move(app));
 }
 
@@ -444,8 +431,6 @@ void PaymentRequestState::AddAutofillPaymentApp(
     return;
 
   available_apps_.push_back(std::move(app));
-  journey_logger_->SetAvailableMethod(
-      JourneyLogger::PaymentMethodCategory::kBasicCard);
 
   if (selected) {
     SetSelectedApp(available_apps_.back()->AsWeakPtr());
@@ -639,9 +624,6 @@ void PaymentRequestState::PopulateProfileCache() {
             ? false
             : profile_comparator()->IsContactInfoComplete(contact_profiles_[0]);
     is_requested_autofill_data_available_ &= has_complete_contact;
-    journey_logger_->SetNumberOfSuggestionsShown(
-        JourneyLogger::Section::SECTION_CONTACT_INFO, contact_profiles_.size(),
-        has_complete_contact);
   }
   if (ShouldShowShippingSection()) {
     bool has_complete_shipping =
@@ -649,9 +631,6 @@ void PaymentRequestState::PopulateProfileCache() {
             ? false
             : profile_comparator()->IsShippingComplete(shipping_profiles_[0]);
     is_requested_autofill_data_available_ &= has_complete_shipping;
-    journey_logger_->SetNumberOfSuggestionsShown(
-        JourneyLogger::Section::SECTION_SHIPPING_ADDRESS,
-        shipping_profiles_.size(), has_complete_shipping);
   }
 }
 
@@ -694,10 +673,6 @@ void PaymentRequestState::SetDefaultProfileSelections() {
   }
 
   SelectDefaultShippingAddressAndNotifyObservers();
-
-  journey_logger_->SetNumberOfSuggestionsShown(
-      JourneyLogger::Section::SECTION_PAYMENT_METHOD, available_apps().size(),
-      selected_app_.get());
 }
 
 void PaymentRequestState::UpdateIsReadyToPayAndNotifyObservers() {
