diff --git a/components/keep_alive_registry/BUILD.gn b/components/keep_alive_registry/BUILD.gn
index 794b93f3c0859..b73919b98d999 100644
--- a/components/keep_alive_registry/BUILD.gn
+++ b/components/keep_alive_registry/BUILD.gn
@@ -16,10 +16,6 @@ source_set("keep_alive_registry") {
   ]
 
   deps = [ "//base" ]
-
-  if (is_win) {
-    deps += [ "//components/browser_watcher:stability_client" ]
-  }
 }
 
 source_set("unit_tests") {
