diff --git a/components/download/internal/common/BUILD.gn b/components/download/internal/common/BUILD.gn
index 44d41af8674b..03bee7676e6e 100644
--- a/components/download/internal/common/BUILD.gn
+++ b/components/download/internal/common/BUILD.gn
@@ -72,7 +72,6 @@ source_set("internal") {
     "//base",
     "//components/download/database",
     "//components/download/public/common:interfaces",
-    "//components/download/quarantine",
     "//components/filename_generation",
     "//components/leveldb_proto",
     "//components/safe_browsing:buildflags",
@@ -177,7 +176,6 @@ source_set("unit_tests") {
     "//components/download/public/common:test_support",
     "//components/leveldb_proto",
     "//components/leveldb_proto:test_support",
-    "//components/ukm:test_support",
     "//crypto",
     "//net",
     "//services/metrics/public/cpp:ukm_builders",
