diff --git a/components/services/storage/public/mojom/cache_storage_control.mojom b/components/services/storage/public/mojom/cache_storage_control.mojom
index 74acfde11e70..d761769e0ff0 100644
--- a/components/services/storage/public/mojom/cache_storage_control.mojom
+++ b/components/services/storage/public/mojom/cache_storage_control.mojom
@@ -39,8 +39,6 @@ interface CacheStorageControl {
   // the cache storage for a particular origin and owner.
   AddReceiver(
       network.mojom.CrossOriginEmbedderPolicy policy,
-      pending_remote<network.mojom.CrossOriginEmbedderPolicyReporter>?
-          coep_reporter,
       url.mojom.Origin origin,
       CacheStorageOwner owner,
       pending_receiver<blink.mojom.CacheStorage> receiver);
