diff --git a/components/password_manager/core/browser/password_save_manager_impl.h b/components/password_manager/core/browser/password_save_manager_impl.h
index 562491b9ff8b0..3e2fe32e9137d 100644
--- a/components/password_manager/core/browser/password_save_manager_impl.h
+++ b/components/password_manager/core/browser/password_save_manager_impl.h
@@ -36,7 +36,6 @@ class PasswordSaveManagerImpl : public PasswordSaveManager {
   // |metrics_recorder| and |votes_uploader| can both be nullptr.
   void Init(PasswordManagerClient* client,
             const FormFetcher* form_fetcher,
-            scoped_refptr<PasswordFormMetricsRecorder> metrics_recorder,
             VotesUploader* votes_uploader) override;
 
   // Create pending credentials from |parsed_submitted_form|, |observed_form|
@@ -157,9 +156,6 @@ class PasswordSaveManagerImpl : public PasswordSaveManager {
   // Handles the user flows related to the generation.
   std::unique_ptr<PasswordGenerationManager> generation_manager_;
 
-  // Takes care of recording metrics and events for |*this|. Can be nullptr.
-  scoped_refptr<PasswordFormMetricsRecorder> metrics_recorder_;
-
   // Can be nullptr.
   VotesUploader* votes_uploader_;
 };
