diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 8255cdfc3392..d944c0d1dc5f 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -20,6 +20,7 @@
 #include "components/ukm/content/source_url_recorder.h"
 #include "content/public/browser/permission_type.h"
 #include "content/public/browser/web_contents.h"
+#include "dumb/base/compiler_specific.h"
 #include "services/metrics/public/cpp/metrics_utils.h"
 #include "services/metrics/public/cpp/ukm_builders.h"
 #include "services/metrics/public/cpp/ukm_recorder.h"
@@ -51,6 +52,7 @@ using content::PermissionType;
 
 namespace {
 
+DUMB_ALLOW_UNUSED_TYPE
 RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
   switch (request_type) {
     case RequestType::kAccessibilityEvents:
@@ -108,8 +110,6 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
   }
 }
 
-const int kPriorCountCap = 10;
-
 std::string GetPermissionRequestString(RequestTypeForUma type) {
   switch (type) {
     case RequestTypeForUma::MULTIPLE:
@@ -178,6 +178,7 @@ void RecordEngagementMetric(const std::vector<PermissionRequest*>& requests,
   base::UmaHistogramPercentageObsoleteDoNotUse(name, engagement_score);
 }
 
+#if 0
 void RecordPermissionActionUkm(
     PermissionAction action,
     PermissionRequestGestureType gesture_type,
@@ -241,6 +242,7 @@ void RecordPermissionActionUkm(
 
   builder.Record(ukm::UkmRecorder::Get());
 }
+#endif
 
 std::string GetPromptDispositionString(
     PermissionPromptDisposition ui_disposition) {
@@ -649,6 +651,7 @@ void PermissionUmaUtil::RecordPermissionAction(
     const content::WebContents* web_contents,
     content::BrowserContext* browser_context,
     base::Optional<PredictionGrantLikelihood> predicted_grant_likelihood) {
+#if 0
   DCHECK(PermissionUtil::IsPermission(permission));
   PermissionDecisionAutoBlocker* autoblocker =
       PermissionsClient::Get()->GetPermissionDecisionAutoBlocker(
@@ -657,21 +660,6 @@ void PermissionUmaUtil::RecordPermissionAction(
       autoblocker->GetDismissCount(requesting_origin, permission);
   int ignore_count = autoblocker->GetIgnoreCount(requesting_origin, permission);
 
-  PermissionsClient::Get()->GetUkmSourceId(
-      browser_context, web_contents, requesting_origin,
-      base::BindOnce(
-          &RecordPermissionActionUkm, action, gesture_type, permission,
-          dismiss_count, ignore_count, source_ui, time_to_decision,
-          ui_disposition, ui_reason,
-          permission == ContentSettingsType::NOTIFICATIONS
-              ? PermissionsClient::Get()
-                    ->HadThreeConsecutiveNotificationPermissionDenies(
-                        browser_context)
-              : base::nullopt,
-          PermissionsClient::Get()->HasPreviouslyAutoRevokedPermission(
-              browser_context, requesting_origin, permission),
-          predicted_grant_likelihood));
-
   switch (permission) {
     case ContentSettingsType::GEOLOCATION:
       base::UmaHistogramEnumeration("Permissions.Action.Geolocation", action,
@@ -744,6 +732,7 @@ void PermissionUmaUtil::RecordPermissionAction(
                    << PermissionUtil::GetPermissionString(permission)
                    << " not accounted for";
   }
+#endif
 }
 
 // static
