diff --git a/components/download/internal/common/BUILD.gn b/components/download/internal/common/BUILD.gn
index 282015c57227..e7b283f51dd0 100644
--- a/components/download/internal/common/BUILD.gn
+++ b/components/download/internal/common/BUILD.gn
@@ -72,7 +72,6 @@ source_set("internal") {
     "//base",
     "//components/download/database",
     "//components/download/public/common:interfaces",
-    "//components/download/quarantine",
     "//components/filename_generation",
     "//components/leveldb_proto",
     "//components/safe_browsing:buildflags",
