diff --git a/components/password_manager/core/browser/password_form_manager.cc b/components/password_manager/core/browser/password_form_manager.cc
index 497cae100e74..7cfca12de75b 100644
--- a/components/password_manager/core/browser/password_form_manager.cc
+++ b/components/password_manager/core/browser/password_form_manager.cc
@@ -626,7 +626,6 @@ std::unique_ptr<PasswordFormManager> PasswordFormManager::Clone() {
   }
   result->is_submitted_ = is_submitted_;
   result->password_save_manager_->Init(result->client_, result->form_fetcher_,
-                                       result->metrics_recorder_,
                                        &result->votes_uploader_);
   return result;
 }
@@ -905,7 +904,7 @@ PasswordFormManager::PasswordFormManager(
         client_->IsCommittedMainFrameSecure(), client_->GetUkmSourceId(),
         client_->GetPrefs());
   }
-  password_save_manager_->Init(client_, form_fetcher_, metrics_recorder_,
+  password_save_manager_->Init(client_, form_fetcher_,
                                &votes_uploader_);
   // TODO(https://crbug.com/1167475): Add test for this metric.
   base::UmaHistogramEnumeration("PasswordManager.FormVisited.PerProfileType",
