diff --git a/components/ntp_tiles/DEPS b/components/ntp_tiles/DEPS
index 3742e149693c..674274de7798 100644
--- a/components/ntp_tiles/DEPS
+++ b/components/ntp_tiles/DEPS
@@ -8,7 +8,6 @@ include_rules = [
   "+components/history/core/test",
   "+components/pref_registry",
   "+components/prefs",
-  "+components/rappor",
   "+components/search_engines",
   "+components/strings/grit/components_strings.h",
   "+components/suggestions",
