diff --git a/components/subresource_filter/content/browser/content_activation_list_utils.h b/components/subresource_filter/content/browser/content_activation_list_utils.h
index 41a7e4590268..791e6e5cc237 100644
--- a/components/subresource_filter/content/browser/content_activation_list_utils.h
+++ b/components/subresource_filter/content/browser/content_activation_list_utils.h
@@ -5,14 +5,11 @@
 #ifndef COMPONENTS_SUBRESOURCE_FILTER_CONTENT_BROWSER_CONTENT_ACTIVATION_LIST_UTILS_H_
 #define COMPONENTS_SUBRESOURCE_FILTER_CONTENT_BROWSER_CONTENT_ACTIVATION_LIST_UTILS_H_
 
-#include "components/safe_browsing/core/db/util.h"
 #include "components/subresource_filter/core/common/activation_list.h"
 
 namespace subresource_filter {
 
 ActivationList GetListForThreatTypeAndMetadata(
-    safe_browsing::SBThreatType threat_type,
-    const safe_browsing::ThreatMetadata& threat_type_metadata,
     bool* warning);
 
 }  // namespace subresource_filter
