diff --git a/components/permissions/permissions_client.cc b/components/permissions/permissions_client.cc
index 02e9d5455283..f0349a9d0e4f 100644
--- a/components/permissions/permissions_client.cc
+++ b/components/permissions/permissions_client.cc
@@ -52,13 +52,6 @@ bool PermissionsClient::IsCookieDeletionDisabled(
 }
 #endif
 
-void PermissionsClient::GetUkmSourceId(content::BrowserContext* browser_context,
-                                       const content::WebContents* web_contents,
-                                       const GURL& requesting_origin,
-                                       GetUkmSourceIdCallback callback) {
-  std::move(callback).Run(base::nullopt);
-}
-
 PermissionRequest::IconId PermissionsClient::GetOverrideIconId(
     ContentSettingsType type) {
 #if defined(OS_ANDROID)
