diff --git a/components/prerender/browser/prerender_util.h b/components/prerender/browser/prerender_util.h
index 038940bd2fc0..d1ea238a2122 100644
--- a/components/prerender/browser/prerender_util.h
+++ b/components/prerender/browser/prerender_util.h
@@ -5,8 +5,6 @@
 #ifndef COMPONENTS_PRERENDER_BROWSER_PRERENDER_UTIL_H_
 #define COMPONENTS_PRERENDER_BROWSER_PRERENDER_UTIL_H_
 
-#include "services/metrics/public/cpp/ukm_source_id.h"
-
 class GURL;
 
 namespace content {
@@ -26,13 +24,6 @@ void ReportPrerenderExternalURL();
 // Report a URL was canceled due to unsupported prerender scheme.
 void ReportUnsupportedPrerenderScheme(const GURL& url);
 
-// Records the metrics for the nostate prefetch to an event with UKM source ID
-// |source_id|.
-void RecordNoStatePrefetchMetrics(
-    content::NavigationHandle* navigation_handle,
-    ukm::SourceId source_id,
-    prerender::PrerenderManager* prerender_manager);
-
 }  // namespace prerender
 
 #endif  // COMPONENTS_PRERENDER_BROWSER_PRERENDER_UTIL_H_
