diff --git a/components/metrics_services_manager/BUILD.gn b/components/metrics_services_manager/BUILD.gn
index 286abdc3ad7f..0fdc175850bc 100644
--- a/components/metrics_services_manager/BUILD.gn
+++ b/components/metrics_services_manager/BUILD.gn
@@ -12,7 +12,6 @@ static_library("metrics_services_manager") {
   deps = [
     "//base",
     "//components/metrics",
-    "//components/rappor",
     "//components/ukm",
     "//components/variations",
     "//components/variations/service",
