diff --git a/components/performance_manager/performance_manager_impl.cc b/components/performance_manager/performance_manager_impl.cc
index b5cfc7f910ba..3601d45c3b17 100644
--- a/components/performance_manager/performance_manager_impl.cc
+++ b/components/performance_manager/performance_manager_impl.cc
@@ -331,7 +331,6 @@ void PerformanceManagerImpl::OnStartImpl(GraphImplCallback on_start) {
   DCHECK(!g_performance_manager);
 
   g_performance_manager = this;
-  graph_.set_ukm_recorder(ukm::UkmRecorder::Get());
   std::move(on_start).Run(&graph_);
 }
 
