diff --git a/components/password_manager/core/browser/password_manager_client.h b/components/password_manager/core/browser/password_manager_client.h
index 56bb574424d7..af9424f48713 100644
--- a/components/password_manager/core/browser/password_manager_client.h
+++ b/components/password_manager/core/browser/password_manager_client.h
@@ -27,7 +27,6 @@
 #include "components/password_manager/core/browser/password_store.h"
 #include "components/safe_browsing/buildflags.h"
 #include "net/cert/cert_status_flags.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
 
 class PrefService;
 
@@ -59,7 +58,7 @@ class Origin;
 
 class GURL;
 
-#if defined(ON_FOCUS_PING_ENABLED)
+#if defined(ON_FOCUS_PING_ENABLED) && BUILDFLAG(FULL_SAFE_BROWSING)
 namespace safe_browsing {
 class PasswordProtectionService;
 }
@@ -72,7 +71,6 @@ class PasswordFeatureManager;
 class BiometricAuthenticator;
 class PasswordFormManagerForUI;
 class PasswordManagerDriver;
-class PasswordManagerMetricsRecorder;
 class HttpAuthManager;
 class PasswordRequirementsService;
 class PasswordStore;
@@ -355,18 +353,6 @@ class PasswordManagerClient {
   // Records a Chrome Sync event that GAIA password reuse was detected.
   virtual void LogPasswordReuseDetectedEvent() = 0;
 #endif
-
-  // Gets a ukm::SourceId that is associated with the WebContents object
-  // and its last committed main frame navigation.
-  virtual ukm::SourceId GetUkmSourceId() = 0;
-
-  // Gets a metrics recorder for the currently committed navigation.
-  // As PasswordManagerMetricsRecorder submits metrics on destruction, a new
-  // instance will be returned for each committed navigation. A caller must not
-  // hold on to the pointer. This method returns a nullptr if the client
-  // does not support metrics recording.
-  virtual PasswordManagerMetricsRecorder* GetMetricsRecorder() = 0;
-
   // Gets the PasswordRequirementsService associated with the client. It is
   // valid that this method returns a nullptr if the PasswordRequirementsService
   // has not been implemented for a specific platform or the context is an
