diff --git a/components/optimization_guide/proto/BUILD.gn b/components/optimization_guide/proto/BUILD.gn
index 8dbebbb19a53..9f305010919f 100644
--- a/components/optimization_guide/proto/BUILD.gn
+++ b/components/optimization_guide/proto/BUILD.gn
@@ -14,7 +14,6 @@ proto_library("optimization_guide_proto") {
     "delay_async_script_execution_metadata.proto",
     "hint_cache.proto",
     "hints.proto",
-    "lite_video_metadata.proto",
     "loading_predictor_metadata.proto",
     "models.proto",
     "performance_hints_metadata.proto",
@@ -30,7 +29,6 @@ if (is_android) {
       "common_types.proto",
       "delay_async_script_execution_metadata.proto",
       "hints.proto",
-      "lite_video_metadata.proto",
       "loading_predictor_metadata.proto",
       "models.proto",
       "performance_hints_metadata.proto",
