diff --git a/components/ukm/content/source_url_recorder.cc b/components/ukm/content/source_url_recorder.cc
index a6de4e2e83cab..17617ae0cc573 100644
--- a/components/ukm/content/source_url_recorder.cc
+++ b/components/ukm/content/source_url_recorder.cc
@@ -365,10 +365,7 @@ void InitializeSourceUrlRecorderForWebContents(
 
 SourceId GetSourceIdForWebContentsDocument(
     const content::WebContents* web_contents) {
-  const internal::SourceUrlRecorderWebContentsObserver* obs =
-      internal::SourceUrlRecorderWebContentsObserver::FromWebContents(
-          web_contents);
-  return obs ? obs->GetLastCommittedSourceId() : kInvalidSourceId;
+  return kInvalidSourceId;
 }
 
 }  // namespace ukm
