diff --git a/components/blocked_content/BUILD.gn b/components/blocked_content/BUILD.gn
index a312dc9d40009..385e4630873dd 100644
--- a/components/blocked_content/BUILD.gn
+++ b/components/blocked_content/BUILD.gn
@@ -17,8 +17,6 @@ source_set("blocked_content") {
     "popup_tracker.h",
     "pref_names.cc",
     "pref_names.h",
-    "safe_browsing_triggered_popup_blocker.cc",
-    "safe_browsing_triggered_popup_blocker.h",
     "url_list_manager.cc",
     "url_list_manager.h",
   ]
@@ -31,13 +29,8 @@ source_set("blocked_content") {
     "//components/embedder_support",
     "//components/pref_registry",
     "//components/prefs",
-    "//components/safe_browsing/content/triggers:ad_popup_trigger",
-    "//components/safe_browsing/core/db:util",
-    "//components/subresource_filter/content/browser",
-    "//components/ukm/content",
     "//components/user_prefs",
     "//content/public/browser",
-    "//services/metrics/public/cpp:ukm_builders",
     "//third_party/blink/public/common",
   ]
   if (is_android) {
