diff --git a/components/metrics/metrics_service_client.h b/components/metrics/metrics_service_client.h
index ea907d78820e..2ab045442493 100644
--- a/components/metrics/metrics_service_client.h
+++ b/components/metrics/metrics_service_client.h
@@ -44,9 +44,6 @@ class MetricsServiceClient {
   // implementation will own the MetricsService instance being returned).
   virtual MetricsService* GetMetricsService() = 0;
 
-  // Returns the UkmService instance that this client is associated with.
-  virtual ukm::UkmService* GetUkmService();
-
   // Registers the client id with other services (e.g. crash reporting), called
   // when metrics recording gets enabled.
   virtual void SetMetricsClientId(const std::string& client_id) = 0;
