diff --git a/components/security_interstitials/content/cert_report_helper.h b/components/security_interstitials/content/cert_report_helper.h
index 047a8b4600a5..3a6ee42b7e69 100644
--- a/components/security_interstitials/content/cert_report_helper.h
+++ b/components/security_interstitials/content/cert_report_helper.h
@@ -109,15 +109,8 @@ class CertReportHelper {
   const GURL request_url_;
   // The SSLInfo used in this helper's report.
   const net::SSLInfo ssl_info_;
-  // The reason for the interstitial, included in this helper's report.
-  CertificateErrorReport::InterstitialReason interstitial_reason_;
-  // True if the user was given the option to proceed through the
-  // certificate chain error being reported.
-  bool overridable_;
   // The time at which the interstitial was constructed.
   const base::Time interstitial_time_;
-  // Helpful for recording metrics about cert reports.
-  security_interstitials::MetricsHelper* metrics_helper_;
   // Appends additional details to a report.
   ClientDetailsCallback client_details_callback_;
   // Default to DID_NOT_PROCEED. If no user action is processed via
