diff --git a/components/password_manager/core/browser/browser_save_password_progress_logger.cc b/components/password_manager/core/browser/browser_save_password_progress_logger.cc
index 152c76e8d622..38b3dcdef8f4 100644
--- a/components/password_manager/core/browser/browser_save_password_progress_logger.cc
+++ b/components/password_manager/core/browser/browser_save_password_progress_logger.cc
@@ -89,7 +89,7 @@ std::string FormSignatureToDebugString(autofill::FormSignature form_signature) {
   return base::StrCat(
       {NumberToString(form_signature.value()), " - ",
        NumberToString(
-           PasswordFormMetricsRecorder::HashFormSignature(form_signature))});
+           static_cast<uint64_t>(form_signature) % 1021)});
 }
 
 BrowserSavePasswordProgressLogger::StringID FormSchemeToStringID(
