diff --git a/components/embedder_support/android/metrics/BUILD.gn b/components/embedder_support/android/metrics/BUILD.gn
index 387ffd0de183..de4ba147f24b 100644
--- a/components/embedder_support/android/metrics/BUILD.gn
+++ b/components/embedder_support/android/metrics/BUILD.gn
@@ -25,7 +25,6 @@ static_library("metrics") {
     "//components/metrics:net",
     "//components/metrics:ui",
     "//components/prefs",
-    "//components/ukm",
     "//components/version_info",
     "//components/version_info/android:channel_getter",
     "//content/public/browser",
