diff --git a/components/component_updater/installer_policies/BUILD.gn b/components/component_updater/installer_policies/BUILD.gn
index 402b5af1caefa..6879f4a692f84 100644
--- a/components/component_updater/installer_policies/BUILD.gn
+++ b/components/component_updater/installer_policies/BUILD.gn
@@ -12,8 +12,6 @@ static_library("installer_policies") {
     "optimization_hints_component_installer.h",
     "origin_trials_component_installer.cc",
     "origin_trials_component_installer.h",
-    "safety_tips_component_installer.cc",
-    "safety_tips_component_installer.h",
     "trust_token_key_commitments_component_installer_policy.cc",
     "trust_token_key_commitments_component_installer_policy.h",
   ]
@@ -26,8 +24,6 @@ static_library("installer_policies") {
     "//components/omnibox/common",
     "//components/optimization_guide/core",
     "//components/prefs:prefs",
-    "//components/reputation/core",
-    "//components/reputation/core:proto",
   ]
 }
 
