diff --git a/content/public/browser/payment_app_provider.h b/content/public/browser/payment_app_provider.h
index e8c4cd0aa97d..be4217fd1152 100644
--- a/content/public/browser/payment_app_provider.h
+++ b/content/public/browser/payment_app_provider.h
@@ -7,7 +7,6 @@
 
 #include "base/callback_forward.h"
 #include "content/common/content_export.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
 #include "third_party/blink/public/mojom/payments/payment_app.mojom.h"
 
 class SkBitmap;
@@ -106,11 +105,6 @@ class CONTENT_EXPORT PaymentAppProvider {
                                             const GURL& sw_scope,
                                             std::string* error_message) = 0;
 
-  // Gets the ukm source id for a payment app with |sw_scope|.
-  // This must ONLY be called when payment app window has been opened.
-  virtual ukm::SourceId GetSourceIdForPaymentAppFromScope(
-      const GURL& sw_scope) = 0;
-
  protected:
   virtual ~PaymentAppProvider() = default;
 };
