diff --git a/content/browser/renderer_host/input/render_widget_host_latency_tracker.cc b/content/browser/renderer_host/input/render_widget_host_latency_tracker.cc
index dd3766c4406d..b3f7805acb02 100644
--- a/content/browser/renderer_host/input/render_widget_host_latency_tracker.cc
+++ b/content/browser/renderer_host/input/render_widget_host_latency_tracker.cc
@@ -209,8 +209,6 @@ void RenderWidgetHostLatencyTracker::OnInputEventAck(
 void RenderWidgetHostLatencyTracker::OnEventStart(ui::LatencyInfo* latency) {
   static uint64_t global_trace_id = 0;
   latency->set_trace_id(++global_trace_id);
-  latency->set_ukm_source_id(
-      render_widget_host_delegate_->GetCurrentPageUkmSourceId());
 }
 
 }  // namespace content
