diff --git a/content/child/BUILD.gn b/content/child/BUILD.gn
index 4b49a079d7038..e6fb1aa0c8e61 100644
--- a/content/child/BUILD.gn
+++ b/content/child/BUILD.gn
@@ -4,6 +4,7 @@
 
 import("//build/config/features.gni")
 import("//build/config/ui.gni")
+import("//dumb/content/child/sources.gni")
 import("//ppapi/buildflags/buildflags.gni")
 import("//third_party/blink/public/public_features.gni")
 if (is_android) {
@@ -114,6 +115,8 @@ target(link_target_type, "child") {
     "//v8",
   ]
 
+  deps += dumb_content_child_deps
+
   if (is_linux || is_chromeos) {
     sources += [
       "child_process_sandbox_support_impl_linux.cc",
