diff --git a/content/browser/webui/content_web_ui_controller_factory.cc b/content/browser/webui/content_web_ui_controller_factory.cc
index 6012229fe08b..d0a353e6193c 100644
--- a/content/browser/webui/content_web_ui_controller_factory.cc
+++ b/content/browser/webui/content_web_ui_controller_factory.cc
@@ -15,7 +15,6 @@
 #include "content/browser/process_internals/process_internals_ui.h"
 #include "content/browser/service_worker/service_worker_internals_ui.h"
 #include "content/browser/tracing/tracing_ui.h"
-#include "content/browser/ukm_internals_ui.h"
 #include "content/browser/webrtc/webrtc_internals_ui.h"
 #include "content/public/browser/storage_partition.h"
 #include "content/public/browser/web_contents.h"
@@ -43,8 +42,7 @@ WebUI::TypeID ContentWebUIControllerFactory::GetWebUIType(
       url.host_piece() == kChromeUIAppCacheInternalsHost ||
       url.host_piece() == kChromeUINetworkErrorsListingHost ||
       url.host_piece() == kChromeUIProcessInternalsHost ||
-      url.host_piece() == kChromeUIConversionInternalsHost ||
-      url.host_piece() == kChromeUIUkmHost) {
+      url.host_piece() == kChromeUIConversionInternalsHost) {
     return const_cast<ContentWebUIControllerFactory*>(this);
   }
   return WebUI::kNoWebUI;
@@ -92,8 +90,6 @@ ContentWebUIControllerFactory::CreateWebUIControllerForURL(WebUI* web_ui,
     return std::make_unique<ProcessInternalsUI>(web_ui);
   if (url.host_piece() == kChromeUIConversionInternalsHost)
     return std::make_unique<ConversionInternalsUI>(web_ui);
-  if (url.host_piece() == kChromeUIUkmHost)
-    return std::make_unique<UkmInternalsUI>(web_ui);
   return nullptr;
 }
 
