diff --git a/content/public/browser/content_browser_client.cc b/content/public/browser/content_browser_client.cc
index e51417f75154b..138554f2787bb 100644
--- a/content/public/browser/content_browser_client.cc
+++ b/content/public/browser/content_browser_client.cc
@@ -61,7 +61,6 @@
 #include "third_party/blink/public/common/loader/url_loader_throttle.h"
 #include "third_party/blink/public/common/renderer_preferences/renderer_preferences.h"
 #include "third_party/blink/public/common/user_agent/user_agent_metadata.h"
-#include "third_party/blink/public/mojom/federated_learning/floc.mojom.h"
 #include "ui/gfx/image/image_skia.h"
 #include "ui/shell_dialogs/select_file_policy.h"
 #include "url/gurl.h"
@@ -318,10 +317,12 @@ std::string ContentBrowserClient::GetApplicationClientGUIDForQuarantineCheck() {
   return std::string();
 }
 
+#if 0
 download::QuarantineConnectionCallback
 ContentBrowserClient::GetQuarantineConnectionCallback() {
   return base::NullCallback();
 }
+#endif
 
 std::string ContentBrowserClient::GetApplicationLocale() {
   return "en-US";
@@ -1100,12 +1101,14 @@ void ContentBrowserClient::AugmentNavigationDownloadPolicy(
     bool user_gesture,
     blink::NavigationDownloadPolicy* download_policy) {}
 
+#if 0
 blink::mojom::InterestCohortPtr ContentBrowserClient::GetInterestCohortForJsApi(
     WebContents* web_contents,
     const GURL& url,
     const absl::optional<url::Origin>& top_frame_origin) {
   return blink::mojom::InterestCohort::New();
 }
+#endif
 
 bool ContentBrowserClient::IsBluetoothScanningBlocked(
     content::BrowserContext* browser_context,
@@ -1196,10 +1199,6 @@ bool ContentBrowserClient::ShouldAllowInsecurePrivateNetworkRequests(
   return false;
 }
 
-ukm::UkmService* ContentBrowserClient::GetUkmService() {
-  return nullptr;
-}
-
 void ContentBrowserClient::OnKeepaliveRequestStarted(BrowserContext*) {}
 
 void ContentBrowserClient::OnKeepaliveRequestFinished() {}
