diff --git a/content/browser/worker_host/shared_worker_connector_impl.cc b/content/browser/worker_host/shared_worker_connector_impl.cc
index 6081a315770c..e988fec3b93f 100644
--- a/content/browser/worker_host/shared_worker_connector_impl.cc
+++ b/content/browser/worker_host/shared_worker_connector_impl.cc
@@ -37,8 +37,7 @@ void SharedWorkerConnectorImpl::Connect(
     mojo::PendingRemote<blink::mojom::SharedWorkerClient> client,
     blink::mojom::SharedWorkerCreationContextType creation_context_type,
     blink::MessagePortDescriptor message_port,
-    mojo::PendingRemote<blink::mojom::BlobURLToken> blob_url_token,
-    ukm::SourceId client_ukm_source_id) {
+    mojo::PendingRemote<blink::mojom::BlobURLToken> blob_url_token) {
   RenderProcessHost* host =
       RenderProcessHost::FromID(client_render_frame_host_id_.child_id);
   // The render process was already terminated.
@@ -64,7 +63,7 @@ void SharedWorkerConnectorImpl::Connect(
   service->ConnectToWorker(
       client_render_frame_host_id_, std::move(info), std::move(client),
       creation_context_type, blink::MessagePortChannel(std::move(message_port)),
-      std::move(blob_url_loader_factory), client_ukm_source_id);
+      std::move(blob_url_loader_factory));
 }
 
 }  // namespace content
