diff --git a/content/browser/service_worker/service_worker_info.h b/content/browser/service_worker/service_worker_info.h
index 72809c0f362d..e4746bee2b54 100644
--- a/content/browser/service_worker/service_worker_info.h
+++ b/content/browser/service_worker/service_worker_info.h
@@ -14,7 +14,6 @@
 #include "base/time/time.h"
 #include "content/browser/service_worker/service_worker_version.h"
 #include "content/common/content_export.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "third_party/blink/public/mojom/service_worker/service_worker_container_type.mojom.h"
 #include "url/gurl.h"
 #include "url/origin.h"
@@ -37,8 +36,7 @@ struct CONTENT_EXPORT ServiceWorkerVersionInfo {
       int64_t version_id,
       int process_id,
       int thread_id,
-      int devtools_agent_route_id,
-      ukm::SourceId ukm_source_id);
+      int devtools_agent_route_id);
   ServiceWorkerVersionInfo(const ServiceWorkerVersionInfo& other);
   ~ServiceWorkerVersionInfo();
 
@@ -53,7 +51,6 @@ struct CONTENT_EXPORT ServiceWorkerVersionInfo {
   int process_id;
   int thread_id;
   int devtools_agent_route_id;
-  ukm::SourceId ukm_source_id = ukm::kInvalidSourceId;
   base::Time script_response_time;
   base::Time script_last_modified;
   std::map<std::string, ServiceWorkerClientInfo> clients;
