diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index b63cf589be716..b274fbdf29ffb 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -78,7 +78,6 @@ source_set("browser") {
     "//components/payments/mojom",
     "//components/power_scheduler",
     "//components/services/filesystem:lib",
-    "//components/services/quarantine:quarantine",
     "//components/services/storage",
     "//components/services/storage:filesystem_proxy_factory",
     "//components/services/storage/dom_storage:local_storage_proto",
@@ -87,8 +86,6 @@ source_set("browser") {
     "//components/system_media_controls",
     "//components/tracing:startup_tracing",
     "//components/ui_devtools",
-    "//components/ukm",
-    "//components/ukm/debug:util",
     "//components/url_formatter",
     "//components/variations",
     "//components/variations/net",
@@ -833,8 +830,6 @@ source_set("browser") {
     "eye_dropper_chooser_impl.h",
     "feature_observer.cc",
     "feature_observer.h",
-    "federated_learning/floc_service_impl.cc",
-    "federated_learning/floc_service_impl.h",
     "field_trial_recorder.cc",
     "field_trial_recorder.h",
     "field_trial_synchronizer.cc",
@@ -1654,8 +1649,6 @@ source_set("browser") {
     "service_worker/service_worker_fetch_dispatcher.h",
     "service_worker/service_worker_host.cc",
     "service_worker/service_worker_host.h",
-    "service_worker/service_worker_identifiability_metrics.cc",
-    "service_worker/service_worker_identifiability_metrics.h",
     "service_worker/service_worker_info.cc",
     "service_worker/service_worker_info.h",
     "service_worker/service_worker_installed_script_loader.cc",
@@ -1793,8 +1786,6 @@ source_set("browser") {
     "tracing/tracing_controller_impl_data_endpoint.cc",
     "tracing/tracing_service_controller.cc",
     "tracing/tracing_service_controller.h",
-    "ukm_internals_ui.cc",
-    "ukm_internals_ui.h",
     "url_loader_factory_getter.cc",
     "url_loader_factory_getter.h",
     "url_loader_factory_params_helper.cc",
@@ -2201,7 +2192,6 @@ source_set("browser") {
     public_deps += [ "//components/services/font/public/mojom" ]
 
     deps += [
-      "//components/crash/content/browser/error_reporting",
       "//components/services/font:lib",
       "//third_party/blink/public/mojom:memory_usage_monitor_linux_mojo_bindings",
     ]
@@ -2876,7 +2866,6 @@ source_set("browser") {
     ]
   }
 
-  if (enable_reporting) {
     sources += [
       "net/cross_origin_embedder_policy_reporter.cc",
       "net/cross_origin_embedder_policy_reporter.h",
@@ -2885,7 +2874,6 @@ source_set("browser") {
       "net/reporting_service_proxy.cc",
       "net/reporting_service_proxy.h",
     ]
-  }
 
   if (use_viz_devtools) {
     sources += [
