diff --git a/content/renderer/media/media_factory.cc b/content/renderer/media/media_factory.cc
index 819de0e9015b..ad6be8cd4e7d 100644
--- a/content/renderer/media/media_factory.cc
+++ b/content/renderer/media/media_factory.cc
@@ -45,6 +45,8 @@
 #include "media/renderers/default_decoder_factory.h"
 #include "media/renderers/default_renderer_factory.h"
 #include "media/video/gpu_video_accelerator_factories.h"
+#include "mojo/public/cpp/bindings/generic_pending_receiver.h"
+#include "mojo/public/cpp/bindings/pending_receiver.h"
 #include "mojo/public/cpp/bindings/pending_remote.h"
 #include "services/device/public/mojom/battery_monitor.mojom.h"
 #include "services/service_manager/public/cpp/connect.h"
@@ -427,10 +429,6 @@ blink::WebMediaPlayer* MediaFactory::CreateMediaPlayer(
   DCHECK_EQ(static_cast<FrameFetchContext*>(fetch_context_.get())->frame(),
             web_frame);
 
-  mojo::PendingRemote<media::mojom::MediaMetricsProvider> metrics_provider;
-  interface_broker_->GetInterface(
-      metrics_provider.InitWithNewPipeAndPassReceiver());
-
   std::unique_ptr<media::PowerStatusHelper> power_status_helper;
   if (base::FeatureList::IsEnabled(media::kMediaPowerExperiment)) {
     // The battery monitor is only available through the blink provider.
@@ -482,7 +480,6 @@ blink::WebMediaPlayer* MediaFactory::CreateMediaPlayer(
               base::Unretained(blink::MainThreadIsolate())),
           initial_cdm, request_routing_token_cb_, media_observer,
           enable_instant_source_buffer_gc, embedded_media_experience_enabled,
-          std::move(metrics_provider),
           base::BindOnce(&blink::WebSurfaceLayerBridge::Create,
                          parent_frame_sink_id,
                          blink::WebSurfaceLayerBridge::ContainsVideo::kYes),
