diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 266fd660ec0c..69f8135fcaa5 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -77,7 +77,6 @@ source_set("browser") {
     "//components/payments/core",
     "//components/payments/core:error_strings",
     "//components/payments/mojom",
-    "//components/rappor",
     "//components/services/filesystem:lib",
     "//components/services/quarantine:quarantine",
     "//components/services/storage",
