diff --git a/content/browser/payments/payment_app_provider_util.cc b/content/browser/payments/payment_app_provider_util.cc
index cb0b745aab43..4b268ca34161 100644
--- a/content/browser/payments/payment_app_provider_util.cc
+++ b/content/browser/payments/payment_app_provider_util.cc
@@ -9,13 +9,6 @@
 
 namespace content {
 
-// static
-ukm::SourceId PaymentAppProviderUtil::GetSourceIdForPaymentAppFromScope(
-    const GURL& sw_scope) {
-  return ukm::UkmRecorder::GetSourceIdForPaymentAppFromScope(
-      sw_scope.GetOrigin());
-}
-
 // static
 bool PaymentAppProviderUtil::IsValidInstallablePaymentApp(
     const GURL& manifest_url,
