diff --git a/content/test/BUILD.gn b/content/test/BUILD.gn
index b17b67fc1e64..f5f160ef676c 100644
--- a/content/test/BUILD.gn
+++ b/content/test/BUILD.gn
@@ -2343,10 +2343,6 @@ test("content_unittests") {
     deps += [ "//services/tracing:test_utils" ]
   }
 
-  if (is_linux || is_chromeos) {
-    deps += [ "//components/crash/content/browser/error_reporting" ]
-  }
-
   if (enable_plugins) {
     sources += [
       "../browser/renderer_host/pepper/browser_ppapi_host_test.cc",
