diff --git a/content/browser/renderer_host/code_cache_host_impl.cc b/content/browser/renderer_host/code_cache_host_impl.cc
index 6c5e19894294..da67da426fa8 100644
--- a/content/browser/renderer_host/code_cache_host_impl.cc
+++ b/content/browser/renderer_host/code_cache_host_impl.cc
@@ -207,7 +207,7 @@ void CodeCacheHostImpl::DidGenerateCacheableMetadataInCacheStorage(
           : render_process_host_impl_->GetStoragePartition()
                 ->GetCacheStorageControl();
   cache_storage_control->AddReceiver(
-      cross_origin_embedder_policy, mojo::NullRemote(), cache_storage_origin,
+      cross_origin_embedder_policy, cache_storage_origin,
       storage::mojom::CacheStorageOwner::kCacheAPI,
       remote.BindNewPipeAndPassReceiver());
 
