diff --git a/content/browser/renderer_host/pepper/pepper_file_io_host.cc b/content/browser/renderer_host/pepper/pepper_file_io_host.cc
index 90caac458fc5..5bc98e0e8d49 100644
--- a/content/browser/renderer_host/pepper/pepper_file_io_host.cc
+++ b/content/browser/renderer_host/pepper/pepper_file_io_host.cc
@@ -437,41 +437,7 @@ void PepperFileIOHost::OnLocalFileOpened(
     ppapi::host::ReplyMessageContext reply_context,
     const base::FilePath& path,
     base::File::Error error_code) {
-#if defined(OS_WIN) || defined(OS_LINUX) || defined(OS_CHROMEOS)
-  // Quarantining a file before its contents are available is only supported on
-  // Windows and Linux.
-  if (!FileOpenForWrite(open_flags_) || error_code != base::File::FILE_OK) {
-    SendFileOpenReply(reply_context, error_code);
-    return;
-  }
-
-  base::PostTaskAndReplyWithResult(
-      task_runner_.get(), FROM_HERE,
-      base::BindOnce(
-          &download::QuarantineFile, path,
-          browser_ppapi_host_->GetDocumentURLForInstance(pp_instance()), GURL(),
-          std::string()),
-      base::BindOnce(&PepperFileIOHost::OnLocalFileQuarantined, AsWeakPtr(),
-                     reply_context, path));
-#else
-  SendFileOpenReply(reply_context, error_code);
-#endif
-}
-
-#if defined(OS_WIN) || defined(OS_LINUX) || defined(OS_CHROMEOS)
-void PepperFileIOHost::OnLocalFileQuarantined(
-    ppapi::host::ReplyMessageContext reply_context,
-    const base::FilePath& path,
-    download::QuarantineFileResult quarantine_result) {
-  base::File::Error file_error =
-      (quarantine_result == download::QuarantineFileResult::OK
-           ? base::File::FILE_OK
-           : base::File::FILE_ERROR_SECURITY);
-  if (file_error != base::File::FILE_OK && file_.IsValid())
-    file_.Close(base::FileProxy::StatusCallback());
-  SendFileOpenReply(reply_context, file_error);
 }
-#endif
 
 void PepperFileIOHost::SendFileOpenReply(
     ppapi::host::ReplyMessageContext reply_context,
