diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index ed9d15a2832c8..68ea7af55cf58 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -359,9 +359,7 @@ class UkmRecorderFactoryImpl : public cc::UkmRecorderFactory {
   ~UkmRecorderFactoryImpl() override = default;
 
   std::unique_ptr<ukm::UkmRecorder> CreateRecorder() override {
-    mojo::PendingRemote<ukm::mojom::UkmRecorderInterface> recorder;
-    process_host_->BindHostReceiver(recorder.InitWithNewPipeAndPassReceiver());
-    return std::make_unique<ukm::MojoUkmRecorder>(std::move(recorder));
+    return std::make_unique<ukm::MojoUkmRecorder>();
   }
 
  private:
