diff --git a/content/renderer/accessibility/ax_image_annotator.cc b/content/renderer/accessibility/ax_image_annotator.cc
index 4290e3d012cc..aadf2557b218 100644
--- a/content/renderer/accessibility/ax_image_annotator.cc
+++ b/content/renderer/accessibility/ax_image_annotator.cc
@@ -18,8 +18,6 @@
 #include "content/renderer/accessibility/ax_image_stopwords.h"
 #include "content/renderer/render_frame_impl.h"
 #include "crypto/sha2.h"
-#include "services/metrics/public/cpp/mojo_ukm_recorder.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
 #include "third_party/blink/public/strings/grit/blink_strings.h"
 #include "third_party/blink/public/web/web_ax_object.h"
 #include "third_party/blink/public/web/web_document.h"
@@ -374,14 +372,6 @@ void AXImageAnnotator::OnImageAnnotated(
   ax::mojom::NameFrom name_from;
   blink::WebVector<blink::WebAXObject> name_objects;
   blink::WebString name = image.GetName(name_from, name_objects);
-  bool has_existing_label = !name.IsEmpty();
-
-  ukm::builders::Accessibility_ImageDescriptions(
-      render_accessibility_->GetMainDocument().GetUkmSourceId())
-      .SetOCR(has_ocr)
-      .SetDescription(has_description)
-      .SetImageAlreadyHasLabel(has_existing_label)
-      .Record(render_accessibility_->ukm_recorder());
 
   image_annotations_.at(image.AxID())
       .set_status(ax::mojom::ImageAnnotationStatus::kAnnotationSucceeded);
