diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index bd83dfa0a326..0c400efcc399 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -482,7 +482,6 @@ source_set("browser_sources") {
     "//net",
     "//ppapi/c",
     "//services/device/public/cpp/geolocation",
-    "//services/metrics/public/cpp:metrics_cpp",
     "//ui/accessibility",
     "//ui/base",
     "//ui/base/clipboard",
