diff --git a/content/browser/content_index/content_index_database.cc b/content/browser/content_index/content_index_database.cc
index 2ce59c40e2d8..f4059ba142a9 100644
--- a/content/browser/content_index/content_index_database.cc
+++ b/content/browser/content_index/content_index_database.cc
@@ -14,7 +14,6 @@
 #include "base/time/time.h"
 #include "content/browser/background_fetch/storage/image_helpers.h"
 #include "content/browser/content_index/content_index.pb.h"
-#include "content/browser/content_index/content_index_metrics.h"
 #include "content/public/browser/browser_context.h"
 #include "content/public/browser/browser_task_traits.h"
 #include "content/public/browser/browser_thread.h"
@@ -170,7 +169,6 @@ void ContentIndexDatabase::AddEntryOnCoreThread(
   if (blocked_origins_.count(origin)) {
     // TODO(crbug.com/973844): Does this need a more specific error?
     std::move(callback).Run(blink::mojom::ContentIndexError::STORAGE_ERROR);
-    content_index::RecordRegistrationBlocked(description->category);
     return;
   }
 
@@ -237,8 +235,6 @@ void ContentIndexDatabase::DidAddEntry(
     blink::mojom::ContentIndexService::AddCallback callback,
     ContentIndexEntry entry,
     blink::ServiceWorkerStatusCode status) {
-  content_index::RecordDatabaseOperationStatus("Add", status);
-
   if (status != blink::ServiceWorkerStatusCode::kOk) {
     std::move(callback).Run(blink::mojom::ContentIndexError::STORAGE_ERROR);
     return;
@@ -298,8 +294,6 @@ void ContentIndexDatabase::DidDeleteEntry(
     const std::string& entry_id,
     blink::mojom::ContentIndexService::DeleteCallback callback,
     blink::ServiceWorkerStatusCode status) {
-  content_index::RecordDatabaseOperationStatus("Delete", status);
-
   if (status != blink::ServiceWorkerStatusCode::kOk) {
     std::move(callback).Run(blink::mojom::ContentIndexError::STORAGE_ERROR);
     return;
@@ -354,8 +348,6 @@ void ContentIndexDatabase::DidGetDescriptions(
     blink::mojom::ContentIndexService::GetDescriptionsCallback callback,
     const std::vector<std::string>& data,
     blink::ServiceWorkerStatusCode status) {
-  content_index::RecordDatabaseOperationStatus("GetDescriptions", status);
-
   if (status == blink::ServiceWorkerStatusCode::kErrorNotFound) {
     std::move(callback).Run(blink::mojom::ContentIndexError::NONE,
                             /* descriptions= */ {});
@@ -385,8 +377,7 @@ void ContentIndexDatabase::DidGetDescriptions(
           service_worker_registration_id,
           {EntryKey(entry.description().id()),
            IconsKey(entry.description().id())},
-          base::BindOnce(&content_index::RecordDatabaseOperationStatus,
-                         "ClearCorruptedData"));
+          base::NullCallback());
       continue;
     }
 
@@ -439,8 +430,6 @@ void ContentIndexDatabase::DidGetSerializedIcons(
     blink::ServiceWorkerStatusCode status) {
   DCHECK_CURRENTLY_ON(ServiceWorkerContext::GetCoreThreadId());
 
-  content_index::RecordDatabaseOperationStatus("GetIcon", status);
-
   if (status != blink::ServiceWorkerStatusCode::kOk || data.empty()) {
     std::move(callback).Run({});
     return;
@@ -525,7 +514,6 @@ void ContentIndexDatabase::DidGetEntries(
     ContentIndexContext::GetAllEntriesCallback callback,
     const std::vector<std::pair<int64_t, std::string>>& user_data,
     blink::ServiceWorkerStatusCode status) {
-  content_index::RecordDatabaseOperationStatus("GetAllEntries", status);
 
   if (status != blink::ServiceWorkerStatusCode::kOk) {
     std::move(callback).Run(blink::mojom::ContentIndexError::STORAGE_ERROR,
@@ -607,7 +595,6 @@ void ContentIndexDatabase::DidGetEntry(
     ContentIndexContext::GetEntryCallback callback,
     const std::vector<std::string>& data,
     blink::ServiceWorkerStatusCode status) {
-  content_index::RecordDatabaseOperationStatus("GetEntry", status);
 
   if (status != blink::ServiceWorkerStatusCode::kOk) {
     std::move(callback).Run(base::nullopt);
@@ -625,8 +612,7 @@ void ContentIndexDatabase::ClearServiceWorkerDataOnCorruption(
 
   service_worker_context_->ClearRegistrationUserDataByKeyPrefixes(
       service_worker_registration_id, {kEntryPrefix, kIconPrefix},
-      base::BindOnce(&content_index::RecordDatabaseOperationStatus,
-                     "ClearCorruptedData"));
+      base::NullCallback());
 }
 
 void ContentIndexDatabase::DeleteItem(int64_t service_worker_registration_id,
@@ -667,8 +653,6 @@ void ContentIndexDatabase::StartActiveWorkerForDispatch(
     scoped_refptr<ServiceWorkerRegistration> registration) {
   DCHECK_CURRENTLY_ON(ServiceWorkerContext::GetCoreThreadId());
 
-  content_index::RecordDisptachStatus("Find", service_worker_status);
-
   if (service_worker_status != blink::ServiceWorkerStatusCode::kOk)
     return;
 
@@ -690,8 +674,6 @@ void ContentIndexDatabase::DeliverMessageToWorker(
     blink::ServiceWorkerStatusCode service_worker_status) {
   DCHECK_CURRENTLY_ON(ServiceWorkerContext::GetCoreThreadId());
 
-  content_index::RecordDisptachStatus("Start", service_worker_status);
-
   if (service_worker_status != blink::ServiceWorkerStatusCode::kOk)
     return;
 
@@ -714,7 +696,6 @@ void ContentIndexDatabase::DidDispatchEvent(
     blink::ServiceWorkerStatusCode service_worker_status) {
   DCHECK_CURRENTLY_ON(ServiceWorkerContext::GetCoreThreadId());
 
-  content_index::RecordDisptachStatus("Dispatch", service_worker_status);
   UnblockOrigin(origin);
 }
 
