diff --git a/content/public/test/mock_render_process_host.h b/content/public/test/mock_render_process_host.h
index 4630d2a8d6dc..1b7b3ba09d92 100644
--- a/content/public/test/mock_render_process_host.h
+++ b/content/public/test/mock_render_process_host.h
@@ -167,7 +167,6 @@ class MockRenderProcessHost : public RenderProcessHost {
   bool IsProcessLockedToSiteForTesting() override;
   void BindCacheStorage(
       const network::CrossOriginEmbedderPolicy&,
-      mojo::PendingRemote<network::mojom::CrossOriginEmbedderPolicyReporter>,
       const url::Origin& origin,
       mojo::PendingReceiver<blink::mojom::CacheStorage> receiver) override;
   void BindFileSystemManager(
