diff --git a/content/browser/worker_host/dedicated_worker_host.h b/content/browser/worker_host/dedicated_worker_host.h
index 7493ce9b5385..29901ae5a46c 100644
--- a/content/browser/worker_host/dedicated_worker_host.h
+++ b/content/browser/worker_host/dedicated_worker_host.h
@@ -60,8 +60,6 @@ class DedicatedWorkerHost final : public blink::mojom::DedicatedWorkerHost,
       const url::Origin& creator_origin,
       const net::IsolationInfo& isolation_info,
       const network::CrossOriginEmbedderPolicy& cross_origin_embedder_policy,
-      mojo::PendingRemote<network::mojom::CrossOriginEmbedderPolicyReporter>
-          coep_reporter,
       mojo::PendingReceiver<blink::mojom::DedicatedWorkerHost> host);
   ~DedicatedWorkerHost() final;
 
@@ -252,14 +250,6 @@ class DedicatedWorkerHost final : public blink::mojom::DedicatedWorkerHost,
   mojo::Remote<blink::mojom::SubresourceLoaderUpdater>
       subresource_loader_updater_;
 
-  // The endpoint of this mojo interface is the RenderFrameHostImpl's COEP
-  // reporter. The COEP endpoint is correct, but the context_url is the
-  // Document's URL.
-  // TODO(arthursonzogni): After landing PlzDedicatedWorker, make the
-  // DedicatedWorkerHost to have its own COEP reporter using the right
-  // context_url.
-  mojo::Remote<network::mojom::CrossOriginEmbedderPolicyReporter>
-      coep_reporter_;  // Never null.
   // Will be set once the worker script started loading.
   base::Optional<GURL> final_response_url_;
 
