diff --git a/content/public/test/navigation_handle_observer.h b/content/public/test/navigation_handle_observer.h
index b5f6fa2de0e2..49819fde6712 100644
--- a/content/public/test/navigation_handle_observer.h
+++ b/content/public/test/navigation_handle_observer.h
@@ -10,7 +10,6 @@
 #include "content/public/browser/reload_type.h"
 #include "content/public/browser/render_frame_host.h"
 #include "content/public/browser/web_contents_observer.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 
 namespace content {
 
@@ -38,7 +37,6 @@ class NavigationHandleObserver : public WebContentsObserver {
   net::Error net_error_code() { return net_error_code_; }
   int64_t navigation_id() { return navigation_id_; }
   bool is_download() { return is_download_; }
-  ukm::SourceId next_page_ukm_source_id() { return next_page_ukm_source_id_; }
   base::Optional<net::AuthChallengeInfo> auth_challenge_info() {
     return auth_challenge_info_;
   }
@@ -70,7 +68,6 @@ class NavigationHandleObserver : public WebContentsObserver {
   net::Error net_error_code_ = net::OK;
   int64_t navigation_id_ = -1;
   bool is_download_ = false;
-  ukm::SourceId next_page_ukm_source_id_ = ukm::kInvalidSourceId;
   base::Optional<net::AuthChallengeInfo> auth_challenge_info_;
   net::ResolveErrorInfo resolve_error_info_;
   base::TimeTicks navigation_start_;
