diff --git a/content/browser/renderer_host/navigation_entry_impl.cc b/content/browser/renderer_host/navigation_entry_impl.cc
index 80b3f2590d12..fd51b48b0e59 100644
--- a/content/browser/renderer_host/navigation_entry_impl.cc
+++ b/content/browser/renderer_host/navigation_entry_impl.cc
@@ -785,7 +785,7 @@ NavigationEntryImpl::ConstructCommitNavigationParams(
           false, network::mojom::IPAddressSpace::kUnknown,
           GURL() /* web_bundle_physical_url */,
           GURL() /* base_url_override_for_web_bundle */,
-          ukm::kInvalidSourceId /* document_ukm_source_id */, frame_policy,
+          frame_policy,
           std::vector<std::string>() /* force_enabled_origin_trials */,
           false /* origin_isolated */,
           std::vector<
