diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 266fd660ec0c..6d13e3224411 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -64,7 +64,6 @@ source_set("browser") {
     "//components/discardable_memory/service",
     "//components/download/database",
     "//components/download/public/common:public",
-    "//components/download/quarantine",
     "//components/filename_generation",
     "//components/link_header_util",
     "//components/metrics",
@@ -77,7 +76,6 @@ source_set("browser") {
     "//components/payments/core",
     "//components/payments/core:error_strings",
     "//components/payments/mojom",
-    "//components/rappor",
     "//components/services/filesystem:lib",
     "//components/services/quarantine:quarantine",
     "//components/services/storage",
