diff --git a/content/browser/service_worker/service_worker_container_host.cc b/content/browser/service_worker/service_worker_container_host.cc
index 860326af92fd..e09f712cb2a2 100644
--- a/content/browser/service_worker/service_worker_container_host.cc
+++ b/content/browser/service_worker/service_worker_container_host.cc
@@ -746,8 +746,7 @@ void ServiceWorkerContainerHost::OnBeginNavigationCommit(
     int container_frame_id,
     const network::CrossOriginEmbedderPolicy& cross_origin_embedder_policy,
     mojo::PendingRemote<network::mojom::CrossOriginEmbedderPolicyReporter>
-        coep_reporter,
-    ukm::SourceId document_ukm_source_id) {
+        coep_reporter) {
   DCHECK_CURRENTLY_ON(ServiceWorkerContext::GetCoreThreadId());
   DCHECK(IsContainerForWindowClient());
 
@@ -783,9 +782,6 @@ void ServiceWorkerContainerHost::OnBeginNavigationCommit(
   if (rfh)
     rfh->AddServiceWorkerContainerHost(client_uuid(), GetWeakPtr());
 
-  DCHECK_EQ(ukm_source_id_, ukm::kInvalidSourceId);
-  ukm_source_id_ = document_ukm_source_id;
-
   TransitionToClientPhase(ClientPhase::kResponseCommitted);
 }
 
@@ -803,8 +799,7 @@ void ServiceWorkerContainerHost::OnEndNavigationCommit() {
 }
 
 void ServiceWorkerContainerHost::CompleteWebWorkerPreparation(
-    const network::CrossOriginEmbedderPolicy& cross_origin_embedder_policy,
-    ukm::SourceId worker_ukm_source_id) {
+    const network::CrossOriginEmbedderPolicy& cross_origin_embedder_policy) {
   DCHECK_CURRENTLY_ON(ServiceWorkerContext::GetCoreThreadId());
   DCHECK(IsContainerForWorkerClient());
 
@@ -819,9 +814,6 @@ void ServiceWorkerContainerHost::CompleteWebWorkerPreparation(
                                      mojo::NullRemote());
   }
 
-  DCHECK_EQ(ukm_source_id_, ukm::kInvalidSourceId);
-  ukm_source_id_ = worker_ukm_source_id;
-
   TransitionToClientPhase(ClientPhase::kResponseCommitted);
   SetExecutionReady();
 }
