diff --git a/content/browser/payments/payment_app_provider_impl.h b/content/browser/payments/payment_app_provider_impl.h
index 8cc0491e317b..01a46c95dce4 100644
--- a/content/browser/payments/payment_app_provider_impl.h
+++ b/content/browser/payments/payment_app_provider_impl.h
@@ -68,8 +68,6 @@ class CONTENT_EXPORT PaymentAppProviderImpl : public PaymentAppProvider {
                                     const GURL& sw_js_url,
                                     const GURL& sw_scope,
                                     std::string* error_message) override;
-  ukm::SourceId GetSourceIdForPaymentAppFromScope(
-      const GURL& sw_scope) override;
 
  private:
   PaymentAppProviderImpl();
