diff --git a/content/browser/contacts/contacts_manager_impl.cc b/content/browser/contacts/contacts_manager_impl.cc
index 62899148c416..445e14cd934d 100644
--- a/content/browser/contacts/contacts_manager_impl.cc
+++ b/content/browser/contacts/contacts_manager_impl.cc
@@ -14,7 +14,6 @@
 #include "content/public/browser/contacts_picker_properties_requested.h"
 #include "mojo/public/cpp/bindings/self_owned_receiver.h"
 #include "services/metrics/public/cpp/metrics_utils.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
 
 #if defined(OS_ANDROID)
 #include "content/browser/contacts/contacts_provider_android.h"
@@ -37,18 +36,9 @@ std::unique_ptr<ContactsProvider> CreateProvider(
 
 void OnContactsSelected(
     blink::mojom::ContactsManager::SelectCallback callback,
-    ukm::SourceId source_id,
     base::Optional<std::vector<blink::mojom::ContactInfoPtr>> contacts,
     int percentage_shared,
     ContactsPickerPropertiesRequested properties_requested) {
-  if (contacts != base::nullopt) {
-    int select_count = contacts.value().size();
-    ukm::builders::ContactsPicker_ShareStatistics(source_id)
-        .SetSelectCount(ukm::GetExponentialBucketMinForCounts1000(select_count))
-        .SetSelectPercentage(percentage_shared)
-        .SetPropertiesRequested(properties_requested)
-        .Record(ukm::UkmRecorder::Get());
-  }
   std::move(callback).Run(std::move(contacts));
 }
 
@@ -64,8 +54,7 @@ void ContactsManagerImpl::Create(
 }
 
 ContactsManagerImpl::ContactsManagerImpl(RenderFrameHostImpl* render_frame_host)
-    : contacts_provider_(CreateProvider(render_frame_host)),
-      source_id_(render_frame_host->GetPageUkmSourceId()) {}
+    : contacts_provider_(CreateProvider(render_frame_host)) {}
 
 ContactsManagerImpl::~ContactsManagerImpl() = default;
 
@@ -80,8 +69,7 @@ void ContactsManagerImpl::Select(bool multiple,
     contacts_provider_->Select(
         multiple, include_names, include_emails, include_tel, include_addresses,
         include_icons,
-        base::BindOnce(&OnContactsSelected, std::move(mojom_callback),
-                       source_id_));
+        base::BindOnce(&OnContactsSelected, std::move(mojom_callback)));
   } else {
     std::move(mojom_callback).Run(base::nullopt);
   }
