diff --git a/content/browser/notifications/notification_database.cc b/content/browser/notifications/notification_database.cc
index a43a64dcfb47..782b70548f2d 100644
--- a/content/browser/notifications/notification_database.cc
+++ b/content/browser/notifications/notification_database.cc
@@ -145,9 +145,8 @@ void UpdateNotificationTimestamps(NotificationDatabaseData* data) {
 
 }  // namespace
 
-NotificationDatabase::NotificationDatabase(const base::FilePath& path,
-                                           UkmCallback callback)
-    : path_(path), record_notification_to_ukm_callback_(std::move(callback)) {}
+NotificationDatabase::NotificationDatabase(const base::FilePath& path)
+    : path_(path) {}
 
 NotificationDatabase::~NotificationDatabase() {
   DCHECK(sequence_checker_.CalledOnValidSequence());
@@ -348,13 +347,6 @@ NotificationDatabase::Status NotificationDatabase::DeleteNotificationData(
   DCHECK(!notification_id.empty());
   DCHECK(origin.is_valid());
 
-  NotificationDatabaseData data;
-  Status status = ReadNotificationData(notification_id, origin, &data);
-  if (status == STATUS_OK && record_notification_to_ukm_callback_) {
-    GetUIThreadTaskRunner({})->PostTask(
-        FROM_HERE, base::BindOnce(record_notification_to_ukm_callback_, data));
-  }
-
   leveldb::WriteBatch batch;
   batch.Delete(CreateDataKey(origin, notification_id));
   batch.Delete(CreateResourcesKey(origin, notification_id));
@@ -506,12 +498,6 @@ NotificationDatabase::DeleteAllNotificationDataInternal(
       continue;
     }
 
-    if (record_notification_to_ukm_callback_) {
-      GetUIThreadTaskRunner({})->PostTask(
-          FROM_HERE, base::BindOnce(record_notification_to_ukm_callback_,
-                                    notification_database_data));
-    }
-
     std::string notification_id = notification_database_data.notification_id;
     DCHECK(!notification_id.empty());
 
