diff --git a/content/browser/url_loader_factory_params_helper.h b/content/browser/url_loader_factory_params_helper.h
index b7b8bc2a9bc5..59cfa1f05061 100644
--- a/content/browser/url_loader_factory_params_helper.h
+++ b/content/browser/url_loader_factory_params_helper.h
@@ -8,7 +8,6 @@
 #include "base/strings/string_piece.h"
 #include "content/common/content_export.h"
 #include "mojo/public/cpp/bindings/pending_remote.h"
-#include "services/network/public/mojom/cross_origin_embedder_policy.mojom-forward.h"
 #include "services/network/public/mojom/network_context.mojom.h"
 #include "services/network/public/mojom/url_loader.mojom-shared.h"
 #include "url/origin.h"
@@ -49,8 +48,6 @@ class URLLoaderFactoryParamsHelper {
       const url::Origin& origin,
       const net::IsolationInfo& isolation_info,
       network::mojom::ClientSecurityStatePtr client_security_state,
-      mojo::PendingRemote<network::mojom::CrossOriginEmbedderPolicyReporter>
-          coep_reporter,
       RenderProcessHost* process,
       network::mojom::TrustTokenRedemptionPolicy trust_token_redemption_policy,
       base::StringPiece debug_tag);
@@ -75,8 +72,6 @@ class URLLoaderFactoryParamsHelper {
       RenderProcessHost* process,
       const url::Origin& request_initiator,
       const net::IsolationInfo& isolation_info,
-      mojo::PendingRemote<network::mojom::CrossOriginEmbedderPolicyReporter>
-          coep_reporter,
       mojo::PendingRemote<network::mojom::AuthenticationAndCertificateObserver>
           auth_cert_observer,
       base::StringPiece debug_tag);
