diff --git a/content/browser/service_worker/service_worker_version.cc b/content/browser/service_worker/service_worker_version.cc
index 51049c143b6d..51823d1b091d 100644
--- a/content/browser/service_worker/service_worker_version.cc
+++ b/content/browser/service_worker/service_worker_version.cc
@@ -265,9 +265,7 @@ ServiceWorkerVersion::ServiceWorkerVersion(
       clock_(base::DefaultClock::GetInstance()),
       ping_controller_(this),
       validator_(std::make_unique<blink::TrialTokenValidator>()),
-      remote_reference_(std::move(remote_reference)),
-      ukm_source_id_(ukm::ConvertToSourceId(ukm::AssignNewSourceId(),
-                                            ukm::SourceIdType::WORKER_ID)) {
+      remote_reference_(std::move(remote_reference)) {
   DCHECK_NE(blink::mojom::kInvalidServiceWorkerVersionId, version_id);
   DCHECK(context_);
   DCHECK(registration);
@@ -1884,8 +1882,6 @@ void ServiceWorkerVersion::StartWorkerInternal() {
 
   params->provider_info = std::move(provider_info);
 
-  params->ukm_source_id = ukm_source_id_;
-
   embedded_worker_->Start(std::move(params),
                           base::BindOnce(&ServiceWorkerVersion::OnStartSent,
                                          weak_factory_.GetWeakPtr()));
