diff --git a/content/browser/renderer_host/compositor_impl_android.cc b/content/browser/renderer_host/compositor_impl_android.cc
index 44d35de2b2db..81f8383bdd44 100644
--- a/content/browser/renderer_host/compositor_impl_android.cc
+++ b/content/browser/renderer_host/compositor_impl_android.cc
@@ -75,7 +75,6 @@
 #include "ui/display/screen.h"
 #include "ui/gfx/ca_layer_params.h"
 #include "ui/gfx/swap_result.h"
-#include "ui/latency/latency_tracker.h"
 
 namespace content {
 
@@ -704,11 +703,6 @@ void CompositorImpl::DidCommit(base::TimeTicks) {
   root_window_->OnCompositingDidCommit();
 }
 
-std::unique_ptr<cc::BeginMainFrameMetrics>
-CompositorImpl::GetBeginMainFrameMetrics() {
-  return nullptr;
-}
-
 std::unique_ptr<ui::WindowAndroidCompositor::ReadbackRef>
 CompositorImpl::TakeReadbackRef() {
   ++pending_readbacks_;
