diff --git a/content/browser/download/download_manager_impl.cc b/content/browser/download/download_manager_impl.cc
index 26d995b3ea90b..f6b9da6c4bde6 100644
--- a/content/browser/download/download_manager_impl.cc
+++ b/content/browser/download/download_manager_impl.cc
@@ -674,6 +674,7 @@ void DownloadManagerImpl::BindWakeLockProvider(
   GetDeviceService().BindWakeLockProvider(std::move(receiver));
 }
 
+#if 0
 download::QuarantineConnectionCallback
 DownloadManagerImpl::GetQuarantineConnectionCallback() {
   if (!delegate_)
@@ -681,6 +682,7 @@ DownloadManagerImpl::GetQuarantineConnectionCallback() {
 
   return delegate_->GetQuarantineConnectionCallback();
 }
+#endif
 
 std::unique_ptr<download::DownloadItemRenameHandler>
 DownloadManagerImpl::GetRenameHandlerForDownload(
@@ -1455,4 +1457,10 @@ DownloadManagerImpl::RetrieveInProgressDownload(uint32_t id) {
   return nullptr;
 }
 
+bool DownloadManagerImpl::ShouldRemoveAfterCompletion() {
+  if (delegate_)
+    return delegate_->ShouldRemoveAfterCompletion();
+  return false;
+}
+
 }  // namespace content
