diff --git a/content/browser/renderer_host/delegated_frame_host.cc b/content/browser/renderer_host/delegated_frame_host.cc
index c3224c882319..a460a26e4acc 100644
--- a/content/browser/renderer_host/delegated_frame_host.cc
+++ b/content/browser/renderer_host/delegated_frame_host.cc
@@ -87,12 +87,6 @@ void DelegatedFrameHost::WasShown(
   SetFrameEvictionStateAndNotifyObservers(FrameEvictionState::kNotStarted);
 
   frame_evictor_->SetVisible(true);
-  if (record_tab_switch_time_request && compositor_) {
-    compositor_->RequestPresentationTimeForNextFrame(
-        tab_switch_time_recorder_.TabWasShown(
-            true /* has_saved_frames */,
-            std::move(record_tab_switch_time_request), base::TimeTicks::Now()));
-  }
 
   // Use the default deadline to synchronize web content with browser UI.
   // TODO(fsamuel): Investigate if there is a better deadline to use here.
@@ -111,7 +105,6 @@ bool DelegatedFrameHost::HasSavedFrame() const {
 }
 
 void DelegatedFrameHost::WasHidden(HiddenCause cause) {
-  tab_switch_time_recorder_.TabWasHidden();
 #if defined(OS_WIN)
   // Ignore if the native window was occluded.
   // Windows needs the frame host to display tab previews.
