diff --git a/content/public/test/mock_render_process_host.cc b/content/public/test/mock_render_process_host.cc
index f3081778b0fb..6df1be8b8039 100644
--- a/content/public/test/mock_render_process_host.cc
+++ b/content/public/test/mock_render_process_host.cc
@@ -451,7 +451,6 @@ bool MockRenderProcessHost::IsProcessLockedToSiteForTesting() {
 
 void MockRenderProcessHost::BindCacheStorage(
     const network::CrossOriginEmbedderPolicy&,
-    mojo::PendingRemote<network::mojom::CrossOriginEmbedderPolicyReporter>,
     const url::Origin& origin,
     mojo::PendingReceiver<blink::mojom::CacheStorage> receiver) {
   cache_storage_receiver_ = std::move(receiver);
