diff --git a/content/browser/devtools/service_worker_devtools_agent_host.h b/content/browser/devtools/service_worker_devtools_agent_host.h
index 2e2e7900a16b..176b0f7b320b 100644
--- a/content/browser/devtools/service_worker_devtools_agent_host.h
+++ b/content/browser/devtools/service_worker_devtools_agent_host.h
@@ -42,8 +42,6 @@ class ServiceWorkerDevToolsAgentHost : public DevToolsAgentHostImpl,
       bool is_installed_version,
       base::Optional<network::CrossOriginEmbedderPolicy>
           cross_origin_embedder_policy,
-      mojo::PendingRemote<network::mojom::CrossOriginEmbedderPolicyReporter>
-          coep_reporter,
       const base::UnguessableToken& devtools_worker_token);
 
   // DevToolsAgentHost overrides.
@@ -65,9 +63,7 @@ class ServiceWorkerDevToolsAgentHost : public DevToolsAgentHostImpl,
       mojo::PendingRemote<blink::mojom::DevToolsAgent> agent_remote,
       mojo::PendingReceiver<blink::mojom::DevToolsAgentHost> host_receiver);
   void UpdateCrossOriginEmbedderPolicy(
-      network::CrossOriginEmbedderPolicy cross_origin_embedder_policy,
-      mojo::PendingRemote<network::mojom::CrossOriginEmbedderPolicyReporter>
-          coep_reporter);
+      network::CrossOriginEmbedderPolicy cross_origin_embedder_policy);
   void WorkerStopped();
   void WorkerVersionInstalled();
   void WorkerVersionDoomed();
@@ -121,8 +117,6 @@ class ServiceWorkerDevToolsAgentHost : public DevToolsAgentHostImpl,
   base::Time version_doomed_time_;
   base::Optional<network::CrossOriginEmbedderPolicy>
       cross_origin_embedder_policy_;
-  mojo::Remote<network::mojom::CrossOriginEmbedderPolicyReporter>
-      coep_reporter_;
   base::ScopedObservation<RenderProcessHost, RenderProcessHostObserver>
       process_observation_{this};
 
