diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index ebc8e7006453..cea0f1803d21 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -354,9 +354,7 @@ class UkmRecorderFactoryImpl : public cc::UkmRecorderFactory {
   ~UkmRecorderFactoryImpl() override = default;
 
   std::unique_ptr<ukm::UkmRecorder> CreateRecorder() override {
-    mojo::PendingRemote<ukm::mojom::UkmRecorderInterface> recorder;
-    process_host_->BindHostReceiver(recorder.InitWithNewPipeAndPassReceiver());
-    return std::make_unique<ukm::MojoUkmRecorder>(std::move(recorder));
+    return std::make_unique<ukm::MojoUkmRecorder>();
   }
 
  private:
