diff --git a/content/child/runtime_features.cc b/content/child/runtime_features.cc
index 39cef2472289b..1e9d60dd7b34b 100644
--- a/content/child/runtime_features.cc
+++ b/content/child/runtime_features.cc
@@ -366,8 +366,6 @@ void SetRuntimeFeaturesFromChromiumFeatures() {
            blink::features::kForceSynchronousHTMLParsing},
           {"IgnoreCrossOriginWindowWhenNamedAccessOnWindow",
            blink::features::kIgnoreCrossOriginWindowWhenNamedAccessOnWindow},
-          {"InterestCohortFeaturePolicy",
-           blink::features::kInterestCohortFeaturePolicy},
           {"LangClientHintHeader", features::kLangClientHintHeader},
           {"LayoutNG", blink::features::kLayoutNG},
           {"LayoutNGTable", blink::features::kLayoutNGTable},
