diff --git a/content/browser/renderer_host/DEPS b/content/browser/renderer_host/DEPS
index ad7f91b632b6..9012b52a9594 100644
--- a/content/browser/renderer_host/DEPS
+++ b/content/browser/renderer_host/DEPS
@@ -19,7 +19,6 @@ include_rules = [
 
 specific_include_rules = {
   ".*_(unit|browser)test\.(cc|mm)": [
-    "+components/rappor/test_rappor_service.h",
     "+content/browser/web_contents",
     "+content/public/browser/web_contents.h",
     "+content/public/browser/web_contents_delegate.h",
