diff --git a/content/public/browser/web_contents.h b/content/public/browser/web_contents.h
index 0426cc1550f1..2c3449f06c70 100644
--- a/content/public/browser/web_contents.h
+++ b/content/public/browser/web_contents.h
@@ -35,7 +35,6 @@
 #include "content/public/common/stop_find_action.h"
 #include "mojo/public/cpp/bindings/remote.h"
 #include "services/data_decoder/public/mojom/web_bundler.mojom.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "third_party/blink/public/mojom/frame/find_in_page.mojom-forward.h"
 #include "third_party/blink/public/mojom/input/pointer_lock_result.mojom.h"
 #include "third_party/blink/public/mojom/loader/pause_subresource_loading_handle.mojom-forward.h"
@@ -440,16 +439,6 @@ class WebContents : public PageNavigator,
   using AccessibilityEventCallback =
       base::RepeatingCallback<void(const std::string&)>;
 
-  // Starts or stops recording accessibility events. |start_recording| is true
-  // when recording should start and false when recording should stop.
-  // |callback| is an optional function which is called when an accessibility
-  // event is received while accessibility events are being recorded. When
-  // |start_recording| is true, it is expected that |callback| has a value; when
-  // |start_recording| is false, it is expected that |callback| does not.
-  virtual void RecordAccessibilityEvents(
-      bool start_recording,
-      base::Optional<AccessibilityEventCallback> callback) = 0;
-
   // Tab navigation state ------------------------------------------------------
 
   // Returns the current navigation properties, which if a navigation is
