diff --git a/content/browser/renderer_host/navigator.h b/content/browser/renderer_host/navigator.h
index 4db476c8078f..c9d6a4696c05 100644
--- a/content/browser/renderer_host/navigator.h
+++ b/content/browser/renderer_host/navigator.h
@@ -187,32 +187,11 @@ class CONTENT_EXPORT Navigator {
   // Cancel a NavigationRequest for |frame_tree_node|.
   void CancelNavigation(FrameTreeNode* frame_tree_node);
 
-  // Called when the network stack started handling the navigation request
-  // so that the |timestamp| when it happened can be recorded into an histogram.
-  // The |url| is used to verify we're tracking the correct navigation.
-  // TODO(carlosk): Remove the URL parameter and rename this method to better
-  // suit naming conventions.
-  void LogResourceRequestTime(base::TimeTicks timestamp, const GURL& url);
-
-  // Called to record the time it took to execute the beforeunload hook for the
-  // current navigation.
-  void LogBeforeUnloadTime(base::TimeTicks renderer_before_unload_start_time,
-                           base::TimeTicks renderer_before_unload_end_time,
-                           base::TimeTicks before_unload_sent_time);
-
   NavigationControllerImpl* controller() { return controller_; }
 
  private:
   friend class NavigatorTestWithBrowserSideNavigation;
 
-  // Holds data used to track browser side navigation metrics.
-  struct NavigationMetricsData;
-
-  void RecordNavigationMetrics(
-      const LoadCommittedDetails& details,
-      const FrameHostMsg_DidCommitProvisionalLoad_Params& params,
-      SiteInstance* site_instance);
-
   // Called when a renderer initiated navigation has started. Returns the
   // pending NavigationEntry to be used. Either null or a new one owned
   // NavigationController.
@@ -220,14 +199,6 @@ class CONTENT_EXPORT Navigator {
       const mojom::CommonNavigationParams& common_params,
       FrameTreeNode* frame_tree_node);
 
-  // Called to record the time it took to execute beforeunload handlers for
-  // renderer-inititated navigations. It records the time it took to execute
-  // beforeunload handlers in the renderer process before sending the
-  // BeginNavigation IPC.
-  void LogRendererInitiatedBeforeUnloadTime(
-      base::TimeTicks renderer_before_unload_start_time,
-      base::TimeTicks renderer_before_unload_end_time);
-
   // The NavigationController that will keep track of session history for all
   // RenderFrameHost objects using this Navigator.
   // TODO(nasko): Move ownership of the NavigationController from
@@ -238,8 +209,6 @@ class CONTENT_EXPORT Navigator {
   // events. Can be nullptr in tests.
   NavigatorDelegate* delegate_;
 
-  std::unique_ptr<Navigator::NavigationMetricsData> navigation_data_;
-
   DISALLOW_COPY_AND_ASSIGN(Navigator);
 };
 
