diff --git a/content/browser/net/cross_origin_embedder_policy_reporter.cc b/content/browser/net/cross_origin_embedder_policy_reporter.cc
index f506654a2f5bb..ea0f2323686be 100644
--- a/content/browser/net/cross_origin_embedder_policy_reporter.cc
+++ b/content/browser/net/cross_origin_embedder_policy_reporter.cc
@@ -87,8 +87,6 @@ void CrossOriginEmbedderPolicyReporter::Clone(
 void CrossOriginEmbedderPolicyReporter::QueueAndNotify(
     std::initializer_list<std::pair<base::StringPiece, base::StringPiece>> body,
     bool report_only) {
-  const absl::optional<std::string>& endpoint =
-      report_only ? report_only_endpoint_ : endpoint_;
   const char* const disposition = report_only ? "reporting" : "enforce";
   if (observer_) {
     std::vector<blink::mojom::ReportBodyElementPtr> list;
@@ -103,6 +101,7 @@ void CrossOriginEmbedderPolicyReporter::QueueAndNotify(
     observer_->Notify(blink::mojom::Report::New(
         kType, context_url_, blink::mojom::ReportBody::New(std::move(list))));
   }
+#if 0
   if (endpoint) {
     base::DictionaryValue body_to_pass;
     for (const auto& pair : body) {
@@ -114,6 +113,7 @@ void CrossOriginEmbedderPolicyReporter::QueueAndNotify(
         kType, *endpoint, context_url_, network_isolation_key_,
         /*user_agent=*/absl::nullopt, std::move(body_to_pass));
   }
+#endif
 }
 
 }  // namespace content
