diff --git a/content/browser/renderer_host/delegated_frame_host.h b/content/browser/renderer_host/delegated_frame_host.h
index 48d95fe06ee8..b25ab268124a 100644
--- a/content/browser/renderer_host/delegated_frame_host.h
+++ b/content/browser/renderer_host/delegated_frame_host.h
@@ -227,8 +227,6 @@ class CONTENT_EXPORT DelegatedFrameHost
   // compositor frame is submitted.
   std::unique_ptr<ui::Layer> stale_content_layer_;
 
-  blink::ContentToVisibleTimeReporter tab_switch_time_recorder_;
-
   base::ObserverList<Observer>::Unchecked observers_;
 
   base::WeakPtrFactory<DelegatedFrameHost> weak_factory_{this};
