diff --git a/content/public/browser/render_process_host.h b/content/public/browser/render_process_host.h
index 0c013dbf045d..06a4dcd1d33e 100644
--- a/content/public/browser/render_process_host.h
+++ b/content/public/browser/render_process_host.h
@@ -466,8 +466,6 @@ class CONTENT_EXPORT RenderProcessHost : public IPC::Sender,
   // exposed here to support MockRenderProcessHost usage in tests.
   virtual void BindCacheStorage(
       const network::CrossOriginEmbedderPolicy& cross_origin_embedder_policy,
-      mojo::PendingRemote<network::mojom::CrossOriginEmbedderPolicyReporter>
-          coep_reporter_remote,
       const url::Origin& origin,
       mojo::PendingReceiver<blink::mojom::CacheStorage> receiver) = 0;
   virtual void BindFileSystemManager(
