diff --git a/content/browser/devtools/service_worker_devtools_manager.h b/content/browser/devtools/service_worker_devtools_manager.h
index 412fcb9774a2..fad36e5bdce4 100644
--- a/content/browser/devtools/service_worker_devtools_manager.h
+++ b/content/browser/devtools/service_worker_devtools_manager.h
@@ -68,8 +68,6 @@ class CONTENT_EXPORT ServiceWorkerDevToolsManager {
       bool is_installed_version,
       base::Optional<network::CrossOriginEmbedderPolicy>
           cross_origin_embedder_policy,
-      mojo::PendingRemote<network::mojom::CrossOriginEmbedderPolicyReporter>
-          coep_reporter,
       base::UnguessableToken* devtools_worker_token,
       bool* pause_on_start);
   void WorkerReadyForInspection(
@@ -80,9 +78,7 @@ class CONTENT_EXPORT ServiceWorkerDevToolsManager {
   void UpdateCrossOriginEmbedderPolicy(
       int worker_process_id,
       int worker_route_id,
-      network::CrossOriginEmbedderPolicy cross_origin_embedder_policy,
-      mojo::PendingRemote<network::mojom::CrossOriginEmbedderPolicyReporter>
-          coep_reporter);
+      network::CrossOriginEmbedderPolicy cross_origin_embedder_policy);
   void WorkerVersionInstalled(int worker_process_id, int worker_route_id);
   // If the worker instance is stopped its worker_process_id and
   // worker_route_id will be invalid. For that case we pass context
