diff --git a/content/browser/browser_child_process_host_impl_receiver_bindings.cc b/content/browser/browser_child_process_host_impl_receiver_bindings.cc
index a10a83f0e862..7cc0f9bea473 100644
--- a/content/browser/browser_child_process_host_impl_receiver_bindings.cc
+++ b/content/browser/browser_child_process_host_impl_receiver_bindings.cc
@@ -19,9 +19,6 @@
 #include "content/public/browser/browser_thread.h"
 #include "content/public/browser/device_service.h"
 #include "services/device/public/mojom/power_monitor.mojom.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
-#include "services/metrics/public/mojom/ukm_interface.mojom.h"
-#include "services/metrics/ukm_recorder_interface.h"
 
 #if defined(OS_MAC)
 #include "content/browser/sandbox_support_mac_impl.h"
@@ -111,12 +108,6 @@ void BrowserChildProcessHostImpl::BindHostReceiver(
     return;
   }
 
-  if (auto r = receiver.As<ukm::mojom::UkmRecorderInterface>()) {
-    metrics::UkmRecorderInterface::Create(ukm::UkmRecorder::Get(),
-                                          std::move(r));
-    return;
-  }
-
   delegate_->BindHostReceiver(std::move(receiver));
 }
 
