diff --git a/content/renderer/accessibility/render_accessibility_impl.h b/content/renderer/accessibility/render_accessibility_impl.h
index 6365bdf4d6f1a..773ca831f7424 100644
--- a/content/renderer/accessibility/render_accessibility_impl.h
+++ b/content/renderer/accessibility/render_accessibility_impl.h
@@ -43,10 +43,6 @@ class AXActionTarget;
 struct AXEvent;
 }
 
-namespace ukm {
-class MojoUkmRecorder;
-}
-
 namespace content {
 
 class AXImageAnnotator;
@@ -124,9 +120,6 @@ class CONTENT_EXPORT RenderAccessibilityImpl : public RenderAccessibility,
   // Returns the page language.
   std::string GetLanguage();
 
-  // Access the UKM recorder.
-  ukm::MojoUkmRecorder* ukm_recorder() const { return ukm_recorder_.get(); }
-
  protected:
   // Send queued events from the renderer to the browser.
   void SendPendingAccessibilityEvents();
@@ -287,9 +280,6 @@ class CONTENT_EXPORT RenderAccessibilityImpl : public RenderAccessibility,
   // The specified page language, or empty if unknown.
   std::string page_language_;
 
-  // The URL-keyed metrics recorder interface.
-  std::unique_ptr<ukm::MojoUkmRecorder> ukm_recorder_;
-
   // The longest amount of time spent serializing the accessibility tree
   // in SendPendingAccessibilityEvents. This is periodically uploaded as
   // a UKM and then reset.
