diff --git a/content/public/test/navigation_handle_observer.cc b/content/public/test/navigation_handle_observer.cc
index c1c38f5a1b00..adf4af43610a 100644
--- a/content/public/test/navigation_handle_observer.cc
+++ b/content/public/test/navigation_handle_observer.cc
@@ -37,7 +37,6 @@ void NavigationHandleObserver::DidStartNavigation(
   navigation_id_ = navigation_handle->GetNavigationId();
   navigation_start_ = navigation_handle->NavigationStart();
   reload_type_ = navigation_handle->GetReloadType();
-  next_page_ukm_source_id_ = navigation_handle->GetNextPageUkmSourceId();
 }
 
 void NavigationHandleObserver::DidFinishNavigation(
