diff --git a/content/browser/frame_host/cross_origin_opener_policy_status.h b/content/browser/frame_host/cross_origin_opener_policy_status.h
index c7875efc8c1a..1cf1ecfd984a 100644
--- a/content/browser/frame_host/cross_origin_opener_policy_status.h
+++ b/content/browser/frame_host/cross_origin_opener_policy_status.h
@@ -120,7 +120,7 @@ class CrossOriginOpenerPolicyStatus {
   // send a report for a navigation away from a current response. If false, the
   // |coop_reporter| field from this CrossOriginOpenerPolicyStatus should be
   // used instead.
-  bool use_current_document_coop_reporter_ = true;
+  ALLOW_UNUSED_TYPE bool use_current_document_coop_reporter_ = true;
 
   // The reporter currently in use by COOP.
   std::unique_ptr<CrossOriginOpenerPolicyReporter> coop_reporter_;
