diff --git a/content/browser/browser_child_process_host_impl_receiver_bindings.cc b/content/browser/browser_child_process_host_impl_receiver_bindings.cc
index a10a83f0e862..ff1b359fae7f 100644
--- a/content/browser/browser_child_process_host_impl_receiver_bindings.cc
+++ b/content/browser/browser_child_process_host_impl_receiver_bindings.cc
@@ -111,12 +111,6 @@ void BrowserChildProcessHostImpl::BindHostReceiver(
     return;
   }
 
-  if (auto r = receiver.As<ukm::mojom::UkmRecorderInterface>()) {
-    metrics::UkmRecorderInterface::Create(ukm::UkmRecorder::Get(),
-                                          std::move(r));
-    return;
-  }
-
   delegate_->BindHostReceiver(std::move(receiver));
 }
 
