diff --git a/content/public/test/mock_render_process_host.h b/content/public/test/mock_render_process_host.h
index d1dc8a0e254f..0fd835e48064 100644
--- a/content/public/test/mock_render_process_host.h
+++ b/content/public/test/mock_render_process_host.h
@@ -164,7 +164,6 @@ class MockRenderProcessHost : public RenderProcessHost {
   bool IsProcessLockedToSiteForTesting() override;
   void BindCacheStorage(
       const network::CrossOriginEmbedderPolicy&,
-      mojo::PendingRemote<network::mojom::CrossOriginEmbedderPolicyReporter>,
       const url::Origin& origin,
       mojo::PendingReceiver<blink::mojom::CacheStorage> receiver) override;
   void BindFileSystemManager(
