diff --git a/content/browser/download/download_manager_impl.cc b/content/browser/download/download_manager_impl.cc
index d62a36a287c09..fb33ce5eedf10 100644
--- a/content/browser/download/download_manager_impl.cc
+++ b/content/browser/download/download_manager_impl.cc
@@ -672,6 +672,7 @@ void DownloadManagerImpl::BindWakeLockProvider(
   GetDeviceService().BindWakeLockProvider(std::move(receiver));
 }
 
+#if 0
 download::QuarantineConnectionCallback
 DownloadManagerImpl::GetQuarantineConnectionCallback() {
   if (!delegate_)
@@ -679,6 +680,7 @@ DownloadManagerImpl::GetQuarantineConnectionCallback() {
 
   return delegate_->GetQuarantineConnectionCallback();
 }
+#endif
 
 std::unique_ptr<download::DownloadItemRenameHandler>
 DownloadManagerImpl::GetRenameHandlerForDownload(
@@ -1454,4 +1456,10 @@ DownloadManagerImpl::RetrieveInProgressDownload(uint32_t id) {
   return nullptr;
 }
 
+bool DownloadManagerImpl::ShouldRemoveAfterCompletion() {
+  if (delegate_)
+    return delegate_->ShouldRemoveAfterCompletion();
+  return false;
+}
+
 }  // namespace content
