diff --git a/content/renderer/render_widget.cc b/content/renderer/render_widget.cc
index 6055365c20f3..2ca14ae39014 100644
--- a/content/renderer/render_widget.cc
+++ b/content/renderer/render_widget.cc
@@ -14,7 +14,6 @@
 #include "base/logging.h"
 #include "base/macros.h"
 #include "cc/trees/layer_tree_host.h"
-#include "cc/trees/ukm_manager.h"
 #include "content/renderer/render_thread_impl.h"
 #include "ppapi/buildflags/buildflags.h"
 #include "third_party/blink/public/platform/web_rect.h"
@@ -147,7 +146,7 @@ void RenderWidget::InitCompositing(const blink::ScreenInfo& screen_info) {
   layer_tree_host_ = webwidget_->InitializeCompositing(
       compositor_deps_->GetWebMainThreadScheduler(),
       compositor_deps_->GetTaskGraphRunner(), for_child_local_root_frame_,
-      screen_info, compositor_deps_->CreateUkmRecorderFactory(),
+      screen_info,
       /*settings=*/nullptr);
   DCHECK(layer_tree_host_);
 }
