diff --git a/content/browser/payments/payment_app_provider_impl.cc b/content/browser/payments/payment_app_provider_impl.cc
index 7b10ed2406c9..7e086d3e64a7 100644
--- a/content/browser/payments/payment_app_provider_impl.cc
+++ b/content/browser/payments/payment_app_provider_impl.cc
@@ -973,12 +973,6 @@ bool PaymentAppProviderImpl::IsValidInstallablePaymentApp(
   return true;
 }
 
-ukm::SourceId PaymentAppProviderImpl::GetSourceIdForPaymentAppFromScope(
-    const GURL& sw_scope) {
-  return ukm::UkmRecorder::GetSourceIdForPaymentAppFromScope(
-      sw_scope.GetOrigin());
-}
-
 PaymentAppProviderImpl::PaymentAppProviderImpl() = default;
 
 PaymentAppProviderImpl::~PaymentAppProviderImpl() = default;
