diff --git a/content/browser/download/download_manager_impl.h b/content/browser/download/download_manager_impl.h
index 3218ad41839b..77e65891adcd 100644
--- a/content/browser/download/download_manager_impl.h
+++ b/content/browser/download/download_manager_impl.h
@@ -34,7 +34,6 @@
 #include "content/public/browser/download_manager_delegate.h"
 #include "content/public/browser/ssl_status.h"
 #include "mojo/public/cpp/system/data_pipe.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "services/network/public/mojom/url_loader.mojom.h"
 #include "url/origin.h"
 
