diff --git a/content/browser/worker_host/shared_worker_host.cc b/content/browser/worker_host/shared_worker_host.cc
index fd4c6cf1249a..d3a56d49cd64 100644
--- a/content/browser/worker_host/shared_worker_host.cc
+++ b/content/browser/worker_host/shared_worker_host.cc
@@ -321,7 +321,6 @@ SharedWorkerHost::CreateNetworkFactoryParamsForSubresources() {
           net::IsolationInfo::Create(net::IsolationInfo::RequestType::kOther,
                                      origin, origin,
                                      net::SiteForCookies::FromOrigin(origin)),
-          /*coep_reporter=*/mojo::NullRemote(),
           /*auth_cert_observer=*/mojo::NullRemote(),
           /*debug_tag=*/"SharedWorkerHost::CreateNetworkFactoryForSubresource");
   return factory_params;
@@ -389,14 +388,9 @@ void SharedWorkerHost::BindCacheStorage(
   // cache.match() for SharedWorker by providing the correct value here.
   network::CrossOriginEmbedderPolicy cross_origin_embedder_policy;
 
-  // TODO(https://crbug.com/1031542): Plumb a CrossOriginEmbedderPolicyReporter
-  // here to handle reports.
-  mojo::PendingRemote<network::mojom::CrossOriginEmbedderPolicyReporter>
-      coep_reporter;
-
   const url::Origin origin = url::Origin::Create(instance().url());
   GetProcessHost()->BindCacheStorage(cross_origin_embedder_policy,
-                                     std::move(coep_reporter), origin,
+                                     origin,
                                      std::move(receiver));
 }
 
