diff --git a/content/renderer/render_thread_impl.h b/content/renderer/render_thread_impl.h
index 939c6ceb1bef..acd7599324d3 100644
--- a/content/renderer/render_thread_impl.h
+++ b/content/renderer/render_thread_impl.h
@@ -201,7 +201,6 @@ class CONTENT_EXPORT RenderThreadImpl
   blink::scheduler::WebThreadScheduler* GetWebMainThreadScheduler() override;
   cc::TaskGraphRunner* GetTaskGraphRunner() override;
   bool IsScrollAnimatorEnabled() override;
-  std::unique_ptr<cc::UkmRecorderFactory> CreateUkmRecorderFactory() override;
   void RequestNewLayerTreeFrameSink(
       RenderWidget* render_widget,
       const GURL& url,
