diff --git a/content/browser/plugin_service_impl.h b/content/browser/plugin_service_impl.h
index 79ac3b705511..f4e044bb6334 100644
--- a/content/browser/plugin_service_impl.h
+++ b/content/browser/plugin_service_impl.h
@@ -122,9 +122,6 @@ class CONTENT_EXPORT PluginServiceImpl : public PluginService {
   // Pulled out of the air, seems reasonable.
   static constexpr int kDefaultMaxPpapiProcessesPerProfile = 15;
 
-  // Helper for recording URLs to UKM.
-  static void RecordBrokerUsage(int render_process_id, int render_frame_id);
-
   // Creates the PluginServiceImpl object, but doesn't actually build the plugin
   // list yet.  It's generated lazily.
   PluginServiceImpl();
