diff --git a/content/browser/plugin_service_impl.cc b/content/browser/plugin_service_impl.cc
index e9cb48e03f17..d98bbd1840b8 100644
--- a/content/browser/plugin_service_impl.cc
+++ b/content/browser/plugin_service_impl.cc
@@ -46,7 +46,6 @@
 #include "content/public/common/process_type.h"
 #include "content/public/common/webplugininfo.h"
 #include "ppapi/shared_impl/ppapi_permissions.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
 
 namespace content {
 namespace {
@@ -69,18 +68,6 @@ void WillLoadPluginsCallback(base::SequenceChecker* sequence_checker) {
 
 }  // namespace
 
-// static
-void PluginServiceImpl::RecordBrokerUsage(int render_process_id,
-                                          int render_frame_id) {
-  RenderFrameHostImpl* rfh =
-      RenderFrameHostImpl::FromID(render_process_id, render_frame_id);
-  if (!rfh)
-    return;
-
-  ukm::SourceId source_id = rfh->GetPageUkmSourceId();
-  ukm::builders::Pepper_Broker(source_id).Record(ukm::UkmRecorder::Get());
-}
-
 // static
 PluginService* PluginService::GetInstance() {
   return PluginServiceImpl::GetInstance();
@@ -314,10 +301,6 @@ void PluginServiceImpl::OpenChannelToPpapiBroker(
     int render_frame_id,
     const base::FilePath& path,
     PpapiPluginProcessHost::BrokerClient* client) {
-  GetUIThreadTaskRunner({})->PostTask(
-      FROM_HERE, base::BindOnce(&PluginServiceImpl::RecordBrokerUsage,
-                                render_process_id, render_frame_id));
-
   PpapiPluginProcessHost* plugin_host = FindOrStartPpapiBrokerProcess(
       render_process_id, path);
   if (plugin_host) {
