diff --git a/content/browser/storage_partition_impl.cc b/content/browser/storage_partition_impl.cc
index 4bb577abb154e..10be118288b3d 100644
--- a/content/browser/storage_partition_impl.cc
+++ b/content/browser/storage_partition_impl.cc
@@ -1862,6 +1862,7 @@ void StoragePartitionImpl::OnCanSendReportingReports(
   std::move(callback).Run(origins_out);
 }
 
+#if 0
 void StoragePartitionImpl::OnCanSendDomainReliabilityUpload(
     const GURL& origin,
     OnCanSendDomainReliabilityUploadCallback callback) {
@@ -1873,6 +1874,7 @@ void StoragePartitionImpl::OnCanSendDomainReliabilityUpload(
           content::PermissionType::BACKGROUND_SYNC, origin, origin) ==
       blink::mojom::PermissionStatus::GRANTED);
 }
+#endif
 
 void StoragePartitionImpl::OnClearSiteData(const GURL& url,
                                            const std::string& header_value,
