diff --git a/content/browser/renderer_host/render_widget_host_view_mac.mm b/content/browser/renderer_host/render_widget_host_view_mac.mm
index 9b6a07b8af23..0878d56d4682 100644
--- a/content/browser/renderer_host/render_widget_host_view_mac.mm
+++ b/content/browser/renderer_host/render_widget_host_view_mac.mm
@@ -454,26 +454,14 @@
   DelegatedFrameHost* delegated_frame_host =
       browser_compositor_->GetDelegatedFrameHost();
 
-  bool has_saved_frame =
-      browser_compositor_->has_saved_frame_before_state_transition();
-
-  auto tab_switch_start_state = TakeRecordContentToVisibleTimeRequest();
-
-  const bool renderer_should_record_presentation_time = !has_saved_frame;
-  host()->WasShown(renderer_should_record_presentation_time
-                       ? tab_switch_start_state.Clone()
-                       : blink::mojom::RecordContentToVisibleTimeRequestPtr());
+  host()->WasShown({});
 
   if (delegated_frame_host) {
     // If the frame for the renderer is already available, then the
     // tab-switching time is the presentation time for the browser-compositor.
-    const bool record_presentation_time = has_saved_frame;
     delegated_frame_host->WasShown(
         browser_compositor_->GetRendererLocalSurfaceId(),
-        browser_compositor_->GetRendererSize(),
-        record_presentation_time
-            ? std::move(tab_switch_start_state)
-            : blink::mojom::RecordContentToVisibleTimeRequestPtr());
+        browser_compositor_->GetRendererSize(), {});
   }
 }
 
