diff --git a/content/test/BUILD.gn b/content/test/BUILD.gn
index 1bcb30608d8d..ceda93817a05 100644
--- a/content/test/BUILD.gn
+++ b/content/test/BUILD.gn
@@ -425,7 +425,6 @@ static_library("test_support") {
     "//services/data_decoder/public/cpp:test_support",
     "//services/data_decoder/public/mojom",
     "//services/device/public/mojom",
-    "//services/metrics/public/cpp:metrics_cpp",
 
     # TODO(jam): remove this by adding a public header for the NetworkContext
     # public testing method.
@@ -1253,7 +1252,6 @@ test("content_browsertests") {
     "//components/services/storage",
     "//components/services/storage/public/cpp",
     "//components/services/storage/public/mojom:test_api",
-    "//components/ukm:test_support",
     "//components/url_formatter:url_formatter",
     "//components/viz/host",
     "//components/viz/test:test_support",
@@ -1303,7 +1301,6 @@ test("content_browsertests") {
     "//services/device/public/mojom:generic_sensor",
     "//services/image_annotation/public/cpp:cpp",
     "//services/image_annotation/public/mojom:mojom",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network:test_support",
     "//services/network/trust_tokens:test_support",
     "//services/service_manager/public/cpp",
@@ -2185,7 +2182,6 @@ test("content_unittests") {
     "//components/services/storage/dom_storage:local_storage_proto",
     "//components/services/storage/public/cpp",
     "//components/system_media_controls:test_support",
-    "//components/ukm:test_support",
     "//components/viz/client",
     "//components/viz/common",
     "//components/viz/host",
@@ -2242,8 +2238,6 @@ test("content_unittests") {
     "//services/device/public/cpp:test_support",
     "//services/device/public/cpp/generic_sensor",
     "//services/device/public/mojom",
-    "//services/metrics/public/cpp:ukm_builders",
-    "//services/metrics/public/mojom",
     "//services/network:network_service",
     "//services/network:test_support",
     "//services/network/public/cpp",
