diff --git a/content/browser/renderer_host/pepper/pepper_file_io_host.cc b/content/browser/renderer_host/pepper/pepper_file_io_host.cc
index f0b57618b37c6..c1b682653d87b 100644
--- a/content/browser/renderer_host/pepper/pepper_file_io_host.cc
+++ b/content/browser/renderer_host/pepper/pepper_file_io_host.cc
@@ -480,6 +480,7 @@ void PepperFileIOHost::OnLocalFileOpened(
     ppapi::host::ReplyMessageContext reply_context,
     const base::FilePath& path,
     base::File::Error error_code) {
+#if 0
 #if defined(OS_WIN) || defined(OS_LINUX) || defined(OS_CHROMEOS)
   // Quarantining a file before its contents are available is only supported on
   // Windows and Linux.
@@ -510,10 +511,12 @@ void PepperFileIOHost::OnLocalFileOpened(
     SendFileOpenReply(reply_context, error_code);
   }
 #else
-  SendFileOpenReply(reply_context, error_code);
 #endif
+#endif
+  SendFileOpenReply(reply_context, error_code);
 }
 
+#if 0
 #if defined(OS_WIN) || defined(OS_LINUX) || defined(OS_CHROMEOS)
 void PepperFileIOHost::OnLocalFileQuarantined(
     ppapi::host::ReplyMessageContext reply_context,
@@ -529,6 +532,7 @@ void PepperFileIOHost::OnLocalFileQuarantined(
   SendFileOpenReply(reply_context, file_error);
 }
 #endif
+#endif
 
 void PepperFileIOHost::SendFileOpenReply(
     ppapi::host::ReplyMessageContext reply_context,
