diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 434c77afe3c6..38d15b90e47e 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -60,7 +60,6 @@ source_set("browser") {
     "//components/discardable_memory/service",
     "//components/download/database",
     "//components/download/public/common:public",
-    "//components/download/quarantine",
     "//components/filename_generation",
     "//components/link_header_util",
     "//components/metrics",
@@ -73,7 +72,6 @@ source_set("browser") {
     "//components/payments/core",
     "//components/payments/core:error_strings",
     "//components/payments/mojom",
-    "//components/rappor",
     "//components/services/filesystem:lib",
     "//components/services/quarantine:quarantine",
     "//components/services/storage",
@@ -1304,6 +1302,10 @@ source_set("browser") {
     "net/browser_online_state_observer.cc",
     "net/browser_online_state_observer.h",
     "net/cookie_store_factory.cc",
+    "net/cross_origin_embedder_policy_reporter.cc",
+    "net/cross_origin_embedder_policy_reporter.h",
+    "net/cross_origin_opener_policy_reporter.cc",
+    "net/cross_origin_opener_policy_reporter.h",
     "net/network_errors_listing_ui.cc",
     "net/network_errors_listing_ui.h",
     "net/network_quality_observer_impl.cc",
@@ -2768,8 +2770,6 @@ source_set("browser") {
 
   if (enable_reporting) {
     sources += [
-      "net/cross_origin_embedder_policy_reporter.cc",
-      "net/cross_origin_embedder_policy_reporter.h",
       "net/cross_origin_opener_policy_reporter.cc",
       "net/cross_origin_opener_policy_reporter.h",
       "net/reporting_service_proxy.cc",
