diff --git a/content/browser/browser_child_process_host_impl.cc b/content/browser/browser_child_process_host_impl.cc
index 1252731a0833..05c89f574cbf 100644
--- a/content/browser/browser_child_process_host_impl.cc
+++ b/content/browser/browser_child_process_host_impl.cc
@@ -48,7 +48,6 @@
 #include "content/public/common/process_type.h"
 #include "content/public/common/result_codes.h"
 #include "content/public/common/sandboxed_process_launcher_delegate.h"
-#include "mojo/public/cpp/bindings/scoped_message_error_crash_key.h"
 #include "mojo/public/cpp/system/platform_handle.h"
 #include "services/tracing/public/cpp/trace_startup.h"
 
@@ -709,7 +708,6 @@ void BrowserChildProcessHostImpl::OnMojoError(
   // It is important to call DumpWithoutCrashing synchronously - this will help
   // to preserve the callstack and the crash keys present when the bad mojo
   // message was received.
-  mojo::debug::ScopedMessageErrorCrashKey scoped_error_key(error);
   base::debug::DumpWithoutCrashing();
 
   if (task_runner->BelongsToCurrentThread()) {
