diff --git a/content/renderer/compositor/compositor_dependencies.h b/content/renderer/compositor/compositor_dependencies.h
index 7a544ab70f95..b085f07506df 100644
--- a/content/renderer/compositor/compositor_dependencies.h
+++ b/content/renderer/compositor/compositor_dependencies.h
@@ -43,8 +43,6 @@ class CONTENT_EXPORT CompositorDependencies {
   virtual blink::scheduler::WebThreadScheduler* GetWebMainThreadScheduler() = 0;
   virtual cc::TaskGraphRunner* GetTaskGraphRunner() = 0;
   virtual bool IsScrollAnimatorEnabled() = 0;
-  virtual std::unique_ptr<cc::UkmRecorderFactory>
-  CreateUkmRecorderFactory() = 0;
 
   virtual ~CompositorDependencies() {}
 };
