diff --git a/content/browser/service_worker/service_worker_version.h b/content/browser/service_worker/service_worker_version.h
index 2c14dea5aa27..9d57174ac66d 100644
--- a/content/browser/service_worker/service_worker_version.h
+++ b/content/browser/service_worker/service_worker_version.h
@@ -614,12 +614,6 @@ class CONTENT_EXPORT ServiceWorkerVersion
     return receiver_;
   }
 
-  void set_reporting_observer_receiver(
-      mojo::PendingReceiver<blink::mojom::ReportingObserver>
-          reporting_observer_receiver) {
-    reporting_observer_receiver_ = std::move(reporting_observer_receiver);
-  }
-
  private:
   friend class base::RefCounted<ServiceWorkerVersion>;
   friend class EmbeddedWorkerInstanceTest;
@@ -1124,9 +1118,6 @@ class CONTENT_EXPORT ServiceWorkerVersion
   // Used only when ServiceWorkerTerminationOnNoControllee is on.
   base::CancelableOnceClosure stop_on_no_controllee_callback_;
 
-  mojo::PendingReceiver<blink::mojom::ReportingObserver>
-      reporting_observer_receiver_;
-
   // Lives while the ServiceWorkerVersion is alive.
   // See comments at the definition of storage::mojom::ServiceWorkerVersionRef
   // for more details.
