diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index c853d52ad829..f0e10e214f59 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -454,7 +454,6 @@ source_set("browser_sources") {
     "//net",
     "//ppapi/c",
     "//services/device/public/cpp/geolocation",
-    "//services/metrics/public/cpp:metrics_cpp",
     "//ui/accessibility",
     "//ui/base",
     "//ui/base/clipboard",
