diff --git a/content/renderer/worker/shared_worker_factory_impl.cc b/content/renderer/worker/shared_worker_factory_impl.cc
index 9369b9c60b74..300a4cc382bd 100644
--- a/content/renderer/worker/shared_worker_factory_impl.cc
+++ b/content/renderer/worker/shared_worker_factory_impl.cc
@@ -46,8 +46,7 @@ void SharedWorkerFactoryImpl::CreateSharedWorker(
     mojo::PendingRemote<blink::mojom::SharedWorkerHost> host,
     mojo::PendingReceiver<blink::mojom::SharedWorker> receiver,
     mojo::PendingRemote<blink::mojom::BrowserInterfaceBroker>
-        browser_interface_broker,
-    ukm::SourceId ukm_source_id) {
+        browser_interface_broker) {
   // Bound to the lifetime of the underlying blink::WebSharedWorker instance.
   new EmbeddedSharedWorkerStub(
       std::move(info), token, constructor_origin, user_agent, ua_metadata,
@@ -58,7 +57,6 @@ void SharedWorkerFactoryImpl::CreateSharedWorker(
       std::move(main_script_load_params),
       std::move(subresource_loader_factories), std::move(controller_info),
       std::move(host), std::move(receiver), std::move(browser_interface_broker),
-      ukm_source_id,
       RenderThreadImpl::current()
           ->resource_dispatcher()
           ->cors_exempt_header_list());
