diff --git a/content/browser/renderer_host/navigation_request.cc b/content/browser/renderer_host/navigation_request.cc
index 9e83a156d7e1..2a38de428525 100644
--- a/content/browser/renderer_host/navigation_request.cc
+++ b/content/browser/renderer_host/navigation_request.cc
@@ -99,7 +99,6 @@
 #include "net/http/http_request_headers.h"
 #include "net/http/http_status_code.h"
 #include "net/url_request/redirect_info.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "services/network/public/cpp/content_security_policy/content_security_policy.h"
 #include "services/network/public/cpp/cross_origin_embedder_policy.h"
 #include "services/network/public/cpp/cross_origin_resource_policy.h"
@@ -1062,9 +1061,7 @@ NavigationRequest::NavigationRequest(
           frame_tree_node->current_frame_host()->GetRoutingID())),
       initiator_routing_id_(initiator_routing_id),
       client_security_state_(network::mojom::ClientSecurityState::New()),
-      coop_status_(frame_tree_node, common_params_->initiator_origin),
-      previous_page_ukm_source_id_(
-          frame_tree_node_->current_frame_host()->GetPageUkmSourceId()) {
+      coop_status_(frame_tree_node, common_params_->initiator_origin) {
   DCHECK(browser_initiated_ || common_params_->initiator_origin.has_value());
   DCHECK(!IsRendererDebugURL(common_params_->url));
   DCHECK(common_params_->method == "POST" || !common_params_->post_data);
@@ -1169,13 +1166,6 @@ NavigationRequest::NavigationRequest(
   // Add necessary headers that may not be present in the
   // mojom::BeginNavigationParams.
   if (entry) {
-    // TODO(altimin, crbug.com/933147): Remove this logic after we are done
-    // with implementing back-forward cache.
-    if (frame_tree_node->IsMainFrame() && entry->back_forward_cache_metrics()) {
-      entry->back_forward_cache_metrics()
-          ->MainFrameDidStartNavigationToDocument();
-    }
-
     // If this NavigationRequest is for the current pending entry, make sure
     // that we will discard the pending entry if all of associated its requests
     // go away, by creating a ref to it.
@@ -1654,10 +1644,6 @@ NavigationRequest::TakeCoepReporter() {
   return std::move(coep_reporter_);
 }
 
-ukm::SourceId NavigationRequest::GetPreviousPageUkmSourceId() {
-  return previous_page_ukm_source_id_;
-}
-
 NavigationEntry* NavigationRequest::GetNavigationEntry() {
   if (nav_entry_id_ == 0)
     return nullptr;
@@ -2610,8 +2596,6 @@ bool NavigationRequest::ShouldKeepErrorPageInCurrentProcess(int net_error) {
 }
 
 void NavigationRequest::OnRequestStarted(base::TimeTicks timestamp) {
-  frame_tree_node_->navigator().LogResourceRequestTime(timestamp,
-                                                       common_params_->url);
 }
 namespace {
 
@@ -4537,21 +4521,6 @@ int64_t NavigationRequest::GetNavigationId() {
   return navigation_id_;
 }
 
-ukm::SourceId NavigationRequest::GetNextPageUkmSourceId() {
-  // If the navigation is restoring from back-forward cache, the UKM id
-  // will get restored, too.
-  if (rfh_restored_from_back_forward_cache_)
-    return rfh_restored_from_back_forward_cache_->GetPageUkmSourceId();
-
-  // If this is the same document or a child frame navigation the UKM id will
-  // not change from it.
-  if (IsSameDocument() || !IsInMainFrame())
-    return previous_page_ukm_source_id_;
-
-  return ukm::ConvertToSourceId(navigation_id_,
-                                base::UkmSourceId::Type::NAVIGATION_ID);
-}
-
 const GURL& NavigationRequest::GetURL() {
   return common_params().url;
 }
