diff --git a/content/browser/plugin_service_impl.cc b/content/browser/plugin_service_impl.cc
index 7caf00713119..53fe8fa9ad68 100644
--- a/content/browser/plugin_service_impl.cc
+++ b/content/browser/plugin_service_impl.cc
@@ -45,7 +45,6 @@
 #include "content/public/common/process_type.h"
 #include "content/public/common/webplugininfo.h"
 #include "ppapi/shared_impl/ppapi_permissions.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
 
 namespace content {
 namespace {
