diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 266fd660ec0c..95040602b8b1 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -64,7 +64,6 @@ source_set("browser") {
     "//components/discardable_memory/service",
     "//components/download/database",
     "//components/download/public/common:public",
-    "//components/download/quarantine",
     "//components/filename_generation",
     "//components/link_header_util",
     "//components/metrics",
@@ -77,7 +76,6 @@ source_set("browser") {
     "//components/payments/core",
     "//components/payments/core:error_strings",
     "//components/payments/mojom",
-    "//components/rappor",
     "//components/services/filesystem:lib",
     "//components/services/quarantine:quarantine",
     "//components/services/storage",
@@ -88,8 +86,6 @@ source_set("browser") {
     "//components/system_media_controls",
     "//components/tracing:startup_tracing",
     "//components/ui_devtools",
-    "//components/ukm",
-    "//components/ukm/debug:util",
     "//components/url_formatter",
     "//components/variations",
     "//components/variations/net",
@@ -814,8 +810,6 @@ source_set("browser") {
     "eye_dropper_chooser_impl.h",
     "feature_observer.cc",
     "feature_observer.h",
-    "federated_learning/floc_service_impl.cc",
-    "federated_learning/floc_service_impl.h",
     "field_trial_recorder.cc",
     "field_trial_recorder.h",
     "field_trial_synchronizer.cc",
@@ -1749,8 +1743,6 @@ source_set("browser") {
     "tracing/tracing_controller_impl_data_endpoint.cc",
     "tracing/tracing_service_controller.cc",
     "tracing/tracing_service_controller.h",
-    "ukm_internals_ui.cc",
-    "ukm_internals_ui.h",
     "url_loader_factory_getter.cc",
     "url_loader_factory_getter.h",
     "url_loader_factory_params_helper.cc",
