diff --git a/content/browser/service_worker/service_worker_info.cc b/content/browser/service_worker/service_worker_info.cc
index ea14403c2e64..22e1f62702c3 100644
--- a/content/browser/service_worker/service_worker_info.cc
+++ b/content/browser/service_worker/service_worker_info.cc
@@ -22,8 +22,7 @@ ServiceWorkerVersionInfo::ServiceWorkerVersionInfo()
       version_id(blink::mojom::kInvalidServiceWorkerVersionId),
       process_id(ChildProcessHost::kInvalidUniqueID),
       thread_id(ServiceWorkerConsts::kInvalidEmbeddedWorkerThreadId),
-      devtools_agent_route_id(MSG_ROUTING_NONE),
-      ukm_source_id(ukm::kInvalidSourceId) {}
+      devtools_agent_route_id(MSG_ROUTING_NONE) {}
 
 ServiceWorkerVersionInfo::ServiceWorkerVersionInfo(
     EmbeddedWorkerStatus running_status,
@@ -35,8 +34,7 @@ ServiceWorkerVersionInfo::ServiceWorkerVersionInfo(
     int64_t version_id,
     int process_id,
     int thread_id,
-    int devtools_agent_route_id,
-    ukm::SourceId ukm_source_id)
+    int devtools_agent_route_id)
     : running_status(running_status),
       status(status),
       fetch_handler_existence(fetch_handler_existence),
@@ -46,8 +44,7 @@ ServiceWorkerVersionInfo::ServiceWorkerVersionInfo(
       version_id(version_id),
       process_id(process_id),
       thread_id(thread_id),
-      devtools_agent_route_id(devtools_agent_route_id),
-      ukm_source_id(ukm_source_id) {}
+      devtools_agent_route_id(devtools_agent_route_id) {}
 
 ServiceWorkerVersionInfo::ServiceWorkerVersionInfo(
     const ServiceWorkerVersionInfo& other) = default;
