diff --git a/content/browser/service_worker/service_worker_context_core_observer.h b/content/browser/service_worker/service_worker_context_core_observer.h
index b6959b62c9e6..c5a8fa212d50 100644
--- a/content/browser/service_worker/service_worker_context_core_observer.h
+++ b/content/browser/service_worker/service_worker_context_core_observer.h
@@ -109,13 +109,11 @@ class ServiceWorkerContextCoreObserver {
   // Called when a container host representing a client is execution ready. See
   // https://html.spec.whatwg.org/multipage/webappapis.html#concept-environment-execution-ready-flag
   virtual void OnClientIsExecutionReady(
-      ukm::SourceId source_id,
       const GURL& url,
       blink::mojom::ServiceWorkerClientType type) {}
 
   // Called when a container host representing a client is destroyed.
-  virtual void OnClientDestroyed(ukm::SourceId source_id,
-                                 const GURL& url,
+  virtual void OnClientDestroyed(const GURL& url,
                                  blink::mojom::ServiceWorkerClientType type) {}
 
  protected:
