diff --git a/content/browser/worker_host/worker_script_fetch_initiator.cc b/content/browser/worker_host/worker_script_fetch_initiator.cc
index 4e65906a7145..f475a1482ca8 100644
--- a/content/browser/worker_host/worker_script_fetch_initiator.cc
+++ b/content/browser/worker_host/worker_script_fetch_initiator.cc
@@ -48,7 +48,6 @@
 #include "net/base/isolation_info.h"
 #include "net/base/load_flags.h"
 #include "net/http/http_request_headers.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "services/network/public/cpp/constants.h"
 #include "services/network/public/cpp/shared_url_loader_factory.h"
 #include "third_party/blink/public/common/loader/url_loader_factory_bundle.h"
@@ -78,7 +77,6 @@ void WorkerScriptFetchInitiator::Start(
     scoped_refptr<network::SharedURLLoaderFactory> url_loader_factory_override,
     StoragePartitionImpl* storage_partition,
     const std::string& storage_domain,
-    ukm::SourceId worker_source_id,
     DevToolsAgentHostImpl* devtools_agent_host,
     const base::UnguessableToken& devtools_worker_token,
     CompletionCallback callback) {
@@ -185,7 +183,7 @@ void WorkerScriptFetchInitiator::Start(
       std::move(subresource_loader_factories),
       std::move(service_worker_context), service_worker_handle,
       std::move(appcache_host), std::move(blob_url_loader_factory),
-      std::move(url_loader_factory_override), worker_source_id,
+      std::move(url_loader_factory_override),
       devtools_agent_host, devtools_worker_token, std::move(callback));
 }
 
@@ -308,7 +306,6 @@ void WorkerScriptFetchInitiator::CreateScriptLoader(
     base::WeakPtr<AppCacheHost> appcache_host,
     scoped_refptr<network::SharedURLLoaderFactory> blob_url_loader_factory,
     scoped_refptr<network::SharedURLLoaderFactory> url_loader_factory_override,
-    ukm::SourceId worker_source_id,
     DevToolsAgentHostImpl* devtools_agent_host,
     const base::UnguessableToken& devtools_worker_token,
     CompletionCallback callback) {
@@ -354,7 +351,7 @@ void WorkerScriptFetchInitiator::CreateScriptLoader(
         request_initiator,
         /*navigation_id=*/base::nullopt,
         /* TODO(https://crbug.com/1103288): The UKM ID could be computed */
-        ukm::kInvalidSourceIdObj, &default_factory_receiver,
+        &default_factory_receiver,
         &factory_params->header_client, &bypass_redirect_checks,
         nullptr /* disable_secure_dns */, &factory_params->factory_override);
     factory_bundle_for_browser_info->set_bypass_redirect_checks(
@@ -399,7 +396,7 @@ void WorkerScriptFetchInitiator::CreateScriptLoader(
       std::make_unique<WorkerScriptLoaderFactory>(
           worker_process_id, worker_token, service_worker_handle,
           std::move(appcache_host), browser_context_getter,
-          std::move(url_loader_factory), worker_source_id),
+          std::move(url_loader_factory)),
       std::move(throttles), std::move(resource_request),
       base::BindOnce(WorkerScriptFetchInitiator::DidCreateScriptLoader,
                      std::move(callback),
