diff --git a/content/public/browser/platform_notification_service.h b/content/public/browser/platform_notification_service.h
index ca61088079c4..4b5f74904a49 100644
--- a/content/public/browser/platform_notification_service.h
+++ b/content/public/browser/platform_notification_service.h
@@ -80,10 +80,6 @@ class CONTENT_EXPORT PlatformNotificationService {
   // Reads the value of the next persistent notification ID from the profile and
   // increments the value, as it is called once per notification write.
   virtual int64_t ReadNextPersistentNotificationId() = 0;
-
-  // Records a given notification to UKM.
-  virtual void RecordNotificationUkmEvent(
-      const NotificationDatabaseData& data) = 0;
 };
 
 }  // namespace content
