diff --git a/content/browser/file_system_access/file_system_access_manager_impl.h b/content/browser/file_system_access/file_system_access_manager_impl.h
index ec28b88254197..6b3651c0088aa 100644
--- a/content/browser/file_system_access/file_system_access_manager_impl.h
+++ b/content/browser/file_system_access/file_system_access_manager_impl.h
@@ -11,7 +11,6 @@
 #include "base/memory/weak_ptr.h"
 #include "base/threading/sequence_bound.h"
 #include "base/types/pass_key.h"
-#include "components/download/public/common/quarantine_connection.h"
 #include "components/services/storage/public/mojom/file_system_access_context.mojom.h"
 #include "content/browser/blob_storage/chrome_blob_storage_context.h"
 #include "content/browser/file_system_access/file_system_chooser.h"
@@ -170,8 +169,7 @@ class CONTENT_EXPORT FileSystemAccessManagerImpl
       const SharedHandleState& handle_state,
       mojo::PendingReceiver<blink::mojom::FileSystemAccessFileWriter> receiver,
       bool has_transient_user_activation,
-      bool auto_close,
-      download::QuarantineConnectionCallback quarantine_connection_callback);
+      bool auto_close);
 
   // Create a transfer token for a specific file or directory.
   void CreateTransferToken(
