diff --git a/content/browser/renderer_host/render_widget_host_view_aura.cc b/content/browser/renderer_host/render_widget_host_view_aura.cc
index f6cd4515bf8f..8816ac55941b 100644
--- a/content/browser/renderer_host/render_widget_host_view_aura.cc
+++ b/content/browser/renderer_host/render_widget_host_view_aura.cc
@@ -1442,14 +1442,6 @@ bool RenderWidgetHostViewAura::IsTextEditCommandEnabled(
 void RenderWidgetHostViewAura::SetTextEditCommandForNextKeyEvent(
     ui::TextEditCommand command) {}
 
-ukm::SourceId RenderWidgetHostViewAura::GetClientSourceForMetrics() const {
-  RenderFrameHostImpl* frame = GetFocusedFrame();
-  if (frame) {
-    return frame->GetPageUkmSourceId();
-  }
-  return ukm::SourceId();
-}
-
 bool RenderWidgetHostViewAura::ShouldDoLearning() {
   return GetTextInputManager() && GetTextInputManager()->should_do_learning();
 }
