diff --git a/content/renderer/render_view_impl.cc b/content/renderer/render_view_impl.cc
index be0a2a33bc8e7..1515b7e9ca8ed 100644
--- a/content/renderer/render_view_impl.cc
+++ b/content/renderer/render_view_impl.cc
@@ -79,7 +79,7 @@ WindowOpenDisposition RenderViewImpl::NavigationPolicyToDisposition(
     case blink::kWebNavigationPolicyNewWindow:
       return WindowOpenDisposition::NEW_WINDOW;
     case blink::kWebNavigationPolicyNewPopup:
-      return WindowOpenDisposition::NEW_POPUP;
+      return WindowOpenDisposition::NEW_FOREGROUND_TAB;
     default:
       NOTREACHED() << "Unexpected WebNavigationPolicy";
       return WindowOpenDisposition::IGNORE_ACTION;
