diff --git a/content/public/browser/network_context_client_base.h b/content/public/browser/network_context_client_base.h
index fd39485d3020..6b972d03a094 100644
--- a/content/public/browser/network_context_client_base.h
+++ b/content/public/browser/network_context_client_base.h
@@ -75,7 +75,6 @@ class CONTENT_EXPORT NetworkContextClientBase
 #if defined(OS_CHROMEOS)
   void OnTrustAnchorUsed() override;
 #endif
-  void OnSCTReportReady(const std::string& cache_key) override;
 };
 
 }  // namespace content
