diff --git a/content/browser/storage_partition_impl.h b/content/browser/storage_partition_impl.h
index a4712173d769..e3ef8f2d8a66 100644
--- a/content/browser/storage_partition_impl.h
+++ b/content/browser/storage_partition_impl.h
@@ -269,7 +269,6 @@ class CONTENT_EXPORT StoragePartitionImpl
 #if defined(OS_CHROMEOS)
   void OnTrustAnchorUsed() override;
 #endif
-  void OnSCTReportReady(const std::string& cache_key) override;
 
   scoped_refptr<URLLoaderFactoryGetter> url_loader_factory_getter() {
     return url_loader_factory_getter_;
