diff --git a/content/browser/background_fetch/background_fetch_data_manager.cc b/content/browser/background_fetch/background_fetch_data_manager.cc
index 66d736c87bef..7ba9e6152eae 100644
--- a/content/browser/background_fetch/background_fetch_data_manager.cc
+++ b/content/browser/background_fetch/background_fetch_data_manager.cc
@@ -94,7 +94,7 @@ BackgroundFetchDataManager::GetOrOpenCacheStorage(
   network::CrossOriginEmbedderPolicy cross_origin_embedder_policy;
   DCHECK(cache_storage_context_);
   cache_storage_context_->AddReceiver(
-      cross_origin_embedder_policy, mojo::NullRemote(), origin,
+      cross_origin_embedder_policy, origin,
       storage::mojom::CacheStorageOwner::kBackgroundFetch,
       remote.BindNewPipeAndPassReceiver());
 
