diff --git a/content/browser/worker_host/dedicated_worker_host.cc b/content/browser/worker_host/dedicated_worker_host.cc
index 887472c53fdf..ae5ba4901896 100644
--- a/content/browser/worker_host/dedicated_worker_host.cc
+++ b/content/browser/worker_host/dedicated_worker_host.cc
@@ -31,7 +31,6 @@
 #include "mojo/public/cpp/bindings/self_owned_receiver.h"
 #include "mojo/public/cpp/system/message_pipe.h"
 #include "net/base/isolation_info.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "services/network/public/mojom/fetch_api.mojom.h"
 #include "third_party/blink/public/common/features.h"
 #include "third_party/blink/public/common/tokens/tokens.h"
@@ -221,8 +220,6 @@ void DedicatedWorkerHost::StartScriptLoad(
       appcache_host ? appcache_host->GetWeakPtr() : nullptr,
       std::move(blob_url_loader_factory), nullptr, storage_partition_impl,
       partition_domain,
-      // TODO(crbug.com/1138622): Propagate dedicated worker ukm::SourceId here.
-      ukm::kInvalidSourceId,
       // TODO(crbug.com/1143102): pass DevToolsAgentHostImpl for the worker.
       nullptr, base::UnguessableToken(),
       base::BindOnce(&DedicatedWorkerHost::DidStartScriptLoad,
@@ -341,8 +338,6 @@ DedicatedWorkerHost::CreateNetworkFactoryForSubresources(
       /*frame=*/nullptr, worker_process_host_->GetID(),
       ContentBrowserClient::URLLoaderFactoryType::kWorkerSubResource,
       worker_origin_, /*navigation_id=*/base::nullopt,
-      ukm::SourceIdObj::FromInt64(
-          ancestor_render_frame_host->GetPageUkmSourceId()),
       &default_factory_receiver, &factory_params->header_client,
       bypass_redirect_checks,
       /*disable_secure_dns=*/nullptr, &factory_params->factory_override);
