diff --git a/content/browser/worker_host/worker_script_loader_factory.cc b/content/browser/worker_host/worker_script_loader_factory.cc
index 1e18547ec1f8..912ac7725dd0 100644
--- a/content/browser/worker_host/worker_script_loader_factory.cc
+++ b/content/browser/worker_host/worker_script_loader_factory.cc
@@ -27,14 +27,12 @@ WorkerScriptLoaderFactory::WorkerScriptLoaderFactory(
     ServiceWorkerMainResourceHandle* service_worker_handle,
     base::WeakPtr<AppCacheHost> appcache_host,
     const BrowserContextGetter& browser_context_getter,
-    scoped_refptr<network::SharedURLLoaderFactory> loader_factory,
-    ukm::SourceId worker_source_id)
+    scoped_refptr<network::SharedURLLoaderFactory> loader_factory)
     : process_id_(process_id),
       worker_token_(worker_token),
       appcache_host_(std::move(appcache_host)),
       browser_context_getter_(browser_context_getter),
-      loader_factory_(std::move(loader_factory)),
-      worker_source_id_(worker_source_id) {
+      loader_factory_(std::move(loader_factory)) {
   DCHECK_CURRENTLY_ON(BrowserThread::UI);
 
   if (service_worker_handle) {
@@ -67,7 +65,7 @@ void WorkerScriptLoaderFactory::CreateLoaderAndStart(
       process_id_, worker_token_, routing_id, request_id, options,
       resource_request, std::move(client), service_worker_handle_,
       appcache_host_, browser_context_getter_, loader_factory_,
-      traffic_annotation, worker_source_id_);
+      traffic_annotation);
   script_loader_ = script_loader->GetWeakPtr();
   mojo::MakeSelfOwnedReceiver(std::move(script_loader), std::move(receiver));
 }
