diff --git a/content/public/browser/content_browser_client.cc b/content/public/browser/content_browser_client.cc
index c84ae68f2f27..8f0e65956911 100644
--- a/content/public/browser/content_browser_client.cc
+++ b/content/public/browser/content_browser_client.cc
@@ -54,7 +54,6 @@
 #include "third_party/blink/public/common/loader/url_loader_throttle.h"
 #include "third_party/blink/public/common/renderer_preferences/renderer_preferences.h"
 #include "third_party/blink/public/common/user_agent/user_agent_metadata.h"
-#include "third_party/blink/public/mojom/federated_learning/floc.mojom.h"
 #include "ui/gfx/image/image_skia.h"
 #include "ui/shell_dialogs/select_file_policy.h"
 #include "url/gurl.h"
@@ -1053,13 +1052,6 @@ void ContentBrowserClient::AugmentNavigationDownloadPolicy(
     bool user_gesture,
     NavigationDownloadPolicy* download_policy) {}
 
-blink::mojom::InterestCohortPtr ContentBrowserClient::GetInterestCohortForJsApi(
-    WebContents* web_contents,
-    const GURL& url,
-    const base::Optional<url::Origin>& top_frame_origin) {
-  return blink::mojom::InterestCohort::New();
-}
-
 bool ContentBrowserClient::IsBluetoothScanningBlocked(
     content::BrowserContext* browser_context,
     const url::Origin& requesting_origin,
@@ -1144,10 +1136,6 @@ bool ContentBrowserClient::ShouldAllowInsecurePrivateNetworkRequests(
   return false;
 }
 
-ukm::UkmService* ContentBrowserClient::GetUkmService() {
-  return nullptr;
-}
-
 void ContentBrowserClient::OnKeepaliveRequestStarted() {}
 
 void ContentBrowserClient::OnKeepaliveRequestFinished() {}
