diff --git a/content/browser/browser_main_loop.cc b/content/browser/browser_main_loop.cc
index 339b5f24416a..aad5c8231c40 100644
--- a/content/browser/browser_main_loop.cc
+++ b/content/browser/browser_main_loop.cc
@@ -1134,12 +1134,6 @@ void BrowserMainLoop::ShutdownThreadsAndCleanUp() {
     }
   }
 
-  // Must happen after the I/O thread is shutdown since this class lives on the
-  // I/O thread and isn't threadsafe.
-  {
-    TRACE_EVENT0("shutdown", "BrowserMainLoop::Subsystem:GamepadService");
-    device::GamepadService::GetInstance()->Terminate();
-  }
   {
     TRACE_EVENT0("shutdown", "BrowserMainLoop::Subsystem:DeleteDataSources");
     URLDataManager::DeleteDataSources();
@@ -1280,16 +1274,6 @@ int BrowserMainLoop::BrowserThreadsStarted() {
     midi_service_.reset(new midi::MidiService);
   }
 
-  {
-    TRACE_EVENT0("startup", "BrowserThreadsStarted::Subsystem:Devices");
-    device::GamepadService::GetInstance()->StartUp(
-        base::BindRepeating(&BindHidManager));
-#if !defined(OS_ANDROID)
-    device::FidoHidDiscovery::SetHidManagerBinder(
-        base::BindRepeating(&BindHidManager));
-#endif
-  }
-
 #if defined(OS_WIN)
   system_message_window_.reset(new media::SystemMessageWindowWin);
 #elif (defined(OS_LINUX) || defined(OS_CHROMEOS)) && defined(USE_UDEV)
