diff --git a/content/browser/background_fetch/background_fetch_registration_service_impl.cc b/content/browser/background_fetch/background_fetch_registration_service_impl.cc
index d33a6b4c449c..1069cfc049f8 100644
--- a/content/browser/background_fetch/background_fetch_registration_service_impl.cc
+++ b/content/browser/background_fetch/background_fetch_registration_service_impl.cc
@@ -13,7 +13,6 @@
 #include "base/optional.h"
 #include "base/task/post_task.h"
 #include "content/browser/background_fetch/background_fetch_context.h"
-#include "content/browser/background_fetch/background_fetch_metrics.h"
 #include "content/browser/background_fetch/background_fetch_registration_id.h"
 #include "content/browser/background_fetch/background_fetch_registration_notifier.h"
 #include "content/browser/background_fetch/background_fetch_request_match_params.h"
