diff --git a/content/browser/notifications/platform_notification_context_impl.cc b/content/browser/notifications/platform_notification_context_impl.cc
index 15ba957ffdd3..2d7dd78af0af 100644
--- a/content/browser/notifications/platform_notification_context_impl.cc
+++ b/content/browser/notifications/platform_notification_context_impl.cc
@@ -147,10 +147,6 @@ void PlatformNotificationContextImpl::Initialize() {
     return;
   }
 
-  ukm_callback_ = base::BindRepeating(
-      &PlatformNotificationServiceProxy::RecordNotificationUkmEvent,
-      service_proxy_->AsWeakPtr());
-
   service->GetDisplayedNotifications(
       base::BindOnce(&PlatformNotificationContextImpl::DidGetNotifications,
                      this));
@@ -1132,7 +1128,7 @@ void PlatformNotificationContextImpl::OpenDatabase(
   }
 
   database_ =
-      std::make_unique<NotificationDatabase>(GetDatabasePath(), ukm_callback_);
+      std::make_unique<NotificationDatabase>(GetDatabasePath());
   NotificationDatabase::Status status =
       database_->Open(/* create_if_missing= */ true);
 
@@ -1143,8 +1139,7 @@ void PlatformNotificationContextImpl::OpenDatabase(
   // away the contents of the directory and try re-opening the database.
   if (status == NotificationDatabase::STATUS_ERROR_CORRUPTED) {
     if (DestroyDatabase()) {
-      database_ = std::make_unique<NotificationDatabase>(GetDatabasePath(),
-                                                         ukm_callback_);
+      database_ = std::make_unique<NotificationDatabase>(GetDatabasePath());
       status = database_->Open(/* create_if_missing= */ true);
 
       UMA_HISTOGRAM_ENUMERATION(
