diff --git a/content/browser/service_worker/service_worker_main_resource_handle.cc b/content/browser/service_worker/service_worker_main_resource_handle.cc
index c826e42c4205..02f1d8373632 100644
--- a/content/browser/service_worker/service_worker_main_resource_handle.cc
+++ b/content/browser/service_worker/service_worker_main_resource_handle.cc
@@ -40,8 +40,6 @@ void ServiceWorkerMainResourceHandle::OnBeginNavigationCommit(
     int render_process_id,
     int render_frame_id,
     const network::CrossOriginEmbedderPolicy& cross_origin_embedder_policy,
-    mojo::PendingRemote<network::mojom::CrossOriginEmbedderPolicyReporter>
-        coep_reporter,
     blink::mojom::ServiceWorkerContainerInfoForClientPtr* out_container_info,
     ukm::SourceId document_ukm_source_id) {
   DCHECK_CURRENTLY_ON(BrowserThread::UI);
@@ -53,7 +51,7 @@ void ServiceWorkerMainResourceHandle::OnBeginNavigationCommit(
   if (container_host_) {
     container_host_->OnBeginNavigationCommit(
         render_process_id, render_frame_id, cross_origin_embedder_policy,
-        std::move(coep_reporter), document_ukm_source_id);
+        document_ukm_source_id);
   }
 }
 
