diff --git a/content/test/fake_compositor_dependencies.cc b/content/test/fake_compositor_dependencies.cc
index b9e219ef517f..a432748188af 100644
--- a/content/test/fake_compositor_dependencies.cc
+++ b/content/test/fake_compositor_dependencies.cc
@@ -9,7 +9,6 @@
 #include "base/single_thread_task_runner.h"
 #include "base/threading/thread_task_runner_handle.h"
 #include "cc/test/fake_layer_tree_frame_sink.h"
-#include "cc/test/test_ukm_recorder_factory.h"
 #include "cc/trees/render_frame_metadata_observer.h"
 #include "third_party/khronos/GLES2/gl2.h"
 #include "ui/gfx/buffer_types.h"
@@ -57,9 +56,4 @@ bool FakeCompositorDependencies::IsScrollAnimatorEnabled() {
   return false;
 }
 
-std::unique_ptr<cc::UkmRecorderFactory>
-FakeCompositorDependencies::CreateUkmRecorderFactory() {
-  return std::make_unique<cc::TestUkmRecorderFactory>();
-}
-
 }  // namespace content
