diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index e3dd6ac510518..ac8ac09bf9ba0 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -15,6 +15,7 @@ import("//components/viz/common/debugger/viz_debugger.gni")
 import("//content/common/features.gni")
 import("//content/public/common/zygote/features.gni")
 import("//device/vr/buildflags/buildflags.gni")
+import("//dumb/content/browser/sources.gni")
 import("//gpu/vulkan/features.gni")
 import("//media/media_options.gni")
 import("//net/features.gni")
@@ -79,7 +80,6 @@ source_set("browser") {
     "//components/payments/mojom",
     "//components/power_scheduler",
     "//components/services/filesystem:lib",
-    "//components/services/quarantine:quarantine",
     "//components/services/storage",
     "//components/services/storage:filesystem_proxy_factory",
     "//components/services/storage/dom_storage:local_storage_proto",
@@ -88,8 +88,6 @@ source_set("browser") {
     "//components/system_media_controls",
     "//components/tracing:startup_tracing",
     "//components/ui_devtools",
-    "//components/ukm",
-    "//components/ukm/debug:util",
     "//components/url_formatter",
     "//components/variations",
     "//components/variations/net",
@@ -257,6 +255,8 @@ source_set("browser") {
     "//v8:v8_version",
   ]
 
+  deps += dumb_content_browser_deps
+
   if (is_chromeos_ash) {
     deps += [
       "//chromeos/services/machine_learning/public/cpp",
@@ -848,8 +848,6 @@ source_set("browser") {
     "eye_dropper_chooser_impl.h",
     "feature_observer.cc",
     "feature_observer.h",
-    "federated_learning/floc_service_impl.cc",
-    "federated_learning/floc_service_impl.h",
     "field_trial_recorder.cc",
     "field_trial_recorder.h",
     "field_trial_synchronizer.cc",
@@ -1678,8 +1676,6 @@ source_set("browser") {
     "service_worker/service_worker_fetch_dispatcher.h",
     "service_worker/service_worker_host.cc",
     "service_worker/service_worker_host.h",
-    "service_worker/service_worker_identifiability_metrics.cc",
-    "service_worker/service_worker_identifiability_metrics.h",
     "service_worker/service_worker_info.cc",
     "service_worker/service_worker_info.h",
     "service_worker/service_worker_installed_script_loader.cc",
@@ -1819,8 +1815,6 @@ source_set("browser") {
     "tracing/tracing_controller_impl_data_endpoint.cc",
     "tracing/tracing_service_controller.cc",
     "tracing/tracing_service_controller.h",
-    "ukm_internals_ui.cc",
-    "ukm_internals_ui.h",
     "url_loader_factory_getter.cc",
     "url_loader_factory_getter.h",
     "url_loader_factory_params_helper.cc",
@@ -2234,7 +2228,6 @@ source_set("browser") {
     public_deps += [ "//components/services/font/public/mojom" ]
 
     deps += [
-      "//components/crash/content/browser/error_reporting",
       "//components/services/font:lib",
       "//third_party/blink/public/mojom:memory_usage_monitor_linux_mojo_bindings",
     ]
@@ -2917,7 +2910,6 @@ source_set("browser") {
     ]
   }
 
-  if (enable_reporting) {
     sources += [
       "net/cross_origin_embedder_policy_reporter.cc",
       "net/cross_origin_embedder_policy_reporter.h",
@@ -2926,7 +2918,6 @@ source_set("browser") {
       "net/reporting_service_proxy.cc",
       "net/reporting_service_proxy.h",
     ]
-  }
 
   if (use_viz_devtools) {
     sources += [
