diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index 53e97db35b55..754e65bc2e52 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -473,7 +473,6 @@ source_set("browser_sources") {
     "//net",
     "//ppapi/c",
     "//services/device/public/cpp/geolocation",
-    "//services/metrics/public/cpp:metrics_cpp",
     "//services/service_manager/embedder:embedder_result_codes",
     "//ui/accessibility",
     "//ui/base",
