diff --git a/content/browser/renderer_host/navigation_controller_impl.h b/content/browser/renderer_host/navigation_controller_impl.h
index cbd419080c7a..146fdded6d0a 100644
--- a/content/browser/renderer_host/navigation_controller_impl.h
+++ b/content/browser/renderer_host/navigation_controller_impl.h
@@ -583,8 +583,7 @@ class CONTENT_EXPORT NavigationControllerImpl : public NavigationController {
   void SetShouldSkipOnBackForwardUIIfNeeded(
       bool replace_entry,
       bool previous_document_was_activated,
-      bool is_renderer_initiated,
-      ukm::SourceId previous_page_load_ukm_source_id);
+      bool is_renderer_initiated);
 
   // This function sets all same document entries with the same value
   // of skippable flag. This is to avoid back button abuse by inserting
