diff --git a/content/public/renderer/render_frame_observer.h b/content/public/renderer/render_frame_observer.h
index 8ab07f180b0e..4d1bb2498e97 100644
--- a/content/public/renderer/render_frame_observer.h
+++ b/content/public/renderer/render_frame_observer.h
@@ -275,8 +275,7 @@ class CONTENT_EXPORT RenderFrameObserver : public IPC::Listener,
       mojo::ScopedInterfaceEndpointHandle* handle);
 
   // Submit throughput data to the browser process. Only called for local roots.
-  virtual void OnThroughputDataAvailable(ukm::SourceId source_id,
-                                         int aggregated_percent,
+  virtual void OnThroughputDataAvailable(int aggregated_percent,
                                          int impl_percent,
                                          base::Optional<int> main_percent) {}
 
