diff --git a/device/vr/BUILD.gn b/device/vr/BUILD.gn
index dc301049f023..b31f835371b8 100644
--- a/device/vr/BUILD.gn
+++ b/device/vr/BUILD.gn
@@ -88,14 +88,12 @@ if (enable_vr) {
     deps = [
       ":vr_base",
       "//base",
-      "//components/ukm:ukm",
       "//device/base",
       "//device/vr/buildflags",
       "//device/vr/public/mojom:isolated_xr_service",
       "//gpu/ipc/common:interfaces",
       "//mojo/public/cpp/bindings",
       "//services/device/public/cpp/generic_sensor",
-      "//services/metrics/public/cpp:ukm_builders",
       "//ui/display",
       "//ui/gfx",
       "//ui/gfx/mojom",
