diff --git a/dumb/build/win/BUILD.gn b/dumb/build/win/BUILD.gn
new file mode 100644
index 000000000000..0d417ec6b834
--- /dev/null
+++ b/dumb/build/win/BUILD.gn
@@ -0,0 +1,35 @@
+
+group("copy_exe_and_pdb") {
+  public_deps = [
+    ":copy_exe",
+    ":copy_pdb",
+  ]
+}
+
+copy("copy_exe") {
+  sources = [
+    "$root_out_dir/chrome.exe",
+  ]
+
+  outputs = [
+    "$root_out_dir/dumb.exe",
+  ]
+
+  deps = [
+    "//chrome:reorder_imports"
+  ]
+}
+
+copy("copy_pdb") {
+  sources = [
+    "$root_out_dir/chrome.exe.pdb",
+  ]
+
+  outputs = [
+    "$root_out_dir/dumb.exe.pdb",
+  ]
+
+  deps = [
+    "//chrome:reorder_imports"
+  ]
+}
