diff --git a/extensions/shell/BUILD.gn b/extensions/shell/BUILD.gn
index 2441d5d26864..968c6ec6c211 100644
--- a/extensions/shell/BUILD.gn
+++ b/extensions/shell/BUILD.gn
@@ -39,7 +39,6 @@ source_set("app_shell_lib") {
     "//apps",
     "//base",
     "//build:chromeos_buildflags",
-    "//components/feedback",
     "//components/guest_view/browser",
     "//components/guest_view/renderer",
     "//components/keep_alive_registry",
@@ -213,8 +212,6 @@ source_set("app_shell_lib") {
     ]
     deps += [
       "//build:branding_buildflags",
-      "//components/crash/core/app",
-      "//components/crash/core/common",
       "//components/upload_list",
       "//components/version_info:generate_version_info",
     ]
@@ -309,7 +306,6 @@ test("app_shell_unittests") {
     "//base/test:test_support",
     "//build:chromeos_buildflags",
     "//components/crx_file",
-    "//components/feedback",
     "//components/keep_alive_registry",
     "//components/prefs",
     "//components/user_prefs",
