diff --git a/extensions/browser/extension_protocols.cc b/extensions/browser/extension_protocols.cc
index fe04db6a1901..d72ad28a00ec 100644
--- a/extensions/browser/extension_protocols.cc
+++ b/extensions/browser/extension_protocols.cc
@@ -592,20 +592,6 @@ class ExtensionURLLoaderFactory
                               &follow_symlinks_anywhere);
     }
 
-    // If the extension is the Media Router Component Extension used to support
-    // Casting scenarios, log metrics needed to track migration away from this
-    // extension.
-    // TODO(crbug.com/1097594): Remove this metric logging once migration away
-    // from the Media Router Component Extension completes.
-    const MediaRouterExtensionAccessLogger* media_router_access_logger =
-        ExtensionsBrowserClient::Get()->GetMediaRouterAccessLogger();
-    if (media_router_access_logger && request.request_initiator.has_value() &&
-        (extension.get()->id() == extension_misc::kCastExtensionIdRelease ||
-         extension.get()->id() == extension_misc::kCastExtensionIdDev)) {
-      media_router_access_logger->LogMediaRouterComponentExtensionUse(
-          request.request_initiator.value(), browser_context_);
-    }
-
     if (IsBackgroundPageURL(request.url)) {
       // Handle background page requests immediately with a simple generated
       // chunk of HTML.
