diff --git a/extensions/browser/api/web_request/BUILD.gn b/extensions/browser/api/web_request/BUILD.gn
index 7027a03d1197..76bd273dcde0 100644
--- a/extensions/browser/api/web_request/BUILD.gn
+++ b/extensions/browser/api/web_request/BUILD.gn
@@ -44,7 +44,6 @@ source_set("web_request") {
 
   deps = [
     "//build:chromeos_buildflags",
-    "//components/ukm/content",
     "//components/web_cache/browser",
     "//content/public/browser",
     "//content/public/common",
