diff --git a/extensions/browser/api/guest_view/web_view/web_view_internal_api.cc b/extensions/browser/api/guest_view/web_view/web_view_internal_api.cc
index 75eb3aa3d78c..a63977103af6 100644
--- a/extensions/browser/api/guest_view/web_view/web_view_internal_api.cc
+++ b/extensions/browser/api/guest_view/web_view/web_view_internal_api.cc
@@ -15,7 +15,6 @@
 #include "base/strings/string_number_conversions.h"
 #include "base/strings/stringprintf.h"
 #include "base/strings/utf_string_conversions.h"
-#include "components/crash/core/common/crash_key.h"
 #include "content/public/browser/browser_context.h"
 #include "content/public/browser/render_frame_host.h"
 #include "content/public/browser/render_view_host.h"
@@ -278,9 +277,6 @@ bool WebViewInternalExtensionFunction::PreRunValidation(std::string* error) {
 
   int instance_id = 0;
   EXTENSION_FUNCTION_PRERUN_VALIDATE(args_->GetInteger(0, &instance_id));
-  // TODO(780728): Remove crash key once the cause of the kill is known.
-  static crash_reporter::CrashKeyString<128> name_key("webview-function");
-  crash_reporter::ScopedCrashKeyString name_key_scope(&name_key, name());
   guest_ = WebViewGuest::From(source_process_id(), instance_id);
   if (!guest_) {
     *error = "Could not find guest";
