diff --git a/extensions/browser/extensions_browser_client.cc b/extensions/browser/extensions_browser_client.cc
index cd58b8873b33..dbce5d8ea6dc 100644
--- a/extensions/browser/extensions_browser_client.cc
+++ b/extensions/browser/extensions_browser_client.cc
@@ -108,11 +108,6 @@ void ExtensionsBrowserClient::SetLastSaveFilePath(
     content::BrowserContext* context,
     const base::FilePath& path) {}
 
-const MediaRouterExtensionAccessLogger*
-ExtensionsBrowserClient::GetMediaRouterAccessLogger() const {
-  return nullptr;
-}
-
 bool ExtensionsBrowserClient::HasIsolatedStorage(
     const std::string& extension_id,
     content::BrowserContext* context) {
