diff --git a/extensions/browser/extensions_browser_client.cc b/extensions/browser/extensions_browser_client.cc
index a12c9ad106a87..d20326c388526 100644
--- a/extensions/browser/extensions_browser_client.cc
+++ b/extensions/browser/extensions_browser_client.cc
@@ -112,11 +112,6 @@ void ExtensionsBrowserClient::SetLastSaveFilePath(
     content::BrowserContext* context,
     const base::FilePath& path) {}
 
-const MediaRouterExtensionAccessLogger*
-ExtensionsBrowserClient::GetMediaRouterAccessLogger() const {
-  return nullptr;
-}
-
 bool ExtensionsBrowserClient::HasIsolatedStorage(
     const std::string& extension_id,
     content::BrowserContext* context) {
