diff --git a/extensions/browser/api/web_request/web_request_info.h b/extensions/browser/api/web_request/web_request_info.h
index 507e307c5382..42a3574d414b 100644
--- a/extensions/browser/api/web_request/web_request_info.h
+++ b/extensions/browser/api/web_request/web_request_info.h
@@ -23,7 +23,6 @@
 #include "ipc/ipc_message.h"
 #include "net/http/http_request_headers.h"
 #include "net/http/http_response_headers.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "services/network/public/cpp/resource_request.h"
 #include "services/network/public/mojom/fetch_api.mojom-shared.h"
 #include "services/network/public/mojom/url_response_head.mojom-forward.h"
@@ -52,8 +51,7 @@ struct WebRequestInfoInitParams {
       bool is_download,
       bool is_async,
       bool is_service_worker_script,
-      base::Optional<int64_t> navigation_id,
-      ukm::SourceIdObj ukm_source_id);
+      base::Optional<int64_t> navigation_id);
 
   ~WebRequestInfoInitParams();
 
@@ -76,7 +74,6 @@ struct WebRequestInfoInitParams {
   ExtensionApiFrameIdMap::FrameData frame_data;
   bool is_service_worker_script = false;
   base::Optional<int64_t> navigation_id;
-  ukm::SourceIdObj ukm_source_id = ukm::kInvalidSourceIdObj;
   content::GlobalFrameRoutingId parent_routing_id;
 
  private:
@@ -177,9 +174,6 @@ struct WebRequestInfo {
   // Valid if this request corresponds to a navigation.
   const base::Optional<int64_t> navigation_id;
 
-  // UKM source to associate metrics with for this request.
-  const ukm::SourceIdObj ukm_source_id;
-
   // ID of the RenderFrameHost corresponding to the parent frame. Only valid for
   // document subresource and sub-frame requests.
   const content::GlobalFrameRoutingId parent_routing_id;
