diff --git a/extensions/common/extension.cc b/extensions/common/extension.cc
index d271c4e280a2..134ecf4534e2 100644
--- a/extensions/common/extension.cc
+++ b/extensions/common/extension.cc
@@ -380,9 +380,7 @@ bool Extension::ShouldDisplayInNewTabPage() const {
 }
 
 bool Extension::ShouldExposeViaManagementAPI() const {
-  // Hide component extensions because they are only extensions as an
-  // implementation detail of Chrome.
-  return !extensions::Manifest::IsComponentLocation(location());
+  return true;
 }
 
 Extension::ManifestData* Extension::GetManifestData(const std::string& key)
