diff --git a/extensions/browser/api/BUILD.gn b/extensions/browser/api/BUILD.gn
index cd771e5daac0..e5ed78882de5 100644
--- a/extensions/browser/api/BUILD.gn
+++ b/extensions/browser/api/BUILD.gn
@@ -130,7 +130,6 @@ source_set("api") {
     public_deps += [
       "//extensions/browser/api/cec_private",
       "//extensions/browser/api/clipboard",
-      "//extensions/browser/api/crash_report_private",
       "//extensions/browser/api/diagnostics",
       "//extensions/browser/api/networking_config",
       "//extensions/browser/api/system_power_source",
