diff --git a/extensions/browser/BUILD.gn b/extensions/browser/BUILD.gn
index 1933de183e66..454f392b20b0 100644
--- a/extensions/browser/BUILD.gn
+++ b/extensions/browser/BUILD.gn
@@ -539,7 +539,6 @@ source_set("browser_tests") {
       "//chromeos/dbus/upstart",
       "//chromeos/login/login_state",
       "//chromeos/network",
-      "//components/crash/content/browser/error_reporting:mock_crash_endpoint",
       "//extensions/common:mojom",
     ]
   }
@@ -593,7 +592,6 @@ source_set("test_support") {
       "api/feedback_private/mock_feedback_service.h",
     ]
     deps += [
-      "//components/crash/core/app:app",
       "//net:test_support",
     ]
   }
@@ -783,7 +781,6 @@ source_set("unit_tests") {
       "//chromeos/dbus/upstart",
       "//chromeos/login/login_state",
       "//chromeos/network:test_support",
-      "//components/feedback",
       "//extensions/shell:app_shell_lib",
     ]
   }
