diff --git a/extensions/common/BUILD.gn b/extensions/common/BUILD.gn
index 91a607d2ef3e..3abac59a66bc 100644
--- a/extensions/common/BUILD.gn
+++ b/extensions/common/BUILD.gn
@@ -186,8 +186,6 @@ if (enable_extensions) {
       "hashed_extension_id.h",
       "host_id.cc",
       "host_id.h",
-      "identifiability_metrics.cc",
-      "identifiability_metrics.h",
       "image_util.cc",
       "image_util.h",
       "install_warning.cc",
