diff --git a/extensions/shell/BUILD.gn b/extensions/shell/BUILD.gn
index 2441d5d26864..dbf16a792939 100644
--- a/extensions/shell/BUILD.gn
+++ b/extensions/shell/BUILD.gn
@@ -213,8 +213,6 @@ source_set("app_shell_lib") {
     ]
     deps += [
       "//build:branding_buildflags",
-      "//components/crash/core/app",
-      "//components/crash/core/common",
       "//components/upload_list",
       "//components/version_info:generate_version_info",
     ]
