diff --git a/extensions/browser/BUILD.gn b/extensions/browser/BUILD.gn
index b4aae04fbfdf..49eabff71974 100644
--- a/extensions/browser/BUILD.gn
+++ b/extensions/browser/BUILD.gn
@@ -517,7 +517,6 @@ source_set("browser_tests") {
     sources += [
       "api/audio/audio_apitest_chromeos.cc",
       "api/cec_private/cec_private_apitest.cc",
-      "api/crash_report_private/crash_report_private_apitest.cc",
       "api/media_perception_private/media_perception_private_apitest.cc",
       "api/system_power_source/system_power_source_apitest.cc",
       "api/virtual_keyboard/virtual_keyboard_apitest.cc",
@@ -535,7 +534,6 @@ source_set("browser_tests") {
       "//chromeos/dbus/upstart",
       "//chromeos/login/login_state",
       "//chromeos/network",
-      "//components/crash/content/browser/error_reporting:mock_crash_endpoint",
     ]
   }
 }
@@ -585,7 +583,6 @@ source_set("test_support") {
       "api/feedback_private/mock_feedback_service.h",
     ]
     deps += [
-      "//components/crash/core/app:app",
       "//net:test_support",
     ]
   }
