diff --git a/extensions/common/BUILD.gn b/extensions/common/BUILD.gn
index 75f49369144e7..a42a54a9e3480 100644
--- a/extensions/common/BUILD.gn
+++ b/extensions/common/BUILD.gn
@@ -218,8 +218,6 @@ if (enable_extensions) {
       "guest_view/extensions_guest_view_messages.h",
       "hashed_extension_id.cc",
       "hashed_extension_id.h",
-      "identifiability_metrics.cc",
-      "identifiability_metrics.h",
       "image_util.cc",
       "image_util.h",
       "install_warning.cc",
