diff --git a/extensions/browser/BUILD.gn b/extensions/browser/BUILD.gn
index 488170149c68..5707221cc727 100644
--- a/extensions/browser/BUILD.gn
+++ b/extensions/browser/BUILD.gn
@@ -412,7 +412,6 @@ source_set("browser_sources") {
     "//google_apis",
     "//ppapi/buildflags",
     "//services/device/public/mojom",
-    "//services/metrics/public/cpp:metrics_cpp",
     "//services/preferences/public/cpp",
     "//services/service_manager/public/cpp",
     "//skia",
@@ -543,7 +542,6 @@ source_set("browser_tests") {
       "//chromeos/dbus/upstart",
       "//chromeos/login/login_state",
       "//chromeos/network",
-      "//components/crash/content/browser/error_reporting:mock_crash_endpoint",
     ]
   }
 }
@@ -595,7 +593,6 @@ source_set("test_support") {
       "api/feedback_private/mock_feedback_service.h",
     ]
     deps += [
-      "//components/crash/core/app:app",
       "//net:test_support",
     ]
   }
