diff --git a/extensions/browser/extension_protocols.cc b/extensions/browser/extension_protocols.cc
index b25f70ff1ba55..de44d8108cdf9 100644
--- a/extensions/browser/extension_protocols.cc
+++ b/extensions/browser/extension_protocols.cc
@@ -70,7 +70,6 @@
 #include "extensions/common/extension.h"
 #include "extensions/common/extension_resource.h"
 #include "extensions/common/file_util.h"
-#include "extensions/common/identifiability_metrics.h"
 #include "extensions/common/manifest_handlers/background_info.h"
 #include "extensions/common/manifest_handlers/csp_info.h"
 #include "extensions/common/manifest_handlers/icons_handler.h"
@@ -161,10 +160,6 @@ class ResultRecordingClient : public network::mojom::URLLoaderClient {
   }
 
   void OnComplete(const network::URLLoaderCompletionStatus& status) override {
-    RecordExtensionResourceAccessResult(
-        ukm_source_id_, url_,
-        status.error_code == net::OK ? ExtensionResourceAccessResult::kSuccess
-                                     : ExtensionResourceAccessResult::kFailure);
     real_client_->OnComplete(status);
   }
 
@@ -593,20 +588,6 @@ class ExtensionURLLoaderFactory : public network::SelfDeletingURLLoaderFactory {
                               &follow_symlinks_anywhere);
     }
 
-    // If the extension is the Media Router Component Extension used to support
-    // Casting scenarios, log metrics needed to track migration away from this
-    // extension.
-    // TODO(crbug.com/1097594): Remove this metric logging once migration away
-    // from the Media Router Component Extension completes.
-    const MediaRouterExtensionAccessLogger* media_router_access_logger =
-        ExtensionsBrowserClient::Get()->GetMediaRouterAccessLogger();
-    if (media_router_access_logger && request.request_initiator.has_value() &&
-        (extension.get()->id() == extension_misc::kCastExtensionIdRelease ||
-         extension.get()->id() == extension_misc::kCastExtensionIdDev)) {
-      media_router_access_logger->LogMediaRouterComponentExtensionUse(
-          request.request_initiator.value(), browser_context_);
-    }
-
     if (IsBackgroundPageURL(request.url)) {
       // Handle background page requests immediately with a simple generated
       // chunk of HTML.
