diff --git a/extensions/browser/api/web_request/BUILD.gn b/extensions/browser/api/web_request/BUILD.gn
index 4d5b49eb1bed3..3191fd2a1f067 100644
--- a/extensions/browser/api/web_request/BUILD.gn
+++ b/extensions/browser/api/web_request/BUILD.gn
@@ -72,7 +72,6 @@ source_set("web_request") {
   deps = [
     "//build:chromeos_buildflags",
     "//components/keyed_service/content:content",
-    "//components/ukm/content",
     "//components/web_cache/browser",
     "//content/public/browser",
     "//content/public/common",
