diff --git a/extensions/common/BUILD.gn b/extensions/common/BUILD.gn
index 1960ab9dc10e..673fb9dd1390 100644
--- a/extensions/common/BUILD.gn
+++ b/extensions/common/BUILD.gn
@@ -185,8 +185,6 @@ if (enable_extensions) {
       "hashed_extension_id.h",
       "host_id.cc",
       "host_id.h",
-      "identifiability_metrics.cc",
-      "identifiability_metrics.h",
       "image_util.cc",
       "image_util.h",
       "install_warning.cc",
