diff --git a/extensions/shell/browser/system_logs/BUILD.gn b/extensions/shell/browser/system_logs/BUILD.gn
index 78a193451a2f..8597a6ca25af 100644
--- a/extensions/shell/browser/system_logs/BUILD.gn
+++ b/extensions/shell/browser/system_logs/BUILD.gn
@@ -10,7 +10,6 @@ assert(enable_extensions,
 source_set("system_logs") {
   deps = [
     "//base",
-    "//components/feedback",
     "//components/strings",
     "//components/version_info",
     "//content/public/browser",
