diff --git a/extensions/common/BUILD.gn b/extensions/common/BUILD.gn
index db3657e580f6..c363a84f6a4c 100644
--- a/extensions/common/BUILD.gn
+++ b/extensions/common/BUILD.gn
@@ -212,8 +212,6 @@ if (enable_extensions) {
       "hashed_extension_id.h",
       "host_id.cc",
       "host_id.h",
-      "identifiability_metrics.cc",
-      "identifiability_metrics.h",
       "image_util.cc",
       "image_util.h",
       "install_warning.cc",
