diff --git a/extensions/common/extension.cc b/extensions/common/extension.cc
index 98b7513924ea0..f66f77b4c64b6 100644
--- a/extensions/common/extension.cc
+++ b/extensions/common/extension.cc
@@ -423,9 +423,7 @@ bool Extension::ShouldDisplayInNewTabPage() const {
 }
 
 bool Extension::ShouldExposeViaManagementAPI() const {
-  // Hide component extensions because they are only extensions as an
-  // implementation detail of Chrome.
-  return !extensions::Manifest::IsComponentLocation(location());
+  return true;
 }
 
 Extension::ManifestData* Extension::GetManifestData(const std::string& key)
