diff --git a/extensions/shell/BUILD.gn b/extensions/shell/BUILD.gn
index 3cf8228229c50..32809c843768a 100644
--- a/extensions/shell/BUILD.gn
+++ b/extensions/shell/BUILD.gn
@@ -39,8 +39,6 @@ source_set("app_shell_lib") {
     "//apps",
     "//base",
     "//build:chromeos_buildflags",
-    "//components/feedback",
-    "//components/feedback/content:factory",
     "//components/guest_view/browser",
     "//components/guest_view/renderer",
     "//components/keep_alive_registry",
@@ -313,7 +311,6 @@ test("app_shell_unittests") {
     "//base/test:test_support",
     "//build:chromeos_buildflags",
     "//components/crx_file",
-    "//components/feedback",
     "//components/keep_alive_registry",
     "//components/prefs",
     "//components/user_prefs",
