diff --git a/extensions/renderer/BUILD.gn b/extensions/renderer/BUILD.gn
index 2ef54dde7aa2..4caf38837b11 100644
--- a/extensions/renderer/BUILD.gn
+++ b/extensions/renderer/BUILD.gn
@@ -260,7 +260,6 @@ source_set("renderer") {
     "//extensions/common/api",
     "//gin",
     "//mojo/public/js:resources",
-    "//services/metrics/public/cpp:metrics_cpp",
     "//skia",
     "//storage/common",
     "//third_party/blink/public:blink",
