diff --git a/extensions/browser/api/web_request/web_request_api.h b/extensions/browser/api/web_request/web_request_api.h
index ccc3ff02175d..b09a58fdd21b 100644
--- a/extensions/browser/api/web_request/web_request_api.h
+++ b/extensions/browser/api/web_request/web_request_api.h
@@ -197,7 +197,6 @@ class WebRequestAPI : public BrowserContextKeyedAPI,
       int render_process_id,
       content::ContentBrowserClient::URLLoaderFactoryType type,
       base::Optional<int64_t> navigation_id,
-      base::UkmSourceId ukm_source_id,
       mojo::PendingReceiver<network::mojom::URLLoaderFactory>* factory_receiver,
       mojo::PendingRemote<network::mojom::TrustedURLLoaderHeaderClient>*
           header_client);
