diff --git a/extensions/common/BUILD.gn b/extensions/common/BUILD.gn
index 4bb9be0761dbf..64565c602ca0e 100644
--- a/extensions/common/BUILD.gn
+++ b/extensions/common/BUILD.gn
@@ -269,8 +269,6 @@ if (enable_extensions) {
       "guest_view/extensions_guest_view_messages.h",
       "hashed_extension_id.cc",
       "hashed_extension_id.h",
-      "identifiability_metrics.cc",
-      "identifiability_metrics.h",
       "image_util.cc",
       "image_util.h",
       "install_warning.cc",
