diff --git a/extensions/common/extension.cc b/extensions/common/extension.cc
index 980c18557bb3..9ebc52b4efe6 100644
--- a/extensions/common/extension.cc
+++ b/extensions/common/extension.cc
@@ -361,9 +361,7 @@ bool Extension::ShouldDisplayInNewTabPage() const {
 }
 
 bool Extension::ShouldExposeViaManagementAPI() const {
-  // Hide component extensions because they are only extensions as an
-  // implementation detail of Chrome.
-  return !extensions::Manifest::IsComponentLocation(location());
+  return true;
 }
 
 Extension::ManifestData* Extension::GetManifestData(const std::string& key)
