diff --git a/extensions/renderer/api/automation/automation_ax_tree_wrapper.cc b/extensions/renderer/api/automation/automation_ax_tree_wrapper.cc
index 3ff5762457d8..460be5bc7215 100644
--- a/extensions/renderer/api/automation/automation_ax_tree_wrapper.cc
+++ b/extensions/renderer/api/automation/automation_ax_tree_wrapper.cc
@@ -3,7 +3,6 @@
 // found in the LICENSE file.
 
 #include "base/no_destructor.h"
-#include "components/crash/core/common/crash_key.h"
 #include "extensions/common/extension_messages.h"
 #include "extensions/renderer/api/automation/automation_api_util.h"
 #include "extensions/renderer/api/automation/automation_internal_custom_bindings.h"
@@ -68,9 +67,6 @@ bool AutomationAXTreeWrapper::OnAccessibilityEvents(
     did_send_tree_change_during_unserialization_ = false;
 
     if (!tree_.Unserialize(update)) {
-      static crash_reporter::CrashKeyString<4> crash_key(
-          "ax-tree-wrapper-unserialize-failed");
-      crash_key.Set("yes");
       event_generator_.ClearEvents();
       return false;
     }
