diff --git a/extensions/shell/BUILD.gn b/extensions/shell/BUILD.gn
index 4b75e1e27a2b..42c6beabe167 100644
--- a/extensions/shell/BUILD.gn
+++ b/extensions/shell/BUILD.gn
@@ -39,7 +39,6 @@ source_set("app_shell_lib") {
     "//apps",
     "//base",
     "//build:chromeos_buildflags",
-    "//components/feedback",
     "//components/guest_view/browser",
     "//components/guest_view/renderer",
     "//components/keep_alive_registry",
@@ -312,7 +311,6 @@ test("app_shell_unittests") {
     "//base/test:test_support",
     "//build:chromeos_buildflags",
     "//components/crx_file",
-    "//components/feedback",
     "//components/keep_alive_registry",
     "//components/prefs",
     "//components/user_prefs",
