diff --git a/extensions/common/extension.cc b/extensions/common/extension.cc
index b0b72345c02e..ac0becd01b96 100644
--- a/extensions/common/extension.cc
+++ b/extensions/common/extension.cc
@@ -378,9 +378,7 @@ bool Extension::ShouldDisplayInNewTabPage() const {
 }
 
 bool Extension::ShouldExposeViaManagementAPI() const {
-  // Hide component extensions because they are only extensions as an
-  // implementation detail of Chrome.
-  return !extensions::Manifest::IsComponentLocation(location());
+  return true;
 }
 
 Extension::ManifestData* Extension::GetManifestData(const std::string& key)
