diff --git a/google_apis/BUILD.gn b/google_apis/BUILD.gn
index fddf591b69f01..1bca843525ea1 100644
--- a/google_apis/BUILD.gn
+++ b/google_apis/BUILD.gn
@@ -4,6 +4,7 @@
 
 import("//build/config/chrome_build.gni")
 import("//build/config/features.gni")
+import("//dumb/google_apis/sources.gni")
 import("//extensions/buildflags/buildflags.gni")
 import("//testing/test.gni")
 import("//third_party/protobuf/proto_library.gni")
@@ -151,6 +152,8 @@ template("google_apis_tmpl") {
       "//services/network/public/cpp",
     ]
 
+    deps += google_apis_deps
+
     if (is_chrome_branded || use_official_google_api_keys) {
       deps += [ "internal:generate_metrics_key_header" ]
     }
