diff --git a/gpu/ipc/BUILD.gn b/gpu/ipc/BUILD.gn
index c418f423f54c..718c8d232db9 100644
--- a/gpu/ipc/BUILD.gn
+++ b/gpu/ipc/BUILD.gn
@@ -36,7 +36,6 @@ component("gl_in_process_context") {
     # Mac even though it's not directly referenced, because it's being
     # implicitly depended upon by gpu/config/gpu_crash_keys.h but deps (even
     # public ones) are not transitive for static libraries.
-    "//components/crash/core/common:crash_key",
     "//gpu/command_buffer/client",
     "//gpu/command_buffer/client:gles2_cmd_helper",
     "//gpu/command_buffer/client:gles2_implementation",
