diff --git a/gpu/vulkan/BUILD.gn b/gpu/vulkan/BUILD.gn
index 28ee6232cad0..455e33a06aa7 100644
--- a/gpu/vulkan/BUILD.gn
+++ b/gpu/vulkan/BUILD.gn
@@ -90,8 +90,6 @@ if (enable_vulkan) {
       "vulkan_command_buffer.h",
       "vulkan_command_pool.cc",
       "vulkan_command_pool.h",
-      "vulkan_crash_keys.cc",
-      "vulkan_crash_keys.h",
       "vulkan_cxx.h",
       "vulkan_device_queue.cc",
       "vulkan_device_queue.h",
@@ -117,7 +115,6 @@ if (enable_vulkan) {
       ":buildflags",
       "//base",
       "//base/util/type_safety",
-      "//components/crash/core/common:crash_key",
       "//gpu/ipc/common:vulkan_ycbcr_info",
       "//ui/gfx",
     ]
