diff --git a/gpu/vulkan/vulkan_device_queue.cc b/gpu/vulkan/vulkan_device_queue.cc
index e30bbd11632f..d44e508516d2 100644
--- a/gpu/vulkan/vulkan_device_queue.cc
+++ b/gpu/vulkan/vulkan_device_queue.cc
@@ -14,7 +14,6 @@
 #include "gpu/config/gpu_info.h"  // nogncheck
 #include "gpu/config/vulkan_info.h"
 #include "gpu/vulkan/vulkan_command_pool.h"
-#include "gpu/vulkan/vulkan_crash_keys.h"
 #include "gpu/vulkan/vulkan_fence_helper.h"
 #include "gpu/vulkan/vulkan_function_pointers.h"
 #include "gpu/vulkan/vulkan_util.h"
@@ -176,22 +175,12 @@ bool VulkanDeviceQueue::Initialize(
     }
   }
 
-  crash_keys::vulkan_device_api_version.Set(
-      VkVersionToString(vk_physical_device_properties_.apiVersion));
-  crash_keys::vulkan_device_driver_version.Set(base::StringPrintf(
-      "0x%08x", vk_physical_device_properties_.driverVersion));
-  crash_keys::vulkan_device_vendor_id.Set(
-      base::StringPrintf("0x%04x", vk_physical_device_properties_.vendorID));
-  crash_keys::vulkan_device_id.Set(
-      base::StringPrintf("0x%04x", vk_physical_device_properties_.deviceID));
   static const char* kDeviceTypeNames[] = {
       "other", "integrated", "discrete", "virtual", "cpu",
   };
   uint32_t gpu_type = vk_physical_device_properties_.deviceType;
   if (gpu_type >= base::size(kDeviceTypeNames))
     gpu_type = 0;
-  crash_keys::vulkan_device_type.Set(kDeviceTypeNames[gpu_type]);
-  crash_keys::vulkan_device_name.Set(vk_physical_device_properties_.deviceName);
 
   // Disable all physical device features by default.
   enabled_device_features_2_ = {VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_FEATURES_2};
