diff --git a/gpu/config/gpu_util.cc b/gpu/config/gpu_util.cc
index 674cd223a9d3..ccd6a3f5647f 100644
--- a/gpu/config/gpu_util.cc
+++ b/gpu/config/gpu_util.cc
@@ -29,7 +29,6 @@
 #include "build/build_config.h"
 #include "gpu/config/device_perf_info.h"
 #include "gpu/config/gpu_blocklist.h"
-#include "gpu/config/gpu_crash_keys.h"
 #include "gpu/config/gpu_driver_bug_list.h"
 #include "gpu/config/gpu_driver_bug_workaround_type.h"
 #include "gpu/config/gpu_feature_type.h"
@@ -624,32 +623,6 @@ GpuFeatureInfo ComputeGpuFeatureInfo(const GPUInfo& gpu_info,
   return gpu_feature_info;
 }
 
-void SetKeysForCrashLogging(const GPUInfo& gpu_info) {
-  const GPUInfo::GPUDevice& active_gpu = gpu_info.active_gpu();
-#if !defined(OS_ANDROID)
-  crash_keys::gpu_vendor_id.Set(
-      base::StringPrintf("0x%04x", active_gpu.vendor_id));
-  crash_keys::gpu_device_id.Set(
-      base::StringPrintf("0x%04x", active_gpu.device_id));
-#endif  // !OS_ANDROID
-#if defined(OS_WIN)
-  crash_keys::gpu_sub_sys_id.Set(
-      base::StringPrintf("0x%08x", active_gpu.sub_sys_id));
-  crash_keys::gpu_revision.Set(base::StringPrintf("%u", active_gpu.revision));
-#endif  // OS_WIN
-  crash_keys::gpu_driver_version.Set(active_gpu.driver_version);
-  crash_keys::gpu_pixel_shader_version.Set(gpu_info.pixel_shader_version);
-  crash_keys::gpu_vertex_shader_version.Set(gpu_info.vertex_shader_version);
-  crash_keys::gpu_generation_intel.Set(
-      base::StringPrintf("%d", GetIntelGpuGeneration(gpu_info)));
-#if defined(OS_MAC)
-  crash_keys::gpu_gl_version.Set(gpu_info.gl_version);
-#elif defined(OS_POSIX)
-  crash_keys::gpu_vendor.Set(gpu_info.gl_vendor);
-  crash_keys::gpu_renderer.Set(gpu_info.gl_renderer);
-#endif
-}
-
 void CacheGPUInfo(const GPUInfo& gpu_info) {
   DCHECK(!g_gpu_info_cache);
   g_gpu_info_cache = new GPUInfo;
