diff --git a/gpu/ipc/service/context_url.cc b/gpu/ipc/service/context_url.cc
index 4fd16df84124..06ff99de90a4 100644
--- a/gpu/ipc/service/context_url.cc
+++ b/gpu/ipc/service/context_url.cc
@@ -7,7 +7,6 @@
 #include <utility>
 
 #include "base/hash/hash.h"
-#include "components/crash/core/common/crash_key.h"
 
 namespace gpu {
 
@@ -19,12 +18,6 @@ void ContextUrl::SetActiveUrl(const gpu::ContextUrl& active_url) {
     return;
 
   last_url_hash = active_url.hash();
-
-  // Note that the url is intentionally excluded from WebView and WebLayer
-  // crash dumps using an allowlist for privacy reasons. See
-  // kWebViewCrashKeyAllowList and kWebLayerCrashKeyAllowList.
-  static crash_reporter::CrashKeyString<1024> crash_key("gpu-url-chunk");
-  crash_key.Set(active_url.url().possibly_invalid_spec());
 }
 
 ContextUrl::ContextUrl(GURL url)
