diff --git a/gpu/ipc/service/BUILD.gn b/gpu/ipc/service/BUILD.gn
index 6ee0b6469aeb..46a9359a03e3 100644
--- a/gpu/ipc/service/BUILD.gn
+++ b/gpu/ipc/service/BUILD.gn
@@ -78,7 +78,6 @@ component("service") {
     # Mac even though it's not directly referenced, because it's being
     # implicitly depended upon by gpu/config/gpu_crash_keys.h but deps (even
     # public ones) are not transitive for static libraries.
-    "//components/crash/core/common:crash_key",
     "//gpu/command_buffer/common",
     "//gpu/command_buffer/service",
     "//gpu/command_buffer/service:gles2",
@@ -106,6 +105,7 @@ component("service") {
     deps += [ "//ui/accelerated_widget_mac" ]
     lib_dirs = [ "$mac_sdk_path/usr/lib" ]
     frameworks = [
+      "CoreFoundation.framework",
       "CoreGraphics.framework",
       "IOSurface.framework",
       "OpenGL.framework",
