diff --git a/gpu/config/BUILD.gn b/gpu/config/BUILD.gn
index 569657ae262f..1dfc8c1ea09d 100644
--- a/gpu/config/BUILD.gn
+++ b/gpu/config/BUILD.gn
@@ -131,8 +131,6 @@ source_set("config_sources") {
     "gpu_blocklist.h",
     "gpu_control_list.cc",
     "gpu_control_list.h",
-    "gpu_crash_keys.cc",
-    "gpu_crash_keys.h",
     "gpu_domain_guilt.h",
     "gpu_driver_bug_list.cc",
     "gpu_driver_bug_list.h",
@@ -180,7 +178,7 @@ source_set("config_sources") {
 
   configs += [ "//gpu:gpu_implementation" ]
 
-  public_deps = [ "//components/crash/core/common:crash_key" ]
+  public_deps = []
 
   if (enable_vulkan) {
     public_deps += [ ":vulkan_info" ]
