diff --git a/gpu/config/gpu_util.h b/gpu/config/gpu_util.h
index b4d6dfb3087b..238ae7d9736a 100644
--- a/gpu/config/gpu_util.h
+++ b/gpu/config/gpu_util.h
@@ -41,8 +41,6 @@ ComputeGpuFeatureInfo(const GPUInfo& gpu_info,
                       base::CommandLine* command_line,
                       bool* needs_more_info);
 
-GPU_EXPORT void SetKeysForCrashLogging(const GPUInfo& gpu_info);
-
 // Cache GPUInfo so it can be accessed later.
 GPU_EXPORT void CacheGPUInfo(const GPUInfo& gpu_info);
 
