diff --git a/gpu/vulkan/BUILD.gn b/gpu/vulkan/BUILD.gn
index 043ce01bf12c..a9437bc4904b 100644
--- a/gpu/vulkan/BUILD.gn
+++ b/gpu/vulkan/BUILD.gn
@@ -83,8 +83,6 @@ if (enable_vulkan) {
       "vulkan_command_buffer.h",
       "vulkan_command_pool.cc",
       "vulkan_command_pool.h",
-      "vulkan_crash_keys.cc",
-      "vulkan_crash_keys.h",
       "vulkan_cxx.h",
       "vulkan_device_queue.cc",
       "vulkan_device_queue.h",
@@ -110,7 +108,6 @@ if (enable_vulkan) {
       ":buildflags",
       "//base",
       "//base/util/type_safety",
-      "//components/crash/core/common:crash_key",
       "//gpu/ipc/common:vulkan_ycbcr_info",
       "//ui/gfx",
     ]
