diff --git a/gpu/command_buffer/service/service_font_manager.cc b/gpu/command_buffer/service/service_font_manager.cc
index b1f298d31179..1fd628355675 100644
--- a/gpu/command_buffer/service/service_font_manager.cc
+++ b/gpu/command_buffer/service/service_font_manager.cc
@@ -12,7 +12,6 @@
 #include "base/metrics/histogram_macros.h"
 #include "base/rand_util.h"
 #include "base/strings/stringprintf.h"
-#include "components/crash/core/common/crash_key.h"
 #include "gpu/command_buffer/common/buffer.h"
 #include "gpu/command_buffer/common/discardable_handle.h"
 
@@ -126,8 +125,6 @@ class ServiceFontManager::SkiaDiscardableManager
         ", gpc: %" PRIu64,
         data.memorySize, data.bytesRead, data.typefaceSize, data.strikeCount,
         data.glyphImagesCount, data.glyphPathsCount);
-    static crash_reporter::CrashKeyString<128> crash_key("oop_read_failure");
-    crash_reporter::ScopedCrashKeyString auto_clear(&crash_key, str);
 
     ++dump_count_;
     base::debug::DumpWithoutCrashing();
