diff --git a/gpu/ipc/service/gpu_watchdog_thread.cc b/gpu/ipc/service/gpu_watchdog_thread.cc
index fac06bb181cf..4cc6e897b09b 100644
--- a/gpu/ipc/service/gpu_watchdog_thread.cc
+++ b/gpu/ipc/service/gpu_watchdog_thread.cc
@@ -26,7 +26,6 @@
 #include "base/threading/thread_task_runner_handle.h"
 #include "base/time/time.h"
 #include "build/build_config.h"
-#include "gpu/config/gpu_crash_keys.h"
 #include "gpu/config/gpu_finch_features.h"
 #include "gpu/ipc/common/result_codes.h"
 
@@ -680,14 +679,6 @@ void GpuWatchdogThread::DeliberatelyTerminateToRecoverFromHang() {
 
   GpuWatchdogHistogram(GpuWatchdogThreadEvent::kGpuWatchdogKill);
 
-  crash_keys::gpu_watchdog_crashed_in_gpu_init.Set(
-      in_gpu_initialization_ ? "1" : "0");
-
-  crash_keys::gpu_watchdog_kill_after_power_resume.Set(
-      WithinOneMinFromPowerResumed() ? "1" : "0");
-
-  crash_keys::num_of_processors.Set(base::NumberToString(num_of_processors_));
-
   // Check the arm_disarm_counter value one more time.
   auto last_arm_disarm_counter = ReadArmDisarmCounter();
   base::debug::Alias(&last_arm_disarm_counter);
