diff --git a/headless/app/headless_shell.cc b/headless/app/headless_shell.cc
index 4b9e87f79e28..72a35b6fb1d8 100644
--- a/headless/app/headless_shell.cc
+++ b/headless/app/headless_shell.cc
@@ -645,11 +645,6 @@ int HeadlessShellMain(HINSTANCE instance,
   std::string process_type =
       base::CommandLine::ForCurrentProcess()->GetSwitchValueASCII(
           ::switches::kProcessType);
-  if (process_type == crash_reporter::switches::kCrashpadHandler) {
-    return crash_reporter::RunAsCrashpadHandler(
-        *base::CommandLine::ForCurrentProcess(), base::FilePath(),
-        ::switches::kProcessType, switches::kUserDataDir);
-  }
 #endif  // defined(HEADLESS_USE_CRASHPAD)
   RunChildProcessIfNeeded(instance, sandbox_info);
   HeadlessBrowser::Options::Builder builder(0, nullptr);
