diff --git a/headless/BUILD.gn b/headless/BUILD.gn
index 1035a2618457..e9ed5b90382a 100644
--- a/headless/BUILD.gn
+++ b/headless/BUILD.gn
@@ -272,13 +272,6 @@ source_set("headless_shared_sources") {
   sources += generated_devtools_api
   sources += get_target_outputs(":protocol_sources")
 
-  if (!is_fuchsia) {
-    sources += [
-      "lib/headless_crash_reporter_client.cc",
-      "lib/headless_crash_reporter_client.h",
-    ]
-  }
-
   deps = [
     ":gen_devtools_client_api",
     ":protocol_sources",
@@ -291,13 +284,6 @@ source_set("headless_shared_sources") {
     "//url",
   ]
 
-  if (!is_fuchsia) {
-    deps += [ "//components/crash/content/browser" ]
-  }
-  if (is_component_build && is_win) {
-    deps += [ "//components/crash/core/app:crash_export_thunks" ]
-  }
-
   configs += [ ":inside_headless_component" ]
 
   if (headless_use_prefs) {
@@ -450,7 +436,6 @@ component("headless_non_renderer") {
     }
 
     deps += [
-      "//components/crash/core/common:crash_key",
       "//components/security_state/content",
       "//gin",
       "//third_party/blink/public:blink",
@@ -459,13 +444,6 @@ component("headless_non_renderer") {
       "//v8",
     ]
 
-    if (!is_fuchsia) {
-      deps += [ "//components/crash/content/browser" ]
-    }
-    if (is_win) {
-      deps += [ "//components/crash/core/app:crash_export_thunks" ]
-    }
-
     if (enable_basic_printing) {
       deps += [
         "//components/printing/browser",
@@ -491,7 +469,6 @@ component("headless_non_renderer") {
   }
 
   if (is_mac) {
-    deps += [ ":mac_helpers" ]
     frameworks = [
       "AppKit.framework",
       "CoreFoundation.framework",
@@ -568,7 +545,6 @@ if (!is_component_build) {
 
     deps = [
       ":headless",
-      "//components/crash/core/common:crash_key",
       "//third_party/blink/public:blink_headers",
       "//ui/base",
       "//v8",
@@ -631,16 +607,6 @@ test("headless_unittests") {
   }
 }
 
-if (is_mac) {
-  copy("mac_helpers") {
-    sources = [ "$root_out_dir/chrome_crashpad_handler" ]
-
-    deps = [ "//components/crash/core/app:chrome_crashpad_handler" ]
-
-    outputs = [ "$root_out_dir/Helpers/{{source_file_part}}" ]
-  }
-}
-
 test("headless_browsertests") {
   configs += [ "//net:net_test_config" ]
   sources = [
@@ -677,10 +643,6 @@ test("headless_browsertests") {
     "//third_party/tlslite/",
   ]
 
-  if (is_mac) {
-    data_deps = [ ":mac_helpers" ]
-  }
-
   defines = [ "HAS_OUT_OF_PROC_TEST_RUNNER" ]
 
   if (headless_use_prefs) {
@@ -772,10 +734,6 @@ static_library("headless_shell_lib") {
     public_deps += [ ":headless_non_renderer" ]
   }
 
-  if (!is_fuchsia) {
-    deps += [ "//components/crash/content/browser" ]
-  }
-
   if (enable_basic_printing) {
     deps += [
       "//components/printing/browser",
@@ -789,8 +747,6 @@ static_library("headless_shell_lib") {
     defines = [ "HEADLESS_USE_CRASHPAD" ]
 
     deps += [
-      "//components/crash/core/app:crash_export_thunks",
-      "//components/crash/core/app:run_as_crashpad_handler",
       "//content:sandbox_helper_win",
       "//sandbox",
     ]
