diff --git a/headless/lib/headless_macros.h b/headless/lib/headless_macros.h
index f7bb9af1171e..376b8b3b04ba 100644
--- a/headless/lib/headless_macros.h
+++ b/headless/lib/headless_macros.h
@@ -7,8 +7,4 @@
 
 #include "build/build_config.h"
 
-#if defined(OS_POSIX) && !defined(OS_MAC)
-#define HEADLESS_USE_BREAKPAD
-#endif  // defined(OS_POSIX) && !defined(OS_MAC)
-
 #endif  // HEADLESS_LIB_HEADLESS_MACROS_H_
