diff --git a/ios/chrome/browser/ui/settings/BUILD.gn b/ios/chrome/browser/ui/settings/BUILD.gn
index eb36f3cc37f7..e6cb10c34b24 100644
--- a/ios/chrome/browser/ui/settings/BUILD.gn
+++ b/ios/chrome/browser/ui/settings/BUILD.gn
@@ -90,7 +90,6 @@ source_set("settings") {
     "resources:settings_privacy",
     "resources:settings_safe_browsing",
     "resources:settings_safe_state",
-    "resources:settings_safety_check",
     "resources:settings_search_engine",
     "resources:settings_unsafe_state",
     "resources:settings_voice_search",
