diff --git a/ios/chrome/browser/ios_chrome_field_trials.cc b/ios/chrome/browser/ios_chrome_field_trials.cc
index d2afe960de0a..c81577384b3f 100644
--- a/ios/chrome/browser/ios_chrome_field_trials.cc
+++ b/ios/chrome/browser/ios_chrome_field_trials.cc
@@ -5,7 +5,6 @@
 #include "ios/chrome/browser/ios_chrome_field_trials.h"
 
 #include "base/path_service.h"
-#include "components/metrics/persistent_histograms.h"
 #include "ios/chrome/browser/chrome_paths.h"
 
 void IOSChromeFieldTrials::SetupFieldTrials() {
