diff --git a/ios/chrome/browser/ui/webui/BUILD.gn b/ios/chrome/browser/ui/webui/BUILD.gn
index dc4afcfa1b7d..a89fce5eb2d2 100644
--- a/ios/chrome/browser/ui/webui/BUILD.gn
+++ b/ios/chrome/browser/ui/webui/BUILD.gn
@@ -13,8 +13,6 @@ source_set("webui") {
     "autofill_and_password_manager_internals/internals_ui_handler.mm",
     "autofill_and_password_manager_internals/password_manager_internals_ui_ios.h",
     "autofill_and_password_manager_internals/password_manager_internals_ui_ios.mm",
-    "crashes_ui.cc",
-    "crashes_ui.h",
     "flags_ui.cc",
     "flags_ui.h",
     "inspect/inspect_ui.h",
@@ -29,8 +27,6 @@ source_set("webui") {
     "suggestions_ui.h",
     "terms_ui.h",
     "terms_ui.mm",
-    "ukm_internals_ui.cc",
-    "ukm_internals_ui.h",
     "user_actions_handler.h",
     "user_actions_handler.mm",
     "user_actions_ui.h",
@@ -56,8 +52,6 @@ source_set("webui") {
     "//components/resources",
     "//components/strings",
     "//components/suggestions",
-    "//components/ukm",
-    "//components/ukm/debug:util",
     "//components/version_info",
     "//components/version_ui",
     "//google_apis",
