diff --git a/media/blink/webmediaplayer_impl.h b/media/blink/webmediaplayer_impl.h
index 0c1918a588ff..fd3ab4cc0e07 100644
--- a/media/blink/webmediaplayer_impl.h
+++ b/media/blink/webmediaplayer_impl.h
@@ -45,7 +45,6 @@
 #include "media/blink/webmediaplayer_params.h"
 #include "media/filters/pipeline_controller.h"
 #include "media/learning/common/media_learning_tasks.h"
-#include "media/mojo/mojom/playback_events_recorder.mojom.h"
 #include "media/renderers/paint_canvas_video_renderer.h"
 #include "mojo/public/cpp/bindings/remote.h"
 #include "services/media_session/public/cpp/media_position.h"
@@ -87,7 +86,6 @@ class MediaLog;
 class MemoryDumpProviderProxy;
 class UrlIndex;
 class VideoFrameCompositor;
-class WatchTimeReporter;
 
 // The canonical implementation of blink::WebMediaPlayer that's backed by
 // Pipeline. Handles normal resource loading, Media Source, and
@@ -498,11 +496,8 @@ class MEDIA_BLINK_EXPORT WebMediaPlayerImpl
   // thus we think we are idle forever).
   bool IsPrerollAttemptNeeded();
 
-  void CreateWatchTimeReporter();
   void UpdateSecondaryProperties();
 
-  void CreateVideoDecodeStatsReporter();
-
   // Returns true if the player is hidden.
   bool IsHidden() const;
 
@@ -895,7 +890,6 @@ class MEDIA_BLINK_EXPORT WebMediaPlayerImpl
   base::OneShotTimer background_pause_timer_;
 
   // Monitors the watch time of the played content.
-  std::unique_ptr<WatchTimeReporter> watch_time_reporter_;
   std::string audio_decoder_name_;
   std::string video_decoder_name_;
 
@@ -909,9 +903,6 @@ class MEDIA_BLINK_EXPORT WebMediaPlayerImpl
 
   bool have_reported_time_to_play_ready_ = false;
 
-  // Records pipeline statistics for describing media capabilities.
-  std::unique_ptr<VideoDecodeStatsReporter> video_decode_stats_reporter_;
-
   // Elapsed time since we've last reached BUFFERING_HAVE_NOTHING.
   std::unique_ptr<base::ElapsedTimer> underflow_timer_;
 
@@ -1001,9 +992,6 @@ class MEDIA_BLINK_EXPORT WebMediaPlayerImpl
 
   base::CancelableClosure update_background_status_cb_;
 
-  mojo::Remote<mojom::MediaMetricsProvider> media_metrics_provider_;
-  mojo::Remote<mojom::PlaybackEventsRecorder> playback_events_recorder_;
-
   base::Optional<ReadyState> stale_state_override_for_testing_;
 
   // True if we attempt to start the media pipeline in a suspended state for
