diff --git a/media/learning/mojo/mojo_learning_task_controller_service.cc b/media/learning/mojo/mojo_learning_task_controller_service.cc
index 92dc1e179c90..22ab35ac7c97 100644
--- a/media/learning/mojo/mojo_learning_task_controller_service.cc
+++ b/media/learning/mojo/mojo_learning_task_controller_service.cc
@@ -18,9 +18,8 @@ static const size_t kMaxInFlightObservations = 16;
 
 MojoLearningTaskControllerService::MojoLearningTaskControllerService(
     const LearningTask& task,
-    ukm::SourceId source_id,
     std::unique_ptr<::media::learning::LearningTaskController> impl)
-    : task_(task), source_id_(source_id), impl_(std::move(impl)) {}
+    : task_(task), impl_(std::move(impl)) {}
 
 MojoLearningTaskControllerService::~MojoLearningTaskControllerService() =
     default;
@@ -40,7 +39,7 @@ void MojoLearningTaskControllerService::BeginObservation(
 
   // Since we own |impl_|, we don't need to keep track of in-flight
   // observations.  We'll release |impl_| on destruction, which cancels them.
-  impl_->BeginObservation(id, features, default_target, source_id_);
+  impl_->BeginObservation(id, features, default_target);
 }
 
 void MojoLearningTaskControllerService::CompleteObservation(
