diff --git a/media/mojo/mojom/BUILD.gn b/media/mojo/mojom/BUILD.gn
index 05942bd5ea39..94103995b505 100644
--- a/media/mojo/mojom/BUILD.gn
+++ b/media/mojo/mojom/BUILD.gn
@@ -28,21 +28,17 @@ mojom("mojom") {
     "interface_factory.mojom",
     "key_system_support.mojom",
     "media_log.mojom",
-    "media_metrics_provider.mojom",
     "media_service.mojom",
     "media_types.mojom",
     "output_protection.mojom",
     "platform_verification.mojom",
-    "playback_events_recorder.mojom",
     "provision_fetcher.mojom",
     "renderer.mojom",
     "renderer_extensions.mojom",
     "video_decode_perf_history.mojom",
-    "video_decode_stats_recorder.mojom",
     "video_decoder.mojom",
     "video_encode_accelerator.mojom",
     "video_encoder_info.mojom",
-    "watch_time_recorder.mojom",
   ]
 
   if (enable_media_drm_storage) {
