diff --git a/media/learning/mojo/public/cpp/mojo_learning_task_controller.cc b/media/learning/mojo/public/cpp/mojo_learning_task_controller.cc
index 3d7b16cb6f54..00d316d2ff3c 100644
--- a/media/learning/mojo/public/cpp/mojo_learning_task_controller.cc
+++ b/media/learning/mojo/public/cpp/mojo_learning_task_controller.cc
@@ -19,13 +19,7 @@ MojoLearningTaskController::~MojoLearningTaskController() = default;
 void MojoLearningTaskController::BeginObservation(
     base::UnguessableToken id,
     const FeatureVector& features,
-    const base::Optional<TargetValue>& default_target,
-    const base::Optional<ukm::SourceId>& source_id) {
-  // We don't need to keep track of in-flight observations, since the service
-  // side handles it for us.  Also note that |source_id| is ignored; the service
-  // has no reason to trust it.  It will fill it in for us.  DCHECK in case
-  // somebody actually tries to send us one, expecting it to be used.
-  DCHECK(!source_id);
+    const base::Optional<TargetValue>& default_target) {
   controller_->BeginObservation(id, features, default_target);
 }
 
