diff --git a/media/blink/BUILD.gn b/media/blink/BUILD.gn
index fa583550b880..cfa9bafe95b1 100644
--- a/media/blink/BUILD.gn
+++ b/media/blink/BUILD.gn
@@ -47,14 +47,8 @@ component("blink") {
     "texttrack_impl.h",
     "url_index.cc",
     "url_index.h",
-    "video_decode_stats_reporter.cc",
-    "video_decode_stats_reporter.h",
     "video_frame_compositor.cc",
     "video_frame_compositor.h",
-    "watch_time_component.cc",
-    "watch_time_component.h",
-    "watch_time_reporter.cc",
-    "watch_time_reporter.h",
     "webcontentdecryptionmodule_impl.cc",
     "webcontentdecryptionmodule_impl.h",
     "webcontentdecryptionmoduleaccess_impl.cc",
@@ -159,7 +153,6 @@ test("media_blink_unittests") {
     "test_response_generator.cc",
     "test_response_generator.h",
     "url_index_unittest.cc",
-    "video_decode_stats_reporter_unittest.cc",
     "video_frame_compositor_unittest.cc",
     "watch_time_component_unittest.cc",
     "watch_time_reporter_unittest.cc",
