diff --git a/media/blink/webmediaplayer_impl.h b/media/blink/webmediaplayer_impl.h
index 995d07a7ea2c..b48c6bdab824 100644
--- a/media/blink/webmediaplayer_impl.h
+++ b/media/blink/webmediaplayer_impl.h
@@ -45,7 +45,6 @@
 #include "media/blink/webmediaplayer_params.h"
 #include "media/filters/pipeline_controller.h"
 #include "media/learning/common/media_learning_tasks.h"
-#include "media/mojo/mojom/playback_events_recorder.mojom.h"
 #include "media/renderers/paint_canvas_video_renderer.h"
 #include "mojo/public/cpp/bindings/remote.h"
 #include "services/media_session/public/cpp/media_position.h"
@@ -63,7 +62,6 @@ class WebAudioSourceProviderImpl;
 class WebLocalFrame;
 class WebMediaPlayerClient;
 class WebMediaPlayerEncryptedMediaClient;
-class WatchTimeReporter;
 }  // namespace blink
 
 namespace base {
@@ -500,11 +498,8 @@ class MEDIA_BLINK_EXPORT WebMediaPlayerImpl
   // thus we think we are idle forever).
   bool IsPrerollAttemptNeeded();
 
-  void CreateWatchTimeReporter();
   void UpdateSecondaryProperties();
 
-  void CreateVideoDecodeStatsReporter();
-
   // Returns true if the player is hidden.
   bool IsHidden() const;
 
@@ -897,7 +892,6 @@ class MEDIA_BLINK_EXPORT WebMediaPlayerImpl
   base::OneShotTimer background_pause_timer_;
 
   // Monitors the watch time of the played content.
-  std::unique_ptr<blink::WatchTimeReporter> watch_time_reporter_;
   std::string audio_decoder_name_;
   std::string video_decoder_name_;
 
@@ -911,9 +905,6 @@ class MEDIA_BLINK_EXPORT WebMediaPlayerImpl
 
   bool have_reported_time_to_play_ready_ = false;
 
-  // Records pipeline statistics for describing media capabilities.
-  std::unique_ptr<VideoDecodeStatsReporter> video_decode_stats_reporter_;
-
   // Elapsed time since we've last reached BUFFERING_HAVE_NOTHING.
   std::unique_ptr<base::ElapsedTimer> underflow_timer_;
 
@@ -1002,9 +993,6 @@ class MEDIA_BLINK_EXPORT WebMediaPlayerImpl
 
   base::CancelableClosure update_background_status_cb_;
 
-  mojo::Remote<mojom::MediaMetricsProvider> media_metrics_provider_;
-  mojo::Remote<mojom::PlaybackEventsRecorder> playback_events_recorder_;
-
   base::Optional<ReadyState> stale_state_override_for_testing_;
 
   // True if we attempt to start the media pipeline in a suspended state for
