diff --git a/media/blink/webmediaplayer_params.h b/media/blink/webmediaplayer_params.h
index 70db6159b127..79f683d44eab 100644
--- a/media/blink/webmediaplayer_params.h
+++ b/media/blink/webmediaplayer_params.h
@@ -21,7 +21,6 @@
 #include "media/base/routing_token_callback.h"
 #include "media/blink/media_blink_export.h"
 #include "media/blink/power_status_helper.h"
-#include "media/mojo/mojom/media_metrics_provider.mojom.h"
 #include "mojo/public/cpp/bindings/pending_remote.h"
 #include "third_party/blink/public/platform/web_media_player.h"
 #include "third_party/blink/public/platform/web_video_frame_submitter.h"
@@ -78,7 +77,6 @@ class MEDIA_BLINK_EXPORT WebMediaPlayerParams {
       base::WeakPtr<MediaObserver> media_observer,
       bool enable_instant_source_buffer_gc,
       bool embedded_media_experience_enabled,
-      mojo::PendingRemote<mojom::MediaMetricsProvider> metrics_provider,
       CreateSurfaceLayerBridgeCB bridge_callback,
       scoped_refptr<viz::RasterContextProvider> raster_context_provider,
       blink::WebMediaPlayer::SurfaceLayerMode use_surface_layer_for_video,
@@ -99,10 +97,6 @@ class MEDIA_BLINK_EXPORT WebMediaPlayerParams {
 
   std::unique_ptr<MediaLog> take_media_log() { return std::move(media_log_); }
 
-  mojo::PendingRemote<mojom::MediaMetricsProvider> take_metrics_provider() {
-    return std::move(metrics_provider_);
-  }
-
   const scoped_refptr<base::SingleThreadTaskRunner>& media_task_runner() const {
     return media_task_runner_;
   }
@@ -191,7 +185,6 @@ class MEDIA_BLINK_EXPORT WebMediaPlayerParams {
   base::WeakPtr<MediaObserver> media_observer_;
   bool enable_instant_source_buffer_gc_;
   const bool embedded_media_experience_enabled_;
-  mojo::PendingRemote<mojom::MediaMetricsProvider> metrics_provider_;
   CreateSurfaceLayerBridgeCB create_bridge_callback_;
   scoped_refptr<viz::RasterContextProvider> raster_context_provider_;
   blink::WebMediaPlayer::SurfaceLayerMode use_surface_layer_for_video_;
