diff --git a/media/learning/impl/learning_task_controller_impl.h b/media/learning/impl/learning_task_controller_impl.h
index 1453f1b779c9..383ecd69bcc7 100644
--- a/media/learning/impl/learning_task_controller_impl.h
+++ b/media/learning/impl/learning_task_controller_impl.h
@@ -54,8 +54,7 @@ class COMPONENT_EXPORT(LEARNING_IMPL) LearningTaskControllerImpl
   void BeginObservation(
       base::UnguessableToken id,
       const FeatureVector& features,
-      const base::Optional<TargetValue>& default_target,
-      const base::Optional<ukm::SourceId>& source_id) override;
+      const base::Optional<TargetValue>& default_target) override;
   void CompleteObservation(base::UnguessableToken id,
                            const ObservationCompletion& completion) override;
   void CancelObservation(base::UnguessableToken id) override;
@@ -68,7 +67,7 @@ class COMPONENT_EXPORT(LEARNING_IMPL) LearningTaskControllerImpl
 
  private:
   // Add |example| to the training data, and process it.
-  void AddFinishedExample(LabelledExample example, ukm::SourceId source_id);
+  void AddFinishedExample(LabelledExample example);
 
   // Called by |training_cb_| when the model is trained.  |training_weight| and
   // |training_size| are the training set's total weight and number of examples.
