diff --git a/media/blink/webmediaplayer_impl.cc b/media/blink/webmediaplayer_impl.cc
index 7fa36146a488..befade686bf9 100644
--- a/media/blink/webmediaplayer_impl.cc
+++ b/media/blink/webmediaplayer_impl.cc
@@ -58,7 +58,6 @@
 #include "media/remoting/remoting_constants.h"
 #include "mojo/public/cpp/bindings/pending_remote.h"
 #include "net/base/data_url.h"
-#include "third_party/blink/public/common/media/watch_time_reporter.h"
 #include "third_party/blink/public/platform/web_encrypted_media_types.h"
 #include "third_party/blink/public/platform/web_fullscreen_video_status.h"
 #include "third_party/blink/public/platform/web_media_player_client.h"
@@ -355,7 +354,6 @@ WebMediaPlayerImpl::WebMediaPlayerImpl(
       create_bridge_callback_(params->create_bridge_callback()),
       request_routing_token_cb_(params->request_routing_token_cb()),
       overlay_routing_token_(OverlayInfo::RoutingToken()),
-      media_metrics_provider_(params->take_metrics_provider()),
       is_background_suspend_enabled_(params->IsBackgroundSuspendEnabled()),
       is_background_video_playback_enabled_(
           params->IsBackgroundVideoPlaybackEnabled()),
@@ -452,13 +450,6 @@ WebMediaPlayerImpl::WebMediaPlayerImpl(
       base::BindRepeating(&WebMediaPlayerImpl::OnMainThreadMemoryDump,
                           weak_this_, media_log_->id()));
 
-  media_metrics_provider_->AcquirePlaybackEventsRecorder(
-      playback_events_recorder_.BindNewPipeAndPassReceiver());
-
-  // MediaMetricsProvider may drop the request for PlaybackEventsRecorder if
-  // it's not interested in recording these events.
-  playback_events_recorder_.reset_on_disconnect();
-
 #if defined(OS_ANDROID)
   renderer_factory_selector_->SetRemotePlayStateChangeCB(
       BindToCurrentLoop(base::BindRepeating(
@@ -483,9 +474,6 @@ WebMediaPlayerImpl::~WebMediaPlayerImpl() {
   delegate_->RemoveObserver(delegate_id_);
   delegate_ = nullptr;
 
-  // Finalize any watch time metrics before destroying the pipeline.
-  watch_time_reporter_.reset();
-
   // Unregister dump providers on their corresponding threads.
   media_task_runner_->DeleteSoon(FROM_HERE,
                                  std::move(media_thread_mem_dumper_));
@@ -678,13 +666,6 @@ void WebMediaPlayerImpl::SetIsEffectivelyFullscreen(
 }
 
 void WebMediaPlayerImpl::OnHasNativeControlsChanged(bool has_native_controls) {
-  if (!watch_time_reporter_)
-    return;
-
-  if (has_native_controls)
-    watch_time_reporter_->OnNativeControlsEnabled();
-  else
-    watch_time_reporter_->OnNativeControlsDisabled();
 }
 
 void WebMediaPlayerImpl::OnDisplayTypeChanged(blink::DisplayType display_type) {
@@ -696,19 +677,12 @@ void WebMediaPlayerImpl::OnDisplayTypeChanged(blink::DisplayType display_type) {
                        display_type == blink::DisplayType::kPictureInPicture));
   }
 
-  if (!watch_time_reporter_)
-    return;
-
   switch (display_type) {
     case blink::DisplayType::kInline:
-      watch_time_reporter_->OnDisplayTypeInline();
       break;
     case blink::DisplayType::kFullscreen:
-      watch_time_reporter_->OnDisplayTypeFullscreen();
       break;
     case blink::DisplayType::kPictureInPicture:
-      watch_time_reporter_->OnDisplayTypePictureInPicture();
-
       // Resumes playback if it was paused when hidden.
       if (paused_when_hidden_) {
         paused_when_hidden_ = false;
@@ -777,12 +751,6 @@ void WebMediaPlayerImpl::DoLoad(LoadType load_type,
   if (smoothness_helper_)
     smoothness_helper_.reset();
 
-  media_metrics_provider_->Initialize(
-      load_type == kLoadTypeMediaSource,
-      load_type == kLoadTypeURL ? blink::GetMediaURLScheme(loaded_url_)
-                                : mojom::MediaURLScheme::kUnknown,
-      mojom::MediaStreamType::kNone);
-
   if (demuxer_override_ || load_type == kLoadTypeMediaSource) {
     // If a demuxer override was specified or a Media Source pipeline will be
     // used, the pipeline can start immediately.
@@ -856,24 +824,7 @@ void WebMediaPlayerImpl::Play() {
   // Try to create the smoothness helper, in case we were paused before.
   UpdateSmoothnessHelper();
 
-  if (playback_events_recorder_)
-    playback_events_recorder_->OnPlaying();
-
-  watch_time_reporter_->SetAutoplayInitiated(client_->WasAutoplayInitiated());
-
-  // If we're seeking we'll trigger the watch time reporter upon seek completed;
-  // we don't want to start it here since the seek time is unstable. E.g., when
-  // playing content with a positive start time we would have a zero seek time.
-  if (!Seeking()) {
-    DCHECK(watch_time_reporter_);
-    watch_time_reporter_->OnPlaying();
-  }
-
-  if (video_decode_stats_reporter_)
-    video_decode_stats_reporter_->OnPlaying();
-
   simple_watch_timer_.Start();
-  media_metrics_provider_->SetHasPlayed();
   media_log_->AddEvent<MediaLogEvent::kPlay>();
 
   MaybeUpdateBufferSizesForPlayback();
@@ -910,15 +861,6 @@ void WebMediaPlayerImpl::Pause() {
   if (observer_)
     observer_->OnPaused();
 
-  if (playback_events_recorder_)
-    playback_events_recorder_->OnPaused();
-
-  DCHECK(watch_time_reporter_);
-  watch_time_reporter_->OnPaused();
-
-  if (video_decode_stats_reporter_)
-    video_decode_stats_reporter_->OnPaused();
-
   simple_watch_timer_.Stop();
   media_log_->AddEvent<MediaLogEvent::kPause>();
 
@@ -965,14 +907,6 @@ void WebMediaPlayerImpl::DoSeek(base::TimeDelta time, bool time_updated) {
     return;
   }
 
-  if (playback_events_recorder_)
-    playback_events_recorder_->OnSeeking();
-
-  // Call this before setting |seeking_| so that the current media time can be
-  // recorded by the reporter.
-  if (watch_time_reporter_)
-    watch_time_reporter_->OnSeeking();
-
   // Send the seek updates only when the seek pipeline hasn't started,
   // OnPipelineSeeked is not called yet.
   if (!seeking_)
@@ -1016,8 +950,6 @@ void WebMediaPlayerImpl::SetVolume(double volume) {
   DCHECK(main_task_runner_->BelongsToCurrentThread());
   volume_ = volume;
   pipeline_controller_->SetVolume(volume_ * volume_multiplier_);
-  if (watch_time_reporter_)
-    watch_time_reporter_->OnVolumeChange(volume);
   client_->DidPlayerMutedStatusChange(volume == 0.0);
 
   if (delegate_has_audio_ != HasUnmutedAudio()) {
@@ -1463,15 +1395,10 @@ void WebMediaPlayerImpl::OnEncryptedMediaInitData(
   const bool was_encrypted = is_encrypted_;
   is_encrypted_ = true;
   if (!was_encrypted) {
-    media_metrics_provider_->SetIsEME();
-    if (watch_time_reporter_)
-      CreateWatchTimeReporter();
-
     // |was_encrypted| = false means we didn't have a CDM prior to observing
     // encrypted media init data. Reset the reporter until the CDM arrives. See
     // SetCdmInternal().
     DCHECK(!cdm_config_);
-    video_decode_stats_reporter_.reset();
   }
 
   encrypted_client_->Encrypted(
@@ -1524,9 +1451,6 @@ void WebMediaPlayerImpl::SetCdmInternal(
 
   // Recreate the watch time reporter if necessary.
   if (!was_encrypted) {
-    media_metrics_provider_->SetIsEME();
-    if (watch_time_reporter_)
-      CreateWatchTimeReporter();
   }
 
   WebContentDecryptionModuleImpl* web_cdm =
@@ -1596,10 +1520,6 @@ void WebMediaPlayerImpl::OnPipelineSeeked(bool time_updated) {
   if (paused_) {
     paused_time_ = pipeline_controller_->GetMediaTime();
   } else {
-    DCHECK(watch_time_reporter_);
-    watch_time_reporter_->OnPlaying();
-    if (playback_events_recorder_)
-      playback_events_recorder_->OnPlaying();
   }
   if (time_updated)
     should_notify_time_changed_ = true;
@@ -1833,11 +1753,6 @@ void WebMediaPlayerImpl::OnError(PipelineStatus status) {
   MaybeSetContainerNameForMetrics();
   simple_watch_timer_.Stop();
   media_log_->NotifyError(status);
-  media_metrics_provider_->OnError(status);
-  if (playback_events_recorder_)
-    playback_events_recorder_->OnError(status);
-  if (watch_time_reporter_)
-    watch_time_reporter_->OnError(status);
 
   if (ready_state_ == WebMediaPlayer::kReadyStateHaveNothing) {
     // Any error that occurs before reaching ReadyStateHaveMetadata should
@@ -1866,9 +1781,6 @@ void WebMediaPlayerImpl::OnEnded() {
   ended_ = true;
   client_->TimeChanged();
 
-  if (playback_events_recorder_)
-    playback_events_recorder_->OnEnded();
-
   // We don't actually want this to run until |client_| calls seek() or pause(),
   // but that should have already happened in timeChanged() and so this is
   // expected to be a no-op.
@@ -1882,7 +1794,6 @@ void WebMediaPlayerImpl::OnMetadata(const PipelineMetadata& metadata) {
   // Cache the |time_to_metadata_| to use for adjusting the TimeToFirstFrame and
   // TimeToPlayReady metrics later if we end up doing a suspended startup.
   time_to_metadata_ = base::TimeTicks::Now() - load_start_time_;
-  media_metrics_provider_->SetTimeToMetadata(time_to_metadata_);
   RecordTimingUMA("Media.TimeToMetadata", time_to_metadata_);
 
   MaybeSetContainerNameForMetrics();
@@ -1897,13 +1808,11 @@ void WebMediaPlayerImpl::OnMetadata(const PipelineMetadata& metadata) {
       VIDEO_ROTATION_MAX + 1);
 
   if (HasAudio()) {
-    media_metrics_provider_->SetHasAudio(metadata.audio_decoder_config.codec());
     RecordEncryptionScheme("Audio",
                            metadata.audio_decoder_config.encryption_scheme());
   }
 
   if (HasVideo()) {
-    media_metrics_provider_->SetHasVideo(metadata.video_decoder_config.codec());
     RecordEncryptionScheme("Video",
                            metadata.video_decoder_config.encryption_scheme());
 
@@ -1962,7 +1871,6 @@ void WebMediaPlayerImpl::OnMetadata(const PipelineMetadata& metadata) {
   // TODO(dalecurtis): Don't create these until kReadyStateHaveFutureData; when
   // we create them early we just increase the chances of needing to throw them
   // away unnecessarily.
-  CreateWatchTimeReporter();
   CreateVideoDecodeStatsReporter();
 
   // SetReadyState() may trigger all sorts of calls into this class (e.g.,
@@ -2046,29 +1954,6 @@ void WebMediaPlayerImpl::CreateVideoDecodeStatsReporter() {
   } else if (cdm_config_) {
     DCHECK(!key_system_.empty());
   }
-
-  mojo::PendingRemote<mojom::VideoDecodeStatsRecorder> recorder;
-  media_metrics_provider_->AcquireVideoDecodeStatsRecorder(
-      recorder.InitWithNewPipeAndPassReceiver());
-
-  // Create capabilities reporter and synchronize its initial state.
-  video_decode_stats_reporter_ = std::make_unique<VideoDecodeStatsReporter>(
-      std::move(recorder),
-      base::BindRepeating(&WebMediaPlayerImpl::GetPipelineStatistics,
-                          base::Unretained(this)),
-      pipeline_metadata_.video_decoder_config.profile(),
-      pipeline_metadata_.natural_size, key_system_, cdm_config_,
-      frame_->GetTaskRunner(blink::TaskType::kInternalMedia));
-
-  if (delegate_->IsFrameHidden())
-    video_decode_stats_reporter_->OnHidden();
-  else
-    video_decode_stats_reporter_->OnShown();
-
-  if (paused_)
-    video_decode_stats_reporter_->OnPaused();
-  else
-    video_decode_stats_reporter_->OnPlaying();
 }
 
 void WebMediaPlayerImpl::OnProgress() {
@@ -2126,9 +2011,6 @@ void WebMediaPlayerImpl::OnBufferingStateChangeInternal(
       SerializableBufferingState<SerializableBufferingStateType::kPipeline>{
           state, reason, for_suspended_start});
 
-  if (state == BUFFERING_HAVE_ENOUGH && !for_suspended_start)
-    media_metrics_provider_->SetHaveEnough();
-
   if (state == BUFFERING_HAVE_ENOUGH) {
     TRACE_EVENT1("media", "WebMediaPlayerImpl::BufferingHaveEnough", "id",
                  media_log_->id());
@@ -2139,7 +2021,6 @@ void WebMediaPlayerImpl::OnBufferingStateChangeInternal(
       DCHECK(!for_suspended_start);
       have_reported_time_to_play_ready_ = true;
       const base::TimeDelta elapsed = base::TimeTicks::Now() - load_start_time_;
-      media_metrics_provider_->SetTimeToPlayReady(elapsed);
       RecordTimingUMA("Media.TimeToPlayReady", elapsed);
     }
 
@@ -2174,12 +2055,8 @@ void WebMediaPlayerImpl::OnBufferingStateChangeInternal(
     if (underflow_timer_) {
       auto elapsed = underflow_timer_->Elapsed();
       RecordUnderflowDuration(elapsed);
-      watch_time_reporter_->OnUnderflowComplete(elapsed);
       underflow_timer_.reset();
     }
-
-    if (playback_events_recorder_)
-      playback_events_recorder_->OnBufferingComplete();
   } else {
     // Buffering has underflowed.
     DCHECK_EQ(state, BUFFERING_HAVE_NOTHING);
@@ -2190,11 +2067,7 @@ void WebMediaPlayerImpl::OnBufferingStateChangeInternal(
     if (ready_state_ == WebMediaPlayer::kReadyStateHaveEnoughData &&
         !seeking_) {
       underflow_timer_ = std::make_unique<base::ElapsedTimer>();
-      watch_time_reporter_->OnUnderflow();
       client_->DidBufferUnderflow();
-
-      if (playback_events_recorder_)
-        playback_events_recorder_->OnBuffering();
     }
 
     // It shouldn't be possible to underflow if we've not advanced past
@@ -2235,9 +2108,6 @@ void WebMediaPlayerImpl::OnDurationChange() {
                                   content_type);
   delegate_->DidMediaMetadataChange(delegate_id_, delegate_has_audio_,
                                     HasVideo(), content_type);
-
-  if (watch_time_reporter_)
-    watch_time_reporter_->OnDurationChanged(GetPipelineMediaDuration());
 }
 
 void WebMediaPlayerImpl::OnAddTextTrack(const TextTrackConfig& config,
@@ -2306,21 +2176,6 @@ void WebMediaPlayerImpl::OnVideoNaturalSizeChange(const gfx::Size& size) {
 
   pipeline_metadata_.natural_size = rotated_size;
 
-  if (using_media_player_renderer_ && old_size.IsEmpty()) {
-    // If we are using MediaPlayerRenderer and this is the first size change, we
-    // now know that there is a video track. This condition is paired with code
-    // in CreateWatchTimeReporter() that guesses the existence of a video track.
-    CreateWatchTimeReporter();
-  } else {
-    UpdateSecondaryProperties();
-  }
-
-  if (video_decode_stats_reporter_ &&
-      !video_decode_stats_reporter_->MatchesBucketedNaturalSize(
-          pipeline_metadata_.natural_size)) {
-    CreateVideoDecodeStatsReporter();
-  }
-
   // Create or replace the smoothness helper now that we have a size.
   UpdateSmoothnessHelper();
 
@@ -2356,49 +2211,20 @@ void WebMediaPlayerImpl::OnAudioConfigChange(const AudioDecoderConfig& config) {
   DCHECK(main_task_runner_->BelongsToCurrentThread());
   DCHECK_NE(ready_state_, WebMediaPlayer::kReadyStateHaveNothing);
 
-  const bool codec_change =
-      pipeline_metadata_.audio_decoder_config.codec() != config.codec();
-  const bool codec_profile_change =
-      pipeline_metadata_.audio_decoder_config.profile() != config.profile();
-
   pipeline_metadata_.audio_decoder_config = config;
 
   if (observer_)
     observer_->OnMetadataChanged(pipeline_metadata_);
-
-  if (codec_change) {
-    media_metrics_provider_->SetHasAudio(
-        pipeline_metadata_.audio_decoder_config.codec());
-  }
-
-  if (codec_change || codec_profile_change)
-    UpdateSecondaryProperties();
 }
 
 void WebMediaPlayerImpl::OnVideoConfigChange(const VideoDecoderConfig& config) {
   DCHECK(main_task_runner_->BelongsToCurrentThread());
   DCHECK_NE(ready_state_, WebMediaPlayer::kReadyStateHaveNothing);
 
-  const bool codec_change =
-      pipeline_metadata_.video_decoder_config.codec() != config.codec();
-  const bool codec_profile_change =
-      pipeline_metadata_.video_decoder_config.profile() != config.profile();
-
   pipeline_metadata_.video_decoder_config = config;
 
   if (observer_)
     observer_->OnMetadataChanged(pipeline_metadata_);
-
-  if (codec_change) {
-    media_metrics_provider_->SetHasVideo(
-        pipeline_metadata_.video_decoder_config.codec());
-  }
-
-  if (codec_change || codec_profile_change)
-    UpdateSecondaryProperties();
-
-  if (video_decode_stats_reporter_ && codec_profile_change)
-    CreateVideoDecodeStatsReporter();
 }
 
 void WebMediaPlayerImpl::OnVideoAverageKeyframeDistanceUpdate() {
@@ -2406,31 +2232,18 @@ void WebMediaPlayerImpl::OnVideoAverageKeyframeDistanceUpdate() {
 }
 
 void WebMediaPlayerImpl::OnAudioDecoderChange(const AudioDecoderInfo& info) {
-  media_metrics_provider_->SetAudioPipelineInfo(info);
   if (info.decoder_type == audio_decoder_type_)
     return;
 
   audio_decoder_type_ = info.decoder_type;
-
-  // If there's no current reporter, there's nothing to be done.
-  if (!watch_time_reporter_)
-    return;
-
-  UpdateSecondaryProperties();
 }
 
 void WebMediaPlayerImpl::OnVideoDecoderChange(const VideoDecoderInfo& info) {
-  media_metrics_provider_->SetVideoPipelineInfo(info);
   if (info.decoder_type == video_decoder_type_)
     return;
 
   video_decoder_type_ = info.decoder_type;
 
-  // If there's no current reporter, there's nothing to be done.
-  if (!watch_time_reporter_)
-    return;
-
-  UpdateSecondaryProperties();
 }
 
 void WebMediaPlayerImpl::OnFrameHidden() {
@@ -2440,12 +2253,6 @@ void WebMediaPlayerImpl::OnFrameHidden() {
   if (IsHidden())
     video_locked_when_paused_when_hidden_ = true;
 
-  if (watch_time_reporter_)
-    watch_time_reporter_->OnHidden();
-
-  if (video_decode_stats_reporter_)
-    video_decode_stats_reporter_->OnHidden();
-
   UpdateBackgroundVideoOptimizationState();
   UpdatePlayState();
 
@@ -2473,12 +2280,6 @@ void WebMediaPlayerImpl::OnFrameShown() {
   // Foreground videos don't require user gesture to continue playback.
   video_locked_when_paused_when_hidden_ = false;
 
-  if (watch_time_reporter_)
-    watch_time_reporter_->OnShown();
-
-  if (video_decode_stats_reporter_)
-    video_decode_stats_reporter_->OnShown();
-
   // Notify the compositor of our page visibility status.
   vfc_task_runner_->PostTask(
       FROM_HERE,
@@ -2549,9 +2350,6 @@ void WebMediaPlayerImpl::FlingingStarted() {
 
   is_flinging_ = true;
 
-  // Capabilities reporting should only be performed for local playbacks.
-  video_decode_stats_reporter_.reset();
-
   // Requests to restart media pipeline. A flinging renderer will be created via
   // the |renderer_factory_selector_|.
   ScheduleRestart();
@@ -2596,7 +2394,6 @@ void WebMediaPlayerImpl::DataSourceInitialized(bool success) {
 
   if (!success) {
     SetNetworkState(WebMediaPlayer::kNetworkStateFormatError);
-    media_metrics_provider_->OnError(PIPELINE_ERROR_NETWORK);
 
     // Not really necessary, since the pipeline was never started, but it at
     // least this makes sure that the error handling code is in sync.
@@ -2765,10 +2562,6 @@ void WebMediaPlayerImpl::StartPipeline() {
     // MediaResource::Type::URL for the moment.
     using_media_player_renderer_ = true;
 
-    // MediaPlayerRenderer does not provide pipeline stats, so nuke capabilities
-    // reporter.
-    video_decode_stats_reporter_.reset();
-
     SetDemuxer(std::make_unique<MediaUrlDemuxer>(
         media_task_runner_, loaded_url_,
         frame_->GetDocument().SiteForCookies().RepresentativeUrl(),
@@ -3296,77 +3089,6 @@ void WebMediaPlayerImpl::ScheduleIdlePauseTimer() {
                                 &blink::WebMediaPlayerClient::ResumePlayback);
 }
 
-void WebMediaPlayerImpl::CreateWatchTimeReporter() {
-  if (!HasVideo() && !HasAudio())
-    return;
-
-  // MediaPlayerRenderer does not know about tracks until playback starts.
-  // Assume audio-only unless the natural size has been detected.
-  bool has_video = pipeline_metadata_.has_video;
-  if (using_media_player_renderer_) {
-    has_video = !pipeline_metadata_.natural_size.IsEmpty();
-  }
-
-  // Create the watch time reporter and synchronize its initial state.
-  watch_time_reporter_ = std::make_unique<blink::WatchTimeReporter>(
-      mojom::PlaybackProperties::New(
-          pipeline_metadata_.has_audio, has_video, false, false,
-          !!chunk_demuxer_, is_encrypted_, embedded_media_experience_enabled_,
-          mojom::MediaStreamType::kNone),
-      pipeline_metadata_.natural_size,
-      base::BindRepeating(&WebMediaPlayerImpl::GetCurrentTimeInternal,
-                          base::Unretained(this)),
-      base::BindRepeating(&WebMediaPlayerImpl::GetPipelineStatistics,
-                          base::Unretained(this)),
-      media_metrics_provider_.get(),
-      frame_->GetTaskRunner(blink::TaskType::kInternalMedia));
-  watch_time_reporter_->OnVolumeChange(volume_);
-  watch_time_reporter_->OnDurationChanged(GetPipelineMediaDuration());
-
-  if (delegate_->IsFrameHidden())
-    watch_time_reporter_->OnHidden();
-  else
-    watch_time_reporter_->OnShown();
-
-  if (client_->HasNativeControls())
-    watch_time_reporter_->OnNativeControlsEnabled();
-  else
-    watch_time_reporter_->OnNativeControlsDisabled();
-
-  switch (client_->GetDisplayType()) {
-    case blink::DisplayType::kInline:
-      watch_time_reporter_->OnDisplayTypeInline();
-      break;
-    case blink::DisplayType::kFullscreen:
-      watch_time_reporter_->OnDisplayTypeFullscreen();
-      break;
-    case blink::DisplayType::kPictureInPicture:
-      watch_time_reporter_->OnDisplayTypePictureInPicture();
-      break;
-  }
-
-  UpdateSecondaryProperties();
-
-  // If the WatchTimeReporter was recreated in the middle of playback, we want
-  // to resume playback here too since we won't get another play() call. When
-  // seeking, the seek completion will restart it if necessary.
-  if (!paused_ && !seeking_)
-    watch_time_reporter_->OnPlaying();
-}
-
-void WebMediaPlayerImpl::UpdateSecondaryProperties() {
-  watch_time_reporter_->UpdateSecondaryProperties(
-      mojom::SecondaryPlaybackProperties::New(
-          pipeline_metadata_.audio_decoder_config.codec(),
-          pipeline_metadata_.video_decoder_config.codec(),
-          pipeline_metadata_.audio_decoder_config.profile(),
-          pipeline_metadata_.video_decoder_config.profile(),
-          audio_decoder_type_, video_decoder_type_,
-          pipeline_metadata_.audio_decoder_config.encryption_scheme(),
-          pipeline_metadata_.video_decoder_config.encryption_scheme(),
-          pipeline_metadata_.natural_size));
-}
-
 bool WebMediaPlayerImpl::IsHidden() const {
   DCHECK(main_task_runner_->BelongsToCurrentThread());
 
@@ -3641,9 +3363,6 @@ void WebMediaPlayerImpl::SwitchToRemoteRenderer(
   DCHECK(!disable_pipeline_auto_suspend_);
   disable_pipeline_auto_suspend_ = true;
 
-  // Capabilities reporting should only be performed for local playbacks.
-  video_decode_stats_reporter_.reset();
-
   // Requests to restart media pipeline. A remote renderer will be created via
   // the |renderer_factory_selector_|.
   ScheduleRestart();
@@ -3709,9 +3428,6 @@ void WebMediaPlayerImpl::RecordVideoNaturalSize(const gfx::Size& natural_size) {
     UMA_HISTOGRAM_VIDEO_HEIGHT("Media.VideoHeight.Initial.EME", height);
 
   UMA_HISTOGRAM_VIDEO_HEIGHT("Media.VideoHeight.Initial.All", height);
-
-  if (playback_events_recorder_)
-    playback_events_recorder_->OnNaturalSizeChanged(natural_size);
 }
 
 #undef UMA_HISTOGRAM_VIDEO_HEIGHT
@@ -3728,7 +3444,6 @@ void WebMediaPlayerImpl::OnFirstFrame(base::TimeTicks frame_time) {
   has_first_frame_ = true;
   needs_first_frame_ = false;
   const base::TimeDelta elapsed = frame_time - load_start_time_;
-  media_metrics_provider_->SetTimeToFirstFrame(elapsed);
   RecordTimingUMA("Media.TimeToFirstFrame", elapsed);
 
   // Needed to signal HTMLVideoElement that it should remove the poster image.
@@ -3774,11 +3489,6 @@ void WebMediaPlayerImpl::MaybeSetContainerNameForMetrics() {
   // Container has already been set.
   if (highest_ready_state_ >= WebMediaPlayer::kReadyStateHaveMetadata)
     return;
-
-  // Only report metrics for demuxers that provide container information.
-  auto container = demuxer_->GetContainerForMetrics();
-  if (container.has_value())
-    media_metrics_provider_->SetContainerName(container.value());
 }
 
 void WebMediaPlayerImpl::MaybeUpdateBufferSizesForPlayback() {
@@ -3795,9 +3505,6 @@ void WebMediaPlayerImpl::MaybeUpdateBufferSizesForPlayback() {
 
 void WebMediaPlayerImpl::OnSimpleWatchTimerTick() {
   RecordSimpleWatchTimeUMA(reported_renderer_type_);
-
-  if (playback_events_recorder_)
-    playback_events_recorder_->OnPipelineStatistics(GetPipelineStatistics());
 }
 
 GURL WebMediaPlayerImpl::GetSrcAfterRedirects() {
@@ -3849,8 +3556,6 @@ WebMediaPlayerImpl::GetLearningTaskController(const char* task_name) {
   DCHECK_EQ(task.name, task_name);
 
   mojo::Remote<media::learning::mojom::LearningTaskController> remote_ltc;
-  media_metrics_provider_->AcquireLearningTaskController(
-      task.name, remote_ltc.BindNewPipeAndPassReceiver());
   return std::make_unique<learning::MojoLearningTaskController>(
       task, std::move(remote_ltc));
 }
