diff --git a/media/mojo/services/video_decode_perf_history.h b/media/mojo/services/video_decode_perf_history.h
index 4c50c743dd5f..5bace70f8211 100644
--- a/media/mojo/services/video_decode_perf_history.h
+++ b/media/mojo/services/video_decode_perf_history.h
@@ -160,15 +160,6 @@ class MEDIA_MOJO_EXPORT VideoDecodePerfHistory
   // nonempty.
   void OnSaveDone(base::OnceClosure save_done_cb, bool success);
 
-  // Report UKM metrics to grade the claims of the API by evaluating how well
-  // |past_stats| predicts |new_stats|.
-  void ReportUkmMetrics(ukm::SourceId source_id,
-                        bool is_top_frame,
-                        uint64_t player_id,
-                        const VideoDecodeStatsDB::VideoDescKey& video_key,
-                        const VideoDecodeStatsDB::DecodeStatsEntry& new_stats,
-                        VideoDecodeStatsDB::DecodeStatsEntry* past_stats);
-
   void AssessStats(const VideoDecodeStatsDB::VideoDescKey& key,
                    const VideoDecodeStatsDB::DecodeStatsEntry* stats,
                    bool* is_smooth,
