diff --git a/media/learning/impl/learning_task_controller_helper.cc b/media/learning/impl/learning_task_controller_helper.cc
index 753dfb0940ee..a562cfa7a6df 100644
--- a/media/learning/impl/learning_task_controller_helper.cc
+++ b/media/learning/impl/learning_task_controller_helper.cc
@@ -26,13 +26,9 @@ LearningTaskControllerHelper::~LearningTaskControllerHelper() = default;
 
 void LearningTaskControllerHelper::BeginObservation(
     base::UnguessableToken id,
-    FeatureVector features,
-    base::Optional<ukm::SourceId> source_id) {
+    FeatureVector features) {
   auto& pending_example = pending_examples_[id];
 
-  if (source_id)
-    pending_example.source_id = *source_id;
-
   // Start feature prediction, so that we capture the current values.
   if (!feature_provider_.is_null()) {
     feature_provider_.Post(
@@ -103,7 +99,7 @@ void LearningTaskControllerHelper::ProcessExampleIfFinished(
   if (!iter->second.features_done || !iter->second.target_done)
     return;
 
-  add_example_cb_.Run(std::move(iter->second.example), iter->second.source_id);
+  add_example_cb_.Run(std::move(iter->second.example));
   pending_examples_.erase(iter);
 
   // TODO(liberato): If we receive FeatureVector f1 then f2, and start filling
