diff --git a/media/mojo/services/BUILD.gn b/media/mojo/services/BUILD.gn
index 3a87515b49cd6..4df60107f3a2b 100644
--- a/media/mojo/services/BUILD.gn
+++ b/media/mojo/services/BUILD.gn
@@ -98,8 +98,6 @@ component("services") {
     "//media/learning/mojo:impl",
     "//media/mojo/common",
     "//media/mojo/common:mojo_shared_buffer_video_frame",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/service_manager/public/mojom",
   ]
 
