diff --git a/media/learning/impl/distribution_reporter.h b/media/learning/impl/distribution_reporter.h
index 5ec60e9205ba..5da21cece4f6 100644
--- a/media/learning/impl/distribution_reporter.h
+++ b/media/learning/impl/distribution_reporter.h
@@ -15,7 +15,6 @@
 #include "media/learning/common/learning_task.h"
 #include "media/learning/common/target_histogram.h"
 #include "media/learning/impl/model.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 
 namespace media {
 namespace learning {
@@ -30,19 +29,6 @@ class COMPONENT_EXPORT(LEARNING_IMPL) DistributionReporter {
     // What value was observed?
     TargetValue observed;
 
-    // UKM source id to use when logging this result.
-    // This will be filled in by the LearningTaskController.  For example, the
-    // MojoLearningTaskControllerService will be created in the browser by the
-    // MediaMetricsProvider, which gets the SourceId via callback from the
-    // RenderFrameHostDelegate on construction.
-    //
-    // TODO(liberato): Right now, this is not filled in anywhere.  When the
-    // mojo service is created (MediaMetricsProvider), record the source id and
-    // memorize it in any MojoLearningTaskControllerService that's created by
-    // the MediaMetricsProvider, either directly or in a wrapper for the
-    // mojo controller.
-    ukm::SourceId source_id = ukm::kInvalidSourceId;
-
     // Total weight of the training data used to create this model.
     double total_training_weight = 0.;
 
