diff --git a/media/cdm/cdm_module.cc b/media/cdm/cdm_module.cc
index 28cba10d248b..abec2ac05b5b 100644
--- a/media/cdm/cdm_module.cc
+++ b/media/cdm/cdm_module.cc
@@ -12,7 +12,6 @@
 #include "base/notreached.h"
 #include "base/time/time.h"
 #include "build/build_config.h"
-#include "components/crash/core/common/crash_key.h"
 
 #if BUILDFLAG(ENABLE_CDM_HOST_VERIFICATION)
 #include "base/feature_list.h"
@@ -174,9 +173,6 @@ bool CdmModule::Initialize(const base::FilePath& cdm_path) {
   std::string cdm_version = get_cdm_version_func_();
   DVLOG(2) << __func__ << ": cdm_version = " << cdm_version;
 
-  static crash_reporter::CrashKeyString<32> cdm_version_key("cdm-version");
-  cdm_version_key.Set(cdm_version);
-
 #if defined(OS_WIN)
   // Load DXVA before sandbox lockdown to give CDM access to Output Protection
   // Manager (OPM).
