diff --git a/media/blink/BUILD.gn b/media/blink/BUILD.gn
index 5e5ede151daf..44806a562bd1 100644
--- a/media/blink/BUILD.gn
+++ b/media/blink/BUILD.gn
@@ -47,8 +47,6 @@ component("blink") {
     "texttrack_impl.h",
     "url_index.cc",
     "url_index.h",
-    "video_decode_stats_reporter.cc",
-    "video_decode_stats_reporter.h",
     "video_frame_compositor.cc",
     "video_frame_compositor.h",
     "webcontentdecryptionmodule_impl.cc",
@@ -155,9 +153,7 @@ test("media_blink_unittests") {
     "test_response_generator.cc",
     "test_response_generator.h",
     "url_index_unittest.cc",
-    "video_decode_stats_reporter_unittest.cc",
     "video_frame_compositor_unittest.cc",
-    "watch_time_reporter_unittest.cc",
     "webmediaplayer_impl_unittest.cc",
   ]
 
