diff --git a/media/blink/watch_time_reporter.h b/media/blink/watch_time_reporter.h
index c467629d6292..aef62a243703 100644
--- a/media/blink/watch_time_reporter.h
+++ b/media/blink/watch_time_reporter.h
@@ -18,7 +18,6 @@
 #include "media/base/video_codecs.h"
 #include "media/blink/media_blink_export.h"
 #include "media/blink/watch_time_component.h"
-#include "media/mojo/mojom/media_metrics_provider.mojom.h"
 #include "media/mojo/mojom/watch_time_recorder.mojom.h"
 #include "mojo/public/cpp/bindings/remote.h"
 #include "third_party/blink/public/platform/web_media_player.h"
@@ -85,7 +84,6 @@ class MEDIA_BLINK_EXPORT WatchTimeReporter : base::PowerObserver {
                     const gfx::Size& natural_size,
                     GetMediaTimeCB get_media_time_cb,
                     GetPipelineStatsCB get_pipeline_stats_cb,
-                    mojom::MediaMetricsProvider* provider,
                     scoped_refptr<base::SequencedTaskRunner> task_runner,
                     const base::TickClock* tick_clock = nullptr);
   ~WatchTimeReporter() override;
@@ -164,7 +162,6 @@ class MEDIA_BLINK_EXPORT WatchTimeReporter : base::PowerObserver {
                     const gfx::Size& natural_size,
                     GetMediaTimeCB get_media_time_cb,
                     GetPipelineStatsCB get_pipeline_stats_cb,
-                    mojom::MediaMetricsProvider* provider,
                     scoped_refptr<base::SequencedTaskRunner> task_runner,
                     const base::TickClock* tick_clock);
 
