diff --git a/net/BUILD.gn b/net/BUILD.gn
index 66b48b19c1e20..fc22b047cc8b0 100644
--- a/net/BUILD.gn
+++ b/net/BUILD.gn
@@ -8,6 +8,7 @@ import("//build/config/chromeos/ui_mode.gni")
 import("//build/config/compiler/compiler.gni")
 import("//build/config/crypto.gni")
 import("//build/config/features.gni")
+import("//dumb/net/sources.gni")
 import("//net/features.gni")
 import("//testing/libfuzzer/fuzzer_test.gni")
 import("//testing/test.gni")
@@ -390,6 +391,8 @@ component("net") {
     "//net/http:transport_security_state_generated_files",
   ]
 
+  deps += net_deps
+
   public_deps = [
     ":net_export_header",
     ":net_public_deps",
@@ -1807,6 +1810,8 @@ component("extras") {
     "//sql:sql",
   ]
 
+  deps += net_extra_deps
+
   if (enable_reporting) {
     sources += [
       "extras/sqlite/sqlite_persistent_reporting_and_nel_store.cc",
