diff --git a/rlz/lib/lib_values.cc b/rlz/lib/lib_values.cc
index d9cfbbe1480a..4c219de19bf4 100644
--- a/rlz/lib/lib_values.cc
+++ b/rlz/lib/lib_values.cc
@@ -41,7 +41,7 @@ const char kSetDccResponseVariable[] = "set_dcc";
 //
 
 const char kFinancialPingPath[] = "/tools/pso/ping";
-const char kFinancialServer[]   = "clients1.google.com";
+const char kFinancialServer[]   = "no-thanks.invalid";
 const int kFinancialPort = 443;
 
 // Ping times in 100-nanosecond intervals.
