diff --git a/services/preferences/public/mojom/preferences.mojom b/services/preferences/public/mojom/preferences.mojom
index edad7de7b214..0c75404a91ba 100644
--- a/services/preferences/public/mojom/preferences.mojom
+++ b/services/preferences/public/mojom/preferences.mojom
@@ -21,7 +21,6 @@ struct TrackedPersistentPrefStoreConfiguration {
   string legacy_device_id;
   string registry_seed;
   mojo_base.mojom.String16 registry_path;
-  pending_remote<TrackedPreferenceValidationDelegate>? validation_delegate;
   pending_remote<ResetOnLoadObserver>? reset_on_load_observer;
 };
 
