diff --git a/services/network/BUILD.gn b/services/network/BUILD.gn
index 96eb95c55c6f8..98d9a8b860b70 100644
--- a/services/network/BUILD.gn
+++ b/services/network/BUILD.gn
@@ -3,6 +3,7 @@
 # found in the LICENSE file.
 
 import("//build/config/chromeos/ui_mode.gni")
+import("//dumb/services/network/sources.gni")
 import("//mojo/public/tools/bindings/mojom.gni")
 import("//net/features.gni")
 import("//services/network/public/cpp/features.gni")
@@ -70,8 +71,6 @@ component("network_service") {
     "network_quality_estimator_manager.h",
     "network_service.cc",
     "network_service.h",
-    "network_service_network_delegate.cc",
-    "network_service_network_delegate.h",
     "network_service_proxy_delegate.cc",
     "network_service_proxy_delegate.h",
     "origin_policy/origin_policy_constants.h",
@@ -225,7 +224,6 @@ component("network_service") {
     "//build:chromeos_buildflags",
     "//components/content_settings/core/common",
     "//components/cookie_config",
-    "//components/domain_reliability",
     "//components/network_session_configurator/browser",
     "//components/network_session_configurator/common",
     "//components/os_crypt",
@@ -253,6 +251,8 @@ component("network_service") {
     deps += [ ":network_sandbox_hook" ]
   }
 
+  deps += dumb_services_network_deps
+
   public_deps = [ "//services/network/public/cpp:buildflags" ]
 
   if (is_ct_supported) {
