diff --git a/services/network/network_service.cc b/services/network/network_service.cc
index 01e7a291e2a5..687e479ebeaa 100644
--- a/services/network/network_service.cc
+++ b/services/network/network_service.cc
@@ -27,6 +27,7 @@
 #include "build/chromeos_buildflags.h"
 #include "components/network_session_configurator/common/network_features.h"
 #include "components/os_crypt/os_crypt.h"
+#include "dumb/services/network/url_purify/url_purify_delegate.h"
 #include "mojo/public/cpp/bindings/scoped_message_error_crash_key.h"
 #include "mojo/public/cpp/system/functions.h"
 #include "net/base/features.h"
@@ -284,6 +285,7 @@ NetworkService::NetworkService(
     : net_log_(net::NetLog::Get()), registry_(std::move(registry)) {
   DCHECK(!g_network_service);
   g_network_service = this;
+  LOG(INFO) << "NetworkService::NetworkService";
 
   // |registry_| is nullptr when an in-process NetworkService is
   // created directly, like in most unit tests.
@@ -385,6 +387,8 @@ void NetworkService::Initialize(mojom::NetworkServiceParamsPtr params,
   sct_auditing_cache_ =
       std::make_unique<SCTAuditingCache>(kMaxSCTAuditingCacheEntries);
 #endif
+
+  url_purify_delegate_ = std::make_unique<dumb::URLPurifyDelegate>();
 }
 
 NetworkService::~NetworkService() {
@@ -415,6 +419,7 @@ void NetworkService::set_os_crypt_is_configured() {
 
 std::unique_ptr<NetworkService> NetworkService::Create(
     mojo::PendingReceiver<mojom::NetworkService> receiver) {
+  LOG(INFO) << "NetworkService::Create";
   return std::make_unique<NetworkService>(nullptr, std::move(receiver));
 }
 
