diff --git a/services/viz/public/cpp/compositing/quads_mojom_traits.cc b/services/viz/public/cpp/compositing/quads_mojom_traits.cc
index 308e76e04176..5e48bebac539 100644
--- a/services/viz/public/cpp/compositing/quads_mojom_traits.cc
+++ b/services/viz/public/cpp/compositing/quads_mojom_traits.cc
@@ -5,7 +5,6 @@
 #include "services/viz/public/cpp/compositing/quads_mojom_traits.h"
 
 #include "services/viz/public/cpp/compositing/compositor_render_pass_id_mojom_traits.h"
-#include "services/viz/public/cpp/crash_keys.h"
 #include "ui/gfx/mojom/color_space_mojom_traits.h"
 #include "ui/gfx/mojom/transform_mojom_traits.h"
 
@@ -85,7 +84,6 @@ bool StructTraits<
   }
   // CompositorRenderPass ids are never zero.
   if (!quad->render_pass_id) {
-    viz::SetDeserializationCrashKeyString("Draw quad invalid render pass ID");
     return false;
   }
   quad->force_anti_aliasing_off = data.force_anti_aliasing_off();
@@ -220,11 +218,9 @@ bool StructTraits<viz::mojom::YUVVideoQuadStateDataView, viz::DrawQuad>::Read(
   quad->resource_multiplier = data.resource_multiplier();
   quad->bits_per_channel = data.bits_per_channel();
   if (quad->bits_per_channel < viz::YUVVideoDrawQuad::kMinBitsPerChannel) {
-    viz::SetDeserializationCrashKeyString("Bits per channel too small");
     return false;
   }
   if (quad->bits_per_channel > viz::YUVVideoDrawQuad::kMaxBitsPerChannel) {
-    viz::SetDeserializationCrashKeyString("Bits per channel too big");
     return false;
   }
   return true;
@@ -238,7 +234,6 @@ bool StructTraits<viz::mojom::DrawQuadDataView, viz::DrawQuad>::Read(
     return false;
   }
   if (!out->rect.Contains(out->visible_rect)) {
-    viz::SetDeserializationCrashKeyString("Rect does not contain visible rect");
     return false;
   }
 
