diff --git a/services/preferences/tracked/pref_hash_filter.cc b/services/preferences/tracked/pref_hash_filter.cc
index c9ced554c9734..82ecacab50fc1 100644
--- a/services/preferences/tracked/pref_hash_filter.cc
+++ b/services/preferences/tracked/pref_hash_filter.cc
@@ -57,9 +57,6 @@ PrefHashFilter::PrefHashFilter(
         tracked_preferences,
     mojo::PendingRemote<prefs::mojom::ResetOnLoadObserver>
         reset_on_load_observer,
-    scoped_refptr<base::RefCountedData<
-        mojo::Remote<prefs::mojom::TrackedPreferenceValidationDelegate>>>
-        delegate,
     size_t reporting_ids_count)
     : pref_hash_store_(std::move(pref_hash_store)),
       external_validation_hash_store_pair_(
@@ -67,8 +64,7 @@ PrefHashFilter::PrefHashFilter(
               ? absl::make_optional(
                     std::move(external_validation_hash_store_pair))
               : absl::nullopt),
-      reset_on_load_observer_(std::move(reset_on_load_observer)),
-      delegate_(std::move(delegate)) {
+      reset_on_load_observer_(std::move(reset_on_load_observer)) {
   DCHECK(pref_hash_store_);
   DCHECK_GE(reporting_ids_count, tracked_preferences.size());
   // Verify that, if |external_validation_hash_store_pair_| is present, both its
@@ -77,8 +73,6 @@ PrefHashFilter::PrefHashFilter(
          (external_validation_hash_store_pair_->first &&
           external_validation_hash_store_pair_->second));
 
-  prefs::mojom::TrackedPreferenceValidationDelegate* delegate_ptr =
-      (delegate_ ? delegate_->data.get() : nullptr);
   for (size_t i = 0; i < tracked_preferences.size(); ++i) {
     const prefs::mojom::TrackedPreferenceMetadata& metadata =
         *tracked_preferences[i];
@@ -88,12 +82,12 @@ PrefHashFilter::PrefHashFilter(
       case PrefTrackingStrategy::ATOMIC:
         tracked_preference = std::make_unique<TrackedAtomicPreference>(
             metadata.name, metadata.reporting_id, reporting_ids_count,
-            metadata.enforcement_level, metadata.value_type, delegate_ptr);
+            metadata.enforcement_level, metadata.value_type);
         break;
       case PrefTrackingStrategy::SPLIT:
         tracked_preference = std::make_unique<TrackedSplitPreference>(
             metadata.name, metadata.reporting_id, reporting_ids_count,
-            metadata.enforcement_level, metadata.value_type, delegate_ptr);
+            metadata.enforcement_level, metadata.value_type);
         break;
     }
     DCHECK(tracked_preference);
