diff --git a/services/network/public/mojom/BUILD.gn b/services/network/public/mojom/BUILD.gn
index 92458987d18ed..3122078a0da5e 100644
--- a/services/network/public/mojom/BUILD.gn
+++ b/services/network/public/mojom/BUILD.gn
@@ -734,7 +734,10 @@ mojom("cookies_mojom") {
           [ "//services/network/public/cpp/cookie_manager_mojom_traits.h" ]
       traits_sources =
           [ "//services/network/public/cpp/cookie_manager_mojom_traits.cc" ]
-      traits_public_deps = [ "//net" ]
+      traits_public_deps =
+          [ "//net",
+            "//dumb/components/contextual_identity",
+          ]
     },
   ]
   cpp_typemaps += shared_cpp_typemaps
