diff --git a/services/network/public/cpp/BUILD.gn b/services/network/public/cpp/BUILD.gn
index 1ba78613e28e..4e9c472fcb4a 100644
--- a/services/network/public/cpp/BUILD.gn
+++ b/services/network/public/cpp/BUILD.gn
@@ -178,7 +178,6 @@ component("cookies_mojom_support") {
     "site_for_cookies_mojom_traits.h",
   ]
   deps = [
-    ":crash_keys",
     "//net",
     "//services/network/public/mojom:cookies_mojom_shared",
   ]
@@ -277,7 +276,6 @@ component("cpp_base") {
 
   public_deps = [
     ":cookies_mojom_support",
-    ":crash_keys",
     ":cross_origin_embedder_policy",
     ":ip_address_mojom_support",
     ":schemeful_site_mojom_support",
