diff --git a/services/network/public/mojom/network_context.mojom b/services/network/public/mojom/network_context.mojom
index 5aca8515f5ce..113ab21fff08 100644
--- a/services/network/public/mojom/network_context.mojom
+++ b/services/network/public/mojom/network_context.mojom
@@ -645,9 +645,6 @@ struct URLLoaderFactoryParams {
   // URLLoader based on the Fetch client state.
   ClientSecurityState? client_security_state;
 
-  // Used to report CORP violations caused by COEP.
-  pending_remote<CrossOriginEmbedderPolicyReporter>? coep_reporter;
-
   // Used to notify clients about cookie reads or writes.
   pending_remote<CookieAccessObserver>? cookie_observer;
 
