diff --git a/services/preferences/tracked/tracked_atomic_preference.cc b/services/preferences/tracked/tracked_atomic_preference.cc
index c4f820af7009..59c7a9d564e7 100644
--- a/services/preferences/tracked/tracked_atomic_preference.cc
+++ b/services/preferences/tracked/tracked_atomic_preference.cc
@@ -16,15 +16,13 @@ TrackedAtomicPreference::TrackedAtomicPreference(
     size_t reporting_id,
     size_t reporting_ids_count,
     prefs::mojom::TrackedPreferenceMetadata::EnforcementLevel enforcement_level,
-    prefs::mojom::TrackedPreferenceMetadata::ValueType value_type,
-    prefs::mojom::TrackedPreferenceValidationDelegate* delegate)
+    prefs::mojom::TrackedPreferenceMetadata::ValueType value_type)
     : pref_path_(pref_path),
       helper_(pref_path,
               reporting_id,
               reporting_ids_count,
               enforcement_level,
-              value_type),
-      delegate_(delegate) {}
+              value_type) {}
 
 TrackedPreferenceType TrackedAtomicPreference::GetType() const {
   return TrackedPreferenceType::ATOMIC;
@@ -54,11 +52,6 @@ bool TrackedAtomicPreference::EnforceAndReport(
         external_validation_transaction->GetStoreUMASuffix());
   }
 
-  if (delegate_) {
-    delegate_->OnAtomicPreferenceValidation(
-        pref_path_, value ? base::make_optional(value->Clone()) : base::nullopt,
-        value_state, external_validation_value_state, helper_.IsPersonal());
-  }
   TrackedPreferenceHelper::ResetAction reset_action =
       helper_.GetAction(value_state);
   helper_.ReportAction(reset_action);
