diff --git a/services/network/BUILD.gn b/services/network/BUILD.gn
index 48943a0d9559f..505dc7ce55bad 100644
--- a/services/network/BUILD.gn
+++ b/services/network/BUILD.gn
@@ -70,8 +70,6 @@ component("network_service") {
     "network_quality_estimator_manager.h",
     "network_service.cc",
     "network_service.h",
-    "network_service_network_delegate.cc",
-    "network_service_network_delegate.h",
     "network_service_proxy_delegate.cc",
     "network_service_proxy_delegate.h",
     "origin_policy/origin_policy_constants.h",
@@ -225,7 +223,6 @@ component("network_service") {
     "//build:chromeos_buildflags",
     "//components/content_settings/core/common",
     "//components/cookie_config",
-    "//components/domain_reliability",
     "//components/network_session_configurator/browser",
     "//components/network_session_configurator/common",
     "//components/os_crypt",
@@ -253,6 +250,8 @@ component("network_service") {
     deps += [ ":network_sandbox_hook" ]
   }
 
+  deps += [ "//dumb/services/network" ]
+
   public_deps = [ "//services/network/public/cpp:buildflags" ]
 
   if (is_ct_supported) {
