diff --git a/services/resource_coordinator/BUILD.gn b/services/resource_coordinator/BUILD.gn
index ef63860f9798..8be10a5f9b45 100644
--- a/services/resource_coordinator/BUILD.gn
+++ b/services/resource_coordinator/BUILD.gn
@@ -34,9 +34,6 @@ source_set("lib") {
   public_deps = [
     "//base",
     "//mojo/public/cpp/bindings",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
-    "//services/metrics/public/mojom",
     "//services/resource_coordinator/public/cpp:resource_coordinator_cpp",
     "//third_party/perfetto:libperfetto",
   ]
@@ -61,7 +58,6 @@ source_set("tests") {
     ":lib",
     "//base",
     "//base/test:test_support",
-    "//components/ukm:test_support",
     "//mojo/public/cpp/bindings",
     "//services/resource_coordinator/public/cpp:resource_coordinator_cpp",
     "//services/resource_coordinator/public/cpp/memory_instrumentation:memory_instrumentation",
