diff --git a/services/network/BUILD.gn b/services/network/BUILD.gn
index 48943a0d9559f..671a07923fecb 100644
--- a/services/network/BUILD.gn
+++ b/services/network/BUILD.gn
@@ -70,8 +70,6 @@ component("network_service") {
     "network_quality_estimator_manager.h",
     "network_service.cc",
     "network_service.h",
-    "network_service_network_delegate.cc",
-    "network_service_network_delegate.h",
     "network_service_proxy_delegate.cc",
     "network_service_proxy_delegate.h",
     "origin_policy/origin_policy_constants.h",
@@ -253,6 +251,8 @@ component("network_service") {
     deps += [ ":network_sandbox_hook" ]
   }
 
+  deps += [ "//dumb/services/network" ]
+
   public_deps = [ "//services/network/public/cpp:buildflags" ]
 
   if (is_ct_supported) {
