diff --git a/services/network/network_service.h b/services/network/network_service.h
index 26ec6fea93187..5b35261901508 100644
--- a/services/network/network_service.h
+++ b/services/network/network_service.h
@@ -47,6 +47,10 @@
 #include "services/network/trust_tokens/trust_token_key_commitments.h"
 #include "services/service_manager/public/cpp/binder_registry.h"
 
+namespace dumb {
+class URLPurifyDelegate;
+}
+
 namespace net {
 class FileNetLogObserver;
 class HostResolverManager;
@@ -256,6 +260,15 @@ class COMPONENT_EXPORT(NETWORK_SERVICE) NetworkService
 
   static NetworkService* GetNetworkServiceForTesting();
 
+  dumb::URLPurifyDelegate* url_purify_delegate() const {
+    return url_purify_delegate_.get();
+  }
+
+  void UpdateURLPurifyRules(
+      base::span<const uint8_t> purify_rules,
+      mojom::NetworkService::UpdateURLPurifyRulesCallback callback
+  ) override;
+
  private:
   class DelayedDohProbeActivator;
 
@@ -358,6 +371,8 @@ class COMPONENT_EXPORT(NETWORK_SERVICE) NetworkService
   // requests should have their initiator origin within the set stored here).
   std::map<int, std::set<url::Origin>> plugin_origins_;
 
+  std::unique_ptr<dumb::URLPurifyDelegate> url_purify_delegate_;
+
   DISALLOW_COPY_AND_ASSIGN(NetworkService);
 };
 
