diff --git a/services/preferences/tracked/segregated_pref_store.cc b/services/preferences/tracked/segregated_pref_store.cc
index f658c7ddef3b..bf1fdceed384 100644
--- a/services/preferences/tracked/segregated_pref_store.cc
+++ b/services/preferences/tracked/segregated_pref_store.cc
@@ -53,11 +53,8 @@ void SegregatedPrefStore::AggregatingObserver::OnInitializationCompleted(
 SegregatedPrefStore::SegregatedPrefStore(
     const scoped_refptr<PersistentPrefStore>& default_pref_store,
     const scoped_refptr<PersistentPrefStore>& selected_pref_store,
-    const std::set<std::string>& selected_pref_names,
-    mojo::Remote<prefs::mojom::TrackedPreferenceValidationDelegate>
-        validation_delegate)
-    : validation_delegate_(std::move(validation_delegate)),
-      default_pref_store_(default_pref_store),
+    const std::set<std::string>& selected_pref_names)
+    : default_pref_store_(default_pref_store),
       selected_pref_store_(selected_pref_store),
       selected_preference_names_(selected_pref_names),
       aggregating_observer_(this) {
