diff --git a/services/BUILD.gn b/services/BUILD.gn
index 9027402467ca..a2f60382061f 100644
--- a/services/BUILD.gn
+++ b/services/BUILD.gn
@@ -25,7 +25,6 @@ test("services_unittests") {
   deps = [
     "//services/cert_verifier:tests",
     "//services/cert_verifier/cert_net_url_loader:tests",
-    "//services/metrics/public/cpp:tests",
     "//services/network:tests",
     "//services/network/public/cpp:tests",
     "//services/test:run_all_unittests",
