diff --git a/services/preferences/tracked/tracked_split_preference.cc b/services/preferences/tracked/tracked_split_preference.cc
index 053f315bd85e..df21bb5f5a04 100644
--- a/services/preferences/tracked/tracked_split_preference.cc
+++ b/services/preferences/tracked/tracked_split_preference.cc
@@ -20,15 +20,13 @@ TrackedSplitPreference::TrackedSplitPreference(
     size_t reporting_id,
     size_t reporting_ids_count,
     prefs::mojom::TrackedPreferenceMetadata::EnforcementLevel enforcement_level,
-    prefs::mojom::TrackedPreferenceMetadata::ValueType value_type,
-    prefs::mojom::TrackedPreferenceValidationDelegate* delegate)
+    prefs::mojom::TrackedPreferenceMetadata::ValueType value_type)
     : pref_path_(pref_path),
       helper_(pref_path,
               reporting_id,
               reporting_ids_count,
               enforcement_level,
-              value_type),
-      delegate_(delegate) {}
+              value_type) {}
 
 TrackedPreferenceType TrackedSplitPreference::GetType() const {
   return TrackedPreferenceType::SPLIT;
@@ -74,11 +72,6 @@ bool TrackedSplitPreference::EnforceAndReport(
         external_validation_transaction->GetStoreUMASuffix());
   }
 
-  if (delegate_) {
-    delegate_->OnSplitPreferenceValidation(
-        pref_path_, invalid_keys, external_validation_invalid_keys, value_state,
-        external_validation_value_state, helper_.IsPersonal());
-  }
   TrackedPreferenceHelper::ResetAction reset_action =
       helper_.GetAction(value_state);
   helper_.ReportAction(reset_action);
