diff --git a/services/network/network_service_network_delegate.cc b/services/network/network_service_network_delegate.cc
index 25e64eb7c66c..e65488fa548e 100644
--- a/services/network/network_service_network_delegate.cc
+++ b/services/network/network_service_network_delegate.cc
@@ -11,6 +11,7 @@
 #include "base/strings/utf_string_conversions.h"
 #include "build/build_config.h"
 #include "components/domain_reliability/monitor.h"
+#include "dumb/services/network/url_purify/url_purify_delegate.h"
 #include "net/base/features.h"
 #include "net/base/isolation_info.h"
 #include "net/base/load_flags.h"
