diff --git a/services/resource_coordinator/public/mojom/BUILD.gn b/services/resource_coordinator/public/mojom/BUILD.gn
index ec433ce1e59b..ac424a216419 100644
--- a/services/resource_coordinator/public/mojom/BUILD.gn
+++ b/services/resource_coordinator/public/mojom/BUILD.gn
@@ -16,7 +16,6 @@ mojom_component("mojom") {
 
   public_deps = [
     "//mojo/public/mojom/base",
-    "//services/metrics/public/mojom",
   ]
 
   enabled_features = []
