diff --git a/services/network/public/cpp/site_for_cookies_mojom_traits.cc b/services/network/public/cpp/site_for_cookies_mojom_traits.cc
index 282d0ef1e087..200a8aafccd2 100644
--- a/services/network/public/cpp/site_for_cookies_mojom_traits.cc
+++ b/services/network/public/cpp/site_for_cookies_mojom_traits.cc
@@ -4,7 +4,6 @@
 
 #include "services/network/public/cpp/site_for_cookies_mojom_traits.h"
 #include "net/base/features.h"
-#include "services/network/public/cpp/crash_keys.h"
 
 namespace mojo {
 
@@ -21,9 +20,6 @@ bool StructTraits<network::mojom::SiteForCookiesDataView, net::SiteForCookies>::
 
   bool result = net::SiteForCookies::FromWire(scheme, registrable_domain,
                                               data.schemefully_same(), out);
-  if (!result) {
-    network::debug::SetDeserializationCrashKeyString("site_for_cookie");
-  }
   return result;
 }
 
