diff --git a/services/network/sct_auditing_cache.cc b/services/network/sct_auditing_cache.cc
index df60a69c5a53..02cd28ad7e9c 100644
--- a/services/network/sct_auditing_cache.cc
+++ b/services/network/sct_auditing_cache.cc
@@ -138,8 +138,6 @@ void SCTAuditingCache::MaybeEnqueueReport(
   double sampling_rate = features::kSCTAuditingSamplingRate.Get();
   if (base::RandDouble() > sampling_rate)
     return;
-
-  context->client()->OnSCTReportReady(net::HashValue(cache_key).ToString());
 }
 
 sct_auditing::TLSConnectionReport* SCTAuditingCache::GetPendingReport(
