diff --git a/services/network/public/mojom/network_service.mojom b/services/network/public/mojom/network_service.mojom
index f7b1e990333be..c2649c51b59ce 100644
--- a/services/network/public/mojom/network_service.mojom
+++ b/services/network/public/mojom/network_service.mojom
@@ -360,4 +360,7 @@ interface NetworkService {
   // Sets the list of ports which will be permitted even if they normally would
   // be restricted.
   SetExplicitlyAllowedPorts(array<uint16> ports);
+
+  // Update URL purify rules.
+  UpdateURLPurifyRules(mojo_base.mojom.ReadOnlyBuffer purify_rules) => ();
 };
