diff --git a/services/viz/public/cpp/compositing/compositor_render_pass_mojom_traits.cc b/services/viz/public/cpp/compositing/compositor_render_pass_mojom_traits.cc
index 4f939e73ab79..be0b6b7fae21 100644
--- a/services/viz/public/cpp/compositing/compositor_render_pass_mojom_traits.cc
+++ b/services/viz/public/cpp/compositing/compositor_render_pass_mojom_traits.cc
@@ -8,7 +8,6 @@
 #include "components/viz/common/quads/compositor_render_pass.h"
 #include "services/viz/public/cpp/compositing/compositor_render_pass_id_mojom_traits.h"
 #include "services/viz/public/cpp/compositing/shared_quad_state_mojom_traits.h"
-#include "services/viz/public/cpp/crash_keys.h"
 #include "ui/gfx/mojom/display_color_spaces_mojom_traits.h"
 
 namespace mojo {
@@ -31,7 +30,6 @@ bool StructTraits<viz::mojom::CompositorRenderPassDataView,
   }
   // CompositorRenderPass ids are never zero.
   if (!(*out)->id) {
-    viz::SetDeserializationCrashKeyString("Invalid render pass ID");
     return false;
   }
   (*out)->has_transparent_background = data.has_transparent_background();
@@ -52,7 +50,6 @@ bool StructTraits<viz::mojom::CompositorRenderPassDataView,
     viz::DrawQuad* quad =
         AllocateAndConstruct(quad_state_data_view.tag(), &(*out)->quad_list);
     if (!quad) {
-      viz::SetDeserializationCrashKeyString("AllocateAndConstruct quad failed");
       return false;
     }
     if (!quads.Read(i, quad))
@@ -72,7 +69,6 @@ bool StructTraits<viz::mojom::CompositorRenderPassDataView,
     }
     quad->shared_quad_state = last_sqs;
     if (!quad->shared_quad_state) {
-      viz::SetDeserializationCrashKeyString("No shared quad state");
       return false;
     }
   }
