diff --git a/services/metrics/public/cpp/BUILD.gn b/services/metrics/public/cpp/BUILD.gn
index 68bfc8ee1522..2b866a06b9c9 100644
--- a/services/metrics/public/cpp/BUILD.gn
+++ b/services/metrics/public/cpp/BUILD.gn
@@ -6,23 +6,9 @@ import("//mojo/public/tools/bindings/mojom.gni")
 
 component("metrics_cpp") {
   sources = [
-    "delegating_ukm_recorder.cc",
-    "delegating_ukm_recorder.h",
     "metrics_export.h",
     "metrics_utils.cc",
     "metrics_utils.h",
-    "mojo_ukm_recorder.cc",
-    "mojo_ukm_recorder.h",
-    "ukm_entry_builder.cc",
-    "ukm_entry_builder.h",
-    "ukm_entry_builder_base.cc",
-    "ukm_entry_builder_base.h",
-    "ukm_recorder.cc",
-    "ukm_recorder.h",
-    "ukm_source.cc",
-    "ukm_source.h",
-    "ukm_source_id.cc",
-    "ukm_source_id.h",
   ]
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
@@ -37,7 +23,6 @@ component("metrics_cpp") {
 
   deps = [
     "//mojo/public/cpp/bindings",
-    "//third_party/metrics_proto",
   ]
 }
 
@@ -46,7 +31,6 @@ source_set("tests") {
 
   sources = [
     "metrics_utils_unittest.cc",
-    "ukm_source_id_unittest.cc",
   ]
 
   deps = [
