diff --git a/services/network/BUILD.gn b/services/network/BUILD.gn
index 294695ec053c..744649bbfdbc 100644
--- a/services/network/BUILD.gn
+++ b/services/network/BUILD.gn
@@ -25,8 +25,6 @@ component("network_service") {
     "cors/cors_url_loader_factory.h",
     "cors/preflight_controller.cc",
     "cors/preflight_controller.h",
-    "crash_keys.cc",
-    "crash_keys.h",
     "crl_set_distributor.cc",
     "crl_set_distributor.h",
     "cross_origin_read_blocking_exception_for_plugin.cc",
@@ -237,7 +235,6 @@ component("network_service") {
     "//net:extras",
     "//services/network/first_party_sets",
     "//services/network/public/cpp",
-    "//services/network/public/cpp:crash_keys",
     "//services/network/public/cpp/cert_verifier:cert_verifier_creation",
     "//services/network/public/cpp/cert_verifier:mojo_cert_verifier",
     "//services/network/public/mojom",
