diff --git a/services/preferences/tracked/pref_hash_filter.h b/services/preferences/tracked/pref_hash_filter.h
index c5d2d7aa64148..20790fcb79d06 100644
--- a/services/preferences/tracked/pref_hash_filter.h
+++ b/services/preferences/tracked/pref_hash_filter.h
@@ -33,12 +33,6 @@ class DictionaryValue;
 class Time;
 }  // namespace base
 
-namespace prefs {
-namespace mojom {
-class TrackedPreferenceValidationDelegate;
-}
-}
-
 namespace user_prefs {
 class PrefRegistrySyncable;
 }  // namespace user_prefs
@@ -67,9 +61,6 @@ class PrefHashFilter : public InterceptablePrefFilter {
           tracked_preferences,
       mojo::PendingRemote<prefs::mojom::ResetOnLoadObserver>
           reset_on_load_observer,
-      scoped_refptr<base::RefCountedData<
-          mojo::Remote<prefs::mojom::TrackedPreferenceValidationDelegate>>>
-          delegate,
       size_t reporting_ids_count);
 
   ~PrefHashFilter() override;
@@ -146,9 +137,6 @@ class PrefHashFilter : public InterceptablePrefFilter {
 
   // Notified if a reset occurs in a call to FilterOnLoad.
   mojo::Remote<prefs::mojom::ResetOnLoadObserver> reset_on_load_observer_;
-  scoped_refptr<base::RefCountedData<
-      mojo::Remote<prefs::mojom::TrackedPreferenceValidationDelegate>>>
-      delegate_;
 
   TrackedPreferencesMap tracked_paths_;
 
