diff --git a/services/network/public/cpp/BUILD.gn b/services/network/public/cpp/BUILD.gn
index d66d36e59a30..412b3a2f41c6 100644
--- a/services/network/public/cpp/BUILD.gn
+++ b/services/network/public/cpp/BUILD.gn
@@ -173,7 +173,6 @@ component("cookies_mojom_support") {
     "site_for_cookies_mojom_traits.h",
   ]
   deps = [
-    ":crash_keys",
     "//net",
     "//services/network/public/mojom:cookies_mojom_shared",
   ]
@@ -242,7 +241,6 @@ component("cpp_base") {
 
   public_deps = [
     ":cookies_mojom_support",
-    ":crash_keys",
     ":ip_address_mojom_support",
     "//services/network/public/mojom:url_loader_base",
     "//third_party/webrtc_overrides:webrtc_component",
