diff --git a/services/viz/public/cpp/compositing/begin_frame_args_mojom_traits.cc b/services/viz/public/cpp/compositing/begin_frame_args_mojom_traits.cc
index b36230af3f6e..b387968cf109 100644
--- a/services/viz/public/cpp/compositing/begin_frame_args_mojom_traits.cc
+++ b/services/viz/public/cpp/compositing/begin_frame_args_mojom_traits.cc
@@ -5,7 +5,6 @@
 #include "services/viz/public/cpp/compositing/begin_frame_args_mojom_traits.h"
 
 #include "mojo/public/cpp/base/time_mojom_traits.h"
-#include "services/viz/public/cpp/crash_keys.h"
 
 namespace mojo {
 
@@ -66,8 +65,6 @@ bool StructTraits<viz::mojom::BeginFrameAckDataView, viz::BeginFrameAck>::Read(
     viz::mojom::BeginFrameAckDataView data,
     viz::BeginFrameAck* out) {
   if (data.sequence_number() < viz::BeginFrameArgs::kStartingFrameNumber) {
-    viz::SetDeserializationCrashKeyString(
-        "Invalid begin frame ack sequence number");
     return false;
   }
   out->frame_id.source_id = data.source_id();
