diff --git a/services/viz/public/mojom/BUILD.gn b/services/viz/public/mojom/BUILD.gn
index 32c01030d69b..d54452697529 100644
--- a/services/viz/public/mojom/BUILD.gn
+++ b/services/viz/public/mojom/BUILD.gn
@@ -203,7 +203,6 @@ mojom("mojom") {
       ]
       traits_public_deps = [
         "//components/viz/common",
-        "//services/viz/public/cpp:crash_keys",
       ]
     },
     {
@@ -335,7 +334,6 @@ mojom("mojom") {
       traits_sources = [ "//services/viz/public/cpp/hit_test/hit_test_region_list_mojom_traits.cc" ]
       traits_public_deps = [
         "//components/viz/common",
-        "//services/viz/public/cpp:crash_keys",
         "//ui/gfx/geometry/mojom",
       ]
     },
@@ -479,7 +477,6 @@ mojom("mojom") {
       traits_headers = [ "//services/viz/public/cpp/hit_test/hit_test_region_list_mojom_traits.h" ]
       traits_public_deps = [
         "//components/viz/common",
-        "//services/viz/public/cpp:crash_keys",
         "//ui/gfx/geometry/mojom",
       ]
     },
