diff --git a/services/BUILD.gn b/services/BUILD.gn
index 5eed24831daf..8464899388fc 100644
--- a/services/BUILD.gn
+++ b/services/BUILD.gn
@@ -26,7 +26,6 @@ test("services_unittests") {
     "//services/cert_verifier:tests",
     "//services/cert_verifier/cert_net_url_loader:tests",
     "//services/content:tests",
-    "//services/metrics/public/cpp:tests",
     "//services/network:tests",
     "//services/network/public/cpp:tests",
     "//services/test:run_all_unittests",
