diff --git a/services/device/battery/battery_status_service.h b/services/device/battery/battery_status_service.h
index 619e256a0535..b75ff34d0a2e 100644
--- a/services/device/battery/battery_status_service.h
+++ b/services/device/battery/battery_status_service.h
@@ -57,15 +57,10 @@ class BatteryStatusService {
   // Updates current battery status and sends new status to interested
   // render processes. Can be called on any thread via a callback.
   void NotifyConsumers(const mojom::BatteryStatus& status);
-  void NotifyConsumersOnMainThread(const mojom::BatteryStatus& status);
-  void ConsumersChanged();
 
   scoped_refptr<base::SingleThreadTaskRunner> main_thread_task_runner_;
-  std::unique_ptr<BatteryStatusManager> battery_fetcher_;
   BatteryUpdateCallbackList callback_list_;
   BatteryUpdateCallback update_callback_;
-  mojom::BatteryStatus status_;
-  bool status_updated_;
   bool is_shutdown_;
 
   DISALLOW_COPY_AND_ASSIGN(BatteryStatusService);
