diff --git a/services/network/url_loader_factory.cc b/services/network/url_loader_factory.cc
index 1d0354234fad..509e6f69a0dc 100644
--- a/services/network/url_loader_factory.cc
+++ b/services/network/url_loader_factory.cc
@@ -76,7 +76,6 @@ URLLoaderFactory::URLLoaderFactory(
       params_(std::move(params)),
       resource_scheduler_client_(std::move(resource_scheduler_client)),
       header_client_(std::move(params_->header_client)),
-      coep_reporter_(std::move(params_->coep_reporter)),
       cors_url_loader_factory_(cors_url_loader_factory),
       cookie_observer_(std::move(params_->cookie_observer)),
       auth_cert_observer_(std::move(params_->auth_cert_observer)) {
@@ -287,7 +286,7 @@ void URLLoaderFactory::CreateLoaderAndStart(
       std::move(receiver), options, url_request, std::move(client),
       std::move(data_pipe_use_tracker),
       static_cast<net::NetworkTrafficAnnotationTag>(traffic_annotation),
-      params_.get(), coep_reporter_ ? coep_reporter_.get() : nullptr,
+      params_.get(),
       request_id, keepalive_request_size,
       context_->require_network_isolation_key(), resource_scheduler_client_,
       std::move(keepalive_statistics_recorder),
