diff --git a/services/network/cors/cors_url_loader_factory.cc b/services/network/cors/cors_url_loader_factory.cc
index 25386af0f335..f8ede8e99628 100644
--- a/services/network/cors/cors_url_loader_factory.cc
+++ b/services/network/cors/cors_url_loader_factory.cc
@@ -19,7 +19,6 @@
 #include "net/http/http_util.h"
 #include "services/network/cors/cors_url_loader.h"
 #include "services/network/cors/preflight_controller.h"
-#include "services/network/crash_keys.h"
 #include "services/network/cross_origin_read_blocking_exception_for_plugin.h"
 #include "services/network/network_context.h"
 #include "services/network/network_service.h"
@@ -94,7 +93,6 @@ bool VerifyTrustTokenParamsIntegrityIfPresent(
     //
     // TODO(crbug.com/1118183): Re-add a ReportBadMessage (and false return)
     // once the false positives in crbug.com/1117458 have been resolved.
-    base::debug::DumpWithoutCrashing();
     HistogramWhetherTrustTokenFeaturePolicyConsistentWithBrowserOpinion(
         IsConsistent(false));
   } else if (DoesTrustTokenOperationRequireFeaturePolicy(
@@ -255,7 +253,6 @@ void CorsURLLoaderFactory::CreateLoaderAndStart(
     const ResourceRequest& resource_request,
     mojo::PendingRemote<mojom::URLLoaderClient> client,
     const net::MutableNetworkTrafficAnnotationTag& traffic_annotation) {
-  debug::ScopedRequestCrashKeys request_crash_keys(resource_request);
 
   if (!IsValidRequest(resource_request, options)) {
     mojo::Remote<mojom::URLLoaderClient>(std::move(client))
@@ -429,11 +426,6 @@ bool CorsURLLoaderFactory::IsValidRequest(const ResourceRequest& request,
       NOTREACHED();
       if (base::FeatureList::IsEnabled(
               features::kRequestInitiatorSiteLockEnfocement)) {
-        url::debug::ScopedOriginCrashKey initiator_lock_crash_key(
-            debug::GetRequestInitiatorOriginLockCrashKey(),
-            base::OptionalOrNullptr(request_initiator_origin_lock_));
-        mojo::ReportBadMessage(
-            "CorsURLLoaderFactory: lock VS initiator mismatch");
         return false;
       }
       break;
