diff --git a/services/preferences/tracked/tracked_atomic_preference.cc b/services/preferences/tracked/tracked_atomic_preference.cc
index f7252f30f7cb3..ba72dc5b21a9f 100644
--- a/services/preferences/tracked/tracked_atomic_preference.cc
+++ b/services/preferences/tracked/tracked_atomic_preference.cc
@@ -16,15 +16,13 @@ TrackedAtomicPreference::TrackedAtomicPreference(
     size_t reporting_id,
     size_t reporting_ids_count,
     prefs::mojom::TrackedPreferenceMetadata::EnforcementLevel enforcement_level,
-    prefs::mojom::TrackedPreferenceMetadata::ValueType value_type,
-    prefs::mojom::TrackedPreferenceValidationDelegate* delegate)
+    prefs::mojom::TrackedPreferenceMetadata::ValueType value_type)
     : pref_path_(pref_path),
       helper_(pref_path,
               reporting_id,
               reporting_ids_count,
               enforcement_level,
-              value_type),
-      delegate_(delegate) {}
+              value_type) {}
 
 TrackedPreferenceType TrackedAtomicPreference::GetType() const {
   return TrackedPreferenceType::ATOMIC;
@@ -54,11 +52,13 @@ bool TrackedAtomicPreference::EnforceAndReport(
         external_validation_transaction->GetStoreUMASuffix());
   }
 
+#if 0
   if (delegate_) {
     delegate_->OnAtomicPreferenceValidation(
         pref_path_, value ? absl::make_optional(value->Clone()) : absl::nullopt,
         value_state, external_validation_value_state, helper_.IsPersonal());
   }
+#endif
   TrackedPreferenceHelper::ResetAction reset_action =
       helper_.GetAction(value_state);
   helper_.ReportAction(reset_action);
