diff --git a/services/network/sct_auditing_cache.cc b/services/network/sct_auditing_cache.cc
index c6b01f76ab21..094d09c0227b 100644
--- a/services/network/sct_auditing_cache.cc
+++ b/services/network/sct_auditing_cache.cc
@@ -83,8 +83,6 @@ void SCTAuditingCache::MaybeEnqueueReport(
   double sampling_rate = features::kSCTAuditingSamplingRate.Get();
   if (base::RandDouble() > sampling_rate)
     return;
-
-  context->client()->OnSCTReportReady(net::HashValue(cache_key).ToString());
 }
 
 SCTAuditReport* SCTAuditingCache::GetPendingReport(
