diff --git a/services/network/public/cpp/url_request_mojom_traits.cc b/services/network/public/cpp/url_request_mojom_traits.cc
index 8626ff6086fe..c0cb1fccfb2e 100644
--- a/services/network/public/cpp/url_request_mojom_traits.cc
+++ b/services/network/public/cpp/url_request_mojom_traits.cc
@@ -12,7 +12,6 @@
 #include "mojo/public/cpp/base/file_path_mojom_traits.h"
 #include "mojo/public/cpp/base/time_mojom_traits.h"
 #include "mojo/public/cpp/base/unguessable_token_mojom_traits.h"
-#include "services/network/public/cpp/crash_keys.h"
 #include "services/network/public/cpp/http_request_headers_mojom_traits.h"
 #include "services/network/public/cpp/isolation_info_mojom_traits.h"
 #include "services/network/public/cpp/network_ipc_param_traits.h"
@@ -170,7 +169,6 @@ bool StructTraits<
     return false;
   }
   if (!data.ReadUrl(&out->url)) {
-    network::debug::SetDeserializationCrashKeyString("url");
     return false;
   }
   if (!data.ReadSiteForCookies(&out->site_for_cookies) ||
@@ -178,15 +176,12 @@ bool StructTraits<
     return false;
   }
   if (!data.ReadRequestInitiator(&out->request_initiator)) {
-    network::debug::SetDeserializationCrashKeyString("request_initiator");
     return false;
   }
   if (!data.ReadIsolatedWorldOrigin(&out->isolated_world_origin)) {
-    network::debug::SetDeserializationCrashKeyString("isolated_world_origin");
     return false;
   }
   if (!data.ReadReferrer(&out->referrer)) {
-    network::debug::SetDeserializationCrashKeyString("referrer");
     return false;
   }
   if (!data.ReadReferrerPolicy(&out->referrer_policy) ||
@@ -213,7 +208,6 @@ bool StructTraits<
   if (!data.ReadTrustTokenParams(&out->trust_token_params.as_ptr())) {
     // We don't return false here to avoid duplicate reports.
     out->trust_token_params = base::nullopt;
-    base::debug::DumpWithoutCrashing();
   }
 
   out->force_ignore_site_for_cookies = data.force_ignore_site_for_cookies();
@@ -265,7 +259,6 @@ bool StructTraits<network::mojom::URLRequestBodyDataView,
 bool StructTraits<network::mojom::DataElementDataView, network::DataElement>::
     Read(network::mojom::DataElementDataView data, network::DataElement* out) {
   if (!data.ReadPath(&out->path_)) {
-    network::debug::SetDeserializationCrashKeyString("data_element_path");
     return false;
   }
   if (!data.ReadExpectedModificationTime(&out->expected_modification_time_)) {
