diff --git a/third_party/blink/renderer/platform/loader/fetch/resource_loader.cc b/third_party/blink/renderer/platform/loader/fetch/resource_loader.cc
index a184223e52dda..26d1ab597cb13 100644
--- a/third_party/blink/renderer/platform/loader/fetch/resource_loader.cc
+++ b/third_party/blink/renderer/platform/loader/fetch/resource_loader.cc
@@ -37,8 +37,6 @@
 #include "base/metrics/histogram_functions.h"
 #include "base/metrics/histogram_macros.h"
 #include "mojo/public/cpp/bindings/pending_remote.h"
-#include "services/metrics/public/cpp/mojo_ukm_recorder.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
 #include "services/network/public/cpp/cross_origin_embedder_policy.h"
 #include "services/network/public/cpp/features.h"
 #include "services/network/public/mojom/blocked_by_response_reason.mojom-shared.h"
@@ -118,6 +116,7 @@ bool IsThrottlableRequestContext(mojom::blink::RequestContextType context) {
          context != mojom::blink::RequestContextType::AUDIO;
 }
 
+#if 0
 void LogMixedAutoupgradeMetrics(blink::MixedContentAutoupgradeStatus status,
                                 absl::optional<int> response_or_error_code,
                                 ukm::SourceId source_id,
@@ -150,6 +149,7 @@ void LogMixedAutoupgradeMetrics(blink::MixedContentAutoupgradeStatus status,
   }
   builder.Record(recorder);
 }
+#endif
 
 bool CanHandleDataURLRequestLocally(const ResourceRequestHead& request) {
   if (!request.Url().ProtocolIsData())
@@ -526,6 +526,7 @@ void ResourceLoader::Start() {
   }
 
   if (request.IsAutomaticUpgrade()) {
+#if 0
     mojo::PendingRemote<ukm::mojom::UkmRecorderInterface> pending_recorder;
     Platform::Current()->GetBrowserInterfaceBroker()->GetInterface(
         pending_recorder.InitWithNewPipeAndPassReceiver());
@@ -534,6 +535,7 @@ void ResourceLoader::Start() {
     LogMixedAutoupgradeMetrics(MixedContentAutoupgradeStatus::kStarted,
                                absl::nullopt, request.GetUkmSourceId(),
                                recorder.get(), resource_);
+#endif
   }
   if (resource_->GetResourceRequest().IsDownloadToNetworkCacheOnly()) {
     // The download-to-cache requests are throttled in net/, they are fire-and
@@ -939,6 +941,7 @@ void ResourceLoader::DidReceiveResponseInternal(
   }
 
   if (request.IsAutomaticUpgrade()) {
+#if 0
     mojo::PendingRemote<ukm::mojom::UkmRecorderInterface> pending_recorder;
     Platform::Current()->GetBrowserInterfaceBroker()->GetInterface(
         pending_recorder.InitWithNewPipeAndPassReceiver());
@@ -948,6 +951,7 @@ void ResourceLoader::DidReceiveResponseInternal(
                                response.HttpStatusCode(),
                                request.GetUkmSourceId(), recorder.get(),
                                resource_);
+#endif
   }
 
   if (fetcher_->GetProperties().IsDetached()) {
@@ -1223,6 +1227,7 @@ void ResourceLoader::DidFail(const WebURLError& error,
   response_end_time_for_error_cases_ = response_end_time;
 
   if (request.IsAutomaticUpgrade()) {
+#if 0
     mojo::PendingRemote<ukm::mojom::UkmRecorderInterface> pending_recorder;
     Platform::Current()->GetBrowserInterfaceBroker()->GetInterface(
         pending_recorder.InitWithNewPipeAndPassReceiver());
@@ -1231,6 +1236,7 @@ void ResourceLoader::DidFail(const WebURLError& error,
     LogMixedAutoupgradeMetrics(MixedContentAutoupgradeStatus::kFailed,
                                error.reason(), request.GetUkmSourceId(),
                                recorder.get(), resource_);
+#endif
   }
   resource_->SetEncodedDataLength(encoded_data_length);
   resource_->SetEncodedBodyLength(encoded_body_length);
