diff --git a/third_party/blink/renderer/core/html/parser/BUILD.gn b/third_party/blink/renderer/core/html/parser/BUILD.gn
index 24c6c997ae83..77400d72e22a 100644
--- a/third_party/blink/renderer/core/html/parser/BUILD.gn
+++ b/third_party/blink/renderer/core/html/parser/BUILD.gn
@@ -35,8 +35,6 @@ blink_core_sources("parser") {
     "html_meta_charset_parser.h",
     "html_parser_idioms.cc",
     "html_parser_idioms.h",
-    "html_parser_metrics.cc",
-    "html_parser_metrics.h",
     "html_parser_options.cc",
     "html_parser_options.h",
     "html_parser_reentry_permit.cc",
