diff --git a/third_party/blink/renderer/bindings/templates/methods.cc.tmpl b/third_party/blink/renderer/bindings/templates/methods.cc.tmpl
index b1317339a0be..51f85c0e1d5b 100644
--- a/third_party/blink/renderer/bindings/templates/methods.cc.tmpl
+++ b/third_party/blink/renderer/bindings/templates/methods.cc.tmpl
@@ -266,9 +266,6 @@ Document& document = *ToLocalDOMWindow(
 {{method.cpp_type}} impl = {{cpp_value}};
 {% elif method.use_local_result %}
 {{method.cpp_type}} result = {{cpp_value}};
-{% if method.high_entropy == 'Direct' %}
-Dactyloscoper::RecordDirectSurface(execution_context, WebFeature::k{{method.measure_as('Method')}}, result);
-{% endif %}
 {% endif %}
 {# Post-call #}
 {% if method.is_raises_exception %}
@@ -437,9 +434,6 @@ if ({{test}}) {
   {% if method.measure_as and not overloads.measure_all_as %}
   ExecutionContext* execution_context_for_measurement = CurrentExecutionContext(info.GetIsolate());
   UseCounter::Count(execution_context_for_measurement, WebFeature::k{{method.measure_as('Method')}});
-  {% if method.high_entropy %}
-  Dactyloscoper::Record(execution_context_for_measurement, WebFeature::k{{method.measure_as('Method')}});
-  {% endif %}
   {% endif %}
   {% if method.deprecate_as and not overloads.deprecate_all_as %}
   Deprecation::CountDeprecation(CurrentExecutionContext(info.GetIsolate()), WebFeature::k{{method.deprecate_as}});
@@ -464,9 +458,6 @@ void {{v8_class_or_partial}}::{{method.camel_case_name}}MethodCallback{{world_su
   {% if method.measure_as %}
   ExecutionContext* execution_context_for_measurement = CurrentExecutionContext(info.GetIsolate());
   UseCounter::Count(execution_context_for_measurement, WebFeature::k{{method.measure_as('Method')}});
-  {% if method.high_entropy %}
-  Dactyloscoper::Record(execution_context_for_measurement, WebFeature::k{{method.measure_as('Method')}});
-  {% endif %}
   {% endif %}
   {% if method.deprecate_as %}
   Deprecation::CountDeprecation(CurrentExecutionContext(info.GetIsolate()), WebFeature::k{{method.deprecate_as}});
