diff --git a/third_party/blink/renderer/bindings/bindings.gni b/third_party/blink/renderer/bindings/bindings.gni
index deb188d211de..9b9df889a4b5 100644
--- a/third_party/blink/renderer/bindings/bindings.gni
+++ b/third_party/blink/renderer/bindings/bindings.gni
@@ -147,8 +147,6 @@ bindings_core_v8_files =
                     "core/v8/v8_intersection_observer_delegate.h",
                     "core/v8/v8_iterator_result_value.cc",
                     "core/v8/v8_iterator_result_value.h",
-                    "core/v8/v8_metrics.cc",
-                    "core/v8/v8_metrics.h",
                     "core/v8/v8_object_builder.cc",
                     "core/v8/v8_object_builder.h",
                     "core/v8/v8_object_parser.cc",
