diff --git a/third_party/blink/renderer/modules/xr/BUILD.gn b/third_party/blink/renderer/modules/xr/BUILD.gn
index 8ca0b86424dd..83acd9fc02a0 100644
--- a/third_party/blink/renderer/modules/xr/BUILD.gn
+++ b/third_party/blink/renderer/modules/xr/BUILD.gn
@@ -109,7 +109,6 @@ blink_modules_sources("xr") {
 
   deps = [
     "//device/vr/public/mojom:vr_service_blink",
-    "//services/metrics/public/cpp:ukm_builders",
   ]
 
   public_deps = [ "//third_party/blink/renderer/modules/gamepad:gamepad" ]
