diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
index 0a8bfc09b5bba..f01702ceac09c 100644
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -2382,5 +2382,17 @@
       name: "XSLT",
       status: "stable",
     },
+    {
+      name: "FingerprintingClientRectsNoise",
+      status: "stable",
+    },
+    {
+      name: "FingerprintingCanvasMeasureTextNoise",
+      status: "stable",
+    },
+    {
+      name: "FingerprintingCanvasImageDataNoise",
+      status: "stable",
+    },
   ],
 }
