diff --git a/third_party/blink/renderer/core/frame/local_dom_window.cc b/third_party/blink/renderer/core/frame/local_dom_window.cc
index 1b7d52acbd16..db95a4d614c0 100644
--- a/third_party/blink/renderer/core/frame/local_dom_window.cc
+++ b/third_party/blink/renderer/core/frame/local_dom_window.cc
@@ -2059,14 +2059,4 @@ bool LocalDOMWindow::CrossOriginIsolatedCapability() const {
              mojom::blink::FeaturePolicyFeature::kCrossOriginIsolated);
 }
 
-ukm::UkmRecorder* LocalDOMWindow::UkmRecorder() {
-  DCHECK(document_);
-  return document_->UkmRecorder();
-}
-
-ukm::SourceId LocalDOMWindow::UkmSourceID() const {
-  DCHECK(document_);
-  return document_->UkmSourceID();
-}
-
 }  // namespace blink
