diff --git a/third_party/blink/renderer/core/css/css_font_selector.cc b/third_party/blink/renderer/core/css/css_font_selector.cc
index 2c0b865d5f337..4f1e5b3650f70 100644
--- a/third_party/blink/renderer/core/css/css_font_selector.cc
+++ b/third_party/blink/renderer/core/css/css_font_selector.cc
@@ -98,15 +98,11 @@ void CSSFontSelector::FontCacheInvalidated() {
 scoped_refptr<FontData> CSSFontSelector::GetFontData(
     const FontDescription& font_description,
     const AtomicString& family_name) {
-  Document& document = GetTreeScope()->GetDocument();
   if (CSSSegmentedFontFace* face =
           font_face_cache_->Get(font_description, family_name)) {
-    document.GetFontMatchingMetrics()->ReportWebFontFamily(family_name);
     return face->GetFontData(font_description);
   }
 
-  document.GetFontMatchingMetrics()->ReportSystemFontFamily(family_name);
-
   // Try to return the correct font based off our settings, in case we were
   // handed the generic font family name.
   AtomicString settings_family_name = FamilyNameFromSettings(
@@ -114,17 +110,10 @@ scoped_refptr<FontData> CSSFontSelector::GetFontData(
   if (settings_family_name.IsEmpty())
     return nullptr;
 
-  document.GetFontMatchingMetrics()->ReportFontFamilyLookupByGenericFamily(
-      family_name, font_description.GetScript(),
-      font_description.GenericFamily(), settings_family_name);
-
   scoped_refptr<SimpleFontData> font_data =
       FontCache::GetFontCache()->GetFontData(font_description,
                                              settings_family_name);
 
-  document.GetFontMatchingMetrics()->ReportFontLookupByUniqueOrFamilyName(
-      settings_family_name, font_description, font_data.get());
-
   return font_data;
 }
 
@@ -170,39 +159,28 @@ void CSSFontSelector::ReportNotDefGlyph() const {
 void CSSFontSelector::ReportEmojiSegmentGlyphCoverage(
     unsigned num_clusters,
     unsigned num_broken_clusters) {
-  GetDocument().GetFontMatchingMetrics()->ReportEmojiSegmentGlyphCoverage(
-      num_clusters, num_broken_clusters);
 }
 
 void CSSFontSelector::ReportSuccessfulFontFamilyMatch(
     const AtomicString& font_family_name) {
-  GetDocument().GetFontMatchingMetrics()->ReportSuccessfulFontFamilyMatch(
-      font_family_name);
 }
 
 void CSSFontSelector::ReportFailedFontFamilyMatch(
     const AtomicString& font_family_name) {
-  GetDocument().GetFontMatchingMetrics()->ReportFailedFontFamilyMatch(
-      font_family_name);
 }
 
 void CSSFontSelector::ReportSuccessfulLocalFontMatch(
     const AtomicString& font_name) {
-  GetDocument().GetFontMatchingMetrics()->ReportSuccessfulLocalFontMatch(
-      font_name);
 }
 
 void CSSFontSelector::ReportFailedLocalFontMatch(
     const AtomicString& font_name) {
-  GetDocument().GetFontMatchingMetrics()->ReportFailedLocalFontMatch(font_name);
 }
 
 void CSSFontSelector::ReportFontLookupByUniqueOrFamilyName(
     const AtomicString& name,
     const FontDescription& font_description,
     SimpleFontData* resulting_font_data) {
-  GetDocument().GetFontMatchingMetrics()->ReportFontLookupByUniqueOrFamilyName(
-      name, font_description, resulting_font_data);
 }
 
 void CSSFontSelector::ReportFontLookupByUniqueNameOnly(
@@ -210,8 +188,6 @@ void CSSFontSelector::ReportFontLookupByUniqueNameOnly(
     const FontDescription& font_description,
     SimpleFontData* resulting_font_data,
     bool is_loading_fallback) {
-  GetDocument().GetFontMatchingMetrics()->ReportFontLookupByUniqueNameOnly(
-      name, font_description, resulting_font_data, is_loading_fallback);
 }
 
 void CSSFontSelector::ReportFontLookupByFallbackCharacter(
@@ -219,16 +195,11 @@ void CSSFontSelector::ReportFontLookupByFallbackCharacter(
     FontFallbackPriority fallback_priority,
     const FontDescription& font_description,
     SimpleFontData* resulting_font_data) {
-  GetDocument().GetFontMatchingMetrics()->ReportFontLookupByFallbackCharacter(
-      fallback_character, fallback_priority, font_description,
-      resulting_font_data);
 }
 
 void CSSFontSelector::ReportLastResortFallbackFontLookup(
     const FontDescription& font_description,
     SimpleFontData* resulting_font_data) {
-  GetDocument().GetFontMatchingMetrics()->ReportLastResortFallbackFontLookup(
-      font_description, resulting_font_data);
 }
 
 void CSSFontSelector::Trace(Visitor* visitor) const {
