diff --git a/third_party/blink/renderer/core/dom/BUILD.gn b/third_party/blink/renderer/core/dom/BUILD.gn
index 7c33da9b84fa..fe0b01f6c4a9 100644
--- a/third_party/blink/renderer/core/dom/BUILD.gn
+++ b/third_party/blink/renderer/core/dom/BUILD.gn
@@ -325,8 +325,5 @@ blink_core_sources("dom") {
 
   deps = [
     ":dom_hotpath",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
-    "//services/metrics/public/mojom",
   ]
 }
