diff --git a/third_party/blink/renderer/modules/filesystem/dom_window_file_system.cc b/third_party/blink/renderer/modules/filesystem/dom_window_file_system.cc
index 3312630ba15e..406dec2b526e 100644
--- a/third_party/blink/renderer/modules/filesystem/dom_window_file_system.cc
+++ b/third_party/blink/renderer/modules/filesystem/dom_window_file_system.cc
@@ -25,9 +25,6 @@
 
 #include "third_party/blink/renderer/modules/filesystem/dom_window_file_system.h"
 
-#include "services/metrics/public/cpp/mojo_ukm_recorder.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "third_party/blink/public/mojom/filesystem/file_system.mojom-blink.h"
 #include "third_party/blink/renderer/core/dom/dom_exception.h"
 #include "third_party/blink/renderer/core/fileapi/file_error.h"
@@ -77,19 +74,10 @@ void DOMWindowFileSystem::webkitRequestFileSystem(
     return;
   }
 
-  auto* ukm_recorder = window.document()->UkmRecorder();
-  const ukm::SourceId source_id = window.document()->UkmSourceID();
-
   if (file_system_type == mojom::blink::FileSystemType::kTemporary) {
     UseCounter::Count(window, WebFeature::kRequestedFileSystemTemporary);
-    ukm::builders::FileSystemAPI_WebRequest(source_id)
-        .SetTemporary(true)
-        .Record(ukm_recorder->Get());
   } else if (file_system_type == mojom::blink::FileSystemType::kPersistent) {
     UseCounter::Count(window, WebFeature::kRequestedFileSystemPersistent);
-    ukm::builders::FileSystemAPI_WebRequest(source_id)
-        .SetPersistent(true)
-        .Record(ukm_recorder->Get());
   }
 
   auto success_callback_wrapper =
