diff --git a/third_party/blink/renderer/platform/scheduler/BUILD.gn b/third_party/blink/renderer/platform/scheduler/BUILD.gn
index c0e3c87512bd..c764f1095f55 100644
--- a/third_party/blink/renderer/platform/scheduler/BUILD.gn
+++ b/third_party/blink/renderer/platform/scheduler/BUILD.gn
@@ -52,8 +52,6 @@ blink_platform_sources("scheduler") {
     "common/throttling/wake_up_budget_pool.h",
     "common/tracing_helper.cc",
     "common/tracing_helper.h",
-    "common/ukm_task_sampler.cc",
-    "common/ukm_task_sampler.h",
     "common/unprioritized_resource_loading_task_runner_handle.cc",
     "common/unprioritized_resource_loading_task_runner_handle.h",
     "common/web_resource_loading_task_runner_handle.cc",
@@ -168,8 +166,6 @@ blink_platform_sources("scheduler") {
     "//cc",
     "//components/scheduling_metrics",
     "//device/base/synchronization",
-    "//services/metrics/public/cpp:ukm_builders",
-    "//services/metrics/public/mojom",
     "//third_party/blink/renderer/platform:make_platform_generated",
     "//third_party/blink/renderer/platform/heap:heap",
     "//third_party/blink/renderer/platform/instrumentation",
@@ -226,7 +222,6 @@ source_set("unit_tests") {
     "common/throttling/budget_pool_unittest.cc",
     "common/throttling/task_queue_throttler_unittest.cc",
     "common/tracing_helper_unittest.cc",
-    "common/ukm_task_sampler_unittest.cc",
     "common/worker_pool_unittest.cc",
     "main_thread/agent_scheduling_strategy_unittest.cc",
     "main_thread/auto_advancing_virtual_time_domain_unittest.cc",
@@ -251,7 +246,6 @@ source_set("unit_tests") {
   deps = [
     "//base",
     "//base/test:test_support",
-    "//components/ukm:test_support",
     "//testing/gmock",
     "//testing/gtest",
     "//third_party/blink/public:test_support",
