diff --git a/third_party/blink/renderer/core/frame/web_view_frame_widget.cc b/third_party/blink/renderer/core/frame/web_view_frame_widget.cc
index c7a658d5352e..e764351b8673 100644
--- a/third_party/blink/renderer/core/frame/web_view_frame_widget.cc
+++ b/third_party/blink/renderer/core/frame/web_view_frame_widget.cc
@@ -6,7 +6,6 @@
 
 #include "third_party/blink/public/platform/platform.h"
 #include "third_party/blink/renderer/core/exported/web_view_impl.h"
-#include "third_party/blink/renderer/core/frame/local_frame_ukm_aggregator.h"
 #include "third_party/blink/renderer/core/frame/screen_metrics_emulator.h"
 #include "third_party/blink/renderer/core/frame/web_local_frame_impl.h"
 #include "third_party/blink/renderer/platform/scheduler/public/thread.h"
@@ -100,12 +99,6 @@ void WebViewFrameWidget::EndCommitCompositorFrame(
   DCHECK(commit_compositor_frame_start_time_.has_value());
 
   WebFrameWidgetBase::EndCommitCompositorFrame(commit_start_time);
-  web_view_->MainFrameImpl()
-      ->GetFrame()
-      ->View()
-      ->EnsureUkmAggregator()
-      .RecordImplCompositorSample(commit_compositor_frame_start_time_.value(),
-                                  commit_start_time, base::TimeTicks::Now());
   commit_compositor_frame_start_time_.reset();
 }
 
@@ -119,11 +112,6 @@ void WebViewFrameWidget::RecordEndOfFrameMetrics(
   web_view_->RecordEndOfFrameMetrics(frame_begin_time, trackers);
 }
 
-std::unique_ptr<cc::BeginMainFrameMetrics>
-WebViewFrameWidget::GetBeginMainFrameMetrics() {
-  return web_view_->GetBeginMainFrameMetrics();
-}
-
 void WebViewFrameWidget::UpdateLifecycle(WebLifecycleUpdate requested_update,
                                          DocumentUpdateReason reason) {
   web_view_->UpdateLifecycle(requested_update, reason);
