diff --git a/third_party/blink/renderer/platform/widget/widget_base.cc b/third_party/blink/renderer/platform/widget/widget_base.cc
index 59a70580ded0..feeab1a7cff4 100644
--- a/third_party/blink/renderer/platform/widget/widget_base.cc
+++ b/third_party/blink/renderer/platform/widget/widget_base.cc
@@ -9,7 +9,6 @@
 #include "build/build_config.h"
 #include "cc/trees/layer_tree_host.h"
 #include "cc/trees/layer_tree_settings.h"
-#include "cc/trees/ukm_manager.h"
 #include "mojo/public/cpp/bindings/pending_associated_receiver.h"
 #include "mojo/public/cpp/bindings/pending_associated_remote.h"
 #include "third_party/blink/public/common/features.h"
@@ -125,7 +124,6 @@ void WidgetBase::InitializeCompositing(
     cc::TaskGraphRunner* task_graph_runner,
     bool for_child_local_root_frame,
     const ScreenInfo& screen_info,
-    std::unique_ptr<cc::UkmRecorderFactory> ukm_recorder_factory,
     const cc::LayerTreeSettings* settings) {
   scoped_refptr<base::SingleThreadTaskRunner> main_thread_task_runner =
       main_thread_scheduler->CompositorTaskRunner();
@@ -149,7 +147,7 @@ void WidgetBase::InitializeCompositing(
     settings = &default_settings.value();
   }
   screen_info_ = screen_info;
-  layer_tree_view_->Initialize(*settings, std::move(ukm_recorder_factory));
+  layer_tree_view_->Initialize(*settings);
 
   FrameWidget* frame_widget = client_->FrameWidget();
 
