diff --git a/third_party/blink/renderer/modules/encryptedmedia/media_key_system_access.cc b/third_party/blink/renderer/modules/encryptedmedia/media_key_system_access.cc
index 61ff322aab2a..205762afd574 100644
--- a/third_party/blink/renderer/modules/encryptedmedia/media_key_system_access.cc
+++ b/third_party/blink/renderer/modules/encryptedmedia/media_key_system_access.cc
@@ -9,8 +9,6 @@
 #include "base/macros.h"
 #include "base/memory/ptr_util.h"
 #include "media/base/eme_constants.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
 #include "third_party/blink/public/platform/web_content_decryption_module.h"
 #include "third_party/blink/public/platform/web_encrypted_media_types.h"
 #include "third_party/blink/public/platform/web_media_key_system_configuration.h"
@@ -149,14 +147,6 @@ void ReportMetrics(ExecutionContext* execution_context,
   LocalFrame* frame = document->GetFrame();
   if (!frame)
     return;
-
-  ukm::builders::Media_EME_CreateMediaKeys builder(document->UkmSourceID());
-  builder.SetKeySystem(KeySystemForUkm::kWidevine);
-  builder.SetIsAdFrame(
-      static_cast<int>(frame->IsAdRoot() || frame->IsAdSubframe()));
-  builder.SetIsCrossOrigin(static_cast<int>(frame->IsCrossOriginToMainFrame()));
-  builder.SetIsTopFrame(static_cast<int>(frame->IsMainFrame()));
-  builder.Record(document->UkmRecorder());
 }
 
 }  // namespace
