diff --git a/third_party/blink/renderer/platform/scheduler/main_thread/page_scheduler_impl.cc b/third_party/blink/renderer/platform/scheduler/main_thread/page_scheduler_impl.cc
index c1b6afdf5ac6..e4fb148dc8b5 100644
--- a/third_party/blink/renderer/platform/scheduler/main_thread/page_scheduler_impl.cc
+++ b/third_party/blink/renderer/platform/scheduler/main_thread/page_scheduler_impl.cc
@@ -1033,14 +1033,6 @@ PageSchedulerImpl::PageLifecycleStateTracker::
   }
 }
 
-FrameSchedulerImpl* PageSchedulerImpl::SelectFrameForUkmAttribution() {
-  for (FrameSchedulerImpl* frame_scheduler : frame_schedulers_) {
-    if (frame_scheduler->GetUkmRecorder())
-      return frame_scheduler;
-  }
-  return nullptr;
-}
-
 WebScopedVirtualTimePauser PageSchedulerImpl::CreateWebScopedVirtualTimePauser(
     const String& name,
     WebScopedVirtualTimePauser::VirtualTaskDuration duration) {
