diff --git a/third_party/blink/common/BUILD.gn b/third_party/blink/common/BUILD.gn
index 7bdd25267f93..75dff46a72b0 100644
--- a/third_party/blink/common/BUILD.gn
+++ b/third_party/blink/common/BUILD.gn
@@ -123,7 +123,6 @@ source_set("common") {
     "manifest/manifest_icon_selector.cc",
     "manifest/manifest_util.cc",
     "media/watch_time_component.cc",
-    "media/watch_time_reporter.cc",
     "mediastream/media_devices.cc",
     "mediastream/media_devices_mojom_traits.cc",
     "mediastream/media_stream_controls.cc",
@@ -145,7 +144,6 @@ source_set("common") {
     "notifications/platform_notification_data.cc",
     "origin_trials/trial_token.cc",
     "origin_trials/trial_token_validator.cc",
-    "page/content_to_visible_time_reporter.cc",
     "page/drag_mojom_traits.cc",
     "page/page_zoom.cc",
     "page_state/page_state.cc",
@@ -178,7 +176,6 @@ source_set("common") {
   public_deps = [
     ":make_generated_document_policy_features",
     ":make_generated_feature_policy_features",
-    "//third_party/blink/common/privacy_budget:privacy_budget",
     "//third_party/blink/public/common:common_export",
     "//third_party/blink/public/common:headers",
   ]
@@ -190,9 +187,6 @@ source_set("common") {
     "//media/capture/mojom:video_capture_mojom_support",
     "//mojo/public/cpp/system",
     "//net",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
-    "//services/metrics/public/mojom:mojom",
     "//services/network/public/cpp:cpp",
     "//ui/base:base",
     "//ui/display",
@@ -225,10 +219,6 @@ test("blink_common_unittests") {
   visibility = [ "*" ]
   deps = [
     ":common_unittests_sources",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
-    "//services/metrics/public/mojom:mojom",
-    "//third_party/blink/common/privacy_budget:unit_tests",
     "//third_party/blink/public/common",
     "//ui/gfx:test_support",
   ]
@@ -269,7 +259,6 @@ source_set("common_unittests_sources") {
     "notifications/notification_mojom_traits_unittest.cc",
     "origin_trials/trial_token_unittest.cc",
     "origin_trials/trial_token_validator_unittest.cc",
-    "page/content_to_visible_time_reporter_unittest.cc",
     "page_state/page_state_serialization_unittest.cc",
     "service_worker/service_worker_scope_match_unittest.cc",
     "test/run_all_unittests.cc",
