diff --git a/third_party/blink/renderer/modules/canvas/canvas2d/canvas_gradient.h b/third_party/blink/renderer/modules/canvas/canvas2d/canvas_gradient.h
index 6da679686115..a3c94176ae44 100644
--- a/third_party/blink/renderer/modules/canvas/canvas2d/canvas_gradient.h
+++ b/third_party/blink/renderer/modules/canvas/canvas2d/canvas_gradient.h
@@ -27,7 +27,6 @@
 #ifndef THIRD_PARTY_BLINK_RENDERER_MODULES_CANVAS_CANVAS2D_CANVAS_GRADIENT_H_
 #define THIRD_PARTY_BLINK_RENDERER_MODULES_CANVAS_CANVAS2D_CANVAS_GRADIENT_H_
 
-#include "third_party/blink/renderer/modules/canvas/canvas2d/identifiability_study_helper.h"
 #include "third_party/blink/renderer/modules/modules_export.h"
 #include "third_party/blink/renderer/platform/bindings/script_wrappable.h"
 #include "third_party/blink/renderer/platform/graphics/gradient.h"
@@ -56,11 +55,8 @@ class MODULES_EXPORT CanvasGradient final : public ScriptWrappable {
 
   void addColorStop(double value, const String& color, ExceptionState&);
 
-  IdentifiableToken GetIdentifiableToken() const;
-
  private:
   scoped_refptr<Gradient> gradient_;
-  IdentifiabilityStudyHelper identifiability_study_helper_;
 };
 
 }  // namespace blink
