diff --git a/third_party/blink/renderer/core/page/scrolling/text_fragment_selector_generator.cc b/third_party/blink/renderer/core/page/scrolling/text_fragment_selector_generator.cc
index e3c98ded9137..b6f13b8323fa 100644
--- a/third_party/blink/renderer/core/page/scrolling/text_fragment_selector_generator.cc
+++ b/third_party/blink/renderer/core/page/scrolling/text_fragment_selector_generator.cc
@@ -365,9 +365,6 @@ void TextFragmentSelectorGenerator::NotifySelectorReady(
       "SharedHighlights.LinkGenerated",
       selector.Type() != TextFragmentSelector::SelectorType::kInvalid);
 
-  ukm::UkmRecorder* recorder = selection_frame_->GetDocument()->UkmRecorder();
-  ukm::SourceId source_id = selection_frame_->GetDocument()->UkmSourceID();
-
   if (selector.Type() != TextFragmentSelector::SelectorType::kInvalid) {
     UMA_HISTOGRAM_COUNTS_1000("SharedHighlights.LinkGenerated.ParamLength",
                               selector.ToString().length());
@@ -380,8 +377,6 @@ void TextFragmentSelectorGenerator::NotifySelectorReady(
     UMA_HISTOGRAM_ENUMERATION(
         "SharedHighlights.LinkGenerated.SelectorParameters",
         TextFragmentAnchorMetrics::GetParametersForSelector(selector));
-
-    shared_highlighting::LogLinkGeneratedSuccessUkmEvent(recorder, source_id);
   } else {
     UMA_HISTOGRAM_EXACT_LINEAR(
         "SharedHighlights.LinkGenerated.Error.Iterations", iteration_,
@@ -389,12 +384,6 @@ void TextFragmentSelectorGenerator::NotifySelectorReady(
     UMA_HISTOGRAM_TIMES("SharedHighlights.LinkGenerated.Error.TimeToGenerate",
                         base::DefaultTickClock::GetInstance()->NowTicks() -
                             generation_start_time_);
-
-    LinkGenerationError error =
-        error_.has_value() ? error_.value() : LinkGenerationError::kUnknown;
-    shared_highlighting::LogLinkGenerationErrorReason(error);
-    shared_highlighting::LogLinkGeneratedErrorUkmEvent(recorder, source_id,
-                                                       error);
   }
 
   std::move(pending_generate_selector_callback_).Run(selector.ToString());
