diff --git a/third_party/blink/renderer/platform/wtf/BUILD.gn b/third_party/blink/renderer/platform/wtf/BUILD.gn
index 661dd85c715e..5ffe89d2fd2c 100644
--- a/third_party/blink/renderer/platform/wtf/BUILD.gn
+++ b/third_party/blink/renderer/platform/wtf/BUILD.gn
@@ -220,7 +220,6 @@ component("wtf") {
   public_deps = [
     "//base",
     "//base/third_party/double_conversion",
-    "//components/crash/core/common:crash_key",
     "//third_party/icu",
   ]
 
