diff --git a/third_party/blink/renderer/core/frame/deprecation.cc b/third_party/blink/renderer/core/frame/deprecation.cc
index 3b3506c29acf..07fb652bf2a7 100644
--- a/third_party/blink/renderer/core/frame/deprecation.cc
+++ b/third_party/blink/renderer/core/frame/deprecation.cc
@@ -7,16 +7,12 @@
 #include "base/time/time.h"
 #include "build/build_config.h"
 #include "third_party/blink/public/mojom/feature_policy/feature_policy.mojom-blink.h"
-#include "third_party/blink/public/mojom/reporting/reporting.mojom-blink.h"
 #include "third_party/blink/public/platform/platform.h"
 #include "third_party/blink/renderer/core/execution_context/execution_context.h"
-#include "third_party/blink/renderer/core/frame/deprecation_report_body.h"
 #include "third_party/blink/renderer/core/frame/frame_console.h"
 #include "third_party/blink/renderer/core/frame/local_dom_window.h"
 #include "third_party/blink/renderer/core/frame/local_frame.h"
 #include "third_party/blink/renderer/core/frame/local_frame_client.h"
-#include "third_party/blink/renderer/core/frame/report.h"
-#include "third_party/blink/renderer/core/frame/reporting_context.h"
 #include "third_party/blink/renderer/core/inspector/console_message.h"
 #include "third_party/blink/renderer/core/loader/document_loader.h"
 #include "third_party/blink/renderer/core/origin_trials/origin_trial_context.h"
@@ -637,22 +633,6 @@ DeprecationInfo GetDeprecationInfo(WebFeature feature) {
   }
 }
 
-Report* CreateReportInternal(const KURL& context_url,
-                             const DeprecationInfo& info) {
-  base::Optional<base::Time> optional_removal_date;
-  if (info.anticipated_removal != kUnknown) {
-    base::Time removal_date;
-    bool result = base::Time::FromUTCExploded(
-        MilestoneDate(info.anticipated_removal), &removal_date);
-    DCHECK(result);
-    optional_removal_date = removal_date;
-  }
-  DeprecationReportBody* body = MakeGarbageCollected<DeprecationReportBody>(
-      info.id, optional_removal_date, info.message);
-  return MakeGarbageCollected<Report>(ReportType::kDeprecation, context_url,
-                                      body);
-}
-
 }  // anonymous namespace
 
 Deprecation::Deprecation() : mute_count_(0) {}
@@ -772,12 +752,6 @@ void Deprecation::CountDeprecation(ExecutionContext* context,
       mojom::ConsoleMessageSource::kDeprecation,
       mojom::ConsoleMessageLevel::kWarning, info.message);
   context->AddConsoleMessage(console_message);
-
-  Report* report = CreateReportInternal(context->Url(), info);
-
-  // Send the deprecation report to the Reporting API and any
-  // ReportingObservers.
-  ReportingContext::From(context)->QueueReport(report);
 }
 
 // static
