diff --git a/third_party/blink/renderer/modules/peerconnection/BUILD.gn b/third_party/blink/renderer/modules/peerconnection/BUILD.gn
index ac8dae244ed34..5d2c313c2ed7e 100644
--- a/third_party/blink/renderer/modules/peerconnection/BUILD.gn
+++ b/third_party/blink/renderer/modules/peerconnection/BUILD.gn
@@ -25,8 +25,6 @@ blink_modules_sources("peerconnection") {
     "byte_buffer_queue.h",
     "call_setup_state_tracker.cc",
     "call_setup_state_tracker.h",
-    "identifiability_metrics.cc",
-    "identifiability_metrics.h",
     "media_stream_remote_video_source.cc",
     "media_stream_remote_video_source.h",
     "media_stream_track_metrics.cc",
