diff --git a/third_party/crashpad/crashpad/handler/handler_main.cc b/third_party/crashpad/crashpad/handler/handler_main.cc
index 737ee4e09f67..fd69a4ed895e 100644
--- a/third_party/crashpad/crashpad/handler/handler_main.cc
+++ b/third_party/crashpad/crashpad/handler/handler_main.cc
@@ -44,7 +44,6 @@
 #include "client/crashpad_info.h"
 #include "client/prune_crash_reports.h"
 #include "client/simple_string_dictionary.h"
-#include "handler/crash_report_upload_thread.h"
 #include "handler/prune_crash_reports_thread.h"
 #include "tools/tool_support.h"
 #include "util/file/file_io.h"
@@ -971,21 +970,6 @@ int HandlerMain(int argc,
   }
 
   ScopedStoppable upload_thread;
-  if (!options.url.empty()) {
-    // TODO(scottmg): options.rate_limit should be removed when we have a
-    // configurable database setting to control upload limiting.
-    // See https://crashpad.chromium.org/bug/23.
-    CrashReportUploadThread::Options upload_thread_options;
-    upload_thread_options.identify_client_via_url =
-        options.identify_client_via_url;
-    upload_thread_options.rate_limit = options.rate_limit;
-    upload_thread_options.upload_gzip = options.upload_gzip;
-    upload_thread_options.watch_pending_reports = options.periodic_tasks;
-
-    upload_thread.Reset(new CrashReportUploadThread(
-        database.get(), options.url, upload_thread_options));
-    upload_thread.Get()->Start();
-  }
 
 #if defined(OS_LINUX) || defined(OS_CHROMEOS) || defined(OS_ANDROID)
   std::unique_ptr<ExceptionHandlerServer::Delegate> exception_handler;
@@ -1022,7 +1006,6 @@ int HandlerMain(int argc,
 #else
   exception_handler = std::make_unique<CrashReportExceptionHandler>(
       database.get(),
-      static_cast<CrashReportUploadThread*>(upload_thread.Get()),
       &options.annotations,
 #if defined(ATTACHMENTS_SUPPORTED)
       &options.attachments,
