diff --git a/third_party/blink/renderer/core/html/media/autoplay_uma_helper.cc b/third_party/blink/renderer/core/html/media/autoplay_uma_helper.cc
index 9f4e7f1432192..97b9c890abd82 100644
--- a/third_party/blink/renderer/core/html/media/autoplay_uma_helper.cc
+++ b/third_party/blink/renderer/core/html/media/autoplay_uma_helper.cc
@@ -22,6 +22,7 @@
 
 namespace blink {
 
+#if 0
 namespace {
 
 constexpr base::TimeDelta kMaxOffscreenDurationUma =
@@ -48,6 +49,7 @@ int64_t GetUserGestureStatusForUkmMetric(LocalFrame* frame) {
 }
 
 }  // namespace
+#endif
 
 AutoplayUmaHelper::AutoplayUmaHelper(HTMLMediaElement* element)
     : ExecutionContextLifecycleObserver(
@@ -92,6 +94,7 @@ void AutoplayUmaHelper::OnAutoplayInitiated(AutoplaySource source) {
   DCHECK(frame);
   DCHECK(element_->GetDocument().GetPage());
 
+#if 0
   ukm::UkmRecorder* ukm_recorder = element_->GetDocument().UkmRecorder();
   DCHECK(ukm_recorder);
   ukm::builders::Media_Autoplay_Attempt(element_->GetDocument().UkmSourceID())
@@ -105,6 +108,7 @@ void AutoplayUmaHelper::OnAutoplayInitiated(AutoplaySource source) {
           element_->GetDocument()))
       .SetUserGestureStatus(GetUserGestureStatusForUkmMetric(frame))
       .Record(ukm_recorder);
+#endif
 }
 
 void AutoplayUmaHelper::RecordAutoplayUnmuteStatus(
@@ -121,6 +125,7 @@ void AutoplayUmaHelper::RecordAutoplayUnmuteStatus(
       source = static_cast<int>(AutoplaySource::kAttribute);
     }
 
+#if 0
     ukm::UkmRecorder* ukm_recorder = element_->GetDocument().UkmRecorder();
     DCHECK(ukm_recorder);
     ukm::builders::Media_Autoplay_Muted_UnmuteAction(
@@ -128,6 +133,7 @@ void AutoplayUmaHelper::RecordAutoplayUnmuteStatus(
         .SetSource(source)
         .SetResult(status == AutoplayUnmuteActionStatus::kSuccess)
         .Record(ukm_recorder);
+#endif
   }
 }
 
@@ -263,12 +269,6 @@ void AutoplayUmaHelper::MaybeStopRecordingMutedVideoOffscreenDuration() {
 
   DCHECK(sources_.Contains(AutoplaySource::kMethod));
 
-  UMA_HISTOGRAM_CUSTOM_TIMES(
-      "Media.Video.Autoplay.Muted.PlayMethod.OffscreenDuration",
-      muted_video_autoplay_offscreen_duration_,
-      base::TimeDelta::FromMilliseconds(1), kMaxOffscreenDurationUma,
-      kOffscreenDurationUmaBucketCount);
-
   muted_video_offscreen_duration_intersection_observer_->disconnect();
   muted_video_offscreen_duration_intersection_observer_ = nullptr;
   muted_video_autoplay_offscreen_duration_ = base::TimeDelta();
