diff --git a/third_party/blink/renderer/core/timing/event_timing.cc b/third_party/blink/renderer/core/timing/event_timing.cc
index 3b3b1fa63b94..ad1207b4190e 100644
--- a/third_party/blink/renderer/core/timing/event_timing.cc
+++ b/third_party/blink/renderer/core/timing/event_timing.cc
@@ -116,14 +116,6 @@ void EventTiming::DidDispatchEvent(const Event& event, Document& document) {
   performance_->RegisterEventTiming(event.type(), event_timestamp_,
                                     processing_start_, processing_end,
                                     event.cancelable(), target);
-  if (should_log_event_) {
-    InteractiveDetector* interactive_detector =
-        InteractiveDetector::From(document);
-    if (interactive_detector) {
-      interactive_detector->RecordInputEventTimingUKM(
-          event, event_timestamp_, processing_start_, processing_end);
-    }
-  }
 }
 
 // static
