diff --git a/third_party/blink/renderer/core/dom/document_data.h b/third_party/blink/renderer/core/dom/document_data.h
index 40d00499ab7c2..1be278ebe9221 100644
--- a/third_party/blink/renderer/core/dom/document_data.h
+++ b/third_party/blink/renderer/core/dom/document_data.h
@@ -6,7 +6,6 @@
 #define THIRD_PARTY_BLINK_RENDERER_CORE_DOM_DOCUMENT_DATA_H_
 
 #include "services/network/public/mojom/trust_tokens.mojom-blink.h"
-#include "third_party/blink/public/mojom/federated_learning/floc.mojom-blink.h"
 #include "third_party/blink/public/mojom/permissions/permission.mojom-blink.h"
 #include "third_party/blink/renderer/bindings/core/v8/script_regexp.h"
 #include "third_party/blink/renderer/platform/mojo/heap_mojo_remote.h"
@@ -25,12 +24,10 @@ class DocumentData final : public GarbageCollected<DocumentData> {
  public:
   explicit DocumentData(ExecutionContext* context)
       : permission_service_(context),
-        floc_service_(context),
         has_trust_tokens_answerer_(context) {}
 
   void Trace(Visitor* visitor) const {
     visitor->Trace(permission_service_);
-    visitor->Trace(floc_service_);
     visitor->Trace(has_trust_tokens_answerer_);
     visitor->Trace(pending_has_trust_tokens_resolvers_);
     visitor->Trace(email_regexp_);
@@ -41,9 +38,6 @@ class DocumentData final : public GarbageCollected<DocumentData> {
   // storage or not.
   HeapMojoRemote<mojom::blink::PermissionService> permission_service_;
 
-  // Mojo remote used to query the floc (i.e. interestCohort).
-  HeapMojoRemote<mojom::blink::FlocService> floc_service_;
-
   // Mojo remote used to answer API calls asking whether the user has trust
   // tokens (https://github.com/wicg/trust-token-api). The other endpoint
   // is in the network service, which may crash and restart. To handle this:
