diff --git a/third_party/blink/renderer/platform/widget/widget_base_client.h b/third_party/blink/renderer/platform/widget/widget_base_client.h
index cbc6daa043e8..f9f11320b5cb 100644
--- a/third_party/blink/renderer/platform/widget/widget_base_client.h
+++ b/third_party/blink/renderer/platform/widget/widget_base_client.h
@@ -118,8 +118,7 @@ class WidgetBaseClient {
   virtual void WillBeginMainFrame() {}
   virtual void DidCompletePageScaleAnimation() {}
 
-  virtual void SubmitThroughputData(ukm::SourceId source_id,
-                                    int aggregated_percent,
+  virtual void SubmitThroughputData(int aggregated_percent,
                                     int impl_percent,
                                     base::Optional<int> main_percent) {}
   virtual void FocusChangeComplete() {}
