diff --git a/third_party/blink/renderer/core/frame/build.gni b/third_party/blink/renderer/core/frame/build.gni
index bc6ce85c2dc4f..e9c1dfecdf306 100644
--- a/third_party/blink/renderer/core/frame/build.gni
+++ b/third_party/blink/renderer/core/frame/build.gni
@@ -29,8 +29,6 @@ blink_core_sources_frame = [
   "csp/source_list_directive.h",
   "csp/trusted_types_directive.cc",
   "csp/trusted_types_directive.h",
-  "dactyloscoper.cc",
-  "dactyloscoper.h",
   "frame_content_as_text.h",
   "frame_content_as_text.cc",
   "deprecated_schedule_style_recalc_during_layout.cc",
