diff --git a/third_party/blink/renderer/platform/widget/widget_base.h b/third_party/blink/renderer/platform/widget/widget_base.h
index 53a539f143f5..efae8e135eb3 100644
--- a/third_party/blink/renderer/platform/widget/widget_base.h
+++ b/third_party/blink/renderer/platform/widget/widget_base.h
@@ -12,7 +12,6 @@
 #include "mojo/public/cpp/bindings/associated_receiver.h"
 #include "mojo/public/cpp/bindings/associated_remote.h"
 #include "third_party/blink/public/common/metrics/document_update_reason.h"
-#include "third_party/blink/public/common/page/content_to_visible_time_reporter.h"
 #include "third_party/blink/public/mojom/input/input_handler.mojom-blink.h"
 #include "third_party/blink/public/mojom/page/record_content_to_visible_time_request.mojom-blink-forward.h"
 #include "third_party/blink/public/mojom/page/widget.mojom-blink.h"
@@ -31,7 +30,6 @@ class AnimationHost;
 class LayerTreeHost;
 class LayerTreeSettings;
 class TaskGraphRunner;
-class UkmRecorderFactory;
 }  // namespace cc
 
 namespace ui {
@@ -79,7 +77,6 @@ class PLATFORM_EXPORT WidgetBase : public mojom::blink::Widget,
       cc::TaskGraphRunner* task_graph_runner,
       bool for_child_local_root_frame,
       const ScreenInfo& screen_info,
-      std::unique_ptr<cc::UkmRecorderFactory> ukm_recorder_factory,
       const cc::LayerTreeSettings* settings);
 
   // Shutdown the compositor.
@@ -397,9 +394,6 @@ class PLATFORM_EXPORT WidgetBase : public mojom::blink::Widget,
   // Whether or not this RenderWidget is currently pasting.
   bool is_pasting_ = false;
 
-  // Object to record tab switch time into this RenderWidget
-  ContentToVisibleTimeReporter tab_switch_time_recorder_;
-
   // Properties of the screen hosting the WidgetBase. Rects in this structure
   // do not include any scaling by device scale factor, so are logical pixels
   // not physical device pixels.
