diff --git a/third_party/blink/renderer/platform/scheduler/common/scheduler_helper.h b/third_party/blink/renderer/platform/scheduler/common/scheduler_helper.h
index b12d21eef9fd..8efa65a48856 100644
--- a/third_party/blink/renderer/platform/scheduler/common/scheduler_helper.h
+++ b/third_party/blink/renderer/platform/scheduler/common/scheduler_helper.h
@@ -103,15 +103,8 @@ class PLATFORM_EXPORT SchedulerHelper
   double GetSamplingRateForRecordingCPUTime() const;
   bool HasCPUTimingForEachTask() const;
 
-  bool ShouldRecordTaskUkm(bool task_has_thread_time) {
-    return ukm_task_sampler_.ShouldRecordTaskUkm(task_has_thread_time);
-  }
-
   // Test helpers.
   void SetWorkBatchSizeForTesting(int work_batch_size);
-  void SetUkmTaskSamplingRateForTest(double rate) {
-    ukm_task_sampler_.SetUkmTaskSamplingRate(rate);
-  }
 
  protected:
   void InitDefaultQueues(
@@ -131,7 +124,6 @@ class PLATFORM_EXPORT SchedulerHelper
 
   Observer* observer_;  // NOT OWNED
 
-  UkmTaskSampler ukm_task_sampler_;
   base::Optional<base::SimpleTaskExecutor> simple_task_executor_;
 
   DISALLOW_COPY_AND_ASSIGN(SchedulerHelper);
