diff --git a/third_party/blink/renderer/platform/wtf/BUILD.gn b/third_party/blink/renderer/platform/wtf/BUILD.gn
index f7200b2d4f74..5557eb9d17e5 100644
--- a/third_party/blink/renderer/platform/wtf/BUILD.gn
+++ b/third_party/blink/renderer/platform/wtf/BUILD.gn
@@ -216,7 +216,6 @@ component("wtf") {
   public_deps = [
     "//base",
     "//base/third_party/double_conversion",
-    "//components/crash/core/common:crash_key",
     "//third_party/icu",
   ]
 
