diff --git a/third_party/blink/renderer/core/loader/interactive_detector.h b/third_party/blink/renderer/core/loader/interactive_detector.h
index c465a86bbef8..4e954244e0fb 100644
--- a/third_party/blink/renderer/core/loader/interactive_detector.h
+++ b/third_party/blink/renderer/core/loader/interactive_detector.h
@@ -24,10 +24,6 @@ namespace base {
 class TickClock;
 }  // namespace base
 
-namespace ukm {
-class UkmRecorder;
-}  // namespace ukm
-
 namespace blink {
 
 class Document;
@@ -126,15 +122,6 @@ class CORE_EXPORT InteractiveDetector
   // The caller owns the |clock| which must outlive the InteractiveDetector.
   void SetTickClockForTesting(const base::TickClock* clock);
 
-  ukm::UkmRecorder* GetUkmRecorder() const;
-
-  void SetUkmRecorderForTesting(ukm::UkmRecorder* test_ukm_recorder);
-
-  void RecordInputEventTimingUKM(const Event& event,
-                                 base::TimeTicks event_timestamp,
-                                 base::TimeTicks processing_start,
-                                 base::TimeTicks processing_end);
-
   void DidObserveFirstScrollDelay(base::TimeDelta first_scroll_delay,
                                   base::TimeTicks first_scroll_timestamp);
 
@@ -240,8 +227,6 @@ class CORE_EXPORT InteractiveDetector
   // pending_pointerdown_delay_.
   base::TimeTicks pending_pointerdown_timestamp_;
 
-  ukm::UkmRecorder* ukm_recorder_;
-
   DISALLOW_COPY_AND_ASSIGN(InteractiveDetector);
 };
 
