diff --git a/third_party/crashpad/crashpad/handler/linux/crash_report_exception_handler.h b/third_party/crashpad/crashpad/handler/linux/crash_report_exception_handler.h
index ae2843017129b..bc9eae5ea007a 100644
--- a/third_party/crashpad/crashpad/handler/linux/crash_report_exception_handler.h
+++ b/third_party/crashpad/crashpad/handler/linux/crash_report_exception_handler.h
@@ -65,7 +65,6 @@ class CrashReportExceptionHandler : public ExceptionHandlerServer::Delegate {
   //!     minidump streams. `nullptr` if not required.
   CrashReportExceptionHandler(
       CrashReportDatabase* database,
-      CrashReportUploadThread* upload_thread,
       const std::map<std::string, std::string>* process_annotations,
       const std::vector<base::FilePath>* attachments,
       bool write_minidump_to_database,
@@ -107,7 +106,6 @@ class CrashReportExceptionHandler : public ExceptionHandlerServer::Delegate {
                           ProcessSnapshotSanitized* sanitized_snapshot);
 
   CrashReportDatabase* database_;  // weak
-  CrashReportUploadThread* upload_thread_;  // weak
   const std::map<std::string, std::string>* process_annotations_;  // weak
   const std::vector<base::FilePath>* attachments_;  // weak
   bool write_minidump_to_database_;
