diff --git a/third_party/blink/renderer/modules/service_worker/service_worker_thread.cc b/third_party/blink/renderer/modules/service_worker/service_worker_thread.cc
index 2ab749238df4..04f1d5d828c3 100644
--- a/third_party/blink/renderer/modules/service_worker/service_worker_thread.cc
+++ b/third_party/blink/renderer/modules/service_worker/service_worker_thread.cc
@@ -48,8 +48,7 @@ ServiceWorkerThread::ServiceWorkerThread(
     mojo::PendingRemote<mojom::blink::CacheStorage> cache_storage_remote,
     scoped_refptr<base::SingleThreadTaskRunner>
         parent_thread_default_task_runner,
-    const blink::ServiceWorkerToken& service_worker_token,
-    ukm::SourceId ukm_source_id)
+    const blink::ServiceWorkerToken& service_worker_token)
     : WorkerThread(*global_scope_proxy,
                    std::move(parent_thread_default_task_runner)),
       global_scope_proxy_(std::move(global_scope_proxy)),
@@ -57,8 +56,7 @@ ServiceWorkerThread::ServiceWorkerThread(
           ThreadCreationParams(GetThreadType()))),
       installed_scripts_manager_(std::move(installed_scripts_manager)),
       cache_storage_remote_(std::move(cache_storage_remote)),
-      service_worker_token_(service_worker_token),
-      ukm_source_id_(ukm_source_id) {}
+      service_worker_token_(service_worker_token) {}
 
 ServiceWorkerThread::~ServiceWorkerThread() {
   global_scope_proxy_->Detach();
@@ -77,8 +75,7 @@ WorkerOrWorkletGlobalScope* ServiceWorkerThread::CreateWorkerGlobalScope(
     std::unique_ptr<GlobalScopeCreationParams> creation_params) {
   return ServiceWorkerGlobalScope::Create(
       this, std::move(creation_params), std::move(installed_scripts_manager_),
-      std::move(cache_storage_remote_), time_origin_, service_worker_token_,
-      ukm_source_id_);
+      std::move(cache_storage_remote_), time_origin_, service_worker_token_);
 }
 
 }  // namespace blink
