diff --git a/third_party/blink/renderer/modules/xr/xr_system.h b/third_party/blink/renderer/modules/xr/xr_system.h
index 289801f61202..2b6cfe417e76 100644
--- a/third_party/blink/renderer/modules/xr/xr_system.h
+++ b/third_party/blink/renderer/modules/xr/xr_system.h
@@ -165,8 +165,7 @@ class XRSystem final : public EventTargetWithInlineData,
   class PendingRequestSessionQuery final
       : public GarbageCollected<PendingRequestSessionQuery> {
    public:
-    PendingRequestSessionQuery(int64_t ukm_source_id,
-                               ScriptPromiseResolver* resolver,
+    PendingRequestSessionQuery(ScriptPromiseResolver* resolver,
                                device::mojom::blink::XRSessionMode mode,
                                RequestedXRSessionFeatureSet required_features,
                                RequestedXRSessionFeatureSet optional_features);
@@ -247,8 +246,6 @@ class XRSystem final : public EventTargetWithInlineData,
     RequestedXRSessionFeatureSet optional_features_;
     SensorRequirement sensor_requirement_ = SensorRequirement::kNone;
 
-    const int64_t ukm_source_id_;
-
     Member<Element> dom_overlay_element_;
 
     Vector<device::mojom::blink::XRTrackedImage> tracked_images_;
