diff --git a/third_party/blink/renderer/platform/fonts/font_platform_data.h b/third_party/blink/renderer/platform/fonts/font_platform_data.h
index 78508e79f9eea..91292534d3cdd 100644
--- a/third_party/blink/renderer/platform/fonts/font_platform_data.h
+++ b/third_party/blink/renderer/platform/fonts/font_platform_data.h
@@ -33,7 +33,6 @@
 
 #include "base/memory/scoped_refptr.h"
 #include "build/build_config.h"
-#include "third_party/blink/public/common/privacy_budget/identifiable_token.h"
 #include "third_party/blink/public/platform/web_font_render_style.h"
 #include "third_party/blink/renderer/platform/fonts/font_description.h"
 #include "third_party/blink/renderer/platform/fonts/font_orientation.h"
@@ -135,16 +134,6 @@ class PLATFORM_EXPORT FontPlatformData {
                    float device_scale_factor = 1,
                    const FontDescription* = nullptr) const;
 
-  // Computes a digest from the typeface. The digest only depends on the
-  // underlying font itself, and does not vary by the style (size, weight,
-  // italics, etc). This is aimed at discovering the fingerprinting information
-  // a particular local font may provide websites.
-  //
-  // The digest algorithm is designed for fast computation, rather than to be
-  // robust against an attacker with control of local fonts looking to attack
-  // the fingerprinting algorithm.
-  IdentifiableToken ComputeTypefaceDigest() const;
-
   // Gets the postscript name from the typeface.
   String GetPostScriptName() const;
 
