diff --git a/third_party/blink/renderer/modules/mediastream/BUILD.gn b/third_party/blink/renderer/modules/mediastream/BUILD.gn
index 4ad4ee6632b6d..0e9d51f9d863b 100644
--- a/third_party/blink/renderer/modules/mediastream/BUILD.gn
+++ b/third_party/blink/renderer/modules/mediastream/BUILD.gn
@@ -15,8 +15,6 @@ blink_modules_sources("mediastream") {
     "frame_queue_transferring_optimizer.h",
     "frame_queue_underlying_source.cc",
     "frame_queue_underlying_source.h",
-    "identifiability_metrics.cc",
-    "identifiability_metrics.h",
     "input_device_info.cc",
     "input_device_info.h",
     "local_media_stream_audio_source.cc",
