diff --git a/third_party/blink/public/web/web_navigation_params.h b/third_party/blink/public/web/web_navigation_params.h
index 122b5e742309..b957e4e824d9 100644
--- a/third_party/blink/public/web/web_navigation_params.h
+++ b/third_party/blink/public/web/web_navigation_params.h
@@ -389,10 +389,6 @@ struct BLINK_EXPORT WebNavigationParams {
   // computation in the document.
   WebURL web_bundle_claimed_url;
 
-  // UKM source id to be associated with the Document that will be installed
-  // in the current frame.
-  ukm::SourceId document_ukm_source_id = ukm::kInvalidSourceId;
-
   // The frame policy specified by the frame owner element.
   // Should be base::nullopt for top level navigations
   base::Optional<FramePolicy> frame_policy;
