diff --git a/third_party/blink/public/mojom/loader/previews_resource_loading_hints.mojom b/third_party/blink/public/mojom/loader/previews_resource_loading_hints.mojom
index ac066d0843a4..0f11351abdb5 100644
--- a/third_party/blink/public/mojom/loader/previews_resource_loading_hints.mojom
+++ b/third_party/blink/public/mojom/loader/previews_resource_loading_hints.mojom
@@ -57,7 +57,7 @@ interface PreviewsResourceLoadingHintsReceiver {
 
   // Sends the hints for reducing the data-costs of streaming
   // media/videos from the browser to renderers.
-  SetLiteVideoHint(LiteVideoHint lite_video_hint);
+  // SetLiteVideoHint(LiteVideoHint lite_video_hint);
 
   // Sends the hints for Blink.
   // TODO(https://crbug.com/1113980): Move this to its own mojo plumbing.
