diff --git a/third_party/blink/renderer/modules/media_capabilities/BUILD.gn b/third_party/blink/renderer/modules/media_capabilities/BUILD.gn
index 4af36e9fbc54..75c6395015a2 100644
--- a/third_party/blink/renderer/modules/media_capabilities/BUILD.gn
+++ b/third_party/blink/renderer/modules/media_capabilities/BUILD.gn
@@ -10,8 +10,6 @@ blink_modules_sources("media_capabilities") {
   sources = [
     "media_capabilities.cc",
     "media_capabilities.h",
-    "media_capabilities_identifiability_metrics.cc",
-    "media_capabilities_identifiability_metrics.h",
   ]
   deps = [
     "//media",
