diff --git a/third_party/blink/renderer/modules/credentialmanager/BUILD.gn b/third_party/blink/renderer/modules/credentialmanager/BUILD.gn
index 0b14647807e5..fa7ff18d2e0e 100644
--- a/third_party/blink/renderer/modules/credentialmanager/BUILD.gn
+++ b/third_party/blink/renderer/modules/credentialmanager/BUILD.gn
@@ -18,8 +18,6 @@ blink_modules_sources("credentialmanager") {
     "credential_manager_proxy.h",
     "credential_manager_type_converters.cc",
     "credential_manager_type_converters.h",
-    "credential_metrics.cc",
-    "credential_metrics.h",
     "credentials_container.cc",
     "credentials_container.h",
     "federated_credential.cc",
@@ -35,5 +33,4 @@ blink_modules_sources("credentialmanager") {
     "scoped_promise_resolver.cc",
     "scoped_promise_resolver.h",
   ]
-  deps = [ "//services/metrics/public/cpp:ukm_builders" ]
 }
