diff --git a/third_party/blink/renderer/modules/webgpu/BUILD.gn b/third_party/blink/renderer/modules/webgpu/BUILD.gn
index a52893f9a326..0bb8627441b6 100644
--- a/third_party/blink/renderer/modules/webgpu/BUILD.gn
+++ b/third_party/blink/renderer/modules/webgpu/BUILD.gn
@@ -81,7 +81,6 @@ blink_modules_sources("webgpu") {
   ]
   deps = [
     "//gpu/command_buffer/client:webgpu_interface",
-    "//services/metrics/public/cpp:ukm_builders",
     "//third_party/dawn/src/dawn:dawn_headers",
   ]
 }
