diff --git a/third_party/blink/renderer/modules/encryptedmedia/media_key_system_access.cc b/third_party/blink/renderer/modules/encryptedmedia/media_key_system_access.cc
index 3ff12c560f673..61c4fe9b99d3d 100644
--- a/third_party/blink/renderer/modules/encryptedmedia/media_key_system_access.cc
+++ b/third_party/blink/renderer/modules/encryptedmedia/media_key_system_access.cc
@@ -9,8 +9,6 @@
 #include "base/macros.h"
 #include "base/memory/ptr_util.h"
 #include "media/base/eme_constants.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
 #include "third_party/blink/public/platform/web_content_decryption_module.h"
 #include "third_party/blink/public/platform/web_encrypted_media_types.h"
 #include "third_party/blink/public/platform/web_media_key_system_configuration.h"
@@ -130,6 +128,7 @@ Vector<String> ConvertSessionTypes(
   return result;
 }
 
+#if 0
 void ReportMetrics(ExecutionContext* execution_context,
                    const String& key_system) {
   // TODO(xhwang): Report other key systems here and for
@@ -157,6 +156,7 @@ void ReportMetrics(ExecutionContext* execution_context,
   builder.SetIsTopFrame(static_cast<int>(frame->IsMainFrame()));
   builder.Record(document->UkmRecorder());
 }
+#endif
 
 }  // namespace
 
@@ -221,8 +221,6 @@ ScriptPromise MediaKeySystemAccess::createMediaKeys(ScriptState* script_state) {
       helper->Result(),
       execution_context->GetTaskRunner(TaskType::kInternalMedia));
 
-  ReportMetrics(execution_context, key_system_);
-
   // 3. Return promise.
   return promise;
 }
