diff --git a/third_party/blink/renderer/core/loader/mixed_content_checker.cc b/third_party/blink/renderer/core/loader/mixed_content_checker.cc
index 25500205c82b..fa9e985d4632 100644
--- a/third_party/blink/renderer/core/loader/mixed_content_checker.cc
+++ b/third_party/blink/renderer/core/loader/mixed_content_checker.cc
@@ -925,7 +925,6 @@ void MixedContentChecker::UpgradeInsecureRequest(
             MixedContentChecker::CreateConsoleMessageAboutFetchAutoupgrade(
                 fetch_client_settings_object->GlobalObjectUrl(),
                 resource_request.Url()));
-        resource_request.SetUkmSourceId(window->document()->UkmSourceID());
         CreateMixedContentIssue(fetch_client_settings_object->GlobalObjectUrl(),
                                 resource_request.Url(), context,
                                 window->document()->GetFrame(),
