diff --git a/third_party/blink/common/media/watch_time_reporter.cc b/third_party/blink/common/media/watch_time_reporter.cc
index 3cf5182ee775..7024240bfcff 100644
--- a/third_party/blink/common/media/watch_time_reporter.cc
+++ b/third_party/blink/common/media/watch_time_reporter.cc
@@ -96,9 +96,6 @@ WatchTimeReporter::WatchTimeReporter(
 
   base::PowerMonitor::AddObserver(this);
 
-  provider->AcquireWatchTimeRecorder(properties_->Clone(),
-                                     recorder_.BindNewPipeAndPassReceiver());
-
   reporting_timer_.SetTaskRunner(task_runner);
 
   base_component_ = CreateBaseComponent();
