diff --git a/third_party/blink/public/mojom/loader/previews_resource_loading_hints.mojom b/third_party/blink/public/mojom/loader/previews_resource_loading_hints.mojom
index ac066d0843a4..404495073c8d 100644
--- a/third_party/blink/public/mojom/loader/previews_resource_loading_hints.mojom
+++ b/third_party/blink/public/mojom/loader/previews_resource_loading_hints.mojom
@@ -12,9 +12,6 @@ import "third_party/blink/public/mojom/optimization_guide/optimization_guide.moj
 // hints are provided to the renderer and aid in optimizing the loading of
 // resources.
 struct PreviewsResourceLoadingHints {
-  // SourceId to use when recording UKM for the resource loading hints. This
-  // is based on the browser NavigationHandle's navigation id.
-  int64 ukm_source_id;
   // List of subresources to block. The list contains the URL patterns of
   // subresources to block.
   // TODO(tbansal): https://crbug.com/856243. Update the mojom message below
@@ -55,10 +52,6 @@ interface PreviewsResourceLoadingHintsReceiver {
   // Sends the public image URL hints from the browser to renderers.
   SetCompressPublicImagesHints(CompressPublicImagesHints images_hints);
 
-  // Sends the hints for reducing the data-costs of streaming
-  // media/videos from the browser to renderers.
-  SetLiteVideoHint(LiteVideoHint lite_video_hint);
-
   // Sends the hints for Blink.
   // TODO(https://crbug.com/1113980): Move this to its own mojo plumbing.
   SetBlinkOptimizationGuideHints(BlinkOptimizationGuideHints hints);
