diff --git a/third_party/blink/renderer/modules/mediastream/BUILD.gn b/third_party/blink/renderer/modules/mediastream/BUILD.gn
index c2884478b0aa2..91b60deef0948 100644
--- a/third_party/blink/renderer/modules/mediastream/BUILD.gn
+++ b/third_party/blink/renderer/modules/mediastream/BUILD.gn
@@ -13,8 +13,6 @@ blink_modules_sources("mediastream") {
     "capture_handle_change_event.cc",
     "capture_handle_change_event.h",
     "dom_window_media_stream.h",
-    "identifiability_metrics.cc",
-    "identifiability_metrics.h",
     "input_device_info.cc",
     "input_device_info.h",
     "local_media_stream_audio_source.cc",
