diff --git a/third_party/blink/renderer/bindings/generated_in_core.gni b/third_party/blink/renderer/bindings/generated_in_core.gni
index c70a2af45861..47e327ca8f5b 100644
--- a/third_party/blink/renderer/bindings/generated_in_core.gni
+++ b/third_party/blink/renderer/bindings/generated_in_core.gni
@@ -51,8 +51,6 @@ generated_callback_function_sources_in_core = [
   "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_no_argument_constructor.h",
   "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_performance_observer_callback.cc",
   "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_performance_observer_callback.h",
-  "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_reporting_observer_callback.cc",
-  "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_reporting_observer_callback.h",
   "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_resize_observer_callback.cc",
   "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_resize_observer_callback.h",
   "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_scroll_state_callback.cc",
@@ -293,8 +291,6 @@ generated_dictionary_sources_in_core = [
   "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_readable_stream_get_reader_options.h",
   "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_readable_writable_pair.cc",
   "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_readable_writable_pair.h",
-  "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_reporting_observer_options.cc",
-  "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_reporting_observer_options.h",
   "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_request_init.cc",
   "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_request_init.h",
   "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_resize_observer_options.cc",
@@ -1063,12 +1059,6 @@ generated_interface_sources_in_core = [
   "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_readable_stream_default_controller.h",
   "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_readable_stream_default_reader.cc",
   "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_readable_stream_default_reader.h",
-  "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_report.cc",
-  "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_report.h",
-  "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_report_body.cc",
-  "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_report_body.h",
-  "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_reporting_observer.cc",
-  "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_reporting_observer.h",
   "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_request.cc",
   "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_request.h",
   "$root_gen_dir/third_party/blink/renderer/bindings/core/v8/v8_resize_observer.cc",
