diff --git a/third_party/blink/renderer/modules/canvas/canvas2d/canvas_gradient.cc b/third_party/blink/renderer/modules/canvas/canvas2d/canvas_gradient.cc
index 7510eb5b5681b..9905acf19ce8f 100644
--- a/third_party/blink/renderer/modules/canvas/canvas2d/canvas_gradient.cc
+++ b/third_party/blink/renderer/modules/canvas/canvas2d/canvas_gradient.cc
@@ -39,8 +39,6 @@ CanvasGradient::CanvasGradient(const FloatPoint& p0, const FloatPoint& p1)
                                  kSpreadMethodPad,
                                  Gradient::ColorInterpolation::kUnpremultiplied,
                                  Gradient::DegenerateHandling::kDisallow)) {
-  identifiability_study_helper_.MaybeUpdateBuilder(
-      CanvasOps::kCreateLinearGradient, p0.X(), p0.Y(), p1.X(), p1.Y());
 }
 
 CanvasGradient::CanvasGradient(const FloatPoint& p0,
@@ -56,8 +54,6 @@ CanvasGradient::CanvasGradient(const FloatPoint& p0,
                                  kSpreadMethodPad,
                                  Gradient::ColorInterpolation::kUnpremultiplied,
                                  Gradient::DegenerateHandling::kDisallow)) {
-  identifiability_study_helper_.MaybeUpdateBuilder(
-      CanvasOps::kCreateRadialGradient, p0.X(), p0.Y(), r0, p1.X(), p1.Y(), r1);
 }
 
 // CanvasRenderingContext2D.createConicGradient only takes one angle argument
@@ -90,14 +86,8 @@ void CanvasGradient::addColorStop(double value,
                                           "') could not be parsed as a color.");
     return;
   }
-  identifiability_study_helper_.MaybeUpdateBuilder(CanvasOps::kAddColorStop,
-                                                   value, color.Rgb());
 
   gradient_->AddColorStop(value, color);
 }
 
-IdentifiableToken CanvasGradient::GetIdentifiableToken() const {
-  return identifiability_study_helper_.GetToken();
-}
-
 }  // namespace blink
