diff --git a/third_party/blink/renderer/core/workers/worker_global_scope.h b/third_party/blink/renderer/core/workers/worker_global_scope.h
index 0a35a7462158b..bedfd56340efb 100644
--- a/third_party/blink/renderer/core/workers/worker_global_scope.h
+++ b/third_party/blink/renderer/core/workers/worker_global_scope.h
@@ -54,7 +54,6 @@ struct BlinkTransferableMessage;
 class ConsoleMessage;
 class FetchClientSettingsObjectSnapshot;
 class FontFaceSet;
-class FontMatchingMetrics;
 class InstalledScriptsManager;
 class OffscreenFontSelector;
 class WorkerResourceTimingNotifier;
@@ -122,7 +121,6 @@ class CORE_EXPORT WorkerGlobalScope
   }
   HttpsState GetHttpsState() const override { return https_state_; }
   scheduler::WorkerScheduler* GetScheduler() final;
-  ukm::UkmRecorder* UkmRecorder() final;
   ScriptWrappable* ToScriptWrappable() final { return this; }
 
   void AddConsoleMessageImpl(ConsoleMessage*, bool discard_duplicates) final;
@@ -222,10 +220,6 @@ class CORE_EXPORT WorkerGlobalScope
   // match the actual worker type.
   virtual WorkerToken GetWorkerToken() const = 0;
 
-  // Tracks and reports metrics of attempted font match attempts (both
-  // successful and not successful) by the worker.
-  FontMatchingMetrics* GetFontMatchingMetrics();
-
   bool IsUrlValid() { return url_.IsValid(); }
 
  protected:
@@ -292,10 +286,6 @@ class CORE_EXPORT WorkerGlobalScope
 
   Member<OffscreenFontSelector> font_selector_;
 
-  // Tracks and reports UKM metrics of the number of attempted font family match
-  // attempts (both successful and not successful) by the worker.
-  std::unique_ptr<FontMatchingMetrics> font_matching_metrics_;
-
   blink::BrowserInterfaceBrokerProxy browser_interface_broker_proxy_;
 
   // State transition about worker top-level script evaluation.
@@ -317,8 +307,6 @@ class CORE_EXPORT WorkerGlobalScope
 
   HttpsState https_state_;
 
-  std::unique_ptr<ukm::UkmRecorder> ukm_recorder_;
-
   // |worker_main_script_load_params_for_modules_| is used to load a root module
   // script for dedicated workers (when PlzDedicatedWorker is enabled) and
   // shared workers.
