diff --git a/third_party/blink/renderer/platform/instrumentation/histogram.cc b/third_party/blink/renderer/platform/instrumentation/histogram.cc
index 0a8270d1352e..a5513761635a 100644
--- a/third_party/blink/renderer/platform/instrumentation/histogram.cc
+++ b/third_party/blink/renderer/platform/instrumentation/histogram.cc
@@ -23,12 +23,10 @@ CustomCountHistogram::CustomCountHistogram(base::HistogramBase* histogram)
     : histogram_(histogram) {}
 
 void CustomCountHistogram::Count(base::HistogramBase::Sample sample) {
-  histogram_->Add(sample);
 }
 
 void CustomCountHistogram::CountMany(base::HistogramBase::Sample sample,
                                      int count) {
-  histogram_->AddCount(sample, count);
 }
 
 void CustomCountHistogram::CountMicroseconds(base::TimeDelta delta) {
