diff --git a/third_party/blink/renderer/platform/graphics/video_frame_submitter.cc b/third_party/blink/renderer/platform/graphics/video_frame_submitter.cc
index 7a51d0953051..6261af5f45c1 100644
--- a/third_party/blink/renderer/platform/graphics/video_frame_submitter.cc
+++ b/third_party/blink/renderer/platform/graphics/video_frame_submitter.cc
@@ -36,8 +36,7 @@ VideoFrameSubmitter::VideoFrameSubmitter(
       resource_provider_(std::move(resource_provider)),
       rotation_(media::VIDEO_ROTATION_0),
       roughness_reporter_(std::make_unique<cc::VideoPlaybackRoughnessReporter>(
-          std::move(roughness_reporting_callback))),
-      frame_trackers_(false, nullptr) {
+          std::move(roughness_reporting_callback))) {
   DETACH_FROM_THREAD(thread_checker_);
 }
 
@@ -66,8 +65,6 @@ void VideoFrameSubmitter::StartRendering() {
   if (compositor_frame_sink_) {
     compositor_frame_sink_->SetNeedsBeginFrame(IsDrivingFrameUpdates());
   }
-
-  frame_trackers_.StartSequence(cc::FrameSequenceTrackerType::kVideo);
 }
 
 void VideoFrameSubmitter::StopRendering() {
@@ -77,8 +74,6 @@ void VideoFrameSubmitter::StopRendering() {
 
   is_rendering_ = false;
 
-  frame_trackers_.StopSequence(cc::FrameSequenceTrackerType::kVideo);
-
   UpdateSubmissionState();
 }
 
@@ -202,10 +197,6 @@ void VideoFrameSubmitter::OnBeginFrame(
 #endif
     if (!presentation_failure &&
         !ignorable_submitted_frames_.contains(pair.key)) {
-      frame_trackers_.NotifyFramePresented(
-          pair.key, gfx::PresentationFeedback(
-                        feedback.timestamp, feedback.interval, feedback.flags));
-
       // We assume that presentation feedback is reliable if
       // 1. (kHWCompletion) OS told us that the frame was shown at that time
       //  or
@@ -223,11 +214,7 @@ void VideoFrameSubmitter::OnBeginFrame(
         "media", "VideoFrameSubmitter", TRACE_ID_LOCAL(pair.key),
         feedback.timestamp);
   }
-  frame_trackers_.NotifyBeginImplFrame(args);
 
-  base::ScopedClosureRunner end_frame(
-      base::BindOnce(&cc::FrameSequenceTrackerCollection::NotifyFrameEnd,
-                     base::Unretained(&frame_trackers_), args, args));
   base::ScopedClosureRunner roughness_processing(
       base::BindOnce(&cc::VideoPlaybackRoughnessReporter::ProcessFrameWindow,
                      base::Unretained(roughness_reporter_.get())));
@@ -237,7 +224,6 @@ void VideoFrameSubmitter::OnBeginFrame(
   viz::BeginFrameAck current_begin_frame_ack(args, false);
   if (args.type == viz::BeginFrameArgs::MISSED || !is_rendering_) {
     compositor_frame_sink_->DidNotProduceFrame(current_begin_frame_ack);
-    frame_trackers_.NotifyImplFrameCausedNoDamage(current_begin_frame_ack);
     return;
   }
 
@@ -249,7 +235,6 @@ void VideoFrameSubmitter::OnBeginFrame(
                                     args.frame_time + args.interval,
                                     args.frame_time + 2 * args.interval)) {
     compositor_frame_sink_->DidNotProduceFrame(current_begin_frame_ack);
-    frame_trackers_.NotifyImplFrameCausedNoDamage(current_begin_frame_ack);
     return;
   }
 
@@ -258,7 +243,6 @@ void VideoFrameSubmitter::OnBeginFrame(
   auto video_frame = video_frame_provider_->GetCurrentFrame();
   if (!SubmitFrame(current_begin_frame_ack, std::move(video_frame))) {
     compositor_frame_sink_->DidNotProduceFrame(current_begin_frame_ack);
-    frame_trackers_.NotifyImplFrameCausedNoDamage(current_begin_frame_ack);
     return;
   }
 
@@ -500,8 +484,6 @@ bool VideoFrameSubmitter::SubmitFrame(
   compositor_frame_sink_->SubmitCompositorFrame(
       child_local_surface_id_allocator_.GetCurrentLocalSurfaceId(),
       std::move(compositor_frame), base::nullopt, 0);
-  frame_trackers_.NotifySubmitFrame(frame_token, false, begin_frame_ack,
-                                    last_begin_frame_args_);
   resource_provider_->ReleaseFrameResources();
 
   waiting_for_compositor_ack_ = true;
@@ -528,8 +510,6 @@ void VideoFrameSubmitter::SubmitEmptyFrame() {
   compositor_frame_sink_->SubmitCompositorFrame(
       child_local_surface_id_allocator_.GetCurrentLocalSurfaceId(),
       std::move(compositor_frame), base::nullopt, 0);
-  frame_trackers_.NotifySubmitFrame(frame_token, false, begin_frame_ack,
-                                    last_begin_frame_args_);
 
   // We don't set |waiting_for_compositor_ack_| here since we want to allow a
   // subsequent real frame to replace it at any time if needed.
