diff --git a/third_party/blink/renderer/core/loader/document_loader.cc b/third_party/blink/renderer/core/loader/document_loader.cc
index e773f04ee2f3..def5b6d2b14d 100644
--- a/third_party/blink/renderer/core/loader/document_loader.cc
+++ b/third_party/blink/renderer/core/loader/document_loader.cc
@@ -248,7 +248,6 @@ struct SameSizeAsDocumentLoader
   const KURL web_bundle_claimed_url;
   ukm::SourceId ukm_source_id;
   UseCounterHelper use_counter;
-  Dactyloscoper dactyloscoper;
   const base::TickClock* clock;
   const Vector<OriginTrialFeature> initiator_origin_trial_features;
   const Vector<String> force_enabled_origin_trials;
