diff --git a/third_party/blink/renderer/modules/encryptedmedia/media_key_system_access_initializer_base.cc b/third_party/blink/renderer/modules/encryptedmedia/media_key_system_access_initializer_base.cc
index 478ceb87ab26..4731347126cb 100644
--- a/third_party/blink/renderer/modules/encryptedmedia/media_key_system_access_initializer_base.cc
+++ b/third_party/blink/renderer/modules/encryptedmedia/media_key_system_access_initializer_base.cc
@@ -6,8 +6,6 @@
 
 #include "base/metrics/histogram_functions.h"
 #include "media/base/eme_constants.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
 #include "third_party/blink/renderer/bindings/modules/v8/v8_media_key_system_media_capability.h"
 #include "third_party/blink/renderer/core/dom/document.h"
 #include "third_party/blink/renderer/core/frame/local_dom_window.h"
@@ -219,20 +217,6 @@ void MediaKeySystemAccessInitializerBase::GenerateWarningAndReportMetrics()
   LocalFrame* frame = GetFrame();
   if (!document || !frame)
     return;
-
-  ukm::builders::Media_EME_RequestMediaKeySystemAccess builder(
-      document->UkmSourceID());
-  builder.SetKeySystem(KeySystemForUkm::kWidevine);
-  builder.SetIsAdFrame(
-      static_cast<int>(frame->IsAdRoot() || frame->IsAdSubframe()));
-  builder.SetIsCrossOrigin(static_cast<int>(frame->IsCrossOriginToMainFrame()));
-  builder.SetIsTopFrame(static_cast<int>(frame->IsMainFrame()));
-  builder.SetVideoCapabilities(static_cast<int>(has_video_capabilities));
-  builder.SetVideoCapabilities_HasEmptyRobustness(
-      static_cast<int>(has_empty_robustness));
-  builder.SetVideoCapabilities_HasHwSecureAllRobustness(
-      static_cast<int>(has_hw_secure_all));
-  builder.Record(document->UkmRecorder());
 }
 
 }  // namespace blink
