diff --git a/third_party/blink/common/BUILD.gn b/third_party/blink/common/BUILD.gn
index 03bcd09f2c546..02ec838b3a01a 100644
--- a/third_party/blink/common/BUILD.gn
+++ b/third_party/blink/common/BUILD.gn
@@ -203,7 +203,6 @@ source_set("common") {
   public_deps = [
     ":make_generated_document_policy_features",
     ":make_generated_permissions_policy_features",
-    "//third_party/blink/common/privacy_budget:privacy_budget",
     "//third_party/blink/public/common:common_export",
     "//third_party/blink/public/common:headers",
   ]
