diff --git a/third_party/blink/renderer/modules/peerconnection/BUILD.gn b/third_party/blink/renderer/modules/peerconnection/BUILD.gn
index 48f247fe314d..b9df6ee8860e 100644
--- a/third_party/blink/renderer/modules/peerconnection/BUILD.gn
+++ b/third_party/blink/renderer/modules/peerconnection/BUILD.gn
@@ -142,7 +142,6 @@ blink_modules_sources("peerconnection") {
   public_deps = [ "//third_party/webrtc_overrides:webrtc_component" ]
   deps = [
     "//jingle:webrtc_glue",
-    "//services/metrics/public/cpp:ukm_builders",
     "//third_party/abseil-cpp:absl",
     "//third_party/blink/renderer/modules/crypto",
     "//third_party/blink/renderer/modules/mediastream",
