diff --git a/third_party/crashpad/crashpad/handler/BUILD.gn b/third_party/crashpad/crashpad/handler/BUILD.gn
index a697b338e8eeb..e135f56798b13 100644
--- a/third_party/crashpad/crashpad/handler/BUILD.gn
+++ b/third_party/crashpad/crashpad/handler/BUILD.gn
@@ -92,10 +92,6 @@ if (crashpad_is_android) {
 
 static_library("common") {
   sources = [
-    "crash_report_upload_thread.cc",
-    "crash_report_upload_thread.h",
-    "minidump_to_upload_parameters.cc",
-    "minidump_to_upload_parameters.h",
   ]
   if (crashpad_is_mac || crashpad_is_ios) {
     sources += [
