diff --git a/third_party/blink/public/common/BUILD.gn b/third_party/blink/public/common/BUILD.gn
index 114300a77ca8..e6d5ce5ee9d5 100644
--- a/third_party/blink/public/common/BUILD.gn
+++ b/third_party/blink/public/common/BUILD.gn
@@ -136,7 +136,6 @@ source_set("headers") {
     "media/display_type.h",
     "media/video_capture.h",
     "media/watch_time_component.h",
-    "media/watch_time_reporter.h",
     "mediastream/media_devices.h",
     "mediastream/media_devices_mojom_traits.h",
     "mediastream/media_stream_controls.h",
