diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
index 9158a16d4e33b..1e5bc7899b638 100644
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -2471,5 +2471,17 @@
       name: "XSLT",
       status: "stable",
     },
+    {
+      name: "FingerprintingClientRectsNoise",
+      status: "stable",
+    },
+    {
+      name: "FingerprintingCanvasMeasureTextNoise",
+      status: "stable",
+    },
+    {
+      name: "FingerprintingCanvasImageDataNoise",
+      status: "stable",
+    },
   ],
 }
