diff --git a/third_party/blink/renderer/platform/BUILD.gn b/third_party/blink/renderer/platform/BUILD.gn
index 145eb0363bf1..30797eda1ed8 100644
--- a/third_party/blink/renderer/platform/BUILD.gn
+++ b/third_party/blink/renderer/platform/BUILD.gn
@@ -606,8 +606,6 @@ component("platform") {
     "fonts/font_height.cc",
     "fonts/font_height.h",
     "fonts/font_invalidation_reason.h",
-    "fonts/font_matching_metrics.cc",
-    "fonts/font_matching_metrics.h",
     "fonts/font_metrics.cc",
     "fonts/font_metrics.h",
     "fonts/font_metrics_override.h",
@@ -821,8 +819,6 @@ component("platform") {
     "graphics/begin_frame_provider.h",
     "graphics/bitmap_image.cc",
     "graphics/bitmap_image.h",
-    "graphics/bitmap_image_metrics.cc",
-    "graphics/bitmap_image_metrics.h",
     "graphics/box_reflection.cc",
     "graphics/box_reflection.h",
     "graphics/canvas_2d_layer_bridge.cc",
@@ -1000,8 +996,6 @@ component("platform") {
     "graphics/graphics_types.cc",
     "graphics/graphics_types.h",
     "graphics/graphics_types_3d.h",
-    "graphics/identifiability_paint_op_digest.cc",
-    "graphics/identifiability_paint_op_digest.h",
     "graphics/image.cc",
     "graphics/image.h",
     "graphics/image_data_buffer.cc",
@@ -1358,8 +1352,6 @@ component("platform") {
     "peerconnection/webrtc_util.h",
     "peerconnection/webrtc_video_track_source.cc",
     "peerconnection/webrtc_video_track_source.h",
-    "privacy_budget/identifiability_digest_helpers.cc",
-    "privacy_budget/identifiability_digest_helpers.h",
     "resolution_units.h",
     "search_engine_utils.cc",
     "search_engine_utils.h",
@@ -1622,7 +1614,6 @@ component("platform") {
     "//media/capture:capture_switches",
     "//media/capture/mojom:video_capture",
     "//mojo/public/cpp/bindings:wtf_support",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/mojom:mojom_shared",
     "//services/service_manager/public/cpp",
     "//services/viz/public/cpp/gpu",
