diff --git a/third_party/crashpad/crashpad/handler/mac/crash_report_exception_handler.h b/third_party/crashpad/crashpad/handler/mac/crash_report_exception_handler.h
index b5a59e43221a..3b2e5ed3dee4 100644
--- a/third_party/crashpad/crashpad/handler/mac/crash_report_exception_handler.h
+++ b/third_party/crashpad/crashpad/handler/mac/crash_report_exception_handler.h
@@ -22,7 +22,6 @@
 
 #include "base/macros.h"
 #include "client/crash_report_database.h"
-#include "handler/crash_report_upload_thread.h"
 #include "handler/user_stream_data_source.h"
 #include "util/mach/exc_server_variants.h"
 
@@ -55,7 +54,6 @@ class CrashReportExceptionHandler final
   //!     minidump streams. `nullptr` if not required.
   CrashReportExceptionHandler(
       CrashReportDatabase* database,
-      CrashReportUploadThread* upload_thread,
       const std::map<std::string, std::string>* process_annotations,
       const UserStreamDataSources* user_stream_data_sources);
 
@@ -83,7 +81,6 @@ class CrashReportExceptionHandler final
 
  private:
   CrashReportDatabase* database_;  // weak
-  CrashReportUploadThread* upload_thread_;  // weak
   const std::map<std::string, std::string>* process_annotations_;  // weak
   const UserStreamDataSources* user_stream_data_sources_;  // weak
 
