diff --git a/third_party/blink/renderer/platform/widget/compositing/test/stub_layer_tree_view_delegate.h b/third_party/blink/renderer/platform/widget/compositing/test/stub_layer_tree_view_delegate.h
index 4e3d04c3d3cc..2e1f1df8e28f 100644
--- a/third_party/blink/renderer/platform/widget/compositing/test/stub_layer_tree_view_delegate.h
+++ b/third_party/blink/renderer/platform/widget/compositing/test/stub_layer_tree_view_delegate.h
@@ -44,10 +44,6 @@ class StubLayerTreeViewDelegate : public LayerTreeViewDelegate {
   void RecordEndOfFrameMetrics(
       base::TimeTicks frame_begin_time,
       cc::ActiveFrameSequenceTrackers trackers) override {}
-  std::unique_ptr<cc::BeginMainFrameMetrics> GetBeginMainFrameMetrics()
-      override {
-    return nullptr;
-  }
   void BeginUpdateLayers() override {}
   void EndUpdateLayers() override {}
   void UpdateVisualState() override {}
