diff --git a/third_party/blink/public/web/modules/media/webmediaplayer_util.h b/third_party/blink/public/web/modules/media/webmediaplayer_util.h
index 47e0a655e734..34ed3536a49b 100644
--- a/third_party/blink/public/web/modules/media/webmediaplayer_util.h
+++ b/third_party/blink/public/web/modules/media/webmediaplayer_util.h
@@ -11,7 +11,6 @@
 #include "media/base/eme_constants.h"
 #include "media/base/pipeline_status.h"
 #include "media/base/ranges.h"
-#include "media/mojo/mojom/media_metrics_provider.mojom-shared.h"
 #include "third_party/blink/public/platform/web_common.h"
 #include "third_party/blink/public/platform/web_encrypted_media_types.h"
 #include "third_party/blink/public/platform/web_media_player.h"
@@ -27,22 +26,12 @@ namespace blink {
 
 class WebLocalFrame;
 
-// Translates a |url| into the appropriate URL scheme.
-BLINK_MODULES_EXPORT media::mojom::MediaURLScheme GetMediaURLScheme(
-    const WebURL& url);
-
 BLINK_MODULES_EXPORT WebTimeRanges
 ConvertToWebTimeRanges(const media::Ranges<base::TimeDelta>& ranges);
 
 BLINK_MODULES_EXPORT WebMediaPlayer::NetworkState PipelineErrorToNetworkState(
     media::PipelineStatus error);
 
-// Report various metrics to UMA.
-BLINK_MODULES_EXPORT void ReportMetrics(WebMediaPlayer::LoadType load_type,
-                                        const WebURL& url,
-                                        const WebLocalFrame& frame,
-                                        media::MediaLog* media_log);
-
 // Wraps a WebSetSinkIdCompleteCallback into a
 // media::OutputDeviceStatusCB and binds it to the current thread
 BLINK_MODULES_EXPORT media::OutputDeviceStatusCB ConvertToOutputDeviceStatusCB(
