diff --git a/third_party/blink/public/mojom/BUILD.gn b/third_party/blink/public/mojom/BUILD.gn
index 1b41669fdb4e..bfa59b1af378 100644
--- a/third_party/blink/public/mojom/BUILD.gn
+++ b/third_party/blink/public/mojom/BUILD.gn
@@ -64,7 +64,6 @@ mojom("mojom_platform") {
     "feature_policy/feature_policy_feature.mojom",
     "feature_policy/policy_disposition.mojom",
     "feature_policy/policy_value.mojom",
-    "federated_learning/floc.mojom",
     "fetch/fetch_api_request.mojom",
     "fetch/fetch_api_response.mojom",
     "file/file_utilities.mojom",
