diff --git a/third_party/blink/renderer/modules/service_worker/cross_origin_resource_policy_checker.h b/third_party/blink/renderer/modules/service_worker/cross_origin_resource_policy_checker.h
index 900b1b8afe68..96b1317fd639 100644
--- a/third_party/blink/renderer/modules/service_worker/cross_origin_resource_policy_checker.h
+++ b/third_party/blink/renderer/modules/service_worker/cross_origin_resource_policy_checker.h
@@ -26,9 +26,7 @@ class CrossOriginResourcePolicyChecker {
  public:
   // |reporter| can be null if reporting is not necessary.
   CrossOriginResourcePolicyChecker(
-      network::CrossOriginEmbedderPolicy policy,
-      mojo::PendingRemote<
-          network::mojom::blink::CrossOriginEmbedderPolicyReporter> reporter);
+      network::CrossOriginEmbedderPolicy policy);
 
   CrossOriginResourcePolicyChecker(const CrossOriginResourcePolicyChecker&) =
       delete;
@@ -44,7 +42,6 @@ class CrossOriginResourcePolicyChecker {
 
  private:
   const network::CrossOriginEmbedderPolicy policy_;
-  mojo::Remote<network::mojom::CrossOriginEmbedderPolicyReporter> reporter_;
 
   base::WeakPtrFactory<CrossOriginResourcePolicyChecker> weak_factory_{this};
 };
