diff --git a/third_party/blink/renderer/modules/service_worker/service_worker_thread.h b/third_party/blink/renderer/modules/service_worker/service_worker_thread.h
index 8be9eba85c65..c8ca932c45e8 100644
--- a/third_party/blink/renderer/modules/service_worker/service_worker_thread.h
+++ b/third_party/blink/renderer/modules/service_worker/service_worker_thread.h
@@ -55,8 +55,7 @@ class MODULES_EXPORT ServiceWorkerThread final : public WorkerThread {
       mojo::PendingRemote<mojom::blink::CacheStorage> cache_storage_remote,
       scoped_refptr<base::SingleThreadTaskRunner>
           parent_thread_default_task_runner,
-      const ServiceWorkerToken& service_worker_token,
-      ukm::SourceId ukm_source_id);
+      const ServiceWorkerToken& service_worker_token);
   ~ServiceWorkerThread() override;
 
   WorkerBackingThread& GetWorkerBackingThread() override {
@@ -82,8 +81,6 @@ class MODULES_EXPORT ServiceWorkerThread final : public WorkerThread {
   mojo::PendingRemote<mojom::blink::CacheStorage> cache_storage_remote_;
 
   const ServiceWorkerToken service_worker_token_;
-
-  const ukm::SourceId ukm_source_id_;
 };
 
 }  // namespace blink
