diff --git a/third_party/blink/renderer/core/exported/web_external_widget_impl.cc b/third_party/blink/renderer/core/exported/web_external_widget_impl.cc
index cdaa34f918dc..0fc3bc0bf8bf 100644
--- a/third_party/blink/renderer/core/exported/web_external_widget_impl.cc
+++ b/third_party/blink/renderer/core/exported/web_external_widget_impl.cc
@@ -5,7 +5,6 @@
 #include "third_party/blink/renderer/core/exported/web_external_widget_impl.h"
 
 #include "cc/trees/layer_tree_host.h"
-#include "cc/trees/ukm_manager.h"
 #include "third_party/blink/public/mojom/input/input_handler.mojom-blink.h"
 #include "third_party/blink/public/platform/scheduler/web_render_widget_scheduling_state.h"
 #include "third_party/blink/renderer/platform/widget/input/widget_input_handler_manager.h"
@@ -48,11 +47,10 @@ cc::LayerTreeHost* WebExternalWidgetImpl::InitializeCompositing(
     cc::TaskGraphRunner* task_graph_runner,
     bool for_child_local_root_frame,
     const ScreenInfo& screen_info,
-    std::unique_ptr<cc::UkmRecorderFactory> ukm_recorder_factory,
     const cc::LayerTreeSettings* settings) {
   widget_base_->InitializeCompositing(
       main_thread_scheduler, task_graph_runner, for_child_local_root_frame,
-      screen_info, std::move(ukm_recorder_factory), settings);
+      screen_info, settings);
   return widget_base_->LayerTreeHost();
 }
 
