diff --git a/third_party/blink/renderer/core/html/BUILD.gn b/third_party/blink/renderer/core/html/BUILD.gn
index efc66908410c..9be28a76ac52 100644
--- a/third_party/blink/renderer/core/html/BUILD.gn
+++ b/third_party/blink/renderer/core/html/BUILD.gn
@@ -32,7 +32,6 @@ blink_core_sources("html") {
     "canvas/image_element_base.h",
     "canvas/text_metrics.cc",
     "canvas/text_metrics.h",
-    "canvas/ukm_parameters.h",
     "collection_items_cache.h",
     "collection_type.h",
     "cross_origin_attribute.cc",
@@ -599,7 +598,6 @@ blink_core_sources("html") {
   ]
 
   deps = [
-    "//services/metrics/public/cpp:metrics_cpp",
     "//skia:skcms",
     "//third_party/blink/public/strings",
     "//third_party/blink/renderer/core/html/parser:parser",
