diff --git a/third_party/blink/renderer/core/frame/local_dom_window.cc b/third_party/blink/renderer/core/frame/local_dom_window.cc
index 58c922f8da2a..40372ee15bc0 100644
--- a/third_party/blink/renderer/core/frame/local_dom_window.cc
+++ b/third_party/blink/renderer/core/frame/local_dom_window.cc
@@ -2061,14 +2061,4 @@ bool LocalDOMWindow::CrossOriginIsolatedCapability() const {
              mojom::blink::FeaturePolicyFeature::kCrossOriginIsolated);
 }
 
-ukm::UkmRecorder* LocalDOMWindow::UkmRecorder() {
-  DCHECK(document_);
-  return document_->UkmRecorder();
-}
-
-ukm::SourceId LocalDOMWindow::UkmSourceID() const {
-  DCHECK(document_);
-  return document_->UkmSourceID();
-}
-
 }  // namespace blink
