diff --git a/third_party/blink/renderer/core/workers/worklet_global_scope.cc b/third_party/blink/renderer/core/workers/worklet_global_scope.cc
index b3a72cb65caad..d0884c6aa1152 100644
--- a/third_party/blink/renderer/core/workers/worklet_global_scope.cc
+++ b/third_party/blink/renderer/core/workers/worklet_global_scope.cc
@@ -5,7 +5,6 @@
 #include "third_party/blink/renderer/core/workers/worklet_global_scope.h"
 
 #include <memory>
-#include "services/metrics/public/cpp/mojo_ukm_recorder.h"
 #include "third_party/blink/public/common/browser_interface_broker_proxy.h"
 #include "third_party/blink/public/common/thread_safe_browser_interface_broker_proxy.h"
 #include "third_party/blink/public/mojom/fetch/fetch_api_request.mojom-blink.h"
@@ -301,6 +300,7 @@ bool WorkletGlobalScope::DirectSocketCapability() const {
   return parent_direct_socket_capability_;
 }
 
+#if 0
 ukm::UkmRecorder* WorkletGlobalScope::UkmRecorder() {
   if (ukm_recorder_)
     return ukm_recorder_.get();
@@ -312,6 +312,7 @@ ukm::UkmRecorder* WorkletGlobalScope::UkmRecorder() {
 
   return ukm_recorder_.get();
 }
+#endif
 
 void WorkletGlobalScope::Trace(Visitor* visitor) const {
   visitor->Trace(frame_);
