diff --git a/third_party/blink/renderer/platform/wtf/BUILD.gn b/third_party/blink/renderer/platform/wtf/BUILD.gn
index c26830abf20c..b1b0fe3b2575 100644
--- a/third_party/blink/renderer/platform/wtf/BUILD.gn
+++ b/third_party/blink/renderer/platform/wtf/BUILD.gn
@@ -209,7 +209,6 @@ component("wtf") {
   public_deps = [
     "//base",
     "//base/third_party/double_conversion",
-    "//components/crash/core/common:crash_key",
     "//third_party/icu",
   ]
 
