diff --git a/third_party/blink/renderer/bindings/idl_in_core.gni b/third_party/blink/renderer/bindings/idl_in_core.gni
index bd98dc8f11909..b34fbe0107b44 100644
--- a/third_party/blink/renderer/bindings/idl_in_core.gni
+++ b/third_party/blink/renderer/bindings/idl_in_core.gni
@@ -163,7 +163,6 @@ static_idl_files_in_core = get_path_info(
           "//third_party/blink/renderer/core/dom/idle_deadline.idl",
           "//third_party/blink/renderer/core/dom/idle_request_callback.idl",
           "//third_party/blink/renderer/core/dom/idle_request_options.idl",
-          "//third_party/blink/renderer/core/dom/interest_cohort.idl",
           "//third_party/blink/renderer/core/dom/iterator.idl",
           "//third_party/blink/renderer/core/dom/mutation_observer.idl",
           "//third_party/blink/renderer/core/dom/mutation_observer_init.idl",
