diff --git a/third_party/blink/public/web/web_document.h b/third_party/blink/public/web/web_document.h
index 8a30460d06ad..539afe38722e 100644
--- a/third_party/blink/public/web/web_document.h
+++ b/third_party/blink/public/web/web_document.h
@@ -32,7 +32,6 @@
 #define THIRD_PARTY_BLINK_PUBLIC_WEB_WEB_DOCUMENT_H_
 
 #include "net/cookies/site_for_cookies.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "services/network/public/mojom/referrer_policy.mojom-shared.h"
 #include "third_party/blink/public/platform/web_security_origin.h"
 #include "third_party/blink/public/platform/web_vector.h"
@@ -86,7 +85,6 @@ class WebDocument : public WebNode {
   BLINK_EXPORT bool IsXHTMLDocument() const;
   BLINK_EXPORT bool IsPluginDocument() const;
   BLINK_EXPORT WebURL BaseURL() const;
-  BLINK_EXPORT ukm::SourceId GetUkmSourceId() const;
 
   // The SiteForCookies is used to compute whether this document
   // appears in a "third-party" context for the purpose of third-party
