diff --git a/third_party/blink/public/common/media/watch_time_component.h b/third_party/blink/public/common/media/watch_time_component.h
index 13e578065141..9d607d663903 100644
--- a/third_party/blink/public/common/media/watch_time_component.h
+++ b/third_party/blink/public/common/media/watch_time_component.h
@@ -12,7 +12,6 @@
 #include "base/time/time.h"
 #include "media/base/timestamp_constants.h"
 #include "media/base/watch_time_keys.h"
-#include "media/mojo/mojom/watch_time_recorder.mojom.h"
 
 namespace blink {
 
@@ -50,8 +49,7 @@ class WatchTimeComponent {
   WatchTimeComponent(T initial_value,
                      std::vector<media::WatchTimeKey> keys_to_finalize,
                      ValueToKeyCB value_to_key_cb,
-                     GetMediaTimeCB get_media_time_cb,
-                     media::mojom::WatchTimeRecorder* recorder);
+                     GetMediaTimeCB get_media_time_cb);
   ~WatchTimeComponent();
 
   // Called when the main WatchTimeReporter timer is started. Reinitializes
@@ -110,7 +108,6 @@ class WatchTimeComponent {
   const std::vector<media::WatchTimeKey> keys_to_finalize_;
   const ValueToKeyCB value_to_key_cb_;
   const GetMediaTimeCB get_media_time_cb_;
-  media::mojom::WatchTimeRecorder* const recorder_;
 
   // The current value which will be used to select keys for reporting WatchTime
   // during the next RecordWatchTime() call.
