diff --git a/third_party/blink/renderer/modules/credentialmanager/BUILD.gn b/third_party/blink/renderer/modules/credentialmanager/BUILD.gn
index 9f9d84c202b6..90a5f88bf03e 100644
--- a/third_party/blink/renderer/modules/credentialmanager/BUILD.gn
+++ b/third_party/blink/renderer/modules/credentialmanager/BUILD.gn
@@ -37,5 +37,4 @@ blink_modules_sources("credentialmanager") {
     "scoped_promise_resolver.cc",
     "scoped_promise_resolver.h",
   ]
-  deps = [ "//services/metrics/public/cpp:ukm_builders" ]
 }
