diff --git a/third_party/blink/renderer/platform/loader/BUILD.gn b/third_party/blink/renderer/platform/loader/BUILD.gn
index 4f8c6195b485..2bcba53ceda0 100644
--- a/third_party/blink/renderer/platform/loader/BUILD.gn
+++ b/third_party/blink/renderer/platform/loader/BUILD.gn
@@ -149,7 +149,6 @@ blink_platform_sources("loader") {
     "//components/link_header_util",
     "//components/web_package",
     "//net",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/cpp",
     "//services/network/public/mojom:mojom_blink",
     "//third_party/blink/renderer/platform/blob:blob",
