diff --git a/third_party/blink/renderer/modules/peerconnection/rtc_peer_connection_controller.h b/third_party/blink/renderer/modules/peerconnection/rtc_peer_connection_controller.h
index f628dbbea6d1..a66a707850b6 100644
--- a/third_party/blink/renderer/modules/peerconnection/rtc_peer_connection_controller.h
+++ b/third_party/blink/renderer/modules/peerconnection/rtc_peer_connection_controller.h
@@ -41,7 +41,6 @@ class RTCPeerConnectionController
   void Trace(Visitor*) const override;
 
  private:
-  bool has_reported_ukm_ = false;
 };
 
 }  // namespace blink
