diff --git a/third_party/blink/common/BUILD.gn b/third_party/blink/common/BUILD.gn
index c373c2cf7ce96..a18574d2bc32f 100644
--- a/third_party/blink/common/BUILD.gn
+++ b/third_party/blink/common/BUILD.gn
@@ -196,7 +196,6 @@ source_set("common") {
   public_deps = [
     ":make_generated_document_policy_features",
     ":make_generated_permissions_policy_features",
-    "//third_party/blink/common/privacy_budget:privacy_budget",
     "//third_party/blink/public/common:common_export",
     "//third_party/blink/public/common:headers",
   ]
