diff --git a/third_party/blink/renderer/modules/webaudio/BUILD.gn b/third_party/blink/renderer/modules/webaudio/BUILD.gn
index 899b871673fe..159e278834e4 100644
--- a/third_party/blink/renderer/modules/webaudio/BUILD.gn
+++ b/third_party/blink/renderer/modules/webaudio/BUILD.gn
@@ -153,7 +153,6 @@ blink_modules_sources("webaudio") {
     sources += [ "cpu/arm/oscillator_kernel_neon.cc" ]
   }
   deps = [
-    "//services/metrics/public/cpp:ukm_builders",
     "//third_party/blink/renderer/modules/mediastream",
   ]
 }
