diff --git a/third_party/blink/renderer/core/workers/dedicated_worker_thread.h b/third_party/blink/renderer/core/workers/dedicated_worker_thread.h
index 8958bdb6f4a5..668f8b0812fc 100644
--- a/third_party/blink/renderer/core/workers/dedicated_worker_thread.h
+++ b/third_party/blink/renderer/core/workers/dedicated_worker_thread.h
@@ -64,7 +64,6 @@ class CORE_EXPORT DedicatedWorkerThread : public WorkerThread {
 
   std::unique_ptr<WorkerBackingThread> worker_backing_thread_;
   DedicatedWorkerObjectProxy& worker_object_proxy_;
-  ukm::SourceId ukm_source_id_;
 };
 
 }  // namespace blink
