diff --git a/third_party/blink/renderer/platform/scheduler/main_thread/page_scheduler_impl.h b/third_party/blink/renderer/platform/scheduler/main_thread/page_scheduler_impl.h
index 27dfaabae38b..639d4804c0ac 100644
--- a/third_party/blink/renderer/platform/scheduler/main_thread/page_scheduler_impl.h
+++ b/third_party/blink/renderer/platform/scheduler/main_thread/page_scheduler_impl.h
@@ -140,14 +140,6 @@ class PLATFORM_EXPORT PageSchedulerImpl : public PageScheduler {
   // frames or pages that are stored in the back-forward cache
   bool has_ipc_detection_enabled_ = false;
 
-  // Generally UKMs are associated with the main frame of a page, but the
-  // implementation allows to request a recorder from any local frame with
-  // the same result (e.g. for OOPIF support), therefore we need to select
-  // any frame here.
-  // Note that selecting main frame doesn't work for OOPIFs where the main
-  // frame it not a local one.
-  FrameSchedulerImpl* SelectFrameForUkmAttribution();
-
   void AsValueInto(base::trace_event::TracedValue* state) const;
 
   base::WeakPtr<PageSchedulerImpl> GetWeakPtr() {
