diff --git a/third_party/blink/renderer/modules/filesystem/BUILD.gn b/third_party/blink/renderer/modules/filesystem/BUILD.gn
index 49d30b126d33..edfe648b40d1 100644
--- a/third_party/blink/renderer/modules/filesystem/BUILD.gn
+++ b/third_party/blink/renderer/modules/filesystem/BUILD.gn
@@ -66,7 +66,6 @@ blink_modules_sources("filesystem") {
 
   deps = [
     "//components/services/filesystem/public/mojom:mojom_blink",
-    "//services/metrics/public/cpp:ukm_builders",
     "//third_party/blink/renderer/platform",
   ]
 }
