diff --git a/third_party/blink/public/mojom/BUILD.gn b/third_party/blink/public/mojom/BUILD.gn
index a96bd8fc65f9..02fcb64ad696 100644
--- a/third_party/blink/public/mojom/BUILD.gn
+++ b/third_party/blink/public/mojom/BUILD.gn
@@ -62,7 +62,6 @@ mojom("mojom_platform") {
     "feature_policy/feature_policy_feature.mojom",
     "feature_policy/policy_disposition.mojom",
     "feature_policy/policy_value.mojom",
-    "federated_learning/floc.mojom",
     "fetch/fetch_api_request.mojom",
     "fetch/fetch_api_response.mojom",
     "file/file_utilities.mojom",
@@ -147,7 +146,6 @@ mojom("mojom_platform") {
     "quota/quota_types.mojom",
     "renderer_preference_watcher.mojom",
     "renderer_preferences.mojom",
-    "reporting/reporting.mojom",
     "screen_enumeration/screen_enumeration.mojom",
     "scroll/scroll_enums.mojom",
     "scroll/scroll_into_view_params.mojom",
