diff --git a/third_party/blink/renderer/platform/loader/BUILD.gn b/third_party/blink/renderer/platform/loader/BUILD.gn
index a13b84af65d2..8063cf3dad1c 100644
--- a/third_party/blink/renderer/platform/loader/BUILD.gn
+++ b/third_party/blink/renderer/platform/loader/BUILD.gn
@@ -142,7 +142,6 @@ blink_platform_sources("loader") {
     ":make_platform_loader_generated_fetch_initiator_type_names",
     "//components/link_header_util",
     "//components/web_package",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/cpp",
     "//services/network/public/mojom:mojom_blink",
     "//third_party/blink/renderer/platform/blob:blob",
