diff --git a/third_party/blink/renderer/modules/xr/BUILD.gn b/third_party/blink/renderer/modules/xr/BUILD.gn
index c933239ebd76..0ef71a362d35 100644
--- a/third_party/blink/renderer/modules/xr/BUILD.gn
+++ b/third_party/blink/renderer/modules/xr/BUILD.gn
@@ -107,7 +107,6 @@ blink_modules_sources("xr") {
 
   deps = [
     "//device/vr/public/mojom:mojom_blink",
-    "//services/metrics/public/cpp:ukm_builders",
   ]
 
   public_deps = [ "//third_party/blink/renderer/modules/gamepad:gamepad" ]
