diff --git a/third_party/blink/renderer/modules/encryptedmedia/BUILD.gn b/third_party/blink/renderer/modules/encryptedmedia/BUILD.gn
index de8b1ffc5578..8aeb0531847a 100644
--- a/third_party/blink/renderer/modules/encryptedmedia/BUILD.gn
+++ b/third_party/blink/renderer/modules/encryptedmedia/BUILD.gn
@@ -33,5 +33,4 @@ blink_modules_sources("encryptedmedia") {
     "navigator_request_media_key_system_access.cc",
     "navigator_request_media_key_system_access.h",
   ]
-  deps = [ "//services/metrics/public/cpp:ukm_builders" ]
 }
