diff --git a/third_party/blink/renderer/bindings/scripts/v8_utilities.py b/third_party/blink/renderer/bindings/scripts/v8_utilities.py
index 2ecd69233208..034ff96987c6 100644
--- a/third_party/blink/renderer/bindings/scripts/v8_utilities.py
+++ b/third_party/blink/renderer/bindings/scripts/v8_utilities.py
@@ -447,7 +447,6 @@ def measure_as(definition_or_member, interface):
 def high_entropy(definition_or_member):
     extended_attributes = definition_or_member.extended_attributes
     if 'HighEntropy' in extended_attributes:
-        includes.add('core/frame/dactyloscoper.h')
         if not ('Measure' in extended_attributes
                 or 'MeasureAs' in extended_attributes):
             raise Exception(
