diff --git a/third_party/blink/renderer/core/execution_context/execution_context.h b/third_party/blink/renderer/core/execution_context/execution_context.h
index 050e047a1636..1b8212a3c000 100644
--- a/third_party/blink/renderer/core/execution_context/execution_context.h
+++ b/third_party/blink/renderer/core/execution_context/execution_context.h
@@ -33,7 +33,6 @@
 
 #include "base/macros.h"
 #include "base/optional.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "services/network/public/mojom/ip_address_space.mojom-blink-forward.h"
 #include "services/network/public/mojom/referrer_policy.mojom-blink-forward.h"
 #include "third_party/blink/public/common/tokens/tokens.h"
@@ -64,10 +63,6 @@ class SingleThreadTaskRunner;
 class UnguessableToken;
 }  // namespace base
 
-namespace ukm {
-class UkmRecorder;
-}  // namespace ukm
-
 namespace blink {
 
 class Agent;
@@ -383,9 +378,6 @@ class CORE_EXPORT ExecutionContext : public Supplementable<ExecutionContext>,
   // to cross-origin isolated contexts.
   bool SharedArrayBufferTransferAllowed() const;
 
-  virtual ukm::UkmRecorder* UkmRecorder() { return nullptr; }
-  virtual ukm::SourceId UkmSourceID() const { return ukm::kInvalidSourceId; }
-
   // Returns the token that uniquely identifies this ExecutionContext.
   virtual ExecutionContextToken GetExecutionContextToken() const = 0;
 
