diff --git a/third_party/blink/renderer/modules/webaudio/audio_context.cc b/third_party/blink/renderer/modules/webaudio/audio_context.cc
index 783ee92eeb03..04c0bb486771 100644
--- a/third_party/blink/renderer/modules/webaudio/audio_context.cc
+++ b/third_party/blink/renderer/modules/webaudio/audio_context.cc
@@ -6,8 +6,6 @@
 
 #include "base/metrics/histogram_functions.h"
 #include "build/build_config.h"
-#include "services/metrics/public/cpp/ukm_builders.h"
-#include "services/metrics/public/cpp/ukm_recorder.h"
 #include "third_party/blink/public/common/browser_interface_broker_proxy.h"
 #include "third_party/blink/public/platform/web_audio_latency_hint.h"
 #include "third_party/blink/renderer/bindings/core/v8/script_promise_resolver.h"
@@ -531,16 +529,6 @@ void AudioContext::RecordAutoplayMetrics() {
   if (!autoplay_status_.has_value() || !GetDocument())
     return;
 
-  ukm::UkmRecorder* ukm_recorder = GetDocument()->UkmRecorder();
-  DCHECK(ukm_recorder);
-  ukm::builders::Media_Autoplay_AudioContext(GetDocument()->UkmSourceID())
-      .SetStatus(static_cast<int>(autoplay_status_.value()))
-      .SetUnlockType(autoplay_unlock_type_
-                         ? static_cast<int>(autoplay_unlock_type_.value())
-                         : -1)
-      .SetSourceNodeStarted(source_node_started_)
-      .Record(ukm_recorder);
-
   // Record autoplay_status_ value.
   base::UmaHistogramEnumeration("WebAudio.Autoplay", autoplay_status_.value());
 
