diff --git a/third_party/blink/renderer/modules/canvas/canvas2d/canvas_rendering_context_2d_api_test.cc b/third_party/blink/renderer/modules/canvas/canvas2d/canvas_rendering_context_2d_api_test.cc
index 74548784ac49e..d85c8d931df87 100644
--- a/third_party/blink/renderer/modules/canvas/canvas2d/canvas_rendering_context_2d_api_test.cc
+++ b/third_party/blink/renderer/modules/canvas/canvas2d/canvas_rendering_context_2d_api_test.cc
@@ -487,8 +487,6 @@ class StudyParticipationRaii {
 TEST_F(CanvasRenderingContext2DAPITest, IdentifiabilityStudyMaxOperations) {
   StudyParticipationRaii study_participation_raii;
   constexpr int kMaxOperations = 5;
-  IdentifiabilityStudyHelper::ScopedMaxOperationsSetter max_operations_setter(
-      kMaxOperations);
   CreateContext(kNonOpaque);
 
   int64_t last_digest = INT64_C(0);
