diff --git a/third_party/blink/renderer/core/css/media_query_evaluator.cc b/third_party/blink/renderer/core/css/media_query_evaluator.cc
index 2286a3bf90de..098635344664 100644
--- a/third_party/blink/renderer/core/css/media_query_evaluator.cc
+++ b/third_party/blink/renderer/core/css/media_query_evaluator.cc
@@ -64,25 +64,6 @@
 
 namespace blink {
 
-namespace {
-
-void RecordMediaQueryResult(Document* doc,
-                            const MediaQueryExp& expr,
-                            bool result) {
-  IdentifiableTokenBuilder input_builder;
-  input_builder.AddToken(IdentifiabilityBenignStringToken(expr.MediaFeature()));
-  input_builder.AddToken(
-      IdentifiabilityBenignStringToken(expr.ExpValue().CssText()));
-  IdentifiableSurface surface = IdentifiableSurface::FromTypeAndToken(
-      IdentifiableSurface::Type::kMediaQuery, input_builder.GetToken());
-
-  IdentifiabilityMetricBuilder(doc->UkmSourceID())
-      .Set(surface, result)
-      .Record(doc->UkmRecorder());
-}
-
-}  // namespace
-
 enum MediaFeaturePrefix { kMinPrefix, kMaxPrefix, kNoPrefix };
 
 using EvalFunc = bool (*)(const MediaQueryExpValue&,
@@ -1003,13 +984,6 @@ bool MediaQueryEvaluator::Eval(const MediaQueryExp& expr) const {
   EvalFunc func = g_function_map->at(expr.MediaFeature().Impl());
   if (func) {
     bool result = func(expr.ExpValue(), kNoPrefix, *media_values_);
-    Document* doc = nullptr;
-    if (!skip_ukm_reporting_ && (doc = media_values_->GetDocument()) &&
-        (IdentifiabilityStudySettings::Get()->ShouldSample(
-            IdentifiableSurface::Type::kMediaQuery))) {
-      RecordMediaQueryResult(doc, expr, result);
-    }
-
     return result;
   }
 
