diff --git a/third_party/blink/renderer/platform/scheduler/main_thread/page_scheduler_impl.cc b/third_party/blink/renderer/platform/scheduler/main_thread/page_scheduler_impl.cc
index ac1fc7430707..11fc8402ad61 100644
--- a/third_party/blink/renderer/platform/scheduler/main_thread/page_scheduler_impl.cc
+++ b/third_party/blink/renderer/platform/scheduler/main_thread/page_scheduler_impl.cc
@@ -1042,14 +1042,6 @@ PageSchedulerImpl::PageLifecycleStateTracker::
   }
 }
 
-FrameSchedulerImpl* PageSchedulerImpl::SelectFrameForUkmAttribution() {
-  for (FrameSchedulerImpl* frame_scheduler : frame_schedulers_) {
-    if (frame_scheduler->GetUkmRecorder())
-      return frame_scheduler;
-  }
-  return nullptr;
-}
-
 WebScopedVirtualTimePauser PageSchedulerImpl::CreateWebScopedVirtualTimePauser(
     const String& name,
     WebScopedVirtualTimePauser::VirtualTaskDuration duration) {
