diff --git a/third_party/blink/renderer/core/exported/web_shared_worker_impl.h b/third_party/blink/renderer/core/exported/web_shared_worker_impl.h
index 22e76cadb4a5..a0afd9b47cf4 100644
--- a/third_party/blink/renderer/core/exported/web_shared_worker_impl.h
+++ b/third_party/blink/renderer/core/exported/web_shared_worker_impl.h
@@ -37,7 +37,6 @@
 
 #include "base/memory/scoped_refptr.h"
 #include "base/memory/weak_ptr.h"
-#include "services/metrics/public/cpp/ukm_source_id.h"
 #include "services/network/public/mojom/content_security_policy.mojom-blink-forward.h"
 #include "services/network/public/mojom/fetch_api.mojom-shared.h"
 #include "services/network/public/mojom/ip_address_space.mojom-blink-forward.h"
@@ -93,8 +92,7 @@ class CORE_EXPORT WebSharedWorkerImpl final : public WebSharedWorker {
       const blink::SharedWorkerToken& token,
       const base::UnguessableToken& appcache_host_id,
       CrossVariantMojoRemote<mojom::SharedWorkerHostInterfaceBase> host,
-      WebSharedWorkerClient*,
-      ukm::SourceId ukm_source_id);
+      WebSharedWorkerClient*);
 
   void StartWorkerContext(
       const WebURL&,
