diff --git a/third_party/blink/renderer/core/workers/worklet_global_scope.h b/third_party/blink/renderer/core/workers/worklet_global_scope.h
index c373f37e5b682..3fd0878b6c516 100644
--- a/third_party/blink/renderer/core/workers/worklet_global_scope.h
+++ b/third_party/blink/renderer/core/workers/worklet_global_scope.h
@@ -65,7 +65,6 @@ class CORE_EXPORT WorkletGlobalScope
   scoped_refptr<base::SingleThreadTaskRunner> GetTaskRunner(TaskType) final;
   FrameOrWorkerScheduler* GetScheduler() final;
   bool CrossOriginIsolatedCapability() const final;
-  ukm::UkmRecorder* UkmRecorder() final;
 
   // WorkerOrWorkletGlobalScope
   void Dispose() override;
@@ -183,8 +182,6 @@ class CORE_EXPORT WorkletGlobalScope
   // The token identifying the LocalFrame that caused this scope to be created.
   const LocalFrameToken frame_token_;
 
-  std::unique_ptr<ukm::UkmRecorder> ukm_recorder_;
-
   // This is inherited at construction to make sure it is possible to used
   // restricted API between the document and the worklet (e.g.
   // SharedArrayBuffer passing via postMessage).
