diff --git a/third_party/blink/renderer/core/frame/dom_window.h b/third_party/blink/renderer/core/frame/dom_window.h
index 0a3d30b0e1bf..d0d686ed1ea4 100644
--- a/third_party/blink/renderer/core/frame/dom_window.h
+++ b/third_party/blink/renderer/core/frame/dom_window.h
@@ -141,8 +141,6 @@ class CORE_EXPORT DOMWindow : public EventTargetWithInlineData {
   void InstallCoopAccessMonitor(
       network::mojom::blink::CoopAccessReportType report_type,
       LocalFrame* accessing_frame,
-      mojo::PendingRemote<
-          network::mojom::blink::CrossOriginOpenerPolicyReporter> reporter,
       bool endpoint_defined,
       const WTF::String& reported_window_url);
   // Whenever we detect that the enforcement of a report-only COOP policy would
@@ -195,8 +193,6 @@ class CORE_EXPORT DOMWindow : public EventTargetWithInlineData {
   struct CoopAccessMonitor {
     network::mojom::blink::CoopAccessReportType report_type;
     blink::LocalFrameToken accessing_main_frame;
-    mojo::Remote<network::mojom::blink::CrossOriginOpenerPolicyReporter>
-        reporter;
     bool endpoint_defined;
     WTF::String reported_window_url;
   };
