diff --git a/third_party/blink/renderer/core/core_idl_files.gni b/third_party/blink/renderer/core/core_idl_files.gni
index 159ece16d9ea..9bb9f65acbc8 100644
--- a/third_party/blink/renderer/core/core_idl_files.gni
+++ b/third_party/blink/renderer/core/core_idl_files.gni
@@ -198,7 +198,6 @@ core_interface_idl_files_core_only =
                     "frame/navigator_ua_data.idl",
                     "frame/report.idl",
                     "frame/report_body.idl",
-                    "frame/reporting_observer.idl",
                     "frame/scheduling.idl",
                     "frame/test_report_body.idl",
                     "frame/user_activation.idl",
@@ -701,7 +700,6 @@ core_dictionary_idl_files =
                     "frame/impression_params.idl",
                     "frame/is_input_pending_options.idl",
                     "frame/navigator_ua_brand_version.idl",
-                    "frame/reporting_observer_options.idl",
                     "frame/scroll_into_view_options.idl",
                     "frame/scroll_options.idl",
                     "frame/scroll_to_options.idl",
