diff --git a/third_party/blink/renderer/modules/canvas/canvas2d/canvas_pattern.h b/third_party/blink/renderer/modules/canvas/canvas2d/canvas_pattern.h
index 6bd134ac011af..789c99a2a8030 100644
--- a/third_party/blink/renderer/modules/canvas/canvas2d/canvas_pattern.h
+++ b/third_party/blink/renderer/modules/canvas/canvas2d/canvas_pattern.h
@@ -27,7 +27,6 @@
 #define THIRD_PARTY_BLINK_RENDERER_MODULES_CANVAS_CANVAS2D_CANVAS_PATTERN_H_
 
 #include "third_party/blink/renderer/bindings/core/v8/v8_dom_matrix_2d_init.h"
-#include "third_party/blink/renderer/modules/canvas/canvas2d/identifiability_study_helper.h"
 #include "third_party/blink/renderer/platform/bindings/script_wrappable.h"
 #include "third_party/blink/renderer/platform/graphics/pattern.h"
 #include "third_party/blink/renderer/platform/transforms/affine_transform.h"
@@ -54,13 +53,10 @@ class CanvasPattern final : public ScriptWrappable {
 
   void setTransform(DOMMatrix2DInit*, ExceptionState&);
 
-  IdentifiableToken GetIdentifiableToken() const;
-
  private:
   scoped_refptr<Pattern> pattern_;
   AffineTransform pattern_transform_;
   bool origin_clean_;
-  IdentifiabilityStudyHelper identifiability_study_helper_;
 };
 
 }  // namespace blink
