diff --git a/third_party/blink/renderer/core/BUILD.gn b/third_party/blink/renderer/core/BUILD.gn
index 991cd09342e6..9a16f5a96ed9 100644
--- a/third_party/blink/renderer/core/BUILD.gn
+++ b/third_party/blink/renderer/core/BUILD.gn
@@ -268,9 +268,6 @@ component("core") {
     "//mojo/public/cpp/bindings:bindings",
     "//mojo/public/cpp/system",
     "//printing/buildflags",
-    "//services/metrics/public/cpp:metrics_cpp",
-    "//services/metrics/public/cpp:ukm_builders",
-    "//services/metrics/public/mojom",
     "//services/service_manager/public/cpp",
     "//skia",
     "//skia:skcms",
@@ -335,7 +332,6 @@ blink_core_sources("core_hot") {
   }
 
   deps = [
-    "//services/metrics/public/cpp:ukm_builders",
     "//third_party/blink/renderer/core/probe:probe",
   ]
 }
@@ -353,7 +349,6 @@ source_set("testing") {
     ":core",
     ":generated_testing_idls",
     "//build:chromeos_buildflags",
-    "//components/ukm:test_support",
     "//third_party/blink/renderer/bindings/core/v8:testing",
     "//third_party/blink/renderer/platform:test_support",
     "//ui/base/cursor:cursor_base",
@@ -1099,7 +1094,6 @@ target(core_generated_target_type, "core_generated") {
     sources += get_target_outputs(current)
   }
   deps = [
-    "//services/metrics/public/cpp:ukm_builders",
     "//third_party/blink/renderer/core/probe:probe",
   ]
   public_deps = [ ":all_generators" ]
@@ -1154,10 +1148,6 @@ target(core_generated_target_type, "core_generated") {
       "/wd4702",
     ]
   }
-
-  if (is_debug || dcheck_always_on) {
-    deps += [ "//components/crash/core/common:crash_key" ]
-  }
 }
 
 # Fuzzer for blink::TextResourceDecoder.
@@ -1285,7 +1275,6 @@ source_set("unit_tests") {
     "frame/local_frame_back_forward_cache_test.cc",
     "frame/local_frame_client_impl_test.cc",
     "frame/local_frame_test.cc",
-    "frame/local_frame_ukm_aggregator_test.cc",
     "frame/local_frame_view_test.cc",
     "frame/location_report_body_test.cc",
     "frame/mhtml_archive_test.cc",
@@ -1623,10 +1612,8 @@ source_set("unit_tests") {
     "//build:chromeos_buildflags",
     "//components/paint_preview/common:common",
     "//components/shared_highlighting/core/common",
-    "//components/ukm:test_support",
     "//mojo/public/cpp/system",
     "//mojo/public/cpp/test_support:test_utils",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network/trust_tokens:test_support",
     "//skia",
     "//skia:skcms",
