diff --git a/third_party/blink/renderer/core/workers/worklet_global_scope.h b/third_party/blink/renderer/core/workers/worklet_global_scope.h
index ae9943f436073..c8af7dad91bd4 100644
--- a/third_party/blink/renderer/core/workers/worklet_global_scope.h
+++ b/third_party/blink/renderer/core/workers/worklet_global_scope.h
@@ -67,7 +67,6 @@ class CORE_EXPORT WorkletGlobalScope
   FrameOrWorkerScheduler* GetScheduler() final;
   bool CrossOriginIsolatedCapability() const final;
   bool DirectSocketCapability() const final;
-  ukm::UkmRecorder* UkmRecorder() final;
 
   // WorkerOrWorkletGlobalScope
   void Dispose() override;
@@ -185,8 +184,6 @@ class CORE_EXPORT WorkletGlobalScope
   // The token identifying the LocalFrame that caused this scope to be created.
   const LocalFrameToken frame_token_;
 
-  std::unique_ptr<ukm::UkmRecorder> ukm_recorder_;
-
   // This is inherited at construction to make sure it is possible to used
   // restricted API between the document and the worklet (e.g.
   // SharedArrayBuffer passing via postMessage).
