diff --git a/third_party/blink/renderer/core/core_idl_files.gni b/third_party/blink/renderer/core/core_idl_files.gni
index 96315711f9c0..f6bb00695a03 100644
--- a/third_party/blink/renderer/core/core_idl_files.gni
+++ b/third_party/blink/renderer/core/core_idl_files.gni
@@ -187,22 +187,12 @@ core_interface_idl_files_core_only =
                     "fileapi/file_reader.idl",
                     "fileapi/file_reader_sync.idl",
                     "frame/bar_prop.idl",
-                    "frame/coop_access_violation_report_body.idl",
-                    "frame/csp/csp_violation_report_body.idl",
-                    "frame/deprecation_report_body.idl",
-                    "frame/document_policy_violation_report_body.idl",
                     "frame/external.idl",
-                    "frame/feature_policy_violation_report_body.idl",
                     "frame/fragment_directive.idl",
                     "frame/history.idl",
-                    "frame/intervention_report_body.idl",
                     "frame/location.idl",
                     "frame/navigator_ua_data.idl",
-                    "frame/report.idl",
-                    "frame/report_body.idl",
-                    "frame/reporting_observer.idl",
                     "frame/scheduling.idl",
-                    "frame/test_report_body.idl",
                     "frame/user_activation.idl",
                     "frame/visual_viewport.idl",
                     "frame/window_controls_overlay.idl",
