diff --git a/third_party/blink/renderer/bindings/core/v8/generated.gni b/third_party/blink/renderer/bindings/core/v8/generated.gni
index 01938ee00099..6c8c7a27ae40 100644
--- a/third_party/blink/renderer/bindings/core/v8/generated.gni
+++ b/third_party/blink/renderer/bindings/core/v8/generated.gni
@@ -186,8 +186,6 @@ generated_core_callback_function_files = [
   "$bindings_core_v8_output_dir/v8_no_argument_constructor.h",
   "$bindings_core_v8_output_dir/v8_performance_observer_callback.cc",
   "$bindings_core_v8_output_dir/v8_performance_observer_callback.h",
-  "$bindings_core_v8_output_dir/v8_reporting_observer_callback.cc",
-  "$bindings_core_v8_output_dir/v8_reporting_observer_callback.h",
   "$bindings_core_v8_output_dir/v8_resize_observer_callback.cc",
   "$bindings_core_v8_output_dir/v8_resize_observer_callback.h",
   "$bindings_core_v8_output_dir/v8_scroll_state_callback.cc",
