diff --git a/third_party/blink/renderer/modules/plugins/navigator_plugins.h b/third_party/blink/renderer/modules/plugins/navigator_plugins.h
index e45d7795df25..7f424a4cbdc8 100644
--- a/third_party/blink/renderer/modules/plugins/navigator_plugins.h
+++ b/third_party/blink/renderer/modules/plugins/navigator_plugins.h
@@ -35,8 +35,6 @@ class NavigatorPlugins final : public GarbageCollected<NavigatorPlugins>,
   DOMPluginArray* plugins(LocalFrame*) const;
   DOMMimeTypeArray* mimeTypes(LocalFrame*) const;
 
-  void RecordMimeTypes(LocalFrame*) const;
-
   mutable Member<DOMPluginArray> plugins_;
   mutable Member<DOMMimeTypeArray> mime_types_;
 };
