diff --git a/third_party/blink/renderer/core/frame/build.gni b/third_party/blink/renderer/core/frame/build.gni
index 9c164212e432..c8274d378923 100644
--- a/third_party/blink/renderer/core/frame/build.gni
+++ b/third_party/blink/renderer/core/frame/build.gni
@@ -32,8 +32,6 @@ blink_core_sources_frame = [
   "csp/source_list_directive.h",
   "csp/string_list_directive.cc",
   "csp/string_list_directive.h",
-  "dactyloscoper.cc",
-  "dactyloscoper.h",
   "deprecated_schedule_style_recalc_during_layout.cc",
   "deprecated_schedule_style_recalc_during_layout.h",
   "deprecation.cc",
