diff --git a/third_party/blink/renderer/core/frame/build.gni b/third_party/blink/renderer/core/frame/build.gni
index 393d5d2f006d..d70379f5b63b 100644
--- a/third_party/blink/renderer/core/frame/build.gni
+++ b/third_party/blink/renderer/core/frame/build.gni
@@ -161,10 +161,6 @@ blink_core_sources_frame = [
   "report.h",
   "report_body.cc",
   "report_body.h",
-  "reporting_context.cc",
-  "reporting_context.h",
-  "reporting_observer.cc",
-  "reporting_observer.h",
   "resize_viewport_anchor.cc",
   "resize_viewport_anchor.h",
   "root_frame_viewport.cc",
