diff --git a/third_party/blink/renderer/core/core_idl_files.gni b/third_party/blink/renderer/core/core_idl_files.gni
index 1ada6225e4b18..b730803d4ec83 100644
--- a/third_party/blink/renderer/core/core_idl_files.gni
+++ b/third_party/blink/renderer/core/core_idl_files.gni
@@ -672,7 +672,6 @@ core_dictionary_idl_files =
           "dom/get_inner_html_options.idl",
           "dom/get_root_node_options.idl",
           "dom/idle_request_options.idl",
-          "dom/interest_cohort.idl",
           "dom/mutation_observer_init.idl",
           "dom/pointer_lock_options.idl",
           "dom/shadow_root_init.idl",
