diff --git a/third_party/blink/public/mojom/frame/BUILD.gn b/third_party/blink/public/mojom/frame/BUILD.gn
index 1549c3acdc4a..f2b3111098b4 100644
--- a/third_party/blink/public/mojom/frame/BUILD.gn
+++ b/third_party/blink/public/mojom/frame/BUILD.gn
@@ -25,7 +25,6 @@ mojom_component("frame") {
     "lifecycle.mojom",
     "media_player_action.mojom",
     "policy_container.mojom",
-    "reporting_observer.mojom",
     "sudden_termination_disabler_type.mojom",
     "tree_scope_type.mojom",
     "user_activation_notification_type.mojom",
