diff --git a/third_party/blink/public/mojom/worker/shared_worker_connector.mojom b/third_party/blink/public/mojom/worker/shared_worker_connector.mojom
index f975ea7d34fd..74f16978a506 100644
--- a/third_party/blink/public/mojom/worker/shared_worker_connector.mojom
+++ b/third_party/blink/public/mojom/worker/shared_worker_connector.mojom
@@ -24,6 +24,5 @@ interface SharedWorkerConnector {
           pending_remote<SharedWorkerClient> client,
           SharedWorkerCreationContextType creation_context_type,
           MessagePortDescriptor message_port,
-          pending_remote<BlobURLToken>? blob_url_token,
-          int64 client_ukm_source_id);
+          pending_remote<BlobURLToken>? blob_url_token);
 };
