diff --git a/third_party/blink/renderer/core/frame/BUILD.gn b/third_party/blink/renderer/core/frame/BUILD.gn
index 4d1fd63d45c2..746f4da7b35a 100644
--- a/third_party/blink/renderer/core/frame/BUILD.gn
+++ b/third_party/blink/renderer/core/frame/BUILD.gn
@@ -36,8 +36,6 @@ blink_core_sources("frame") {
     "csp/source_list_directive.h",
     "csp/string_list_directive.cc",
     "csp/string_list_directive.h",
-    "dactyloscoper.cc",
-    "dactyloscoper.h",
     "deprecated_schedule_style_recalc_during_layout.cc",
     "deprecated_schedule_style_recalc_during_layout.h",
     "deprecation.cc",
@@ -102,8 +100,6 @@ blink_core_sources("frame") {
     "local_frame.cc",
     "local_frame.h",
     "local_frame_client.h",
-    "local_frame_ukm_aggregator.cc",
-    "local_frame_ukm_aggregator.h",
     "local_frame_view.cc",
     "local_frame_view.h",
     "location.cc",
