diff --git a/third_party/blink/renderer/bindings/idl_in_core.gni b/third_party/blink/renderer/bindings/idl_in_core.gni
index 6d69a99e937f..02ae8695be58 100644
--- a/third_party/blink/renderer/bindings/idl_in_core.gni
+++ b/third_party/blink/renderer/bindings/idl_in_core.gni
@@ -282,8 +282,6 @@ static_idl_files_in_core = get_path_info(
           "//third_party/blink/renderer/core/frame/navigator_window_controls_overlay.idl",
           "//third_party/blink/renderer/core/frame/report.idl",
           "//third_party/blink/renderer/core/frame/report_body.idl",
-          "//third_party/blink/renderer/core/frame/reporting_observer.idl",
-          "//third_party/blink/renderer/core/frame/reporting_observer_options.idl",
           "//third_party/blink/renderer/core/frame/scheduling.idl",
           "//third_party/blink/renderer/core/frame/screen.idl",
           "//third_party/blink/renderer/core/frame/scroll_into_view_options.idl",
