diff --git a/third_party/blink/renderer/core/frame/web_view_frame_widget.h b/third_party/blink/renderer/core/frame/web_view_frame_widget.h
index 42e2d7eecbb2..edda7ff56ba8 100644
--- a/third_party/blink/renderer/core/frame/web_view_frame_widget.h
+++ b/third_party/blink/renderer/core/frame/web_view_frame_widget.h
@@ -114,8 +114,6 @@ class CORE_EXPORT WebViewFrameWidget : public WebFrameWidgetBase {
   void RecordEndOfFrameMetrics(
       base::TimeTicks frame_begin_time,
       cc::ActiveFrameSequenceTrackers trackers) override;
-  std::unique_ptr<cc::BeginMainFrameMetrics> GetBeginMainFrameMetrics()
-      override;
   void BeginUpdateLayers() override;
   void EndUpdateLayers() override;
   void DidBeginMainFrame() override;
