diff --git a/third_party/blink/renderer/modules/canvas/canvas2d/canvas_pattern.cc b/third_party/blink/renderer/modules/canvas/canvas2d/canvas_pattern.cc
index 6b1438626fd51..9c3774aaaa4f1 100644
--- a/third_party/blink/renderer/modules/canvas/canvas2d/canvas_pattern.cc
+++ b/third_party/blink/renderer/modules/canvas/canvas2d/canvas_pattern.cc
@@ -58,9 +58,6 @@ CanvasPattern::CanvasPattern(scoped_refptr<Image> image,
                              bool origin_clean)
     : pattern_(Pattern::CreateImagePattern(std::move(image), repeat)),
       origin_clean_(origin_clean) {
-  identifiability_study_helper_.MaybeUpdateBuilder(
-      CanvasOps::kCreatePattern, image ? image->width() : 0,
-      image ? image->height() : 0, repeat);
 }
 
 void CanvasPattern::setTransform(DOMMatrix2DInit* transform,
@@ -71,14 +68,8 @@ void CanvasPattern::setTransform(DOMMatrix2DInit* transform,
   if (!m) {
     return;
   }
-  identifiability_study_helper_.MaybeUpdateBuilder(
-      m->m11(), m->m12(), m->m21(), m->m22(), m->m41(), m->m42());
 
   pattern_transform_ = m->GetAffineTransform();
 }
 
-IdentifiableToken CanvasPattern::GetIdentifiableToken() const {
-  return identifiability_study_helper_.GetToken();
-}
-
 }  // namespace blink
