diff --git a/third_party/blink/renderer/core/html/parser/html_document_parser.h b/third_party/blink/renderer/core/html/parser/html_document_parser.h
index 65a17838def1..36f6c0b46aac 100644
--- a/third_party/blink/renderer/core/html/parser/html_document_parser.h
+++ b/third_party/blink/renderer/core/html/parser/html_document_parser.h
@@ -59,7 +59,6 @@ class DocumentEncodingData;
 class DocumentFragment;
 class Element;
 class HTMLDocument;
-class HTMLParserMetrics;
 class HTMLParserScheduler;
 class HTMLParserScriptRunner;
 class HTMLPreloadScanner;
@@ -269,8 +268,6 @@ class CORE_EXPORT HTMLDocumentParser : public ScriptableDocumentParser,
   Member<HTMLDocumentParserState> task_runner_state_;
   PreloadRequestStream queued_preloads_;
 
-  // Metrics gathering and reporting
-  std::unique_ptr<HTMLParserMetrics> metrics_reporter_;
   // A timer for how long we are inactive after yielding
   std::unique_ptr<base::ElapsedTimer> yield_timer_;
 
