diff --git a/third_party/blink/renderer/modules/BUILD.gn b/third_party/blink/renderer/modules/BUILD.gn
index 79788bf60dbc..25a77687b9fd 100644
--- a/third_party/blink/renderer/modules/BUILD.gn
+++ b/third_party/blink/renderer/modules/BUILD.gn
@@ -491,14 +491,12 @@ source_set("unit_tests") {
     "//base/test:test_support",
     "//build:chromecast_buildflags",
     "//components/schema_org/common:mojom_blink",
-    "//components/ukm:test_support",
     "//media:test_support",
     "//media/mojo/mojom",
     "//media/webrtc:webrtc",
     "//mojo/public/cpp/bindings",
     "//net:test_support",
     "//services/device/public/cpp:test_support",
-    "//services/metrics/public/cpp:ukm_builders",
     "//skia",
     "//testing/gmock",
     "//testing/gtest",
