diff --git a/third_party/blink/renderer/platform/loader/BUILD.gn b/third_party/blink/renderer/platform/loader/BUILD.gn
index 1069a982c978..1618c8c693d8 100644
--- a/third_party/blink/renderer/platform/loader/BUILD.gn
+++ b/third_party/blink/renderer/platform/loader/BUILD.gn
@@ -143,7 +143,6 @@ blink_platform_sources("loader") {
     "//components/link_header_util",
     "//components/web_package",
     "//net",
-    "//services/metrics/public/cpp:ukm_builders",
     "//services/network/public/cpp",
     "//services/network/public/mojom:mojom_blink",
     "//third_party/blink/renderer/platform/blob:blob",
