diff --git a/third_party/blink/renderer/core/loader/interactive_detector.h b/third_party/blink/renderer/core/loader/interactive_detector.h
index f5abcecf3e825..e82fa299a1f96 100644
--- a/third_party/blink/renderer/core/loader/interactive_detector.h
+++ b/third_party/blink/renderer/core/loader/interactive_detector.h
@@ -244,8 +244,6 @@ class CORE_EXPORT InteractiveDetector
   // pending_pointerdown_delay_.
   base::TimeTicks pending_pointerdown_timestamp_;
 
-  ukm::UkmRecorder* ukm_recorder_;
-
   DISALLOW_COPY_AND_ASSIGN(InteractiveDetector);
 };
 
