diff --git a/third_party/blink/public/BUILD.gn b/third_party/blink/public/BUILD.gn
index 1d66f1fb76265..9b0ec392b3000 100644
--- a/third_party/blink/public/BUILD.gn
+++ b/third_party/blink/public/BUILD.gn
@@ -193,7 +193,6 @@ source_set("blink_headers") {
     "platform/web_dedicated_worker.h",
     "platform/web_dedicated_worker_host_factory_client.h",
     "platform/web_distillability.h",
-    "platform/web_document_subresource_filter.h",
     "platform/web_double_size.h",
     "platform/web_drag_data.h",
     "platform/web_effective_connection_type.h",
