diff --git a/tools/perf/chrome_telemetry_build/BUILD.gn b/tools/perf/chrome_telemetry_build/BUILD.gn
index 4725b3aa6cd5..8e76ad96dd9b 100644
--- a/tools/perf/chrome_telemetry_build/BUILD.gn
+++ b/tools/perf/chrome_telemetry_build/BUILD.gn
@@ -100,7 +100,6 @@ group("telemetry_chrome_test_without_chrome") {
   testonly = true
 
   data = [
-    "//components/crash/content/tools/generate_breakpad_symbols.py",
     "//testing/scripts/common.py",
     "//testing/xvfb.py",
     "//tools/perf/chrome_telemetry_build/",
