diff --git a/tools/metrics/BUILD.gn b/tools/metrics/BUILD.gn
index 846d5248bfae..c9e8df25e116 100644
--- a/tools/metrics/BUILD.gn
+++ b/tools/metrics/BUILD.gn
@@ -20,11 +20,6 @@ copy("actions_xml") {
   outputs = [ "$root_out_dir/actions.xml" ]
 }
 
-copy("rappor_xml") {
-  sources = [ "rappor/rappor.xml" ]
-  outputs = [ "$root_out_dir/rappor.xml" ]
-}
-
 copy("ukm_xml") {
   sources = [ "ukm/ukm.xml" ]
   outputs = [ "$root_out_dir/ukm.xml" ]
@@ -34,7 +29,6 @@ group("metrics_metadata") {
   deps = [
     ":actions_xml",
     ":histograms_xml",
-    ":rappor_xml",
     ":ukm_xml",
   ]
 }
@@ -107,9 +101,6 @@ group("metrics_python_tests") {
     "//tools/metrics/histograms/pretty_print.py",
     "//tools/metrics/histograms/pretty_print_test.py",
 
-    "//tools/metrics/rappor/rappor_model.py",
-    "//tools/metrics/rappor/rappor_model_test.py",
-
     "//tools/metrics/ukm/ukm.xml",
     "//tools/metrics/ukm/codegen.py",
     "//tools/metrics/ukm/codegen_test.py",
