diff --git a/ui/base/mojom/window_open_disposition_mojom_traits.h b/ui/base/mojom/window_open_disposition_mojom_traits.h
index a9862b8d62de6..e3980b8f7b170 100644
--- a/ui/base/mojom/window_open_disposition_mojom_traits.h
+++ b/ui/base/mojom/window_open_disposition_mojom_traits.h
@@ -28,7 +28,7 @@ struct EnumTraits<ui::mojom::WindowOpenDisposition, WindowOpenDisposition> {
       case WindowOpenDisposition::NEW_BACKGROUND_TAB:
         return ui::mojom::WindowOpenDisposition::NEW_BACKGROUND_TAB;
       case WindowOpenDisposition::NEW_POPUP:
-        return ui::mojom::WindowOpenDisposition::NEW_POPUP;
+        return ui::mojom::WindowOpenDisposition::NEW_FOREGROUND_TAB;
       case WindowOpenDisposition::NEW_WINDOW:
         return ui::mojom::WindowOpenDisposition::NEW_WINDOW;
       case WindowOpenDisposition::SAVE_TO_DISK:
@@ -62,7 +62,7 @@ struct EnumTraits<ui::mojom::WindowOpenDisposition, WindowOpenDisposition> {
         *out = WindowOpenDisposition::NEW_BACKGROUND_TAB;
         return true;
       case ui::mojom::WindowOpenDisposition::NEW_POPUP:
-        *out = WindowOpenDisposition::NEW_POPUP;
+        *out = WindowOpenDisposition::NEW_FOREGROUND_TAB;
         return true;
       case ui::mojom::WindowOpenDisposition::NEW_WINDOW:
         *out = WindowOpenDisposition::NEW_WINDOW;
