diff --git a/ui/base/ime/dummy_text_input_client.cc b/ui/base/ime/dummy_text_input_client.cc
index 14dc45935f40..6aeb4d55bcb7 100644
--- a/ui/base/ime/dummy_text_input_client.cc
+++ b/ui/base/ime/dummy_text_input_client.cc
@@ -138,10 +138,6 @@ bool DummyTextInputClient::IsTextEditCommandEnabled(
 void DummyTextInputClient::SetTextEditCommandForNextKeyEvent(
     TextEditCommand command) {}
 
-ukm::SourceId DummyTextInputClient::GetClientSourceForMetrics() const {
-  return ukm::SourceId{};
-}
-
 bool DummyTextInputClient::ShouldDoLearning() {
   return false;
 }
