diff --git a/ui/compositor/compositor.h b/ui/compositor/compositor.h
index 9a731f852201..876655996f9f 100644
--- a/ui/compositor/compositor.h
+++ b/ui/compositor/compositor.h
@@ -347,8 +347,7 @@ class COMPOSITOR_EXPORT Compositor : public cc::LayerTreeHostClient,
       override;
   void NotifyThroughputTrackerResults(
       cc::CustomTrackerResults results) override;
-  void SubmitThroughputData(ukm::SourceId source_id,
-                            int aggregated_percent,
+  void SubmitThroughputData(int aggregated_percent,
                             int impl_percent,
                             base::Optional<int> main_percent) override {}
   void DidObserveFirstScrollDelay(
