diff --git a/ui/gfx/win/window_impl.cc b/ui/gfx/win/window_impl.cc
index 9c4de6c604db..ac0dfe07e483 100644
--- a/ui/gfx/win/window_impl.cc
+++ b/ui/gfx/win/window_impl.cc
@@ -16,7 +16,6 @@
 #include "base/synchronization/lock.h"
 #include "base/win/win_util.h"
 #include "base/win/wrapped_window_proc.h"
-#include "ui/gfx/win/crash_id_helper.h"
 #include "ui/gfx/win/hwnd_util.h"
 
 namespace gfx {
@@ -306,8 +305,6 @@ LRESULT CALLBACK WindowImpl::WndProc(HWND hwnd,
   if (!window)
     return 0;
 
-  auto logger =
-      CrashIdHelper::Get()->OnWillProcessMessages(window->debugging_id_);
   return window->OnWndProc(message, w_param, l_param);
 }
 
