diff --git a/ui/latency/BUILD.gn b/ui/latency/BUILD.gn
index 5130c405ed99..4b04d23caba5 100644
--- a/ui/latency/BUILD.gn
+++ b/ui/latency/BUILD.gn
@@ -9,8 +9,6 @@ source_set("latency") {
     "latency_histogram_macros.h",
     "latency_info.cc",
     "latency_info.h",
-    "latency_tracker.cc",
-    "latency_tracker.h",
   ]
 
   deps = [
@@ -19,7 +17,7 @@ source_set("latency") {
     "//ui/gfx",
   ]
 
-  public_deps = [ "//services/metrics/public/cpp:metrics_cpp" ]
+  public_deps = []
 }
 
 source_set("test_support") {
