diff --git a/ui/latency/mojom/latency_info_mojom_traits.h b/ui/latency/mojom/latency_info_mojom_traits.h
index bde8c1255209..2e455e250ef1 100644
--- a/ui/latency/mojom/latency_info_mojom_traits.h
+++ b/ui/latency/mojom/latency_info_mojom_traits.h
@@ -50,7 +50,6 @@ struct COMPONENT_EXPORT(LATENCY_SHARED_MOJOM_TRAITS)
   static const ui::LatencyInfo::LatencyMap& latency_components(
       const ui::LatencyInfo& info);
   static int64_t trace_id(const ui::LatencyInfo& info);
-  static ukm::SourceId ukm_source_id(const ui::LatencyInfo& info);
   static bool coalesced(const ui::LatencyInfo& info);
   static bool began(const ui::LatencyInfo& info);
   static bool terminated(const ui::LatencyInfo& info);
