diff --git a/ui/latency/mojom/latency_info_mojom_traits.cc b/ui/latency/mojom/latency_info_mojom_traits.cc
index 4760725e9f29..d9caec453d80 100644
--- a/ui/latency/mojom/latency_info_mojom_traits.cc
+++ b/ui/latency/mojom/latency_info_mojom_traits.cc
@@ -75,13 +75,6 @@ int64_t StructTraits<ui::mojom::LatencyInfoDataView, ui::LatencyInfo>::trace_id(
   return info.trace_id();
 }
 
-// static
-ukm::SourceId
-StructTraits<ui::mojom::LatencyInfoDataView, ui::LatencyInfo>::ukm_source_id(
-    const ui::LatencyInfo& info) {
-  return info.ukm_source_id();
-}
-
 // static
 bool StructTraits<ui::mojom::LatencyInfoDataView, ui::LatencyInfo>::coalesced(
     const ui::LatencyInfo& info) {
@@ -134,7 +127,6 @@ bool StructTraits<ui::mojom::LatencyInfoDataView, ui::LatencyInfo>::Read(
   if (!data.ReadLatencyComponents(&out->latency_components_))
     return false;
   out->trace_id_ = data.trace_id();
-  out->ukm_source_id_ = data.ukm_source_id();
   out->coalesced_ = data.coalesced();
   out->began_ = data.began();
   out->terminated_ = data.terminated();
