diff --git a/ui/latency/ipc/latency_info_param_traits.cc b/ui/latency/ipc/latency_info_param_traits.cc
index 4bfa9f84638e..6cf64ebf5bc8 100644
--- a/ui/latency/ipc/latency_info_param_traits.cc
+++ b/ui/latency/ipc/latency_info_param_traits.cc
@@ -34,7 +34,6 @@ namespace IPC {
 void ParamTraits<ui::LatencyInfo>::Write(base::Pickle* m, const param_type& p) {
   WriteParam(m, p.latency_components_);
   WriteParam(m, p.trace_id_);
-  WriteParam(m, p.ukm_source_id_);
   WriteParam(m, p.coalesced_);
   WriteParam(m, p.began_);
   WriteParam(m, p.terminated_);
@@ -51,8 +50,6 @@ bool ParamTraits<ui::LatencyInfo>::Read(const base::Pickle* m,
 
   if (!ReadParam(m, iter, &p->trace_id_))
     return false;
-  if (!ReadParam(m, iter, &p->ukm_source_id_))
-    return false;
   if (!ReadParam(m, iter, &p->coalesced_))
     return false;
   if (!ReadParam(m, iter, &p->began_))
@@ -74,8 +71,6 @@ void ParamTraits<ui::LatencyInfo>::Log(const param_type& p, std::string* l) {
   l->append(" ");
   LogParam(p.trace_id_, l);
   l->append(" ");
-  LogParam(p.ukm_source_id_, l);
-  l->append(" ");
   LogParam(p.coalesced_, l);
   l->append(" ");
   LogParam(p.began_, l);
