diff --git a/ui/latency/mojom/latency_info.mojom b/ui/latency/mojom/latency_info.mojom
index 77a5c42e6cc3..cfa417c16bab 100644
--- a/ui/latency/mojom/latency_info.mojom
+++ b/ui/latency/mojom/latency_info.mojom
@@ -60,7 +60,6 @@ enum SourceEventType {
 struct LatencyInfo {
   map<LatencyComponentType, mojo_base.mojom.TimeTicks> latency_components;
   int64 trace_id;
-  int64 ukm_source_id;
   bool coalesced;
   bool began;
   bool terminated;
