diff --git a/ui/latency/latency_info.cc b/ui/latency/latency_info.cc
index f369ec6df9fe..0e487abd42c0 100644
--- a/ui/latency/latency_info.cc
+++ b/ui/latency/latency_info.cc
@@ -117,7 +117,6 @@ LatencyInfo::LatencyInfo() : LatencyInfo(SourceEventType::UNKNOWN) {}
 
 LatencyInfo::LatencyInfo(SourceEventType type)
     : trace_id_(-1),
-      ukm_source_id_(ukm::kInvalidSourceId),
       coalesced_(false),
       began_(false),
       terminated_(false),
@@ -133,7 +132,6 @@ LatencyInfo::~LatencyInfo() {}
 
 LatencyInfo::LatencyInfo(int64_t trace_id, bool terminated)
     : trace_id_(trace_id),
-      ukm_source_id_(ukm::kInvalidSourceId),
       coalesced_(false),
       began_(false),
       terminated_(terminated),
@@ -180,12 +178,8 @@ void LatencyInfo::CopyLatencyFrom(const LatencyInfo& other,
                                   LatencyComponentType type) {
   // Don't clobber an existing trace_id_ or ukm_source_id_.
   if (trace_id_ == -1) {
-    DCHECK_EQ(ukm_source_id_, ukm::kInvalidSourceId);
     DCHECK(latency_components().empty());
     trace_id_ = other.trace_id();
-    ukm_source_id_ = other.ukm_source_id();
-  } else {
-    DCHECK_NE(ukm_source_id_, ukm::kInvalidSourceId);
   }
 
   for (const auto& lc : other.latency_components()) {
@@ -209,10 +203,6 @@ void LatencyInfo::AddNewLatencyFrom(const LatencyInfo& other) {
     trace_id_ = other.trace_id();
   }
 
-  if (ukm_source_id_ == ukm::kInvalidSourceId) {
-    ukm_source_id_ = other.ukm_source_id();
-  }
-
   for (const auto& lc : other.latency_components()) {
     if (!FindLatency(lc.first, nullptr)) {
       AddLatencyNumberWithTimestamp(lc.first, lc.second);
