diff --git a/ui/native_theme/native_theme_base.cc b/ui/native_theme/native_theme_base.cc
index 3047394c87c3..f55f88dec316 100644
--- a/ui/native_theme/native_theme_base.cc
+++ b/ui/native_theme/native_theme_base.cc
@@ -1590,11 +1590,11 @@ SkColor NativeThemeBase::GetControlColor(ControlColorId color_id,
     case kDisabledBorder:
       return SkColorSetARGB(0x4D, 0x76, 0x76, 0x76);
     case kAccent:
-      return SkColorSetRGB(0x00, 0x75, 0xFF);
+      return SkColorSetRGB(0x4E, 0x4E, 0x4E);
     case kHoveredAccent:
-      return SkColorSetRGB(0x00, 0x5C, 0xC8);
+      return SkColorSetRGB(0x61, 0x61, 0x61);
     case kPressedAccent:
-      return SkColorSetRGB(0x37, 0x93, 0xFF);
+      return SkColorSetRGB(0x83, 0x83, 0x83);
     case kDisabledAccent:
       return SkColorSetARGB(0x4D, 0x76, 0x76, 0x76);
     case kBackground:
@@ -1614,13 +1614,13 @@ SkColor NativeThemeBase::GetControlColor(ControlColorId color_id,
     case kProgressValue:
       return SkColorSetRGB(0x00, 0x75, 0xFF);
     case kSlider:
-      return SkColorSetRGB(0x00, 0x75, 0xFF);
+      return SkColorSetRGB(0x4E, 0x4E, 0x4E);
     case kHoveredSlider:
-      return SkColorSetRGB(0x00, 0x5C, 0xC8);
+      return SkColorSetRGB(0x61, 0x61, 0x61);
     case kPressedSlider:
-      return SkColorSetRGB(0x37, 0x93, 0xFF);
+      return SkColorSetRGB(0x8D, 0x8D, 0x8D);
     case kDisabledSlider:
-      return SkColorSetRGB(0xCB, 0xCB, 0xCB);
+      return SkColorSetARGB(0x4D, 0x76, 0x76, 0x76);
     case kAutoCompleteBackground:
       return SkColorSetRGB(0xE8, 0xF0, 0xFE);
     case kScrollbarArrowBackground:
