diff --git a/ui/native_theme/native_theme_base.cc b/ui/native_theme/native_theme_base.cc
index 313206e211c57..b9ba600b7b4ff 100644
--- a/ui/native_theme/native_theme_base.cc
+++ b/ui/native_theme/native_theme_base.cc
@@ -1718,11 +1718,11 @@ SkColor NativeThemeBase::GetControlColor(ControlColorId color_id,
     case kButtonDisabledBorder:
       return SkColorSetARGB(0x4D, 0x76, 0x76, 0x76);
     case kAccent:
-      return SkColorSetRGB(0x00, 0x75, 0xFF);
+      return SkColorSetRGB(0x63, 0x63, 0x63);
     case kHoveredAccent:
-      return SkColorSetRGB(0x00, 0x5C, 0xC8);
+      return SkColorSetRGB(0x4E, 0x4E, 0x4E);
     case kPressedAccent:
-      return SkColorSetRGB(0x37, 0x93, 0xFF);
+      return SkColorSetRGB(0x70, 0x70, 0x70);
     case kDisabledAccent:
       return SkColorSetARGB(0x4D, 0x76, 0x76, 0x76);
     case kBackground:
