diff --git a/ui/views/BUILD.gn b/ui/views/BUILD.gn
index 4f506ca30ca9..c1608e1416f1 100644
--- a/ui/views/BUILD.gn
+++ b/ui/views/BUILD.gn
@@ -595,7 +595,6 @@ component("views") {
     ]
     public_deps += [ "//components/remote_cocoa/common:mojo" ]
     deps += [
-      "//components/crash/core/common",
       "//components/remote_cocoa/app_shim",
       "//components/remote_cocoa/browser",
       "//ui/accelerated_widget_mac",
