diff --git a/ui/views/controls/textfield/textfield.cc b/ui/views/controls/textfield/textfield.cc
index 432332a7fadb..5d246407e289 100644
--- a/ui/views/controls/textfield/textfield.cc
+++ b/ui/views/controls/textfield/textfield.cc
@@ -1818,12 +1818,6 @@ void Textfield::SetTextEditCommandForNextKeyEvent(ui::TextEditCommand command) {
   scheduled_text_edit_command_ = command;
 }
 
-ukm::SourceId Textfield::GetClientSourceForMetrics() const {
-  // TODO(shend): Implement this method.
-  NOTIMPLEMENTED_LOG_ONCE();
-  return ukm::SourceId();
-}
-
 bool Textfield::ShouldDoLearning() {
   if (should_do_learning_.has_value())
     return should_do_learning_.value();
