diff --git a/ui/views/controls/prefix_selector.h b/ui/views/controls/prefix_selector.h
index 9df6de750fbe..8abe4763b769 100644
--- a/ui/views/controls/prefix_selector.h
+++ b/ui/views/controls/prefix_selector.h
@@ -73,7 +73,6 @@ class VIEWS_EXPORT PrefixSelector : public ui::TextInputClient {
 
   bool IsTextEditCommandEnabled(ui::TextEditCommand command) const override;
   void SetTextEditCommandForNextKeyEvent(ui::TextEditCommand command) override;
-  ukm::SourceId GetClientSourceForMetrics() const override;
   bool ShouldDoLearning() override;
 
 #if defined(OS_WIN) || defined(OS_CHROMEOS)
