diff --git a/ui/views/controls/prefix_selector.cc b/ui/views/controls/prefix_selector.cc
index c0cc92ea1804..9e3e01039b27 100644
--- a/ui/views/controls/prefix_selector.cc
+++ b/ui/views/controls/prefix_selector.cc
@@ -151,12 +151,6 @@ bool PrefixSelector::IsTextEditCommandEnabled(
 void PrefixSelector::SetTextEditCommandForNextKeyEvent(
     ui::TextEditCommand command) {}
 
-ukm::SourceId PrefixSelector::GetClientSourceForMetrics() const {
-  // TODO(shend): Implement this method.
-  NOTIMPLEMENTED_LOG_ONCE();
-  return ukm::SourceId();
-}
-
 bool PrefixSelector::ShouldDoLearning() {
   // TODO(https://crbug.com/311180): Implement this method.
   NOTIMPLEMENTED_LOG_ONCE();
