diff --git a/weblayer/browser/browser_main_parts_impl.cc b/weblayer/browser/browser_main_parts_impl.cc
index c738afc0de23..60fa50a47576 100644
--- a/weblayer/browser/browser_main_parts_impl.cc
+++ b/weblayer/browser/browser_main_parts_impl.cc
@@ -39,9 +39,6 @@
 
 #if defined(OS_ANDROID)
 #include "base/command_line.h"
-#include "components/crash/content/browser/child_exit_observer_android.h"
-#include "components/crash/content/browser/child_process_crash_observer_android.h"
-#include "components/crash/core/common/crash_key.h"
 #include "components/javascript_dialogs/android/app_modal_dialog_view_android.h"  // nogncheck
 #include "components/javascript_dialogs/app_modal_dialog_manager.h"  // nogncheck
 #include "content/public/browser/web_contents.h"
@@ -113,14 +110,6 @@ int BrowserMainPartsImpl::PreCreateThreads() {
   // Make sure permissions client has been set.
   WebLayerPermissionsClient::GetInstance();
 #if defined(OS_ANDROID)
-  // The ChildExitObserver needs to be created before any child process is
-  // created because it needs to be notified during process creation.
-  crash_reporter::ChildExitObserver::Create();
-  crash_reporter::ChildExitObserver::GetInstance()->RegisterClient(
-      std::make_unique<crash_reporter::ChildProcessCrashObserver>());
-
-  crash_reporter::InitializeCrashKeys();
-
   // MediaSession was implemented in M85, and requires both implementation and
   // client libraries to be at least that new. The version check has to be in
   // the browser process, but the command line flag is automatically propagated
