diff --git a/weblayer/app/content_main_delegate_impl.cc b/weblayer/app/content_main_delegate_impl.cc
index dcb4dd913233..71a4ec97e615 100644
--- a/weblayer/app/content_main_delegate_impl.cc
+++ b/weblayer/app/content_main_delegate_impl.cc
@@ -46,8 +46,6 @@
 #include "ui/base/ui_base_switches.h"
 #include "weblayer/browser/android/exception_filter.h"
 #include "weblayer/browser/android_descriptors.h"
-#include "weblayer/common/crash_reporter/crash_keys.h"
-#include "weblayer/common/crash_reporter/crash_reporter_client.h"
 #endif
 
 #if defined(OS_WIN)
