diff --git a/weblayer/browser/weblayer_impl_android.cc b/weblayer/browser/weblayer_impl_android.cc
index e6423f34ab38..60919aac5b79 100644
--- a/weblayer/browser/weblayer_impl_android.cc
+++ b/weblayer/browser/weblayer_impl_android.cc
@@ -7,7 +7,6 @@
 #include "base/android/jni_android.h"
 #include "base/android/jni_array.h"
 #include "base/android/jni_string.h"
-#include "components/crash/core/common/crash_key.h"
 #include "components/page_info/android/page_info_client.h"
 #include "weblayer/browser/android/metrics/weblayer_metrics_service_client.h"
 #include "weblayer/browser/default_search_engine.h"
@@ -15,7 +14,6 @@
 #include "weblayer/browser/java/jni/WebLayerImpl_jni.h"
 #include "weblayer/browser/url_bar/page_info_client_impl.h"
 #include "weblayer/browser/user_agent.h"
-#include "weblayer/common/crash_reporter/crash_keys.h"
 
 using base::android::JavaParamRef;
 
@@ -32,9 +30,6 @@ static jboolean JNI_WebLayerImpl_IsRemoteDebuggingEnabled(JNIEnv* env) {
 
 static void JNI_WebLayerImpl_SetIsWebViewCompatMode(JNIEnv* env,
                                                     jboolean value) {
-  static crash_reporter::CrashKeyString<1> crash_key(
-      crash_keys::kWeblayerWebViewCompatMode);
-  crash_key.Set(value ? "1" : "0");
 }
 
 static base::android::ScopedJavaLocalRef<jstring>
