diff --git a/weblayer/renderer/content_renderer_client_impl.cc b/weblayer/renderer/content_renderer_client_impl.cc
index 76ca9ee102e6..8232cb1c343f 100644
--- a/weblayer/renderer/content_renderer_client_impl.cc
+++ b/weblayer/renderer/content_renderer_client_impl.cc
@@ -12,7 +12,6 @@
 #include "components/error_page/common/error.h"
 #include "components/grit/components_scaled_resources.h"
 #include "components/js_injection/renderer/js_communication.h"
-#include "components/page_load_metrics/renderer/metrics_render_frame_observer.h"
 #include "components/prerender/common/prerender_types.mojom.h"
 #include "components/prerender/common/prerender_url_loader_throttle.h"
 #include "components/prerender/renderer/prerender_helper.h"
