diff --git a/weblayer/browser/browser_main_parts_impl.cc b/weblayer/browser/browser_main_parts_impl.cc
index c738afc0de23..99aa60527992 100644
--- a/weblayer/browser/browser_main_parts_impl.cc
+++ b/weblayer/browser/browser_main_parts_impl.cc
@@ -13,7 +13,6 @@
 #include "cc/base/switches.h"
 #include "components/captive_portal/core/buildflags.h"
 #include "components/prefs/pref_service.h"
-#include "components/startup_metric_utils/browser/startup_metric_utils.h"
 #include "components/translate/core/browser/translate_download_manager.h"
 #include "content/public/browser/browser_thread.h"
 #include "content/public/browser/devtools_agent_host.h"
@@ -192,12 +191,9 @@ void BrowserMainPartsImpl::PreMainMessageLoopRun() {
   // On Android, retrieve the application start time from Java and record it. On
   // other platforms, the application start time was already recorded in the
   // constructor of ContentMainDelegateImpl.
-  startup_metric_utils::RecordApplicationStartTime(GetApplicationStartTime());
 #endif  // defined(OS_ANDROID)
   // Record the time at which the main message loop starts. Must be recorded
   // after application start time (see startup_metric_utils.h).
-  startup_metric_utils::RecordBrowserMainMessageLoopStart(
-      base::TimeTicks::Now(), /* is_first_run */ false);
 
 #if defined(OS_ANDROID)
   memory_metrics_logger_ = std::make_unique<metrics::MemoryMetricsLogger>();
