diff --git a/weblayer/browser/autofill_client_impl.cc b/weblayer/browser/autofill_client_impl.cc
index 5abdd76b80da..2e32e613ef87 100644
--- a/weblayer/browser/autofill_client_impl.cc
+++ b/weblayer/browser/autofill_client_impl.cc
@@ -54,16 +54,6 @@ autofill::StrikeDatabase* AutofillClientImpl::GetStrikeDatabase() {
   return nullptr;
 }
 
-ukm::UkmRecorder* AutofillClientImpl::GetUkmRecorder() {
-  NOTREACHED();
-  return nullptr;
-}
-
-ukm::SourceId AutofillClientImpl::GetUkmSourceId() {
-  NOTREACHED();
-  return ukm::kInvalidSourceId;
-}
-
 autofill::AddressNormalizer* AutofillClientImpl::GetAddressNormalizer() {
   NOTREACHED();
   return nullptr;
