diff --git a/weblayer/browser/browser_impl.cc b/weblayer/browser/browser_impl.cc
index 4bf9dca0a301..88f79a4e85e0 100644
--- a/weblayer/browser/browser_impl.cc
+++ b/weblayer/browser/browser_impl.cc
@@ -31,7 +31,6 @@
 #include "base/android/jni_string.h"
 #include "base/json/json_writer.h"
 #include "components/metrics/metrics_service.h"
-#include "components/ukm/ukm_service.h"
 #include "weblayer/browser/android/metrics/weblayer_metrics_service_client.h"
 #include "weblayer/browser/browser_process.h"
 #include "weblayer/browser/java/jni/BrowserImpl_jni.h"
@@ -66,15 +65,6 @@ void UpdateMetricsService() {
     else
       metrics_service->OnAppEnterBackground();
   }
-
-  auto* ukm_service =
-      WebLayerMetricsServiceClient::GetInstance()->GetUkmService();
-  if (ukm_service) {
-    if (foreground)
-      ukm_service->OnAppEnterForeground();
-    else
-      ukm_service->OnAppEnterBackground();
-  }
 }
 #endif  // defined(OS_ANDROID)
 
