diff --git a/weblayer/BUILD.gn b/weblayer/BUILD.gn
index 4143f8bc79cb..eda1fd1f0d6a 100644
--- a/weblayer/BUILD.gn
+++ b/weblayer/BUILD.gn
@@ -373,7 +373,6 @@ source_set("weblayer_lib_base") {
     "//components/autofill/core/common",
     "//components/background_sync",
     "//components/base32",
-    "//components/blocked_content",
     "//components/browsing_data/content",
     "//components/captive_portal/core:buildflags",
     "//components/cdm/renderer",
