diff --git a/weblayer/browser/content_browser_client_impl.h b/weblayer/browser/content_browser_client_impl.h
index b1a565bea42a..3933de049246 100644
--- a/weblayer/browser/content_browser_client_impl.h
+++ b/weblayer/browser/content_browser_client_impl.h
@@ -154,7 +154,6 @@ class ContentBrowserClientImpl : public content::ContentBrowserClient {
 #endif  // OS_ANDROID
   content::SpeechRecognitionManagerDelegate*
   CreateSpeechRecognitionManagerDelegate() override;
-  ukm::UkmService* GetUkmService() override;
 
   void CreateFeatureListAndFieldTrials();
 
