diff --git a/weblayer/browser/browser_main_parts_impl.cc b/weblayer/browser/browser_main_parts_impl.cc
index c8cf179bf4f0..4edad6c61ed8 100644
--- a/weblayer/browser/browser_main_parts_impl.cc
+++ b/weblayer/browser/browser_main_parts_impl.cc
@@ -45,9 +45,6 @@
 
 #if defined(OS_ANDROID)
 #include "base/command_line.h"
-#include "components/crash/content/browser/child_exit_observer_android.h"
-#include "components/crash/content/browser/child_process_crash_observer_android.h"
-#include "components/crash/core/common/crash_key.h"
 #include "components/javascript_dialogs/android/app_modal_dialog_view_android.h"  // nogncheck
 #include "components/javascript_dialogs/app_modal_dialog_manager.h"  // nogncheck
 #include "components/metrics/metrics_service.h"
