diff --git a/weblayer/browser/content_browser_client_impl.h b/weblayer/browser/content_browser_client_impl.h
index 967ff9294a9e..9ff828fecce1 100644
--- a/weblayer/browser/content_browser_client_impl.h
+++ b/weblayer/browser/content_browser_client_impl.h
@@ -137,7 +137,6 @@ class ContentBrowserClientImpl : public content::ContentBrowserClient {
 #endif  // OS_ANDROID
   content::SpeechRecognitionManagerDelegate*
   CreateSpeechRecognitionManagerDelegate() override;
-  ukm::UkmService* GetUkmService() override;
 
   void CreateFeatureListAndFieldTrials();
 
