diff --git a/weblayer/test/BUILD.gn b/weblayer/test/BUILD.gn
index 60f7b969ebc8..1134e7594a21 100644
--- a/weblayer/test/BUILD.gn
+++ b/weblayer/test/BUILD.gn
@@ -102,7 +102,6 @@ test("weblayer_browsertests") {
     "//components/favicon/content",
     "//components/network_session_configurator/common",
     "//components/network_time",
-    "//components/page_load_metrics/browser:browser",
     "//components/prefs",
     "//components/prerender/browser",
     "//components/security_interstitials/content:security_interstitial_page",
@@ -143,7 +142,6 @@ test("weblayer_browsertests") {
     "../browser/navigation_browsertest.cc",
     "../browser/navigation_error_navigation_throttle_browsertest.cc",
     "../browser/no_state_prefetch/no_state_prefetch_browsertest.cc",
-    "../browser/page_load_metrics_browsertest.cc",
     "../browser/popup_blocker_browsertest.cc",
     "../browser/prefetch_browsertest.cc",
     "../browser/profile_browsertest.cc",
@@ -202,7 +200,6 @@ test("weblayer_browsertests") {
       "//content/public/test/android:android_test_message_pump_support_java",
       "//content/test:android_test_message_pump_support",
       "//content/test:android_test_message_pump_support",
-      "//services/metrics/public/cpp:ukm_builders",
       "//services/tracing:test_utils",
       "//testing/android/native_test:native_test_support",
       "//third_party/metrics_proto",
