diff --git a/weblayer/browser/content_browser_client_impl.cc b/weblayer/browser/content_browser_client_impl.cc
index 7511ea0b467e..bdc51b069290 100644
--- a/weblayer/browser/content_browser_client_impl.cc
+++ b/weblayer/browser/content_browser_client_impl.cc
@@ -987,12 +987,4 @@ ContentBrowserClientImpl::CreateSpeechRecognitionManagerDelegate() {
   return new WebLayerSpeechRecognitionManagerDelegate();
 }
 
-ukm::UkmService* ContentBrowserClientImpl::GetUkmService() {
-#if defined(OS_ANDROID)
-  return WebLayerMetricsServiceClient::GetInstance()->GetUkmService();
-#else
-  return nullptr;
-#endif
-}
-
 }  // namespace weblayer
