diff --git a/weblayer/BUILD.gn b/weblayer/BUILD.gn
index ecf8ce7c3ebb..652cd15c1d50 100644
--- a/weblayer/BUILD.gn
+++ b/weblayer/BUILD.gn
@@ -404,8 +404,6 @@ source_set("weblayer_lib_base") {
     "//components/no_state_prefetch/renderer",
     "//components/omnibox/browser:location_bar",
     "//components/page_info",
-    "//components/page_load_metrics/browser",
-    "//components/page_load_metrics/renderer",
     "//components/password_manager/content/browser",
     "//components/permissions",
     "//components/policy/core/browser",
@@ -432,8 +430,6 @@ source_set("weblayer_lib_base") {
     "//components/translate/content/renderer",
     "//components/translate/core/browser",
     "//components/translate/core/common",
-    "//components/ukm",
-    "//components/ukm/content",
     "//components/unified_consent:unified_consent",
     "//components/user_prefs",
     "//components/variations",
@@ -459,7 +455,6 @@ source_set("weblayer_lib_base") {
     "//net",
     "//net:net_resources",
     "//sandbox",
-    "//services/metrics/public/cpp:metrics_cpp",
     "//services/network/public/cpp",
     "//services/network/public/mojom",
     "//services/preferences/tracked",
@@ -592,10 +587,6 @@ source_set("weblayer_lib_base") {
       "browser/weblayer_impl_android.h",
       "browser/webrtc/media_stream_manager.cc",
       "browser/webrtc/media_stream_manager.h",
-      "common/crash_reporter/crash_keys.cc",
-      "common/crash_reporter/crash_keys.h",
-      "common/crash_reporter/crash_reporter_client.cc",
-      "common/crash_reporter/crash_reporter_client.h",
     ]
     deps += [
       "//android_webview:generate_aw_resources",
@@ -609,8 +600,6 @@ source_set("weblayer_lib_base") {
       "//components/browser_ui/sms/android",
       "//components/cdm/browser",
       "//components/content_settings/android",
-      "//components/crash/android:crash_android",
-      "//components/crash/android:crashpad_main",
       "//components/embedder_support/android:browser_context",
       "//components/embedder_support/android:context_menu",
       "//components/embedder_support/android:util",
