diff --git a/weblayer/test/BUILD.gn b/weblayer/test/BUILD.gn
index a3aa76288787..47ce86e8accb 100644
--- a/weblayer/test/BUILD.gn
+++ b/weblayer/test/BUILD.gn
@@ -154,7 +154,6 @@ test("weblayer_browsertests") {
     "../browser/navigation_error_navigation_throttle_browsertest.cc",
     "../browser/new_tab_delegate_browsertest.cc",
     "../browser/no_state_prefetch/no_state_prefetch_browsertest.cc",
-    "../browser/page_load_metrics_browsertest.cc",
     "../browser/popup_blocker_browsertest.cc",
     "../browser/prefetch_browsertest.cc",
     "../browser/profile_browsertest.cc",
@@ -213,7 +212,6 @@ test("weblayer_browsertests") {
       "//content/public/test/android:android_test_message_pump_support_java",
       "//content/test:android_test_message_pump_support",
       "//content/test:android_test_message_pump_support",
-      "//services/metrics/public/cpp:ukm_builders",
       "//services/tracing:test_utils",
       "//testing/android/native_test:native_test_support",
       "//third_party/metrics_proto",
