diff --git a/weblayer/app/content_main_delegate_impl.cc b/weblayer/app/content_main_delegate_impl.cc
index 9b81005db1f7..f8c472fc7d0f 100644
--- a/weblayer/app/content_main_delegate_impl.cc
+++ b/weblayer/app/content_main_delegate_impl.cc
@@ -14,7 +14,6 @@
 #include "base/logging.h"
 #include "base/path_service.h"
 #include "build/build_config.h"
-#include "components/startup_metric_utils/browser/startup_metric_utils.h"
 #include "content/public/browser/browser_main_runner.h"
 #include "content/public/common/content_features.h"
 #include "content/public/common/content_switches.h"
@@ -126,7 +125,6 @@ ContentMainDelegateImpl::ContentMainDelegateImpl(MainParams params)
   // start time is sampled when the Java code is entered, and it is retrieved
   // from C++ after initializing the JNI (see
   // BrowserMainPartsImpl::PreMainMessageLoopRun()).
-  startup_metric_utils::RecordApplicationStartTime(base::TimeTicks::Now());
 #endif
 }
 
