diff --git a/weblayer/BUILD.gn b/weblayer/BUILD.gn
index e86bccceef46..a04044f14f81 100644
--- a/weblayer/BUILD.gn
+++ b/weblayer/BUILD.gn
@@ -417,7 +417,6 @@ source_set("weblayer_lib_base") {
     "//components/site_isolation",
     "//components/spellcheck:buildflags",
     "//components/ssl_errors",
-    "//components/startup_metric_utils/browser",
     "//components/strings",
     "//components/translate/content/browser",
     "//components/translate/content/renderer",
