--- a/chrome/browser/themes/theme_properties.cc
+++ b/chrome/browser/themes/theme_properties.cc
@@ -49,23 +49,26 @@ SkColor GetLightModeColor(int id) {
   constexpr SkColor kDefaultColorNTPLink = SkColorSetRGB(0x06, 0x37, 0x74);
 #endif  // OS_WIN
 
+  constexpr SkColor kLightToolbar = SkColorSetRGB(247, 247, 247);
+  constexpr SkColor kLightFrame = SkColorSetRGB(206, 206, 206);
+
   switch (id) {
     // Properties stored in theme pack.  If you change these defaults, you must
     // increment the version number in browser_theme_pack.cc.
     case ThemeProperties::COLOR_FRAME:
     case ThemeProperties::COLOR_BACKGROUND_TAB:
-      return SkColorSetRGB(0xDE, 0xE1, 0xE6);
+    case ThemeProperties::COLOR_TOOLBAR_CONTENT_AREA_SEPARATOR:
+      return kLightFrame;
     case ThemeProperties::COLOR_FRAME_INACTIVE:
     case ThemeProperties::COLOR_BACKGROUND_TAB_INACTIVE:
-      return color_utils::HSLShift(
-          GetLightModeColor(ThemeProperties::COLOR_FRAME),
-          ThemeProperties::GetDefaultTint(ThemeProperties::TINT_FRAME_INACTIVE,
-                                          false));
+      return color_utils::HSLShift(kLightFrame, { -1, -1, 0.6 });
     case ThemeProperties::COLOR_DOWNLOAD_SHELF:
     case ThemeProperties::COLOR_INFOBAR:
     case ThemeProperties::COLOR_TOOLBAR:
     case ThemeProperties::COLOR_STATUS_BUBBLE:
-      return SK_ColorWHITE;
+    case ThemeProperties::COLOR_TOOLBAR_TOP_SEPARATOR:
+    case ThemeProperties::COLOR_TOOLBAR_TOP_SEPARATOR_INACTIVE:
+      return kLightToolbar;
     case ThemeProperties::COLOR_HOVER_CARD_NO_PREVIEW_FOREGROUND:
       return gfx::kGoogleGrey300;
     case ThemeProperties::COLOR_HOVER_CARD_NO_PREVIEW_BACKGROUND:
@@ -103,11 +106,6 @@ SkColor GetLightModeColor(int id) {
     case ThemeProperties::COLOR_TAB_ALERT_CAPTURING:
     case ThemeProperties::COLOR_TAB_PIP_PLAYING:
       return gfx::kGoogleBlue600;
-    case ThemeProperties::COLOR_TOOLBAR_CONTENT_AREA_SEPARATOR:
-      return SkColorSetRGB(0xB6, 0xB4, 0xB6);
-    case ThemeProperties::COLOR_TOOLBAR_TOP_SEPARATOR:
-    case ThemeProperties::COLOR_TOOLBAR_TOP_SEPARATOR_INACTIVE:
-      return SkColorSetA(SK_ColorBLACK, 0x40);
 #if defined(OS_WIN)
     case ThemeProperties::COLOR_ACCENT_BORDER:
       NOTREACHED();
