diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 817eb3acd82f..4dd7008ec76d 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2032,6 +2032,7 @@ static_library("browser") {
     "//components/download/content/factory",
     "//components/download/database",
     "//components/download/public/background_service:public",
+    "//components/dumb_center:manager",
     "//components/embedder_support",
     "//components/encrypted_messages",
     "//components/error_page/common",
diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index ef167ee8b5cb..3eb1db2dcf78 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -56,6 +56,7 @@ jumbo_source_set("browser") {
     "//components/discardable_memory/service",
     "//components/download/database",
     "//components/download/public/common:public",
+    "//components/dumb_center:manager",
     "//components/filename_generation",
     "//components/link_header_util",
     "//components/metrics",
diff --git a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
index b36da920302f..fcf84c561e1b 100644
--- a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
+++ b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
@@ -10,6 +10,7 @@
 
 #include "base/metrics/histogram_functions.h"
 #include "base/numerics/checked_math.h"
+#include "components/dumb_center/dumb_center.h"
 #include "third_party/blink/public/common/features.h"
 #include "third_party/blink/renderer/core/css/cssom/css_url_image_value.h"
 #include "third_party/blink/renderer/core/css/parser/css_parser.h"
@@ -802,6 +803,10 @@ bool BaseRenderingContext2D::IsPointInPathInternal(
     const double x,
     const double y,
     const String& winding_rule_string) {
+  if(!dumb_center::AllowFingerprinting()) {
+    return false;
+  }
+
   cc::PaintCanvas* c = GetOrCreatePaintCanvas();
   if (!c)
     return false;
@@ -831,6 +836,10 @@ bool BaseRenderingContext2D::isPointInStroke(Path2D* dom_path,
 bool BaseRenderingContext2D::IsPointInStrokeInternal(const Path& path,
                                                      const double x,
                                                      const double y) {
+  if(!dumb_center::AllowFingerprinting()) {
+    return false;
+  }
+
   cc::PaintCanvas* c = GetOrCreatePaintCanvas();
   if (!c)
     return false;
@@ -1555,6 +1564,10 @@ ImageData* BaseRenderingContext2D::getImageData(
     int sw,
     int sh,
     ExceptionState& exception_state) {
+  if(!dumb_center::AllowFingerprinting()) {
+    return nullptr;
+  }
+
   if (!base::CheckMul(sw, sh).IsValid<int>()) {
     exception_state.ThrowRangeError("Out of memory at ImageData creation");
     return nullptr;
diff --git a/third_party/blink/renderer/modules/canvas/canvas2d/canvas_rendering_context_2d.cc b/third_party/blink/renderer/modules/canvas/canvas2d/canvas_rendering_context_2d.cc
index e916fc83dabc..23e6078fddcb 100644
--- a/third_party/blink/renderer/modules/canvas/canvas2d/canvas_rendering_context_2d.cc
+++ b/third_party/blink/renderer/modules/canvas/canvas2d/canvas_rendering_context_2d.cc
@@ -35,6 +35,7 @@
 
 #include "base/metrics/histogram_functions.h"
 #include "base/rand_util.h"
+#include "components/dumb_center/dumb_center.h"
 #include "third_party/blink/public/common/features.h"
 #include "third_party/blink/public/platform/platform.h"
 #include "third_party/blink/public/platform/task_type.h"
@@ -799,6 +800,10 @@ void CanvasRenderingContext2D::strokeText(const String& text,
 }
 
 TextMetrics* CanvasRenderingContext2D::measureText(const String& text) {
+  if(!dumb_center::AllowFingerprinting()) {
+    return MakeGarbageCollected<TextMetrics>();
+  }
+
   // The style resolution required for fonts is not available in frame-less
   // documents.
   if (!canvas()->GetDocument().GetFrame())
diff --git a/third_party/blink/renderer/platform/BUILD.gn b/third_party/blink/renderer/platform/BUILD.gn
index b34305f0ee0d..98f8735e21d1 100644
--- a/third_party/blink/renderer/platform/BUILD.gn
+++ b/third_party/blink/renderer/platform/BUILD.gn
@@ -1563,6 +1563,7 @@ jumbo_component("platform") {
   deps = [
     "//base/allocator:buildflags",
     "//cc/ipc",
+    "//components/dumb_center:manager",
     "//components/paint_preview/common",
     "//components/viz/client",
     "//components/viz/common",
