--- a/content/browser/browser_main_loop.cc
+++ b/content/browser/browser_main_loop.cc
@@ -1113,12 +1113,6 @@ void BrowserMainLoop::ShutdownThreadsAnd
     }
   }
 
-  // Must happen after the I/O thread is shutdown since this class lives on the
-  // I/O thread and isn't threadsafe.
-  {
-    TRACE_EVENT0("shutdown", "BrowserMainLoop::Subsystem:GamepadService");
-    device::GamepadService::GetInstance()->Terminate();
-  }
   {
     TRACE_EVENT0("shutdown", "BrowserMainLoop::Subsystem:DeleteDataSources");
     URLDataManager::DeleteDataSources();
@@ -1259,16 +1253,6 @@ int BrowserMainLoop::BrowserThreadsStart
     midi_service_.reset(new midi::MidiService);
   }
 
-  {
-    TRACE_EVENT0("startup", "BrowserThreadsStarted::Subsystem:Devices");
-    device::GamepadService::GetInstance()->StartUp(
-        base::BindRepeating(&BindHidManager));
-#if !defined(OS_ANDROID)
-    device::FidoHidDiscovery::SetHidManagerBinder(
-        base::BindRepeating(&BindHidManager));
-#endif
-  }
-
 #if defined(OS_WIN)
   system_message_window_.reset(new media::SystemMessageWindowWin);
 #elif defined(OS_LINUX) && defined(USE_UDEV)
